{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "gpuType": "T4"
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU",
    "gpuClass": "standard"
  },
  "cells": [
    {
      "cell_type": "code",
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/gdrive')\n",
        "\n",
        "# Now you can access your Google Drive as a local file system\n",
        "with open('/content/CollatedText.txt', 'r') as file:\n",
        "    corpus = [line.strip() for line in file]\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yDbNwuejJYsR",
        "outputId": "afd05f80-4645-4664-8f39-fe1b37ec0fd6"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Drive already mounted at /content/gdrive; to attempt to forcibly remount, call drive.mount(\"/content/gdrive\", force_remount=True).\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# -*- coding: utf-8 -*-\n",
        "import os\n",
        "os.environ[\"LC_ALL\"] = \"en_US.utf8\"\n",
        "os.environ[\"LANG\"] = \"en_US.utf8\""
      ],
      "metadata": {
        "id": "ds4kfSZDC_rX"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "\n",
        "# Install necessary packages\n",
        "!pip install bertopic==0.14.1\n",
        "!pip install -U pyldavis==3.4.1\n",
        "!pip install numpy==1.23.5\n",
        "!pip install --upgrade gensim nltk numpy\n",
        "!pip install --upgrade gensim\n",
        "!pip install --upgrade nltk\n",
        "!pip install sklearn"
      ],
      "metadata": {
        "id": "HJ89X-1J0_RI",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "f18f45cd-a972-4477-be55-e06c6446ab3b"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting bertopic==0.14.1\n",
            "  Downloading bertopic-0.14.1-py2.py3-none-any.whl (120 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m120.7/120.7 kB\u001b[0m \u001b[31m2.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: numpy>=1.20.0 in /usr/local/lib/python3.10/dist-packages (from bertopic==0.14.1) (1.22.4)\n",
            "Collecting hdbscan>=0.8.29 (from bertopic==0.14.1)\n",
            "  Downloading hdbscan-0.8.29.tar.gz (5.2 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m5.2/5.2 MB\u001b[0m \u001b[31m35.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Installing build dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Getting requirements to build wheel ... \u001b[?25l\u001b[?25hdone\n",
            "  Preparing metadata (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "Collecting umap-learn>=0.5.0 (from bertopic==0.14.1)\n",
            "  Downloading umap-learn-0.5.3.tar.gz (88 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m88.2/88.2 kB\u001b[0m \u001b[31m6.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: pandas>=1.1.5 in /usr/local/lib/python3.10/dist-packages (from bertopic==0.14.1) (1.5.3)\n",
            "Requirement already satisfied: scikit-learn>=0.22.2.post1 in /usr/local/lib/python3.10/dist-packages (from bertopic==0.14.1) (1.2.2)\n",
            "Requirement already satisfied: tqdm>=4.41.1 in /usr/local/lib/python3.10/dist-packages (from bertopic==0.14.1) (4.65.0)\n",
            "Collecting sentence-transformers>=0.4.1 (from bertopic==0.14.1)\n",
            "  Downloading sentence-transformers-2.2.2.tar.gz (85 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m86.0/86.0 kB\u001b[0m \u001b[31m7.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: plotly>=4.7.0 in /usr/local/lib/python3.10/dist-packages (from bertopic==0.14.1) (5.13.1)\n",
            "Requirement already satisfied: cython>=0.27 in /usr/local/lib/python3.10/dist-packages (from hdbscan>=0.8.29->bertopic==0.14.1) (0.29.34)\n",
            "Requirement already satisfied: scipy>=1.0 in /usr/local/lib/python3.10/dist-packages (from hdbscan>=0.8.29->bertopic==0.14.1) (1.10.1)\n",
            "Requirement already satisfied: joblib>=1.0 in /usr/local/lib/python3.10/dist-packages (from hdbscan>=0.8.29->bertopic==0.14.1) (1.2.0)\n",
            "Requirement already satisfied: python-dateutil>=2.8.1 in /usr/local/lib/python3.10/dist-packages (from pandas>=1.1.5->bertopic==0.14.1) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas>=1.1.5->bertopic==0.14.1) (2022.7.1)\n",
            "Requirement already satisfied: tenacity>=6.2.0 in /usr/local/lib/python3.10/dist-packages (from plotly>=4.7.0->bertopic==0.14.1) (8.2.2)\n",
            "Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from scikit-learn>=0.22.2.post1->bertopic==0.14.1) (3.1.0)\n",
            "Collecting transformers<5.0.0,>=4.6.0 (from sentence-transformers>=0.4.1->bertopic==0.14.1)\n",
            "  Downloading transformers-4.29.2-py3-none-any.whl (7.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m7.1/7.1 MB\u001b[0m \u001b[31m86.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: torch>=1.6.0 in /usr/local/lib/python3.10/dist-packages (from sentence-transformers>=0.4.1->bertopic==0.14.1) (2.0.1+cu118)\n",
            "Requirement already satisfied: torchvision in /usr/local/lib/python3.10/dist-packages (from sentence-transformers>=0.4.1->bertopic==0.14.1) (0.15.2+cu118)\n",
            "Requirement already satisfied: nltk in /usr/local/lib/python3.10/dist-packages (from sentence-transformers>=0.4.1->bertopic==0.14.1) (3.8.1)\n",
            "Collecting sentencepiece (from sentence-transformers>=0.4.1->bertopic==0.14.1)\n",
            "  Downloading sentencepiece-0.1.99-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.3/1.3 MB\u001b[0m \u001b[31m53.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting huggingface-hub>=0.4.0 (from sentence-transformers>=0.4.1->bertopic==0.14.1)\n",
            "  Downloading huggingface_hub-0.14.1-py3-none-any.whl (224 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m224.5/224.5 kB\u001b[0m \u001b[31m18.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: numba>=0.49 in /usr/local/lib/python3.10/dist-packages (from umap-learn>=0.5.0->bertopic==0.14.1) (0.56.4)\n",
            "Collecting pynndescent>=0.5 (from umap-learn>=0.5.0->bertopic==0.14.1)\n",
            "  Downloading pynndescent-0.5.10.tar.gz (1.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.1/1.1 MB\u001b[0m \u001b[31m51.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (3.12.0)\n",
            "Requirement already satisfied: fsspec in /usr/local/lib/python3.10/dist-packages (from huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (2023.4.0)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (2.27.1)\n",
            "Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (6.0)\n",
            "Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (4.5.0)\n",
            "Requirement already satisfied: packaging>=20.9 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (23.1)\n",
            "Requirement already satisfied: llvmlite<0.40,>=0.39.0dev0 in /usr/local/lib/python3.10/dist-packages (from numba>=0.49->umap-learn>=0.5.0->bertopic==0.14.1) (0.39.1)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from numba>=0.49->umap-learn>=0.5.0->bertopic==0.14.1) (67.7.2)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.1->pandas>=1.1.5->bertopic==0.14.1) (1.16.0)\n",
            "Requirement already satisfied: sympy in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (1.11.1)\n",
            "Requirement already satisfied: networkx in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (3.1)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (3.1.2)\n",
            "Requirement already satisfied: triton==2.0.0 in /usr/local/lib/python3.10/dist-packages (from torch>=1.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (2.0.0)\n",
            "Requirement already satisfied: cmake in /usr/local/lib/python3.10/dist-packages (from triton==2.0.0->torch>=1.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (3.25.2)\n",
            "Requirement already satisfied: lit in /usr/local/lib/python3.10/dist-packages (from triton==2.0.0->torch>=1.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (16.0.5)\n",
            "Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers<5.0.0,>=4.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (2022.10.31)\n",
            "Collecting tokenizers!=0.11.3,<0.14,>=0.11.1 (from transformers<5.0.0,>=4.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1)\n",
            "  Downloading tokenizers-0.13.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (7.8 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m7.8/7.8 MB\u001b[0m \u001b[31m69.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: click in /usr/local/lib/python3.10/dist-packages (from nltk->sentence-transformers>=0.4.1->bertopic==0.14.1) (8.1.3)\n",
            "Requirement already satisfied: pillow!=8.3.*,>=5.3.0 in /usr/local/lib/python3.10/dist-packages (from torchvision->sentence-transformers>=0.4.1->bertopic==0.14.1) (8.4.0)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->torch>=1.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (2.1.2)\n",
            "Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (1.26.15)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (2022.12.7)\n",
            "Requirement already satisfied: charset-normalizer~=2.0.0 in /usr/local/lib/python3.10/dist-packages (from requests->huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (2.0.12)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->huggingface-hub>=0.4.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (3.4)\n",
            "Requirement already satisfied: mpmath>=0.19 in /usr/local/lib/python3.10/dist-packages (from sympy->torch>=1.6.0->sentence-transformers>=0.4.1->bertopic==0.14.1) (1.3.0)\n",
            "Building wheels for collected packages: hdbscan, sentence-transformers, umap-learn, pynndescent\n",
            "  Building wheel for hdbscan (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for hdbscan: filename=hdbscan-0.8.29-cp310-cp310-linux_x86_64.whl size=3541439 sha256=f7ae8f283216b706652009b79cb9fccf41ede9762abc66274de83b1826eeb55d\n",
            "  Stored in directory: /root/.cache/pip/wheels/dc/52/e3/6c6b60b126b4d5c4370cb5ac071b82950f91649d62d72f7f56\n",
            "  Building wheel for sentence-transformers (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for sentence-transformers: filename=sentence_transformers-2.2.2-py3-none-any.whl size=125926 sha256=33572d8be8e4f4209609cabe9aaf7b9de6f2195a78f8d4da0dc14e8e1a0a32fd\n",
            "  Stored in directory: /root/.cache/pip/wheels/62/f2/10/1e606fd5f02395388f74e7462910fe851042f97238cbbd902f\n",
            "  Building wheel for umap-learn (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for umap-learn: filename=umap_learn-0.5.3-py3-none-any.whl size=82816 sha256=e7c665a7a281c37664f4e27f3d85d8d4e6808762fd484c2847eb8e2b24cf7c0f\n",
            "  Stored in directory: /root/.cache/pip/wheels/a0/e8/c6/a37ea663620bd5200ea1ba0907ab3c217042c1d035ef606acc\n",
            "  Building wheel for pynndescent (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for pynndescent: filename=pynndescent-0.5.10-py3-none-any.whl size=55622 sha256=866823609da1ae5744909e2d4b7ccab587688366c09722c27b4bc2fabf4facf5\n",
            "  Stored in directory: /root/.cache/pip/wheels/4a/38/5d/f60a40a66a9512b7e5e83517ebc2d1b42d857be97d135f1096\n",
            "Successfully built hdbscan sentence-transformers umap-learn pynndescent\n",
            "Installing collected packages: tokenizers, sentencepiece, huggingface-hub, transformers, pynndescent, hdbscan, umap-learn, sentence-transformers, bertopic\n",
            "Successfully installed bertopic-0.14.1 hdbscan-0.8.29 huggingface-hub-0.14.1 pynndescent-0.5.10 sentence-transformers-2.2.2 sentencepiece-0.1.99 tokenizers-0.13.3 transformers-4.29.2 umap-learn-0.5.3\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting pyldavis==3.4.1\n",
            "  Downloading pyLDAvis-3.4.1-py3-none-any.whl (2.6 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.6/2.6 MB\u001b[0m \u001b[31m26.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting numpy>=1.24.2 (from pyldavis==3.4.1)\n",
            "  Downloading numpy-1.24.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m17.3/17.3 MB\u001b[0m \u001b[31m41.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: scipy in /usr/local/lib/python3.10/dist-packages (from pyldavis==3.4.1) (1.10.1)\n",
            "Collecting pandas>=2.0.0 (from pyldavis==3.4.1)\n",
            "  Downloading pandas-2.0.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m12.3/12.3 MB\u001b[0m \u001b[31m54.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: joblib>=1.2.0 in /usr/local/lib/python3.10/dist-packages (from pyldavis==3.4.1) (1.2.0)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from pyldavis==3.4.1) (3.1.2)\n",
            "Requirement already satisfied: numexpr in /usr/local/lib/python3.10/dist-packages (from pyldavis==3.4.1) (2.8.4)\n",
            "Collecting funcy (from pyldavis==3.4.1)\n",
            "  Downloading funcy-2.0-py2.py3-none-any.whl (30 kB)\n",
            "Requirement already satisfied: scikit-learn>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from pyldavis==3.4.1) (1.2.2)\n",
            "Requirement already satisfied: gensim in /usr/local/lib/python3.10/dist-packages (from pyldavis==3.4.1) (4.3.1)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from pyldavis==3.4.1) (67.7.2)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.10/dist-packages (from pandas>=2.0.0->pyldavis==3.4.1) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas>=2.0.0->pyldavis==3.4.1) (2022.7.1)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.10/dist-packages (from pandas>=2.0.0->pyldavis==3.4.1) (2023.3)\n",
            "Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from scikit-learn>=1.0.0->pyldavis==3.4.1) (3.1.0)\n",
            "Requirement already satisfied: smart-open>=1.8.1 in /usr/local/lib/python3.10/dist-packages (from gensim->pyldavis==3.4.1) (6.3.0)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->pyldavis==3.4.1) (2.1.2)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.2->pandas>=2.0.0->pyldavis==3.4.1) (1.16.0)\n",
            "Installing collected packages: funcy, numpy, pandas, pyldavis\n",
            "  Attempting uninstall: numpy\n",
            "    Found existing installation: numpy 1.22.4\n",
            "    Uninstalling numpy-1.22.4:\n",
            "      Successfully uninstalled numpy-1.22.4\n",
            "  Attempting uninstall: pandas\n",
            "    Found existing installation: pandas 1.5.3\n",
            "    Uninstalling pandas-1.5.3:\n",
            "      Successfully uninstalled pandas-1.5.3\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "google-colab 1.0.0 requires pandas==1.5.3, but you have pandas 2.0.1 which is incompatible.\n",
            "numba 0.56.4 requires numpy<1.24,>=1.18, but you have numpy 1.24.3 which is incompatible.\n",
            "tensorflow 2.12.0 requires numpy<1.24,>=1.22, but you have numpy 1.24.3 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed funcy-2.0 numpy-1.24.3 pandas-2.0.1 pyldavis-3.4.1\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.colab-display-data+json": {
              "pip_warning": {
                "packages": [
                  "numpy"
                ]
              }
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting numpy==1.23.5\n",
            "  Downloading numpy-1.23.5-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m17.1/17.1 MB\u001b[0m \u001b[31m65.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: numpy\n",
            "  Attempting uninstall: numpy\n",
            "    Found existing installation: numpy 1.24.3\n",
            "    Uninstalling numpy-1.24.3:\n",
            "      Successfully uninstalled numpy-1.24.3\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "pyldavis 3.4.1 requires numpy>=1.24.2, but you have numpy 1.23.5 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed numpy-1.23.5\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.colab-display-data+json": {
              "pip_warning": {
                "packages": [
                  "numpy"
                ]
              }
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: gensim in /usr/local/lib/python3.10/dist-packages (4.3.1)\n",
            "Requirement already satisfied: nltk in /usr/local/lib/python3.10/dist-packages (3.8.1)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (1.23.5)\n",
            "Collecting numpy\n",
            "  Using cached numpy-1.24.3-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)\n",
            "Requirement already satisfied: scipy>=1.7.0 in /usr/local/lib/python3.10/dist-packages (from gensim) (1.10.1)\n",
            "Requirement already satisfied: smart-open>=1.8.1 in /usr/local/lib/python3.10/dist-packages (from gensim) (6.3.0)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.10/dist-packages (from nltk) (8.1.3)\n",
            "Requirement already satisfied: joblib in /usr/local/lib/python3.10/dist-packages (from nltk) (1.2.0)\n",
            "Requirement already satisfied: regex>=2021.8.3 in /usr/local/lib/python3.10/dist-packages (from nltk) (2022.10.31)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from nltk) (4.65.0)\n",
            "Installing collected packages: numpy\n",
            "  Attempting uninstall: numpy\n",
            "    Found existing installation: numpy 1.23.5\n",
            "    Uninstalling numpy-1.23.5:\n",
            "      Successfully uninstalled numpy-1.23.5\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "numba 0.56.4 requires numpy<1.24,>=1.18, but you have numpy 1.24.3 which is incompatible.\n",
            "tensorflow 2.12.0 requires numpy<1.24,>=1.22, but you have numpy 1.24.3 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed numpy-1.24.3\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/vnd.colab-display-data+json": {
              "pip_warning": {
                "packages": [
                  "numpy"
                ]
              }
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: gensim in /usr/local/lib/python3.10/dist-packages (4.3.1)\n",
            "Requirement already satisfied: numpy>=1.18.5 in /usr/local/lib/python3.10/dist-packages (from gensim) (1.24.3)\n",
            "Requirement already satisfied: scipy>=1.7.0 in /usr/local/lib/python3.10/dist-packages (from gensim) (1.10.1)\n",
            "Requirement already satisfied: smart-open>=1.8.1 in /usr/local/lib/python3.10/dist-packages (from gensim) (6.3.0)\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: nltk in /usr/local/lib/python3.10/dist-packages (3.8.1)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.10/dist-packages (from nltk) (8.1.3)\n",
            "Requirement already satisfied: joblib in /usr/local/lib/python3.10/dist-packages (from nltk) (1.2.0)\n",
            "Requirement already satisfied: regex>=2021.8.3 in /usr/local/lib/python3.10/dist-packages (from nltk) (2022.10.31)\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.10/dist-packages (from nltk) (4.65.0)\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting sklearn\n",
            "  Downloading sklearn-0.0.post5.tar.gz (3.7 kB)\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Building wheels for collected packages: sklearn\n",
            "  Building wheel for sklearn (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for sklearn: filename=sklearn-0.0.post5-py3-none-any.whl size=2950 sha256=b508a9a07d28f4384aadf570e7a6e5bc78c6ef6f39ca51c7a61b051070e16711\n",
            "  Stored in directory: /root/.cache/pip/wheels/38/1f/8d/4f812c590e074c1e928f5cec67bf5053b71f38e2648739403a\n",
            "Successfully built sklearn\n",
            "Installing collected packages: sklearn\n",
            "Successfully installed sklearn-0.0.post5\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip uninstall numba -y \n",
        "!pip install numba"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "kcHbIhRZ867J",
        "outputId": "eaa27527-76d2-44b8-a698-35ca985dc8b2"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Found existing installation: numba 0.56.4\n",
            "Uninstalling numba-0.56.4:\n",
            "  Successfully uninstalled numba-0.56.4\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting numba\n",
            "  Downloading numba-0.57.0-cp310-cp310-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (3.6 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.6/3.6 MB\u001b[0m \u001b[31m19.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hCollecting llvmlite<0.41,>=0.40.0dev0 (from numba)\n",
            "  Downloading llvmlite-0.40.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (42.1 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m42.1/42.1 MB\u001b[0m \u001b[31m9.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: numpy<1.25,>=1.21 in /usr/local/lib/python3.10/dist-packages (from numba) (1.24.3)\n",
            "Installing collected packages: llvmlite, numba\n",
            "  Attempting uninstall: llvmlite\n",
            "    Found existing installation: llvmlite 0.39.1\n",
            "    Uninstalling llvmlite-0.39.1:\n",
            "      Successfully uninstalled llvmlite-0.39.1\n",
            "Successfully installed llvmlite-0.40.0 numba-0.57.0\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install pyLDAvis\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "x631-phSCMZJ",
        "outputId": "8969d853-f578-4c4f-e234-f2f5b812a7bc"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: pyLDAvis in /usr/local/lib/python3.10/dist-packages (3.4.1)\n",
            "Requirement already satisfied: numpy>=1.24.2 in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (1.24.3)\n",
            "Requirement already satisfied: scipy in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (1.10.1)\n",
            "Requirement already satisfied: pandas>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (2.0.1)\n",
            "Requirement already satisfied: joblib>=1.2.0 in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (1.2.0)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (3.1.2)\n",
            "Requirement already satisfied: numexpr in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (2.8.4)\n",
            "Requirement already satisfied: funcy in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (2.0)\n",
            "Requirement already satisfied: scikit-learn>=1.0.0 in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (1.2.2)\n",
            "Requirement already satisfied: gensim in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (4.3.1)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from pyLDAvis) (67.7.2)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.10/dist-packages (from pandas>=2.0.0->pyLDAvis) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas>=2.0.0->pyLDAvis) (2022.7.1)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.10/dist-packages (from pandas>=2.0.0->pyLDAvis) (2023.3)\n",
            "Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.10/dist-packages (from scikit-learn>=1.0.0->pyLDAvis) (3.1.0)\n",
            "Requirement already satisfied: smart-open>=1.8.1 in /usr/local/lib/python3.10/dist-packages (from gensim->pyLDAvis) (6.3.0)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->pyLDAvis) (2.1.2)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.2->pandas>=2.0.0->pyLDAvis) (1.16.0)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def prepare_corpus(doc_clean):\n",
        "    \"\"\"\n",
        "    This function prepares the corpus by creating a dictionary and document-term matrix.\n",
        "    Input: \n",
        "    - doc_clean: A list of tokenized documents\n",
        "    Return: \n",
        "    - dictionary: A mapping from word IDs to words\n",
        "    - doc_term_matrix: A list of bag-of-word vectors representing each document\n",
        "    \"\"\"\n",
        "    # Creating the term dictionary of our corpus, where every unique term is assigned an index. \n",
        "    dictionary = gensim.corpora.Dictionary(doc_clean)\n",
        "\n",
        "    # Converting list of documents (corpus) into Document Term Matrix using dictionary prepared above.\n",
        "    doc_term_matrix = [dictionary.doc2bow(doc) for doc in doc_clean]\n",
        "    \n",
        "    return dictionary, doc_term_matrix\n"
      ],
      "metadata": {
        "id": "BS5ecSkz-E4l"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "import nltk\n",
        "nltk.download('stopwords')  # add this line to download the stopwords\n",
        "nltk.download('wordnet')  # this downloads the wordnet\n",
        "import requests\n",
        "import re\n",
        "from nltk.tokenize import RegexpTokenizer\n",
        "from nltk.corpus import stopwords\n",
        "from gensim import corpora\n",
        "from gensim.models import LsiModel, LdaModel\n",
        "from nltk.stem import WordNetLemmatizer\n",
        "from gensim.models.coherencemodel import CoherenceModel\n",
        "\n",
        "# Load the articles as a list\n",
        "with open('/content/CollatedText.txt', 'r', encoding=\"utf-8\") as a:\n",
        "    docs = [doc.strip() for doc in a.readlines()]\n",
        "\n",
        "def preprocess_data(doc_set):\n",
        "    tokenizer = RegexpTokenizer(r'\\w+')\n",
        "    en_stop = set(stopwords.words('english'))\n",
        "    lemmatizer = WordNetLemmatizer()   # Use NLTK's WordNetLemmatizer\n",
        "    texts = []\n",
        "\n",
        "    for i in doc_set:\n",
        "        raw = i.lower()\n",
        "        # remove references like [3,4] and (Fig. 1)\n",
        "        raw = re.sub(r'\\[[^\\]]*\\]', '', raw)\n",
        "        raw = re.sub(r'\\(Fig\\. .*?\\)', '', raw)\n",
        "        # split into tokens\n",
        "        tokens = tokenizer.tokenize(raw)\n",
        "        # remove stopwords\n",
        "        stopped_tokens = [i for i in tokens if not i in en_stop]\n",
        "        # lemmatize tokens\n",
        "        lemmatized_tokens = [lemmatizer.lemmatize(i) for i in stopped_tokens]\n",
        "        texts.append(lemmatized_tokens)\n",
        "\n",
        "    return texts\n",
        "\n",
        "def prepare_corpus(doc_clean):\n",
        "    dictionary = corpora.Dictionary(doc_clean)\n",
        "    doc_term_matrix = [dictionary.doc2bow(doc) for doc in doc_clean]\n",
        "    return dictionary, doc_term_matrix\n",
        "\n",
        "clean_docs = preprocess_data(docs)\n",
        "dictionary, doc_term_matrix = prepare_corpus(clean_docs)\n",
        "\n",
        "# LSI Model\n",
        "lsi_model = LsiModel(doc_term_matrix, num_topics=20, id2word = dictionary)\n",
        "print(\"LSI Model:\")\n",
        "for idx, topic in lsi_model.print_topics(-1):\n",
        "    print('Topic: {} Word: {}'.format(idx, topic))\n",
        "\n",
        "# LDA Model\n",
        "lda_model = LdaModel(doc_term_matrix, num_topics=20, id2word = dictionary)\n",
        "print(\"LDA Model:\")\n",
        "for idx, topic in lda_model.print_topics(-1):\n",
        "    print('Topic: {} Word: {}'.format(idx, topic))\n"
      ],
      "metadata": {
        "id": "ourcjejj588f",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f3ecb3ed-5c27-4400-f806-a4cf01265752"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "[nltk_data] Downloading package stopwords to /root/nltk_data...\n",
            "[nltk_data]   Package stopwords is already up-to-date!\n",
            "[nltk_data] Downloading package wordnet to /root/nltk_data...\n",
            "WARNING:gensim.models.ldamodel:too few updates, training might not converge; consider increasing the number of passes or iterations to improve accuracy\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "LSI Model:\n",
            "Topic: 0 Word: 0.660*\"virus\" + 0.317*\"group\" + 0.254*\"family\" + 0.240*\"acid\" + 0.240*\"type\" + 0.229*\"symmetry\" + 0.225*\"belongs\" + 0.113*\"ivof\" + 0.113*\"ssnucleic\" + 0.113*\"virushas\"\n",
            "Topic: 1 Word: -0.410*\"g\" + -0.265*\"protein\" + -0.262*\"sg\" + -0.222*\"g3bp1\" + -0.211*\"viral\" + -0.207*\"virus\" + -0.205*\"quadruplex\" + -0.186*\"rna\" + -0.185*\"cell\" + -0.147*\"structure\"\n",
            "Topic: 2 Word: 0.503*\"g\" + -0.372*\"sg\" + 0.298*\"quadruplex\" + -0.275*\"protein\" + -0.268*\"g3bp1\" + -0.236*\"viral\" + 0.168*\"structure\" + -0.138*\"mrna\" + 0.137*\"dna\" + -0.135*\"infection\"\n",
            "Topic: 3 Word: 0.472*\"virus\" + -0.274*\"quadruplex\" + -0.205*\"type\" + -0.187*\"protein\" + -0.183*\"symmetry\" + 0.182*\"rna\" + -0.179*\"belongs\" + 0.177*\"e\" + -0.171*\"acid\" + -0.164*\"cell\"\n",
            "Topic: 4 Word: 0.457*\"sg\" + -0.440*\"g3bp1\" + -0.385*\"viral\" + 0.343*\"cell\" + -0.291*\"protein\" + 0.167*\"ndv\" + 0.134*\"formation\" + 0.124*\"1\" + 0.121*\"eif2a\" + 0.119*\"mrna\"\n",
            "Topic: 5 Word: -0.648*\"protein\" + 0.386*\"g3bp1\" + 0.293*\"sg\" + 0.249*\"viral\" + 0.135*\"infection\" + -0.133*\"1\" + -0.130*\"stress\" + -0.120*\"binding\" + 0.115*\"g\" + -0.112*\"cell\"\n",
            "Topic: 6 Word: 0.706*\"cell\" + -0.419*\"sg\" + 0.204*\"g3bp1\" + -0.198*\"protein\" + 0.186*\"infected\" + 0.160*\"dna\" + 0.104*\"replication\" + -0.088*\"induced\" + 0.083*\"damage\" + -0.081*\"g\"\n",
            "Topic: 7 Word: 0.329*\"g\" + 0.249*\"e\" + -0.244*\"dna\" + -0.238*\"structure\" + -0.189*\"ap\" + 0.188*\"mrna\" + -0.181*\"site\" + -0.152*\"specie\" + -0.143*\"formation\" + -0.139*\"form\"\n",
            "Topic: 8 Word: -0.598*\"mrna\" + -0.380*\"viral\" + 0.229*\"g3bp1\" + 0.200*\"protein\" + -0.185*\"dna\" + 0.178*\"pkr\" + 0.140*\"infection\" + 0.113*\"cell\" + -0.103*\"ap\" + -0.100*\"site\"\n",
            "Topic: 9 Word: -0.364*\"specie\" + 0.337*\"dna\" + -0.239*\"mrna\" + -0.207*\"virus\" + 0.188*\"rna\" + 0.172*\"sg\" + -0.167*\"quadruplex\" + -0.157*\"genus\" + -0.144*\"viroid\" + -0.142*\"name\"\n",
            "Topic: 10 Word: -0.335*\"specie\" + -0.248*\"type\" + -0.221*\"dna\" + -0.201*\"e\" + -0.189*\"viroid\" + 0.171*\"quadruplex\" + 0.170*\"rna\" + 0.162*\"genome\" + -0.155*\"strand\" + 0.154*\"c\"\n",
            "Topic: 11 Word: -0.399*\"rna\" + -0.261*\"pkr\" + -0.222*\"region\" + -0.211*\"form\" + -0.205*\"structure\" + 0.203*\"dna\" + 0.172*\"protein\" + 0.149*\"viral\" + -0.143*\"domain\" + -0.141*\"mrna\"\n",
            "Topic: 12 Word: -0.310*\"ap\" + 0.288*\"domain\" + -0.287*\"site\" + -0.228*\"genome\" + -0.196*\"known\" + 0.189*\"dna\" + 0.174*\"guanine\" + -0.174*\"specie\" + -0.167*\"damage\" + 0.142*\"satellite\"\n",
            "Topic: 13 Word: 0.410*\"structure\" + -0.307*\"pkr\" + -0.266*\"dna\" + -0.192*\"domain\" + -0.182*\"genus\" + -0.162*\"viroid\" + -0.160*\"specie\" + 0.150*\"form\" + -0.140*\"c\" + -0.139*\"stranded\"\n",
            "Topic: 14 Word: -0.486*\"viral\" + 0.410*\"g3bp1\" + 0.322*\"1\" + 0.218*\"mrna\" + -0.184*\"pkr\" + 0.163*\"tia\" + -0.145*\"replication\" + 0.142*\"binding\" + 0.142*\"infection\" + -0.134*\"cell\"\n",
            "Topic: 15 Word: 0.409*\"rna\" + -0.268*\"mrna\" + 0.258*\"1\" + -0.257*\"pkr\" + 0.213*\"ndv\" + 0.206*\"specie\" + 0.195*\"viroid\" + 0.165*\"genus\" + -0.140*\"virus\" + 0.137*\"tia\"\n",
            "Topic: 16 Word: 0.401*\"1\" + 0.393*\"pkr\" + -0.236*\"sg\" + -0.215*\"gene\" + 0.185*\"tia\" + 0.163*\"viral\" + 0.158*\"eif2a\" + 0.153*\"kinase\" + 0.130*\"ap\" + -0.129*\"mrna\"\n",
            "Topic: 17 Word: -0.508*\"domain\" + 0.343*\"guanine\" + 0.267*\"base\" + -0.164*\"region\" + 0.158*\"quadruplex\" + 0.151*\"run\" + 0.150*\"pkr\" + -0.148*\"structure\" + -0.130*\"c\" + 0.127*\"stress\"\n",
            "Topic: 18 Word: -0.319*\"structure\" + 0.236*\"domain\" + 0.227*\"region\" + 0.223*\"guanine\" + 0.199*\"base\" + -0.194*\"repeat\" + -0.192*\"gene\" + -0.169*\"pkr\" + -0.147*\"genus\" + 0.141*\"quadruplex\"\n",
            "Topic: 19 Word: 0.450*\"gene\" + 0.203*\"1\" + 0.199*\"expression\" + -0.163*\"mrna\" + -0.159*\"genus\" + -0.158*\"g\" + -0.157*\"satellite\" + 0.149*\"ndv\" + -0.128*\"protein\" + 0.121*\"system\"\n",
            "LDA Model:\n",
            "Topic: 0 Word: 0.015*\"protein\" + 0.014*\"cell\" + 0.013*\"rna\" + 0.012*\"viral\" + 0.010*\"fold\" + 0.010*\"non\" + 0.010*\"g3bp1\" + 0.009*\"domain\" + 0.008*\"pkr\" + 0.007*\"g\"\n",
            "Topic: 1 Word: 0.027*\"g\" + 0.019*\"quadruplex\" + 0.017*\"structure\" + 0.012*\"viral\" + 0.010*\"protein\" + 0.010*\"region\" + 0.010*\"guanine\" + 0.008*\"form\" + 0.008*\"quadruplexes\" + 0.007*\"domain\"\n",
            "Topic: 2 Word: 0.014*\"mrna\" + 0.014*\"sg\" + 0.011*\"tia\" + 0.009*\"1\" + 0.009*\"g3bp1\" + 0.008*\"state\" + 0.008*\"translation\" + 0.008*\"gene\" + 0.008*\"viral\" + 0.007*\"level\"\n",
            "Topic: 3 Word: 0.023*\"protein\" + 0.018*\"sg\" + 0.014*\"virus\" + 0.012*\"cell\" + 0.012*\"viral\" + 0.012*\"replication\" + 0.011*\"g3bp1\" + 0.011*\"rna\" + 0.008*\"ndv\" + 0.008*\"mrna\"\n",
            "Topic: 4 Word: 0.022*\"cell\" + 0.014*\"icp27\" + 0.013*\"infected\" + 0.013*\"mutant\" + 0.012*\"region\" + 0.010*\"g3bp1\" + 0.010*\"mrna\" + 0.009*\"infection\" + 0.008*\"viral\" + 0.008*\"virus\"\n",
            "Topic: 5 Word: 0.024*\"sg\" + 0.021*\"g\" + 0.018*\"g3bp1\" + 0.014*\"cell\" + 0.011*\"quadruplexes\" + 0.011*\"quadruplex\" + 0.010*\"ndv\" + 0.009*\"viral\" + 0.008*\"b\" + 0.008*\"infection\"\n",
            "Topic: 6 Word: 0.024*\"g\" + 0.016*\"quadruplex\" + 0.015*\"virus\" + 0.013*\"structure\" + 0.013*\"cell\" + 0.012*\"specie\" + 0.011*\"dna\" + 0.011*\"formation\" + 0.011*\"sg\" + 0.010*\"type\"\n",
            "Topic: 7 Word: 0.021*\"virus\" + 0.017*\"g\" + 0.015*\"quadruplex\" + 0.015*\"protein\" + 0.014*\"sg\" + 0.012*\"viral\" + 0.012*\"1\" + 0.010*\"sequence\" + 0.008*\"g3bp1\" + 0.008*\"cell\"\n",
            "Topic: 8 Word: 0.042*\"protein\" + 0.022*\"sg\" + 0.021*\"mrna\" + 0.015*\"g3bp1\" + 0.013*\"translation\" + 0.011*\"infection\" + 0.011*\"stress\" + 0.011*\"rna\" + 0.010*\"binding\" + 0.008*\"factor\"\n",
            "Topic: 9 Word: 0.043*\"virus\" + 0.021*\"g3bp1\" + 0.016*\"rna\" + 0.015*\"group\" + 0.011*\"sg\" + 0.011*\"protein\" + 0.010*\"viral\" + 0.009*\"family\" + 0.007*\"cell\" + 0.007*\"stranded\"\n",
            "Topic: 10 Word: 0.020*\"g3bp1\" + 0.015*\"protein\" + 0.012*\"g\" + 0.011*\"virus\" + 0.009*\"viral\" + 0.007*\"quadruplex\" + 0.006*\"effect\" + 0.006*\"replication\" + 0.005*\"structure\" + 0.005*\"many\"\n",
            "Topic: 11 Word: 0.025*\"mrna\" + 0.015*\"protein\" + 0.015*\"sg\" + 0.010*\"1\" + 0.010*\"viral\" + 0.010*\"binding\" + 0.009*\"induced\" + 0.008*\"virus\" + 0.007*\"pkr\" + 0.007*\"dna\"\n",
            "Topic: 12 Word: 0.040*\"virus\" + 0.031*\"sg\" + 0.029*\"cell\" + 0.014*\"infection\" + 0.013*\"formation\" + 0.011*\"eif2a\" + 0.011*\"protein\" + 0.008*\"infected\" + 0.008*\"pkr\" + 0.007*\"1\"\n",
            "Topic: 13 Word: 0.024*\"cell\" + 0.013*\"virus\" + 0.011*\"acid\" + 0.011*\"g3bp1\" + 0.011*\"sg\" + 0.010*\"g\" + 0.010*\"viral\" + 0.009*\"nucleic\" + 0.008*\"ndv\" + 0.008*\"pkr\"\n",
            "Topic: 14 Word: 0.021*\"protein\" + 0.017*\"g\" + 0.014*\"gene\" + 0.011*\"viral\" + 0.009*\"sg\" + 0.008*\"region\" + 0.008*\"structure\" + 0.007*\"quadruplex\" + 0.007*\"pkr\" + 0.006*\"cell\"\n",
            "Topic: 15 Word: 0.041*\"g3bp1\" + 0.030*\"viral\" + 0.028*\"virus\" + 0.020*\"replication\" + 0.020*\"sg\" + 0.013*\"protein\" + 0.009*\"infection\" + 0.007*\"family\" + 0.007*\"formation\" + 0.006*\"study\"\n",
            "Topic: 16 Word: 0.026*\"g3bp1\" + 0.024*\"sg\" + 0.017*\"protein\" + 0.016*\"viral\" + 0.012*\"infection\" + 0.011*\"g\" + 0.009*\"formation\" + 0.009*\"cell\" + 0.009*\"virus\" + 0.009*\"mrna\"\n",
            "Topic: 17 Word: 0.017*\"pkr\" + 0.014*\"kinase\" + 0.013*\"domain\" + 0.013*\"sg\" + 0.012*\"protein\" + 0.010*\"infection\" + 0.008*\"formation\" + 0.007*\"virus\" + 0.007*\"structural\" + 0.007*\"quadruplex\"\n",
            "Topic: 18 Word: 0.029*\"virus\" + 0.022*\"g\" + 0.017*\"dna\" + 0.014*\"structure\" + 0.013*\"quadruplex\" + 0.008*\"protein\" + 0.007*\"viral\" + 0.007*\"stranded\" + 0.007*\"domain\" + 0.006*\"also\"\n",
            "Topic: 19 Word: 0.014*\"sg\" + 0.014*\"cell\" + 0.014*\"mrna\" + 0.013*\"region\" + 0.011*\"quadruplexes\" + 0.011*\"structure\" + 0.010*\"viral\" + 0.009*\"virus\" + 0.007*\"g\" + 0.007*\"rna\"\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import pyLDAvis\n",
        "import pyLDAvis.gensim_models as gensimvis\n",
        "\n",
        "pyLDAvis.enable_notebook()\n",
        "vis = gensimvis.prepare(lda_model, doc_term_matrix, dictionary)\n",
        "pyLDAvis.display(vis)\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 882
        },
        "id": "0SGmDkKACKQs",
        "outputId": "821b2c82-c930-47c8-f4b9-29edadeccb34"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ],
            "text/html": [
              "\n",
              "<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.jsdelivr.net/gh/bmabey/pyLDAvis@3.4.0/pyLDAvis/js/ldavis.v1.0.0.css\">\n",
              "\n",
              "\n",
              "<div id=\"ldavis_el94351398835099700003693728938\" style=\"background-color:white;\"></div>\n",
              "<script type=\"text/javascript\">\n",
              "\n",
              "var ldavis_el94351398835099700003693728938_data = {\"mdsDat\": {\"x\": [-0.05087851605967665, 0.012276991169915414, 0.035006128197275437, -0.12736158724133087, 0.0644031549132629, 0.024135546220332954, -0.10944594257972731, -0.12997140444881083, 0.04104284616919997, -0.034198079485833956, 0.010017895852022264, 0.07061828214860148, 0.03386049506788735, 0.0023034030717748112, -0.04213988327099668, 0.0362849365193373, 0.0276789255704449, 0.06518704884241712, 0.09698009737291667, -0.02580033802901252], \"y\": [-0.0211603928743945, -0.11689189329316221, -0.08389977062594496, -0.04007540270032853, 0.016740185115747385, -0.10626071818227953, 0.05272841674961519, 0.028586738742107418, -0.05878308472806867, 0.014620174230183493, -0.028807214596497734, 0.0838553324330985, 0.08079690524162556, 0.0682142420382392, -3.322608649894947e-05, 0.049913484553505486, 0.04321946718970108, 0.06638502496784449, -0.04115814342245088, -0.007990124752041312], \"topics\": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20], \"cluster\": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1], \"Freq\": [9.558449503560553, 8.651650604387973, 7.2175995574103755, 7.187323172085989, 5.964907025719553, 5.5779377476396075, 5.432265466122614, 5.247794148962075, 5.22510082032573, 4.864562961229593, 4.741257624556305, 4.088904955871717, 3.9738357150457473, 3.8930255467020283, 3.8260823611265233, 3.7897544967053136, 3.2964420991753784, 2.9738152697254447, 2.7223972321652687, 1.7668936914822082]}, \"tinfo\": {\"Term\": [\"g3bp1\", \"mrna\", \"g\", \"protein\", \"virus\", \"sg\", \"cell\", \"replication\", \"viral\", \"quadruplex\", \"domain\", \"structure\", \"icp27\", \"kinase\", \"1\", \"infected\", \"tia\", \"gene\", \"rna\", \"dna\", \"region\", \"ndv\", \"mutant\", \"binding\", \"induced\", \"specie\", \"quadruplexes\", \"expression\", \"pkr\", \"acid\", \"international\", \"hairpin\", \"distinguished\", \"relationship\", \"decoy\", \"generated\", \"set\", \"fibroblast\", \"fraction\", \"universal\", \"algae\", \"asos\", \"seems\", \"sitia\", \"july\", \"gfp\", \"sterically\", \"causal\", \"unpublished\", \"posi\", \"embryo\", \"ab\", \"cytoskeleton\", \"controversial\", \"aaron\", \"exclusive\", \"program\", \"scale\", \"situated\", \"contrary\", \"extracellular\", \"chicken\", \"biochemical\", \"criterion\", \"trans\", \"suggests\", \"taxonomy\", \"seem\", \"sequence\", \"structural\", \"g3bp\", \"difference\", \"r\", \"ictv\", \"remove\", \"separated\", \"interacts\", \"ggggcc\", \"1\", \"quadruplex\", \"system\", \"classification\", \"g\", \"virus\", \"modified\", \"expression\", \"protein\", \"host\", \"viral\", \"ndv\", \"used\", \"sg\", \"form\", \"tia\", \"role\", \"formation\", \"potential\", \"structure\", \"rna\", \"found\", \"cell\", \"gene\", \"g3bp1\", \"mrna\", \"region\", \"infection\", \"replication\", \"domain\", \"function\", \"unassembled\", \"suitable\", \"pv\", \"cytotoxic\", \"nuclease\", \"subversion\", \"inefficient\", \"sing\", \"mosquito\", \"yeast\", \"togaviridae\", \"qualifies\", \"hunov\", \"borne\", \"prod\", \"ischemia\", \"amazingly\", \"arthritis\", \"remodel\", \"date\", \"dearth\", \"react\", \"autoinhibitory\", \"account\", \"uct\", \"subdivided\", \"cvb3\", \"prominent\", \"intact\", \"affected\", \"enveloped\", \"metal\", \"envelopedcapsidhelicalcapsid\", \"belongs\", \"group\", \"sars\", \"ivof\", \"ssnucleic\", \"hepatitis\", \"virushas\", \"picornavirus\", \"necessary\", \"modulate\", \"virus\", \"intracellular\", \"s\", \"cleavage\", \"sense\", \"invading\", \"vof\", \"coronavirus\", \"nakedcapsidicosahedralcapsid\", \"sequestration\", \"rna\", \"symmetry\", \"family\", \"positive\", \"\\u00df\", \"norovirus\", \"g3bp1\", \"stranded\", \"expression\", \"single\", \"ifn\", \"type\", \"acid\", \"viral\", \"effect\", \"sg\", \"protein\", \"n\", \"form\", \"replication\", \"genome\", \"cell\", \"dna\", \"translation\", \"stress\", \"non\", \"g\", \"mrna\", \"quadruplex\", \"s51d\", \"s51a\", \"32\", \"dtt\", \"defective\", \"ef\", \"parental\", \"colocalize\", \"tobacco\", \"denatured\", \"noc\", \"145\", \"marc\", \"brought\", \"expected\", \"substrate\", \"hypo\", \"displayed\", \"endoribonuclease\", \"viable\", \"maintained\", \"transfecting\", \"hyper\", \"microbe\", \"conclusion\", \"sjincluding\", \"defines\", \"chaperon\", \"manipulates\", \"groundwork\", \"denv\", \"37\", \"transfection\", \"dot\", \"shape\", \"sig3bp1\", \"paramyxoviridae\", \"exposed\", \"inducer\", \"transfected\", \"wild\", \"unnamed\", \"mosaic\", \"eif2a\", \"cell\", \"virus\", \"sg\", \"fig\", \"infected\", \"formation\", \"infection\", \"satellite\", \"treatment\", \"reduced\", \"mutant\", \"ndv\", \"nsp1\\u00df\", \"1\", \"vhs\", \"pkr\", \"type\", \"protein\", \"family\", \"induced\", \"translation\", \"12\", \"h\", \"stress\", \"study\", \"prrsv\", \"g3bp1\", \"g\", \"structure\", \"e\", \"rna\", \"idr\", \"intriguingly\", \"acetylated\", \"acape\", \"effectively\", \"suppressed\", \"art\", \"epidemic\", \"reagent\", \"address\", \"driving\", \"albeit\", \"analog\", \"limited\", \"aumaivirus\", \"dissociates\", \"72\", \"tmpyp_\", \"arrested\", \"solution\", \"optimal\", \"march\", \"physicochemical\", \"force\", \"occupancy\", \"2021\", \"executive\", \"intrin\", \"summarize\", \"reserved\", \"accepted\", \"allowing\", \"tetramolecular\", \"turn\", \"native\", \"globular\", \"need\", \"sedis\", \"incertae\", \"development\", \"described\", \"dsdna\", \"dna\", \"currently\", \"state\", \"although\", \"double\", \"research\", \"determine\", \"definition\", \"g\", \"ape\", \"typical\", \"varidnaviria\", \"duplodnaviria\", \"structure\", \"virus\", \"ssrna\", \"strand\", \"satellite\", \"quadruplex\", \"stranded\", \"e\", \"also\", \"reported\", \"host\", \"may\", \"domain\", \"pathway\", \"region\", \"form\", \"protein\", \"transcription\", \"viral\", \"infection\", \"g3bp1\", \"c\", \"cell\", \"family\", \"formation\", \"rna\", \"participate\", \"targeted\", \"common\", \"marking\", \"dihydro\", \"recog\", \"lapping\", \"reference\", \"reviewed\", \"node\", \"deriva\", \"fifth\", \"heterodimerization\", \"tissue\", \"0j0antisense\", \"alternatively\", \"heterochromatin\", \"nizes\", \"radical\", \"combating\", \"chaperone\", \"establishing\", \"indirectly\", \"medical\", \"mismatching\", \"coded\", \"redundant\", \"cytopathic\", \"spared\", \"application\", \"network\", \"39\", \"oxoguanine\", \"fmrp\", \"expressed\", \"profile\", \"recognition\", \"fusion\", \"protein\", \"arrest\", \"fmr\", \"polyadenylated\", \"mrna\", \"translation\", \"factor\", \"stress\", \"binding\", \"oxidized\", \"nucleotide\", \"sg\", \"expansion\", \"component\", \"marker\", \"bacterial\", \"initiation\", \"certain\", \"infection\", \"rna\", \"g3bp1\", \"mediated\", \"also\", \"degradation\", \"form\", \"2\", \"cellular\", \"function\", \"pkr\", \"gene\", \"1\", \"viral\", \"c\", \"role\", \"induced\", \"virus\", \"cell\", \"dna\", \"g\", \"flaviviridae\", \"unexplored\", \"sucrose\", \"virinae\", \"100\", \"distinguish\", \"gated\", \"virineae\", \"predominately\", \"interactors\", \"\\u03b2c1\", \"differentially\", \"played\", \"retrieved\", \"investi\", \"10000\", \"viridae\", \"atf4\", \"neutralizing\", \"concentrated\", \"subgenus\", \"alter\", \"note\", \"growing\", \"suborder\", \"mechanistic\", \"determinant\", \"bacterium\", \"proteomic\", \"earlier\", \"provided\", \"utilized\", \"insight\", \"titer\", \"reduces\", \"replication\", \"g3bp1\", \"enhances\", \"smaller\", \"viral\", \"mnv\", \"knockdown\", \"granule\", \"overexpression\", \"different\", \"virus\", \"study\", \"sg\", \"satellite\", \"post\", \"mechanism\", \"interactome\", \"chromatin\", \"however\", \"family\", \"associated\", \"recent\", \"depletion\", \"similarly\", \"role\", \"protein\", \"stranded\", \"infection\", \"complex\", \"formation\", \"e\", \"tia\", \"guanine\", \"prrsv\", \"induced\", \"1\", \"stress\", \"rna\", \"cell\", \"function\", \"quadruplex\", \"actual\", \"stoichiometry\", \"hydrogen\", \"ntf2l\", \"stack\", \"cal\", \"0xoguanine\", \"transport\", \"fo\", \"compete\", \"dichroism\", \"influenced\", \"catalogue\", \"thermostability\", \"000000\", \"self\", \"polymorphic\", \"inconsistent\", \"settled\", \"hypervariable\", \"ikb\", \"glycine\", \"underway\", \"preliminary\", \"blue\", \"computational\", \"wrapped\", \"threshold\", \"q325\", \"architecture\", \"tetrad\", \"cation\", \"uv\", \"motif\", \"rich\", \"hoogsteen\", \"top\", \"guanine\", \"four\", \"huge\", \"g\", \"structure\", \"terminal\", \"quadruplex\", \"strand\", \"within\", \"repair\", \"bond\", \"sequence\", \"short\", \"interaction\", \"3c\", \"form\", \"region\", \"run\", \"quadruplexes\", \"base\", \"nucleic\", \"domain\", \"viral\", \"specific\", \"dna\", \"protein\", \"acid\", \"1\", \"also\", \"rna\", \"role\", \"g3bp1\", \"cell\", \"virus\", \"3\", \"viroid\", \"subsequent\", \"cadang\", \"right\", \"rc\", \"last\", \"hypoxia\", \"crowding\", \"original\", \"modulating\", \"blumei\", \"incorporation\", \"la\", \"nonrandom\", \"tuber\", \"tautomeric\", \"vaccinia\", \"coconut\", \"decouples\", \"lowest\", \"damaging\", \"hop\", \"spindle\", \"allowed\", \"wrn\", \"synergistically\", \"rely\", \"assessment\", \"radioactive\", \"bloom\", \"intensity\", \"specie\", \"genus\", \"damage\", \"neither\", \"p\", \"molecular\", \"change\", \"stabilization\", \"g\", \"type\", \"fact\", \"quadruplex\", \"1982\", \"sequencing\", \"though\", \"structure\", \"instance\", \"dna\", \"formation\", \"arsenite\", \"year\", \"wide\", \"ap\", \"system\", \"cell\", \"virus\", \"however\", \"pkr\", \"sg\", \"site\", \"region\", \"interaction\", \"binding\", \"response\", \"endogenous\", \"base\", \"protein\", \"infection\", \"e\", \"g3bp1\", \"known\", \"correspondingly\", \"event\", \"potentiates\", \"accumulat\", \"fundamental\", \"confused\", \"explained\", \"fortunately\", \"antagonizing\", \"correlation\", \"ns6pro\", \"coincidental\", \"manual\", \"worth\", \"seemingly\", \"ten\", \"tween\", \"fails\", \"proteasome\", \"coimmunoprecipitation\", \"served\", \"remodels\", \"processed\", \"sembly\", \"calicivirus\", \"interval\", \"ubiquitylation\", \"description\", \"72hpi\", \"hydrolase\", \"tested\", \"dm\", \"ttp\", \"indicated\", \"stable\", \"are\", \"assay\", \"old\", \"distinct\", \"bona\", \"piperidine\", \"g3bp1\", \"fide\", \"sg\", \"world\", \"stage\", \"mechanism\", \"viral\", \"cleavage\", \"induce\", \"infection\", \"study\", \"assembly\", \"protein\", \"formation\", \"alphavirus\", \"e\", \"late\", \"mrna\", \"g\", \"cell\", \"function\", \"virus\", \"role\", \"induced\", \"stress\", \"activity\", \"quadruplex\", \"replication\", \"dna\", \"nsp1\\u00df\", \"interstrand\", \"serve\", \"purpose\", \"assemble\", \"unusual\", \"plasm\", \"consideration\", \"ner\", \"hierarchy\", \"reliable\", \"evasion\", \"discussed\", \"grouped\", \"exonucleolytic\", \"activator\", \"molten\", \"ct\", \"ternary\", \"encapsu\", \"glyceraldehyde\", \"polyinosinic\", \"adenocarcinoma\", \"rationale\", \"ratio\", \"directionality\", \"apoptosis\", \"globule\", \"disappeared\", \"value\", \"hold\", \"treat\", \"rapidly\", \"telomerase\", \"product\", \"would\", \"telomeric\", \"survival\", \"5\", \"quadruplexes\", \"environmental\", \"region\", \"3\", \"phaginae\", \"drought\", \"shown\", \"gapdh\", \"mrna\", \"cleaved\", \"transcriptase\", \"ii\", \"variety\", \"condition\", \"structure\", \"promoter\", \"involved\", \"reverse\", \"double\", \"cell\", \"stranded\", \"human\", \"sg\", \"domain\", \"viral\", \"level\", \"genome\", \"type\", \"virus\", \"rna\", \"dna\", \"translation\", \"form\", \"g\", \"also\", \"group\", \"complex\", \"protein\", \"overhang\", \"coat\", \"replicates\", \"contribute\", \"mitogenic\", \"enylated\", \"healing\", \"papovaviridaeare\", \"sneaky\", \"nonsegmented\", \"easily\", \"feasible\", \"synonymous\", \"examine\", \"provides\", \"upstream\", \"healthy\", \"charge\", \"pur\", \"dead\", \"downstream\", \"sponse\", \"rescued\", \"persist\", \"rnai\", \"occurs\", \"repairing\", \"advance\", \"intermedi\", \"coupled\", \"belong\", \"helper\", \"naked\", \"d\", \"capsid\", \"nucleic\", \"acid\", \"signaling\", \"cell\", \"molecule\", \"pathway\", \"selective\", \"symmetry\", \"response\", \"ndv\", \"growth\", \"icosahedral\", \"whether\", \"process\", \"virion\", \"early\", \"pkr\", \"ammatory\", \"translation\", \"innate\", \"1\", \"eif2a\", \"g3bp1\", \"virus\", \"viral\", \"g\", \"sg\", \"complex\", \"quadruplexes\", \"tia\", \"infection\", \"function\", \"type\", \"group\", \"formation\", \"replication\", \"rna\", \"protein\", \"mediates\", \"corresponding\", \"modu\", \"somewhat\", \"additionally\", \"critically\", \"knowledge\", \"450\", \"ikk\", \"complete\", \"grouping\", \"daunting\", \"eumetazoa\", \"tristetraprolin\", \"partial\", \"585\", \"rossman\", \"sufficient\", \"1css\", \"going\", \"citrate\", \"synthase\", \"frequency\", \"lastly\", \"nf\", \"1i31\", \"concurrently\", \"gadd45\", \"complementing\", \"challenge\", \"notion\", \"categorization\", \"essential\", \"fold\", \"flag\", \"non\", \"evidence\", \"gc\", \"across\", \"mutation\", \"g4s\", \"increased\", \"14\", \"respectively\", \"rna\", \"repeat\", \"domain\", \"subunit\", \"increase\", \"cell\", \"protein\", \"neuron\", \"residue\", \"expression\", \"4\", \"viral\", \"poly\", \"transcript\", \"pkr\", \"well\", \"g3bp1\", \"may\", \"vhs\", \"binding\", \"g\", \"mrna\", \"structure\", \"complex\", \"including\", \"gene\", \"sg\", \"however\", \"translation\", \"statistical\", \"heme\", \"nonderepressible\", \"participation\", \"nocodazole\", \"challenging\", \"surprise\", \"descriptor\", \"representing\", \"repurposing\", \"adapting\", \"ticulum\", \"student\", \"added\", \"reiterates\", \"examination\", \"learn\", \"five\", \"categorize\", \"filled\", \"taught\", \"bar\", \"attempting\", \"05\", \"powerful\", \"facilitated\", \"earliest\", \"tailed\", \"avg\", \"sure\", \"significance\", \"endoplasmic\", \"evolutionarily\", \"id\", \"pdb\", \"kinase\", \"microtubule\", \"disruption\", \"inhibitor\", \"mean\", \"structural\", \"domain\", \"pkr\", \"regulated\", \"near\", \"rejected\", \"ogg\", \"dependent\", \"full\", \"r\", \"fold\", \"cgas\", \"realm\", \"indicate\", \"defined\", \"infection\", \"pathway\", \"formation\", \"protein\", \"sg\", \"stress\", \"one\", \"small\", \"quadruplex\", \"region\", \"like\", \"c\", \"specific\", \"virus\", \"nsp1\\u00df\", \"cellular\", \"response\", \"function\", \"gene\", \"mrna\", \"ring\", \"ligase\", \"camp\", \"reactivity\", \"harmful\", \"translating\", \"noticeably\", \"continuum\", \"meaningful\", \"biochemistry\", \"aromatic\", \"strategically\", \"conservation\", \"globin\", \"opposed\", \"tend\", \"rfp\", \"recognize\", \"keep\", \"needed\", \"band\", \"presynaptic\", \"brain\", \"nanopore\", \"repur\", \"fragmented\", \"necessarily\", \"prohibited\", \"interchangeable\", \"neutralization\", \"prediction\", \"seen\", \"discovery\", \"ftd\", \"regulator\", \"corf\", \"ire\", \"ra\", \"increasingly\", \"linear\", \"gene\", \"dynamic\", \"disorder\", \"affinity\", \"fragile\", \"4\", \"protein\", \"al\", \"g\", \"terminal\", \"chromosome\", \"platform\", \"would\", \"isgs\", \"region\", \"showed\", \"viral\", \"internal\", \"ligand\", \"target\", \"structure\", \"binding\", \"pkr\", \"sg\", \"quadruplex\", \"quadruplexes\", \"domain\", \"1\", \"cell\", \"translation\", \"g3bp1\", \"dna\", \"c\", \"factor\", \"also\", \"virus\", \"ndv\", \"joining\", \"review\", \"stained\", \"diagonally\", \"exogenous\", \"align\", \"recognizing\", \"ascertain\", \"pursuit\", \"41\", \"bird\", \"mation\", \"repress\", \"strongly\", \"pretreatment\", \"soreson\", \"stone\", \"scram\", \"uncapped\", \"confirm\", \"42\", \"rotate\", \"hybridize\", \"irradiation\", \"adjacent\", \"bled\", \"next\", \"axis\", \"monitored\", \"action\", \"55\", \"b\", \"tumor\", \"zikv\", \"nsp3\", \"run\", \"ifn\", \"loop\", \"ligand\", \"sg\", \"g\", \"implicated\", \"quadruplexes\", \"ndv\", \"modulated\", \"g3bp1\", \"two\", \"associate\", \"neurological\", \"guanine\", \"cell\", \"canonical\", \"intramolecular\", \"sirnas\", \"quadruplex\", \"base\", \"one\", \"nsp1\\u00df\", \"promoter\", \"replication\", \"infection\", \"viral\", \"genome\", \"virus\", \"used\", \"binding\", \"c\", \"component\", \"pkr\", \"complex\", \"region\", \"origin\", \"heterodimeric\", \"baseline\", \"veev\", \"virtually\", \"30utrs\", \"radiation\", \"sthe\", \"nega\", \"exudes\", \"jxwn06\", \"geographical\", \"recovered\", \"diversity\", \"exclusively\", \"exploration\", \"eif3a\", \"ionizing\", \"hydroxyl\", \"story\", \"plastic\", \"identify\", \"warrant\", \"give\", \"majority\", \"abundantly\", \"despite\", \"clustered\", \"pivotal\", \"randomly\", \"icp27\", \"hence\", \"stabilizes\", \"h\", \"mutant\", \"drbm\", \"36\", \"colocalized\", \"picornaviridae\", \"infected\", \"6\", \"regulates\", \"time\", \"cell\", \"treated\", \"lane\", \"region\", \"30\", \"either\", \"alphavirus\", \"mrna\", \"vhs\", \"degradation\", \"2\", \"g4s\", \"ndv\", \"infection\", \"c\", \"dna\", \"g3bp1\", \"viral\", \"binding\", \"virus\", \"pkr\", \"1\", \"nsp1\\u00df\", \"rna\", \"protein\", \"fig\", \"replication\", \"sg\", \"j\", \"jev\", \"dramatically\", \"influx\", \"herpesviridae\", \"crna\", \"encapsulates\", \"brief\", \"equilibrium\", \"pseudo\", \"sudden\", \"produce\", \"ifna\", \"total\", \"shuttling\", \"biomolecular\", \"transla\", \"degrades\", \"abrogates\", \"fluidity\", \"abuts\", \"polyribo\", \"cascade\", \"japanese\", \"incapable\", \"assembled\", \"elongating\", \"characterizes\", \"caprin\", \"condensate\", \"detect\", \"coding\", \"mrna\", \"invading\", \"conformation\", \"pathogen\", \"open\", \"induced\", \"signaling\", \"core\", \"binding\", \"model\", \"1\", \"specifically\", \"protein\", \"ap\", \"sg\", \"dynamic\", \"end\", \"new\", \"expression\", \"known\", \"lower\", \"addition\", \"critical\", \"name\", \"viral\", \"pkr\", \"dna\", \"gene\", \"strand\", \"virus\", \"kinase\", \"complex\", \"function\", \"g3bp1\", \"stress\", \"ndv\", \"site\", \"translation\", \"found\", \"representative\", \"ance\", \"stall\", \"blood\", \"cyclin\", \"dogma\", \"qpcr\", \"recycling\", \"corroborated\", \"hy\", \"endothelial\", \"investigate\", \"checkpoint\", \"throughput\", \"rylated\", \"cdknla\", \"incidence\", \"unavailable\", \"heterodimer\", \"700000\", \"pothesis\", \"harbour\", \"biased\", \"vessel\", \"vascular\", \"phospho\", \"unexpectedly\", \"limitation\", \"capped\", \"appear\", \"measured\", \"state\", \"rig\", \"tia\", \"structured\", \"polysomes\", \"tiar\", \"np\", \"initiation\", \"demonstrate\", \"endogenous\", \"mrna\", \"level\", \"result\", \"ammatory\", \"1\", \"late\", \"eif2a\", \"sg\", \"gene\", \"factor\", \"translation\", \"high\", \"prrsv\", \"verify\", \"depletion\", \"g3bp1\", \"domain\", \"dependent\", \"time\", \"viral\", \"ndv\", \"infection\", \"also\", \"cell\", \"response\", \"protein\", \"expression\", \"brachygtaviriformidae\", \"maturation\", \"taken\", \"bicaudaviridae\", \"44\", \"nipulating\", \"ampullaviridae\", \"hek\", \"measurement\", \"fuselloviridae\", \"anelloviridae\", \"ongoing\", \"293ft\", \"individually\", \"elevates\", \"localization\", \"downregulation\", \"exception\", \"nucleolar\", \"guttaviridae\", \"globuloviridae\", \"upregulation\", \"bartogtaviriformidae\", \"augmenting\", \"halspiviridae\", \"clue\", \"clavaviridae\", \"sensing\", \"8a\", \"nucleus\", \"nucleolin\", \"impairs\", \"synthesis\", \"protein\", \"replication\", \"sg\", \"rna\", \"flag\", \"induced\", \"induces\", \"\\u00df\", \"ndv\", \"viral\", \"binding\", \"cell\", \"virus\", \"function\", \"modulate\", \"g3bp1\", \"mrna\", \"oxidative\", \"belongs\", \"either\", \"tia\", \"knockdown\", \"kd\", \"symmetry\", \"stress\", \"cleavage\", \"1\", \"infection\", \"translation\", \"cellular\", \"pkr\", \"group\", \"family\", \"within\", \"acid\", \"exact\", \"bruton\", \"intrinsically\", \"cdd\", \"nificant\", \"sig\", \"2009\", \"encoded\", \"4b\", \"tyrosine\", \"fairly\", \"achieve\", \"denaturing\", \"hydration\", \"conclude\", \"n_g\", \"pabp\", \"toxin\", \"deoxyguanosine\", \"deletion\", \"unaffected\", \"updated\", \"center\", \"predominant\", \"layer\", \"40\", \"btk\", \"field\", \"describe\", \"coli\", \"section\", \"ribosome\", \"g3bp1\", \"effect\", \"poly\", \"protein\", \"showed\", \"g\", \"still\", \"end\", \"virus\", \"similar\", \"viral\", \"many\", \"base\", \"quadruplex\", \"replication\", \"forming\", \"subunit\", \"genome\", \"binding\", \"role\", \"structure\", \"stress\", \"quadruplexes\", \"found\", \"n\"], \"Freq\": [169.0, 91.0, 155.0, 185.0, 246.0, 198.0, 150.0, 61.0, 142.0, 100.0, 48.0, 83.0, 14.0, 19.0, 67.0, 37.0, 30.0, 46.0, 86.0, 70.0, 66.0, 56.0, 25.0, 50.0, 34.0, 21.0, 49.0, 32.0, 72.0, 37.0, 4.204550948316974, 1.6520291157008336, 1.6518436293410177, 4.688461576281528, 3.1415287786123995, 1.5437147439662937, 1.5056030786228873, 1.9358508078130872, 5.289556815658952, 1.3776304791471485, 1.3346176020655514, 1.3273138588478501, 1.651863429336728, 1.6522280660577324, 0.8463139094527212, 0.8463118502531674, 0.8462892782580573, 0.8462831798593785, 0.8462826254594986, 0.8462811998598075, 0.846277952660511, 0.8462767646607684, 0.8462743094613002, 0.8462648054633592, 0.8462530838658986, 0.8462530838658986, 0.846252371066053, 0.8462522126660873, 0.8462513414662761, 0.8462477774670482, 2.457999029093765, 2.4581197298676165, 1.6521514004743414, 1.6520191365029955, 1.586532630690134, 1.652135085277876, 1.9760902598955252, 1.611916700390874, 13.514818128120522, 6.288170708117075, 2.45827923863306, 2.185370987356495, 4.097678174670123, 4.059389734965014, 1.5328621287174278, 1.6054750489864074, 4.608923251112882, 2.125915401837082, 16.23138767719701, 20.427513014539326, 4.7803110635830555, 4.3378388170412014, 22.756157246056794, 29.195868567743783, 2.3820706391430564, 6.901430114459229, 20.210968856651924, 6.146746783555461, 16.773589492533233, 9.523617360782904, 5.486932993698949, 18.886590545168108, 8.320001282336898, 6.102389090765207, 6.7630113540465695, 8.708073595064072, 3.52710122308115, 9.026267446129866, 9.066545389403965, 4.443909770061761, 10.436025239916884, 6.358928606387931, 10.65678980648997, 7.641336629364279, 6.925598147623414, 6.9369117067724195, 6.161956348260428, 5.564733354444111, 4.9821538038554065, 1.939838710060359, 1.091594070184699, 1.0915821702467288, 1.0915424559959124, 1.0915320614717816, 1.091531918099035, 1.0915117025417602, 1.091502311626856, 1.0914620955714265, 1.0914518444200425, 1.0914398011093256, 1.0914365752225266, 1.0914317722355145, 1.0914289047805819, 1.0914250337164229, 1.0914246752845562, 1.0914240301071965, 1.0914236716753298, 1.0914165030379983, 1.0914070404367209, 1.091395929048857, 1.0913930615939245, 1.0913823803243006, 1.091379082751128, 1.091377433964542, 1.0913673978722778, 1.091361089471426, 1.0913598708030798, 1.0913563581707872, 1.0913509100064154, 4.799581335861973, 2.1307567269270207, 3.6531424587908834, 6.91674059149866, 18.905335680327855, 2.817065147057929, 3.8541080377413346, 3.6701180787372993, 3.329925232753476, 3.6310188970039916, 3.0814413376421443, 3.1706664990520284, 3.1706168920816946, 53.15527376555971, 1.850829037888659, 4.2432807416651785, 6.5528754713188535, 6.408636753299796, 2.1308840419260275, 3.033287019977216, 3.5938561075866606, 2.4224293679965174, 2.130819954308284, 20.176263966315293, 6.4611186339489874, 11.387063834016741, 6.269622538167916, 3.151384298357838, 2.1306960802551966, 26.08989197460237, 8.609287407463164, 7.702515575076684, 5.744207874541017, 5.233415510597359, 8.549939119232397, 7.550776168444963, 12.26339017691097, 4.202624245157982, 13.921908905190449, 13.092170676076968, 4.572478044964499, 6.6579516368505915, 6.989331360099105, 5.736585605839211, 8.922244306260783, 6.446980934149353, 5.603087227015656, 5.3490966714517825, 4.482451143155464, 5.643647950528105, 4.7564774665698115, 4.693858272497442, 2.3142930821698706, 2.3140000424432516, 1.5556579633648702, 1.555598398555133, 1.5555800985232255, 1.5555674200697474, 1.5554615669440095, 1.5553878883841739, 1.5444279633925238, 2.014406512240534, 1.474390751082297, 2.314699510329483, 2.3142454781653017, 2.3143253763438243, 2.3140375993714795, 1.7689882529662915, 0.7968686099772266, 0.796864363891392, 0.7968562903479035, 0.796853120734534, 0.7968428942461152, 0.79684110012534, 0.7968266275510866, 0.7968193912639598, 0.7968098824238512, 0.7968078490869727, 0.7968010912320527, 0.7968009118199751, 0.7968007922119235, 0.7967979216186831, 2.314496894289935, 1.5555914612881354, 2.3143337489074423, 2.314323701831101, 1.555388247208329, 1.5555330925589148, 1.555640739805428, 1.5556085652395257, 1.555489196403948, 4.590129783563207, 2.7678551867085353, 1.5435549442233027, 1.536035903662417, 11.433447527069506, 29.630730376709923, 41.40266581944394, 31.644928053292222, 6.4874421661527935, 8.57129287398332, 13.019877053890394, 14.500904138112169, 4.533237496132743, 3.4471978221642186, 4.007774446622554, 5.199628971752388, 7.548423474861902, 5.207678115198324, 7.610678508897545, 4.590898624119414, 7.70792224707538, 6.269932532009528, 11.273735374091828, 5.946318595219885, 5.365881292995992, 5.6955445266074625, 3.2749792120866537, 3.0733052055537704, 4.654469825155756, 3.831137993548093, 3.6611503128504856, 5.54172283095898, 5.225634632781117, 4.259191049688608, 3.552396931075473, 3.7355127915256077, 1.963593578066418, 1.9633196335307688, 1.9575981232442534, 3.8662235391694426, 1.6194132366662428, 1.0057273330511727, 1.0057242362868568, 1.0057210204162208, 1.0057160179507871, 1.0057072040831185, 1.005697437364891, 1.005695174344814, 1.0056913629425788, 1.005675521802039, 1.0056751644830795, 1.0056750453767598, 1.0056696855923666, 1.005656703003503, 1.005653725345507, 1.0056422911388017, 1.0056370504607284, 1.0056363358228093, 1.005617755236913, 1.0056165641737145, 1.0056045344354099, 1.0056019140963732, 1.005598221800458, 1.0055919091655061, 1.0055894079327894, 1.0055883359759106, 1.963363941081752, 1.958579440213477, 1.9631599119558532, 1.9502631987490837, 1.9052377939760858, 1.872736180522415, 2.2637378117862488, 4.107339514517082, 3.9650753529016107, 2.9208940680623874, 2.6284966284917592, 2.673789188740175, 17.112496643381284, 1.9626859879091798, 5.2456490946559615, 4.533492874851194, 4.635152024542169, 5.024536787514072, 2.6574790075128956, 1.8808119463270363, 22.467099435180963, 4.236447906680116, 1.8865882455206813, 1.6124433730414058, 1.6125057847530058, 14.355230122887498, 29.551509037091233, 2.152585173765508, 6.516631204472991, 4.521464565822433, 13.716155158814772, 6.839281172859609, 5.598552544709129, 6.610862407632995, 3.375546950396974, 5.084217152987429, 4.767825030645598, 6.6783337557002165, 4.777737058583309, 6.23969376521857, 5.544656934978127, 8.3446611851596, 3.3175869566074954, 6.876678175165333, 5.701753883029489, 6.455124700903732, 4.110806937700483, 5.2923735510815835, 4.214476125645204, 4.5123486445265835, 4.135037450984983, 1.8119661378104603, 1.8119293660644535, 2.72456100388478, 0.928128160996355, 0.9281139267720944, 0.9281132348306372, 0.9281116532501638, 0.9281057223233885, 0.9280772538748672, 0.9280706310066347, 0.9280700379139571, 0.9280673689969083, 0.9280538267141047, 0.9280460176605173, 0.9280298064606648, 0.9280287191240894, 0.9280266432997181, 0.9280172526656572, 0.9280134964120328, 0.9280133975632532, 0.9279972852121804, 0.927996889817062, 0.9279920462268622, 0.9279914531341846, 0.9279809751635483, 0.9279809751635483, 0.9279785039440586, 0.9279730672611812, 0.9279695087051161, 0.9279660489978305, 1.8112828950459472, 1.8119194811864947, 1.8117249467882652, 1.8120060727174139, 2.6958636215898775, 1.8118170738508415, 1.5851025542753054, 1.244196441787888, 35.79555658065799, 2.328728776228991, 1.8118799416746594, 2.805561647830463, 17.77817126702738, 11.091029976555172, 6.784518409429812, 9.321363999296294, 8.293251286234359, 1.8120060727174139, 1.8117621139293905, 18.45845331289419, 1.8120278194489232, 3.69099761401539, 2.705631857988775, 1.8118676844259904, 3.334991361958092, 2.695595148304516, 9.521193528950674, 9.076432539281685, 12.877806282165634, 4.39639239003412, 5.6160447786990595, 3.336762732088311, 5.466886715042017, 4.21820481218276, 4.259299808413603, 4.462904570658014, 5.196740117539836, 4.425203646123108, 4.634349870951229, 5.591722047993608, 3.700787201750678, 3.57880227220729, 3.4814125006058982, 5.11843527751108, 4.538009477993944, 3.8619328546059295, 4.05147657565149, 1.7297833896072714, 0.8860751679081103, 0.8860701763623617, 0.8860519664639829, 0.8860297818162116, 0.8860168407716785, 0.8859989081813968, 0.885992160351033, 0.8859774630218846, 0.8859754294291723, 0.8860143449988043, 0.885932539110148, 0.8859297660291766, 0.88592717782027, 0.8859201526818091, 0.8859132199793806, 0.8859093376660206, 0.8859064721490169, 0.8859064721490169, 0.8859062872769521, 0.885900186498815, 0.8858988923943617, 0.8858916823838361, 0.885869775044162, 0.8858667246550935, 0.8858628423417335, 0.8859276400004319, 0.8858610860571183, 0.8858598843886973, 0.8858508256575242, 1.7296138619238866, 1.729626618096355, 1.7296199627020237, 2.5731592113339943, 3.1174562167027653, 15.801521184993161, 32.23866210816104, 1.9035224586272788, 1.7295940806129573, 23.849052449190843, 2.5730684391501972, 3.8043520707592657, 3.3668308843455796, 1.8755136013279032, 4.632501644360303, 22.33327751636576, 5.105052759348815, 15.77354782313022, 3.416540024606466, 2.729135217755682, 4.244208191350869, 1.7297484487870318, 1.7296611891724651, 4.510398452260109, 5.698986804873215, 2.2385507572101715, 2.573453342789027, 2.5734627712643303, 1.6006439667223176, 4.709485330079339, 10.605717687006775, 4.503494220129588, 7.147630254054441, 4.260326447189021, 5.63635880472691, 3.5569666265332676, 3.514383195136485, 3.476346877044535, 2.980067247672033, 3.3578664379253746, 3.5504591298537207, 3.3684999093462316, 3.6452548692682383, 3.5436691486592213, 3.0910439145546555, 3.095662018732357, 1.6448906574514108, 1.4936116708536264, 2.452991791971888, 2.452011812568482, 1.6483370593878426, 0.8445182710202246, 0.8445041375675023, 0.8445008067538033, 0.8444969358081533, 0.8444923446865684, 0.8444806418276264, 0.8444788413877892, 0.8444748804201472, 0.8444636276711646, 0.8444620972973028, 0.8444583163736447, 0.8444570560657586, 0.8444523749221818, 0.8444442729429142, 0.8444421124151096, 0.8444384215134433, 0.844427528852428, 0.8444212273129976, 0.8444193368511685, 0.8444168162353964, 0.8444123151358034, 0.8444100645860069, 0.8444068237942998, 0.8444032229146253, 0.8443971914411706, 4.864502530357826, 1.6486899455959403, 1.7701956887593429, 4.051397973529678, 5.225357485175118, 1.648539788913515, 1.648485055542463, 7.382777730695837, 3.2505039996776453, 1.4086877144130865, 20.596416707698868, 13.168307502781175, 3.257043557254444, 14.629652501069836, 5.600086989466096, 5.016072558055299, 2.8043281814576955, 1.6486465549958629, 5.449893578068171, 1.6483858513074314, 4.89562745400001, 1.6486024442198506, 6.529304432013182, 7.5218595475942935, 2.4440778143377466, 6.072481712815754, 4.1752131408723, 4.26130297212944, 5.6699429748868395, 9.57339880699107, 3.3400802026356042, 5.5265631480089406, 7.948019336011771, 4.01889859411575, 4.74466345435175, 3.9620299014169036, 5.073157303534614, 3.758738998603201, 4.864291518808902, 4.391155014246304, 4.447550911443893, 3.2574497364817234, 6.138824862707998, 3.850667587687886, 1.5632656905233144, 1.5632138593925404, 1.563102544212355, 1.563089847324615, 1.5630691496582994, 1.56300201269025, 0.8007884484718938, 0.8007375739559496, 0.8007332257067236, 0.8007183546943707, 0.8007147021650208, 0.8007127889353614, 0.8006866124750209, 0.8006836556655472, 0.8006800031361974, 0.8006767854317701, 0.8006716544976835, 0.8006706978828537, 0.8006685237582407, 0.800667567143411, 0.8006666974935659, 0.8006663496336277, 0.8006602620847114, 0.8006599142247733, 0.8006529570260117, 0.8006480869868786, 0.8006473043020179, 0.8006438257026371, 1.5631987274852341, 9.026858948027968, 5.375738083080077, 7.438161043501675, 1.6637166830526913, 2.326028612548883, 2.325085912116687, 2.639006895337669, 1.5630361029641817, 18.259312859539325, 7.6787722840720445, 1.060181864608971, 12.163415304630117, 1.0455743561591577, 2.133956356298407, 1.5633006504470914, 9.54327302766448, 1.5632789092009614, 8.187708070759719, 8.082442870617287, 1.5631761165892588, 1.5628244301918603, 2.0259404536956103, 3.057100972425481, 2.82879388572493, 9.519653337868867, 11.508274681687407, 3.742981978096296, 5.619389271649325, 7.9128722823214375, 2.966245870218003, 4.500477257079523, 3.38346038345238, 3.440946673239622, 3.087810743479036, 2.3256684035830015, 2.6927641265886963, 4.101520520555199, 3.3081191387633817, 2.6226642615167144, 2.7457400644187966, 2.4459587180309104, 1.7705887046772804, 1.77044652567558, 0.9070473932431631, 0.9069960460153869, 0.9069851358118459, 0.9069601982037523, 0.9069491148223773, 0.9069429670092709, 0.9069362996626625, 0.9069266882928764, 0.9069235710918647, 0.9069227052026948, 0.9069179428122602, 0.9069150853779995, 0.9069118815880708, 0.9068918795482457, 0.9068904941255739, 0.9068894550585699, 0.9068878098691471, 0.9068845194903014, 0.9068814022892897, 0.9068801900444518, 0.9068755142429342, 0.9068718775084206, 0.9068639979169744, 0.9068602745935437, 0.9068546463139392, 0.9068533474801843, 0.9068526547688484, 0.906846247188991, 1.7706477583186688, 1.315054976231109, 2.842740641050546, 2.6342838846168926, 4.596693190284019, 2.682634789509465, 2.1765435242101008, 1.7705038475386286, 2.6341661236897833, 2.987718716081817, 1.1876070872115856, 19.54088526371013, 2.819173562335044, 17.723119280321185, 2.6343254472970483, 3.257734131272476, 4.362058352931892, 12.246107050279958, 3.737965962867897, 3.0091491266816286, 9.136721207865909, 4.360812511594213, 3.1774752104684123, 12.3234662814318, 7.020347456184265, 2.891216407161421, 4.265823430587166, 3.286728603305026, 6.62644546754104, 7.836950214550863, 6.998545059597057, 4.121131964871138, 6.8402840715214035, 3.617544331517676, 3.519289116561697, 3.6912457004601142, 2.979215684433219, 3.4362558868572144, 3.008496938958846, 2.9719699238593424, 2.855701443323718, 1.640274373805413, 1.536180825153087, 1.5043237119575426, 0.8403191090716701, 0.8402773509244073, 0.8402619536152814, 0.8402594545755804, 0.8402493778025922, 0.84024623384942, 0.8402454277075809, 0.8402417194551213, 0.840232690666524, 0.840225032319053, 0.8402086676397204, 0.8402077002695134, 0.840200444992962, 0.8401943989291691, 0.8401841609278132, 0.8401806945179053, 0.8401798077618824, 0.8401782760923882, 0.8401702952881815, 0.8401607022002969, 0.8401582031605959, 0.8401577194754923, 0.8401532050811937, 0.8401519152542513, 0.8401490131436307, 0.8401378077720679, 0.8401346638188956, 1.6404646232794287, 2.4401455579453435, 3.0167123893296686, 3.240210699811749, 2.703086937556789, 3.4056677652524967, 2.048576186495214, 3.240709862838488, 7.7865465953473105, 2.0342273454452426, 8.842046162791569, 4.553223571408398, 1.0850958558521886, 1.078116118581333, 3.582768420959735, 1.6401800552102443, 9.4236962021562, 1.640096377687351, 1.6684843400927316, 2.2170104949759386, 1.520389473896321, 2.8358515656283694, 7.688722250548902, 3.8309878770780967, 2.572500504734319, 1.8749845049849891, 2.804737715208428, 9.579281577092688, 4.352024434021359, 3.5047490464063507, 9.593924337457178, 4.409195045876519, 6.822040449214095, 3.2406157054716873, 3.63635653848216, 3.8870418212578053, 6.519725006214707, 4.703530974496263, 4.197578943645288, 3.863291270395739, 3.781344050345772, 4.740485483768483, 3.433513516656933, 3.502702413505373, 3.1028989480828275, 3.360198785876239, 1.6953522125407596, 1.9938532184848, 1.411816587286677, 1.3392393230447164, 0.8684236694338785, 0.8683682770183463, 0.8683476128973746, 0.8683310344581161, 0.8683277344844248, 0.868318777412977, 0.8683013346948946, 0.868299684708049, 0.8682970918715772, 0.8682841276892186, 0.8682613421565883, 0.868255292204821, 0.8682522279435362, 0.8682487708282406, 0.8682484565450319, 0.8682415423144406, 0.8682400494691993, 0.8682387923363646, 0.8682381637699472, 0.8682372994911233, 0.8682257495832038, 0.8682237853131495, 0.8682124711176366, 0.8681975426652235, 0.8681896070142041, 0.8681786071018998, 2.5211945143075942, 3.1541997105923594, 1.694890059082376, 1.6951185429750961, 2.521911394306624, 6.181635803223909, 7.440759307841341, 3.34882493039029, 16.125738181977454, 3.2444181627630932, 4.951717065774311, 1.6185245821844296, 3.6254274680605967, 5.088365204931736, 5.603645725479318, 2.5216048110365437, 1.6948532879469589, 2.4147332773451686, 3.1704076099478864, 2.5221518209612745, 2.5218163236359943, 5.5623671402831105, 1.695111628744505, 4.701327319125077, 1.695205913707113, 5.071855279412655, 3.895135574981796, 7.3750954884826125, 8.536788055573005, 6.549873896052947, 6.771201560111095, 7.305430843877218, 3.5482391977222907, 3.887540292677303, 3.296596403919996, 4.538945042273418, 3.556289562112972, 3.654127810678477, 3.4894496949873135, 3.770648624584761, 3.5431603810698054, 3.6220533235320644, 3.9580365306529, 1.744796839570821, 1.6234138006787262, 0.8936755954167961, 0.893650320863966, 0.8936373109064771, 0.8936336518559334, 0.8936324321724187, 0.8936185413323915, 0.8936165762867292, 0.8936140691595047, 0.8936020756049446, 0.8935995684777202, 0.8935947575038571, 0.8935932667795615, 0.8935850000357404, 0.8935845257143736, 0.8935822896279302, 0.8935767332919193, 0.8935676534257552, 0.8935598610033009, 0.8935576926770527, 0.8935534915449469, 0.8935431919952681, 0.8935408881486295, 0.8935351962922281, 0.8935307918795365, 0.8935295044358267, 0.8935219152939581, 0.893503281240263, 0.8935003675518671, 1.7443372899266028, 1.7445092653021597, 3.229057896713043, 5.701810645130006, 2.5958624526045417, 5.666683217828337, 2.934905739336957, 1.8184375356106537, 1.5468400368270927, 2.2429813143175745, 3.4344937987682194, 1.7447069895519132, 1.7445897644141224, 1.744703737062541, 7.536565377839587, 3.3247762195708064, 5.025743867565332, 1.7446535945180524, 2.1256496574997663, 8.225368903785922, 8.885058855279846, 1.2718245782744875, 1.738782715680783, 3.7000432935641143, 2.591882760816835, 6.884229846897842, 1.4258907987050193, 1.6922802488812934, 4.758855973392763, 2.8134098119595676, 5.570835608518027, 2.6148255498077795, 2.59492329629831, 3.1710695347362967, 4.262355610538874, 3.512540262703521, 3.1206597440376322, 2.4093958816905667, 2.057146674984546, 2.1474619353930793, 2.369640975134554, 2.026000836756051, 2.0452520503095117, 1.53505464293248, 1.535029091852153, 1.5349902384052638, 0.7863358799854693, 0.7863106581716413, 0.7862795095608301, 0.7862733193506478, 0.7862602145439852, 0.7862566584657953, 0.7862481633901196, 0.7862468463241233, 0.7862443438987304, 0.7862394049012446, 0.7862390756347455, 0.7862354537032559, 0.7862351902900566, 0.7862291976397737, 0.7862239293757887, 0.7862188586717032, 0.7862163562463104, 0.7862153684468132, 0.7862043709457445, 0.7862012099873535, 0.7861959417233686, 0.7861879076207915, 0.7861850100755997, 0.7861844173959015, 0.7861750003740283, 0.7861744735476298, 0.786164595552658, 1.5349127949246848, 1.5345059532384457, 1.1553241016930513, 2.283493726915189, 2.2836396578275724, 7.884866882693118, 3.0324256569936914, 1.5347054887368763, 1.5345237336293949, 1.5343359200183309, 3.781240368198528, 7.632888450424146, 9.472628399489597, 2.283692340467422, 1.534689420531722, 1.5350267211333597, 1.518294319597306, 2.593941723608117, 1.5349790433442956, 2.2831720993989064, 3.0321891119407662, 1.5344960752434738, 1.7956879109825559, 1.534883292646369, 1.5349170095358726, 5.693960788008129, 3.0187410778795405, 4.7811444921248745, 7.007603466312413, 7.218559507409645, 3.7762331466940164, 3.0340517066726513, 2.488105461108524, 3.7810125157811783, 3.3425217336524238, 2.2836217457300236, 2.6538392509851976, 2.28357038015617, 4.181333671686051, 2.283055670764839, 2.2834679124216626, 2.283412332236621, 2.2696152756863914, 2.2837660961632116, 2.282587058683376, 1.9977229353718735, 0.7527918429859515, 0.7527482314298894, 0.7527358447157417, 0.7527230064026406, 0.7527141034518469, 0.7526945556685823, 0.7526922331596796, 0.7526853946612438, 0.7526665565334775, 0.752666040420388, 0.7526620405439444, 0.752661588944991, 0.7526612018601739, 0.7526570084413218, 0.7526536537062402, 0.7526475893774386, 0.7526452023543997, 0.7526406863648667, 0.752640492822458, 0.7526357832905165, 0.7526239772035943, 0.752622815949143, 0.7526199128130147, 0.7526150742528006, 0.7526078486695478, 0.7526028810810614, 0.7526004940580225, 0.7525688176171549, 0.7525602372370421, 1.4693222254455798, 1.4694971877829175, 1.469469317676085, 1.4543377850900956, 1.3962973844134026, 2.8885979968293305, 1.3150192499388915, 1.9832918972765967, 1.4693397732906224, 1.4693192577953151, 7.510317167112036, 2.024444045438943, 2.1863546569130112, 1.3276571821330019, 1.4694131903776027, 2.902670336327411, 11.456646361524673, 1.4541247594124078, 9.258782898829088, 2.186299948925525, 1.4693867395817661, 1.4694712531001706, 1.7309596918345833, 1.2332386793717605, 4.60412306866612, 1.954249439561266, 5.90785982841218, 1.1403945795900134, 2.670190549900967, 2.2793363008331076, 4.324490158157539, 3.3103705166362785, 3.700193829807878, 5.06651239833334, 3.996433969253579, 3.119816918978146, 2.9031059357749416, 3.0945906014465168, 3.3933682108967034, 2.639190475237194, 3.0218431711999414, 2.5160766008133053, 2.1859249927660094, 2.104119906827127, 2.18690431735332, 2.49571929416794, 2.0158356661910393, 2.37662745804288, 1.2692750292045696, 0.8184033126591067, 0.818380867369698, 0.8183772532976745, 0.8183752877497319, 0.8183569637705254, 0.818353730127136, 0.818347389649902, 0.8183345184811167, 0.8183289388611507, 0.818326402670257, 0.8183198719787059, 0.8183172089782675, 0.8183090297626356, 0.8183062399526526, 0.818299265427695, 0.8182922909027376, 0.8182717477564991, 0.8182700992324182, 0.8182633149217777, 0.818262997897916, 0.8182590033972585, 0.8182558331586415, 0.8182427717755393, 0.8182423913469052, 0.8182361142744434, 0.8182357972505817, 0.8182285057017625, 0.8182284422969902, 1.5976458089676988, 4.576446396888371, 2.3766299942337734, 1.597603835008409, 1.9044320825114165, 2.641506220498271, 3.847752340852281, 2.664139441652203, 4.254235010329536, 13.246465183375978, 11.417403875460842, 1.2227706721215206, 5.771725774199193, 5.660875971572696, 1.3082826596251083, 10.037806317716562, 3.156578185658521, 1.5975779658612939, 1.597713144835925, 3.8894315947601803, 7.794161492990796, 1.5975664261927278, 1.947094490819235, 1.2999969239755869, 5.7652442847514065, 2.9606348670258873, 3.0176867347974055, 2.9297717064220943, 2.698465010063993, 3.914881509520588, 4.555224058728571, 4.633295115769308, 2.79746725054994, 4.215705705511537, 2.2322459147543023, 2.5407382839121717, 2.3334904016093168, 2.0423937657909232, 2.376369020190818, 2.211557698396703, 2.235391045081505, 1.5999181981879935, 0.8194919497980961, 0.8194851671003315, 0.8194783215998098, 0.8194541425383343, 0.8194211710908675, 0.819419977838483, 0.8194131951407184, 0.819408045314638, 0.8194028326858003, 0.8194012626168733, 0.8193992529286468, 0.8193960499880357, 0.8193952335521937, 0.8193877600241013, 0.8193873832075588, 0.8193806005097942, 0.8193797840739522, 0.8193641461874394, 0.8193573634896748, 0.8193568610676182, 0.8193562958428044, 0.8193522136635942, 0.819340846364563, 0.8193199330464555, 0.8193068700729829, 0.819302976302044, 0.8192994593476475, 0.8192875268238025, 0.8192856427410901, 7.601067429111721, 1.5971933121638295, 1.5972994488232939, 3.6140620291347565, 6.9779877107307815, 2.301645181231981, 1.5996882144915685, 1.5997325532380666, 1.5998075397300195, 7.047147614092863, 2.6637937629763417, 1.8890364682569798, 3.5139853333065663, 11.97364206621415, 2.0784210711569435, 1.5555410164025558, 6.473737340230933, 1.5994653903094507, 2.274300358377524, 2.380023775700243, 5.188751731776357, 2.802207522625095, 2.3779595746805553, 3.0335593143103288, 2.380124511322599, 3.822690527253497, 4.605625620183205, 3.0811751086728405, 3.9405288476278746, 5.342732536427912, 4.578670174422641, 3.101819128557947, 4.1490998184211305, 3.1070671781505355, 3.028794594736224, 2.498470277974178, 2.992234597729982, 3.147862844305245, 2.3084379274376965, 2.464757506762891, 2.316028268658321, 3.035748346533288, 1.5367638930094607, 0.7870910123732604, 0.7870823265655762, 0.7870781202310373, 0.7870698168174022, 0.7870692705401893, 0.7870626605859139, 0.7870514072753294, 0.7870391160380404, 0.7870333254995844, 0.787033052360978, 0.7870208157514104, 0.7870163362782651, 0.7870135502644795, 0.78701229382689, 0.7870084698864002, 0.7870061755221062, 0.7870024062093376, 0.7869965610431602, 0.7869767311803341, 0.7869466859336278, 0.786932100332045, 0.7869099214772037, 0.7869090474336632, 0.786887687994641, 0.7868591723241308, 0.7866834895724812, 1.5365130425133242, 1.128390101588181, 1.5364756771519659, 1.5366191295480578, 11.871962098689087, 1.5361869150172582, 1.5361622232872378, 1.5365815456758145, 1.4226622067753385, 4.003755282371948, 2.3160757539887062, 1.5366557301213182, 4.523323035678842, 1.425492141248738, 4.772266805831563, 1.444177116292182, 7.014374221581911, 2.3311783523257836, 6.894966326268583, 1.5366422917018823, 1.8818701520038315, 1.6454885726375923, 2.893476783264613, 2.3504029400000714, 1.2552211394359156, 1.53659837101397, 1.5367098115653894, 1.291758344539549, 4.534219736498816, 3.4244689411790854, 3.2437429583005915, 2.6980257888585566, 2.1672359720237204, 3.6658413091618365, 1.926897816060319, 2.138720083002554, 2.035066385485779, 1.7703648121068982, 1.6229998693611392, 1.591939421082561, 1.5711358737522787, 1.5686958719534048, 1.5367396383012104, 0.7838534939193449, 0.7838398922973169, 0.783832500111432, 0.7838041633988734, 0.7838037198677203, 0.7838004180246918, 0.7837877034649698, 0.7837655269073153, 0.7837599088460429, 0.7837562620343397, 0.7837516295978518, 0.7837421183186799, 0.7837399006629145, 0.7837384715069767, 0.7837358103200582, 0.7837206809796139, 0.783707769294935, 0.7837059951703227, 0.7836914572047492, 0.7836866769245436, 0.7836838678939074, 0.783682931550362, 0.783671251896664, 0.7836619870236883, 0.7836618884612098, 0.7836424716529523, 0.7836416831531245, 0.783565395794793, 0.7835171987428238, 0.7834937901541884, 1.5301857307244011, 3.4989770532406252, 1.508282092572739, 4.630186503576892, 1.5302049504077015, 2.276330797382853, 2.987857074792841, 2.985421595949981, 2.516915696441229, 1.52990206791138, 2.226748745466289, 6.041001541070706, 3.022577481956602, 2.8240531349582008, 1.5301493611698476, 3.9763947749271877, 2.2722134484074736, 3.02196777446482, 5.877237617602322, 3.2329376056189436, 2.702339513052032, 3.3670756193104436, 1.530213722468285, 2.250834655453635, 1.1018175278832585, 1.5302266341529638, 3.6826241826146258, 2.4191710894229477, 1.5303404738155904, 1.6501867310047693, 3.211926450709778, 2.2222497625742945, 2.2296360347104303, 1.8353819812280128, 2.0978737709998403, 1.7632575077368575, 1.9963648658040398, 1.7322081586459885, 0.7270799239390875, 0.7270583590601623, 0.7270454111098452, 0.7270384634291873, 0.7270376062478074, 0.7270319217818146, 0.7270074695550836, 0.7270049882405628, 0.7269819796877347, 0.726979498373214, 0.7269619938271408, 0.7269586553312403, 0.7269497677137753, 0.7269449404291622, 0.7269418275073091, 0.7269363686153636, 0.726931766904798, 0.7269305939197518, 0.7269089388112077, 0.7269025776230728, 0.7269008632603131, 0.7268975698792219, 0.7268935095463699, 0.7268788923481027, 0.7268669369235939, 0.726853808514039, 0.726822318377031, 0.7268011144165816, 0.7264640616750537, 1.0741507698275194, 1.4193893228442593, 1.1729235095413861, 2.523082172206947, 8.730755077070892, 4.4674990603264275, 7.061280469989843, 4.383786907227305, 1.41929954437342, 2.803948569106107, 1.753492999506835, 1.419604430255798, 3.2138176958690536, 4.596524888972662, 3.0156149838970054, 4.6306280757449425, 5.547694853704047, 2.7082941650225973, 1.3267005858758716, 4.42905871623123, 3.123629582559802, 1.419194336637743, 1.592808304465241, 1.4193914883551138, 1.9491021513122457, 1.537352097913001, 1.15732741013827, 1.653072666951234, 2.148713528095091, 1.5767701701589671, 2.2785229107521485, 2.3950053786926597, 1.9183766200951395, 1.752339684517621, 1.8119585443097603, 1.6000667359307645, 1.5776674134896465, 1.490734784831532, 1.4718336647164836, 0.6505247433617911, 0.6504972782803263, 0.6504838092638299, 0.6504749665616951, 0.6504535332571832, 0.6504265366632488, 0.6504246627131275, 0.6504035807742633, 0.6500555531001804, 1.269973069131418, 0.6392003456571291, 0.6285900986314915, 0.628056608456345, 0.5545938401198531, 0.5564275588744482, 1.035383911542086, 1.2700373690449538, 1.2259497643952275, 0.4794033776129705, 0.6503714308174955, 0.6504597407169599, 0.6503660432108969, 0.650374183181736, 0.6505406719378218, 0.650390170318708, 1.2700369005574235, 0.6504695203941552, 1.2698401357946911, 0.6503975489973105, 0.650076517917162, 0.6504285862961939, 1.269661876289406, 4.986546136110365, 1.5465708008328196, 0.8823585501025154, 3.867967037716078, 1.2699511673393757, 3.0789925755188245, 0.9466725768251293, 1.1162050378344153, 2.760330636811128, 1.2544048771333958, 2.2921164689698528, 1.278617366797029, 1.2315558033048621, 1.6607689698568087, 1.4426758438192882, 0.9305953731254195, 0.8728820429002405, 1.2037910069426134, 1.2699408606137088, 1.1840236440884933, 1.3558901685347509, 1.2252441050526903, 1.1823034749990584, 0.9945040410124282, 0.9806978306159214], \"Total\": [169.0, 91.0, 155.0, 185.0, 246.0, 198.0, 150.0, 61.0, 142.0, 100.0, 48.0, 83.0, 14.0, 19.0, 67.0, 37.0, 30.0, 46.0, 86.0, 70.0, 66.0, 56.0, 25.0, 50.0, 34.0, 21.0, 49.0, 32.0, 72.0, 37.0, 5.603748501091272, 2.413758977704754, 2.4135734913449385, 7.183357535144544, 4.8672236908849165, 2.4166424199028835, 2.4200302721427147, 3.181197075096454, 8.954961663364088, 2.394529861986907, 2.357180798323183, 2.3953466775788503, 3.0672526185176676, 3.1364048877599466, 1.6080437714566422, 1.6080417122570883, 1.6080191402619783, 1.6080130418632994, 1.6080124874634196, 1.6080110618637284, 1.6080078146644319, 1.6080066266646893, 1.6080041714652211, 1.60799466746728, 1.6079829458698194, 1.6079829458698194, 1.607982233069974, 1.607982074670008, 1.607981203470197, 1.6079776394709688, 4.700974041917554, 4.757416596083279, 3.1894610000303434, 3.2179536548899, 3.1214816269049153, 3.277564107605687, 4.003704421362182, 3.2625153009110908, 34.91160519345983, 16.070505384789104, 5.460531588507867, 4.779568647031122, 10.307606542009385, 10.44839828850704, 3.266733954930378, 3.455245660354939, 12.383275968775743, 4.930441607560773, 67.13655543724771, 100.54286782276854, 14.581171396461468, 13.074993381626475, 155.0857093034151, 246.96695121258134, 5.712322158392551, 32.03833721888798, 185.03814757509102, 26.606936391188245, 142.64909071060328, 56.23273732890741, 22.772216746713447, 198.60069866234727, 51.638306489357625, 30.729642405272276, 37.3822822700304, 70.82288098120331, 11.254524238741979, 83.31185378326795, 86.44308126247893, 19.259797144114685, 150.55767347470498, 46.42471726568493, 169.03929776373798, 91.57141796795516, 66.47002224727925, 90.32706620958497, 61.602797975814305, 48.575504903848504, 42.846486735471636, 2.816766152371498, 1.84163992589103, 1.84162802595306, 1.8415883117022436, 1.8415779171781128, 1.8415777738053662, 1.8415575582480914, 1.8415481673331875, 1.8415079512777577, 1.841497700126374, 1.8414856568156568, 1.8414824309288578, 1.8414776279418457, 1.841474760486913, 1.841470889422754, 1.8414705309908876, 1.8414698858135277, 1.8414695273816613, 1.8414623587443295, 1.8414528961430523, 1.8414417847551883, 1.8414389173002557, 1.8414282360306318, 1.8414249384574595, 1.841423289670873, 1.8414132535786092, 1.8414069451777573, 1.841405726509411, 1.8414022138771184, 1.8413967657127468, 8.288109581607547, 3.6864911702622307, 7.097224574933559, 14.789958502307682, 47.36868472707816, 5.490030898912845, 7.923306892001452, 7.84733771138721, 7.013209011651796, 7.856227657758491, 6.469555422487545, 6.932776310926109, 6.9492044053482624, 246.96695121258134, 3.6013681310812475, 10.439899603543374, 18.449223751523206, 18.038026882890094, 4.379616104283372, 6.937171789035648, 8.725073608939066, 5.235113535651703, 4.439564502391397, 86.44308126247893, 19.46779138215442, 42.98602391313575, 19.21429964824774, 7.601398232149421, 4.499396361724387, 169.03929776373798, 34.4788844216265, 32.03833721888798, 21.825864450697775, 19.11482290187962, 47.168223224108765, 37.587336643711915, 142.64909071060328, 14.040178539870587, 198.60069866234727, 185.03814757509102, 18.56632329453585, 51.638306489357625, 61.602797975814305, 36.56535764640446, 150.55767347470498, 70.78587171968042, 53.80737617850395, 49.931425588334456, 22.316197131248888, 155.0857093034151, 91.57141796795516, 100.54286782276854, 3.078377793897907, 3.078084754171288, 2.3197426750929067, 2.319683110283169, 2.319664810251262, 2.319652131797784, 2.3195462786720458, 2.3194726001122103, 2.324071454221996, 3.0618882036886848, 2.321521991406626, 3.858156305832382, 3.857590998292742, 3.858592701016013, 3.941805972007264, 3.1311748981118024, 1.5609533217052631, 1.5609490756194286, 1.5609410020759398, 1.5609378324625702, 1.5609276059741515, 1.5609258118533764, 1.5609113392791232, 1.5609041029919963, 1.5608945941518877, 1.5608925608150093, 1.5608858029600892, 1.5608856235480115, 1.56088550393996, 1.5608826333467196, 4.581816275913196, 3.06923430129535, 4.662784039206075, 4.712437609486374, 3.124214222766716, 3.1254270926896814, 3.146703027731577, 3.1651348868041587, 3.205794209200084, 10.4349864950085, 6.239467716792785, 3.281608758944077, 3.284345751680792, 38.53189397009943, 150.55767347470498, 246.96695121258134, 198.60069866234727, 24.588641262600245, 37.6127902785303, 70.82288098120331, 90.32706620958497, 15.96527396954616, 11.113053034749123, 15.178557053119782, 25.747044458685643, 56.23273732890741, 28.38642088993745, 67.13655543724771, 23.44706571181554, 72.68560668675633, 47.168223224108765, 185.03814757509102, 42.98602391313575, 34.86036593195589, 53.80737617850395, 12.097397328049654, 10.285508292470977, 49.931425588334456, 25.09713658377447, 22.058615659975583, 169.03929776373798, 155.0857093034151, 83.31185378326795, 28.866007983872866, 86.44308126247893, 2.717725580583017, 2.7174516360473677, 2.71787948591295, 5.391786323912952, 2.6628664320786717, 1.7598593355677716, 1.7598562388034555, 1.7598530229328195, 1.759848020467386, 1.7598392065997175, 1.75982943988149, 1.7598271768614129, 1.7598233654591777, 1.759807524318638, 1.7598071669996782, 1.7598070478933587, 1.7598016881089655, 1.759788705520102, 1.7597857278621059, 1.7597742936554004, 1.759769052977327, 1.759768338339408, 1.759749757753512, 1.7597485666903132, 1.7597365369520086, 1.7597339166129722, 1.759730224317057, 1.7597239116821048, 1.7597214104493883, 1.7597203384925095, 3.480070719597567, 3.5107852736085277, 3.5230996980755083, 3.5110460095731146, 3.4521345078350443, 3.5045781413254593, 4.378164897260746, 8.806111016429824, 8.77275927024719, 6.766116797864328, 6.002525509323041, 6.13893371613659, 70.78587171968042, 4.284324071048524, 16.819064585410437, 13.949220023795576, 14.89861143676794, 17.043213528280933, 6.934659655241071, 4.254811672038577, 155.0857093034151, 13.979482404980576, 4.329985655499411, 3.454702661024453, 3.455616002160419, 83.31185378326795, 246.96695121258134, 5.261355267377376, 27.578215566260766, 15.96527396954616, 100.54286782276854, 34.4788844216265, 28.866007983872866, 40.51128246110746, 11.175008695910295, 26.606936391188245, 23.217990235381794, 48.575504903848504, 25.295239323983065, 66.47002224727925, 51.638306489357625, 185.03814757509102, 13.345956040403589, 142.64909071060328, 90.32706620958497, 169.03929776373798, 36.486827824532284, 150.55767347470498, 42.98602391313575, 70.82288098120331, 86.44308126247893, 2.5697976749679365, 2.5697609032219297, 4.0816862733935455, 1.6859596981538314, 1.6859454639295708, 1.6859447719881138, 1.6859431904076405, 1.685937259480865, 1.6859087910323438, 1.6859021681641113, 1.6859015750714337, 1.685898906154385, 1.6858853638715814, 1.685877554817994, 1.6858613436181413, 1.6858602562815659, 1.6858581804571948, 1.6858487898231338, 1.6858450335695092, 1.6858449347207298, 1.685828822369657, 1.6858284269745385, 1.685823583384339, 1.6858229902916613, 1.6858125123210248, 1.6858125123210248, 1.6858100411015353, 1.6858046044186576, 1.6858010458625927, 1.6857975861553072, 3.3177592377789145, 3.332250308736623, 3.331923761491893, 3.3821142885260964, 6.511373276613288, 3.937927439564576, 3.324374606188316, 2.4466154075305937, 185.03814757509102, 5.713475223078259, 4.101688027195636, 7.420655913414651, 91.57141796795516, 53.80737617850395, 31.036177909247144, 49.931425588334456, 50.93962956896941, 4.859987215797292, 4.8645750428736685, 198.60069866234727, 4.881489210053338, 15.742232676528676, 9.57742489514896, 4.975604676721405, 13.657667886434567, 9.828786753027, 90.32706620958497, 86.44308126247893, 169.03929776373798, 24.880192929141323, 40.51128246110746, 16.845828738549788, 51.638306489357625, 30.71464919868768, 33.71163644943962, 42.846486735471636, 72.68560668675633, 46.42471726568493, 67.13655543724771, 142.64909071060328, 36.486827824532284, 37.3822822700304, 34.86036593195589, 246.96695121258134, 150.55767347470498, 70.78587171968042, 155.0857093034151, 2.4896186593259557, 1.6459104376267948, 1.645905446081046, 1.645887236182667, 1.6458650515348958, 1.645852110490363, 1.6458341779000814, 1.6458274300697175, 1.6458127327405692, 1.6458106991478567, 1.6458965090076174, 1.6457678088288323, 1.6457650357478608, 1.6457624475389545, 1.6457554224004933, 1.6457484896980648, 1.645744607384705, 1.6457417418677014, 1.6457417418677014, 1.6457415569956366, 1.6457354562174993, 1.645734162113046, 1.6457269521025204, 1.6457050447628465, 1.645701994373778, 1.645698112060418, 1.6458186509497377, 1.6456963557758029, 1.6456951541073817, 1.6456860953762085, 3.2830836416941054, 3.2928426474144232, 3.316255081464304, 5.506836053819567, 7.260414651417116, 61.602797975814305, 169.03929776373798, 4.462918049880798, 4.057563816246818, 142.64909071060328, 7.587428172805895, 14.77209404877583, 12.302013087691996, 5.0860315402808425, 20.92870195290989, 246.96695121258134, 25.09713658377447, 198.60069866234727, 15.96527396954616, 11.062266233148979, 23.947439084068012, 5.13863418271791, 5.144419828124939, 26.99997626119585, 42.98602391313575, 8.359886554669352, 10.766301147657348, 11.037139161664365, 4.78382307106268, 37.3822822700304, 185.03814757509102, 34.4788844216265, 90.32706620958497, 35.424143727262724, 70.82288098120331, 28.866007983872866, 30.729642405272276, 33.37928157366333, 22.058615659975583, 34.86036593195589, 67.13655543724771, 49.931425588334456, 86.44308126247893, 150.55767347470498, 42.846486735471636, 100.54286782276854, 2.4111607218484683, 2.439908434288602, 4.020718188206127, 4.019580046436985, 3.126953383673817, 1.6063332900738505, 1.6063191566211283, 1.6063158258074293, 1.6063119548617792, 1.6063073637401943, 1.6062956608812524, 1.606293888152541, 1.606289899473773, 1.6062786467247905, 1.6062771163509286, 1.6062733354272705, 1.6062721053497033, 1.6062673939758076, 1.6062592919965402, 1.6062571314687355, 1.6062534405670692, 1.606242547906054, 1.6062362463666235, 1.6062343559047945, 1.6062318352890224, 1.6062273341894293, 1.6062250836396328, 1.6062218428479256, 1.6062182419682511, 1.6062122104947967, 9.437641371224991, 3.1691147601716954, 3.9855262758493195, 11.664643992024429, 17.216923598989524, 3.9277166120097036, 4.016224038841257, 33.37928157366333, 10.669121774278358, 3.3171303726216026, 155.0857093034151, 83.31185378326795, 11.29944856184374, 100.54286782276854, 27.578215566260766, 23.340368957389956, 10.946201324228467, 4.737462048225456, 34.91160519345983, 4.833924158426403, 29.895143176073482, 4.940183475814031, 51.638306489357625, 66.47002224727925, 9.748218956566278, 49.98799566206046, 25.75711098904983, 27.488294670112015, 48.575504903848504, 142.64909071060328, 19.61323136197176, 70.78587171968042, 185.03814757509102, 37.587336643711915, 67.13655543724771, 40.51128246110746, 86.44308126247893, 37.3822822700304, 169.03929776373798, 150.55767347470498, 246.96695121258134, 23.350746367424378, 6.902725300407817, 5.420285045810424, 2.327166128223132, 2.327114297092358, 2.3270029819121727, 2.326990285024433, 2.326969587358117, 2.3269024503900675, 1.5646888861717119, 1.5646380116557677, 1.5646336634065416, 1.5646187923941888, 1.564615139864839, 1.5646132266351795, 1.564587050174839, 1.5645840933653652, 1.5645804408360156, 1.5645772231315882, 1.5645720921975017, 1.5645711355826717, 1.564568961458059, 1.564568004843229, 1.5645671351933839, 1.5645667873334457, 1.5645606997845296, 1.5645603519245914, 1.5645533947258299, 1.5645485246866966, 1.564547742001836, 1.5645442634024551, 3.210630840521458, 21.98896762752579, 12.792386711799665, 21.93484213476127, 3.822337274407751, 6.3726277806945575, 6.608442442422717, 8.22233328311557, 4.084805270554137, 155.0857093034151, 47.168223224108765, 2.3936891764777286, 100.54286782276854, 2.3552886968475004, 7.125033157993711, 4.506055291409946, 83.31185378326795, 4.652229522338057, 70.78587171968042, 70.82288098120331, 4.724054548537671, 4.748345452511885, 7.61206262704077, 16.5327777852986, 14.581171396461468, 150.55767347470498, 246.96695121258134, 26.99997626119585, 72.68560668675633, 198.60069866234727, 19.174120944194698, 66.47002224727925, 29.895143176073482, 50.93962956896941, 35.76707126634329, 11.73845500996545, 25.75711098904983, 185.03814757509102, 90.32706620958497, 28.866007983872866, 169.03929776373798, 20.681628134454485, 2.5294266897187327, 3.2461264561331062, 1.6658853782846155, 1.6658340310568394, 1.6658231208532983, 1.6657981832452047, 1.6657870998638298, 1.6657809520507234, 1.665774284704115, 1.6657646733343288, 1.6657615561333172, 1.6657606902441473, 1.6657559278537126, 1.6657530704194519, 1.6657498666295232, 1.665729864589698, 1.6657284791670264, 1.6657274401000224, 1.6657257949105995, 1.6657225045317539, 1.665719387330742, 1.6657181750859043, 1.6657134992843867, 1.665709862549873, 1.6657019829584268, 1.665698259634996, 1.6656926313553917, 1.6656913325216367, 1.6656906398103009, 1.6656842322304435, 3.2918915463944143, 2.4761636316971503, 6.705235820027098, 6.58368284889792, 14.85299398035659, 7.459960433131751, 5.675871300413083, 4.2600638736667, 7.487912443538787, 9.094234471608152, 2.4611784693163736, 169.03929776373798, 9.082345896488158, 198.60069866234727, 9.567269412911942, 14.340744504500346, 23.947439084068012, 142.64909071060328, 18.449223751523206, 12.819873115412708, 90.32706620958497, 25.09713658377447, 14.793722419859215, 185.03814757509102, 70.82288098120331, 13.905238765803619, 28.866007983872866, 18.989219494357915, 91.57141796795516, 155.0857093034151, 150.55767347470498, 42.846486735471636, 246.96695121258134, 37.3822822700304, 34.86036593195589, 49.931425588334456, 19.10335214588043, 100.54286782276854, 61.602797975814305, 70.78587171968042, 28.38642088993745, 2.402289551349499, 2.396936127307579, 2.3952991395967826, 1.6023342866157562, 1.6022925284684935, 1.6022771311593675, 1.6022746321196666, 1.6022645553466783, 1.602261411393506, 1.602260605251667, 1.6022568969992073, 1.60224786821061, 1.6022402098631392, 1.6022238451838064, 1.6022228778135996, 1.602215622537048, 1.6022095764732553, 1.6021993384718993, 1.6021958720619913, 1.6021949853059685, 1.6021934536364741, 1.6021854728322678, 1.602175879744383, 1.6021733807046818, 1.6021728970195783, 1.60216838262528, 1.6021670927983374, 1.6021641906877166, 1.602152985316154, 1.6021498413629816, 3.1649479577517083, 5.6968812109567315, 7.433336775243677, 9.587421068097823, 7.68794660882222, 10.683726398396173, 5.656800784004108, 11.683635202321485, 49.98799566206046, 6.338321272763483, 66.47002224727925, 23.350746367424378, 2.4215676138526554, 2.403664155373576, 17.304210802011998, 4.864257847484813, 91.57141796795516, 4.89666855557101, 5.04423114078806, 8.183343489199787, 4.319468798161315, 13.880037331886944, 83.31185378326795, 24.360551791529474, 12.134458857905408, 6.864101260545973, 14.89861143676794, 150.55767347470498, 34.4788844216265, 23.614220233191933, 198.60069866234727, 48.575504903848504, 142.64909071060328, 24.52845047829212, 36.56535764640446, 47.168223224108765, 246.96695121258134, 86.44308126247893, 70.78587171968042, 53.80737617850395, 51.638306489357625, 155.0857093034151, 40.51128246110746, 47.36868472707816, 35.424143727262724, 185.03814757509102, 2.456029760148696, 3.36611672032203, 2.4188573181100863, 2.442098310221677, 1.6291012170418153, 1.6290458246262831, 1.6290251605053114, 1.629008582066053, 1.6290052820923615, 1.6289963250209138, 1.6289788823028313, 1.6289772323159857, 1.628974639479514, 1.6289616752971554, 1.6289388897645252, 1.6289328398127578, 1.628929775551473, 1.6289263184361773, 1.6289260041529687, 1.6289190899223773, 1.6289175970771361, 1.6289163399443014, 1.628915711377884, 1.62891484709906, 1.6289032971911406, 1.6289013329210862, 1.6288900187255733, 1.6288750902731604, 1.628867154622141, 1.6288561547098366, 4.778695234259622, 7.40379898502038, 4.0519032327200675, 4.080283995630482, 7.308882938736402, 27.488294670112015, 37.587336643711915, 12.603910102379492, 150.55767347470498, 12.48571161795178, 25.295239323983065, 4.731663993977771, 19.46779138215442, 35.76707126634329, 56.23273732890741, 12.263418190659937, 5.841625356305852, 11.903533831147584, 20.321513044180815, 13.88679338734363, 14.040172481633343, 72.68560668675633, 6.399639166330922, 53.80737617850395, 6.51084156965023, 67.13655543724771, 38.53189397009943, 169.03929776373798, 246.96695121258134, 142.64909071060328, 155.0857093034151, 198.60069866234727, 35.424143727262724, 49.98799566206046, 30.729642405272276, 90.32706620958497, 42.846486735471636, 47.168223224108765, 47.36868472707816, 70.82288098120331, 61.602797975814305, 86.44308126247893, 185.03814757509102, 2.504267904893462, 2.497412284399638, 1.6531466607394374, 1.6531213861866074, 1.6531083762291185, 1.6531047171785747, 1.65310349749506, 1.653089606655033, 1.6530876416093707, 1.6530851344821462, 1.653073140927586, 1.6530706338003616, 1.6530658228264985, 1.653064332102203, 1.6530560653583817, 1.653055591037015, 1.6530533549505715, 1.6530477986145606, 1.6530387187483966, 1.6530309263259424, 1.653028757999694, 1.6530245568675883, 1.6530142573179094, 1.653011953471271, 1.6530062616148695, 1.6530018572021778, 1.653000569758468, 1.6529929806165995, 1.6529743465629045, 1.6529714328745087, 3.262548699237711, 3.311847270468284, 7.253875266586066, 18.53546270320531, 7.0558284548897765, 22.316197131248888, 9.060541951081222, 5.209972466991423, 4.019676591891755, 7.400905306137232, 18.107703551916472, 5.6129349795544785, 5.666016260005541, 5.685316522652941, 86.44308126247893, 21.236831722314957, 48.575504903848504, 6.213051808778822, 9.405320871130115, 150.55767347470498, 185.03814757509102, 3.4031870989116837, 6.529665710349835, 32.03833721888798, 15.645873185648606, 142.64909071060328, 4.566505133438799, 6.743697516649187, 72.68560668675633, 21.894458218565248, 169.03929776373798, 23.217990235381794, 23.44706571181554, 50.93962956896941, 155.0857093034151, 91.57141796795516, 83.31185378326795, 35.424143727262724, 16.200554302783353, 46.42471726568493, 198.60069866234727, 26.99997626119585, 53.80737617850395, 2.29964263083289, 2.2996172614475427, 2.299578422732679, 1.5509238678858792, 1.5508986460720513, 1.5508674974612402, 1.5508613072510578, 1.550848202444395, 1.5508446463662053, 1.5508361512905295, 1.5508348342245333, 1.5508325527795215, 1.5508273928016545, 1.5508270635351555, 1.5508234416036657, 1.5508231781904664, 1.5508171855401838, 1.5508119172761987, 1.5508068465721132, 1.5508043441467203, 1.5508033563472232, 1.5507923588461543, 1.5507891978877635, 1.5507839296237786, 1.5507758955212014, 1.5507729979760096, 1.5507724052963114, 1.5507629882744383, 1.5507624614480398, 1.550752583453068, 3.045936845960981, 3.057817609280593, 2.299826388287335, 4.661296858699385, 4.661677507376914, 19.195283846980605, 7.802436098515262, 3.8428230590563306, 3.888460841854179, 4.1407162138342395, 16.070505384789104, 48.575504903848504, 72.68560668675633, 8.142723112966165, 4.662323277764826, 4.6966031015761285, 4.700526341905978, 11.892777130070625, 4.8757718883955485, 10.307606542009385, 18.53546270320531, 5.359910814718979, 7.433896468392109, 5.541579017495163, 5.6700712619088085, 90.32706620958497, 25.295239323983065, 70.82288098120331, 185.03814757509102, 198.60069866234727, 49.931425588334456, 28.500205963720713, 18.05159587448997, 100.54286782276854, 66.47002224727925, 18.863874157083004, 36.486827824532284, 19.61323136197176, 246.96695121258134, 28.38642088993745, 33.71163644943962, 35.76707126634329, 42.846486735471636, 46.42471726568493, 91.57141796795516, 2.9642040538092984, 1.5189791124777405, 1.5189355009216785, 1.5189231142075308, 1.5189102758944295, 1.5189013729436358, 1.5188818251603713, 1.5188795026514685, 1.5188726641530328, 1.5188538260252664, 1.518853309912177, 1.5188493100357332, 1.5188488584367799, 1.5188484713519628, 1.5188442779331106, 1.518840923198029, 1.5188348588692275, 1.5188324718461887, 1.5188279558566558, 1.518827762314247, 1.5188230527823054, 1.5188112466953834, 1.518810085440932, 1.5188071823048037, 1.5188023437445894, 1.5187951181613368, 1.5187901505728505, 1.5187877635498115, 1.5187560871089438, 1.518747506728831, 3.015875617537707, 3.079230268864976, 3.099340161486705, 3.1386684606206203, 3.0693076549732172, 7.635843314035944, 3.024952948504818, 5.362455594560349, 3.747531066763228, 3.8762878843989843, 46.42471726568493, 6.89680051731363, 7.794365978922816, 3.755975780993268, 4.780643370962572, 15.645873185648606, 185.03814757509102, 4.8481206008668, 155.0857093034151, 11.29944856184374, 5.42168092916491, 5.450068918405213, 7.68794660882222, 4.082170079947644, 66.47002224727925, 12.17548277794114, 142.64909071060328, 3.781475386610242, 25.05442145653092, 18.679655845311267, 83.31185378326795, 50.93962956896941, 72.68560668675633, 198.60069866234727, 100.54286782276854, 49.98799566206046, 48.575504903848504, 67.13655543724771, 150.55767347470498, 53.80737617850395, 169.03929776373798, 70.78587171968042, 36.486827824532284, 31.036177909247144, 40.51128246110746, 246.96695121258134, 56.23273732890741, 3.13968635953084, 2.387797710005878, 1.5814622141470667, 1.581439768857658, 1.5814361547856346, 1.581434189237692, 1.5814158652584853, 1.581412631615096, 1.581406291137862, 1.581393419969077, 1.5813878403491108, 1.581385304158217, 1.581378773466666, 1.5813761104662276, 1.5813679312505957, 1.5813651414406127, 1.5813581669156551, 1.5813511923906975, 1.5813306492444592, 1.5813290007203784, 1.5813222164097378, 1.5813218993858762, 1.5813179048852186, 1.5813147346466014, 1.5813016732634992, 1.5813012928348653, 1.5812950157624035, 1.5812946987385417, 1.5812874071897227, 1.5812873437849502, 3.1484377567224398, 15.134697781340625, 6.406743597040574, 3.843710561627079, 4.992742556952465, 9.748218956566278, 19.11482290187962, 11.25325649236817, 25.05442145653092, 198.60069866234727, 155.0857093034151, 3.1933974492188026, 49.98799566206046, 56.23273732890741, 3.7532011285208053, 169.03929776373798, 19.440964466868056, 5.504448349123678, 5.632998362542716, 33.37928157366333, 150.55767347470498, 6.215372486746249, 9.830681525223461, 4.180730666724222, 100.54286782276854, 25.75711098904983, 28.500205963720713, 28.38642088993745, 24.360551791529474, 61.602797975814305, 90.32706620958497, 142.64909071060328, 36.56535764640446, 246.96695121258134, 22.772216746713447, 50.93962956896941, 36.486827824532284, 15.742232676528676, 72.68560668675633, 35.424143727262724, 66.47002224727925, 2.362926074992552, 1.5824998266026549, 1.5824930439048903, 1.5824861984043685, 1.582462019342893, 1.5824290478954264, 1.582427854643042, 1.5824210719452771, 1.582415922119197, 1.582410709490359, 1.5824091394214321, 1.5824071297332056, 1.5824039267925947, 1.5824031103567526, 1.5823956368286602, 1.5823952600121176, 1.582388477314353, 1.582387660878511, 1.5823720229919982, 1.5823652402942336, 1.582364737872177, 1.5823641726473634, 1.5823600904681532, 1.5823487231691218, 1.5823278098510143, 1.5823147468775416, 1.5823108531066028, 1.5823073361522064, 1.5822954036283614, 1.582293519545649, 14.944313276725948, 3.1118120338414452, 3.186807803292693, 10.285508292470977, 25.747044458685643, 6.330194428212883, 3.8736422478943138, 3.8819990712652976, 4.184133671126436, 37.6127902785303, 9.437724608720583, 5.729972179214251, 16.369837871270168, 150.55767347470498, 7.8322592141869025, 4.925432702242157, 66.47002224727925, 5.4754706979628125, 11.392206318449322, 13.905238765803619, 91.57141796795516, 23.44706571181554, 16.845828738549788, 30.71464919868768, 18.107703551916472, 56.23273732890741, 90.32706620958497, 36.486827824532284, 70.78587171968042, 169.03929776373798, 142.64909071060328, 50.93962956896941, 246.96695121258134, 72.68560668675633, 67.13655543724771, 28.38642088993745, 86.44308126247893, 185.03814757509102, 24.588641262600245, 61.602797975814305, 198.60069866234727, 3.8002968724610864, 2.301312418937259, 1.5516395383010588, 1.5516308524933746, 1.5516266461588357, 1.5516183427452004, 1.5516177964679876, 1.551611186513712, 1.5515999332031278, 1.5515876419658388, 1.5515818514273827, 1.5515815782887763, 1.5515693416792087, 1.5515648622060634, 1.5515620761922777, 1.5515608197546882, 1.5515569958141986, 1.5515547014499045, 1.551550932137136, 1.5515450869709586, 1.5515252571081324, 1.551495211861426, 1.5514806262598433, 1.551458447405002, 1.5514575733614615, 1.5514362139224394, 1.5514076982519291, 1.5512320155002797, 3.10519339703269, 2.363236672786983, 3.8638600320662992, 3.9131511140272672, 91.57141796795516, 4.379616104283372, 4.818219102626512, 4.970064884814999, 4.667400926206335, 34.86036593195589, 12.603910102379492, 5.715438428584243, 50.93962956896941, 5.316559756863254, 67.13655543724771, 5.798075918770264, 185.03814757509102, 16.5327777852986, 198.60069866234727, 6.89680051731363, 11.171653438245482, 8.349885256251811, 32.03833721888798, 20.681628134454485, 4.839267688103593, 8.167699210932934, 8.205679372506598, 5.474962714290649, 142.64909071060328, 72.68560668675633, 70.78587171968042, 46.42471726568493, 27.578215566260766, 246.96695121258134, 19.195283846980605, 35.424143727262724, 42.846486735471636, 169.03929776373798, 49.931425588334456, 56.23273732890741, 19.174120944194698, 53.80737617850395, 19.259797144114685, 1.5485576915703627, 1.5485440899483347, 1.5485366977624497, 1.548508361049891, 1.5485079175187382, 1.5485046156757096, 1.5484919011159877, 1.548469724558333, 1.5484641064970608, 1.5484604596853575, 1.5484558272488698, 1.5484463159696977, 1.5484440983139323, 1.5484426691579947, 1.548440007971076, 1.5484248786306318, 1.5484119669459528, 1.5484101928213405, 1.548395654855767, 1.5483908745755615, 1.5483880655449254, 1.5483871292013798, 1.5483754495476818, 1.548366184674706, 1.5483660861122275, 1.5483466693039702, 1.5483458808041424, 1.5482695934458108, 1.5482213963938416, 1.5481979878052063, 3.0536875646206414, 16.819064585410437, 4.525163562393062, 30.729642405272276, 4.8927879771001255, 10.166784852093972, 17.950110153430888, 18.64091349379448, 13.657667886434567, 5.553671374655002, 11.73845500996545, 91.57141796795516, 24.52845047829212, 22.194433177287294, 6.399639166330922, 67.13655543724771, 18.989219494357915, 38.53189397009943, 198.60069866234727, 46.42471726568493, 31.036177909247144, 53.80737617850395, 8.074206627099937, 22.058615659975583, 4.02097529875932, 11.037139161664365, 169.03929776373798, 48.575504903848504, 11.892777130070625, 16.369837871270168, 142.64909071060328, 56.23273732890741, 90.32706620958497, 40.51128246110746, 150.55767347470498, 35.76707126634329, 185.03814757509102, 32.03833721888798, 1.4944894858843996, 1.4944679210054743, 1.494454973055157, 1.4944480253744992, 1.4944471681931195, 1.4944414837271265, 1.4944170315003955, 1.4944145501858748, 1.4943915416330467, 1.494389060318526, 1.4943715557724526, 1.4943682172765522, 1.4943593296590874, 1.4943545023744742, 1.4943516996860051, 1.4943459305606757, 1.49434132885011, 1.4943401558650637, 1.4943185007565196, 1.4943121395683847, 1.494310425205625, 1.494307131824534, 1.4943030714916818, 1.4942884542934145, 1.494276498868906, 1.4942633704593509, 1.494231880322343, 1.4942106763618934, 1.4938736305730336, 2.243208662533867, 3.0742953671061546, 3.090436504839963, 14.557835373086455, 185.03814757509102, 61.602797975814305, 198.60069866234727, 86.44308126247893, 7.0558284548897765, 34.86036593195589, 11.866867623238345, 7.601398232149421, 56.23273732890741, 142.64909071060328, 50.93962956896941, 150.55767347470498, 246.96695121258134, 42.846486735471636, 6.9492044053482624, 169.03929776373798, 91.57141796795516, 10.286861838150193, 14.789958502307682, 11.392206318449322, 30.729642405272276, 14.77209404877583, 6.220776273900312, 19.46779138215442, 49.931425588334456, 18.449223751523206, 67.13655543724771, 90.32706620958497, 53.80737617850395, 33.71163644943962, 72.68560668675633, 47.36868472707816, 42.98602391313575, 23.340368957389956, 37.587336643711915, 1.4215794026391817, 1.4215519375577168, 1.4215384685412205, 1.4215296258390857, 1.4215081925345738, 1.4214811959406393, 1.421479321990518, 1.4214938518548288, 1.4211102221381626, 2.821102132202513, 1.4263986045778294, 1.4308496063053529, 1.4309821592070708, 1.4431688971933772, 1.4584606809668834, 2.917342666725807, 3.617093639590058, 3.7059363182357328, 1.4573575199006172, 2.1675177759310174, 2.1823408761183396, 2.200695333128062, 2.2007576841792527, 2.225826003953845, 2.225542719896735, 4.365563957249666, 2.248336128618677, 4.485493951585466, 2.3051616137168063, 2.3047954409031077, 2.460904183838561, 8.248284225358251, 169.03929776373798, 14.040178539870587, 4.566505133438799, 185.03814757509102, 12.17548277794114, 155.0857093034151, 6.323344259523128, 11.171653438245482, 246.96695121258134, 18.865713844166063, 142.64909071060328, 26.836574714127572, 25.75711098904983, 100.54286782276854, 61.602797975814305, 7.8122364527957275, 6.213051808778822, 36.56535764640446, 50.93962956896941, 37.3822822700304, 83.31185378326795, 49.931425588334456, 49.98799566206046, 19.259797144114685, 18.56632329453585], \"Category\": [\"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Default\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic1\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic2\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic3\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic4\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic5\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic6\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic7\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic8\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic9\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic10\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic11\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic12\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic13\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic14\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic15\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic16\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic17\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic18\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic19\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\", \"Topic20\"], \"logprob\": [30.0, 29.0, 28.0, 27.0, 26.0, 25.0, 24.0, 23.0, 22.0, 21.0, 20.0, 19.0, 18.0, 17.0, 16.0, 15.0, 14.0, 13.0, 12.0, 11.0, 10.0, 9.0, 8.0, 7.0, 6.0, 5.0, 4.0, 3.0, 2.0, 1.0, -5.7795, -6.7137, -6.7138, -5.6706, -6.071, -6.7815, -6.8065, -6.5552, -5.55, -6.8953, -6.9271, -6.9326, -6.7138, -6.7136, -7.3826, -7.3826, -7.3826, -7.3826, -7.3826, -7.3826, -7.3826, -7.3826, -7.3826, -7.3826, -7.3827, -7.3827, -7.3827, -7.3827, -7.3827, -7.3827, -6.3164, -6.3163, -6.7136, -6.7137, -6.7542, -6.7136, -6.5346, -6.7383, -4.6119, -5.377, -6.3163, -6.4339, -5.8053, -5.8147, -6.7886, -6.7423, -5.6877, -6.4615, -4.4288, -4.1988, -5.6512, -5.7483, -4.0909, -3.8417, -6.3477, -5.284, -4.2095, -5.3998, -4.3959, -4.9619, -5.5133, -4.2773, -5.0971, -5.407, -5.3042, -5.0515, -5.9552, -5.0156, -5.0111, -5.7242, -4.8705, -5.3659, -4.8495, -5.1821, -5.2805, -5.2789, -5.3973, -5.4993, -5.6099, -6.4534, -7.0284, -7.0284, -7.0284, -7.0285, -7.0285, -7.0285, -7.0285, -7.0285, -7.0285, -7.0285, -7.0285, -7.0285, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -7.0286, -5.5475, -6.3596, -5.8205, -5.1821, -4.1766, -6.0803, -5.7669, -5.8158, -5.9131, -5.8265, -5.9906, -5.9621, -5.9621, -3.1428, -6.5004, -5.6707, -5.2361, -5.2584, -6.3595, -6.0064, -5.8368, -6.2313, -6.3595, -4.1115, -5.2502, -4.6836, -5.2803, -5.9682, -6.3596, -3.8545, -4.9632, -5.0745, -5.3678, -5.461, -4.9701, -5.0944, -4.6094, -5.6803, -4.4826, -4.544, -5.596, -5.2202, -5.1717, -5.3692, -4.9275, -5.2524, -5.3927, -5.4391, -5.6159, -5.3855, -5.5565, -5.5698, -6.0957, -6.0958, -6.4929, -6.493, -6.493, -6.493, -6.493, -6.4931, -6.5002, -6.2345, -6.5466, -6.0955, -6.0957, -6.0957, -6.0958, -6.3644, -7.1619, -7.1619, -7.1619, -7.1619, -7.1619, -7.1619, -7.1619, -7.1619, -7.162, -7.162, -7.162, -7.162, -7.162, -7.162, -6.0956, -6.493, -6.0957, -6.0957, -6.4931, -6.493, -6.4929, -6.4929, -6.493, -5.4109, -5.9167, -6.5007, -6.5056, -4.4983, -3.546, -3.2115, -3.4802, -5.0649, -4.7864, -4.3683, -4.2606, -5.4234, -5.6972, -5.5466, -5.2862, -4.9135, -5.2847, -4.9053, -5.4107, -4.8926, -5.099, -4.5123, -5.152, -5.2548, -5.1951, -5.7485, -5.8121, -5.397, -5.5916, -5.637, -5.2225, -5.2812, -5.4857, -5.6672, -5.6169, -6.2558, -6.256, -6.2589, -5.5783, -6.4485, -6.9249, -6.9249, -6.9249, -6.9249, -6.9249, -6.9249, -6.9249, -6.9249, -6.9249, -6.9249, -6.9249, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.925, -6.2559, -6.2584, -6.2561, -6.2626, -6.286, -6.3032, -6.1136, -5.5178, -5.5531, -5.8587, -5.9642, -5.9471, -4.0908, -6.2563, -5.2732, -5.4191, -5.3969, -5.3163, -5.9532, -6.2989, -3.8186, -5.4869, -6.2958, -6.4529, -6.4528, -4.2665, -3.5445, -6.1639, -5.0563, -5.4218, -4.312, -5.0079, -5.2081, -5.0419, -5.7141, -5.3045, -5.3687, -5.0317, -5.3666, -5.0997, -5.2178, -4.809, -5.7314, -5.0025, -5.1898, -5.0657, -5.517, -5.2643, -5.4921, -5.4238, -5.5111, -6.1498, -6.1498, -5.7419, -6.8188, -6.8188, -6.8188, -6.8188, -6.8188, -6.8188, -6.8188, -6.8188, -6.8188, -6.8188, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.8189, -6.1501, -6.1498, -6.1499, -6.1497, -5.7525, -6.1499, -6.2835, -6.5257, -3.1664, -5.8989, -6.1498, -5.7126, -3.8662, -4.338, -4.8295, -4.5119, -4.6287, -6.1497, -6.1499, -3.8287, -6.1497, -5.4383, -5.7488, -6.1498, -5.5397, -5.7526, -4.4907, -4.5385, -4.1887, -5.2634, -5.0186, -5.5392, -5.0455, -5.3048, -5.2951, -5.2484, -5.0962, -5.2569, -5.2107, -5.0229, -5.4356, -5.4692, -5.4967, -5.1113, -5.2317, -5.393, -5.3451, -6.1291, -6.7981, -6.7981, -6.7981, -6.7981, -6.7981, -6.7981, -6.7982, -6.7982, -6.7982, -6.7981, -6.7982, -6.7982, -6.7982, -6.7982, -6.7982, -6.7982, -6.7983, -6.7983, -6.7983, -6.7983, -6.7983, -6.7983, -6.7983, -6.7983, -6.7983, -6.7982, -6.7983, -6.7983, -6.7983, -6.1292, -6.1292, -6.1292, -5.732, -5.5401, -3.917, -3.2039, -6.0334, -6.1292, -3.5054, -5.732, -5.341, -5.4631, -6.0482, -5.144, -3.571, -5.0469, -3.9188, -5.4485, -5.6731, -5.2316, -6.1291, -6.1292, -5.1707, -4.9368, -5.8713, -5.7319, -5.7319, -6.2067, -5.1275, -4.3157, -5.1723, -4.7103, -5.2278, -4.9479, -5.4082, -5.4202, -5.4311, -5.5852, -5.4658, -5.41, -5.4626, -5.3837, -5.4119, -5.5486, -5.5471, -6.153, -6.2494, -5.7533, -5.7537, -6.1509, -6.8196, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8197, -6.8198, -6.8198, -6.8198, -6.8198, -6.8198, -6.8198, -6.8198, -5.0687, -6.1507, -6.0796, -5.2516, -4.9971, -6.1508, -6.1508, -4.6515, -5.4718, -6.308, -3.6255, -4.0728, -5.4698, -3.9676, -4.9279, -5.038, -5.6195, -6.1507, -4.955, -6.1508, -5.0623, -6.1507, -4.7743, -4.6328, -5.757, -4.8469, -5.2215, -5.2011, -4.9155, -4.3917, -5.4447, -4.9411, -4.5777, -5.2596, -5.0936, -5.2739, -5.0267, -5.3266, -5.0687, -5.1711, -5.1583, -5.4697, -4.8015, -5.2679, -6.1693, -6.1694, -6.1694, -6.1694, -6.1694, -6.1695, -6.8383, -6.8383, -6.8383, -6.8383, -6.8383, -6.8383, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.8384, -6.1694, -4.4159, -4.9342, -4.6095, -6.107, -5.7719, -5.7723, -5.6457, -6.1695, -3.7114, -4.5776, -6.5577, -4.1177, -6.5715, -5.8581, -6.1693, -4.3603, -6.1693, -4.5135, -4.5264, -6.1694, -6.1696, -5.9101, -5.4986, -5.5762, -4.3627, -4.173, -5.2962, -4.8899, -4.5476, -5.5288, -5.1119, -5.3972, -5.3804, -5.4886, -5.7721, -5.6255, -5.2047, -5.4197, -5.6519, -5.606, -5.7217, -6.0405, -6.0405, -6.7093, -6.7094, -6.7094, -6.7094, -6.7094, -6.7094, -6.7094, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.7095, -6.0404, -6.3379, -5.567, -5.6432, -5.0864, -5.625, -5.834, -6.0405, -5.6432, -5.5173, -6.4398, -3.6393, -5.5753, -3.7369, -5.6431, -5.4307, -5.1388, -4.1066, -5.2932, -5.5101, -4.3995, -5.1391, -5.4557, -4.1003, -4.663, -5.5501, -5.1611, -5.4219, -4.7207, -4.5529, -4.6661, -5.1956, -4.6889, -5.326, -5.3535, -5.3058, -5.5201, -5.3774, -5.5103, -5.5225, -5.5624, -6.0454, -6.111, -6.1319, -6.7142, -6.7143, -6.7143, -6.7143, -6.7143, -6.7143, -6.7143, -6.7143, -6.7143, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7144, -6.7145, -6.7145, -6.0453, -5.6482, -5.4361, -5.3646, -5.5459, -5.3148, -5.8231, -5.3645, -4.4879, -5.8301, -4.3607, -5.0244, -6.4586, -6.4651, -5.2641, -6.0455, -4.297, -6.0455, -6.0284, -5.7441, -6.1213, -5.4979, -4.5005, -5.1971, -5.5954, -5.9117, -5.509, -4.2807, -5.0696, -5.2861, -4.2791, -5.0566, -4.6201, -5.3645, -5.2493, -5.1826, -4.6654, -4.992, -5.1058, -5.1887, -5.2102, -4.9841, -5.3067, -5.2867, -5.4079, -5.3283, -5.9867, -5.8245, -6.1697, -6.2225, -6.6557, -6.6557, -6.6558, -6.6558, -6.6558, -6.6558, -6.6558, -6.6558, -6.6558, -6.6558, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -6.6559, -5.5899, -5.3659, -5.987, -5.9868, -5.5896, -4.693, -4.5076, -5.306, -3.7342, -5.3377, -4.9149, -6.0331, -5.2266, -4.8876, -4.7912, -5.5897, -5.987, -5.633, -5.3607, -5.5895, -5.5896, -4.7986, -5.9868, -4.9667, -5.9868, -4.8909, -5.1549, -4.5165, -4.3702, -4.6351, -4.6019, -4.526, -5.2481, -5.1568, -5.3217, -5.0019, -5.2459, -5.2187, -5.2648, -5.1873, -5.2496, -5.2276, -5.1388, -5.8099, -5.882, -6.479, -6.479, -6.479, -6.479, -6.479, -6.479, -6.479, -6.479, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4791, -6.4792, -6.4792, -6.4792, -5.8102, -5.8101, -5.1944, -4.6258, -5.4126, -4.632, -5.2899, -5.7686, -5.9304, -5.5588, -5.1327, -5.81, -5.81, -5.81, -4.3468, -5.1652, -4.752, -5.81, -5.6125, -4.2593, -4.1822, -6.1261, -5.8134, -5.0582, -5.4142, -4.4373, -6.0118, -5.8405, -4.8066, -5.3322, -4.649, -5.4054, -5.413, -5.2125, -4.9167, -5.1102, -5.2285, -5.4872, -5.6452, -5.6023, -5.5038, -5.6605, -5.651, -5.9095, -5.9095, -5.9095, -6.5784, -6.5784, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5785, -6.5786, -6.5786, -6.5786, -6.5786, -6.5786, -6.5786, -6.5786, -6.5786, -6.5786, -5.9095, -5.9098, -6.1936, -5.5123, -5.5123, -4.2731, -5.2287, -5.9097, -5.9098, -5.9099, -5.008, -4.3056, -4.0896, -5.5122, -5.9097, -5.9095, -5.9204, -5.3848, -5.9095, -5.5125, -5.2287, -5.9098, -5.7526, -5.9096, -5.9095, -4.5986, -5.2332, -4.7733, -4.391, -4.3614, -5.0093, -5.2281, -5.4265, -5.008, -5.1313, -5.5123, -5.362, -5.5123, -4.9074, -5.5125, -5.5123, -5.5123, -5.5184, -5.5122, -5.5127, -5.6255, -6.6014, -6.6015, -6.6015, -6.6015, -6.6015, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6016, -6.6017, -6.6017, -6.6017, -6.6017, -6.6017, -6.6017, -6.6017, -6.6017, -6.6017, -5.9327, -5.9326, -5.9326, -5.9429, -5.9837, -5.2567, -6.0436, -5.6327, -5.9327, -5.9327, -4.3012, -5.6122, -5.5352, -6.0341, -5.9326, -5.2518, -3.8789, -5.9431, -4.0919, -5.5353, -5.9326, -5.9326, -5.7688, -6.1078, -4.7905, -5.6475, -4.5412, -6.1861, -5.3353, -5.4936, -4.8532, -5.1204, -5.0091, -4.6948, -4.9321, -5.1797, -5.2517, -5.1878, -5.0957, -5.347, -5.2116, -5.3948, -5.5354, -5.5736, -5.535, -5.4029, -5.6164, -5.4344, -6.0617, -6.5005, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5006, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -6.5007, -5.8316, -4.7792, -5.4344, -5.8316, -5.6559, -5.3288, -4.9526, -5.3202, -4.8522, -3.7164, -3.865, -6.099, -4.5472, -4.5666, -6.0314, -3.9938, -5.1506, -5.8316, -5.8316, -4.9419, -4.2468, -5.8316, -5.6338, -6.0378, -4.5483, -5.2147, -5.1956, -5.2252, -5.3074, -4.9353, -4.7839, -4.7669, -5.2714, -4.8613, -5.4971, -5.3677, -5.4528, -5.586, -5.4346, -5.5064, -5.4957, -5.8206, -6.4897, -6.4897, -6.4897, -6.4897, -6.4897, -6.4897, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4898, -6.4899, -6.4899, -6.4899, -6.4899, -6.4899, -6.4899, -4.2623, -5.8223, -5.8223, -5.0058, -4.3478, -5.457, -5.8208, -5.8208, -5.8207, -4.338, -5.3108, -5.6545, -5.0338, -3.8079, -5.559, -5.8488, -4.4228, -5.8209, -5.4689, -5.4235, -4.6441, -5.2602, -5.4243, -5.1809, -5.4234, -4.9496, -4.7633, -5.1653, -4.9193, -4.6149, -4.7692, -5.1586, -4.8677, -5.1569, -5.1824, -5.3749, -5.1946, -5.1439, -5.454, -5.3885, -5.4507, -5.0407, -5.7215, -6.3905, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3906, -6.3907, -6.3907, -6.3907, -6.3907, -6.3907, -6.3907, -6.3908, -6.3908, -6.3908, -6.3908, -6.3911, -5.7216, -6.0303, -5.7216, -5.7215, -3.677, -5.7218, -5.7218, -5.7216, -5.7986, -4.7639, -5.3113, -5.7215, -4.6419, -5.7966, -4.5883, -5.7836, -4.2032, -5.3048, -4.2203, -5.7215, -5.5189, -5.6531, -5.0887, -5.2965, -5.9238, -5.7216, -5.7215, -5.8951, -4.6395, -4.9202, -4.9744, -5.1586, -5.3777, -4.8521, -5.4952, -5.3909, -5.4406, -5.5799, -5.6669, -5.6862, -5.6993, -5.7009, -5.7215, -6.2917, -6.2917, -6.2917, -6.2917, -6.2917, -6.2917, -6.2918, -6.2918, -6.2918, -6.2918, -6.2918, -6.2918, -6.2918, -6.2918, -6.2918, -6.2918, -6.2919, -6.2919, -6.2919, -6.2919, -6.2919, -6.2919, -6.2919, -6.2919, -6.2919, -6.2919, -6.2919, -6.292, -6.2921, -6.2921, -5.6227, -4.7957, -5.6372, -4.5155, -5.6227, -5.2256, -4.9536, -4.9544, -5.1251, -5.6229, -5.2476, -4.2496, -4.942, -5.01, -5.6228, -4.6678, -5.2274, -4.9422, -4.277, -4.8747, -5.054, -4.8341, -5.6227, -5.2368, -5.9512, -5.6227, -4.7445, -5.1647, -5.6226, -5.5472, -4.8813, -5.2496, -5.2463, -5.4409, -5.3072, -5.481, -5.3568, -5.4987, -6.2785, -6.2786, -6.2786, -6.2786, -6.2786, -6.2786, -6.2786, -6.2786, -6.2787, -6.2787, -6.2787, -6.2787, -6.2787, -6.2787, -6.2787, -6.2787, -6.2787, -6.2787, -6.2788, -6.2788, -6.2788, -6.2788, -6.2788, -6.2788, -6.2788, -6.2788, -6.2789, -6.2789, -6.2794, -5.8883, -5.6096, -5.8003, -5.0343, -3.7929, -4.463, -4.0052, -4.4819, -5.6096, -4.9288, -5.3982, -5.6094, -4.7923, -4.4345, -4.856, -4.4271, -4.2464, -4.9635, -5.6771, -4.4716, -4.8208, -5.6097, -5.4943, -5.6096, -5.2924, -5.5297, -5.8137, -5.4572, -5.1949, -5.5044, -5.1363, -5.0864, -5.3083, -5.3988, -5.3654, -5.4898, -5.5039, -5.5605, -5.5733, -5.9575, -5.9575, -5.9576, -5.9576, -5.9576, -5.9576, -5.9576, -5.9577, -5.9582, -5.2885, -5.975, -5.9918, -5.9926, -6.117, -6.1137, -5.4927, -5.2885, -5.3238, -6.2627, -5.9577, -5.9576, -5.9577, -5.9577, -5.9575, -5.9577, -5.2885, -5.9576, -5.2886, -5.9577, -5.9582, -5.9576, -5.2888, -3.9208, -5.0915, -5.6527, -4.1748, -5.2885, -4.4029, -5.5823, -5.4176, -4.5122, -5.3009, -4.698, -5.2817, -5.3192, -5.0202, -5.161, -5.5994, -5.6635, -5.342, -5.2885, -5.3586, -5.2231, -5.3244, -5.36, -5.533, -5.547], \"loglift\": [30.0, 29.0, 28.0, 27.0, 26.0, 25.0, 24.0, 23.0, 22.0, 21.0, 20.0, 19.0, 18.0, 17.0, 16.0, 15.0, 14.0, 13.0, 12.0, 11.0, 10.0, 9.0, 8.0, 7.0, 6.0, 5.0, 4.0, 3.0, 2.0, 1.0, 2.0605, 1.9686, 1.9685, 1.9211, 1.9099, 1.8996, 1.8732, 1.851, 1.8213, 1.7949, 1.7789, 1.7574, 1.7289, 1.7068, 1.7059, 1.7059, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.7058, 1.6993, 1.6874, 1.69, 1.681, 1.671, 1.6627, 1.6416, 1.6427, 1.3987, 1.4094, 1.5497, 1.5652, 1.4253, 1.4023, 1.5911, 1.5813, 1.3594, 1.5065, 0.928, 0.754, 1.2325, 1.2444, 0.4286, 0.2125, 1.4731, 0.8125, 0.1334, 0.8825, 0.2072, 0.572, 0.9246, -0.0051, 0.5221, 0.7312, 0.638, 0.2518, 1.1875, 0.1253, 0.0928, 0.8813, -0.3213, 0.3598, -0.4162, -0.1358, 0.0862, -0.2188, 0.0454, 0.1811, 0.196, 2.0744, 1.9244, 1.9244, 1.9244, 1.9244, 1.9244, 1.9244, 1.9244, 1.9244, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9243, 1.9011, 1.8992, 1.7833, 1.6874, 1.5289, 1.7802, 1.7268, 1.6875, 1.7026, 1.6756, 1.7057, 1.6651, 1.6627, 0.9114, 1.7817, 1.5471, 1.4123, 1.4126, 1.727, 1.6202, 1.5604, 1.6768, 1.7134, 0.9924, 1.3445, 1.119, 1.3275, 1.5669, 1.6999, 0.5788, 1.0599, 1.022, 1.1125, 1.152, 0.7396, 0.8424, -0.0063, 1.2412, -0.2104, -0.2011, 1.0461, 0.399, 0.2711, 0.5952, -0.3784, 0.0514, 0.1853, 0.2137, 0.8423, -0.866, -0.5102, -0.6169, 2.3433, 2.3433, 2.2291, 2.2291, 2.2291, 2.2291, 2.229, 2.229, 2.22, 2.2099, 2.1747, 2.1177, 2.1177, 2.1175, 2.096, 2.0576, 1.9563, 1.9563, 1.9563, 1.9563, 1.9563, 1.9563, 1.9563, 1.9563, 1.9562, 1.9562, 1.9562, 1.9562, 1.9562, 1.9562, 1.9457, 1.9491, 1.9282, 1.9176, 1.9312, 1.9309, 1.9242, 1.9183, 1.9055, 1.8074, 1.8158, 1.8744, 1.8687, 1.4137, 1.0031, 0.8427, 0.7919, 1.2962, 1.1497, 0.9349, 0.7994, 1.3697, 1.4581, 1.297, 1.0289, 0.6205, 0.9329, 0.4515, 0.998, 0.3848, 0.6107, -0.1694, 0.6505, 0.7574, 0.3829, 1.322, 1.4207, 0.2558, 0.7491, 0.8327, -0.7892, -0.7618, -0.3449, 0.5336, -0.513, 2.3078, 2.3078, 2.3047, 2.3003, 2.1355, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0733, 2.0605, 2.0492, 2.0481, 2.0449, 2.0385, 2.0062, 1.9732, 1.8702, 1.8387, 1.7928, 1.8071, 1.8017, 1.213, 1.8522, 1.4677, 1.5089, 1.4653, 1.4114, 1.6737, 1.8165, 0.7009, 1.439, 1.8021, 1.8709, 1.8706, 0.8744, 0.5097, 1.7391, 1.1902, 1.3713, 0.6408, 1.0152, 0.9927, 0.82, 1.4357, 0.9778, 1.0498, 0.6486, 0.9662, 0.267, 0.4014, -0.4661, 1.2409, -0.3994, -0.1298, -0.6324, 0.4495, -0.7152, 0.3105, -0.1205, -0.4071, 2.4699, 2.4699, 2.4151, 2.2224, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.2223, 2.214, 2.21, 2.21, 2.1952, 1.9374, 2.043, 2.0786, 2.1431, 1.1765, 1.9218, 2.0022, 1.8466, 1.1801, 1.24, 1.2988, 1.1409, 1.0041, 1.8327, 1.8316, 0.4435, 1.8283, 1.3688, 1.5552, 1.8091, 1.4094, 1.5256, 0.5694, 0.5655, 0.2447, 1.086, 0.8433, 1.2002, 0.5737, 0.8339, 0.7505, 0.5575, 0.1812, 0.4688, 0.146, -0.4198, 0.5309, 0.4731, 0.5154, -1.0571, -0.6826, -0.0892, -0.8256, 2.5222, 2.2671, 2.2671, 2.2671, 2.2671, 2.2671, 2.2671, 2.2671, 2.2671, 2.2671, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.267, 2.2455, 2.2425, 2.2354, 2.1255, 2.0409, 1.5257, 1.2294, 2.0343, 2.0337, 1.0977, 1.805, 1.5298, 1.5906, 1.8887, 1.3783, 0.4832, 1.2938, 0.3534, 1.3446, 1.4868, 1.156, 1.7975, 1.7964, 1.0969, 0.8658, 1.5687, 1.4552, 1.4303, 1.7915, 0.8147, 0.0272, 0.8509, 0.3497, 0.7683, 0.3554, 0.7926, 0.718, 0.6244, 0.8846, 0.5463, -0.0533, 0.1902, -0.2797, -0.8628, 0.2572, -0.5942, 2.5304, 2.4221, 2.4187, 2.4185, 2.2725, 2.2699, 2.2699, 2.2699, 2.2699, 2.2699, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2698, 2.2501, 2.2593, 2.1012, 1.8553, 1.7204, 2.0446, 2.0223, 1.404, 1.7243, 2.0564, 0.894, 1.068, 1.6689, 0.9853, 1.3186, 1.3753, 1.551, 1.8573, 1.0556, 1.837, 1.1035, 1.8153, 0.8449, 0.7339, 1.5294, 0.8048, 1.0933, 1.0486, 0.7649, 0.2114, 1.1426, 0.3627, -0.2348, 0.6772, 0.2631, 0.588, 0.0773, 0.6157, -0.6354, -0.6219, -1.1041, 0.9431, 2.8301, 2.6055, 2.5495, 2.5495, 2.5495, 2.5495, 2.5494, 2.5494, 2.2775, 2.2775, 2.2775, 2.2775, 2.2775, 2.2775, 2.2775, 2.2775, 2.2775, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2774, 2.2276, 2.057, 2.0804, 1.8659, 2.1156, 1.9395, 1.9028, 1.8109, 1.9867, 0.8081, 1.1321, 2.133, 0.8352, 2.1353, 1.7417, 1.8887, 0.7806, 1.8568, 0.7903, 0.7769, 1.8414, 1.8361, 1.6237, 1.2595, 1.3075, 0.1864, -0.1188, 0.9714, 0.3874, -0.2754, 1.0811, 0.2548, 0.7686, 0.2525, 0.4978, 1.3285, 0.6892, -0.8618, -0.3597, 0.5489, -1.1727, 0.8126, 2.595, 2.3455, 2.3438, 2.3438, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3437, 2.3316, 2.3189, 2.0936, 2.0357, 1.7788, 1.9289, 1.9932, 2.0737, 1.907, 1.8386, 2.223, 0.7941, 1.7818, 0.5353, 1.662, 1.4696, 1.2488, 0.4965, 1.3552, 1.5024, 0.6606, 1.2016, 1.4136, 0.2426, 0.6403, 1.3811, 1.0397, 1.1977, 0.3256, -0.0334, -0.1169, 0.6102, -0.6347, 0.6163, 0.6586, 0.347, 1.0935, -0.4245, -0.0676, -0.2187, 0.6551, 2.6416, 2.5783, 2.558, 2.3778, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.3777, 2.366, 2.1753, 2.1214, 1.9384, 1.9779, 1.8799, 2.0075, 1.7408, 1.1638, 1.8867, 1.006, 1.3884, 2.2204, 2.2214, 1.4484, 1.9361, 0.7493, 1.9294, 1.9169, 1.7173, 1.979, 1.4351, 0.6404, 1.1734, 1.472, 1.7255, 1.3532, 0.2684, 0.9535, 1.1155, -0.007, 0.6238, -0.017, 0.9991, 0.7151, 0.5271, -0.6112, 0.112, 0.198, 0.3893, 0.409, -0.4646, 0.5552, 0.4188, 0.5881, -0.9854, 2.6782, 2.5252, 2.5104, 2.4481, 2.4198, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4197, 2.4196, 2.4196, 2.4196, 2.4094, 2.1956, 2.1773, 2.1705, 1.9848, 1.5567, 1.4292, 1.7235, 0.8149, 1.7012, 1.418, 1.9761, 1.3681, 1.0988, 0.7428, 1.4671, 1.8115, 1.4536, 1.191, 1.343, 1.3319, 0.4787, 1.7204, 0.6113, 1.7032, 0.4658, 0.7571, -0.0832, -0.316, -0.0321, -0.0824, -0.2538, 0.7479, 0.4949, 0.8165, 0.0581, 0.56, 0.491, 0.4407, 0.1159, 0.1932, -0.1236, -0.7959, 2.8355, 2.7662, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5818, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5817, 2.5708, 2.5559, 2.3875, 2.018, 2.197, 1.8262, 2.0696, 2.1443, 2.2419, 2.0031, 1.5344, 2.0284, 2.0189, 2.0156, 0.7572, 1.3426, 0.9283, 1.9268, 1.7097, 0.2898, 0.1607, 2.2126, 1.8737, 1.0383, 1.3991, 0.1657, 2.0329, 1.8144, 0.4708, 1.1451, -0.2157, 1.0132, 0.9957, 0.4203, -0.3973, -0.0639, -0.0877, 0.5089, 1.1332, 0.1233, -1.2317, 0.6071, -0.073, 2.8213, 2.8212, 2.8212, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5462, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5461, 2.5401, 2.5359, 2.537, 2.5119, 2.5118, 2.3357, 2.2804, 2.3076, 2.2956, 2.2327, 1.7785, 1.3748, 1.1877, 1.9541, 2.1143, 2.1071, 2.0954, 1.7027, 2.0697, 1.7181, 1.415, 1.9747, 1.8048, 1.9416, 1.9187, 0.4614, 1.0997, 0.5299, -0.0481, -0.0892, 0.6435, 0.9854, 1.2437, -0.0552, 0.2354, 1.114, 0.6045, 1.075, -0.8532, 0.705, 0.5333, 0.4741, 0.2874, 0.2134, -0.4664, 2.8514, 2.544, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5439, 2.5438, 2.5438, 2.5269, 2.5062, 2.4997, 2.4767, 2.4584, 2.2739, 2.4129, 2.2513, 2.3097, 2.2759, 1.4244, 2.0202, 1.9748, 2.2061, 2.0663, 1.5614, 0.464, 2.0418, 0.4276, 1.6034, 1.9404, 1.9353, 1.755, 2.049, 0.5762, 1.4166, 0.0619, 2.0472, 1.0071, 1.1424, 0.2877, 0.5124, 0.2682, -0.4227, 0.0208, 0.472, 0.4286, 0.1689, -0.5465, 0.231, -0.7783, -0.091, 0.4311, 0.5547, 0.3269, -1.3487, -0.0825, 2.9849, 2.6314, 2.6046, 2.6046, 2.6046, 2.6046, 2.6046, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.6045, 2.585, 2.0673, 2.2717, 2.3854, 2.2995, 1.9576, 1.6604, 1.8226, 1.4902, 0.5558, 0.6545, 2.3034, 1.1045, 0.9674, 2.2094, 0.4396, 1.4454, 2.0263, 2.0033, 1.1137, 0.3024, 1.9048, 1.6442, 2.0952, 0.4046, 1.1, 1.0179, 0.9923, 1.063, 0.5074, 0.2762, -0.1638, 0.6929, -0.8071, 0.9408, 0.2651, 0.5137, 1.2211, -0.1572, 0.4896, -0.129, 2.8829, 2.6148, 2.6148, 2.6148, 2.6148, 2.6148, 2.6148, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.6147, 2.5968, 2.6059, 2.5822, 2.227, 1.9673, 2.2612, 2.3885, 2.3864, 2.3115, 1.5981, 2.0079, 2.1632, 1.7342, 0.7412, 1.9462, 2.1203, 0.9439, 2.0423, 1.6616, 1.5077, 0.4022, 1.1485, 1.315, 0.9579, 1.2437, 0.5843, 0.2967, 0.8012, 0.3845, -0.1815, -0.1661, 0.4742, -0.8135, 0.1204, 0.1743, 0.8426, -0.0906, -0.801, 0.9072, 0.0543, -1.1786, 3.1877, 3.0085, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7336, 2.7335, 2.7335, 2.7335, 2.7335, 2.7335, 2.7335, 2.7335, 2.7335, 2.7335, 2.7335, 2.7333, 2.7088, 2.6731, 2.4902, 2.4776, 1.3694, 2.3647, 2.2692, 2.2385, 2.2243, 1.2482, 1.7182, 2.0988, 0.9909, 2.096, 0.7684, 2.0223, 0.1397, 1.4534, 0.0518, 1.9109, 1.6312, 1.7881, 1.0079, 1.2377, 2.0629, 1.7417, 1.7371, 1.9681, -0.0364, 0.3571, 0.3294, 0.567, 0.8688, -0.7979, 1.1136, 0.6051, 0.3652, -1.1466, -0.014, -0.1522, 0.9106, -0.1228, 0.884, 2.8345, 2.8345, 2.8345, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8344, 2.8343, 2.8343, 2.8343, 2.8343, 2.8343, 2.8343, 2.8342, 2.8244, 1.9453, 2.4166, 1.6227, 2.353, 2.0188, 1.7223, 1.6837, 1.8241, 2.2261, 1.853, 0.7968, 1.4216, 1.4537, 2.0844, 0.689, 1.3922, 0.9697, -0.0049, 0.8509, 1.0743, 0.744, 1.8521, 1.2329, 2.2208, 1.5395, -0.3112, 0.5156, 1.4649, 1.2208, -0.2782, 0.2843, -0.1863, 0.421, -0.7581, 0.5055, -1.0139, 0.5978, 2.8832, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.8831, 2.883, 2.883, 2.883, 2.883, 2.883, 2.883, 2.883, 2.883, 2.883, 2.883, 2.883, 2.883, 2.8827, 2.8673, 2.8308, 2.6348, 1.851, 0.5499, 0.9798, 0.267, 0.6221, 2.0, 1.0833, 1.6915, 1.9257, 0.7416, 0.1686, 0.7768, 0.122, -0.1922, 0.8424, 1.9477, -0.0383, 0.2255, 1.6229, 1.3752, 1.521, 0.8458, 1.341, 1.9219, 1.1375, 0.4579, 1.144, 0.2205, -0.0264, 0.2697, 0.6468, -0.0881, 0.2157, 0.2987, 0.8527, 0.3635, 3.2542, 3.2542, 3.2542, 3.2542, 3.2541, 3.2541, 3.2541, 3.2541, 3.2538, 3.2378, 3.2333, 3.2134, 3.2125, 3.0796, 3.0723, 3.0, 2.9893, 2.9297, 2.9241, 2.8322, 2.8255, 2.817, 2.8169, 2.8059, 2.8058, 2.8012, 2.7957, 2.774, 2.7706, 2.7703, 2.7053, 2.1647, 0.5126, 1.8301, 2.392, 0.1681, 1.7755, 0.1166, 2.1369, 1.7325, -0.458, 1.3253, -0.095, 0.992, 0.9955, -0.0674, 0.2817, 1.9083, 2.0733, 0.6223, 0.3443, 0.5837, -0.0822, 0.3284, 0.2916, 1.0724, 1.0951]}, \"token.table\": {\"Topic\": [7, 13, 5, 7, 1, 2, 3, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 6, 6, 1, 2, 3, 5, 6, 10, 13, 16, 18, 3, 6, 7, 12, 3, 15, 7, 8, 12, 12, 1, 2, 3, 5, 7, 8, 9, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 4, 19, 1, 5, 7, 9, 10, 11, 12, 14, 16, 17, 18, 3, 7, 8, 16, 16, 3, 8, 13, 16, 3, 17, 5, 8, 7, 9, 12, 1, 5, 7, 8, 11, 12, 14, 15, 16, 19, 20, 1, 8, 15, 20, 15, 15, 19, 12, 20, 1, 2, 4, 5, 10, 12, 13, 16, 17, 11, 15, 12, 3, 5, 7, 12, 16, 17, 19, 18, 4, 9, 19, 1, 1, 17, 16, 17, 3, 4, 4, 8, 2, 9, 4, 20, 1, 2, 3, 4, 6, 7, 8, 10, 11, 13, 17, 19, 3, 8, 12, 15, 10, 2, 3, 4, 6, 8, 9, 10, 12, 13, 14, 15, 19, 20, 7, 13, 13, 1, 5, 6, 9, 17, 12, 4, 10, 15, 11, 2, 8, 13, 14, 2, 10, 12, 14, 4, 1, 15, 8, 4, 11, 1, 2, 6, 9, 11, 15, 16, 1, 2, 3, 4, 5, 7, 8, 9, 10, 11, 12, 13, 14, 15, 18, 19, 6, 5, 1, 3, 4, 6, 10, 11, 15, 19, 20, 2, 3, 11, 12, 18, 19, 4, 18, 19, 9, 1, 4, 7, 8, 10, 11, 13, 16, 17, 20, 1, 3, 4, 7, 8, 11, 16, 20, 10, 18, 5, 7, 5, 7, 9, 11, 13, 14, 5, 8, 10, 13, 4, 3, 8, 10, 4, 2, 15, 1, 7, 8, 9, 16, 10, 17, 1, 4, 6, 7, 9, 10, 11, 12, 13, 19, 8, 1, 7, 15, 17, 1, 3, 5, 6, 12, 15, 6, 13, 19, 4, 2, 13, 15, 1, 2, 3, 5, 12, 15, 16, 18, 1, 5, 10, 15, 6, 14, 13, 19, 1, 2, 3, 4, 5, 6, 7, 8, 9, 11, 13, 15, 20, 16, 11, 13, 17, 1, 2, 3, 11, 19, 18, 19, 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 19, 20, 1, 14, 17, 15, 15, 18, 8, 7, 8, 1, 6, 9, 10, 12, 13, 16, 1, 3, 7, 8, 2, 19, 14, 17, 3, 16, 20, 11, 20, 1, 2, 3, 4, 5, 6, 7, 8, 10, 12, 13, 14, 15, 16, 8, 7, 9, 14, 3, 7, 12, 13, 15, 19, 18, 7, 17, 1, 9, 10, 11, 13, 17, 17, 7, 1, 12, 13, 3, 7, 1, 20, 18, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 1, 2, 3, 4, 5, 6, 8, 10, 11, 12, 13, 14, 17, 18, 19, 20, 15, 20, 1, 5, 6, 9, 10, 13, 17, 18, 11, 13, 16, 17, 19, 12, 13, 2, 8, 10, 16, 3, 5, 17, 11, 18, 1, 11, 14, 4, 6, 14, 2, 3, 14, 17, 12, 1, 2, 4, 6, 8, 10, 13, 19, 1, 2, 9, 10, 11, 12, 19, 7, 9, 10, 11, 19, 16, 4, 11, 8, 5, 1, 17, 9, 9, 5, 20, 3, 3, 16, 5, 5, 20, 7, 12, 12, 1, 2, 5, 6, 7, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 1, 2, 4, 5, 6, 7, 9, 11, 13, 14, 15, 7, 6, 20, 3, 12, 9, 17, 2, 3, 5, 6, 7, 10, 14, 17, 15, 4, 15, 16, 17, 9, 14, 10, 14, 1, 11, 1, 1, 5, 6, 17, 1, 8, 14, 17, 1, 2, 3, 9, 11, 16, 19, 9, 12, 9, 18, 11, 1, 7, 1, 2, 9, 11, 14, 17, 12, 17, 8, 10, 4, 7, 2, 18, 5, 1, 2, 10, 11, 17, 1, 2, 3, 4, 7, 8, 10, 11, 13, 16, 17, 8, 2, 12, 11, 2, 8, 1, 9, 3, 1, 5, 13, 3, 1, 4, 13, 3, 5, 9, 10, 11, 13, 14, 16, 17, 18, 19, 17, 18, 20, 5, 9, 10, 18, 19, 3, 20, 3, 5, 20, 1, 2, 3, 4, 5, 9, 13, 14, 18, 19, 1, 6, 7, 14, 17, 18, 5, 5, 20, 1, 4, 9, 13, 16, 3, 7, 17, 6, 1, 4, 6, 9, 4, 8, 10, 12, 14, 15, 7, 1, 4, 20, 1, 2, 3, 4, 5, 6, 7, 8, 9, 15, 16, 19, 6, 5, 10, 10, 9, 14, 10, 9, 11, 12, 14, 15, 16, 3, 6, 13, 19, 4, 3, 4, 5, 7, 9, 6, 1, 16, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14, 15, 16, 17, 19, 20, 18, 1, 2, 4, 6, 7, 8, 9, 10, 12, 13, 14, 17, 18, 19, 1, 3, 2, 4, 5, 6, 10, 11, 13, 19, 11, 17, 5, 8, 12, 16, 19, 4, 10, 4, 9, 10, 3, 1, 4, 6, 12, 14, 17, 19, 1, 2, 3, 4, 6, 8, 9, 12, 19, 20, 6, 13, 1, 2, 3, 5, 6, 9, 11, 12, 15, 16, 11, 3, 1, 2, 3, 7, 10, 12, 14, 15, 20, 4, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 17, 18, 16, 1, 2, 4, 5, 12, 14, 15, 16, 19, 19, 17, 1, 10, 17, 20, 2, 3, 11, 14, 16, 17, 20, 1, 5, 7, 8, 14, 15, 18, 3, 13, 3, 18, 2, 4, 6, 2, 6, 10, 17, 2, 3, 11, 19, 3, 5, 8, 10, 14, 11, 4, 17, 9, 12, 13, 17, 19, 5, 12, 10, 9, 14, 3, 8, 9, 11, 12, 13, 20, 13, 11, 19, 1, 16, 4, 15, 10, 1, 5, 14, 3, 9, 9, 16, 3, 9, 3, 5, 8, 12, 19, 1, 2, 4, 6, 7, 12, 14, 16, 17, 18, 19, 20, 1, 14, 16, 13, 8, 9, 1, 2, 4, 5, 6, 7, 8, 9, 11, 12, 13, 14, 15, 17, 18, 20, 9, 20, 1, 2, 3, 4, 6, 8, 9, 11, 12, 13, 14, 19, 11, 1, 18, 1, 6, 9, 10, 12, 13, 16, 4, 14, 20, 5, 1, 3, 4, 5, 7, 8, 9, 10, 12, 13, 14, 16, 18, 20, 13, 13, 3, 12, 16, 19, 6, 17, 5, 12, 14, 5, 12, 7, 1, 3, 4, 7, 8, 10, 12, 13, 14, 16, 18, 4, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 20, 1, 4, 7, 14, 18, 20, 9, 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14, 15, 17, 18, 20, 1, 3, 4, 5, 6, 7, 11, 13, 1, 2, 10, 18, 5, 12, 14, 14, 12, 14, 2, 9, 12, 13, 16, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 15, 16, 17, 18, 19, 20, 9, 19, 5, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 20, 1, 14, 19, 1, 2, 3, 4, 5, 6, 7, 8, 9, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 3, 4, 6, 7, 12, 15, 16, 17, 18, 12, 9, 10, 11, 17, 6, 1, 2, 12, 1, 2, 3, 4, 5, 6, 7, 11, 12, 13, 14, 15, 16, 17, 18, 19, 1, 1, 2, 3, 4, 5, 6, 7, 8, 10, 11, 14, 15, 17, 18, 19, 20, 3, 4, 6, 8, 9, 16, 16, 1, 1, 11, 12, 16, 14, 1, 4, 10, 19, 10, 7, 12, 3, 5, 6, 9, 13, 17, 3, 1, 2, 3, 4, 5, 6, 7, 10, 11, 12, 13, 14, 16, 17, 19, 20, 10, 12, 6, 3, 4, 8, 9, 10, 11, 12, 16, 18, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 15, 16, 20, 19, 3, 4, 9, 14, 16, 1, 19, 18, 14, 11, 11, 19, 3, 4, 7, 11, 13, 13, 16, 1, 2, 3, 14, 17, 5, 18, 16, 5, 10, 4, 6, 8, 12, 14, 17, 18, 10, 7, 11, 20, 8, 1, 2, 3, 4, 6, 7, 8, 9, 10, 11, 12, 14, 17, 18, 19, 1, 2, 3, 4, 5, 6, 7, 8, 10, 11, 12, 13, 15, 19, 7, 1, 2, 3, 4, 5, 7, 8, 10, 11, 14, 15, 2, 18, 15, 20, 1, 7, 9, 16, 3, 7, 3, 8, 2, 10, 11, 13, 17, 3, 9, 12, 13, 16, 1, 4, 5, 8, 13, 17, 8, 12, 13, 16, 4, 2, 3, 5, 7, 10, 11, 14, 15, 16, 17, 18, 19, 17, 2, 9, 10, 13, 14, 7, 12, 5, 19, 15, 17, 1, 4, 9, 10, 11, 18, 1, 2, 4, 6, 8, 11, 12, 13, 14, 15, 17, 7, 8, 1, 2, 3, 7, 9, 12, 1, 9, 12, 14, 18, 8, 14, 17, 1, 4, 8, 13, 14, 6, 7, 8, 9, 16, 5, 19, 3, 5, 7, 9, 11, 13, 15, 1, 2, 3, 5, 6, 7, 9, 10, 13, 14, 15, 17, 18, 19, 20, 3, 4, 3, 4, 6, 8, 9, 11, 12, 13, 15, 19, 2, 1, 2, 3, 5, 7, 8, 9, 11, 12, 15, 16, 17, 18, 19, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 15, 16, 17, 18, 19, 20, 7, 17, 12, 13, 18, 1, 3, 5, 9, 11, 12, 14, 17, 18, 19, 6, 9, 11, 13, 15, 6, 11, 1, 8, 2, 5, 8, 1, 2, 4, 5, 7, 8, 9, 10, 11, 12, 13, 14, 15, 17, 18, 2, 6, 9, 18, 6, 1, 6, 9, 11, 12, 16, 14, 11, 14, 16, 1, 17, 10, 9, 2, 8, 1, 2, 4, 7, 13, 15, 18, 4, 4, 20, 2, 17, 6, 18, 2, 3, 4, 5, 8, 10, 11, 13, 18, 19, 16, 14, 16, 15, 2, 2, 3, 14, 2, 3, 11, 19, 17, 17, 17, 15, 1, 16, 5, 8, 12, 16, 19, 14, 1, 5, 12, 13, 16, 17, 18, 19, 20, 1, 2, 4, 6, 10, 11, 12, 18, 19, 12, 1, 2, 3, 4, 5, 7, 8, 9, 15, 17, 18, 8, 2, 8, 16, 5, 8, 12, 1, 2, 4, 5, 7, 9, 11, 14, 16, 18, 19, 7, 20, 13, 1, 3, 5, 6, 7, 10, 11, 12, 13, 15, 16, 18, 20, 1, 4, 5, 6, 7, 8, 9, 11, 12, 14, 15, 17, 19, 14, 1, 2, 3, 5, 6, 7, 9, 10, 11, 12, 13, 18, 18, 4, 8, 10, 14, 19, 1, 2, 4, 7, 8, 13, 14, 15, 17, 1, 2, 10, 17, 8, 3, 16, 3, 9, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11, 13, 14, 17, 20, 3, 15, 4, 3, 5, 8, 10, 14, 15, 5, 15, 19, 1, 2, 4, 6, 7, 8, 9, 10, 12, 13, 15, 18, 4, 13, 14, 3, 18, 19, 1, 2, 3, 4, 6, 9, 10, 12, 14, 15, 17, 18, 6, 1, 2, 3, 4, 5, 6, 9, 10, 11, 13, 14, 15, 16, 18, 20, 12, 5, 1, 2, 3, 3, 6, 13, 15, 5, 11, 2, 6, 8, 9, 11, 1, 3, 13, 17, 19, 1, 6, 9, 18, 12, 2, 3, 9, 19, 15, 18, 20, 8, 4, 5, 8, 11, 12, 4, 5, 7, 8, 9, 11, 12, 14, 10, 15, 3, 4, 2, 1, 2, 3, 4, 6, 7, 12, 1, 2, 3, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 2, 3, 4, 7, 8, 9, 11, 12, 14, 15, 16, 17, 2, 9, 11, 12, 14, 1, 2, 4, 5, 7, 9, 12, 13, 14, 19, 20, 1, 20, 10, 11, 13, 2, 3, 1, 8, 17, 14, 4, 18, 1, 2, 3, 5, 6, 7, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 3, 7, 10, 13, 14, 2, 3, 6, 14, 4, 6, 14, 16, 8, 20, 10, 5, 13, 1, 5, 12, 15, 17, 5, 12, 14, 6, 2, 4, 9, 15, 16, 17, 18, 15, 12, 20, 19, 5, 3, 13, 5, 1, 2, 4, 5, 6, 7, 12, 15, 16, 17, 18, 13, 8, 11, 2, 9, 18, 6, 14, 3, 12, 1, 2, 5, 6, 7, 8, 9, 10, 13, 15, 16, 17, 18, 19, 9, 1, 3, 5, 6, 7, 8, 9, 10, 12, 13, 14, 15, 16, 17, 19, 2, 6, 7, 15, 1, 7, 2, 1, 2, 3, 4, 6, 7, 10, 11, 13, 17, 19, 19, 19, 1, 4, 5, 20, 19, 4, 11, 1, 7, 13, 2, 9, 16, 1, 2, 3, 4, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 18, 19, 1, 13, 16, 17, 14, 4, 16, 8, 2, 4, 6, 20, 11, 1, 2, 5, 7, 8, 13, 15, 16, 19, 5, 8, 13, 5, 8, 2, 8, 13, 18, 1, 10, 20, 11, 3, 11, 3, 12, 5, 13, 1, 2, 11, 17, 1, 3, 4, 6, 7, 9, 10, 11, 13, 14, 15, 16, 18, 20, 8, 12, 13, 11, 10, 18, 4, 4, 9, 16, 2, 4, 11, 8, 9, 16, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 10, 16, 5, 6, 13, 14, 17, 6, 12, 16, 17, 20, 1, 5, 9, 13, 17, 18, 10, 7, 17, 1, 6, 9, 10, 17, 18, 19, 1, 1, 2, 5, 6, 7, 10, 11, 15, 16, 4, 5, 6, 9, 15, 16, 1, 2, 3, 5, 7, 8, 10, 17, 9, 18, 13, 14, 16, 7, 20, 6, 7, 14, 15, 3, 4, 5, 6, 7, 8, 10, 11, 12, 13, 14, 16, 17, 18, 9, 2, 17, 2, 8, 10, 11, 14, 16, 17, 5, 18, 20, 1, 14, 2, 1, 4, 5, 7, 8, 10, 12, 13, 14, 15, 16, 18, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 6, 6, 10, 11, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11, 14, 16, 17, 18, 17, 11, 10, 15, 2, 7, 18, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 20, 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 18, 19, 20, 2, 1, 2, 6, 13, 17, 5, 13, 14, 19, 16, 5, 8, 16, 9, 10, 12, 16, 10, 10, 8, 2, 14, 4, 1, 2, 4, 11, 13, 16, 4, 6, 10, 11, 13, 14, 15, 16, 5, 5, 7, 14, 15, 16, 18, 1, 3, 4, 6, 7, 9, 14, 16, 18, 4, 6, 17, 18, 19, 5, 5, 1, 2, 3, 4, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 4, 5, 7, 10, 11, 13, 4, 6, 8, 11, 16, 12, 14, 13, 3, 7, 13, 18, 1, 8, 13, 10, 8, 2, 9, 1, 4, 1, 2, 4, 5, 7, 14, 17, 11, 1, 2, 4, 5, 8, 10, 11, 12, 14, 16, 11, 1, 2, 3, 4, 5, 6, 8, 9, 10, 11, 12, 14, 15, 16, 19, 20, 1, 2, 4, 5, 6, 11, 15, 16, 18, 13, 15, 14, 13, 11, 2, 4, 6, 7, 8, 9, 12, 13, 14, 4, 4, 7, 9, 12, 16, 20, 7, 12, 15, 16, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 1, 2, 3, 4, 6, 9, 10, 11, 13, 14, 15, 17, 18, 19, 20, 6, 2, 4, 10, 15, 5, 14, 11, 12, 14, 16, 17, 19, 20, 1, 2, 4, 7, 10, 11, 12, 16, 18, 13, 16, 18, 19, 8, 14, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 19, 20, 11, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11, 15, 16, 20, 12, 15, 1, 2, 6, 7, 15, 17, 18, 2, 3, 9, 10, 11, 13, 19, 3, 3, 2, 5, 8, 3, 4, 6, 11, 1, 15, 2, 20, 1, 4, 9, 10, 11, 1, 6, 9, 1, 6, 14, 3, 8, 11, 7, 9, 1, 2, 3, 4, 5, 6, 8, 9, 11, 19, 1, 2, 1, 3, 4, 5, 7, 10, 11, 12, 13, 14, 15, 16, 20, 1, 4, 6, 8, 17, 18, 2, 9, 19, 10, 9, 1, 7, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 3, 7, 1, 7, 10, 11, 1, 3, 6, 10, 12, 13, 14, 15, 16, 20, 1, 2, 4, 5, 6, 7, 8, 10, 12, 13, 14, 15, 16, 17, 18, 17, 20, 1, 3, 1, 4, 8, 10, 11, 14, 17, 18, 13, 18, 1, 3, 4, 5, 6, 7, 8, 10, 11, 12, 14, 16, 19, 20, 1, 5, 6, 20, 2, 1, 2, 3, 4, 6, 7, 8, 10, 11, 12, 14, 2, 15, 16, 1, 4, 7, 8, 10, 11, 13, 14, 16, 17, 20, 1, 14, 1, 3, 1, 2, 3, 5, 7, 9, 11, 12, 13, 14, 19, 1, 6, 8, 11, 4, 12, 15, 5, 1, 3, 6, 7, 8, 10, 12, 13, 17, 1, 2, 5, 6, 7, 8, 10, 12, 13, 15, 16, 5, 6, 12, 17, 8, 11, 2, 3, 11, 19, 2, 4, 8, 4, 8, 10, 12, 16, 1, 2, 3, 5, 6, 7, 9, 10, 17, 19, 7, 14, 2, 3, 4, 5, 6, 7, 9, 11, 12, 18, 19, 15, 18, 1, 2, 3, 4, 11, 12, 14, 18, 13, 1, 16, 2, 3, 5, 6, 8, 16, 20, 7, 15, 16, 1, 2, 3, 4, 7, 8, 10, 12, 16, 17, 20, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11, 13, 14, 14, 2, 3, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 19, 20, 15, 1, 4, 7, 12, 13, 14, 16, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 20, 4, 7, 13, 18, 13, 1, 3, 4, 6, 7, 8, 9, 10, 11, 13, 14, 16, 2, 6, 6, 1, 8, 3, 1, 5, 8, 12, 20, 2, 6, 17, 12, 1, 9, 2, 4, 4, 13, 13, 9, 10, 12, 14, 1, 2, 3, 10, 11, 13, 17, 19, 8, 11, 12, 1, 2, 4, 5, 12, 13, 14, 15, 18, 19, 1, 4, 5, 6, 8, 11, 17, 13, 19, 1, 2, 3, 4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 19, 20, 5, 13, 8, 1, 10, 4, 5, 10, 16, 3, 4, 5, 6, 10, 12, 15, 9, 14, 1, 2, 4, 7, 13, 14, 19, 10, 8, 9, 3, 4, 7, 14, 20, 1, 4, 7, 8, 15, 16, 20, 7, 18, 1, 2, 3, 5, 6, 9, 10, 11, 13, 14, 16, 17, 18, 19, 1, 3, 5, 6, 10, 11, 13, 14, 17, 18, 19, 13, 1, 2, 3, 4, 7, 9, 11, 12, 16, 18, 5, 1, 6, 13, 20, 4, 3, 2, 1, 7, 10, 17, 5, 8, 20, 1, 14, 2, 5, 12, 15, 19, 2, 4, 10, 1, 4, 5, 7, 10, 13, 16, 19, 2, 3, 12, 15, 16, 3, 3, 7, 16, 17, 14, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 16, 17, 18, 19, 7, 8, 10, 1, 3, 8, 16, 3, 4, 7, 8, 12, 13, 15, 12, 5, 7, 9, 12, 17, 8, 11, 12, 15, 16, 4, 7, 9, 1, 2, 3, 4, 7, 8, 9, 11, 13, 15, 17, 18, 20, 1, 2, 3, 5, 6, 7, 8, 9, 10, 11, 13, 14, 15, 16, 17, 18, 19, 20, 4, 9, 10, 16, 20, 9, 2, 3, 20, 2, 18, 15, 7, 18, 6, 1, 3, 4, 1, 10, 15, 20, 19, 11, 1, 3, 4, 5, 6, 7, 8, 9, 10, 11, 15, 17, 19, 6, 7, 15, 8, 10, 1, 4, 2, 10, 12, 18, 16, 2, 15, 18, 18, 1, 2, 3, 4, 5, 8, 9, 12, 13, 14, 16, 17, 3, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 6, 6, 6, 1, 2, 3, 5, 9, 10, 11, 13, 14, 17, 8, 16, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 14, 15, 16, 17, 18, 19, 20, 2, 3, 11, 19, 2, 3, 19, 16, 1, 2, 3, 4, 5, 6, 7, 9, 10, 11, 12, 13, 15, 17, 20, 1, 2, 3, 6, 8, 11, 15, 18, 19, 1, 2, 4, 8, 3, 8, 9, 16, 1, 2, 3, 5, 6, 7, 11, 12, 13, 14, 15, 16, 19, 20, 2, 7, 9, 11, 16, 9, 1, 7, 10, 14, 19, 7, 8, 1, 6, 8, 15, 2, 9, 15, 20, 2, 8, 16, 17, 19, 6], \"Freq\": [0.6225575835082287, 0.644835157817634, 0.5931685922958718, 0.6225412900531468, 0.23832023993180795, 0.014895014995737997, 0.11916011996590398, 0.07447507497868999, 0.05958005998295199, 0.07447507497868999, 0.029790029991475994, 0.014895014995737997, 0.014895014995737997, 0.07447507497868999, 0.014895014995737997, 0.029790029991475994, 0.04468504498721399, 0.014895014995737997, 0.04468504498721399, 0.07447507497868999, 0.05958005998295199, 0.029790029991475994, 0.6075832274750734, 0.607626260184789, 0.08266240852330675, 0.08266240852330675, 0.24798722556992026, 0.1653248170466135, 0.08266240852330675, 0.08266240852330675, 0.08266240852330675, 0.08266240852330675, 0.08266240852330675, 0.17649084543908847, 0.17649084543908847, 0.17649084543908847, 0.35298169087817693, 0.5183823156611349, 0.25919115783056745, 0.4245764017542635, 0.4245764017542635, 0.6049465077001663, 0.6049599978626585, 0.13023101693672948, 0.06511550846836474, 0.09767326270254711, 0.13023101693672948, 0.06511550846836474, 0.06511550846836474, 0.03255775423418237, 0.03255775423418237, 0.03255775423418237, 0.06511550846836474, 0.03255775423418237, 0.06511550846836474, 0.09767326270254711, 0.03255775423418237, 0.03255775423418237, 0.03255775423418237, 0.7034924705057866, 0.5682677310242099, 0.6691830941545586, 0.04282518358364155, 0.12847555075092465, 0.12847555075092465, 0.04282518358364155, 0.21412591791820773, 0.12847555075092465, 0.0856503671672831, 0.0856503671672831, 0.04282518358364155, 0.04282518358364155, 0.0856503671672831, 0.1826327004858335, 0.1826327004858335, 0.1826327004858335, 0.365265400971667, 0.6319398656956936, 0.8621645932861494, 0.2581549704399247, 0.2581549704399247, 0.5163099408798494, 0.6516283227891443, 0.32581416139457214, 0.6001950077868767, 0.30009750389343837, 0.4048432633709915, 0.20242163168549576, 0.20242163168549576, 0.12782923498539717, 0.12782923498539717, 0.06391461749269858, 0.06391461749269858, 0.06391461749269858, 0.19174385247809572, 0.19174385247809572, 0.06391461749269858, 0.06391461749269858, 0.06391461749269858, 0.06391461749269858, 0.22906547923535783, 0.22906547923535783, 0.22906547923535783, 0.22906547923535783, 0.6323537124743787, 0.6323821860103995, 0.6691437618427574, 0.6049278853210286, 0.7036751860777049, 0.08558979997948794, 0.08558979997948794, 0.08558979997948794, 0.08558979997948794, 0.2567693999384638, 0.08558979997948794, 0.08558979997948794, 0.08558979997948794, 0.08558979997948794, 0.3176178401065205, 0.635235680213041, 0.6049403331757692, 0.211915486297616, 0.105957743148808, 0.105957743148808, 0.105957743148808, 0.317873229446424, 0.105957743148808, 0.105957743148808, 0.6458317576136038, 0.5682458465388635, 0.6003515755566029, 0.6694006638408972, 0.6218971429818628, 0.6218879844259035, 0.6445163863377535, 0.6319855148751843, 0.6445270519565288, 0.18546729041633744, 0.7418691616653498, 0.574700964764095, 0.2873504823820475, 0.5430577044523404, 0.6002998986433117, 0.7358678007491527, 0.6988854702781344, 0.07981411474925233, 0.21283763933133956, 0.07981411474925233, 0.026604704916417445, 0.05320940983283489, 0.10641881966566978, 0.026604704916417445, 0.07981411474925233, 0.1862329344149221, 0.05320940983283489, 0.026604704916417445, 0.026604704916417445, 0.24877623289822337, 0.24877623289822337, 0.49755246579644674, 0.6323961321327041, 0.6241328930245986, 0.1570405014308936, 0.10469366762059575, 0.10469366762059575, 0.05234683381029787, 0.10469366762059575, 0.1570405014308936, 0.05234683381029787, 0.05234683381029787, 0.05234683381029787, 0.05234683381029787, 0.05234683381029787, 0.05234683381029787, 0.05234683381029787, 0.8294760203570087, 0.6448139917491805, 0.6448172227021051, 0.12243349983572395, 0.12243349983572395, 0.2448669996714479, 0.12243349983572395, 0.2448669996714479, 0.6049210169033717, 0.5682337319510884, 0.6241474641710784, 0.6323904014698185, 0.6139206167320671, 0.543066013050659, 0.2662423983297224, 0.2662423983297224, 0.2662423983297224, 0.20626549591633697, 0.20626549591633697, 0.20626549591633697, 0.20626549591633697, 0.5682376162547184, 0.4242355956366883, 0.6323374104375699, 0.6391545622059001, 0.5696731198670886, 0.2848365599335443, 0.07191534189684283, 0.2876613675873713, 0.07191534189684283, 0.21574602569052848, 0.07191534189684283, 0.07191534189684283, 0.14383068379368566, 0.07405344431838529, 0.04936896287892353, 0.07405344431838529, 0.17279137007623233, 0.14810688863677057, 0.09873792575784705, 0.024684481439461763, 0.024684481439461763, 0.07405344431838529, 0.024684481439461763, 0.024684481439461763, 0.04936896287892353, 0.04936896287892353, 0.024684481439461763, 0.04936896287892353, 0.024684481439461763, 0.6076315501138085, 0.5931689748743824, 0.14337719217191047, 0.07168859608595524, 0.35844298042977624, 0.07168859608595524, 0.07168859608595524, 0.07168859608595524, 0.07168859608595524, 0.07168859608595524, 0.07168859608595524, 0.5430444492760295, 0.15625880991245414, 0.3125176198249083, 0.3125176198249083, 0.3125176198249083, 0.6691572559207248, 0.5682388469362534, 0.6457678580100124, 0.6691776192722646, 0.6003214296093099, 0.18145771018997683, 0.06048590339665894, 0.06048590339665894, 0.18145771018997683, 0.12097180679331788, 0.06048590339665894, 0.06048590339665894, 0.06048590339665894, 0.12097180679331788, 0.06048590339665894, 0.14306681335264923, 0.14306681335264923, 0.28613362670529846, 0.07153340667632462, 0.14306681335264923, 0.07153340667632462, 0.14306681335264923, 0.07153340667632462, 0.6241541219041039, 0.645912220450334, 0.5931910261424904, 0.6225827406030914, 0.13404896835092087, 0.26809793670184173, 0.4021469050527626, 0.13404896835092087, 0.13404896835092087, 0.6583914282399147, 0.35004964962855944, 0.17502482481427972, 0.17502482481427972, 0.17502482481427972, 0.5682510002026556, 0.2116825683796453, 0.4233651367592906, 0.2116825683796453, 0.5682282324833023, 0.5430445549766303, 0.6323460303834176, 0.4174761045489967, 0.17618440360464502, 0.17618440360464502, 0.35236880720929004, 0.17618440360464502, 0.6240894976491272, 0.6445640439652602, 0.13519247848771201, 0.06759623924385601, 0.06759623924385601, 0.06759623924385601, 0.20278871773156804, 0.13519247848771201, 0.06759623924385601, 0.06759623924385601, 0.06759623924385601, 0.06759623924385601, 0.6391620229230357, 0.18167124779346916, 0.18167124779346916, 0.3633424955869383, 0.18167124779346916, 0.119618848110021, 0.119618848110021, 0.119618848110021, 0.239237696220042, 0.119618848110021, 0.119618848110021, 0.6076287515592398, 0.6448329672156858, 0.6692148340748959, 0.5682440773922492, 0.5430567319612695, 0.644844084674477, 0.6323931907175415, 0.06607333786558243, 0.06607333786558243, 0.19822001359674732, 0.06607333786558243, 0.13214667573116487, 0.3303666893279122, 0.06607333786558243, 0.13214667573116487, 0.2009805973289128, 0.4019611946578256, 0.2009805973289128, 0.2009805973289128, 0.6076455091428983, 0.6584045443398541, 0.6448316528616611, 0.6692082878486987, 0.1552969198176196, 0.0388242299544049, 0.0388242299544049, 0.0776484599088098, 0.0776484599088098, 0.0776484599088098, 0.1552969198176196, 0.11647268986321468, 0.0388242299544049, 0.0388242299544049, 0.0388242299544049, 0.11647268986321468, 0.0388242299544049, 0.6319143100512113, 0.6277864255691122, 0.20926214185637076, 0.20926214185637076, 0.06761344190681601, 0.47329409334771205, 0.20284032572044802, 0.06761344190681601, 0.13522688381363202, 0.6458381914361432, 0.6691433780371226, 0.019631081114283643, 0.07852432445713457, 0.039262162228567286, 0.15704864891426915, 0.039262162228567286, 0.05889324334285093, 0.05889324334285093, 0.039262162228567286, 0.019631081114283643, 0.039262162228567286, 0.05889324334285093, 0.039262162228567286, 0.05889324334285093, 0.05889324334285093, 0.05889324334285093, 0.09815540557141822, 0.05889324334285093, 0.019631081114283643, 0.6270652000388068, 0.658391204515664, 0.6445122790340287, 0.6323559436116808, 0.6323905536099689, 0.6457827578806216, 0.6391637637820958, 0.6225751339438879, 0.6391272432569208, 0.10995977760656615, 0.2199195552131323, 0.32987933281969845, 0.10995977760656615, 0.10995977760656615, 0.10995977760656615, 0.10995977760656615, 0.21108348516999242, 0.21108348516999242, 0.42216697033998485, 0.21108348516999242, 0.5430430117519424, 0.6691248144902313, 0.6584101656855181, 0.6444913575590302, 0.5183236881864667, 0.25916184409323334, 0.7034565347770831, 0.44477335362412007, 0.44477335362412007, 0.05481430201655623, 0.10962860403311246, 0.10962860403311246, 0.10962860403311246, 0.10962860403311246, 0.027407151008278115, 0.08222145302483434, 0.027407151008278115, 0.05481430201655623, 0.027407151008278115, 0.08222145302483434, 0.05481430201655623, 0.05481430201655623, 0.08222145302483434, 0.8594143648554501, 0.6225358125735073, 0.6003474872641479, 0.6583558020687564, 0.16089140307075958, 0.16089140307075958, 0.16089140307075958, 0.16089140307075958, 0.32178280614151916, 0.16089140307075958, 0.6459024544740349, 0.3220411330758322, 0.6440822661516644, 0.13681981342184216, 0.13681981342184216, 0.13681981342184216, 0.4104594402655265, 0.13681981342184216, 0.13681981342184216, 0.6445455928191005, 0.6225526290911771, 0.3019462910977182, 0.6038925821954364, 0.6448256288076037, 0.3155455310636407, 0.6310910621272814, 0.6218855033919632, 0.703467575928803, 0.6458175748792941, 0.06641972985641338, 0.059777756870772046, 0.19925918956924016, 0.03320986492820669, 0.03320986492820669, 0.026567891942565352, 0.026567891942565352, 0.06641972985641338, 0.04649381089948937, 0.06641972985641338, 0.10627156777026141, 0.053135783885130704, 0.013283945971282676, 0.019925918956924017, 0.053135783885130704, 0.07970367582769607, 0.006641972985641338, 0.013283945971282676, 0.03320986492820669, 0.006641972985641338, 0.11865339156700865, 0.05932669578350432, 0.08899004367525648, 0.02966334789175216, 0.11865339156700865, 0.02966334789175216, 0.05932669578350432, 0.08899004367525648, 0.05932669578350432, 0.05932669578350432, 0.05932669578350432, 0.02966334789175216, 0.02966334789175216, 0.05932669578350432, 0.05932669578350432, 0.02966334789175216, 0.4543889621237145, 0.4543889621237145, 0.10174195708254909, 0.3052258712476473, 0.10174195708254909, 0.10174195708254909, 0.10174195708254909, 0.10174195708254909, 0.10174195708254909, 0.10174195708254909, 0.18657026853019198, 0.37314053706038397, 0.18657026853019198, 0.18657026853019198, 0.18657026853019198, 0.6049711326595677, 0.6448004111485948, 0.24323995770239182, 0.3648599365535877, 0.12161997885119591, 0.12161997885119591, 0.6406619325040127, 0.5931800350846813, 0.6446488919824771, 0.613901309520269, 0.6458095588267466, 0.42039622967779927, 0.21019811483889964, 0.21019811483889964, 0.3887707587677518, 0.3887707587677518, 0.1943853793838759, 0.1844446423655602, 0.1844446423655602, 0.1844446423655602, 0.1844446423655602, 0.6049501529604877, 0.30592749711223305, 0.07648187427805826, 0.07648187427805826, 0.15296374855611652, 0.15296374855611652, 0.15296374855611652, 0.07648187427805826, 0.6692401716019304, 0.10840564497109945, 0.37941975739884803, 0.2168112899421989, 0.10840564497109945, 0.05420282248554972, 0.05420282248554972, 0.10840564497109945, 0.2042204794241762, 0.2042204794241762, 0.4084409588483524, 0.2042204794241762, 0.669226068020787, 0.6319884747749203, 0.2970782308179533, 0.5941564616359066, 0.6391502990171648, 0.5931857740355724, 0.5110970524063599, 0.5110970524063599, 0.6003400910292119, 0.6003263288998806, 0.43387798424668933, 0.43387798424668933, 0.8622649820925865, 0.5151984746220253, 0.5151984746220253, 0.5931743658058657, 0.7349903444455022, 0.24499678148183404, 0.6225458605080147, 0.6049700662804235, 0.6049295218623238, 0.02822933442510825, 0.112917337700433, 0.08468800327532475, 0.112917337700433, 0.0564586688502165, 0.02822933442510825, 0.08468800327532475, 0.112917337700433, 0.0564586688502165, 0.0564586688502165, 0.02822933442510825, 0.0564586688502165, 0.02822933442510825, 0.0564586688502165, 0.02822933442510825, 0.02822933442510825, 0.12704678180636703, 0.06352339090318351, 0.06352339090318351, 0.25409356361273405, 0.06352339090318351, 0.06352339090318351, 0.06352339090318351, 0.12704678180636703, 0.06352339090318351, 0.06352339090318351, 0.12704678180636703, 0.6225768785740672, 0.6076288198163615, 0.6856544115656599, 0.6406582505613393, 0.6049604690372958, 0.42314847747377426, 0.42314847747377426, 0.14409183146830246, 0.14409183146830246, 0.07204591573415123, 0.07204591573415123, 0.07204591573415123, 0.21613774720245368, 0.07204591573415123, 0.07204591573415123, 0.6323794729271691, 0.2075455637654334, 0.2075455637654334, 0.2075455637654334, 0.4150911275308668, 0.600312817037573, 0.658393357867888, 0.6241127332067219, 0.6583800744261319, 0.6218991952705288, 0.40948392446544335, 0.6218926096161002, 0.17496470524444221, 0.34992941048888443, 0.17496470524444221, 0.34992941048888443, 0.3928839129642044, 0.13096130432140146, 0.3928839129642044, 0.13096130432140146, 0.1146122135835592, 0.4584488543342368, 0.1146122135835592, 0.1146122135835592, 0.1146122135835592, 0.1146122135835592, 0.1146122135835592, 0.6003248934306665, 0.8008289270030509, 0.79069300886613, 0.6458012141219097, 0.6139277536008939, 0.6215129907047802, 0.3107564953523901, 0.12186681377663053, 0.12186681377663053, 0.12186681377663053, 0.24373362755326106, 0.12186681377663053, 0.24373362755326106, 0.6049223558606398, 0.6444883850952356, 0.8595117511972762, 0.6241380744965809, 0.46681809471768787, 0.23340904735884394, 0.5430630109323643, 0.6457829428488532, 0.5931885565971897, 0.6218889339626497, 0.5430095280500915, 0.2450809799197521, 0.4901619598395042, 0.2450809799197521, 0.04558956904527928, 0.09117913809055855, 0.04558956904527928, 0.09117913809055855, 0.09117913809055855, 0.31912698331695494, 0.13676870713583783, 0.04558956904527928, 0.04558956904527928, 0.04558956904527928, 0.04558956904527928, 0.6391536740368902, 0.5430494595297622, 0.6049348282843964, 0.6139040337771797, 0.5430527363279886, 0.6391523950778526, 0.6163678085349239, 0.20545593617830799, 0.8621935338077416, 0.17636462644093007, 0.17636462644093007, 0.35272925288186013, 0.6406618588647445, 0.23502802875429674, 0.4700560575085935, 0.23502802875429674, 0.05936187619618929, 0.17808562858856786, 0.11872375239237858, 0.11872375239237858, 0.11872375239237858, 0.05936187619618929, 0.05936187619618929, 0.11872375239237858, 0.05936187619618929, 0.05936187619618929, 0.05936187619618929, 0.6445148205638609, 0.46135723134748846, 0.46135723134748846, 0.18006106817260514, 0.18006106817260514, 0.18006106817260514, 0.3601221363452103, 0.18006106817260514, 0.6531917127446331, 0.6988207320167537, 0.4365081180827973, 0.21825405904139866, 0.21825405904139866, 0.08408464978894811, 0.08408464978894811, 0.08408464978894811, 0.08408464978894811, 0.08408464978894811, 0.08408464978894811, 0.25225394936684437, 0.08408464978894811, 0.16816929957789623, 0.08408464978894811, 0.271809565509511, 0.271809565509511, 0.09060318850317034, 0.09060318850317034, 0.09060318850317034, 0.18120637700634068, 0.5931544372379086, 0.4338090631257805, 0.4338090631257805, 0.333193086292366, 0.49978962943854893, 0.6003513258882918, 0.6448084334906753, 0.6319870700732838, 0.25880854681612886, 0.25880854681612886, 0.5176170936322577, 0.6076003570763638, 0.14420318367668136, 0.43260955103004406, 0.14420318367668136, 0.14420318367668136, 0.4433857838438329, 0.14779526128127765, 0.14779526128127765, 0.14779526128127765, 0.14779526128127765, 0.6323351794310466, 0.6225503961402572, 0.4184478030757693, 0.20922390153788464, 0.20922390153788464, 0.09556254394085456, 0.04778127197042728, 0.09556254394085456, 0.09556254394085456, 0.04778127197042728, 0.2389063598521364, 0.09556254394085456, 0.04778127197042728, 0.14334381591128184, 0.04778127197042728, 0.04778127197042728, 0.04778127197042728, 0.6076191274585835, 0.5931389961269674, 0.6241523632438404, 0.6241557549546515, 0.322649321435023, 0.322649321435023, 0.6241231583704959, 0.1282977990389669, 0.1282977990389669, 0.1282977990389669, 0.2565955980779338, 0.2565955980779338, 0.1282977990389669, 0.6406358898051634, 0.26022535636745314, 0.5204507127349063, 0.26022535636745314, 0.568244115851841, 0.13354857011754803, 0.13354857011754803, 0.13354857011754803, 0.13354857011754803, 0.40064571035264407, 0.6075880047947086, 0.8286468206466425, 0.6319502239695106, 0.40385053200809873, 0.014127112878684414, 0.08476267727210648, 0.028254225757368828, 0.24016091893763503, 0.056508451514737656, 0.028254225757368828, 0.08476267727210648, 0.11301690302947531, 0.04238133863605324, 0.056508451514737656, 0.028254225757368828, 0.028254225757368828, 0.04238133863605324, 0.014127112878684414, 0.056508451514737656, 0.04238133863605324, 0.014127112878684414, 0.014127112878684414, 0.6457843198379085, 0.12351904549168435, 0.020586507581947393, 0.14410555307363176, 0.04117301516389479, 0.12351904549168435, 0.020586507581947393, 0.020586507581947393, 0.08234603032778957, 0.10293253790973697, 0.16469206065557915, 0.06175952274584218, 0.020586507581947393, 0.04117301516389479, 0.020586507581947393, 0.21220440096372836, 0.4244088019274567, 0.13424069810051198, 0.33560174525127995, 0.06712034905025599, 0.06712034905025599, 0.20136104715076797, 0.06712034905025599, 0.06712034905025599, 0.6691911551221676, 0.6139045963984671, 0.6444795813175352, 0.1579730308982494, 0.1579730308982494, 0.1579730308982494, 0.3159460617964988, 0.1579730308982494, 0.5682368855400792, 0.4160314983124507, 0.48868421434724135, 0.16289473811574712, 0.16289473811574712, 0.8621867319436815, 0.289384005449335, 0.57876801089867, 0.14499476931217806, 0.14499476931217806, 0.2899895386243561, 0.2899895386243561, 0.14499476931217806, 0.10392846844898222, 0.13857129126530962, 0.13857129126530962, 0.20785693689796444, 0.13857129126530962, 0.10392846844898222, 0.13857129126530962, 0.034642822816327405, 0.034642822816327405, 0.034642822816327405, 0.6076492976453065, 0.6448399498112856, 0.07122419623463674, 0.07122419623463674, 0.1424483924692735, 0.07122419623463674, 0.07122419623463674, 0.07122419623463674, 0.21367258870391023, 0.07122419623463674, 0.07122419623463674, 0.07122419623463674, 0.6138815001618281, 0.8621982462732263, 0.07122416550190233, 0.2848966620076093, 0.07122416550190233, 0.07122416550190233, 0.14244833100380466, 0.07122416550190233, 0.07122416550190233, 0.07122416550190233, 0.14244833100380466, 0.7510703413083966, 0.025952526516760253, 0.051905053033520505, 0.28547779168436277, 0.07785757955028076, 0.025952526516760253, 0.025952526516760253, 0.025952526516760253, 0.051905053033520505, 0.07785757955028076, 0.025952526516760253, 0.10381010606704101, 0.051905053033520505, 0.051905053033520505, 0.025952526516760253, 0.051905053033520505, 0.07785757955028076, 0.6319560678912494, 0.08777930912123065, 0.08777930912123065, 0.08777930912123065, 0.08777930912123065, 0.08777930912123065, 0.08777930912123065, 0.08777930912123065, 0.1755586182424613, 0.08777930912123065, 0.669186510919833, 0.644575891383525, 0.6218875249736803, 0.6241434130728484, 0.6444886119999022, 0.7034852797253786, 0.2685367941803208, 0.0895122647267736, 0.0895122647267736, 0.0895122647267736, 0.0895122647267736, 0.1790245294535472, 0.0895122647267736, 0.2555702600941203, 0.08519008669804011, 0.17038017339608022, 0.17038017339608022, 0.08519008669804011, 0.08519008669804011, 0.17038017339608022, 0.3270306237248951, 0.6540612474497902, 0.6406392033203507, 0.6458046670770665, 0.2240686449590329, 0.2240686449590329, 0.4481372899180658, 0.6032738769641375, 0.1206547753928275, 0.1206547753928275, 0.1206547753928275, 0.5636005959466833, 0.14090014898667083, 0.14090014898667083, 0.14090014898667083, 0.1577704816411119, 0.1577704816411119, 0.1577704816411119, 0.3155409632822238, 0.1577704816411119, 0.6138562739506781, 0.5682292708362009, 0.6444960318705331, 0.1378573470385349, 0.41357204111560464, 0.1378573470385349, 0.1378573470385349, 0.1378573470385349, 0.5931801742094501, 0.6049365888468662, 0.6241196414088488, 0.6161189426928446, 0.3080594713464223, 0.22073734780967866, 0.11036867390483933, 0.22073734780967866, 0.11036867390483933, 0.331106021714518, 0.434815430022391, 0.7034429439139919, 0.6448188381907094, 0.6138879846989524, 0.6691916804049922, 0.6218971429818628, 0.6319532086198988, 0.5682689233732376, 0.6323366245130213, 0.6241325161936286, 0.20485551784904457, 0.40971103569808914, 0.20485551784904457, 0.5073816454191303, 0.25369082270956517, 0.6003168112430125, 0.6319533591072196, 0.6318846025609364, 0.3159423012804682, 0.15357743405552732, 0.46073230216658195, 0.15357743405552732, 0.15357743405552732, 0.15357743405552732, 0.21848824276289838, 0.24970084887188387, 0.031212606108985484, 0.09363781832695645, 0.031212606108985484, 0.12485042443594194, 0.031212606108985484, 0.031212606108985484, 0.09363781832695645, 0.06242521221797097, 0.031212606108985484, 0.031212606108985484, 0.4254437446721549, 0.21272187233607745, 0.631947189185838, 0.6448397033641605, 0.4177651843133127, 0.4177651843133127, 0.06444092458318154, 0.03222046229159077, 0.0966613868747723, 0.22554323604113538, 0.03222046229159077, 0.0966613868747723, 0.06444092458318154, 0.03222046229159077, 0.06444092458318154, 0.06444092458318154, 0.03222046229159077, 0.06444092458318154, 0.03222046229159077, 0.03222046229159077, 0.0966613868747723, 0.03222046229159077, 0.600338312215084, 0.7010663055829122, 0.09305350055364559, 0.25589712652252533, 0.13958025083046838, 0.09305350055364559, 0.13958025083046838, 0.04652675027682279, 0.04652675027682279, 0.04652675027682279, 0.04652675027682279, 0.04652675027682279, 0.023263375138411396, 0.04652675027682279, 0.6138821219608194, 0.6286941527944665, 0.31434707639723325, 0.11010371234447994, 0.22020742468895987, 0.33031113703343984, 0.11010371234447994, 0.11010371234447994, 0.11010371234447994, 0.11010371234447994, 0.22294088695550124, 0.22294088695550124, 0.22294088695550124, 0.5931553762503158, 0.04066918498343451, 0.24401510990060704, 0.08133836996686902, 0.08133836996686902, 0.08133836996686902, 0.04066918498343451, 0.04066918498343451, 0.04066918498343451, 0.04066918498343451, 0.08133836996686902, 0.04066918498343451, 0.08133836996686902, 0.04066918498343451, 0.04066918498343451, 0.6448266693179903, 0.6448235204152746, 0.1417268016638057, 0.4251804049914171, 0.1417268016638057, 0.1417268016638057, 0.8033358813841329, 0.6445188144369521, 0.4876041246285177, 0.24380206231425885, 0.24380206231425885, 0.5913460721256665, 0.29567303606283324, 0.6225440811626459, 0.10790127184977923, 0.05395063592488961, 0.05395063592488961, 0.05395063592488961, 0.05395063592488961, 0.05395063592488961, 0.3237038155493377, 0.16185190777466885, 0.05395063592488961, 0.05395063592488961, 0.10790127184977923, 0.5682630001402829, 0.1549237483543105, 0.1355582798100217, 0.038730937088577624, 0.11619281126573289, 0.09682734272144407, 0.058096405632866444, 0.1355582798100217, 0.038730937088577624, 0.019365468544288812, 0.07746187417715525, 0.058096405632866444, 0.019365468544288812, 0.019365468544288812, 0.019365468544288812, 0.019365468544288812, 0.019365468544288812, 0.019365468544288812, 0.1270775754291701, 0.028239461206482245, 0.18355649784213457, 0.0705986530162056, 0.028239461206482245, 0.08471838361944672, 0.014119730603241122, 0.11295784482592898, 0.09883811422268785, 0.028239461206482245, 0.05647892241296449, 0.014119730603241122, 0.0705986530162056, 0.014119730603241122, 0.014119730603241122, 0.014119730603241122, 0.014119730603241122, 0.028239461206482245, 0.014119730603241122, 0.38401295430918564, 0.12800431810306187, 0.12800431810306187, 0.12800431810306187, 0.12800431810306187, 0.12800431810306187, 0.6003190268017603, 0.20768650729129307, 0.051921626822823266, 0.051921626822823266, 0.051921626822823266, 0.051921626822823266, 0.051921626822823266, 0.051921626822823266, 0.051921626822823266, 0.051921626822823266, 0.051921626822823266, 0.10384325364564653, 0.051921626822823266, 0.051921626822823266, 0.10384325364564653, 0.051921626822823266, 0.051921626822823266, 0.0937284268711647, 0.0937284268711647, 0.0937284268711647, 0.0937284268711647, 0.0937284268711647, 0.2811852806134941, 0.0937284268711647, 0.0937284268711647, 0.5583496823281388, 0.11166993646562776, 0.11166993646562776, 0.11166993646562776, 0.20917686645984893, 0.20917686645984893, 0.20917686645984893, 0.6584166541242287, 0.6049554597445187, 0.3186064449133523, 0.20509573107388873, 0.20509573107388873, 0.20509573107388873, 0.41019146214777746, 0.20509573107388873, 0.11669568221239043, 0.09335654576991234, 0.023339136442478085, 0.04667827288495617, 0.09335654576991234, 0.07001740932743426, 0.023339136442478085, 0.023339136442478085, 0.09335654576991234, 0.023339136442478085, 0.09335654576991234, 0.04667827288495617, 0.023339136442478085, 0.023339136442478085, 0.04667827288495617, 0.023339136442478085, 0.07001740932743426, 0.023339136442478085, 0.6003038302696638, 0.6691697808513482, 0.4087279091442146, 0.1483050895102269, 0.03868828422005919, 0.03224023685004932, 0.14185704214021702, 0.025792189480039456, 0.019344142110029595, 0.13540899477020715, 0.11606485266017756, 0.05158437896007891, 0.03224023685004932, 0.04513633159006905, 0.025792189480039456, 0.012896094740019728, 0.05803242633008878, 0.07092852107010851, 0.012896094740019728, 0.006448047370009864, 0.006448047370009864, 0.019344142110029595, 0.3662647065734704, 0.1831323532867352, 0.1831323532867352, 0.06507362575165467, 0.15381038814027465, 0.035494704955448, 0.035494704955448, 0.07690519407013732, 0.18930509309572266, 0.029578920796206665, 0.017747352477724, 0.11831568318482666, 0.04141048911468933, 0.035494704955448, 0.011831568318482666, 0.017747352477724, 0.05915784159241333, 0.029578920796206665, 0.011831568318482666, 0.023663136636965332, 0.023663136636965332, 0.029578920796206665, 0.055225114390287364, 0.1656753431708621, 0.11045022878057473, 0.1656753431708621, 0.1656753431708621, 0.11045022878057473, 0.11045022878057473, 0.055225114390287364, 0.055225114390287364, 0.6049632465027044, 0.20558120711406677, 0.41116241422813354, 0.20558120711406677, 0.20558120711406677, 0.6075946249189571, 0.19193959398742563, 0.19193959398742563, 0.38387918797485127, 0.12924149792151626, 0.06462074896075813, 0.021540249653586047, 0.06462074896075813, 0.08616099861434419, 0.021540249653586047, 0.043080499307172095, 0.043080499307172095, 0.043080499307172095, 0.043080499307172095, 0.17232199722868838, 0.043080499307172095, 0.043080499307172095, 0.06462074896075813, 0.06462074896075813, 0.021540249653586047, 0.8275945102711443, 0.10939315946751933, 0.16408973920127898, 0.054696579733759666, 0.10939315946751933, 0.054696579733759666, 0.027348289866879833, 0.08204486960063949, 0.054696579733759666, 0.10939315946751933, 0.054696579733759666, 0.027348289866879833, 0.08204486960063949, 0.027348289866879833, 0.027348289866879833, 0.027348289866879833, 0.027348289866879833, 0.1563429909568951, 0.1563429909568951, 0.07817149547844755, 0.39085747739223775, 0.07817149547844755, 0.07817149547844755, 0.6319486187910442, 0.6218744155562822, 0.4056431774656907, 0.20282158873284534, 0.20282158873284534, 0.6319719448423505, 0.6583935256621594, 0.2853410481016674, 0.5706820962033348, 0.6241546243802853, 0.66920499457962, 0.6241437585132821, 0.6225709817633893, 0.6049493594306906, 0.16257501806765057, 0.16257501806765057, 0.24386252710147585, 0.16257501806765057, 0.16257501806765057, 0.08128750903382528, 0.6406631598276419, 0.08444397439039325, 0.4011088783543679, 0.06333298079279494, 0.042221987195196625, 0.021110993597598313, 0.042221987195196625, 0.021110993597598313, 0.08444397439039325, 0.06333298079279494, 0.021110993597598313, 0.021110993597598313, 0.021110993597598313, 0.021110993597598313, 0.021110993597598313, 0.042221987195196625, 0.021110993597598313, 0.6241261415386763, 0.6049339108122413, 0.6076423008985212, 0.1630866671025897, 0.1630866671025897, 0.08154333355129485, 0.08154333355129485, 0.08154333355129485, 0.24463000065388454, 0.08154333355129485, 0.08154333355129485, 0.08154333355129485, 0.11983481403494467, 0.029958703508736167, 0.029958703508736167, 0.059917407017472335, 0.059917407017472335, 0.0898761105262085, 0.20971092456115317, 0.059917407017472335, 0.029958703508736167, 0.059917407017472335, 0.029958703508736167, 0.11983481403494467, 0.059917407017472335, 0.029958703508736167, 0.6692042268282976, 0.2916725080272415, 0.09722416934241383, 0.09722416934241383, 0.09722416934241383, 0.38889667736965533, 0.8285831429208405, 0.669220188336597, 0.6458333198079317, 0.658366735593475, 0.6138640606937019, 0.613900006623337, 0.6691583669843286, 0.2701315910989032, 0.1350657955494516, 0.1350657955494516, 0.4051973866483548, 0.8697099441413384, 0.32135617097846614, 0.6427123419569323, 0.1425880789148866, 0.42776423674465974, 0.1425880789148866, 0.1425880789148866, 0.6444849361639751, 0.5931697052529091, 0.6458297637712953, 0.6319116016251464, 0.5931601409146422, 0.6241178829428888, 0.1238511777297897, 0.1238511777297897, 0.1238511777297897, 0.1238511777297897, 0.1238511777297897, 0.1238511777297897, 0.2477023554595794, 0.6241613450769871, 0.5092017061222387, 0.25460085306111935, 0.25460085306111935, 0.6391540648309504, 0.2255051055779235, 0.0751683685259745, 0.0751683685259745, 0.18792092131493623, 0.03758418426298725, 0.03758418426298725, 0.03758418426298725, 0.03758418426298725, 0.03758418426298725, 0.0751683685259745, 0.03758418426298725, 0.03758418426298725, 0.03758418426298725, 0.03758418426298725, 0.03758418426298725, 0.07407413920116604, 0.03703706960058302, 0.03703706960058302, 0.07407413920116604, 0.07407413920116604, 0.18518534800291508, 0.03703706960058302, 0.14814827840233208, 0.07407413920116604, 0.03703706960058302, 0.07407413920116604, 0.03703706960058302, 0.07407413920116604, 0.03703706960058302, 0.3014653895588908, 0.21173682427895907, 0.16938945942316724, 0.04234736485579181, 0.12704209456737545, 0.04234736485579181, 0.04234736485579181, 0.04234736485579181, 0.16938945942316724, 0.04234736485579181, 0.04234736485579181, 0.04234736485579181, 0.5430421661530934, 0.645802735061893, 0.6323839102249247, 0.6929195896230608, 0.24871178560419263, 0.49742357120838526, 0.6003538849983256, 0.631962639297154, 0.6406513777149129, 0.622565329304167, 0.6406341471553744, 0.8594869528443919, 0.1711852333906575, 0.1711852333906575, 0.342370466781315, 0.1711852333906575, 0.1711852333906575, 0.20074525636933452, 0.06691508545644484, 0.13383017091288968, 0.06691508545644484, 0.5353206836515587, 0.3828337980185818, 0.09570844950464545, 0.09570844950464545, 0.1914168990092909, 0.09570844950464545, 0.09570844950464545, 0.214532571152103, 0.214532571152103, 0.429065142304206, 0.6319657745580507, 0.7359094730862975, 0.26157710305065573, 0.05231542061013115, 0.05231542061013115, 0.05231542061013115, 0.05231542061013115, 0.05231542061013115, 0.05231542061013115, 0.2092616824405246, 0.05231542061013115, 0.05231542061013115, 0.05231542061013115, 0.05231542061013115, 0.6445087390794506, 0.12219944101329487, 0.12219944101329487, 0.24439888202658974, 0.24439888202658974, 0.12219944101329487, 0.6225667598551332, 0.6049286044062646, 0.32357888551791636, 0.32357888551791636, 0.3131461134737954, 0.644555170034945, 0.113989221543044, 0.455956886172176, 0.227978443086088, 0.113989221543044, 0.113989221543044, 0.6458229601340357, 0.12345256604315001, 0.24690513208630002, 0.12345256604315001, 0.061726283021575006, 0.061726283021575006, 0.061726283021575006, 0.12345256604315001, 0.061726283021575006, 0.061726283021575006, 0.061726283021575006, 0.061726283021575006, 0.6225613517092045, 0.6391333178798103, 0.2126455893853716, 0.2126455893853716, 0.1063227946926858, 0.1063227946926858, 0.1063227946926858, 0.2126455893853716, 0.17815991164026881, 0.17815991164026881, 0.35631982328053763, 0.17815991164026881, 0.17815991164026881, 0.26684235091977715, 0.26684235091977715, 0.26684235091977715, 0.18045398195043835, 0.18045398195043835, 0.18045398195043835, 0.3609079639008767, 0.18045398195043835, 0.1518906701539239, 0.1518906701539239, 0.1518906701539239, 0.4556720104617717, 0.1518906701539239, 0.5931818784931644, 0.6691852558486202, 0.23401167648010857, 0.07800389216003618, 0.07800389216003618, 0.23401167648010857, 0.15600778432007237, 0.07800389216003618, 0.15600778432007237, 0.05737174428701664, 0.02868587214350832, 0.14342936071754162, 0.08605761643052497, 0.08605761643052497, 0.02868587214350832, 0.11474348857403328, 0.05737174428701664, 0.02868587214350832, 0.02868587214350832, 0.05737174428701664, 0.11474348857403328, 0.02868587214350832, 0.08605761643052497, 0.02868587214350832, 0.6238703639367557, 0.31193518196837783, 0.08426823587732163, 0.08426823587732163, 0.08426823587732163, 0.16853647175464326, 0.08426823587732163, 0.08426823587732163, 0.08426823587732163, 0.08426823587732163, 0.08426823587732163, 0.16853647175464326, 0.5430185961449496, 0.05317340152617226, 0.05317340152617226, 0.23928030686777516, 0.07976010228925838, 0.05317340152617226, 0.05317340152617226, 0.07976010228925838, 0.05317340152617226, 0.02658670076308613, 0.05317340152617226, 0.1861069053416029, 0.02658670076308613, 0.02658670076308613, 0.02658670076308613, 0.07749615141665259, 0.04428351509523005, 0.16606318160711267, 0.06642527264284508, 0.11070878773807512, 0.07749615141665259, 0.011070878773807512, 0.03321263632142254, 0.09963790896426761, 0.011070878773807512, 0.05535439386903756, 0.011070878773807512, 0.06642527264284508, 0.05535439386903756, 0.05535439386903756, 0.011070878773807512, 0.022141757547615024, 0.022141757547615024, 0.011070878773807512, 0.6225510831957018, 0.6444831890220938, 0.2571711637767602, 0.5143423275535204, 0.2571711637767602, 0.07321894252482496, 0.07321894252482496, 0.21965682757447486, 0.07321894252482496, 0.07321894252482496, 0.07321894252482496, 0.07321894252482496, 0.07321894252482496, 0.21965682757447486, 0.07321894252482496, 0.307179951870253, 0.1535899759351265, 0.307179951870253, 0.1535899759351265, 0.1535899759351265, 0.6030899164478304, 0.3015449582239152, 0.21495070163637003, 0.42990140327274007, 0.5430644062789927, 0.31146526949749975, 0.6229305389949995, 0.03345024956429538, 0.10035074869288614, 0.06690049912859077, 0.06690049912859077, 0.1672512478214769, 0.10035074869288614, 0.06690049912859077, 0.06690049912859077, 0.06690049912859077, 0.03345024956429538, 0.06690049912859077, 0.03345024956429538, 0.03345024956429538, 0.03345024956429538, 0.03345024956429538, 0.19460424004556853, 0.38920848009113707, 0.19460424004556853, 0.19460424004556853, 0.6076032927224042, 0.4037703764825584, 0.08075407529651168, 0.16150815059302337, 0.08075407529651168, 0.16150815059302337, 0.08075407529651168, 0.6584335750078003, 0.613923607681792, 0.2644470471871593, 0.2644470471871593, 0.7138079089775426, 0.17845197724438566, 0.8325391079008312, 0.6003488292166012, 0.5553445044229719, 0.27767225221148595, 0.10172234726902812, 0.10172234726902812, 0.20344469453805625, 0.10172234726902812, 0.20344469453805625, 0.20344469453805625, 0.10172234726902812, 0.7359836596426322, 0.5682709619170365, 0.7034631999978148, 0.4566610297290557, 0.4566610297290557, 0.6076237005747812, 0.6458086339104115, 0.08240993782335138, 0.08240993782335138, 0.08240993782335138, 0.08240993782335138, 0.08240993782335138, 0.24722981347005413, 0.08240993782335138, 0.08240993782335138, 0.08240993782335138, 0.08240993782335138, 0.6319563939501521, 0.3305836543653622, 0.3305836543653622, 0.632385178035721, 0.5430442590150515, 0.24496774519811915, 0.24496774519811915, 0.24496774519811915, 0.5048397158562651, 0.12620992896406627, 0.12620992896406627, 0.12620992896406627, 0.7894120119245286, 0.6445548069125657, 0.869069311729346, 0.6370063028521288, 0.6218736192076119, 0.631947816204869, 0.1607516419125323, 0.1607516419125323, 0.1607516419125323, 0.1607516419125323, 0.1607516419125323, 0.6584024188809297, 0.1562883895812719, 0.05209612986042396, 0.10419225972084792, 0.4167690388833917, 0.05209612986042396, 0.10419225972084792, 0.05209612986042396, 0.05209612986042396, 0.05209612986042396, 0.20308562821860798, 0.06769520940620266, 0.06769520940620266, 0.27078083762481064, 0.06769520940620266, 0.13539041881240532, 0.06769520940620266, 0.06769520940620266, 0.13539041881240532, 0.6049228021810463, 0.04835209266402259, 0.14505627799206777, 0.04835209266402259, 0.09670418532804519, 0.04835209266402259, 0.04835209266402259, 0.09670418532804519, 0.09670418532804519, 0.04835209266402259, 0.09670418532804519, 0.09670418532804519, 0.6391348099101138, 0.20302784759291906, 0.4060556951858381, 0.4060556951858381, 0.593139795984592, 0.8594793080448982, 0.6049563028870013, 0.0526614587975625, 0.0526614587975625, 0.0526614587975625, 0.1579843763926875, 0.0526614587975625, 0.1579843763926875, 0.105322917595125, 0.0526614587975625, 0.105322917595125, 0.105322917595125, 0.0526614587975625, 0.4493286024392288, 0.4493286024392288, 0.6448213298923935, 0.1630759351692449, 0.08153796758462245, 0.040768983792311225, 0.08153796758462245, 0.040768983792311225, 0.12230695137693368, 0.08153796758462245, 0.08153796758462245, 0.040768983792311225, 0.040768983792311225, 0.040768983792311225, 0.12230695137693368, 0.040768983792311225, 0.15965245922520882, 0.07982622961260441, 0.039913114806302205, 0.07982622961260441, 0.039913114806302205, 0.039913114806302205, 0.11973934441890662, 0.07982622961260441, 0.039913114806302205, 0.11973934441890662, 0.15965245922520882, 0.039913114806302205, 0.039913114806302205, 0.658336899951713, 0.05301137993568093, 0.10602275987136187, 0.15903413980704278, 0.10602275987136187, 0.05301137993568093, 0.15903413980704278, 0.05301137993568093, 0.05301137993568093, 0.10602275987136187, 0.05301137993568093, 0.10602275987136187, 0.05301137993568093, 0.6458823477727879, 0.5682439620135048, 0.25797877500913463, 0.25797877500913463, 0.25797877500913463, 0.6691890944052037, 0.0888631660247137, 0.0888631660247137, 0.0888631660247137, 0.0888631660247137, 0.0888631660247137, 0.0888631660247137, 0.0888631660247137, 0.2665894980741411, 0.0888631660247137, 0.2066428361585178, 0.2066428361585178, 0.2066428361585178, 0.2066428361585178, 0.639152785870349, 0.6406447013767272, 0.6319802974923104, 0.6406619815968675, 0.6003280452307779, 0.14905031817992334, 0.14905031817992334, 0.07452515908996167, 0.14905031817992334, 0.07452515908996167, 0.037262579544980835, 0.037262579544980835, 0.07452515908996167, 0.07452515908996167, 0.037262579544980835, 0.037262579544980835, 0.037262579544980835, 0.037262579544980835, 0.037262579544980835, 0.5184582815765441, 0.25922914078827203, 0.5682566154949932, 0.20882439924044882, 0.31323659886067323, 0.10441219962022441, 0.10441219962022441, 0.10441219962022441, 0.10441219962022441, 0.5931339883717418, 0.6323569577702047, 0.6691344698300399, 0.1722801999418717, 0.12921014995640376, 0.21535024992733962, 0.08614009997093584, 0.04307004998546792, 0.04307004998546792, 0.04307004998546792, 0.04307004998546792, 0.12921014995640376, 0.04307004998546792, 0.04307004998546792, 0.04307004998546792, 0.24150411386778312, 0.48300822773556623, 0.6583830386845686, 0.3274729253856164, 0.6549458507712328, 0.669168669749841, 0.0835162370798379, 0.0835162370798379, 0.04175811853991895, 0.04175811853991895, 0.1670324741596758, 0.1670324741596758, 0.12527435561975683, 0.0835162370798379, 0.04175811853991895, 0.0835162370798379, 0.04175811853991895, 0.04175811853991895, 0.6076448606652393, 0.040192614375941356, 0.08038522875188271, 0.040192614375941356, 0.040192614375941356, 0.16077045750376542, 0.08038522875188271, 0.12057784312782406, 0.08038522875188271, 0.08038522875188271, 0.040192614375941356, 0.040192614375941356, 0.08038522875188271, 0.040192614375941356, 0.040192614375941356, 0.040192614375941356, 0.7986365979821497, 0.5931820871816392, 0.27126065242382424, 0.5425213048476485, 0.6406543477483111, 0.12816509963987935, 0.2563301992797587, 0.384495298919638, 0.2563301992797587, 0.5931857740355724, 0.613835401716683, 0.1317969642973492, 0.3953908928920476, 0.2635939285946984, 0.1317969642973492, 0.1317969642973492, 0.1880915565199996, 0.1880915565199996, 0.1880915565199996, 0.1880915565199996, 0.1880915565199996, 0.35012030913935716, 0.17506015456967858, 0.17506015456967858, 0.17506015456967858, 0.604907007798515, 0.43170409517543235, 0.14390136505847745, 0.14390136505847745, 0.14390136505847745, 0.2664392250127335, 0.2664392250127335, 0.2664392250127335, 0.6391254670732157, 0.151321587304828, 0.151321587304828, 0.302643174609656, 0.151321587304828, 0.151321587304828, 0.08009155029355428, 0.08009155029355428, 0.08009155029355428, 0.08009155029355428, 0.16018310058710855, 0.24027465088066283, 0.08009155029355428, 0.08009155029355428, 0.6241357192713786, 0.6323961067755598, 0.6089492858589821, 0.30447464292949106, 0.5430332241064368, 0.17145829751576455, 0.08572914875788228, 0.08572914875788228, 0.08572914875788228, 0.08572914875788228, 0.3429165950315291, 0.08572914875788228, 0.0873635046560003, 0.05460219041000019, 0.021840876164000076, 0.19656788547600068, 0.021840876164000076, 0.021840876164000076, 0.021840876164000076, 0.07644306657400027, 0.09828394273800034, 0.021840876164000076, 0.04368175232800015, 0.021840876164000076, 0.010920438082000038, 0.021840876164000076, 0.05460219041000019, 0.13104525698400046, 0.06552262849200023, 0.032761314246000116, 0.010920438082000038, 0.0776788187556537, 0.19419704688913425, 0.03883940937782685, 0.03883940937782685, 0.0776788187556537, 0.0776788187556537, 0.03883940937782685, 0.0776788187556537, 0.03883940937782685, 0.03883940937782685, 0.27187586564478794, 0.03883940937782685, 0.135118604905098, 0.135118604905098, 0.135118604905098, 0.270237209810196, 0.135118604905098, 0.053860960198528075, 0.26930480099264037, 0.053860960198528075, 0.10772192039705615, 0.16158288059558423, 0.10772192039705615, 0.053860960198528075, 0.053860960198528075, 0.053860960198528075, 0.053860960198528075, 0.053860960198528075, 0.34277769677372877, 0.34277769677372877, 0.24679760166155149, 0.49359520332310297, 0.24679760166155149, 0.38203564953840613, 0.19101782476920307, 0.36529929140515166, 0.18264964570257583, 0.18264964570257583, 0.658411424208892, 0.5793517012331801, 0.28967585061659007, 0.1778323530919297, 0.03556647061838594, 0.14226588247354377, 0.03556647061838594, 0.03556647061838594, 0.01778323530919297, 0.03556647061838594, 0.05334970592757892, 0.10669941185515784, 0.01778323530919297, 0.03556647061838594, 0.03556647061838594, 0.10669941185515784, 0.07113294123677188, 0.03556647061838594, 0.03556647061838594, 0.05334970592757892, 0.21448534141103404, 0.21448534141103404, 0.21448534141103404, 0.4289706828220681, 0.6584188076429285, 0.4327270728859341, 0.1442423576286447, 0.1442423576286447, 0.1442423576286447, 0.4568123967307228, 0.2284061983653614, 0.6584025027803639, 0.6319451074915777, 0.5232400639762719, 0.26162003198813594, 0.6241166583028058, 0.602816496515554, 0.301408248257777, 0.17752534895973296, 0.17752534895973296, 0.17752534895973296, 0.3550506979194659, 0.17752534895973296, 0.2938422046556868, 0.2938422046556868, 0.6584372949219582, 0.6076287515592398, 0.23952424957008117, 0.11976212478504059, 0.11976212478504059, 0.11976212478504059, 0.11976212478504059, 0.23952424957008117, 0.11976212478504059, 0.6323930639330203, 0.6049583859549759, 0.7034781827159101, 0.6691463070912667, 0.5931730093687182, 0.43075189625668514, 0.6447874608264648, 0.5931542285688886, 0.08962100434215305, 0.1792420086843061, 0.04481050217107652, 0.04481050217107652, 0.04481050217107652, 0.08962100434215305, 0.26886301302645915, 0.04481050217107652, 0.04481050217107652, 0.04481050217107652, 0.04481050217107652, 0.8697246331018021, 0.6391355914525768, 0.6138749269352474, 0.4445040710379876, 0.2222520355189938, 0.2222520355189938, 0.6076342121774433, 0.6583790677029234, 0.3065088347136851, 0.6130176694273702, 0.0536454396579276, 0.1609363189737828, 0.0536454396579276, 0.0536454396579276, 0.0536454396579276, 0.0536454396579276, 0.0536454396579276, 0.1072908793158552, 0.0536454396579276, 0.0536454396579276, 0.0536454396579276, 0.1072908793158552, 0.1609363189737828, 0.0536454396579276, 0.6003260168407718, 0.03522811149307254, 0.1761405574653627, 0.10568433447921763, 0.03522811149307254, 0.03522811149307254, 0.03522811149307254, 0.10568433447921763, 0.03522811149307254, 0.07045622298614508, 0.07045622298614508, 0.03522811149307254, 0.10568433447921763, 0.07045622298614508, 0.03522811149307254, 0.03522811149307254, 0.20029071969823192, 0.20029071969823192, 0.20029071969823192, 0.40058143939646385, 0.24878220820267397, 0.49756441640534793, 0.5430125929899943, 0.03637912107684507, 0.1455164843073803, 0.07275824215369014, 0.07275824215369014, 0.03637912107684507, 0.1455164843073803, 0.07275824215369014, 0.2182747264610704, 0.07275824215369014, 0.03637912107684507, 0.03637912107684507, 0.6692013780822067, 0.32527778908287, 0.20556780215878143, 0.20556780215878143, 0.41113560431756285, 0.20556780215878143, 0.4457900046045772, 0.5682668848440645, 0.6139107260761546, 0.21274213295750155, 0.21274213295750155, 0.4254842659150031, 0.23473826441463314, 0.46947652882926627, 0.23473826441463314, 0.1403498629129836, 0.0701749314564918, 0.0350874657282459, 0.1052623971847377, 0.0350874657282459, 0.0350874657282459, 0.0701749314564918, 0.0701749314564918, 0.0701749314564918, 0.0350874657282459, 0.0350874657282459, 0.1052623971847377, 0.0350874657282459, 0.1052623971847377, 0.0350874657282459, 0.0350874657282459, 0.6691791142496823, 0.2142520035905294, 0.2142520035905294, 0.2142520035905294, 0.2142520035905294, 0.6583953434389141, 0.568256384727368, 0.8464081975168453, 0.6391046864573039, 0.19661694821986528, 0.19661694821986528, 0.39323389643973056, 0.19661694821986528, 0.814322380148567, 0.09721137658244493, 0.09721137658244493, 0.09721137658244493, 0.09721137658244493, 0.19442275316488986, 0.09721137658244493, 0.09721137658244493, 0.09721137658244493, 0.09721137658244493, 0.41152371625568057, 0.20576185812784029, 0.20576185812784029, 0.6002538302690592, 0.3001269151345296, 0.1569211374669382, 0.3138422749338764, 0.3138422749338764, 0.1569211374669382, 0.2764650572091174, 0.2764650572091174, 0.2764650572091174, 0.6138703079953768, 0.635585875875226, 0.317792937937613, 0.8622375929248595, 0.6049401595965836, 0.7782713866860954, 0.6447769750059598, 0.20120461667518513, 0.20120461667518513, 0.20120461667518513, 0.40240923335037027, 0.039533130609753685, 0.039533130609753685, 0.19766565304876843, 0.039533130609753685, 0.039533130609753685, 0.07906626121950737, 0.039533130609753685, 0.19766565304876843, 0.11859939182926105, 0.039533130609753685, 0.039533130609753685, 0.07906626121950737, 0.07906626121950737, 0.039533130609753685, 0.2145150535225873, 0.2145150535225873, 0.4290301070451746, 0.6139056328088012, 0.4129556384382859, 0.645850196099515, 0.5682626155190356, 0.23899810058668222, 0.23899810058668222, 0.47799620117336444, 0.46371037947558064, 0.15457012649186022, 0.15457012649186022, 0.40630942146904275, 0.40630942146904275, 0.6319932407734359, 0.05503152800578907, 0.05503152800578907, 0.11006305601157813, 0.027515764002894533, 0.06878941000723633, 0.013757882001447267, 0.0412736460043418, 0.0825472920086836, 0.0412736460043418, 0.027515764002894533, 0.0825472920086836, 0.06878941000723633, 0.1238209380130254, 0.05503152800578907, 0.027515764002894533, 0.0412736460043418, 0.0412736460043418, 0.013757882001447267, 0.027515764002894533, 0.6241117597905332, 0.6319655488182269, 0.1834839182717377, 0.1834839182717377, 0.1834839182717377, 0.1834839182717377, 0.1834839182717377, 0.6076201512845877, 0.21898584820969022, 0.21898584820969022, 0.21898584820969022, 0.21898584820969022, 0.13475897705919168, 0.40427693117757507, 0.13475897705919168, 0.13475897705919168, 0.13475897705919168, 0.13475897705919168, 0.624144355184023, 0.6225595256678437, 0.6445395334480197, 0.19671902465685362, 0.09835951232842681, 0.09835951232842681, 0.19671902465685362, 0.19671902465685362, 0.19671902465685362, 0.09835951232842681, 0.6218862691410673, 0.15613371577004562, 0.31226743154009123, 0.05204457192334854, 0.10408914384669708, 0.05204457192334854, 0.05204457192334854, 0.05204457192334854, 0.05204457192334854, 0.10408914384669708, 0.18079478091088041, 0.09039739045544021, 0.2711921713663206, 0.18079478091088041, 0.09039739045544021, 0.09039739045544021, 0.3554126247496639, 0.08885315618741597, 0.08885315618741597, 0.08885315618741597, 0.08885315618741597, 0.08885315618741597, 0.08885315618741597, 0.08885315618741597, 0.6002813957282663, 0.6458329292586411, 0.644838498514261, 0.33157866132968833, 0.33157866132968833, 0.4492714157457279, 0.4492714157457279, 0.6076025419579926, 0.6225741569552584, 0.6584096622775157, 0.6323639048435543, 0.04920893428682742, 0.14762680286048227, 0.04920893428682742, 0.04920893428682742, 0.04920893428682742, 0.04920893428682742, 0.09841786857365484, 0.14762680286048227, 0.04920893428682742, 0.09841786857365484, 0.04920893428682742, 0.09841786857365484, 0.04920893428682742, 0.04920893428682742, 0.6003433366119766, 0.5430441533145659, 0.644503656135754, 0.10430333589159899, 0.10430333589159899, 0.312910007674797, 0.10430333589159899, 0.10430333589159899, 0.10430333589159899, 0.10430333589159899, 0.5078813743254609, 0.25394068716273044, 0.25394068716273044, 0.6218974186616423, 0.6584198424556263, 0.5430633703391435, 0.1231499198241948, 0.1641998930989264, 0.0410499732747316, 0.0820999465494632, 0.0820999465494632, 0.1641998930989264, 0.0410499732747316, 0.0410499732747316, 0.0410499732747316, 0.1231499198241948, 0.0820999465494632, 0.0410499732747316, 0.6003389051519554, 0.10808582047593038, 0.07025578330935475, 0.05944720126176171, 0.04323432819037215, 0.19455447685667468, 0.05944720126176171, 0.04323432819037215, 0.021617164095186076, 0.06485149228555823, 0.016212873071389557, 0.021617164095186076, 0.04863861921416867, 0.03783003716657563, 0.05944720126176171, 0.010808582047593038, 0.016212873071389557, 0.03783003716657563, 0.010808582047593038, 0.04863861921416867, 0.021617164095186076, 0.6076459528389363, 0.6091833831464553, 0.30459169157322763, 0.6138965717397521, 0.04533376053214696, 0.09066752106429392, 0.18133504212858784, 0.09066752106429392, 0.04533376053214696, 0.1360012815964409, 0.04533376053214696, 0.04533376053214696, 0.04533376053214696, 0.09066752106429392, 0.04533376053214696, 0.04533376053214696, 0.04533376053214696, 0.09066752106429392, 0.6445011373853267, 0.6139014279657189, 0.8349687798646619, 0.6323485657063339, 0.542997818184533, 0.6225804027568541, 0.6457896223282193, 0.1989201266394639, 0.049730031659865975, 0.01989201266394639, 0.13924408864762475, 0.029838018995919587, 0.029838018995919587, 0.14919009497959793, 0.11935207598367835, 0.029838018995919587, 0.01989201266394639, 0.01989201266394639, 0.009946006331973196, 0.03978402532789278, 0.03978402532789278, 0.059676037991839175, 0.009946006331973196, 0.009946006331973196, 0.009946006331973196, 0.01989201266394639, 0.08001921155314279, 0.020004802888285697, 0.08001921155314279, 0.06001440866485709, 0.040009605776571394, 0.12002881732971418, 0.040009605776571394, 0.020004802888285697, 0.16003842310628558, 0.08001921155314279, 0.040009605776571394, 0.020004802888285697, 0.06001440866485709, 0.12002881732971418, 0.020004802888285697, 0.020004802888285697, 0.020004802888285697, 0.020004802888285697, 0.543040749780921, 0.38806293039006823, 0.09701573259751706, 0.09701573259751706, 0.19403146519503411, 0.09701573259751706, 0.1864817306859185, 0.1864817306859185, 0.372963461371837, 0.1864817306859185, 0.6319403422189988, 0.5931743310253487, 0.6391623426719479, 0.6319939933060885, 0.17553464131860677, 0.35106928263721354, 0.17553464131860677, 0.17553464131860677, 0.6241521748165428, 0.6241512012773177, 0.8594746184452828, 0.5430535819597567, 0.6583611709153105, 0.5682308860593637, 0.13451895708419676, 0.13451895708419676, 0.13451895708419676, 0.13451895708419676, 0.2690379141683935, 0.13451895708419676, 0.18576482048667034, 0.27864723073000547, 0.09288241024333517, 0.09288241024333517, 0.09288241024333517, 0.09288241024333517, 0.09288241024333517, 0.09288241024333517, 0.5931392395616683, 0.6016169165403334, 0.3008084582701667, 0.6584004612335345, 0.6323447373765585, 0.6319498979169746, 0.6457988710662251, 0.1317648306753192, 0.2635296613506384, 0.1317648306753192, 0.0658824153376596, 0.0658824153376596, 0.1317648306753192, 0.0658824153376596, 0.0658824153376596, 0.1317648306753192, 0.13773318026744052, 0.4131995408023216, 0.13773318026744052, 0.13773318026744052, 0.13773318026744052, 0.5931866435832734, 0.5931418825798539, 0.10531063121897066, 0.03008875177684876, 0.04513312766527314, 0.09026625533054627, 0.01504437588842438, 0.12035500710739504, 0.0752218794421219, 0.03008875177684876, 0.1353993829958194, 0.03008875177684876, 0.01504437588842438, 0.04513312766527314, 0.0752218794421219, 0.03008875177684876, 0.09026625533054627, 0.01504437588842438, 0.03008875177684876, 0.01504437588842438, 0.01504437588842438, 0.1228090389574512, 0.1228090389574512, 0.1228090389574512, 0.1228090389574512, 0.1228090389574512, 0.2456180779149024, 0.1745209171569014, 0.1745209171569014, 0.1745209171569014, 0.1745209171569014, 0.3490418343138028, 0.32580637473069674, 0.32580637473069674, 0.6448187286658025, 0.2129198440601487, 0.2129198440601487, 0.4258396881202974, 0.2129198440601487, 0.6960533393385367, 0.13921066786770736, 0.13921066786770736, 0.6241181969539406, 0.6391600333814357, 0.5430466689972896, 0.600341651401161, 0.6122322869242116, 0.3061161434621058, 0.0913558932802183, 0.0913558932802183, 0.2740676798406549, 0.0913558932802183, 0.2740676798406549, 0.0913558932802183, 0.0913558932802183, 0.6139149902719582, 0.23544001597687314, 0.18835201278149852, 0.04708800319537463, 0.09417600639074926, 0.04708800319537463, 0.1412640095861239, 0.04708800319537463, 0.1412640095861239, 0.04708800319537463, 0.04708800319537463, 0.4134183494466408, 0.0973981734134161, 0.11363120231565212, 0.04869908670670805, 0.016233028902236015, 0.03246605780447203, 0.25972846243577624, 0.016233028902236015, 0.04869908670670805, 0.03246605780447203, 0.06493211560894406, 0.03246605780447203, 0.03246605780447203, 0.06493211560894406, 0.03246605780447203, 0.06493211560894406, 0.016233028902236015, 0.08948538897924696, 0.08948538897924696, 0.2684561669377409, 0.08948538897924696, 0.08948538897924696, 0.08948538897924696, 0.17897077795849392, 0.08948538897924696, 0.6457621859641013, 0.6448099120328441, 0.6323595692433766, 0.6584135217585402, 0.6448134441332498, 0.6139053070794619, 0.05867438076397, 0.29337190381984996, 0.11734876152794, 0.05867438076397, 0.05867438076397, 0.05867438076397, 0.05867438076397, 0.11734876152794, 0.05867438076397, 0.5682721158162352, 0.1531471968641445, 0.1531471968641445, 0.1531471968641445, 0.306294393728289, 0.1531471968641445, 0.1531471968641445, 0.17589170207420038, 0.35178340414840076, 0.17589170207420038, 0.17589170207420038, 0.027958677202094464, 0.08387603160628339, 0.05591735440418893, 0.027958677202094464, 0.05591735440418893, 0.05591735440418893, 0.05591735440418893, 0.08387603160628339, 0.05591735440418893, 0.027958677202094464, 0.1397933860104723, 0.05591735440418893, 0.05591735440418893, 0.05591735440418893, 0.05591735440418893, 0.05591735440418893, 0.027958677202094464, 0.05591735440418893, 0.027958677202094464, 0.13516902982095774, 0.04505634327365258, 0.04505634327365258, 0.09011268654730516, 0.04505634327365258, 0.04505634327365258, 0.09011268654730516, 0.04505634327365258, 0.09011268654730516, 0.04505634327365258, 0.04505634327365258, 0.04505634327365258, 0.13516902982095774, 0.04505634327365258, 0.04505634327365258, 0.6076211068586376, 0.2913709929507828, 0.1456854964753914, 0.2913709929507828, 0.4187959456571965, 0.593151898441471, 0.6583994264817572, 0.24247467053223826, 0.12123733526611913, 0.12123733526611913, 0.12123733526611913, 0.12123733526611913, 0.12123733526611913, 0.12123733526611913, 0.11616477174338984, 0.05808238587169492, 0.05808238587169492, 0.2904119293584746, 0.05808238587169492, 0.17424715761508477, 0.05808238587169492, 0.05808238587169492, 0.05808238587169492, 0.22098648727542694, 0.22098648727542694, 0.4419729745508539, 0.22098648727542694, 0.8594335063382683, 0.6747173823710956, 0.1041147523729756, 0.2313661163843902, 0.046273223276878044, 0.046273223276878044, 0.1041147523729756, 0.046273223276878044, 0.05784152909609755, 0.011568305819219511, 0.023136611638439022, 0.05784152909609755, 0.046273223276878044, 0.09254644655375609, 0.011568305819219511, 0.023136611638439022, 0.011568305819219511, 0.03470491745765853, 0.011568305819219511, 0.046273223276878044, 0.011568305819219511, 0.613909985770418, 0.18725448460946276, 0.053501281316989356, 0.053501281316989356, 0.08025192197548403, 0.10700256263397871, 0.1337532032924734, 0.10700256263397871, 0.10700256263397871, 0.026750640658494678, 0.026750640658494678, 0.053501281316989356, 0.053501281316989356, 0.026750640658494678, 0.6049411514789862, 0.632382312790559, 0.10258284148679411, 0.10258284148679411, 0.20516568297358823, 0.20516568297358823, 0.3077485244603823, 0.10258284148679411, 0.6458112647904919, 0.3831454469775143, 0.09578636174437857, 0.09578636174437857, 0.09578636174437857, 0.09578636174437857, 0.09578636174437857, 0.09578636174437857, 0.649754688297548, 0.6496928362608664, 0.5464450119204376, 0.1821483373068125, 0.1821483373068125, 0.3131797180266073, 0.3131797180266073, 0.18790783081596438, 0.12527188721064292, 0.6218974799238488, 0.632370598518469, 0.4063547075775143, 0.4063547075775143, 0.11355750547935066, 0.45423002191740264, 0.22711501095870132, 0.11355750547935066, 0.11355750547935066, 0.6130239448812637, 0.30651197244063183, 0.6003302296661137, 0.6520493251601017, 0.3247564854474513, 0.3247564854474513, 0.4226842824311915, 0.21134214121559575, 0.4226842824311915, 0.6225590489142988, 0.6003446473380408, 0.055438436060240404, 0.3326306163614424, 0.055438436060240404, 0.11087687212048081, 0.055438436060240404, 0.11087687212048081, 0.055438436060240404, 0.055438436060240404, 0.055438436060240404, 0.6692496686175484, 0.5788300446905273, 0.2894150223452637, 0.40101278421373454, 0.028643770300981038, 0.08593131090294312, 0.08593131090294312, 0.14321885150490518, 0.028643770300981038, 0.028643770300981038, 0.028643770300981038, 0.028643770300981038, 0.028643770300981038, 0.028643770300981038, 0.057287540601962075, 0.028643770300981038, 0.14035022403763567, 0.14035022403763567, 0.14035022403763567, 0.28070044807527134, 0.14035022403763567, 0.14035022403763567, 0.4504946372381094, 0.2252473186190547, 0.2252473186190547, 0.8343985378728269, 0.6003412144961977, 0.8264359429806568, 0.6225644919115301, 0.09566935125592392, 0.07049320618857552, 0.16112732843102975, 0.02014091605387872, 0.09063412224245423, 0.08056366421551488, 0.015105687040409039, 0.04028183210775744, 0.09063412224245423, 0.050352290134696796, 0.03524660309428776, 0.01007045802693936, 0.03524660309428776, 0.025176145067348398, 0.06545797717510583, 0.01007045802693936, 0.03524660309428776, 0.030211374080818077, 0.03524660309428776, 0.6401609676524859, 0.32008048382624293, 0.20687126384819657, 0.41374252769639314, 0.20687126384819657, 0.20687126384819657, 0.16426453361040338, 0.08213226680520169, 0.08213226680520169, 0.08213226680520169, 0.08213226680520169, 0.08213226680520169, 0.16426453361040338, 0.08213226680520169, 0.08213226680520169, 0.08213226680520169, 0.05778940232765356, 0.05778940232765356, 0.17336820698296068, 0.05778940232765356, 0.05778940232765356, 0.05778940232765356, 0.05778940232765356, 0.23115760931061424, 0.05778940232765356, 0.05778940232765356, 0.05778940232765356, 0.05778940232765356, 0.05778940232765356, 0.05778940232765356, 0.05778940232765356, 0.6445117571152047, 0.7034915430859908, 0.3199562716849106, 0.6399125433698212, 0.07934045799098567, 0.07934045799098567, 0.15868091598197134, 0.07934045799098567, 0.238021373972957, 0.15868091598197134, 0.15868091598197134, 0.07934045799098567, 0.6566124319524451, 0.32830621597622256, 0.1060124210788064, 0.1060124210788064, 0.0530062105394032, 0.0530062105394032, 0.1060124210788064, 0.1060124210788064, 0.0530062105394032, 0.1060124210788064, 0.0530062105394032, 0.0530062105394032, 0.0530062105394032, 0.1060124210788064, 0.0530062105394032, 0.0530062105394032, 0.20903783127954598, 0.20903783127954598, 0.41807566255909195, 0.20903783127954598, 0.5430213652505957, 0.09163440030143043, 0.2749032009042913, 0.045817200150715214, 0.13745160045214566, 0.13745160045214566, 0.09163440030143043, 0.045817200150715214, 0.045817200150715214, 0.045817200150715214, 0.045817200150715214, 0.045817200150715214, 0.23919263873162677, 0.23919263873162677, 0.23919263873162677, 0.10430725903006965, 0.15646088854510448, 0.05215362951503483, 0.15646088854510448, 0.10430725903006965, 0.05215362951503483, 0.10430725903006965, 0.05215362951503483, 0.10430725903006965, 0.10430725903006965, 0.05215362951503483, 0.6376727723531961, 0.31883638617659804, 0.6218978168661997, 0.6406590851313027, 0.05539676419485843, 0.11079352838971686, 0.11079352838971686, 0.05539676419485843, 0.05539676419485843, 0.11079352838971686, 0.11079352838971686, 0.11079352838971686, 0.11079352838971686, 0.05539676419485843, 0.05539676419485843, 0.2464533018546543, 0.4929066037093086, 0.2464533018546543, 0.6138715515492735, 0.5682546924371771, 0.604916256214423, 0.6323650204461994, 0.5931898087584344, 0.22738675524452545, 0.04547735104890509, 0.09095470209781018, 0.04547735104890509, 0.4092961594401458, 0.04547735104890509, 0.04547735104890509, 0.04547735104890509, 0.04547735104890509, 0.10197197815541068, 0.10197197815541068, 0.10197197815541068, 0.05098598907770534, 0.15295796723311603, 0.10197197815541068, 0.05098598907770534, 0.05098598907770534, 0.10197197815541068, 0.10197197815541068, 0.10197197815541068, 0.34494201663095636, 0.17247100831547818, 0.17247100831547818, 0.17247100831547818, 0.6391544200986926, 0.6139050701856141, 0.509727011518267, 0.12743175287956676, 0.12743175287956676, 0.12743175287956676, 0.19006509714339614, 0.3801301942867923, 0.19006509714339614, 0.24480971154454614, 0.4896194230890923, 0.24480971154454614, 0.313793633543502, 0.627587267087004, 0.1346529866399356, 0.0673264933199678, 0.0673264933199678, 0.0673264933199678, 0.0673264933199678, 0.0673264933199678, 0.33663246659983903, 0.0673264933199678, 0.0673264933199678, 0.0673264933199678, 0.6396001969336127, 0.31980009846680635, 0.0697313866575187, 0.0697313866575187, 0.1394627733150374, 0.0697313866575187, 0.1394627733150374, 0.0697313866575187, 0.20919415997255608, 0.0697313866575187, 0.0697313866575187, 0.0697313866575187, 0.0697313866575187, 0.6323262048592998, 0.6457709406854516, 0.059456338663889906, 0.059456338663889906, 0.11891267732777981, 0.29728169331944954, 0.059456338663889906, 0.059456338663889906, 0.059456338663889906, 0.1783690159916697, 0.8697003496041623, 0.6218831448965714, 0.6319430508914392, 0.15814416532738557, 0.15814416532738557, 0.15814416532738557, 0.15814416532738557, 0.15814416532738557, 0.15814416532738557, 0.15814416532738557, 0.40985144604066565, 0.6323678094700332, 0.631965348160741, 0.03626050415036285, 0.0725210083007257, 0.10878151245108857, 0.25382352905253996, 0.21756302490217713, 0.03626050415036285, 0.0725210083007257, 0.03626050415036285, 0.03626050415036285, 0.0725210083007257, 0.03626050415036285, 0.02900325856751795, 0.2610293271076615, 0.02900325856751795, 0.20302280997262565, 0.02900325856751795, 0.14501629283758974, 0.02900325856751795, 0.02900325856751795, 0.1160130342700718, 0.0580065171350359, 0.02900325856751795, 0.02900325856751795, 0.6583931621080129, 0.10013733717965699, 0.10013733717965699, 0.18024720692338256, 0.06008240230779419, 0.020027467435931395, 0.04005493487186279, 0.08010986974372558, 0.06008240230779419, 0.06008240230779419, 0.020027467435931395, 0.08010986974372558, 0.04005493487186279, 0.020027467435931395, 0.04005493487186279, 0.04005493487186279, 0.04005493487186279, 0.020027467435931395, 0.632360634122123, 0.37335477984899346, 0.12445159328299782, 0.06222579664149891, 0.06222579664149891, 0.24890318656599564, 0.06222579664149891, 0.06222579664149891, 0.10802784467397757, 0.024006187705328346, 0.04801237541065669, 0.16804331393729843, 0.036009281557992524, 0.012003093852664173, 0.15604022008463425, 0.12003093852664173, 0.012003093852664173, 0.09602475082131338, 0.036009281557992524, 0.036009281557992524, 0.024006187705328346, 0.04801237541065669, 0.024006187705328346, 0.012003093852664173, 0.012003093852664173, 0.012003093852664173, 0.012003093852664173, 0.2043824512078456, 0.2043824512078456, 0.2043824512078456, 0.4087649024156912, 0.6448170857966632, 0.11953554900520115, 0.15938073200693487, 0.03984518300173372, 0.1992259150086686, 0.03984518300173372, 0.03984518300173372, 0.15938073200693487, 0.03984518300173372, 0.03984518300173372, 0.03984518300173372, 0.03984518300173372, 0.03984518300173372, 0.5430611504813471, 0.6076310723099841, 0.6076434271932202, 0.18449214230401847, 0.7379685692160739, 0.6387378747849771, 0.16095149868009076, 0.16095149868009076, 0.16095149868009076, 0.3219029973601815, 0.16095149868009076, 0.5430126352652693, 0.6075683158962942, 0.6445035426781042, 0.6049431848480801, 0.6102092695483635, 0.30510463477418176, 0.5429943095505902, 0.5682717696459835, 0.5682272325915052, 0.6448481922069704, 0.6448029848475143, 0.176778366108937, 0.353556732217874, 0.176778366108937, 0.176778366108937, 0.05136689521527707, 0.3082013712916624, 0.1541006856458312, 0.05136689521527707, 0.20546758086110828, 0.05136689521527707, 0.05136689521527707, 0.10273379043055414, 0.6391571912006357, 0.6138830990760652, 0.6049516904303937, 0.13738306202428044, 0.06869153101214022, 0.13738306202428044, 0.06869153101214022, 0.13738306202428044, 0.06869153101214022, 0.06869153101214022, 0.06869153101214022, 0.06869153101214022, 0.20607459303642064, 0.34290797797037015, 0.06858159559407402, 0.06858159559407402, 0.06858159559407402, 0.20574478678222208, 0.06858159559407402, 0.06858159559407402, 0.6448438656075471, 0.6691402672076974, 0.10706835375138964, 0.05353417687569482, 0.05353417687569482, 0.10706835375138964, 0.05353417687569482, 0.10706835375138964, 0.05353417687569482, 0.05353417687569482, 0.05353417687569482, 0.05353417687569482, 0.10706835375138964, 0.05353417687569482, 0.05353417687569482, 0.05353417687569482, 0.05353417687569482, 0.7782825232855043, 0.6448270800466988, 0.6391474924489583, 0.49953737576849866, 0.24976868788424933, 0.13452908569008276, 0.13452908569008276, 0.40358725707024823, 0.13452908569008276, 0.093600300373671, 0.187200600747342, 0.093600300373671, 0.093600300373671, 0.28080090112101297, 0.093600300373671, 0.093600300373671, 0.6003374384155138, 0.658396797667545, 0.08849989400162633, 0.17699978800325267, 0.08849989400162633, 0.265499682004879, 0.08849989400162633, 0.17699978800325267, 0.08849989400162633, 0.624142062718458, 0.3037767149696329, 0.6075534299392658, 0.21191735533604017, 0.10595867766802008, 0.5297933883401005, 0.10595867766802008, 0.10595867766802008, 0.2838409598644766, 0.5676819197289532, 0.6225569903696376, 0.44384719464332173, 0.22192359732166086, 0.22192359732166086, 0.22192359732166086, 0.6225790070361273, 0.645810154885344, 0.19525121447461366, 0.032541869079102274, 0.06508373815820455, 0.032541869079102274, 0.1301674763164091, 0.032541869079102274, 0.032541869079102274, 0.09762560723730683, 0.032541869079102274, 0.032541869079102274, 0.06508373815820455, 0.06508373815820455, 0.16270934539551138, 0.06508373815820455, 0.2228398581295314, 0.1114199290647657, 0.05570996453238285, 0.1114199290647657, 0.05570996453238285, 0.05570996453238285, 0.05570996453238285, 0.05570996453238285, 0.05570996453238285, 0.16712989359714855, 0.05570996453238285, 0.6448149403413818, 0.12217591986723911, 0.061087959933619554, 0.12217591986723911, 0.061087959933619554, 0.061087959933619554, 0.12217591986723911, 0.061087959933619554, 0.061087959933619554, 0.24435183973447822, 0.12217591986723911, 0.5931628884566051, 0.18159247710060214, 0.5447774313018064, 0.18159247710060214, 0.18159247710060214, 0.5682500386911234, 0.8605587390037963, 0.543039798490326, 0.24899009376192957, 0.49798018752385914, 0.24899009376192957, 0.6445105998199577, 0.2698373404527537, 0.2698373404527537, 0.2698373404527537, 0.6407213749911086, 0.3203606874955543, 0.1482866035333211, 0.1482866035333211, 0.2965732070666422, 0.1482866035333211, 0.1482866035333211, 0.19824626827940522, 0.19824626827940522, 0.39649253655881045, 0.14985812885530223, 0.22478719328295332, 0.14985812885530223, 0.14985812885530223, 0.07492906442765111, 0.14985812885530223, 0.07492906442765111, 0.07492906442765111, 0.09583146087235885, 0.4791573043617942, 0.1916629217447177, 0.09583146087235885, 0.09583146087235885, 0.6406454377307291, 0.4289282933079047, 0.21446414665395236, 0.21446414665395236, 0.6445138674878248, 0.6583705945712569, 0.037169625096846855, 0.11150887529054056, 0.11150887529054056, 0.05575443764527028, 0.2044329380326577, 0.018584812548423427, 0.018584812548423427, 0.018584812548423427, 0.018584812548423427, 0.07433925019369371, 0.09292406274211713, 0.037169625096846855, 0.037169625096846855, 0.05575443764527028, 0.018584812548423427, 0.037169625096846855, 0.05575443764527028, 0.037169625096846855, 0.6225425809381794, 0.3159609615541269, 0.6319219231082538, 0.12767708175294526, 0.12767708175294526, 0.2553541635058905, 0.2553541635058905, 0.2699528195014796, 0.08998427316715986, 0.08998427316715986, 0.17996854633431972, 0.08998427316715986, 0.08998427316715986, 0.08998427316715986, 0.6049371343753448, 0.14913718575165022, 0.14913718575165022, 0.4474115572549507, 0.14913718575165022, 0.14913718575165022, 0.6391462845664307, 0.15608553469533626, 0.15608553469533626, 0.3121710693906725, 0.15608553469533626, 0.5696308150183332, 0.2848154075091666, 0.6003379377292425, 0.15431333178519516, 0.10287555452346343, 0.05143777726173172, 0.05143777726173172, 0.10287555452346343, 0.05143777726173172, 0.05143777726173172, 0.05143777726173172, 0.05143777726173172, 0.15431333178519516, 0.05143777726173172, 0.05143777726173172, 0.05143777726173172, 0.08480285511275117, 0.1908064240036901, 0.12720428266912673, 0.02120071377818779, 0.02120071377818779, 0.02120071377818779, 0.16960571022550233, 0.02120071377818779, 0.08480285511275117, 0.08480285511275117, 0.04240142755637558, 0.02120071377818779, 0.02120071377818779, 0.02120071377818779, 0.02120071377818779, 0.02120071377818779, 0.02120071377818779, 0.02120071377818779, 0.46189529460908213, 0.23094764730454106, 0.23094764730454106, 0.35447139207940415, 0.35447139207940415, 0.6003508577607679, 0.5430581906991819, 0.4582235575308786, 0.4582235575308786, 0.710034092931767, 0.6458237000997207, 0.6323788136768158, 0.6225734242157986, 0.6458505250006829, 0.6075664733263858, 0.41761851287593915, 0.6094571738781986, 0.3047285869390993, 0.6218857178015222, 0.6241057623577775, 0.4544018360681498, 0.4544018360681498, 0.6692064694752611, 0.6138988517874977, 0.219565800537254, 0.0878263202149016, 0.13173948032235241, 0.0878263202149016, 0.0878263202149016, 0.0439131601074508, 0.0439131601074508, 0.0439131601074508, 0.0439131601074508, 0.0439131601074508, 0.0878263202149016, 0.0439131601074508, 0.0439131601074508, 0.6073779448800635, 0.5018157858145843, 0.25090789290729215, 0.6391489845454424, 0.624160120266336, 0.28946051169088494, 0.5789210233817699, 0.23150994872926822, 0.46301989745853644, 0.23150994872926822, 0.6458420970139479, 0.631917043578836, 0.24869588239165555, 0.24869588239165555, 0.24869588239165555, 0.6458420559023565, 0.042649259924071266, 0.042649259924071266, 0.21324629962035632, 0.042649259924071266, 0.08529851984814253, 0.08529851984814253, 0.08529851984814253, 0.1279477797722138, 0.08529851984814253, 0.042649259924071266, 0.1279477797722138, 0.042649259924071266, 0.6406405041912385, 0.11917356020508002, 0.08412251308593884, 0.02103062827148471, 0.04907146596679766, 0.04206125654296942, 0.16824502617187767, 0.07010209423828237, 0.08412251308593884, 0.04907146596679766, 0.04907146596679766, 0.04907146596679766, 0.014020418847656473, 0.04206125654296942, 0.035051047119141185, 0.035051047119141185, 0.035051047119141185, 0.02103062827148471, 0.035051047119141185, 0.014020418847656473, 0.6076276935758128, 0.6075750379590501, 0.6075971160339938, 0.1440217294384744, 0.0720108647192372, 0.0720108647192372, 0.1440217294384744, 0.0720108647192372, 0.0720108647192372, 0.21603259415771164, 0.0720108647192372, 0.0720108647192372, 0.0720108647192372, 0.8692218998843133, 0.6319266988886365, 0.11742461838563055, 0.21460361291166963, 0.16601411564865007, 0.12147374315754884, 0.02024562385959147, 0.08908074498220248, 0.016196499087673177, 0.04858949726301954, 0.028343873403428064, 0.028343873403428064, 0.03644212294726465, 0.016196499087673177, 0.008098249543836589, 0.016196499087673177, 0.016196499087673177, 0.016196499087673177, 0.004049124771918294, 0.02429474863150977, 0.012147374315754885, 0.5091502148680432, 0.1272875537170108, 0.1272875537170108, 0.1272875537170108, 0.43245289164405093, 0.14415096388135032, 0.14415096388135032, 0.6319674049060113, 0.045673658147524165, 0.18269463259009666, 0.045673658147524165, 0.045673658147524165, 0.09134731629504833, 0.045673658147524165, 0.09134731629504833, 0.045673658147524165, 0.045673658147524165, 0.045673658147524165, 0.1370209744425725, 0.09134731629504833, 0.045673658147524165, 0.045673658147524165, 0.045673658147524165, 0.16801733236282035, 0.08400866618141017, 0.08400866618141017, 0.08400866618141017, 0.08400866618141017, 0.16801733236282035, 0.16801733236282035, 0.08400866618141017, 0.08400866618141017, 0.13137043781637348, 0.13137043781637348, 0.13137043781637348, 0.26274087563274695, 0.4808102447466564, 0.16027008158221878, 0.16027008158221878, 0.16027008158221878, 0.08568844835534471, 0.12853267253301706, 0.042844224177672356, 0.042844224177672356, 0.042844224177672356, 0.21422112088836176, 0.042844224177672356, 0.08568844835534471, 0.042844224177672356, 0.042844224177672356, 0.08568844835534471, 0.042844224177672356, 0.042844224177672356, 0.042844224177672356, 0.31356909380551296, 0.10452303126850433, 0.31356909380551296, 0.10452303126850433, 0.20904606253700866, 0.600329075034027, 0.13007374411945846, 0.13007374411945846, 0.3902212323583754, 0.2601474882389169, 0.13007374411945846, 0.6225777508928235, 0.6391570490922592, 0.21059967308633742, 0.21059967308633742, 0.42119934617267485, 0.21059967308633742, 0.543036247034886, 0.2601652710230842, 0.5203305420461684, 0.2601652710230842, 0.3946642326028616, 0.13155474420095387, 0.13155474420095387, 0.13155474420095387, 0.13155474420095387, 0.607571614938866], \"Term\": [\"000000\", \"05\", \"0j0antisense\", \"0xoguanine\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"1\", \"100\", \"10000\", \"12\", \"12\", \"12\", \"12\", \"12\", \"12\", \"12\", \"12\", \"12\", \"14\", \"14\", \"14\", \"14\", \"145\", \"145\", \"1982\", \"1982\", \"1css\", \"1i31\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2\", \"2009\", \"2021\", \"293ft\", \"3\", \"3\", \"3\", \"3\", \"3\", \"3\", \"3\", \"3\", \"3\", \"3\", \"3\", \"30\", \"30\", \"30\", \"30\", \"30utrs\", \"32\", \"36\", \"36\", \"36\", \"37\", \"37\", \"39\", \"39\", \"3c\", \"3c\", \"3c\", \"4\", \"4\", \"4\", \"4\", \"4\", \"4\", \"4\", \"4\", \"4\", \"4\", \"4\", \"40\", \"40\", \"40\", \"40\", \"41\", \"42\", \"44\", \"450\", \"4b\", \"5\", \"5\", \"5\", \"5\", \"5\", \"5\", \"5\", \"5\", \"5\", \"55\", \"55\", \"585\", \"6\", \"6\", \"6\", \"6\", \"6\", \"6\", \"6\", \"700000\", \"72\", \"72hpi\", \"8a\", \"aaron\", \"ab\", \"abrogates\", \"abundantly\", \"abuts\", \"acape\", \"acape\", \"accepted\", \"accepted\", \"account\", \"accumulat\", \"acetylated\", \"achieve\", \"acid\", \"acid\", \"acid\", \"acid\", \"acid\", \"acid\", \"acid\", \"acid\", \"acid\", \"acid\", \"acid\", \"acid\", \"across\", \"across\", \"across\", \"action\", \"activator\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"activity\", \"actual\", \"adapting\", \"added\", \"addition\", \"addition\", \"addition\", \"addition\", \"addition\", \"additionally\", \"address\", \"adenocarcinoma\", \"adjacent\", \"advance\", \"affected\", \"affinity\", \"affinity\", \"affinity\", \"al\", \"al\", \"al\", \"al\", \"albeit\", \"algae\", \"align\", \"allowed\", \"allowing\", \"allowing\", \"alphavirus\", \"alphavirus\", \"alphavirus\", \"alphavirus\", \"alphavirus\", \"alphavirus\", \"alphavirus\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"also\", \"alter\", \"alternatively\", \"although\", \"although\", \"although\", \"although\", \"although\", \"although\", \"although\", \"although\", \"although\", \"amazingly\", \"ammatory\", \"ammatory\", \"ammatory\", \"ammatory\", \"ampullaviridae\", \"analog\", \"ance\", \"anelloviridae\", \"antagonizing\", \"ap\", \"ap\", \"ap\", \"ap\", \"ap\", \"ap\", \"ap\", \"ap\", \"ap\", \"ap\", \"ape\", \"ape\", \"ape\", \"ape\", \"ape\", \"ape\", \"ape\", \"ape\", \"apoptosis\", \"appear\", \"application\", \"architecture\", \"are\", \"are\", \"are\", \"are\", \"are\", \"aromatic\", \"arrest\", \"arrest\", \"arrest\", \"arrest\", \"arrested\", \"arsenite\", \"arsenite\", \"arsenite\", \"art\", \"arthritis\", \"ascertain\", \"asos\", \"assay\", \"assay\", \"assay\", \"assay\", \"assemble\", \"assembled\", \"assembly\", \"assembly\", \"assembly\", \"assembly\", \"assembly\", \"assembly\", \"assembly\", \"assembly\", \"assembly\", \"assembly\", \"assessment\", \"associate\", \"associate\", \"associate\", \"associate\", \"associated\", \"associated\", \"associated\", \"associated\", \"associated\", \"associated\", \"atf4\", \"attempting\", \"augmenting\", \"aumaivirus\", \"autoinhibitory\", \"avg\", \"axis\", \"b\", \"b\", \"b\", \"b\", \"b\", \"b\", \"b\", \"b\", \"bacterial\", \"bacterial\", \"bacterial\", \"bacterial\", \"bacterium\", \"band\", \"bar\", \"bartogtaviriformidae\", \"base\", \"base\", \"base\", \"base\", \"base\", \"base\", \"base\", \"base\", \"base\", \"base\", \"base\", \"base\", \"base\", \"baseline\", \"belong\", \"belong\", \"belong\", \"belongs\", \"belongs\", \"belongs\", \"belongs\", \"belongs\", \"biased\", \"bicaudaviridae\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"binding\", \"biochemical\", \"biochemistry\", \"biomolecular\", \"bird\", \"bled\", \"blood\", \"bloom\", \"blue\", \"blumei\", \"bona\", \"bona\", \"bona\", \"bona\", \"bona\", \"bona\", \"bona\", \"bond\", \"bond\", \"bond\", \"bond\", \"borne\", \"brachygtaviriformidae\", \"brain\", \"brief\", \"brought\", \"brought\", \"bruton\", \"btk\", \"btk\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"c\", \"cadang\", \"cal\", \"calicivirus\", \"camp\", \"canonical\", \"canonical\", \"canonical\", \"canonical\", \"canonical\", \"canonical\", \"capped\", \"caprin\", \"caprin\", \"capsid\", \"capsid\", \"capsid\", \"capsid\", \"capsid\", \"capsid\", \"cascade\", \"catalogue\", \"categorization\", \"categorization\", \"categorize\", \"cation\", \"cation\", \"causal\", \"cdd\", \"cdknla\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cell\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"cellular\", \"center\", \"center\", \"certain\", \"certain\", \"certain\", \"certain\", \"certain\", \"certain\", \"certain\", \"certain\", \"cgas\", \"cgas\", \"cgas\", \"cgas\", \"cgas\", \"challenge\", \"challenging\", \"change\", \"change\", \"change\", \"change\", \"chaperon\", \"chaperone\", \"characterizes\", \"charge\", \"checkpoint\", \"chicken\", \"chicken\", \"chicken\", \"chromatin\", \"chromatin\", \"chromatin\", \"chromosome\", \"chromosome\", \"chromosome\", \"chromosome\", \"citrate\", \"classification\", \"classification\", \"classification\", \"classification\", \"classification\", \"classification\", \"classification\", \"clavaviridae\", \"cleavage\", \"cleavage\", \"cleavage\", \"cleavage\", \"cleavage\", \"cleavage\", \"cleavage\", \"cleaved\", \"cleaved\", \"cleaved\", \"cleaved\", \"clue\", \"clustered\", \"coat\", \"coat\", \"coconut\", \"coded\", \"coding\", \"coding\", \"coimmunoprecipitation\", \"coincidental\", \"coli\", \"coli\", \"colocalize\", \"colocalized\", \"colocalized\", \"combating\", \"common\", \"common\", \"compete\", \"complementing\", \"complete\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"complex\", \"component\", \"component\", \"component\", \"component\", \"component\", \"component\", \"component\", \"component\", \"component\", \"component\", \"component\", \"computational\", \"concentrated\", \"conclude\", \"conclusion\", \"concurrently\", \"condensate\", \"condensate\", \"condition\", \"condition\", \"condition\", \"condition\", \"condition\", \"condition\", \"condition\", \"condition\", \"confirm\", \"conformation\", \"conformation\", \"conformation\", \"conformation\", \"confused\", \"conservation\", \"consideration\", \"continuum\", \"contrary\", \"contribute\", \"controversial\", \"core\", \"core\", \"core\", \"core\", \"corf\", \"corf\", \"corf\", \"corf\", \"coronavirus\", \"coronavirus\", \"coronavirus\", \"coronavirus\", \"coronavirus\", \"coronavirus\", \"coronavirus\", \"correlation\", \"corresponding\", \"correspondingly\", \"corroborated\", \"coupled\", \"criterion\", \"criterion\", \"critical\", \"critical\", \"critical\", \"critical\", \"critical\", \"critical\", \"critically\", \"crna\", \"crowding\", \"ct\", \"currently\", \"currently\", \"cvb3\", \"cyclin\", \"cytopathic\", \"cytoskeleton\", \"cytotoxic\", \"d\", \"d\", \"d\", \"damage\", \"damage\", \"damage\", \"damage\", \"damage\", \"damage\", \"damage\", \"damage\", \"damage\", \"damage\", \"damage\", \"damaging\", \"date\", \"daunting\", \"dead\", \"dearth\", \"decouples\", \"decoy\", \"decoy\", \"defective\", \"defined\", \"defined\", \"defined\", \"defines\", \"definition\", \"definition\", \"definition\", \"degradation\", \"degradation\", \"degradation\", \"degradation\", \"degradation\", \"degradation\", \"degradation\", \"degradation\", \"degradation\", \"degradation\", \"degradation\", \"degrades\", \"deletion\", \"deletion\", \"demonstrate\", \"demonstrate\", \"demonstrate\", \"demonstrate\", \"demonstrate\", \"denatured\", \"denaturing\", \"denv\", \"denv\", \"denv\", \"dependent\", \"dependent\", \"dependent\", \"dependent\", \"dependent\", \"dependent\", \"dependent\", \"dependent\", \"dependent\", \"dependent\", \"depletion\", \"depletion\", \"depletion\", \"depletion\", \"depletion\", \"depletion\", \"deriva\", \"describe\", \"describe\", \"described\", \"described\", \"description\", \"descriptor\", \"despite\", \"detect\", \"detect\", \"detect\", \"determinant\", \"determine\", \"determine\", \"determine\", \"determine\", \"development\", \"development\", \"development\", \"development\", \"development\", \"diagonally\", \"dichroism\", \"difference\", \"difference\", \"difference\", \"different\", \"different\", \"different\", \"different\", \"different\", \"different\", \"different\", \"different\", \"different\", \"different\", \"different\", \"different\", \"differentially\", \"dihydro\", \"directionality\", \"disappeared\", \"discovery\", \"discovery\", \"discussed\", \"disorder\", \"disorder\", \"disorder\", \"disorder\", \"disorder\", \"disorder\", \"displayed\", \"disruption\", \"disruption\", \"disruption\", \"dissociates\", \"distinct\", \"distinct\", \"distinct\", \"distinct\", \"distinct\", \"distinguish\", \"distinguished\", \"diversity\", \"dm\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dna\", \"dogma\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"domain\", \"dot\", \"dot\", \"double\", \"double\", \"double\", \"double\", \"double\", \"double\", \"double\", \"downregulation\", \"downstream\", \"dramatically\", \"drbm\", \"drbm\", \"drbm\", \"drbm\", \"drbm\", \"driving\", \"drought\", \"dsdna\", \"dsdna\", \"dsdna\", \"dtt\", \"duplodnaviria\", \"duplodnaviria\", \"dynamic\", \"dynamic\", \"dynamic\", \"dynamic\", \"dynamic\", \"e\", \"e\", \"e\", \"e\", \"e\", \"e\", \"e\", \"e\", \"e\", \"e\", \"earlier\", \"earliest\", \"early\", \"early\", \"early\", \"early\", \"early\", \"early\", \"early\", \"early\", \"early\", \"early\", \"easily\", \"ef\", \"effect\", \"effect\", \"effect\", \"effect\", \"effect\", \"effect\", \"effect\", \"effect\", \"effect\", \"effectively\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif2a\", \"eif3a\", \"either\", \"either\", \"either\", \"either\", \"either\", \"either\", \"either\", \"either\", \"either\", \"elevates\", \"elongating\", \"embryo\", \"encapsu\", \"encapsulates\", \"encoded\", \"end\", \"end\", \"end\", \"end\", \"end\", \"end\", \"end\", \"endogenous\", \"endogenous\", \"endogenous\", \"endogenous\", \"endogenous\", \"endogenous\", \"endogenous\", \"endoplasmic\", \"endoplasmic\", \"endoribonuclease\", \"endothelial\", \"enhances\", \"enhances\", \"enhances\", \"enveloped\", \"enveloped\", \"enveloped\", \"enveloped\", \"envelopedcapsidhelicalcapsid\", \"envelopedcapsidhelicalcapsid\", \"envelopedcapsidhelicalcapsid\", \"envelopedcapsidhelicalcapsid\", \"environmental\", \"environmental\", \"environmental\", \"environmental\", \"environmental\", \"enylated\", \"epidemic\", \"equilibrium\", \"essential\", \"essential\", \"essential\", \"essential\", \"essential\", \"establishing\", \"eumetazoa\", \"evasion\", \"event\", \"event\", \"evidence\", \"evidence\", \"evidence\", \"evidence\", \"evidence\", \"evolutionarily\", \"exact\", \"examination\", \"examine\", \"exception\", \"exclusive\", \"exclusively\", \"executive\", \"exogenous\", \"exonucleolytic\", \"expansion\", \"expansion\", \"expansion\", \"expected\", \"expected\", \"explained\", \"exploration\", \"exposed\", \"exposed\", \"expressed\", \"expressed\", \"expressed\", \"expressed\", \"expressed\", \"expression\", \"expression\", \"expression\", \"expression\", \"expression\", \"expression\", \"expression\", \"expression\", \"expression\", \"expression\", \"expression\", \"expression\", \"extracellular\", \"extracellular\", \"exudes\", \"facilitated\", \"fact\", \"fact\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"factor\", \"fails\", \"fairly\", \"family\", \"family\", \"family\", \"family\", \"family\", \"family\", \"family\", \"family\", \"family\", \"family\", \"family\", \"family\", \"feasible\", \"fibroblast\", \"fibroblast\", \"fide\", \"fide\", \"fide\", \"fide\", \"fide\", \"fide\", \"fide\", \"field\", \"field\", \"field\", \"fifth\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"fig\", \"filled\", \"five\", \"flag\", \"flag\", \"flag\", \"flag\", \"flaviviridae\", \"fluidity\", \"fmr\", \"fmr\", \"fmr\", \"fmrp\", \"fmrp\", \"fo\", \"fold\", \"fold\", \"fold\", \"fold\", \"fold\", \"fold\", \"fold\", \"fold\", \"fold\", \"fold\", \"fold\", \"force\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"form\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"formation\", \"forming\", \"forming\", \"forming\", \"forming\", \"forming\", \"forming\", \"fortunately\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"found\", \"four\", \"four\", \"four\", \"four\", \"four\", \"four\", \"four\", \"four\", \"fraction\", \"fraction\", \"fraction\", \"fraction\", \"fragile\", \"fragile\", \"fragile\", \"fragmented\", \"frequency\", \"ftd\", \"full\", \"full\", \"full\", \"full\", \"full\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"function\", \"fundamental\", \"fuselloviridae\", \"fusion\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g\", \"g3bp\", \"g3bp\", \"g3bp\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g3bp1\", \"g4s\", \"g4s\", \"g4s\", \"g4s\", \"g4s\", \"g4s\", \"g4s\", \"g4s\", \"g4s\", \"gadd45\", \"gapdh\", \"gapdh\", \"gapdh\", \"gapdh\", \"gated\", \"gc\", \"gc\", \"gc\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"gene\", \"generated\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genome\", \"genus\", \"genus\", \"genus\", \"genus\", \"genus\", \"genus\", \"geographical\", \"gfp\", \"ggggcc\", \"ggggcc\", \"ggggcc\", \"give\", \"globin\", \"globular\", \"globular\", \"globule\", \"globuloviridae\", \"glyceraldehyde\", \"glycine\", \"going\", \"granule\", \"granule\", \"granule\", \"granule\", \"granule\", \"granule\", \"groundwork\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"group\", \"grouped\", \"grouping\", \"growing\", \"growth\", \"growth\", \"growth\", \"growth\", \"growth\", \"growth\", \"growth\", \"growth\", \"growth\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guanine\", \"guttaviridae\", \"h\", \"h\", \"h\", \"h\", \"h\", \"hairpin\", \"halspiviridae\", \"harbour\", \"harmful\", \"healing\", \"healthy\", \"hek\", \"helper\", \"helper\", \"helper\", \"helper\", \"heme\", \"hence\", \"hence\", \"hepatitis\", \"hepatitis\", \"hepatitis\", \"hepatitis\", \"herpesviridae\", \"heterochromatin\", \"heterodimer\", \"heterodimeric\", \"heterodimerization\", \"hierarchy\", \"high\", \"high\", \"high\", \"high\", \"high\", \"high\", \"high\", \"hold\", \"hoogsteen\", \"hoogsteen\", \"hoogsteen\", \"hop\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"host\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"however\", \"huge\", \"human\", \"human\", \"human\", \"human\", \"human\", \"human\", \"human\", \"human\", \"human\", \"human\", \"human\", \"hunov\", \"hy\", \"hybridize\", \"hydration\", \"hydrogen\", \"hydrogen\", \"hydrolase\", \"hydroxyl\", \"hyper\", \"hypervariable\", \"hypo\", \"hypoxia\", \"icosahedral\", \"icosahedral\", \"icosahedral\", \"icosahedral\", \"icosahedral\", \"icp27\", \"icp27\", \"icp27\", \"icp27\", \"icp27\", \"ictv\", \"ictv\", \"ictv\", \"ictv\", \"ictv\", \"ictv\", \"id\", \"id\", \"id\", \"identify\", \"idr\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifn\", \"ifna\", \"ii\", \"ii\", \"ii\", \"ii\", \"ii\", \"ikb\", \"ikk\", \"impairs\", \"impairs\", \"implicated\", \"incapable\", \"incertae\", \"incertae\", \"incertae\", \"incertae\", \"incertae\", \"incidence\", \"including\", \"including\", \"including\", \"including\", \"including\", \"including\", \"including\", \"including\", \"including\", \"including\", \"including\", \"inconsistent\", \"incorporation\", \"increase\", \"increase\", \"increase\", \"increase\", \"increase\", \"increase\", \"increased\", \"increased\", \"increased\", \"increased\", \"increased\", \"increasingly\", \"increasingly\", \"increasingly\", \"indicate\", \"indicate\", \"indicate\", \"indicate\", \"indicate\", \"indicated\", \"indicated\", \"indicated\", \"indicated\", \"indicated\", \"indirectly\", \"individually\", \"induce\", \"induce\", \"induce\", \"induce\", \"induce\", \"induce\", \"induce\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"induced\", \"inducer\", \"inducer\", \"induces\", \"induces\", \"induces\", \"induces\", \"induces\", \"induces\", \"induces\", \"induces\", \"induces\", \"induces\", \"inefficient\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infected\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"infection\", \"influenced\", \"influx\", \"inhibitor\", \"inhibitor\", \"inhibitor\", \"initiation\", \"initiation\", \"initiation\", \"initiation\", \"initiation\", \"initiation\", \"initiation\", \"initiation\", \"initiation\", \"initiation\", \"innate\", \"innate\", \"innate\", \"innate\", \"innate\", \"insight\", \"insight\", \"instance\", \"instance\", \"intact\", \"intensity\", \"intensity\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interaction\", \"interactome\", \"interactome\", \"interactome\", \"interactome\", \"interactors\", \"interacts\", \"interacts\", \"interacts\", \"interacts\", \"interacts\", \"interacts\", \"interchangeable\", \"intermedi\", \"internal\", \"internal\", \"international\", \"international\", \"interstrand\", \"interval\", \"intracellular\", \"intracellular\", \"intramolecular\", \"intramolecular\", \"intramolecular\", \"intramolecular\", \"intramolecular\", \"intramolecular\", \"intramolecular\", \"intriguingly\", \"intrin\", \"intrinsically\", \"invading\", \"invading\", \"investi\", \"investigate\", \"involved\", \"involved\", \"involved\", \"involved\", \"involved\", \"involved\", \"involved\", \"involved\", \"involved\", \"involved\", \"ionizing\", \"ire\", \"ire\", \"irradiation\", \"ischemia\", \"isgs\", \"isgs\", \"isgs\", \"ivof\", \"ivof\", \"ivof\", \"ivof\", \"j\", \"japanese\", \"jev\", \"joining\", \"july\", \"jxwn06\", \"kd\", \"kd\", \"kd\", \"kd\", \"kd\", \"keep\", \"kinase\", \"kinase\", \"kinase\", \"kinase\", \"kinase\", \"kinase\", \"kinase\", \"kinase\", \"kinase\", \"knockdown\", \"knockdown\", \"knockdown\", \"knockdown\", \"knockdown\", \"knockdown\", \"knockdown\", \"knockdown\", \"knockdown\", \"knowledge\", \"known\", \"known\", \"known\", \"known\", \"known\", \"known\", \"known\", \"known\", \"known\", \"known\", \"known\", \"la\", \"lane\", \"lane\", \"lane\", \"lapping\", \"last\", \"lastly\", \"late\", \"late\", \"late\", \"late\", \"late\", \"late\", \"late\", \"late\", \"late\", \"late\", \"late\", \"layer\", \"layer\", \"learn\", \"level\", \"level\", \"level\", \"level\", \"level\", \"level\", \"level\", \"level\", \"level\", \"level\", \"level\", \"level\", \"level\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligand\", \"ligase\", \"like\", \"like\", \"like\", \"like\", \"like\", \"like\", \"like\", \"like\", \"like\", \"like\", \"like\", \"like\", \"limitation\", \"limited\", \"linear\", \"linear\", \"linear\", \"localization\", \"loop\", \"loop\", \"loop\", \"loop\", \"loop\", \"loop\", \"loop\", \"loop\", \"loop\", \"lower\", \"lower\", \"lower\", \"lower\", \"lowest\", \"maintained\", \"majority\", \"manipulates\", \"manual\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"many\", \"marc\", \"marc\", \"march\", \"marker\", \"marker\", \"marker\", \"marker\", \"marker\", \"marker\", \"marking\", \"mation\", \"maturation\", \"may\", \"may\", \"may\", \"may\", \"may\", \"may\", \"may\", \"may\", \"may\", \"may\", \"may\", \"may\", \"mean\", \"mean\", \"meaningful\", \"measured\", \"measured\", \"measurement\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanism\", \"mechanistic\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediated\", \"mediates\", \"medical\", \"metal\", \"metal\", \"microbe\", \"microtubule\", \"microtubule\", \"microtubule\", \"microtubule\", \"mismatching\", \"mitogenic\", \"mnv\", \"mnv\", \"mnv\", \"mnv\", \"mnv\", \"model\", \"model\", \"model\", \"model\", \"model\", \"modified\", \"modified\", \"modified\", \"modified\", \"modu\", \"modulate\", \"modulate\", \"modulate\", \"modulate\", \"modulated\", \"modulated\", \"modulated\", \"modulating\", \"molecular\", \"molecular\", \"molecular\", \"molecular\", \"molecular\", \"molecule\", \"molecule\", \"molecule\", \"molecule\", \"molecule\", \"molecule\", \"molecule\", \"molecule\", \"molten\", \"monitored\", \"mosaic\", \"mosaic\", \"mosquito\", \"motif\", \"motif\", \"motif\", \"motif\", \"motif\", \"motif\", \"motif\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mrna\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutant\", \"mutation\", \"mutation\", \"mutation\", \"mutation\", \"mutation\", \"n\", \"n\", \"n\", \"n\", \"n\", \"n\", \"n\", \"n\", \"n\", \"n\", \"n\", \"n_g\", \"n_g\", \"naked\", \"naked\", \"naked\", \"nakedcapsidicosahedralcapsid\", \"nakedcapsidicosahedralcapsid\", \"name\", \"name\", \"name\", \"nanopore\", \"native\", \"native\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"ndv\", \"near\", \"near\", \"near\", \"near\", \"necessarily\", \"necessary\", \"necessary\", \"necessary\", \"necessary\", \"need\", \"need\", \"needed\", \"nega\", \"neither\", \"neither\", \"ner\", \"network\", \"network\", \"neurological\", \"neurological\", \"neurological\", \"neurological\", \"neurological\", \"neuron\", \"neuron\", \"neutralization\", \"neutralizing\", \"new\", \"new\", \"new\", \"new\", \"new\", \"new\", \"new\", \"next\", \"nf\", \"nificant\", \"nipulating\", \"nizes\", \"noc\", \"nocodazole\", \"node\", \"non\", \"non\", \"non\", \"non\", \"non\", \"non\", \"non\", \"non\", \"non\", \"non\", \"non\", \"nonderepressible\", \"nonrandom\", \"nonsegmented\", \"norovirus\", \"norovirus\", \"norovirus\", \"note\", \"noticeably\", \"notion\", \"notion\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"np\", \"ns6pro\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp1\\u00df\", \"nsp3\", \"nsp3\", \"nsp3\", \"nsp3\", \"ntf2l\", \"ntf2l\", \"nuclease\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleic\", \"nucleolar\", \"nucleolin\", \"nucleotide\", \"nucleotide\", \"nucleotide\", \"nucleotide\", \"nucleus\", \"occupancy\", \"occurs\", \"ogg\", \"ogg\", \"ogg\", \"old\", \"old\", \"old\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"one\", \"ongoing\", \"open\", \"open\", \"open\", \"open\", \"opposed\", \"optimal\", \"origin\", \"original\", \"overexpression\", \"overexpression\", \"overexpression\", \"overexpression\", \"overhang\", \"oxidative\", \"oxidative\", \"oxidative\", \"oxidative\", \"oxidative\", \"oxidative\", \"oxidative\", \"oxidative\", \"oxidative\", \"oxidized\", \"oxidized\", \"oxidized\", \"oxoguanine\", \"oxoguanine\", \"p\", \"p\", \"p\", \"p\", \"pabp\", \"pabp\", \"pabp\", \"papovaviridaeare\", \"paramyxoviridae\", \"paramyxoviridae\", \"parental\", \"partial\", \"participate\", \"participation\", \"pathogen\", \"pathogen\", \"pathogen\", \"pathogen\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pathway\", \"pdb\", \"pdb\", \"pdb\", \"persist\", \"phaginae\", \"phospho\", \"physicochemical\", \"picornaviridae\", \"picornaviridae\", \"picornaviridae\", \"picornavirus\", \"picornavirus\", \"picornavirus\", \"piperidine\", \"piperidine\", \"pivotal\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"pkr\", \"plasm\", \"plastic\", \"platform\", \"platform\", \"platform\", \"platform\", \"platform\", \"played\", \"poly\", \"poly\", \"poly\", \"poly\", \"polyadenylated\", \"polyadenylated\", \"polyadenylated\", \"polyadenylated\", \"polyadenylated\", \"polyadenylated\", \"polyinosinic\", \"polymorphic\", \"polyribo\", \"polysomes\", \"polysomes\", \"polysomes\", \"polysomes\", \"polysomes\", \"polysomes\", \"polysomes\", \"posi\", \"positive\", \"positive\", \"positive\", \"positive\", \"positive\", \"positive\", \"positive\", \"positive\", \"positive\", \"post\", \"post\", \"post\", \"post\", \"post\", \"post\", \"potential\", \"potential\", \"potential\", \"potential\", \"potential\", \"potential\", \"potential\", \"potential\", \"potentiates\", \"pothesis\", \"powerful\", \"prediction\", \"prediction\", \"predominant\", \"predominant\", \"predominately\", \"preliminary\", \"presynaptic\", \"pretreatment\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"process\", \"processed\", \"prod\", \"produce\", \"product\", \"product\", \"product\", \"product\", \"product\", \"product\", \"product\", \"profile\", \"profile\", \"profile\", \"program\", \"prohibited\", \"prominent\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"promoter\", \"proteasome\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"protein\", \"proteomic\", \"provided\", \"provided\", \"provides\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"prrsv\", \"pseudo\", \"pur\", \"purpose\", \"pursuit\", \"pv\", \"q325\", \"qpcr\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplex\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"quadruplexes\", \"qualifies\", \"r\", \"r\", \"r\", \"r\", \"r\", \"ra\", \"ra\", \"ra\", \"ra\", \"radiation\", \"radical\", \"radioactive\", \"randomly\", \"rapidly\", \"rapidly\", \"rapidly\", \"rapidly\", \"ratio\", \"rationale\", \"rc\", \"react\", \"reactivity\", \"reagent\", \"realm\", \"realm\", \"realm\", \"realm\", \"realm\", \"realm\", \"recent\", \"recent\", \"recent\", \"recent\", \"recent\", \"recent\", \"recent\", \"recent\", \"recog\", \"recognition\", \"recognition\", \"recognize\", \"recognizing\", \"recovered\", \"recycling\", \"reduced\", \"reduced\", \"reduced\", \"reduced\", \"reduced\", \"reduced\", \"reduced\", \"reduced\", \"reduced\", \"reduces\", \"reduces\", \"reduces\", \"reduces\", \"reduces\", \"redundant\", \"reference\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"region\", \"regulated\", \"regulated\", \"regulated\", \"regulated\", \"regulated\", \"regulated\", \"regulates\", \"regulates\", \"regulates\", \"regulates\", \"regulates\", \"regulator\", \"regulator\", \"reiterates\", \"rejected\", \"rejected\", \"rejected\", \"rejected\", \"relationship\", \"relationship\", \"relationship\", \"reliable\", \"rely\", \"remodel\", \"remodels\", \"remove\", \"remove\", \"repair\", \"repair\", \"repair\", \"repair\", \"repair\", \"repair\", \"repair\", \"repairing\", \"repeat\", \"repeat\", \"repeat\", \"repeat\", \"repeat\", \"repeat\", \"repeat\", \"repeat\", \"repeat\", \"repeat\", \"replicates\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"replication\", \"reported\", \"reported\", \"reported\", \"reported\", \"reported\", \"reported\", \"reported\", \"reported\", \"representative\", \"representing\", \"repress\", \"repur\", \"repurposing\", \"rescued\", \"research\", \"research\", \"research\", \"research\", \"research\", \"research\", \"research\", \"research\", \"research\", \"reserved\", \"residue\", \"residue\", \"residue\", \"residue\", \"residue\", \"residue\", \"respectively\", \"respectively\", \"respectively\", \"respectively\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"response\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"result\", \"retrieved\", \"reverse\", \"reverse\", \"reverse\", \"review\", \"reviewed\", \"rfp\", \"ribosome\", \"ribosome\", \"ribosome\", \"ribosome\", \"ribosome\", \"ribosome\", \"ribosome\", \"rich\", \"rich\", \"rich\", \"rich\", \"rich\", \"rich\", \"rich\", \"rich\", \"rich\", \"rig\", \"rig\", \"rig\", \"rig\", \"right\", \"ring\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rna\", \"rnai\", \"role\", \"role\", \"role\", \"role\", \"role\", \"role\", \"role\", \"role\", \"role\", \"role\", \"role\", \"role\", \"role\", \"rossman\", \"rotate\", \"run\", \"run\", \"run\", \"run\", \"run\", \"run\", \"rylated\", \"s\", \"s\", \"s\", \"s\", \"s\", \"s\", \"s\", \"s51a\", \"s51d\", \"sars\", \"sars\", \"sars\", \"satellite\", \"satellite\", \"satellite\", \"satellite\", \"scale\", \"scram\", \"section\", \"section\", \"sedis\", \"sedis\", \"sedis\", \"sedis\", \"sedis\", \"seem\", \"seem\", \"seemingly\", \"seems\", \"seen\", \"seen\", \"selective\", \"selective\", \"selective\", \"self\", \"sembly\", \"sense\", \"sense\", \"sense\", \"sense\", \"sense\", \"sense\", \"sense\", \"sense\", \"sense\", \"sensing\", \"separated\", \"separated\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequence\", \"sequencing\", \"sequencing\", \"sequencing\", \"sequencing\", \"sequencing\", \"sequencing\", \"sequestration\", \"sequestration\", \"sequestration\", \"serve\", \"served\", \"set\", \"settled\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"sg\", \"shape\", \"shape\", \"short\", \"short\", \"short\", \"short\", \"showed\", \"showed\", \"showed\", \"showed\", \"showed\", \"showed\", \"showed\", \"showed\", \"showed\", \"showed\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shown\", \"shuttling\", \"sig\", \"sig3bp1\", \"sig3bp1\", \"signaling\", \"signaling\", \"signaling\", \"signaling\", \"signaling\", \"signaling\", \"signaling\", \"signaling\", \"significance\", \"significance\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similar\", \"similarly\", \"similarly\", \"similarly\", \"similarly\", \"sing\", \"single\", \"single\", \"single\", \"single\", \"single\", \"single\", \"single\", \"single\", \"single\", \"single\", \"single\", \"sirnas\", \"sirnas\", \"sirnas\", \"site\", \"site\", \"site\", \"site\", \"site\", \"site\", \"site\", \"site\", \"site\", \"site\", \"site\", \"sitia\", \"sitia\", \"situated\", \"sjincluding\", \"small\", \"small\", \"small\", \"small\", \"small\", \"small\", \"small\", \"small\", \"small\", \"small\", \"small\", \"smaller\", \"smaller\", \"smaller\", \"sneaky\", \"solution\", \"somewhat\", \"soreson\", \"spared\", \"specie\", \"specie\", \"specie\", \"specie\", \"specie\", \"specie\", \"specie\", \"specie\", \"specie\", \"specific\", \"specific\", \"specific\", \"specific\", \"specific\", \"specific\", \"specific\", \"specific\", \"specific\", \"specific\", \"specific\", \"specifically\", \"specifically\", \"specifically\", \"specifically\", \"spindle\", \"sponse\", \"ssnucleic\", \"ssnucleic\", \"ssnucleic\", \"ssnucleic\", \"ssrna\", \"ssrna\", \"ssrna\", \"stabilization\", \"stabilization\", \"stabilization\", \"stabilizes\", \"stabilizes\", \"stable\", \"stable\", \"stable\", \"stable\", \"stable\", \"stable\", \"stable\", \"stable\", \"stable\", \"stable\", \"stack\", \"stack\", \"stage\", \"stage\", \"stage\", \"stage\", \"stage\", \"stage\", \"stage\", \"stage\", \"stage\", \"stage\", \"stage\", \"stained\", \"stall\", \"state\", \"state\", \"state\", \"state\", \"state\", \"state\", \"state\", \"state\", \"statistical\", \"sterically\", \"sthe\", \"still\", \"still\", \"still\", \"still\", \"still\", \"still\", \"still\", \"stoichiometry\", \"stone\", \"story\", \"strand\", \"strand\", \"strand\", \"strand\", \"strand\", \"strand\", \"strand\", \"strand\", \"strand\", \"strand\", \"strand\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"stranded\", \"strategically\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"stress\", \"strongly\", \"structural\", \"structural\", \"structural\", \"structural\", \"structural\", \"structural\", \"structural\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structure\", \"structured\", \"structured\", \"structured\", \"structured\", \"student\", \"study\", \"study\", \"study\", \"study\", \"study\", \"study\", \"study\", \"study\", \"study\", \"study\", \"study\", \"study\", \"subdivided\", \"subgenus\", \"suborder\", \"subsequent\", \"subsequent\", \"substrate\", \"subunit\", \"subunit\", \"subunit\", \"subunit\", \"subunit\", \"subversion\", \"sucrose\", \"sudden\", \"sufficient\", \"suggests\", \"suggests\", \"suitable\", \"summarize\", \"suppressed\", \"sure\", \"surprise\", \"survival\", \"survival\", \"survival\", \"survival\", \"symmetry\", \"symmetry\", \"symmetry\", \"symmetry\", \"symmetry\", \"symmetry\", \"symmetry\", \"symmetry\", \"synergistically\", \"synonymous\", \"synthase\", \"synthesis\", \"synthesis\", \"synthesis\", \"synthesis\", \"synthesis\", \"synthesis\", \"synthesis\", \"synthesis\", \"synthesis\", \"synthesis\", \"system\", \"system\", \"system\", \"system\", \"system\", \"system\", \"system\", \"tailed\", \"taken\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"target\", \"targeted\", \"taught\", \"tautomeric\", \"taxonomy\", \"taxonomy\", \"telomerase\", \"telomerase\", \"telomerase\", \"telomerase\", \"telomeric\", \"telomeric\", \"telomeric\", \"telomeric\", \"telomeric\", \"telomeric\", \"telomeric\", \"ten\", \"tend\", \"terminal\", \"terminal\", \"terminal\", \"terminal\", \"terminal\", \"terminal\", \"terminal\", \"ternary\", \"tested\", \"tested\", \"tetrad\", \"tetrad\", \"tetrad\", \"tetrad\", \"tetrad\", \"tetramolecular\", \"tetramolecular\", \"thermostability\", \"though\", \"though\", \"though\", \"though\", \"threshold\", \"throughput\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tia\", \"tiar\", \"tiar\", \"tiar\", \"tiar\", \"tiar\", \"tiar\", \"tiar\", \"tiar\", \"tiar\", \"tiar\", \"tiar\", \"ticulum\", \"time\", \"time\", \"time\", \"time\", \"time\", \"time\", \"time\", \"time\", \"time\", \"time\", \"tissue\", \"titer\", \"titer\", \"titer\", \"titer\", \"tmpyp_\", \"tobacco\", \"togaviridae\", \"top\", \"top\", \"top\", \"total\", \"toxin\", \"toxin\", \"toxin\", \"trans\", \"trans\", \"transcript\", \"transcript\", \"transcript\", \"transcript\", \"transcript\", \"transcriptase\", \"transcriptase\", \"transcriptase\", \"transcription\", \"transcription\", \"transcription\", \"transcription\", \"transcription\", \"transcription\", \"transcription\", \"transcription\", \"transfected\", \"transfected\", \"transfected\", \"transfected\", \"transfected\", \"transfecting\", \"transfection\", \"transfection\", \"transfection\", \"transla\", \"translating\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"translation\", \"transport\", \"treat\", \"treat\", \"treated\", \"treated\", \"treated\", \"treated\", \"treatment\", \"treatment\", \"treatment\", \"treatment\", \"treatment\", \"treatment\", \"treatment\", \"tristetraprolin\", \"ttp\", \"ttp\", \"ttp\", \"ttp\", \"ttp\", \"tuber\", \"tumor\", \"tumor\", \"tumor\", \"tumor\", \"turn\", \"turn\", \"tween\", \"two\", \"two\", \"two\", \"two\", \"two\", \"two\", \"two\", \"two\", \"two\", \"two\", \"two\", \"two\", \"two\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"type\", \"typical\", \"typical\", \"typical\", \"tyrosine\", \"tyrosine\", \"ubiquitylation\", \"uct\", \"unaffected\", \"unaffected\", \"unassembled\", \"unavailable\", \"uncapped\", \"underway\", \"unexpectedly\", \"unexplored\", \"universal\", \"unnamed\", \"unnamed\", \"unpublished\", \"unusual\", \"updated\", \"updated\", \"upregulation\", \"upstream\", \"used\", \"used\", \"used\", \"used\", \"used\", \"used\", \"used\", \"used\", \"used\", \"used\", \"used\", \"used\", \"used\", \"utilized\", \"uv\", \"uv\", \"vaccinia\", \"value\", \"varidnaviria\", \"varidnaviria\", \"variety\", \"variety\", \"variety\", \"vascular\", \"veev\", \"verify\", \"verify\", \"verify\", \"vessel\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"vhs\", \"viable\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viral\", \"viridae\", \"virinae\", \"virineae\", \"virion\", \"virion\", \"virion\", \"virion\", \"virion\", \"virion\", \"virion\", \"virion\", \"virion\", \"virion\", \"viroid\", \"virtually\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virus\", \"virushas\", \"virushas\", \"virushas\", \"virushas\", \"vof\", \"vof\", \"vof\", \"warrant\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"well\", \"whether\", \"whether\", \"whether\", \"whether\", \"whether\", \"whether\", \"whether\", \"whether\", \"whether\", \"wide\", \"wide\", \"wide\", \"wide\", \"wild\", \"wild\", \"wild\", \"wild\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"within\", \"world\", \"world\", \"world\", \"world\", \"world\", \"worth\", \"would\", \"would\", \"would\", \"would\", \"would\", \"wrapped\", \"wrn\", \"year\", \"year\", \"year\", \"year\", \"yeast\", \"zikv\", \"zikv\", \"zikv\", \"\\u00df\", \"\\u00df\", \"\\u00df\", \"\\u00df\", \"\\u00df\", \"\\u03b2c1\"]}, \"R\": 30, \"lambda.step\": 0.01, \"plot.opts\": {\"xlab\": \"PC1\", \"ylab\": \"PC2\"}, \"topic.order\": [8, 10, 13, 19, 9, 16, 2, 7, 17, 20, 14, 1, 18, 15, 6, 5, 12, 3, 4, 11]};\n",
              "\n",
              "function LDAvis_load_lib(url, callback){\n",
              "  var s = document.createElement('script');\n",
              "  s.src = url;\n",
              "  s.async = true;\n",
              "  s.onreadystatechange = s.onload = callback;\n",
              "  s.onerror = function(){console.warn(\"failed to load library \" + url);};\n",
              "  document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
              "}\n",
              "\n",
              "if(typeof(LDAvis) !== \"undefined\"){\n",
              "   // already loaded: just create the visualization\n",
              "   !function(LDAvis){\n",
              "       new LDAvis(\"#\" + \"ldavis_el94351398835099700003693728938\", ldavis_el94351398835099700003693728938_data);\n",
              "   }(LDAvis);\n",
              "}else if(typeof define === \"function\" && define.amd){\n",
              "   // require.js is available: use it to load d3/LDAvis\n",
              "   require.config({paths: {d3: \"https://d3js.org/d3.v5\"}});\n",
              "   require([\"d3\"], function(d3){\n",
              "      window.d3 = d3;\n",
              "      LDAvis_load_lib(\"https://cdn.jsdelivr.net/gh/bmabey/pyLDAvis@3.4.0/pyLDAvis/js/ldavis.v3.0.0.js\", function(){\n",
              "        new LDAvis(\"#\" + \"ldavis_el94351398835099700003693728938\", ldavis_el94351398835099700003693728938_data);\n",
              "      });\n",
              "    });\n",
              "}else{\n",
              "    // require.js not available: dynamically load d3 & LDAvis\n",
              "    LDAvis_load_lib(\"https://d3js.org/d3.v5.js\", function(){\n",
              "         LDAvis_load_lib(\"https://cdn.jsdelivr.net/gh/bmabey/pyLDAvis@3.4.0/pyLDAvis/js/ldavis.v3.0.0.js\", function(){\n",
              "                 new LDAvis(\"#\" + \"ldavis_el94351398835099700003693728938\", ldavis_el94351398835099700003693728938_data);\n",
              "            })\n",
              "         });\n",
              "}\n",
              "</script>"
            ]
          },
          "metadata": {},
          "execution_count": 4
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import drive\n",
        "from sklearn.feature_extraction.text import TfidfVectorizer\n",
        "from sklearn.metrics.pairwise import cosine_similarity\n",
        "\n",
        "# mount the drive\n",
        "drive.mount('/content/gdrive')\n",
        "\n",
        "# open your corpus file from drive and load the content\n",
        "with open('/content/CollatedText.txt', 'r') as file:\n",
        "    corpus = [line.strip() for line in file]\n",
        "\n",
        "# This vectorizer will convert our documents into a matrix of TF-IDF features\n",
        "vectorizer = TfidfVectorizer()\n",
        "\n",
        "# Vectorize our documents\n",
        "tfidf_matrix = vectorizer.fit_transform(corpus)\n",
        "\n",
        "def search(query):\n",
        "    # Vectorize the query to the same feature space as our documents\n",
        "    query_vec = vectorizer.transform([query])\n",
        "\n",
        "    # Compute the cosine similarity between the query and all documents\n",
        "    sims = cosine_similarity(query_vec, tfidf_matrix).flatten()\n",
        "\n",
        "    # Sort the document indices by their similarity score\n",
        "    ranking = sims.argsort()[::-1]\n",
        "\n",
        "    # Print out the ranking\n",
        "    for i in range(len(ranking)):\n",
        "        print(f\"Document #{ranking[i]}: {corpus[ranking[i]]} (score: {sims[ranking[i]]})\")\n",
        "\n",
        "# We try it out\n",
        "search(\"G4\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "vHr1fc5cKPe3",
        "outputId": "fcc561cf-5d5d-4a74-9599-15736cd54853"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/ipykernel/ipkernel.py:283: DeprecationWarning: `should_run_async` will not call `transform_cell` automatically in the future. Please pass the result to `transformed_cell` argument and any exception that happen during thetransform in `preprocessing_exc_tuple` in IPython 7.17 and above.\n",
            "  and should_run_async(code)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Drive already mounted at /content/gdrive; to attempt to forcibly remount, call drive.mount(\"/content/gdrive\", force_remount=True).\n",
            "Document #5: ‘In general, G4 ligands showed promising antiviral activity, with G4-mediated mechanisms of action both at the genome and transcript level.’ (score: 0.4870952927086342)\n",
            "Document #17: ‘However, mapping of G4s in chromatin by G4 ChIP-sequencing with an anti-G4 antibody (13) or foot printing (14) retrieved only about 10000 G4s in highly transcribed regulatory nucleosome-depleted chromatin regions.’ (score: 0.32413775388010746)\n",
            "Document #16: ‘These data have been corroborated by 'G4-seq' high-throughput sequencing method, which identified about 700000 G4s (12).’ (score: 0.23931220490734503)\n",
            "Document #7: ‘The current state of the art of G4 ligands in antiviral research is also reported, with particular focus on the structural and physicochemical requirements for optimal biological activity.’ (score: 0.2220857243385898)\n",
            "Document #4: ‘In this context, G4 ligands have been developed and tested both as tools to study the complexity of G4mediated mechanisms in the viral life cycle, and as therapeutic agents.’ (score: 0.2213675098955331)\n",
            "Document #19: ‘In general, G4 sequences are non-randomly distributed but mainly clustered in pivotal genomic regions, namely telomeres, gene promoters and DNA replication origins (15).’ (score: 0.2174132356558217)\n",
            "Document #20: ‘Moreover, putative G4-forming sequences have been found in coding and non-coding regions of the human transcriptome, i.e., open reading frames and untranslated regions.’ (score: 0.20774594333146165)\n",
            "Document #15: ‘Computational analysis using different algorithms (9,10) indicated that 300000 and up to around 3 000000 potential G4-forming sequences may form in the human genome, correlated with specific gene functions (11).’ (score: 0.20288995149360978)\n",
            "Document #296: ‘Similar to alphaviruses, G3BP1 in MNV-infected cells recruit ribosomes to viral RNA and initiate translation [31].’ (score: 0.0)\n",
            "Document #295: ‘In addition, knockdown of MNV-induced G3BP1 interactors reduces MNV replication, suggesting that the G3BP1 interactome consists of host factors that are required for efficient viral replication complex formation [84,107].’ (score: 0.0)\n",
            "Document #297: ‘During MNV infection, both PKR and GCN2 phosphorylated eIF2a; however, SG formation is not observed in the infected cells, even though they can still form SGs upon arsenite treatment, suggesting that MNV decouples the SG formation from eIF2a phosphorylation during virus infection.’ (score: 0.0)\n",
            "Document #294: ‘Consistently, genetic depletion of G3BP1 severely affects viral replication and translation.’ (score: 0.0)\n",
            "Document #292: remodels G3BP1 interactome in infected cells and induces cytoplasmic G3BP1 granules during infection that are distinct from SGs.’ (score: 0.0)\n",
            "Document #291: ‘MNV (score: 0.0)\n",
            "Document #290: ‘Caliciviruses Remodel the G3BP1 Interactome during Infection The Caliciviridae family comprises small non-enveloped positive-strand RNA viruses, including human norovirus (HuNoV) and the closely related murine norovirus (MNV).’ (score: 0.0)\n",
            "Document #289: ‘4.2.2.’ (score: 0.0)\n",
            "Document #288: both Old World and New World alphaviruses, G3BP1 plays a pro-viral role only for Old World alphaviruses.’ (score: 0.0)\n",
            "Document #287: ‘Instead, these viruses sequester other SG components, such as FXR1, FXR2, and (score: 0.0)\n",
            "Document #293: ‘The proteome of G3BP1 granules contains several proteins that are present in viral replication complexes, suggesting a pro-viral role of G3BP1 during MNV infection.’ (score: 0.0)\n",
            "Document #867:  (score: 0.0)\n",
            "Document #286: ‘Notably, the New World alphaviruses do not have the FGDF motif and thus do not bind G3BP1.’ (score: 0.0)\n",
            "Document #299: ‘4.3.’ (score: 0.0)\n",
            "Document #285: ‘Infection with the mutant virus results in delayed SG disassembly and reduced viral structural protein synthesis in neuronal cells, suggesting that G3BP1 ADP-ribosylation regulates virus production [47,105].’ (score: 0.0)\n",
            "Document #284: ‘G3BP1 ADP-ribosylation, a key driver of SG assembly, is reduced in cells infected by wild-type CHIKV, but not a mutant virus that lacks hydrolase activity.’ (score: 0.0)\n",
            "Document #283: ‘The macrodomain possesses ADP-ribosylhydrolase activity (the ability to remove ADP-ribose from conjugated proteins), and this enzymatic activity is critical for replication and virulence of viruses from several viral families [92,101–104].’ (score: 0.0)\n",
            "Document #282: ‘Recently, our group discovered that the SG disassembly for CHIKV is partly mediated by the macrodomain at the N-terminus of nsP3.’ (score: 0.0)\n",
            "Document #281: ‘However, SFV without FGDF motifs still disassembles SGs, but at a slower rate suggesting an additional mechanism for SG disassembly [77].’ (score: 0.0)\n",
            "Document #280: ‘The binding of nsP3 with G3BP1 is a proposed mechanism for SG disassembly where nsP3-G3BP1 binding inhibits the interaction between other SG components and G3BP1.’ (score: 0.0)\n",
            "Document #279: ‘For initiation of infection, G3BP1 facilitates CHIKV, but not to SFV, translation of genomic RNA by enriching translation factors at cytopathic vacuoles [100].’ (score: 0.0)\n",
            "Document #278: ‘In infected cells, G3BP1 is sequestered through interaction of the NTF2L domain with FGDF motifs in the C-terminal hypervariable region of non-structural protein 3 (nsP3).’ (score: 0.0)\n",
            "Document #277: ‘Cells lacking both G3BP1/G3BP2 do not support replication of the Old World alphaviruses but do support replication of the New World encephalitic alphaviruses, indicating a virus-specific importance of G3BP1 for infection.’ (score: 0.0)\n",
            "Document #276: ‘Notably, the alphavirus genus can be categorized into the Old World alphaviruses (CHIKV, SINV, SFV) and the New World alphaviruses (VEEV, EEEV) based on their geographical origin.’ (score: 0.0)\n",
            "Document #275: ‘Togaviruses Sequester G3BP1 to Facilitate Viral Replication and Translation Togaviridae is a family of enveloped, icosahedral viruses with a positive-sense single-stranded RNA genome including the mosquito-borne alphaviruses that cause rash and arthritis such as CHIKV, SFV, and SING.’ (score: 0.0)\n",
            "Document #298: ‘In contrast, feline calicivirus cleaves G3BP1 through the viral 3C-like proteinase NS6Pro [94].’ (score: 0.0)\n",
            "Document #304: ‘Dengue Virus (DENV) Modulates G3BP1 Function by Interacting with 30UTR vRNA DENV infection induces G3BP1 punctate-like structures at the early stage of infection.’ (score: 0.0)\n",
            "Document #300: ‘G3BP1 Is Differentially Utilized within the Flaviviridae Family Although G3BP1 influences viral propagation of Flaviviridae family, different members (score: 0.0)\n",
            "Document #301: utilize G3BP1 at different stages of infection.’ (score: 0.0)\n",
            "Document #324: ‘Intriguingly, one study suggested a pro-viral function of SG assembly during PRRSV infection.’ (score: 0.0)\n",
            "Document #323: ‘The reason for G3BP1 association with viral replication complexes and the mechanism and role of SG assembly at late infection stages are not known [85,86].’ (score: 0.0)\n",
            "Document #322: ‘In contrast to the above-described viruses, PRRSV induces SGs only at late stages of infection (48–72-h post-infection) through PERK-, rather than PKR-mediated eIF2a phosphorylation.’ (score: 0.0)\n",
            "Document #321: ‘Neither deletion nor overexpression of G3BP1, G3BP2, or both affects viral titers.’ (score: 0.0)\n",
            "Document #320: ‘Depletion or Overexpression of G3BP1 Does Not Affect PRRSV Infection PRRSV—a single-stranded positive-sense enveloped virus in the Arteriviridae family— does not require G3BP1, yet the protein is closely associated with viral replication complexes.’ (score: 0.0)\n",
            "Document #319: ‘4.4.1.’ (score: 0.0)\n",
            "Document #318: ‘Though G3BP1 is a predominant target for many viruses, there are some viruses whose replication is unaffected by the expression level of G3BP1.’ (score: 0.0)\n",
            "Document #317: ‘Interestingly, HCV requires only G3BP1, but not G3BP2 whose knockdown does not alter viral replication [74,75].’ (score: 0.0)\n",
            "Document #316: UTR of viral RNA and is involved in genome amplification [74].’ (score: 0.0)\n",
            "Document #315: ‘During HCV infection, G3BP1 interacts with the 50¬ (score: 0.0)\n",
            "Document #314: ‘In addition, in ZIKV-infected cells, G3BP1 interacts with capsid protein and co-localizes with envelope protein, suggesting a possible role for G3BP1 in virion assembly [70–72].’ (score: 0.0)\n",
            "Document #313: ‘G3BP1 depletion reduces viral replication, titer, and viral protein levels, while overexpression of G3BP1 enhances viral titers.’ (score: 0.0)\n",
            "Document #312: ‘ZIKV and HCV Require G3BP1 for Proper Viral Replication In both ZIKV-and HCV-infected cells, G3BP1 associates with the viral replication complex.’ (score: 0.0)\n",
            "Document #311: ‘4.3.2.’ (score: 0.0)\n",
            "Document #310: and whether a similar decoy phenomenon may be used by other viruses to counter the production of ISG proteins and promote viral replication is unknown.’ (score: 0.0)\n",
            "Document #309: ‘Not all flavivirus sfRNAs have this property (score: 0.0)\n",
            "Document #308: ‘As an IFN countermeasure during DENV-2 infection of Huh-7 cells, a long noncoding RNA from the subgenomic region of flavivirus RNA (sfRNA) produced by all flaviviruses binds G3BP1, G3BP2, and Caprin1, and thereby inhibits the translation of ISGs [21].’ (score: 0.0)\n",
            "Document #307: ‘In DENV-2-infected cells, G3BP1 is also associated with ISG mRNAs and facilitates their translation to inhibit viral replication.’ (score: 0.0)\n",
            "Document #306: ‘The expression of viral protein is reduced in cells containing G3BP1 granules, whereas knockdown of G3BP1 enhances protein level and viral titers.’ (score: 0.0)\n",
            "Document #305: ‘Although the phosphorylation of eIF2a by PKR occurs at the late infection stage [73], it is unclear whether infected cells induce SGs.’ (score: 0.0)\n",
            "Document #273: ‘In this section, we will discuss the different strategies employed by viruses that use G3BP1 to promote viral replication.’ (score: 0.0)\n",
            "Document #303: ‘4.3.1.’ (score: 0.0)\n",
            "Document #302: ‘In this section, we describe how the G3BP1 function is modulated by DENV, ZIKV, and HCV.’ (score: 0.0)\n",
            "Document #274: ‘4.2.1.’ (score: 0.0)\n",
            "Document #269: ‘Such pro-viral activities have been reported for several viruses including CHIKV, hepatitis C virus, and respiratory syncytial virus [23,74,99].’ (score: 0.0)\n",
            "Document #272: ‘The proposed pro-viral functions of G3BP1 include: stabilizing viral mRNAs to prevent degradation, facilitating viral protein translation by recruiting translation factors and ribosomes, acting as a scaffold to build viral replication complexes, and amplifying the viral genome.’ (score: 0.0)\n",
            "Document #271: ‘Binding of the viral genome by G3BP1 and sequestering it into viral replication complexes may be important to prevent G3BP1-mediated activation of the innate immune response.’ (score: 0.0)\n",
            "Document #244: ‘G3BP1 negatively regulates Picornaviridae infection by binding the 30 untranslated regions (30UTRs) or Internal Ribosome Entry Site (IRES) regions of viral RNAs to inhibit viral replication [53,95].’ (score: 0.0)\n",
            "Document #243: ‘Picornaviruses Inhibit G3BP1 Function through Protease-Mediated Cleavage Picornaviruses are small, non-enveloped, positive-sense RNA viruses that include EMCV, EV, PV, FMDV, and CVB3.’ (score: 0.0)\n",
            "Document #242: ‘Here we summarize the strategies shared by viruses in the Picornaviridae and Coronaviridae families to counteract host defense.’ (score: 0.0)\n",
            "Document #241: ‘To counter these antiviral functions, viruses target G3BP1—either cleaving or sequestering it during infection.’ (score: 0.0)\n",
            "Document #240: ‘G3BP1 also assembles SGs in response to infection-mediated translation arrest and limits the availability of translation factors for viral protein synthesis.’ (score: 0.0)\n",
            "Document #239: ‘In response to IFN, G3BP1, G3BP2, and Caprin 1 promote ISG translation to synthesize anti-viral factors.’ (score: 0.0)\n",
            "Document #238: ‘In virus-infected cells, G3BP1 binds to dsRNA resulting from the viral replication intermediates through the RGG domain and elevates RIG-I induced IFN-ß mRNA expression [30].’ (score: 0.0)\n",
            "Document #237: ‘G3BP1 enhances the immune response against invading viruses by inducing the expression of IFN and translation of ISGs.’ (score: 0.0)\n",
            "Document #236: ‘Here, we explore the mechanisms by which different viruses regulate G3BP1 function during infection.’ (score: 0.0)\n",
            "Document #235: (2) recruiting ribosomes to initiate viral RNA translation; (3) serving as a scaffold to build viral replication complexes; (4) assembling SGs; (5) stabilizing and enhancing the IFN-ß response; and (6) activating cGAS and RIG-I pathways.’ (score: 0.0)\n",
            "Document #234: ‘Depending on the virus type, G3BP1 plays different roles in the viral life cycle, including (1) stabilizing viral RNA; (score: 0.0)\n",
            "Document #233: ‘G3BP1 facilitates replication of some viruses whereas it restricts viral spread in other viruses.’ (score: 0.0)\n",
            "Document #232: ‘G3BP1 controls several key cellular processes, including mRNA metabolism, ribosomal quality control, immune response, and SG assembly.’ (score: 0.0)\n",
            "Document #231: ‘Here, we describe the role of G3BP1 during specific viral infections and the ways by which G3BP1 is targeted.’ (score: 0.0)\n",
            "Document #230: ‘On the contrary, viruses cleave G3BP1 when it exerts an anti-viral role (Table 2).’ (score: 0.0)\n",
            "Document #229: ‘In general, viruses use their genome-encoded proteins to sequester G3BP1 if it is required for viral replication.’ (score: 0.0)\n",
            "Document #228: ‘Further clues are obtained from identifying the strategies that viruses employ to target G3BP1—i.e., sequestration, cleavage, or degradation—and modulate its function during infection (Figure 4).’ (score: 0.0)\n",
            "Document #227: ‘Phosphorylation of G3BP1 by BTK is critical for dimerization and phase separation to limit viral spread [88].’ (score: 0.0)\n",
            "Document #226: ‘showed that G3BP1 is phosphorylated at tyrosine residue 40 by Bruton’s tyrosine kinase (BTK).’ (score: 0.0)\n",
            "Document #225: ‘Using polyinosinic:polycytidylic acid as a mimic for viral replication double-stranded intermediates, Kim et al.’ (score: 0.0)\n",
            "Document #224: ‘Similarly, arginine methylation status of G3BP1 partly recruits Tudor domain-containing protein (TDRD3) to regulate innate immune response against enterovirus species [60].’ (score: 0.0)\n",
            "Document #223: ‘Thus, post-translationally modified G3BP1 may act as an antiviral protein, while the unmodified or differently modified G3BP1 could play a pro-viral role during CHIKV infection.’ (score: 0.0)\n",
            "Document #222: ‘However, G3BP1 is also required for establishing CHIKV replication complexes, and post-translational modifications required for this process have not been defined [23].’ (score: 0.0)\n",
            "Document #245: ‘To counteract these effects, the 3C protease or leader protein of these viruses cleaves G3BP1 between Q325 and Q326 at late infection stages.’ (score: 0.0)\n",
            "Document #246: ‘G3BP1 cleavage is essential for the expression of viral proteins, and expression of the non-cleavable G3BP1 Q325E mutant suppresses viral replication [61–69].’ (score: 0.0)\n",
            "Document #247: ‘During Picornaviruses infection, PKR is activated by double-stranded RNA intermedi¬ates, phosphorylated eIF2a, and induces SG formation.’ (score: 0.0)\n",
            "Document #260: ‘Expression of non-cleavable G3BP1 results in persistent SGs and reduces viral replication, but the mechanism behind G3BP1-mediated reduction of viral replication remains unexplored [79,80].’ (score: 0.0)\n",
            "Document #270: ‘For example, during CHIKV infection, depletion of both G3BP1 and G3BP2 significantly decreases viral RNA levels, protein expression, and subsequent viral progeny produc¬tion [23].’ (score: 0.0)\n",
            "Document #326: PRRSV induces SGs, recruits G3BP1 and the viral replicase protein nsP1ß, thereby inhibiting G3BP1-mediated PKR activation [108].’ (score: 0.0)\n",
            "Document #268: ‘Although G3BP1 synergizes host immune responses to fight viral invasion, some viruses require G3BP1 for efficient infection.’ (score: 0.0)\n",
            "Document #267: ‘In COVID-19 patients and mice, the N protein interacts with G3BP1, suppresses SG formation, and potentiates viral infection by antagonizing the G3BP1-mediated host innate immune response pathway [97,98].’ (score: 0.0)\n",
            "Document #266: highlighted the potential antiviral role of G3BP1 in the lung tissue of COVID-19 patients and SARS-CoV-2-infected mice.’ (score: 0.0)\n",
            "Document #265: ‘Recent studies (score: 0.0)\n",
            "Document #264: ‘However, the role of G3BP1 remains unclear given the conflicting reports on the effect of G3BP1 knockdown in structural protein and viral RNA levels [81–83,96].’ (score: 0.0)\n",
            "Document #263: ‘Though it is unclear whether SARS-CoV-2 induces SGs, G3BP1–nucleocapsid binding blocks the interaction of SG proteins such as PKR and USP10 with G3BP1 in arsenite-treated cells [82].’ (score: 0.0)\n",
            "Document #262: ‘The intrin¬sically disordered region (IDR) at the N-terminal region of nucleocapsid is required for condensation with G3BP1, and this IDR is also crucial for viral particle production.’ (score: 0.0)\n",
            "Document #261: ‘SARS-CoV-2—the virus responsible for the COVID-19 pandemic—modulates G3BP1 function by sequestration and condensation with the nucleocapsid protein.’ (score: 0.0)\n",
            "Document #259: ‘However, these viruses utilize different strategies to modulate G3BP1 function; for example, PEDV induces cleavage of G3BP1 between D168 and D169 by caspase-8 to regu¬late SG dynamics in infected cells.’ (score: 0.0)\n",
            "Document #248: ‘SGs are observed early in infection but disassembled at late stages of infection.’ (score: 0.0)\n",
            "Document #258: ‘Overexpression of G3BP1 reduces viral replication, protein synthesis, and virus production, while G3BP1 knockdown enhances viral replica¬tion.’ (score: 0.0)\n",
            "Document #257: ‘Similar to Picornaviridae, G3BP1 nega¬tively regulates coronavirus infection.’ (score: 0.0)\n",
            "Document #256: ‘Coronaviruses Modulate G3BP1 Function Either by Cleavage or Sequestration Coronaviridae is a family of enveloped viruses with large single-stranded positive-sense RNA genomes and includes PEDV and SARS-CoV-2.’ (score: 0.0)\n",
            "Document #255: the fundamental mechanism employed by Picornaviridae to restrict antiviral activities of G3BP1.’ (score: 0.0)\n",
            "Document #254: ‘Thus, protease-mediated cleavage or degradation of G3BP1 is (score: 0.0)\n",
            "Document #253: ‘FMDV infection also causes degradation of G3BP1 through autophagy [62].’ (score: 0.0)\n",
            "Document #252: ‘Catalytic mutant C41A or leaderless FMDV efficiently inhibits arsenite-induced SG formation in infected cells [63].’ (score: 0.0)\n",
            "Document #251: ‘FMDV, which uses the leader protein to cleave G3BP1, also causes SG disassem¬bly.’ (score: 0.0)\n",
            "Document #250: ‘Consistent with the observed effect on G3BP1 cleavage, knockdown of G3BP1 increases viral protein synthesis and virus production, whereas G3BP1 overexpression shows an inhibitory effect [61,64–66].’ (score: 0.0)\n",
            "Document #249: ‘SG disassembly correlates temporally with G3BP1 cleavage, and the expression of the non-cleavable G3BP1 Q325E mutant results in sustained IFN-ß mRNA expression and SG persistence.’ (score: 0.0)\n",
            "Document #325: ‘Cells infected (score: 0.0)\n",
            "Document #331: ‘Although G3BP1 is sequestered within the viral inclusions, the exact role of G3BP1 in EBOV infection is not well understood.’ (score: 0.0)\n",
            "Document #327: ‘4.4.2.’ (score: 0.0)\n",
            "Document #406: ‘The translation process relies on cellular translational machinery that is present within the host cell cytoplasm, eliciting a strong stress re¬sponse of cells.’ (score: 0.0)\n",
            "Document #404: ‘The NP encapsulates (2) viral gRNA as a template to produce (+) cRNA and mRNA.’ (score: 0.0)\n",
            "Document #403: ‘After invasion of host cells by NDV, the NP-wrapped viral genomic RNA, as well as other viral components, is released into the cytoplasm (27).’ (score: 0.0)\n",
            "Document #402: ‘The PP is essential for viral RNA synthesis from bona fide templates (26).’ (score: 0.0)\n",
            "Document #401: ‘The NP encapsu¬lates the RNA genome, which serves as the template for viral transcription and replication (25).’ (score: 0.0)\n",
            "Document #400: ‘The NDV genome contains 6 genes that encode 2 glycoproteins, the fusion protein and hemagglutinin-neuraminidase protein, the matrix protein, and the ribonucleoprotein complex, com¬posed of the nucleocapsid protein (NP), phosphopro¬tein (PP), and large polymerase, responsible for viral transcription and replication (23).’ (score: 0.0)\n",
            "Document #399: ‘Oncolytic NDV can selectively replicate in tumor cells and is an important antineo¬plastic agent (24).’ (score: 0.0)\n",
            "Document #398: ‘NDV is a member of the Paramyxoviridae family with a negative-sense, nonsegmented, single-stranded RNA genome (23).’ (score: 0.0)\n",
            "Document #397: ‘Depletion of endogenous G3BP inhibits SG formation and significantly decreases the viral titers and RNA amounts (22).’ (score: 0.0)\n",
            "Document #396: ‘Respiratory syncytial virus (RSV) also triggers canoni¬cal SGs.’ (score: 0.0)\n",
            "Document #395: ‘For instance, knockdown of SG proteins TIA-1, TIAR, or Ras GTPase¬activating protein-binding protein 1 (G3BP1) impairs HCV RNA replication, virus assembly, and egress (18).’ (score: 0.0)\n",
            "Document #394: ‘However, in situations where the virus induces bona fide SG formation, SG components seem to be required for virus infection.’ (score: 0.0)\n",
            "Document #393: ‘These findings support the hy¬pothesis that SGs (or SG-like structures) are cellular antiviral responses.’ (score: 0.0)\n",
            "Document #392: ‘AVG formation during poxvirus infection leads to adecreaseinbothviral translation and virus titers (20, 21).’ (score: 0.0)\n",
            "Document #391: ‘In other cases, viruses induce SG-like structures (e.g., antiviral granules, AVGs) that re¬strict virus replication.’ (score: 0.0)\n",
            "Document #390: ‘A mutant JEV carrying a core protein incapable of binding to caprin-1 abrogates the SG inhibitory ability and exhibits lower propagation in vitro and in vivo (19).’ (score: 0.0)\n",
            "Document #389: ‘As an example, Japanese encephalitis virus (JEV) inhibits SG formation through interaction between the viral core protein and caprin-1.’ (score: 0.0)\n",
            "Document #388: ‘In cases where viral infection does not induce SGs, accumulat¬ing evidence demonstrates an inverse correlation be¬tween SG formation and viral propagation.’ (score: 0.0)\n",
            "Document #387: ‘SGs generally play an antiviral role (19–21).’ (score: 0.0)\n",
            "Document #386: ‘For example, SGs are served at 48 h post infection (hpi) with HCV and increase to speak at 72hpi (18).’ (score: 0.0)\n",
            "Document #385: ‘Other viru¬ses induce stable SG formation.’ (score: 0.0)\n",
            "Document #384: ‘For example, during the early stages of picor¬navirus infection, v-SGs can be observed but then disappear, probably because of the viral 3C protease, which is responsible for the cleavage of the Ras-GAP SH3 domain-binding protein (G3BP) (15–17).’ (score: 0.0)\n",
            "Document #383: ‘Some viruses induce transient SG for¬mation.’ (score: 0.0)\n",
            "Document #382: ‘For example, wild-type influenza A virus does not induce SGs but an NS1¬mutant virus triggers SG formation in a PKR-dependent manner (13, 14).’ (score: 0.0)\n",
            "Document #405: ‘Viral mRNAs are capped and polyadenylated by L-protein during pro¬tein synthesis (28, 29).’ (score: 0.0)\n",
            "Document #407: ‘In this study, the effect of NDV-induced SGs on its replication and the possible mechanism were investigated.’ (score: 0.0)\n",
            "Document #328: ‘G3BP1 Is Sequestered within Ebola Virus Inclusions EBOV, a single-stranded negative-sense RNA virus, does not induce SG assembly but sequesters SG components G3BP1, eEF2, eIF2, and eIF3 through the viral protein VP5 [55,87].’ (score: 0.0)\n",
            "Document #408: ‘Our results demonstrate that NDV-induced SGs facilitate viral replication through ma-nipulating host protein translation.’ (score: 0.0)\n",
            "Document #431: ‘Together, these results demonstrate that NDV-induced SGs are dependent on the PKR/eIF2a signaling pathway.’ (score: 0.0)\n",
            "Document #430: ‘4C–E).’ (score: 0.0)\n",
            "Document #429: ‘Depletion of endogenous PKR by siRNA reduced the level of eIF2a phosphory¬lation and the percentage of cells with SGs significantly (Fig.’ (score: 0.0)\n",
            "Document #428: ‘4B).’ (score: 0.0)\n",
            "Document #427: ‘Western blot analysis showed that PKR and down¬stream eIF2a was phosphorylated after ARS treatment and virus infection (Fig.’ (score: 0.0)\n",
            "Document #426: ‘4A).’ (score: 0.0)\n",
            "Document #425: ‘PKR were colocalized with either TIA-1 or G3BP1, respectively, after infection (Fig.’ (score: 0.0)\n",
            "Document #424: ‘To examine whether PKR acts as an upstream component of eIF2a in the formation of v-SGs, cells were costained for PKR and TIA-1/G3BP1.’ (score: 0.0)\n",
            "Document #423: ‘3B).’ (score: 0.0)\n",
            "Document #422: ‘(Fig.’ (score: 0.0)\n",
            "Document #421: ‘Conversely, S51D eIF2a transfection led to the accumulation of SG dots that appeared larger than v-SGs.’ (score: 0.0)\n",
            "Document #420: ‘No SG dots were observed in S51A eIF2a¬transfected cells, compared with the SG dots observed in WT eIF2a-transfected cells.’ (score: 0.0)\n",
            "Document #419: ‘To elucidate further the potential role of eIF2a in v-SGs, we measured SG formation by transfecting 2 eIF2a mutant plasmids, S51D and S51A eIF2a, which mimic hyper-and hypo phosphorylated forms of eIF2a,re¬spectively.’ (score: 0.0)\n",
            "Document #418: ‘We noted that the decreased levels of eIF2a and b-actin at 24 hpi were the result of the cell shedding caused by virus infection.’ (score: 0.0)\n",
            "Document #417: ‘Correspondingly, the viral NP accumulated at the late stage of infection.’ (score: 0.0)\n",
            "Document #416: ‘As expected, NDV infection reduced host protein translation over the time).’ (score: 0.0)\n",
            "Document #415: ‘The decrease of eIF2-GTP-tRNAMet ultimately leads to a global decrease in protein syn¬thesis (39).’ (score: 0.0)\n",
            "Document #414: ‘Phosphorylation of eIF2a leads to SG assembly by reducing the levels of the ternary complex eIF2-GTP-tRNAMet.’ (score: 0.0)\n",
            "Document #413: ‘3A).’ (score: 0.0)\n",
            "Document #412: ‘We measured eIF2a phosphoryla¬tion and found that NDV triggered eIF2a phosphorylation at the late stage of infection, consistent with the appear¬ance of v-SGs (Fig.’ (score: 0.0)\n",
            "Document #411: The activation of PKR and eIF2a leads to SG formation in virus-infected cells (38).’ (score: 0.0)\n",
            "Document #410: ‘PKR/eIF2a signaling cascade is essential for NDV-induced SGs (score: 0.0)\n",
            "Document #409: ‘Taken together, these results suggest that NDV induces canonical SGs, which is similar to the SGs formed under oxidative stress, indicating that ongoing protein synthesis is essential for NDV-induced SGs.’ (score: 0.0)\n",
            "Document #381: ‘Some viruses inhibit SG formation.’ (score: 0.0)\n",
            "Document #380: ‘The pattern of virus-induced (v)-SGs varies in the virus species.’ (score: 0.0)\n",
            "Document #379: ‘Recent years have seen a growing list of viruses with replication that is associated with SG formation (12).’ (score: 0.0)\n",
            "Document #378: ‘Some viruses escape translation inhibition through affect¬ing SG formation (11).’ (score: 0.0)\n",
            "Document #351: ‘In this study, Newcastle disease virus (NDV) triggered stable formation of bona fide SGs on HeLa cells through activating the protein kinase R (PKR)/eIF2a pathway.’ (score: 0.0)\n",
            "Document #350: ‘Virus-induced SGs function in different ways, depending on the species of virus; however, the mechanism of SG regulation of virus replication is not well understood.’ (score: 0.0)\n",
            "Document #349: ‘Mammalian cells respond to various environmental stressors to form stress granules (SGs) by arresting cytoplasmic mRNA, protein translation element, and RNA binding proteins.’ (score: 0.0)\n",
            "Document #348: ‘Therefore, not all G3BP1-positive structures are SGs, and should be stained with other markers to ascertain their identities.’ (score: 0.0)\n",
            "Document #347: ‘Given that the interacting partners of G3BP1 are modulated during infection to form new complexes in a virus-specific manner, different terms have been used to differentiate for these viral-induced protein complexes, including atypical SGs (EV), antiviral SGs (IBV), or nsP3 foci (alphaviruses).’ (score: 0.0)\n",
            "Document #346: ‘During norovirus infection, the G3BP1 interactome is modulated to inhibit the formation of SGs.’ (score: 0.0)\n",
            "Document #345: ‘In SIV-infected cells, G3BP1 stably interacts with viral proteins throughout the life cycle, whereas some SG components interact in a time-dependent manner.’ (score: 0.0)\n",
            "Document #344: ‘Recent proteomic studies provided some insights into the G3BP1 interactome during infection.’ (score: 0.0)\n",
            "Document #343: G3BP1 plays critical yet distinct roles during infection by different families of the virus.’ (score: 0.0)\n",
            "Document #342: ‘Furthermore, the differences in the role (score: 0.0)\n",
            "Document #341: ‘Given that G3BP1 is also modified by phosphorylation and ubiquitylation, it is worth checking the status of G3BP1 for other post-translational modifications at different infection stages and determine how these modifications affect a viral life cycle [33,42].’ (score: 0.0)\n",
            "Document #340: ‘Alphaviruses and coronaviruses encode an enzyme to remove ADP-ribosylation indicating that the modified form of G3BP1 may possess antiviral functions.’ (score: 0.0)\n",
            "Document #339: ‘G3BP1 methylation recruits TDRD3 into SGs to regulate innate immune response [60].’ (score: 0.0)\n",
            "Document #338: ‘Intriguingly, the post-translational modification status of heavily regulated G3BP1 may determine the functional consequences of interaction.’ (score: 0.0)\n",
            "Document #337: ‘Besides, some viruses possess conserved motifs (e.g., FGDF) within their proteins to bind and modulate G3BP1 func¬tion [76,109].’ (score: 0.0)\n",
            "Document #336: ‘For pro-viral roles, the NTF2L and EGG domains of G3BP1 play a crucial role to bind viral proteins or viral RNA, respectively.’ (score: 0.0)\n",
            "Document #335: ‘If G3BP1 is required for viral replication, viruses do not cleave G3BP1, but instead recruit it to build viral replication complexes (e.g., alphaviruses).’ (score: 0.0)\n",
            "Document #334: ‘In general, if G3BP1 exerts anti-viral activity, then viruses cleave or sequester G3BP1 (e.g., polioviruses, flaviviruses).’ (score: 0.0)\n",
            "Document #333: ‘These functional modulations of G3BP1 by viruses impact the viral life cycle either posi¬tively or negatively.’ (score: 0.0)\n",
            "Document #332: ‘Viruses employ at least three strategies to target and modulate G3BP1 function during infection: (1) enzymatic activity such as proteolytic cleavage or removing post-translational modification (e.g., poliovirus and alphavirus); (2) binding or sequestration (e.g., alphavirus and coronavirus); and (3) redistribution or remodeling of G3BP1 interaction (e.g., norovirus).’ (score: 0.0)\n",
            "Document #220: ‘One way by which G3BP1 may regulate its function dur¬ing infection is through post-translational modifications.’ (score: 0.0)\n",
            "Document #330: ‘VP5 expression affects IFNa/ß expression and PKR kinase activation in infected cells.’ (score: 0.0)\n",
            "Document #329: ‘Although infection-mediated SG assembly is not observed, EBOV inhibits arsenite-induced SG assembly.’ (score: 0.0)\n",
            "Document #352: ‘NDV-induced SGs contained classic SG markers T-cell internal antigen (TIA)-1, Ras GTPase-activating protein-binding protein (G3BP)-1, eukaryotic initiation factors, and small ribosomal subunit, which could be disassembled in the presence of cycloheximide.’ (score: 0.0)\n",
            "Document #353: ‘Treatment with nocodazole, a microtubule disruption drug, led to the formation of relatively small and circular granules, indicating that NDV infection induces canonical SGs.’ (score: 0.0)\n",
            "Document #354: ‘Furthermore, the role of SGs on NDV replication was investigated by knockdown of TIA-1 and TIA-1-related (TIAR) protein, the 2 critical components involved in SG formation from the HeLa cells, followed by NDV infection.’ (score: 0.0)\n",
            "Document #367: ‘The RNA-binding proteins T-cell internal antigen-1 (TIA-1) and TIA-1¬related (TIAR) protein are then shuttled from the nucleus to the cytoplasm to sequester untranslated mRNAs at SGs (1).’ (score: 0.0)\n",
            "Document #377: ‘Phosphory¬lation of eIF2a also leads to SG formation.’ (score: 0.0)\n",
            "Document #376: ‘In this case, viruses such as hepatitis C(HCV) keep on translating through IRES structure in the presence of activated PKR and phosphorylation of eIF2a (10).’ (score: 0.0)\n",
            "Document #375: ‘Some of theseRNA viruses induce phosphorylation of eIF2a and shut-off of global protein synthesis.’ (score: 0.0)\n",
            "Document #374: ‘Many other RNA viruses contain an internal ribosome entry site (IRES) located in the 59-untranslated region of the genomic (g)RNA, which drives cap-independent ini¬tiation of translation of the viral message (9).’ (score: 0.0)\n",
            "Document #373: ‘For instance, the influenza virus nonstructural protein NS1 interacts with eIF4GI and PABP-1, as well as with viral mRNAs, promoting the specific recruitment of 43S complexes to the viral mRNAs (8).’ (score: 0.0)\n",
            "Document #372: ‘They adopt a se¬ries of methods to rob host translation systems.’ (score: 0.0)\n",
            "Document #371: ‘As obligate intracellular parasites, viruses rely on the host cell translation machinery.’ (score: 0.0)\n",
            "Document #370: ‘This process ultimately leads to reduced translation initiation rates and to SG formation (6, 7).’ (score: 0.0)\n",
            "Document #369: ‘The other non canonical mechanism of SG induction is independent of eIF2a and involves inhibiting the activity of eIF4A, eIF4G, and eIF4E, 3 subunits of the eIF4F complex.’ (score: 0.0)\n",
            "Document #368: ‘The phosphorylation of eIF2a at Ser-51 is mediated by several kinases, including the heme-regulated inhibitor kinase, general control nonderepressible kinase, pro¬tein kinase R (PKR), and the PKR-like endoplasmic re¬ticulum kinase (5).’ (score: 0.0)\n",
            "Document #366: ‘When cells are exposed to stressors, phosphorylation of eIF2a leads to a decreased abundance of eIF2-GTP-tRNAMet (2).’ (score: 0.0)\n",
            "Document #355: ‘Results showed that depletion of TIA-1 or TIAR inhibited viral protein synthesis, reduced extracellular virus yields, but increased global protein translation.’ (score: 0.0)\n",
            "Document #365: ‘The eIF2-GTP-tRNAMet ter¬nary complex associates with the 40S complex and scans for an initiation codon (2).’ (score: 0.0)\n",
            "Document #364: ‘The central region of eIF4G binds to the 40S ribosome subunit, and the N terminus recruits mRNA through binding to poly(A)¬binding protein (PABP) (4).’ (score: 0.0)\n",
            "Document #363: ‘eIF4F consists of 3 initiation factors: eIF4E, eIF4G, and eIF4A.’ (score: 0.0)\n",
            "Document #362: ‘Conventional cap-dependent translation of most cellular mRNA commences with the recognition of 7¬methyl-GTP cap by the eukaryotic initiation factor (eIF) 4F complex of translation initiation factors (3).’ (score: 0.0)\n",
            "Document #361: ‘In this case, stress granules (SGs) are induced to arrest cytoplas¬mic mRNA, protein translation factors, and RNA binding proteins (1, 2).’ (score: 0.0)\n",
            "Document #360: oxidative stress, heat shock, and UV irradiation.’ (score: 0.0)\n",
            "Document #359: ‘Mammalian cells enter a relatively stable state by tempo¬rarily inhibiting cellular protein translation when exposed to various environmental stressors, such as virus infection, (score: 0.0)\n",
            "Document #358: ‘These results demonstrate that NDV triggers stable formation of bona fide SGs, which benefit viral protein translation and virus replication by arresting cellular mRNA.’ (score: 0.0)\n",
            "Document #357: ‘Deletion of TIA-1 or TIAR reduced NP mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #356: ‘FISH revealed that NDV-induced SGs contained predominantly cellular mRNA rather than viral mRNA.’ (score: 0.0)\n",
            "Document #221: ‘For example, ADP-ribosylated G3BP1 is crucial for SG assembly, given that chikungunya virus (CHIKV) actively reduces G3BP1 ADP-ribosylation, disrupting SGs with translation factors released from the con¬densed state [47].’ (score: 0.0)\n",
            "Document #216: ‘On the other hand, the presence of SGs may facilitate the switch from viral translation to genome amplification by sequestering translation factors into the condensed state (Figure 3) [23].’ (score: 0.0)\n",
            "Document #219: ‘G3BP1 can either promote or inhibit the viral life cycle, but the molecular mechanisms are not completely understood.’ (score: 0.0)\n",
            "Document #86: ‘Conse¬quently, the currently available PRRS vaccines are unable to induce sterilizing immunity (30), and the viral persistence in turn facilitates recombination among different strains and allows selection of immune escape mutants in the .eld (30, 31).’ (score: 0.0)\n",
            "Document #84: ‘PRRSV has remained a major threat to the worldwide swine production ever since its .rst emergence in late 1980s (27).’ (score: 0.0)\n",
            "Document #83: ‘The focus of this report is the interplay of PKR signaling with porcine reproductive and respiratory syndrome virus (PRRSV), an economically devastating swine pathogen and a positive-stranded RNA virus in the family Arteriviridae of the order Nido¬virales (25, 26).’ (score: 0.0)\n",
            "Document #82: ‘They specify viral products to either interact directly with PKR (17–19), or sequester and degrade dsRNA (20, 21), or target host PKR regulators, such as ADAR1, P58IPK, and PACT (22–24).’ (score: 0.0)\n",
            "Document #81: ‘To overcome PKR restriction, many viruses have evolved various strategies (2).’ (score: 0.0)\n",
            "Document #80: ‘Additionally, the activated PKR critically mediates the induction of interferons and modu¬lates cellular in.ammatory responses to build up an antiviral state by inducing sustained activation of NF-.B via activating I.B kinase (IKK) and others (14–16).’ (score: 0.0)\n",
            "Document #79: ‘Noticeably, SGs are increasingly emerging as a critical signaling platform for antiviral immunity and a key regulator of PKR activation (10, 13).’ (score: 0.0)\n",
            "Document #78: ‘The activated PKR phosphorylates the a-subunit of eukaryotic translation initiation fac¬tor 2 (eIF2a) to induce general translation arrest and to stimulate formation of stress gran¬ules (SGs) (7, 10), a kind of non membrane bound cytoplasmic aggregate that mainly include stalled translation initiation complexes, together with nucleating factors like Ras GTPase-activating protein-binding protein 1 (G3BP1), T cell intracytoplasmic antigen 1 (TIA1), and many others (11, 12).’ (score: 0.0)\n",
            "Document #77: ‘Alternatively, PKR can be activated via heterodimerization by an endogenous dsRNA binding protein, namely PKR-activating protein (PACT) (8).’ (score: 0.0)\n",
            "Document #76: ‘Binding of DRBMs to viral dsRNAs or 50 triphosphate RNAs induces conformational changes of PKR, leading to autophos¬phorylation of its KD (8, 9).’ (score: 0.0)\n",
            "Document #75: ‘Structurally, PKR is composed of two tan¬dem evolutionarily conserved dsRNA-binding motifs (DRBMs) in the N terminus, an effector/catalytic kinase domain (KD) in the C terminus, and a middle .flexible linker region connecting the two domains (2, 6, 7).’ (score: 0.0)\n",
            "Document #74: ‘This protein was initially discovered as a key component of interferon (IFN)-stimulated genes (3, 4) and later identified as a pattern-recognition receptor that specifically recog¬nizes double-stranded RNAs (dsRNA) (5, 6), a common intermediate produced during infections of both DNA and RNA viruses.’ (score: 0.0)\n",
            "Document #73: ‘At the virus–host interface, one such critical host player is the protein kinase R (PKR), a 551-amino acid-long serine threonine kinase (1, 2).’ (score: 0.0)\n",
            "Document #72: Eukaryotic cells employ an amazingly wide variety of intracellular defenses to restrict invading viral pathogens.’ (score: 0.0)\n",
            "Document #71: ‘PKR j stress granules j PRRSV j in.ammation j nsp1ß (score: 0.0)\n",
            "Document #70: ‘Thus, PRRSV has evolved a unique strategy to evade PKR restriction to suppress host in.ammatory responses.’ (score: 0.0)\n",
            "Document #69: ‘The viral mutant carrying the corresponding mutations induces elevated level of PKR phosphoryla¬tion and pronounced production of in.ammatory cytokines (e.g., tumor necrosis factor, interleukin [IL]-6, and IL-8), whereas small-interfering RNA knockdown of PKR or treatment with C16, a PKR inhibitor, blocks this effect.’ (score: 0.0)\n",
            "Document #68: ‘RNA interference silencing of G3BP1 or mutation of speci.c nsp1ß residues (VS19GG) can abolish the antagonization of PKR activation.’ (score: 0.0)\n",
            "Document #67: ‘We show that nsp1ß is a stress-responsive protein, enters virus-induced stress granules (SGs) during infection, and repurposes SGs into a proviral platform, where it co-opts the SG core component G3BP1 to interact with PKR in a regulated manner.’ (score: 0.0)\n",
            "Document #66: ‘Here, we report that this event is to suppress cellular in.ammation and is mediated by the viral replicase protein nsp1ß.’ (score: 0.0)\n",
            "Document #65: ‘However, this molecule is inactivated in the cells infected with porcine reproductive and respiratory syndrome virus (PRRSV), an economically devastating pathogen to the world swine industry.’ (score: 0.0)\n",
            "Document #64: ‘Protein kinase R (PKR) is a critical host restriction factor against invading viral pathogens.’ (score: 0.0)\n",
            "Document #63: ‘So when is a domain really a domain? And if the native state isn’t always the folded state, which state do we use to determine membership in a domain family? Although our definition of a domain may need to be adapted and expanded, we can take comfort that, for now, when focusing on typical globular domains one only needs to ensure appropriate filtering of the data.’ (score: 0.0)\n",
            "Document #62: ‘The proteome is now seen as existing on a continuum of varying structural content, with structure not necessarily present under physiological conditions, and disorder and order interchangeable states dependent upon the environment and protein interactions.’ (score: 0.0)\n",
            "Document #85: ‘One of the most prominent features of PRRSV is subversion of host immunity, as manifested by crippled induction of interferon and in.ammation, leading to inefficient cytotoxic T cell responses (26, 28, 29).’ (score: 0.0)\n",
            "Document #87: ‘Evolutionarily conserved protein kinase R (PKR) is a powerful mediator of antiviral responses.’ (score: 0.0)\n",
            "Document #218: ‘SG status during infection against different viruses.’ (score: 0.0)\n",
            "Document #88: ‘Enigmatically, PKR activity is disabled by porcine reproductive and respiratory syndrome virus (PRRSV) during infection, while the virus still allows the phosphorylation of its substrate eIF2a.’ (score: 0.0)\n",
            "Document #111: ‘3D).’ (score: 0.0)\n",
            "Document #110: ‘In transfected cells, nsp1ß did not colocalize with PKR, but they became colocalized once the cells were stressed with SG inducers (e.g., DTT or Ars) (Fig.’ (score: 0.0)\n",
            "Document #109: ‘3C).’ (score: 0.0)\n",
            "Document #108: ‘In infected cells, PKR colocalized well with nsp1ß and the SG markers, whereas in mock-infected cells it exhibited a diffusive distribution (Fig.’ (score: 0.0)\n",
            "Document #107: ‘This observation is consistent with previous reports that PKR is a stress-responsive protein and that its activation is dependent on SGs (10, 13, 36).’ (score: 0.0)\n",
            "Document #106: ‘PKR was also relocalized to PRRSV-induced SGs (Fig.3C).’ (score: 0.0)\n",
            "Document #105: ‘it does not require other viral factors, but rather depends on the cellular stress itself.’ (score: 0.0)\n",
            "Document #104: ‘Thus, the sorting of nsp1ß into SGs is an intrinsic property of this and colocalizes with PKR in SGs during infection.’ (score: 0.0)\n",
            "Document #103:  (score: 0.0)\n",
            "Document #102: ‘Treatment with other ER stress inducers (arsenite [Ars] and thapsigargin [TG]) obtained the same result (Fig.’ (score: 0.0)\n",
            "Document #101:  (score: 0.0)\n",
            "Document #100: ‘3A, Right).’ (score: 0.0)\n",
            "Document #99: ‘Indeed, treatment of MARC-145 cells with dithiothreitol (DTT), an endoplasmic reticulum (ER) stress inducer, readily led to relocation of ecto¬pically expressed nsp1ß to SGs (Fig.’ (score: 0.0)\n",
            "Document #98: ‘If this is the case, then exposure of nsp1ß in an arti.cial stress condition should allow it to enter SG.’ (score: 0.0)\n",
            "Document #97: ‘Since PRRSV prefers to induce a stress environment (e.g., induction of unfolded protein responses, phosphorylation of eIF2a, and so forth) conducive to replication (34), we hypothesized that nsp1ß might be a stress-responsive protein.’ (score: 0.0)\n",
            "Document #96: ‘PRRSV nsp1ß Is a Stress-Responsive Protein and Colocalizes with PKR in SGs During Infection.’ (score: 0.0)\n",
            "Document #95: ‘Strategically, PRRSV repur¬poses antiviral SGs into a proviral platform to overcome the PKR restriction.’ (score: 0.0)\n",
            "Document #94: ‘Here, we show that the primary pur¬pose of this selective targeting of PKR signaling is to suppress the cellular in.ammatory responses.’ (score: 0.0)\n",
            "Document #93: ‘We have previously reported that the eIF2a phosphorylation is conducive to utilization of ATF4 for viral replication (34).’ (score: 0.0)\n",
            "Document #92: ‘Interestingly, the activity of PKR is disabled in PRRSV-infected cells while the virus still allows the phosphorylation of its substrate eIF2a (32–34).’ (score: 0.0)\n",
            "Document #91: ‘These findings shed light on the mechanisms of PRRSV-mediated suppression of cellular inflammation, which might help in the rationale design of the vaccine, and reveal a unique strategy for viral evasion of PKR and SG restriction.’ (score: 0.0)\n",
            "Document #90: ‘Mechanistically, PRRSV takes advantage of virus-induced stress granules (SGs) and reprograms the antiviral SGs into a proviral platform by utilizing viral replicase protein nsp1ß to co-opt G3BP1 to inhibit PKR activation.’ (score: 0.0)\n",
            "Document #89: ‘Here, we show that this selective targeting of PKR signaling down-regulates cellular inflammatory responses.’ (score: 0.0)\n",
            "Document #61: ‘The challenge of structural domain categorization is only going to become more complex.’ (score: 0.0)\n",
            "Document #60: ‘At some point we will gather sufficient knowledge to consider categorization of IDPs into domain families, a somewhat daunting notion.’ (score: 0.0)\n",
            "Document #59: ‘Re-examination of our rejected targets here not only demonstrates the importance of filtering structural databases and applying a consistent descriptor for structural domains, it also raises questions for the near future.’ (score: 0.0)\n",
            "Document #58: ‘This reiterates how important it was to survey and re-qualify our consensus data prior to Dynameomics.’ (score: 0.0)\n",
            "Document #31: ‘Greater disorder was indicated in 43 of these folds, which were either listed in DisProt or had predictions for more than 30 contiguous disordered residues, or both.’ (score: 0.0)\n",
            "Document #30: ‘Some 182 of the 488 targets had predicted disordered regions at least ten residues in length and approximately 70 had missing coordinates, some coincidental with disordered regions as observed above for the crystal structures.’ (score: 0.0)\n",
            "Document #29: ‘Disorder predictions were performed on the 488 continuous folds; the fragmented nature of the 86 composite domains prohibited meaningful sequence analysis.’ (score: 0.0)\n",
            "Document #28: ‘For example, at one of the top-levels in the classification hierarchy of CATH, structures are grouped where they have a similar spatial arrangement of secondary structure elements, irrespective of chain directionality or continuity [46].’ (score: 0.0)\n",
            "Document #27: ‘Although it may seem strange to have structural units with spatially separated sequences, remember that these composite domains are present in the underlying databases as a consequence of the different methods used for categorization.’ (score: 0.0)\n",
            "Document #26: ‘Examples of these folds are the Rossman, immunoglobulin-like, and citrate synthase domain 1 folds: PDB IDs 1eq2, 1i31 and 1css, respectively, in Figure 4C.’ (score: 0.0)\n",
            "Document #25: ‘Another 86 folds are composite domains where the structural unit is interrupted by structural elements from a second domain or chain.’ (score: 0.0)\n",
            "Document #24: ‘Most of these non-autonomous folds, being structured in the context of a huge complex or in the presence of binding partners, were rejected because they had large regions buried within a complex making them unlikely to form stable structure in isolation.’ (score: 0.0)\n",
            "Document #23: ‘However, they typically form part of large complexes (PDB ID: 1eq2 in Fig.’ (score: 0.0)\n",
            "Document #22: ‘The 585 folds labeled as “non autonomous”, are not significantly larger than those folds in the other groupings, with only 11 containing over 450 residues.’ (score: 0.0)\n",
            "Document #21:  (score: 0.0)\n",
            "Document #18: ‘These data indicate that G4s are mostly suppressed in chromatin and that, in turn, they may influence the occupancy and positioning of nucleosomes.’ (score: 0.0)\n",
            "Document #14: ‘While RNA G4s are mostly locked in a parallel conformation due to the 2 -hydroxyl group in the sugar which exclusively allows the anti orientation (2), DNA G4s are in principle characterized by higher topological diversity, even though the majority of DNA G4s examined so far adopt the parallel topology.’ (score: 0.0)\n",
            "Document #13: ‘The anti conformation characterizes a parallel folding, while antiparallel G4s are found to adopt both syn and anti orientations (8) (Figure 1).’ (score: 0.0)\n",
            "Document #12: ‘Strands orientation defines the parallel, antiparallel or mixed topology of G4s, which is directly correlated to the conformational state, anti or syn, of the glycosidic bond between the G base and the sugar (1).’ (score: 0.0)\n",
            "Document #11: ‘G4s can fold intramolecularly from a single G-rich strand, or intermolecularly through dimerization or tetramerization of separate filaments: research of biologically relevant G4s has mainly focused on monomolecular G4s (3,4); however, intermolecular G4s are gaining increasing attention (5-7).’ (score: 0.0)\n",
            "Document #10: ‘G4s are highly polymorphic structures whose topology can be influenced by variations in strand stoichiometry and polarity, as well as by the nature and length of loops and their location in the sequence.’ (score: 0.0)\n",
            "Document #9: ‘Each tetrad is composed of four G residues that are linked by the sugar-phosphate backbone and connected through Hoogsteen-type hydrogen bonds.’ (score: 0.0)\n",
            "Document #8: 'G-quadruplexes (G4s) are nucleic acids secondary structures that can form within DNA (1) or RNA (2) guanine (G)-rich strands, when two or more G-tetrads stack on top of each other and coordinate monovalent cations, such as K+ and Na+.’ (score: 0.0)\n",
            "Document #6: ‘This review aims to provide an updated close-up of the literature on G4s in viruses.’ (score: 0.0)\n",
            "Document #3: ‘Viral G4s are usually located in regulatory regions of the genome and implicated in the control of key viral processes; in some cases, they have been involved also in viral latency.’ (score: 0.0)\n",
            "Document #2: ‘Indeed, G4s have been reported in several viruses, including those involved in recent epidemics, such as the Zika and Ebola viruses.’ (score: 0.0)\n",
            "Document #1: ‘G4s have been extensively described in the human genome, especially in telomeres and oncogene promoters; in recent years the presence of G4s in viruses has at-attracted increasing interest.’ (score: 0.0)\n",
            "Document #32: ‘Three of these in particular, the C-terminal domain of topoisomerase II (1bjt), cAMP response binding protein (CBP, 1kbh) domain, and heat shock locus HSLU (1ht2), have disorder predictions that correlate with regions already highlighted in DisProt (Fig.’ (score: 0.0)\n",
            "Document #33: ‘The CBP domain (PDB ID: 1kbh) is an established IDP [47].’ (score: 0.0)\n",
            "Document #34: ‘A first of its kind, this fold representative was one of two domains that are disordered in isolation but become structured upon forming a heterodimer.’ (score: 0.0)\n",
            "Document #47: ‘Many disordered proteins have some fold or structure, albeit less compact than the “typical” native states of globular proteins, with many able to form different conformations with different binding partners.’ (score: 0.0)\n",
            "Document #57: ‘Fortunately, our consensus approach, along with manual inspection, eliminated many of the domains with confirmed or putative disorder.’ (score: 0.0)\n",
            "Document #56: ‘Inspection of our rejected targets here revealed that many contain disordered regions.’ (score: 0.0)\n",
            "Document #55: ‘When we updated our CDD in 2009, the field of intrinsically disordered proteins was still an emerging field.’ (score: 0.0)\n",
            "Document #54: ‘Many of the structures assigned to domain families in CATH, SCOP and Dali were not strictly domains in a structural sense.’ (score: 0.0)\n",
            "Document #53: ‘In the course of this work, it became clear that – in some cases - there was much ambiguity as to what qualifies as a domain and what does not.’ (score: 0.0)\n",
            "Document #52: ‘In our Dynameomics project we generated a consensus set of protein folds and selected targets based on the physical attributes of what constituted a globular structural domain.’ (score: 0.0)\n",
            "Document #51: ‘If the complexity of attempting to categorize well-structured proteins into domain families has taught us anything, we can be sure that as the gaps in protein fold space are filled, defining a domain will only become more challenging.’ (score: 0.0)\n",
            "Document #50: ‘However, at the start of such a revolution in understanding protein structure, or non-structure, it becomes even more important to be able to confidently define domain boundaries and group similar folds together before including ‘dynamic’ or ‘disordered’ protein fold families.’ (score: 0.0)\n",
            "Document #49: ‘Moreover, some disordered regions have already shown attributes consistent with domains, leading to consideration of disordered domain classification [54].’ (score: 0.0)\n",
            "Document #48: ‘Perhaps, this is an indication we should start to consider “dynamic domain” families? After all, conservation of disordered regions has been demonstrated [53].’ (score: 0.0)\n",
            "Document #46: ‘It is possible that as we learn more, the folds we previously rejected for not being bona fide domains may be considered as such and we should be open to adapting or expanding the definition of domains.’ (score: 0.0)\n",
            "Document #35: ‘The other domain is the receptor activator (ACTR) domain, which exists as a molten globule until it forms a complex with the CBP domain [48].’ (score: 0.0)\n",
            "Document #45: ‘However, there has been increasingly more evidence accruing over the last years that all is not what it first seemed [25].’ (score: 0.0)\n",
            "Document #44: ‘Although there were some exceptions to this rule, they were believed to be just that.’ (score: 0.0)\n",
            "Document #43: ‘The dogma that the native, biologically active state is folded and structured is biased because of the limitations of experimental techniques.’ (score: 0.0)\n",
            "Document #42: ‘Much of what was believed for the last 50 years regarding the relationship between structure and function is now in question as more and more proteins with a local or global lack of structure are identified [25].’ (score: 0.0)\n",
            "Document #41: ‘Accordingly, these rejected non-autonomous folds have a high incidence of known and putative disordered regions.’ (score: 0.0)\n",
            "Document #40: ‘Coupled folding and binding is synonymous with the presence of disorder in unbound states.’ (score: 0.0)\n",
            "Document #39: ‘The heat shock locus HSLU domain (PDB ID: 1ht2) has two predicted disordered regions, one that matches with missing coordinates in the X-ray crystal structures [51, 52] and both are listed in DisProt.’ (score: 0.0)\n",
            "Document #38: ‘The C-terminal domain of topoisomerase II (PDB ID: 1bjt) has a region of missing coordinates representing an acidic region sensitive to protease degradation, leading to the suggestion that it is disordered [49, 50].’ (score: 0.0)\n",
            "Document #37: ‘NMR indicates that the unbound form of the CBP domain is 100% disordered [48].’ (score: 0.0)\n",
            "Document #36: ‘Although shown here in the complexed, structured form with the ACTR domain shaded in gray (Fig.’ (score: 0.0)\n",
            "Document #112: ‘Thus, these findings suggest that SGs are likely the battleground for nsp1ß to counteract PKR.’ (score: 0.0)\n",
            "Document #113: ‘Regulated Interaction between nsp1ß and PKR.’ (score: 0.0)\n",
            "Document #114: ‘It is likely that there is an interaction between nsp1ß and PKR.’ (score: 0.0)\n",
            "Document #167: ‘G3BP1 is highly conserved from yeast to humans and plays a key role in RNA metabolism, such as regulating the stability and translation of a subset of mRNAs, binding to specific transcripts during stress conditions, and activating interferon (IFN)-stimulated gene expression [18–21].’ (score: 0.0)\n",
            "Document #191: ‘G3BP1 can also facilitate the expression and translation of IFN-stimulated genes (ISGs).’ (score: 0.0)\n",
            "Document #190: ‘These data suggest that G3BP1-mediated interactions with signaling molecules facilitate the initiation of downstream innate response pathway signaling.’ (score: 0.0)\n",
            "Document #189: ‘G3BP1 is required for activation of cGAS as depletion of G3BP1 dramatically reduces DNA-induced IFN production [34].’ (score: 0.0)\n",
            "Document #188: ‘The local concentration of G3BP1, cGAS, and PKR within cytoplasmic foci is proposed to enhance PKR activity.’ (score: 0.0)\n",
            "Document #187: ‘The binding of the N-terminal region of cGAS with G3BP1 is required for activation of the RNA-dependent protein kinase PKR leading to co-localization as small cytoplasmic foci [35].’ (score: 0.0)\n",
            "Document #186: ‘G3BP1 also interacts with cGAS to form large complexes that enhance interaction with viral DNA to induce IFN [30,34,36].’ (score: 0.0)\n",
            "Document #185: ‘For example, the RGG domain of G3BP1 binds the RIG-I helicase domain and viral dsRNA to prevent RIG-I degradation and enhance Ifn-b mRNA expression [30].’ (score: 0.0)\n",
            "Document #184: ‘G3BP1 facilitates the initial IFN-ß response by binding and augmenting the function of viral RNA and DNA sensing proteins in the cytoplasm.’ (score: 0.0)\n",
            "Document #183: ‘G3BP1 regulates early innate responses through its ability to bind nucleic acids and proteins in these signaling pathways.’ (score: 0.0)\n",
            "Document #182: ‘Here, we focus on the role of G3BP1 during viral infection and the various strategies utilized by viruses to exploit or counteract G3BP1 functions.’ (score: 0.0)\n",
            "Document #181: ‘Given that G3BP1 impacts the life cycle of many viruses, it is important to distinguish what specific functional roles are played by G3BP1 at different infection stages.’ (score: 0.0)\n",
            "Document #180: ‘Knockout of both G3BP1 and G3BP2 is necessary to completely inhibit SG formation induced by many stressors, implying their importance to form protein–protein and protein–RNA interactions within SGs [29].’ (score: 0.0)\n",
            "Document #179: ‘SGs are a class of cytoplasmic biomolecular condensates enriched with mRNA–protein complexes assembled in response to stress, including viral infection.’ (score: 0.0)\n",
            "Document #178: ‘Well-studied functions of G3BP1 include participation in the innate response to infection through activation of viral nucleic acid sensors RIG-I and cGAS and as an essential component of stress granules (SGs) [29,30,32–35].’ (score: 0.0)\n",
            "Document #177: ‘For example, some viruses require G3BP1 for efficient viral replication, while the replication of other viruses is negatively affected by G3BP1 expression.’ (score: 0.0)\n",
            "Document #176: ‘Emergent data suggest that G3BP1 is one of the key proteins that impact the viral life cycle both positively and negatively [30,31].’ (score: 0.0)\n",
            "Document #175: ‘For example, knockdown of G3BP1 increases the expression of G3BP2 to compensate for the loss of G3BP1 expression and function, but not vice versa [23,29].’ (score: 0.0)\n",
            "Document #174: ‘The functions of G3BP1 and G3BP2 are redundant in some, but not all cases [29].’ (score: 0.0)\n",
            "Document #173: ‘The RRM domain is essential for RNA binding and RGG domain increases RNA binding as well as mediates protein–protein interactions.’ (score: 0.0)\n",
            "Document #172: ‘For instance, the NTF2L domain interacts with FGDF sequence motifs present in many proteins and the PxxP domain interacts with SH3 motifs.’ (score: 0.0)\n",
            "Document #171: ‘The N-terminal domains contain motifs for protein binding while the C-terminal domains bind nucleic acids.’ (score: 0.0)\n",
            "Document #170: ‘These domains include N-terminal nuclear transport factor 2-like (NTF2L), followed by an acidic domain, proline rich region (PxxP), an RNA recognition motif (RRM), and an arginine and glycine-rich region (RGG) at the C-terminus [27,28].’ (score: 0.0)\n",
            "Document #169: ‘G3BP1 and its paralog G3BP2 possess similar protein domain architecture composed of regions for protein, RNA and DNA interactions (Figure 1).’ (score: 0.0)\n",
            "Document #192: ‘For example, G3BP1, G3BP2, and their binding partner Caprin1 positively regulate the translation of multiple ISGs in response to IFN (Figure 2a) [21].’ (score: 0.0)\n",
            "Document #193: ‘G3BP1 Forms the Core of Stress Granules (score: 0.0)\n",
            "Document #194: A well-known function of G3BP1 during stress stimuli is its involvement in SG as¬sembly (Figure 2b).’ (score: 0.0)\n",
            "Document #207: ‘G3BP1 is the central node of the protein–protein interaction network within SGs, where RNA serves as a molecular trigger for phase separation.’ (score: 0.0)\n",
            "Document #217: ‘Table 1.’ (score: 0.0)\n",
            "Document #866:  (score: 0.0)\n",
            "Document #215: ‘Thus, viruses may target SGs not only to modulate G3BP1 function but also to disassemble SGs to counteract anti-viral mechanisms in the infected cells.’ (score: 0.0)\n",
            "Document #214: ‘Given that some of the key anti-viral proteins (such as ADAR1, PARP13/ZAP, OAS, IRF3, IRF7, TBK1, and RNase L) localize to SGs and activate the anti-viral response, SGs may negatively affect viral replication [56–60].’ (score: 0.0)\n",
            "Document #213: ‘As demonstrated by one of the earliest studies using Semliki Forest virus (SFV), infection-induced translation arrest results in the assembly of transient SGs.’ (score: 0.0)\n",
            "Document #212: ‘These patterns highlight the importance of modulating SGs upon virus infection (Table 1).’ (score: 0.0)\n",
            "Document #211: ‘At least four distinct temporal patterns of SGs have been observed during infection with different viruses: (1) stable, (2) transient, (3) oscillating, or (4) no SGs (active blocking of SG induction) [53–55].’ (score: 0.0)\n",
            "Document #210: ‘In light of the viral infection, it may be possible that at the early infection stage, an increase in viral RNA concentration may relieve G3BP1 from its auto-inhibitory state, triggering G3BP1 condensation, with the protein serving as the scaffold to recruit other proteins/RNA required for viral replication.’ (score: 0.0)\n",
            "Document #209: ‘Increasing local RNA concentrations can increase their binding to G3BP1, relieving G3BP1 from the autoinhibitory state and resulting in more RNA recruitment and less aberrant RNA–RNA interaction [50–52].’ (score: 0.0)\n",
            "Document #208: ‘In the normal state, G3BP1 adopts a compact, closed auto-inhibitory state with intramolecular interaction between acidic and arginine-rich regions.’ (score: 0.0)\n",
            "Document #206: ‘Recent studies provided more mechanistic insights into the role of G3BP1 as the scaffold for SG assembly.’ (score: 0.0)\n",
            "Document #195: ‘SGs assemble upon stress-induced translation arrest that results in the accumulation of free untranslated mRNAs and mRNA binding proteins in the cyto¬plasm [37].’ (score: 0.0)\n",
            "Document #205: ‘These data suggest that G3BP1/G3BP2 act as a scaffold that recruits other proteins into SGs [47,50].’ (score: 0.0)\n",
            "Document #204: ‘RNAi-mediated knockdown or genetic knockout of both G3BP1 and G3BP2 results in cells that do not form SGs in response to most stressors, emphasizing the importance of G3BP1/G3BP2 in SG assembly [29].’ (score: 0.0)\n",
            "Document #203: ‘G3BP1 and G3BP2 form the core protein–protein interaction network which binds translation factors, mRNA binding proteins, long and translationally incompetent mRNAs as well as specific transcripts during stress conditions [19,48,49].’ (score: 0.0)\n",
            "Document #202: ‘Of note, G3BP1 itself undergoes several post-translational modifications, including phosphorylation, methylation, ubiquitination, and ADP-ribosylation during stress conditions [42,44,47].’ (score: 0.0)\n",
            "Document #201: ‘Several post-translational modifications (PTMs) control the dynamics of SGs [33,41–46].’ (score: 0.0)\n",
            "Document #200: ‘These untranslated mRNAs bound with ribonucleoproteins (mRNPs) cluster together with G3BP1, G3BP2, and several other RNA binding proteins to form RNA/protein-rich condensates in the cytoplasm.’ (score: 0.0)\n",
            "Document #199: ‘Once the elongating ribosomes run off from polysomes, the stalled transla¬tion initiation results in a sudden influx of untranslated mRNAs in the cytoplasm.’ (score: 0.0)\n",
            "Document #198: ‘The initiation complex containing the phospho¬rylated eIF2a becomes unavailable for recycling, which leads to a stall in translation initiation [40].’ (score: 0.0)\n",
            "Document #197: ‘The activated kinase phosphorylates the translation initiation factor eIF2a subunit [38,39].’ (score: 0.0)\n",
            "Document #196: ‘Upon stress, one of the following five kinases is activated: double-stranded RNA (dsRNA)-dependent PKR, protein kinase R-like endoplasmic reticulum kinase (PERK), heme-regulated inhibitor (HRI), general control nonderepressible-2 (GCN2), or microtubule affinity-regulating kinase 2 (MARK2).’ (score: 0.0)\n",
            "Document #168: ‘G3BP1 function is further modulated by post-translational modifications and also implicated in many diseases such as cancer, neurodegeneration, and viral infection—the latter is the focus of this review [22–26].’ (score: 0.0)\n",
            "Document #166: ‘protein-binding protein G3BP1 is an RNA-binding protein targeted by many viruses [17].’ (score: 0.0)\n",
            "Document #115: ‘We tested this hypothesis by coimmunoprecipitation assay (co-IP).’ (score: 0.0)\n",
            "Document #165: ‘Concomitantly, induction of cellular immunity is delayed, usually 1 mo following infection, and it is maintained at a low level, contributing to viral persistence in vivo (26, 30).’ (score: 0.0)\n",
            "Document #138: ‘Thus, nsp1ß is responsible for inhibiting PKR activation dur¬ing infection.’ (score: 0.0)\n",
            "Document #137: as opposed to both WT and the mutant nsp1ß GGK16AAA .’ (score: 0.0)\n",
            "Document #136: ‘This was accom¬panied by a concurrent elevation of PKR phosphorylation in both cell types (score: 0.0)\n",
            "Document #135: ‘Similar to that in the transfection condition, the interaction of nsp1ß with G3BP1 and PKR were almost diminished in VS19GG infected cells (Fig.’ (score: 0.0)\n",
            "Document #134: ‘9A and B).’ (score: 0.0)\n",
            "Document #133: ‘The mutant GGK16AAA displayed a growth property similar to the parental virus in both MARC-145 cells and PAMs, whereas the growth of VS19GG was reduced (Fig.’ (score: 0.0)\n",
            "Document #132: ‘Only two mutants (VS19GG and GGK16AAA) were successfully rescued.’ (score: 0.0)\n",
            "Document #131: ‘ (score: 0.0)\n",
            "Document #130: G3BP1 Contributes to nsp1ß-Mediated Inhibition of PKR Activ¬ity.’ (score: 0.0)\n",
            "Document #129:  (score: 0.0)\n",
            "Document #128: ‘Together, these results suggest that the nsp1ß-PKR interaction is regulated and facilitated by cellular factors during infection.’ (score: 0.0)\n",
            "Document #127: ‘Consistent with this notion, nsp1ß showed partial colocalization with Flag-DRBM in the cytoplasm, but not with the full-length Flag-PKR or Flag-KD in transfected cells (Fig.’ (score: 0.0)\n",
            "Document #126: ‘4C), suggesting that a conformational change of PKR is likely necessary for the interaction.’ (score: 0.0)\n",
            "Document #125: ‘4D,lane 11). (score: 0.0)\n",
            "Document #124: ‘4D,lane 10),but the TF-DRBM mutant was abundantly recovered (Fig.’ (score: 0.0)\n",
            "Document #123: ‘To our surprise, GST-nsp1ß was unable to pull down the full-length PKR (TF-PKR) (Fig.’ (score: 0.0)\n",
            "Document #122: ‘Specifically, PRRSV nsp1ß was expressed as a GST fusion protein, whereas PKR or its deriva¬tives (DRBM and KD) were fused to the C terminus of bacterial chaperone trigger factor (TF) to promote protein solubility (Fig.’ (score: 0.0)\n",
            "Document #121: ‘The relevant proteins were expressed in a bacterial system.’ (score: 0.0)\n",
            "Document #120: ‘To test whether the interaction requires any eukaryotic factor, we carried out the in vitro binding assay.’ (score: 0.0)\n",
            "Document #119: ‘The co-IP assay showed that the mutant Flag-DRBM retained the ability to interact with nsp1ß to the level of full-length PKR (Fig.C, lane 6), whereas the mutant Flag-KD failed to interact suggesting that DRBM is the key region for binding to nsp1ß.’ (score: 0.0)\n",
            "Document #118: ‘WT PKR and its derivatives (Flag-DRBM and Flag-KD) were expressed either individually or in combination with HA-nsp1ß in HEK-293FT cells.’ (score: 0.0)\n",
            "Document #117: ‘The nsp1ß-binding region was further mapped by construction of PKR truncation mutants containing only the DRBM or the KD.’ (score: 0.0)\n",
            "Document #116: ‘The anti¬bodies to PKR could readily pull down nsp1ß from PRRSV-infected MARC-145 cell lysates and the same was true under the condition of transfection (Fig.’ (score: 0.0)\n",
            "Document #139: ‘The Mutant nsp1ß VS19GG Induces Inflammatory Responses via G3BP1-PKR Axis.’ (score: 0.0)\n",
            "Document #140: ‘To investigate the biological significance, we measured the production of interferons and in.ammatory cytokines by RT-qPCR in PRRSV-infected cells.’ (score: 0.0)\n",
            "Document #141: ‘In all treat¬ments, the CT values for IFN-a mRNA were near or over 35 cycles , and therefore the fold-changes are not reliable.’ (score: 0.0)\n",
            "Document #154: ‘As so, cation of the mutant VS19GG (Fig.’ (score: 0.0)\n",
            "Document #164: ‘PRRSV is a rather “sneaky” virus, and the infections often lead to low level production of in.ammatory cytokines in the early infec¬tion (26, 28, 29).’ (score: 0.0)\n",
            "Document #163: ‘Our results also unexpectedly revealed a regulatory mechanism for PRRSV-mediated suppression of host cellular in.ammatory responses.’ (score: 0.0)\n",
            "Document #162: ‘Insight into the Inflammation Regulation by PRRSV.’ (score: 0.0)\n",
            "Document #161: ‘Overall, these results suggest a strategy of two birds (G3BP1 and PKR) with one stone (nsp1ß).’ (score: 0.0)\n",
            "Document #160: ‘Certainly, this is a different story from PKR antagonization and warrants further exploration in the future.’ (score: 0.0)\n",
            "Document #159: ‘9A and B).’ (score: 0.0)\n",
            "Document #158: ‘This notion is in line with the observation that the mutant VS19GG had a reduced growth rate in both cell types (Fig.’ (score: 0.0)\n",
            "Document #157: ‘8B and C).’ (score: 0.0)\n",
            "Document #156: ‘This seemingly confusing result may be explained by WT nsp1ß normally inhibiting the antiviral function of G3BP1 via interaction, whereas the mutant nsp1ß VS19GG fails to do so due to loss of interaction (Fig.’ (score: 0.0)\n",
            "Document #155: ‘10).’ (score: 0.0)\n",
            "Document #153: ‘Cytoplasmic SGs are increasingly emerging as a critical immune signaling platform to detect and restrict invading viral pathogens (10, 11).’ (score: 0.0)\n",
            "Document #142: ‘In contrast, production of IFN-ß mRNA was highly inducible As compared with WT virus, there was a slight up-regulation of IFN-ß mRNA, but for both mutants, suggesting that the mutational effect is less likely due to PKR activation.’ (score: 0.0)\n",
            "Document #152: ‘Repurposing SG into a Proviral Platform.’ (score: 0.0)\n",
            "Document #151: ‘Asterisks (*) indicate the statistical significance: *P < 0.05; ***P < 0.001; NS, no significance.’ (score: 0.0)\n",
            "Document #150: ‘Statistical analysis was performed by two-tailed Student’s t test, and error bars indicate means ± SDs.’ (score: 0.0)\n",
            "Document #149: ‘The relative abundance of p-PKR in each treat¬ment was expressed as fold-changes compared to the intensity of p-PKR in the .rst lane after being normalized against PKR and then ß-actin in the same lane.’ (score: 0.0)\n",
            "Document #148: ‘The signals of protein bands were determined by ImageJ.’ (score: 0.0)\n",
            "Document #147: ‘(B) The same as A, except cells were transfected to express HA-nsp1ß.(C and D) The same as A and B, except that siRNAs were used to target TIA1.’ (score: 0.0)\n",
            "Document #146: ‘The cells were then subject to Western blot analysis with antibodies to the indicated proteins.’ (score: 0.0)\n",
            "Document #145: ‘At 36 h post transfection, the cells were infected with PRRSV strain JXwn06 at an MOI of 0.1 for 36 h before being treated with poly(I:C) for 12 h.’ (score: 0.0)\n",
            "Document #144: ‘(A) MARC-145 cells were transfected with siRNAs targeting G3BP1 or scram¬bled siRNA (siNC).’ (score: 0.0)\n",
            "Document #143: ‘On the other hand, despite a reduced viral growth, the mutant G3BP1 contributes to nsp1ß-mediated inhibition of PKR activation.’ (score: 0.0)\n",
            "Document #432: ‘Microtubule disruption leads to the formation of small SGs after NDV infection (score: 0.0)\n",
            "Document #433: SGs are considered to be dynamic structures.’ (score: 0.0)\n",
            "Document #434: ‘The move¬ment of canonical SGs is associated with cytoskeletal ele¬ments, especially microtubules (40–42).’ (score: 0.0)\n",
            "Document #719: ‘The direct visualization of G-quadruplex structures in human cellsas well as the co-crystal structure of an RNA helicase bound to a G-quadruplexes have provided important confirmations of their relevance to cell biology.’ (score: 0.0)\n",
            "Document #730: ‘Its concentration is used as a measurement of oxidative stress within a cell.’ (score: 0.0)\n",
            "Document #729: ‘Due to Guanine having a lower electron reduction potential than the other nucleotides bases,-oxo-'-deoxyguanosine (-oxo-dG), is a known major product of DNA oxidation.’ (score: 0.0)\n",
            "Document #728: ‘Guanine (G) bases in G-quadruplex have the lowest redox potential causing it to be more susceptible to the formation of -oxoguanine (-oxoG), an endogenous oxidized DNA base damage in the genome.’ (score: 0.0)\n",
            "Document #727: ‘These enzymes participate in BER to repair certain DNA lesions such as ,-dihydro--oxoguanine (-oxoG), which forms under oxidative stress to guanine bases.’ (score: 0.0)\n",
            "Document #726: ‘The BER pathway is signalled when it indicates an oxidative DNA base damage, where structures like, -0xoguanine-DNA glycosylase  (OGG), APE and G-quadruplex play a huge role in its repair.’ (score: 0.0)\n",
            "Document #725: ‘Base excision repair processes in cells have been proved to be reduced with aging as its components in the mitochondria begin to decline, which can lead to the formation of many diseases such as Alzheimer's disease (AD)_These G-quadruplex structures are said to be formed in the promoter regions of DNA through superhelicity, which favors the unwinding of the double helical structure of DNA and in turn loops the strands to form G­ quadruplex structures in guanine rich regions.’ (score: 0.0)\n",
            "Document #724: ‘AP site damage was found to be predominant in PQS regions of the genome, where formation of G-quadruplex structures is regulated and promoted by the DNA repair process, base excision repair (BER).’ (score: 0.0)\n",
            "Document #723: ‘AP site damage was also more prevalent in certain regions of the genome that contain specific active promoter and enhancer markers, some of which were linked to regions responsible for lung adenocarcinoma and colon cancer.’ (score: 0.0)\n",
            "Document #722: ‘Both of these genome-wide mapping sequencing methods, ChIP-sequencing and ARP, have indicated that AP site damage occurrence is nonrandom.’ (score: 0.0)\n",
            "Document #721: ‘T-loops and G-quadruplexes are described as the two tertiary DNA structures that protect telomere ends and regulate telomere length. Many of the genome regulatory processes have been linked to the formation of G-quadruplex structures, attributable to the huge role it plays in DNA repair of apurinic/apyrimidinic sites also known as AP sites. A new technique to map AP sites has been developed known as AP-seq which utilizes a biotin-labeled aldehyde-reactive probe (ARP) to tag certain regions of the genome where AP site damage occurrence has been significant. Another genome-wide mapping sequencing method known as ChIP-sequencing, was utilized to map both; damage in AP sites, and the enzyme responsible for its repair, AP endonuclease  (APE).’ (score: 0.0)\n",
            "Document #720: ‘The potential positive and negative roles of quadruplexes in telomere replication and function remains controversial.’ (score: 0.0)\n",
            "Document #718: ‘The roles of quadruplex structure in translation control are not as well explored.’ (score: 0.0)\n",
            "Document #651: 'virus Papanivirus – Panicum mosaic satellite virus Virtovirus – Tobacco mosaic satellite virus Albetovirus – Tobacco necrosis satellite virus, Family Sarthroviridae, Macronovirus – Macrobrachium satellite virus 1 (extra small virus) (unnamed genus) – Nilaparvata lugens commensal X virus (unnamed genus) – Chronic bee-paralysis satellite virus, Family Lavidaviridae – Virophages Single-stranded DNA satellite viruses, Genus Dependoparvovirus – Adeno-associated.' (score: 0.0)\n",
            "Document #717: ‘There are many studies that implicate quadruplexes in both positive and negative transcriptional regulation, including epigenetic regulation of genes like hTERT.  Function of G-quadruplexes have also been reported in allowing programmed recombination of immunologlobin heavy genes and the pilin antigenic variation system of the pathogenic Neisseria.’ (score: 0.0)\n",
            "Document #716: ‘ The DEAH/RHA helicase, DHX, has also been identified as a key G-quadruplex resolvase_o A metastasis suppressor protein NMH (also known as NME) was found to directly interact with G quadruplex in the promoter of the c-myc gene, and transcriptionally regulate c-myc. More recently, NMH was reported to interact with G-quadruplex in the promoter of the human telomerase (hTERT) gene and regulate hTERT expression.  The telomere-binding-factor- (TRF or TERF) was shown to bind to thousands of non-telomeric G-quadruplexes in the human genome by TRF ChIP-seq.’ (score: 0.0)\n",
            "Document #715: ‘ Quadruplex formation may be potentially damaging for a cell; the helicases WRN and Bloom syndrome protein have a high affinity for resolving DNA G-quadruplexes.’ (score: 0.0)\n",
            "Document #714: ‘It has been suggested that quadruplex formation plays a role in immunoglobulin heavy chain switching. As cells have evolved mechanisms for resolving (i.e., unwinding) quadruplexes that form.’ (score: 0.0)\n",
            "Document #713: ‘In another model, quadruplex formed at the non-coding DNA strand helps to maintain an open conformation of the coding DNA strand and enhance an expression of the respective gene.’ (score: 0.0)\n",
            "Document #712: ‘One model is shown below, with G­ quadruplex formation in or near a promoter blocking transcription of the gene, and hence de­ activating it.’ (score: 0.0)\n",
            "Document #711: ‘There are several possible models for how quadruplexes could influence gene activity, either by upregulation or downregulation.’ (score: 0.0)\n",
            "Document #710: ‘coli.’ (score: 0.0)\n",
            "Document #709: ‘A similar studies have identified putative G-quadruplexes in prokaryotes, namely the bacterium E.’ (score: 0.0)\n",
            "Document #708: ‘ Genome-wide surveys based on a quadruplex folding rule have been performed, which have identified ,000 Putative Quadruplex Sequences (PQS) in the human genome, although not all of these probably form in vivo.’ (score: 0.0)\n",
            "Document #707: ‘ Other genes shown to form G-quadruplexes in their promoter regions include the chicken -globin gene, human ubiquitin­ ligase RFP, and the proto-oncogenes c-kit, bcl-, VEGF, H-ras and N-ras.’ (score: 0.0)\n",
            "Document #731: ‘When DNA undergoes oxidative damage, a possible structural change in guanine, after ionizing radiation, gives rise to an enol form, -0H-Gua.’ (score: 0.0)\n",
            "Document #732: ‘This oxidative product is formed through a tautomeric shift from the original damage guanine, -oxo-Gua, and represents DNA damage that causes changes in the structure.’ (score: 0.0)\n",
            "Document #733: ‘This form allows for the base excision repair (BER) enzyme OGG to bind and remove the oxidative damage with the help of APE, resulting in an AP site.’ (score: 0.0)\n",
            "Document #734: ‘ Moreover, an AP site is a location in DNA that has neither a purine or a pyrimidine base due to DNA damage, they are the most prevalent type of endogenous DNA damage in cells.’ (score: 0.0)\n",
            "Document #757: ‘Some of these oncogenes include c-KIT, PDGF-A, c-Myc and VEGF, showing the importance of this secondary structure in cancer growth and development.’ (score: 0.0)\n",
            "Document #756: ‘The structures most present in the promoter regions of these oncogenes tend to be parallel-stranded G-quadruplex DNA structures.’ (score: 0.0)\n",
            "Document #755: ‘Along with the association of G-quadruplexes in telomeric regions of DNA, G-quadruplex structures have been identified in various human proto-oncogene promoter regions.’ (score: 0.0)\n",
            "Document #754: ‘For therapeutic advances, stabilizing the G­ quadruplexes of cancerous cells can inhibit cell growth and replication leading to the cell's death.’ (score: 0.0)\n",
            "Document #753: ‘This causes replication of damaged and cancerous cells.’ (score: 0.0)\n",
            "Document #752: ‘However, in cancer cells that have mutated helicase these complexes cannot be unwound and leads to potential damage of the cell.’ (score: 0.0)\n",
            "Document #751: ‘G-quadruplexes can inhibit normal cell function, and in healthy cells, are easily and readily unwound by helicase.’ (score: 0.0)\n",
            "Document #750: ‘Therefore, we can conclude that APE has two important roles in genome regulation- Stabilizing the formation of g-quadruplex structures and loading the transcriptional factors onto the AP site G-quadruplex forming sequences are prevalent in eukaryotic cells, especially in telomeres,  untranslated strands, and translocation hot spots.’ (score: 0.0)\n",
            "Document #749: ‘However, not all G-quadruplex structures require APE for formation, in fact some of them formed greater G-quadruplex structures in its absence.’ (score: 0.0)\n",
            "Document #748: ‘When a study downregulated the presence of APE and AcAPE in the cell, the formation of G-quadruplex structures was inhibited, which proves the importance of APE for the formation of these structures.’ (score: 0.0)\n",
            "Document #747: ‘Deacetylation of AcAPE is the driving force behind the loading of these TFs, where APE dissociates from the G-quadruplex structures.’ (score: 0.0)\n",
            "Document #746: ‘After this stage the APE gets acetylated by multiple lysine residues on the chromatin, forming acetylated APE (AcAPE). AcAPE is very crucial to the BER pathway as it acts as a transcriptional coactivator or corepressor, functioning to load transcription factors (TF) into the site of damage allowing it to regulate the gene expression.  AcAPE is also very important since it allows APE to bind for longer periods of time by delay of its dissociation from the sequence, allowing the repair process to be more efficient.’ (score: 0.0)\n",
            "Document #745: ‘This promotes formation of G-quadruplex structures by the folding of the stand.0 This looping process brings four bases in close proximity that will be held together by Hoogsteen base pairing.’ (score: 0.0)\n",
            "Document #744: ‘This AP site then signals cells to engage APE binding, which binds to the open duplex region.SThe binding of APE then plays an important role by stabilizing the formation of G-quadruplex structures in that region.’ (score: 0.0)\n",
            "Document #743: ‘Once the cell is aware of oxidative stress and damage, it recruits OGG to the site, whose main function is to initiate the BER pathway.OGG does so by cleaving the oxidized base and thus creating an AP site, primarily through the process of negative superhelicity.’ (score: 0.0)\n",
            "Document #742: ‘The oxidation of certain purine bases, like guanine, forms oxidized nucleotides that impairs DNA function by mismatching nucleotides in the sequences.This is more common in PQS sequences which form oxidized structures, such as - oxoguanine.’ (score: 0.0)\n",
            "Document #741: ‘APE is considered to be very crucial as AP site damage is known to be the most recurring type of endogenous damage to DNA.’ (score: 0.0)\n",
            "Document #740: ‘APE is mainly in charge of repairing damage caused to AP sites through the BER pathway.’ (score: 0.0)\n",
            "Document #739: ‘AP endonuclease  (APE) is an enzyme responsible for the promotion and the formation of G­ quadruplex structures.’ (score: 0.0)\n",
            "Document #738: ‘Upon insertion of -oxo-dG into thymidine kinase gene of humans, it was determine that if -oxo-dG was left unchecked and not repaired by BER, it can lead to frequent mutations and eventually carcinogenesis.’ (score: 0.0)\n",
            "Document #737: ‘The possible inactivation allows for un-repaired DNA damages to gather in non-replicating cells, like muscle, and can cause aging as well.  Moreover, oxidative DNA damage like -oxo-dG contributes to carcinogenesis through the modulation of gene expression, or the induction of mutations. On the condition that -oxo-dG is repaired by BER, parts of the repair protein is left behind which can lead to epigenetic alterations, or the modulation of gene expression.’ (score: 0.0)\n",
            "Document #736: ‘With the use of genome-wide ChIP-sequencing analyses, cell-based assays, and in vitro biochemical analyses, a connection has been made between oxidized DNA base-derived AP sites, and the formation of the G-quadruplex.  Furthermore, the concentration of -oxo-dG is a known biomarker of oxidative stress within a cell, and excessive amount of oxidative stress has been linked to carcinogenesis and other diseases_SWhen produced, -oxo-dG, has the ability to inactivate OGG, thus preventing the repair of DNA damage caused by the oxidation of guanine.’ (score: 0.0)\n",
            "Document #735: ‘AP sites can be generated spontaneously or after the cleavage of modified bases, like -0H-Gua.iJThe generation of an AP site enables the melting of the duplex DNA to unmask the PQS, adopting aG-quadruplex fold.’ (score: 0.0)\n",
            "Document #706: ‘The proto-oncogene c-myc forms a quadruplex in a nuclease hypersensitive region critical for gene activity.’ (score: 0.0)\n",
            "Document #705: ‘In addition, there was found more than one-billion-year conserved G-quadruplex locus in plants and algae, in gene encoding large subunit of RNA polymerase _i Although these studies predicted G-quadruplex-mediated gene regulation, it is unlikely that all pGs would form in vivo.’ (score: 0.0)\n",
            "Document #704: ‘Here also, like vertebrates G-quadruplexes were enriched within gene promoters.’ (score: 0.0)\n",
            "Document #677: ‘The hydrogen bonds in these layers are represented by blue dashed lines.’ (score: 0.0)\n",
            "Document #675: ‘The backbone is represented by a tube.’ (score: 0.0)\n",
            "Document #674: ‘These structures, formed from two separate G-D Structure of the intramolecular human telomeric G-quadruplex in potassium solution.’ (score: 0.0)\n",
            "Document #673: ‘Longer sequences, which contain two contiguous runs of three or more guanine bases, where the guanine regions are separated by one or more bases, only require two such sequences to provide enough guanine bases to form a quadruplex.’ (score: 0.0)\n",
            "Document #672: ‘The term G DNA was originally reserved for these tetramolecular structures that might play a role in meiosis However, as currently used in molecular biology, the term G can mean G-quadruplexes of any molecularity.’ (score: 0.0)\n",
            "Document #671: ‘Such a quadruplex is described as tetramolecular, reflecting the requirement of four separate strands.’ (score: 0.0)\n",
            "Document #670: ‘Short sequences, consisting of only a single contiguous run of three or more guanine bases, require four individual strands to form a quadruplex.’ (score: 0.0)\n",
            "Document #669: ‘The length of the nucleic acid sequences involved in tetrad formation determines how the quadruplex folds.’ (score: 0.0)\n",
            "Document #668: ‘Early evidence for the formation of G­ quadruplexes in vivo in cells was established by isolating them from cells, and later by the observation that specific DNA helicases could be identified where small molecules specific for these DNA structures accumulated in cells.’ (score: 0.0)\n",
            "Document #667: ‘Current research consists of identifying the biological function of these G­ Quadruplex structures for specific oncogenes and discovering effective therapeutic treatments for cancer based on interactions with G-quadruplexes.’ (score: 0.0)\n",
            "Document #666: ‘With the abundance of G-quadruplexes in vivo, these structures hold a biologically relevant role through interactions with the promoter regions of oncogenes and the telomeric regions of DNA strands.’ (score: 0.0)\n",
            "Document #665: ‘In 00, the prevalence of G-quadruplexes within gene promoters of several bacterial genomes was reported predicting G-quadruplex-mediated gene regulation.’ (score: 0.0)\n",
            "Document #664: ‘Interest in in vivo function of G-quadruplexes surged after large scale genome­ wide analysis showed the prevalence of potential G-quadruplex (pG)-forming sequences within gene promoters of human, chimpanzee, mouse, and rat - presented in the First International G-quadruplex Meeting held in April 00 in Louisville, Kentucky.’ (score: 0.0)\n",
            "Document #663: ‘The importance of discovering G-quadruplex structure was described through the statement, \"If G-quadruplexes form so readily in vitro, Nature will have found a way of using them in vivo\" - Aaron Klug, Nobel Prize Winner in Chemistry ().’ (score: 0.0)\n",
            "Document #662: ‘Their activities in basic genetic processes are an active area of research in telomere, gene regulation, and functional genomics research. The identification of structures with a high guanine association became apparent in the early 0s, through the identification of gel-like substances associated with guanines. More specifically, this research detailed the four-stranded DNA structures with a high association of guanines, which was later identified in eukaryotic telomeric regions of DNA.’ (score: 0.0)\n",
            "Document #661: ‘G-quadruplex structures can be computationally predicted from DNA or RNA sequence motifs, but their actual structures can be quite varied within and between the motifs, which can number over 00,000 per genome.’ (score: 0.0)\n",
            "Document #660: ‘They can be formed of DNA, RNA, LNA, and PNA, and may be intramolecular, bimolecular, or tetramolecular. Depending on the direction of the strands or parts of a strand that form the tetrads, structures may be described as parallel or antiparallel.’ (score: 0.0)\n",
            "Document #659: ‘The quadruplex structure is further stabilized by the presence of a cation, especially potassium, which sits in a central channel between each pair of tetrads.’ (score: 0.0)\n",
            "Document #658: ‘The placement and bonding to form G-quadruplexes is not random and serve very unusual functional purposes.’ (score: 0.0)\n",
            "Document #657: ‘Four guanine bases can associate through Hoogsteen hydrogen bonding to form a square planar structure called a guanine tetrad (G-tetrad or G-quartet), and two or more guanine tetrads (from G-tracts, continuous runs of guanine) can stack on top of each other to form a G-quadruplex.’ (score: 0.0)\n",
            "Document #656: ‘They are helical in shape and contain guanine tetrads that can form from one, two or four strands The unimolecular forms often occur naturally near the ends of the chromosomes, better known as the telomeric regions, and in transcriptional regulatory regions of multiple genes, both  in  microbes  and  across vertebrates SJincluding oncogenes in humans.’ (score: 0.0)\n",
            "Document #655: In molecular biology, G-quadruplex secondary structures (G) are formed in nucleic acids by sequences that are rich in guanine.’ (score: 0.0)\n",
            "Document #654: Viruses can be placed in one of the seven following groups including the dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses), ssDNA viruses (+ strand or \"sense\") DNA (e.g. Parvoviruses), dsRNA viruses (e.g. Reoviruses), (+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses) V, or ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses) VII.' (score: 0.0)\n",
            "Document #653: Defective interfering particles are defective viruses that have lost their ability to replicate except in the presence of a helper virus, which is normally the parental virus. They can also interfere with the helper virus.' (score: 0.0)\n",
            "Document #676: ‘The center of this structure contains three layers of G-tetrads.’ (score: 0.0)\n",
            "Document #678: ‘ Guanine-rich strands, are termed bimolecular quadruplexes.’ (score: 0.0)\n",
            "Document #703: ‘coli and hundreds of other microbial genomes.’ (score: 0.0)\n",
            "Document #679: ‘Finally, sequences which contain four distinct runs of guanine bases can form stable quadruplex structures by themselves, and a quadruplex formed entirely from a single strand is called an intramolecular quadruplex.’ (score: 0.0)\n",
            "Document #702: ‘Similarly, large number of G­ quadruplexes were found in the E.’ (score: 0.0)\n",
            "Document #701: ‘A large number of the non-telomeric G-quadruplexes were found within gene promoters, and were conserved across the species.’ (score: 0.0)\n",
            "Document #700: ‘Analysis of human, chimpanzee, mouse and rat genomes showed enormous number of potential G-quadruplex (pG)-forming sequences in non-telomeric regions.’ (score: 0.0)\n",
            "Document #699: ‘Quadruplexes are present in locations other than at the telomere.’ (score: 0.0)\n",
            "Document #698: ‘This is an active target of drug discovery, including telomestatin.’ (score: 0.0)\n",
            "Document #697: ‘The presence of these structures prevent telomere elongation by the telomerase complex.  Telomeric repeats in a variety of organisms have been shown to form these quadruplex structures in vitro, and subsequently they have also been shown to form in vivo. The human telomeric repeat (which is the same for all vertebrates) consists of many repeats of the sequenced (GGTTAG), and the quadruplexes formed by this structure can be in bead-like structures of  nm to  nm in size and have been well studied by NMR, TEM and X-ray crystal structure determination. The formation of these quadruplexes in telomeres has been shown to decrease the activity of the enzyme telomerase, which is responsible for maintaining length of telomeres and is involved in around % of all cancers.’ (score: 0.0)\n",
            "Document #696: ‘At these’ end protrusions, the G-rich overhang can form secondary structures such as G-quadruplexes if the overhang is longer than four TTAGGG repeats.’ (score: 0.0)\n",
            "Document #695: ‘The heterodimeric complex ribonucleoprotein enzyme telomerase adds TTAGGG repeats at the’ end of DNA strands.’ (score: 0.0)\n",
            "Document #694: ‘The repeats end with a’ protrusion of between 0 and 0 single-stranded TTAGGG repeats.’ (score: 0.0)\n",
            "Document #693: ‘These telomeric regions are characterized by long regions of double-stranded CCCTAA:TTAGGG repeats.’ (score: 0.0)\n",
            "Document #692: ‘Telomeres, rich in guanine and with a propensity to form g-quadruplexes, are located at the terminal ends of chromosomes and help maintain genome integrity by protecting these vulnerable terminal ends from instability.’ (score: 0.0)\n",
            "Document #691: ‘Telomeres function to provide this signaling.’ (score: 0.0)\n",
            "Document #690: ‘Because repair enzymes would naturally recognize ends of linear chromosomes as damaged DNA and would process them as such to harmful effect for the cell, clear signaling and tight regulation is needed at the ends of linear chromosomes.’ (score: 0.0)\n",
            "Document #689: ‘In addition, the propensity of g­ quadruplex formation during transcription in RNA sequences with the potential to form mutually exclusive hairpin or G-quadruplex structures depends heavily on the position of the hairpin-forming sequence.’ (score: 0.0)\n",
            "Document #688: ‘This effect seems to be mediated by alteration of the hydration of the DNA and its effect on Hoogsteen base pair bonding These quadruplexes seemed to readily occur at the ends of chromosome.’ (score: 0.0)\n",
            "Document #687: ‘For instance, quantitative assessments of the thermodynamics of molecular crowding indicate that the antiparallel g-quadruplex is stabilized by molecular crowding.’ (score: 0.0)\n",
            "Document #686: ‘Depending on cell type and cell cycle, mediating factors such as DNA-binding proteins on chromatin, composed of DNA tightly wound around histone proteins, and other environmental conditions and stresses affect the dynamic formation of quadruplexes.’ (score: 0.0)\n",
            "Document #685: ‘Following sequencing of the human genome, many guanine-rich sequences that had the potential to form quadruplexes were discovered.’ (score: 0.0)\n",
            "Document #684: ‘In quadruplexes formed from double-stranded DNA, possible interstrand topologies have also been discussed.  Interstrand quadruplexes contain guanines that originate from both strands of dsDNA.’ (score: 0.0)\n",
            "Document #683: ‘The loops joining runs of guanine bases in intramolecular antiparallel quadruplexes are either diagonal, joining two diagonally opposite runs of guanine bases, or lateral (edgewise) type loops, joining two adjacent runs of guanine base pairs.’ (score: 0.0)\n",
            "Document #682: ‘If one or more of the runs of guanine bases has a’-' direction opposite to the other runs of guanine bases, the quadruplex is said to have adopted an antiparallel topology.’ (score: 0.0)\n",
            "Document #681: ‘For intramolecular quadruplexes, this means that any loop regions present must be of the propeller type, positioned to the sides of the quadruplex.’ (score: 0.0)\n",
            "Document #680: ‘Depending on how the individual runs of guanine bases are arranged in a bimolecular or intramolecular quadruplex, a quadruplex can adopt one of a number of topologies with varying loop configurations. If all strands of DNA proceed in the same direction, the quadruplex is termed parallel.’ (score: 0.0)\n",
            "Document #758: ‘While the formation of G­ quadruplex structure vary to some extent for the different promoter regions of oncogenes, the consistent stabilization of these structures have been found in cancer development.’ (score: 0.0)\n",
            "Document #759: ‘Current therapeutic research actively focuses on targeting this stabilization of G-quadruplex structures to arrest unregulated cell growth and division.’ (score: 0.0)\n",
            "Document #760: ‘One particular gene region, the c-myc pathway, plays an integral role in the regulation of a protein product, c-Myc.’ (score: 0.0)\n",
            "Document #840: ‘Small-molecule ligands, composed primarily of lead, can target GGGGCC repeat regions as well and ultimately decreased both repeat-associated non-ATG translation and RNA foci in neuron cells derived from patients with Amyotrophic lateral sclerosis (ALS).’ (score: 0.0)\n",
            "Document #838: ‘A disadvantage of using small-molecule ligands as a therapeutic technique is that specificity is difficult to manage due to the variability of G-quadruplexes in their primary sequences, orientation, thermodynamic stability, and nucleic acid strand stoichiometry.’ (score: 0.0)\n",
            "Document #837: ‘Approximately ,000 various G­ quadruplex ligands exist in which they are able to interact via their aromatic rings; this allows the small-molecule ligands to stack on the planar terminal tetrads within the G-quadruplex regions.’ (score: 0.0)\n",
            "Document #836: ‘These can be used to target G-quadruplex regions that cause neurological disorders.’ (score: 0.0)\n",
            "Document #835: ‘Decoys have been successfully demonstrated to inhibit oncogenic KRAS in SCIO mice leading to reduced tumour growth and increased median survival time. Another commonly used technique is the utilization of small-molecule ligands.’ (score: 0.0)\n",
            "Document #834: ‘When introduced to cancer cells the decoy can intercept associated transcription factors and bind them leading to the regulation of gene expression.’ (score: 0.0)\n",
            "Document #833: ‘These decoys are typically composed of a G-rich sequence that can form a stable G-quadruplex structure and a short linker region that can be modified to optimize their properties.’ (score: 0.0)\n",
            "Document #832: ‘The strategy involves designing synthetic oligonucleotides that mimic the G-quadruplex structure and compete with the endogenous G-quadruplexes for binding to transcription factors.’ (score: 0.0)\n",
            "Document #831: ‘Antisense oligonucleotides (ASOs) are commonly used to target Corf RNA of the G-quadruplex GGGGCC expansion repeat region, which has lowered the toxicity in cellular models of Corf.0sJ00 ASOs have previously been used to restore normal phenotypes in other neurological diseases that have gain-of-function mechanisms, the only difference is that it was used in the absence of G­ quadruplex expansion repeat regions.0J0JoJThe G-quadruplex decoy strategy is another promising approach for targeting cancer cells by exploiting the unique structural features of the G-quadruplex.’ (score: 0.0)\n",
            "Document #830: ‘Therefore, these techniques are especially advantageous for targeting neurological diseases that have a gain-of-function mechanism, which is when the altered gene product has a new function or new expression of a gene; this has been detected in the Corf (chromosome  open reading frame ).0Antisense therapy is the process by which synthesized strands of nucleic acids are used to bind directly and specifically to the mRNA produced by a certain gene, which will inactivate it.’ (score: 0.0)\n",
            "Document #829: ‘This repeat expansion promotes DNA methylation and other epigenetic heterochromatin modifications of FMR that prevent the transcription of the gene, leading to pathological low levels of FMRP.0J0Antisense-mediated interventions and small-molecule ligands are common strategies used to target neurological diseases linked to G-quadruplex expansion repeats.’ (score: 0.0)\n",
            "Document #828: ‘ FMRP acts as a negative regulator of translation, and its binding stabilizes G-quadruplex structures in mRNA transcripts, inhibiting ribosome elongation of mRNA in the neuron's dendrite and controlling the timing of the transcript's expression.0o Mutations of this gene can cause the development of Fragile X Syndrome, autism, and other neurological disorders.0 Specifically, Fragile X Syndrome is caused by an increase from 0 to over 00 CGG repeats within exon  of the FMR gene.’ (score: 0.0)\n",
            "Document #827: ‘Fragile X mental retardation protein (FMRP) is a widely expressed protein coded by the FMR gene that binds to G-quadruplex secondary structures in neurons and is involved in synaptic plasticity.’ (score: 0.0)\n",
            "Document #826: ‘Nucleolin is involved in the synthesis and maturation of ribosomes within the nucleus, and separation of nucleolin by the mutated RNA transcripts impairs nucleolar function and ribosomal RNA synthesis.’ (score: 0.0)\n",
            "Document #825: ‘Normal individuals typically have around  to  GC repeats, but individuals with FTD or ALS have from 00 to several thousand GC repeats_The transcribed RNA of these repeats have been shown to form stable G-quadruplexes, with evidence showing that the GC repeats in DNA have the ability to form mixed parallel-antiparallel G- quadruplex structures as well_ These RNA transcripts containing GC repeats were shown to bind and separate a wide variety of proteins, including nucleolin.’ (score: 0.0)\n",
            "Document #824: ‘These two diseases have a causal relationship to GGGGCC (GC) repeats within the st intron of Corf gene.’ (score: 0.0)\n",
            "Document #823: ‘The second mechanism is through mutations that affect the expression of G-quadruplex binding proteins, as seen in the fragile X mental retardation gene  (FMR) gene and Fragile X Syndrome. The Corf gene codes for the protein Corf which is found throughout the brain in neuronal cytoplasm and at presynaptic terminals. Mutations of the Corf gene have been linked to the development of FTD and ALS.’ (score: 0.0)\n",
            "Document #822: ‘The first is through expansions of G-repeats within genes that lead to the formation of G-quadruplex structures that directly cause disease, as is the case with the Corf gene and amyotrophic lateral sclerosis (ALS) or frontotemporal dementia (FTD).’ (score: 0.0)\n",
            "Document #821: ‘G-quadruplexes have been implicated in neurological disorders through two main mechanisms.’ (score: 0.0)\n",
            "Document #820: ‘The novel approach combines solid-state nanopores and DNA nanotechnology for label-free detection of G-quadruplexes, for their mapping on dsDNA, and for monitoring G-quadruplex formation.’ (score: 0.0)\n",
            "Document #819: ‘Firstly, it was shown that biological nanopores can detect G-quadruplexes based on size exclusion and specific interaction of G-quadruplex and protein nanocavity.’ (score: 0.0)\n",
            "Document #818: ‘Another approach for detection of G-quadruplexes includes nanopore-based methods.’ (score: 0.0)\n",
            "Document #817: ‘Likewise, the thermostability of the G-quadruplex structure can be identified by observing the UV signal at  nm_ Upon G-quadruplex melting, the UV absorbance at  nm decreases, leading to a hypochromic shift that is a distinctive feature of G-quadruplex structure.’ (score: 0.0)\n",
            "Document #816: ‘To verify G-quadruplex formation, one should also perform the CD experiments under non-G-quadruplex stabilizing (Li+) and G-quadruplex stabilizing conditions (such as K+ or with G-quadruplex ligands), and scan toward the far-UV region (0- 0 nm).’ (score: 0.0)\n",
            "Document #839: ‘As of now, no single small-molecule ligand has been able to be perfectly specific for a single G-quadruplex sequence.  However, a cationic porphyrin known as TMPyP is able to bind to the Corf GGGGCC repeat region, which causes the G-quadruplex repeat region to unfold and lose its interactions with proteins causing it to lose its functionality.’ (score: 0.0)\n",
            "Document #841: ‘This provides evidence that small-molecule ligands are an effective and efficient process to target GGGGCC regions, and that specificity for small-molecule ligand binding is a feasible goal for the scientific community.’ (score: 0.0)\n",
            "Document #814: ‘The topology of the G-quadruplex structure can be determined by monitoring the positive or negative circular dichroism (CD) signals at specific wavelengths.’ (score: 0.0)\n",
            "Document #842: ‘Metal complexes have a number of features that make them particularly suitable as G DNA binders and therefore as potential drugs.’ (score: 0.0)\n",
            "Document #865: ‘These similarities have medical applications in terms of studying the structure and functions of both immune proteins and stress proteins, as well as the role each plays in combating disease.’ (score: 0.0)\n",
            "Document #864: ‘In addition, many stress proteins overlap with immune proteins.’ (score: 0.0)\n",
            "Document #863: ‘Early research has suggested that cells which are better able to synthesize stress proteins and do so at the appropriate time are better able to withstand damage caused by ischemia and reperfusion.’ (score: 0.0)\n",
            "Document #862: ‘Other toxins that either directly or indirectly lead to the release of free radicals can generate misfolded proteins.’ (score: 0.0)\n",
            "Document #861: ‘For example, many heavy metals can react with sulthydryl groups stabilizing proteins,  resulting  in  conformational changes.’ (score: 0.0)\n",
            "Document #860: ‘Many toxins end up activating similar stress proteins to heat or other stress-induced pathways because it is fairly common for some types of toxins to achieve their effects - at least in part - by denaturing vital cellular proteins.’ (score: 0.0)\n",
            "Document #859: ‘Heat shock proteins can also participate in marking malformed proteins for degradation via ubiquitin tags.’ (score: 0.0)\n",
            "Document #858: ‘This role is especially crucial since elevated temperature would, on its own, increase the concentrations of malformed proteins.’ (score: 0.0)\n",
            "Document #857: ‘These can help defend a cell against damage by acting as 'chaperons' in protein folding, ensuring that proteins assume their necessary shape and do not become denatured.’ (score: 0.0)\n",
            "Document #856: The heat shock response involves a class of stress proteins called heat shock proteins.’ (score: 0.0)\n",
            "Document #855: ‘Indeed, if a cell under normal (meaning unstressed) conditions has denatured proteins artificially injected into it, it will trigger a stress response.’ (score: 0.0)\n",
            "Document #854: ‘Because conditions such as high temperatures often cause proteins to denature, this mechanism enables cells to determine when they are subject to high temperature without the need of specialized thermosensitive proteins.’ (score: 0.0)\n",
            "Document #853: ‘Although stress response pathways are mediated in different ways depending on the stressor involved, cell type, etc., a general characteristic of many pathways - especially ones where heat is the principal stressor - is that they are initiated by the presence and detection of denatured proteins.’ (score: 0.0)\n",
            "Document #852: ‘As expected, such cells were also highly vulnerable to stress, and ceased to be viable at elevated temperature ranges.’ (score: 0.0)\n",
            "Document #851: ‘For example, studies in Drosophila have indicated that when DNA encoding certain stress proteins exhibit mutation defects, the resulting cells have impaired or lost abilities such as normal mitotic division and proteasome-mediated protein degradation.’ (score: 0.0)\n",
            "Document #850: ‘Stress proteins can exhibit widely varied functions within a cell- both during normal life processes and in response to stress.’ (score: 0.0)\n",
            "Document #849: ‘The essential character of these stress proteins in promoting the survival of cells has contributed to them being remarkably well conserved across phyla, with nearly identical stress proteins being expressed in the simplest prokaryotic cells as well as the most complex eukaryotic ones.’ (score: 0.0)\n",
            "Document #848: ‘Stress proteins often are further subdivided into two general categories: those that only are activated by stress, or those that are involved both in stress responses and in normal cellular functioning.’ (score: 0.0)\n",
            "Document #847: ‘Cellular stress responses are primarily mediated through what are classified as stress proteins.’ (score: 0.0)\n",
            "Document #846: ‘The various processes involved in cellular stress responses serve the adaptive purpose of protecting a cell against unfavorable environmental conditions, both through short term mechanisms that minimize acute damage to the cell's overall integrity, and through longer term mechanisms which provide the cell a measure of resiliency against similar adverse conditions.’ (score: 0.0)\n",
            "Document #845: ‘Cellular stress responses can also be caused by some viral infections.’ (score: 0.0)\n",
            "Document #844: ‘Cellular stress response is the wide range of molecular changes that cells undergo in response to environmental stressors, including extremes of temperature, exposure to toxins, and mechanical damage.’ (score: 0.0)\n",
            "Document #843: ‘While the metal plays largely a structural role in most G binders, there are also examples where it interacts directly with Gs by electrostatic interactions or direct coordination with nucleobases. (score: 0.0)\n",
            "Document #815: ‘Parallel G-quadruplexes have negative and positive CD signals at 0 and  nm, respectively, whereas antiparallel G-quadruplexes place these signals at  and  nm, respectively.’ (score: 0.0)\n",
            "Document #813: ‘In the DNA polymerase stop assay, the formation of a G-quadruplex in a DNA template can act as a roadblock and cause polymerase stalling, which halts the primer extension.  The dimethyl sulfate (DMS) followed by the piperidine cleavage assay is based on the fact that the formation of a G­ quadruplex will prohibit the N guanine methylation caused by DMS, leading to a protection pattern observed at the DNA G-quadruplex region after piperidine cleavage.’ (score: 0.0)\n",
            "Document #761: ‘With this product, the c-Myc protein functions in the processes of apoptosis and cell growth or development and as a transcriptional control on human telomerase reverse transcriptase.’ (score: 0.0)\n",
            "Document #786: ‘This main pathway of targeting this region results in the lack of telomerase elongation, leading to arrested cell development.’ (score: 0.0)\n",
            "Document #784: ‘To address this issue analog of TMPyP, it was synthesized known as Me which targets only G quadruplex DNA which inhibits cancer growth more effectively than TMPyP_ Ligand design and development remains an important field of research into therapeutic reagents due to the abundance of G-quadruplexes and their multiple conformational differences.’ (score: 0.0)\n",
            "Document #783: ‘However, TMPyP has been limited for used due to its non-selectivity toward cancer cell telomeres and normal double stranded DNA (dsDNA).’ (score: 0.0)\n",
            "Document #782: ‘These complexes have a high affinity for porphyrin rings which makes them effective anticancer agents.’ (score: 0.0)\n",
            "Document #781: ‘Telomeres are generally made up of G-quadruplexes and remain important targets for therapeutic research and discoveries.’ (score: 0.0)\n",
            "Document #780: ‘However, research has also identified the presence of G-quadruplex structures within this region due to the interaction of TMPyP with this promoter sequence.’ (score: 0.0)\n",
            "Document #779: ‘The presence of a guanine-rich sequence in the promoter region of PDGF-A has exhibited the ability to form intramolecular parallel G­ quadruplex structures and remains suggested to play a role in transcriptional regulation of PDGF-A.’ (score: 0.0)\n",
            "Document #778: ‘High levels of expression of PDGF have been associated with increased cell growth and cancer.’ (score: 0.0)\n",
            "Document #777: ‘Another oncogene pathway involving PDGF-A, platelet-derived growth factor, involves the process of wound healing and function as mitogenic growth factors for cells.’ (score: 0.0)\n",
            "Document #776: ‘The research on this pathway suggested the formation of a G­ quadruplex in the promoter region and an applicable target for therapeutic treatments.’ (score: 0.0)\n",
            "Document #775: ‘In certain types of cancers, the RET protein has shown increased expression levels.’ (score: 0.0)\n",
            "Document #774: ‘The guanine rich sequence in the promoter region for this pathway exudes a necessity for baseline transcription of this receptor tyrosine kinase.’ (score: 0.0)\n",
            "Document #773: ‘ The RET oncogene functions in the transcription of kinase which has been abundant in certain types of cancer.’ (score: 0.0)\n",
            "Document #772: ‘Current research on this pathway is focusing on discovering the biological function of this specific quadruplex formation on the c-kit pathway, while this quadruplex sequence has been noticed in various species.’ (score: 0.0)\n",
            "Document #771: ‘The rich guanine sequence of this promoter region has shown the ability to form a variety of quadruplexes.’ (score: 0.0)\n",
            "Document #770: ‘However, more research is necessary to determine whether the formation of G-quadruplex regulates the expression of this gene in a positive or negative manner.i The c-kit oncogene deals with a pathway that encodes an RTK, which was shown to have elevated expression levels in certain types of cancer.’ (score: 0.0)\n",
            "Document #769: ‘Through recent research into this specific gene pathway, the polypurine and polypyrimidine region allows for the transcription of this specific gene and the formation of an intramolecular G-quadruplex structure.’ (score: 0.0)\n",
            "Document #768: ‘Interaction of this G-quadruplex with TRF (also known as TERF) resulted in epigenetic regulation of p, which was tested using the G-quadruplex-binding ligand A.oHypoxia inducible factor la, HIF-a, remains involved in cancer signaling through its binding to Hypoxia Response Element, HRE, in the presence of hypoxia to begin the process of angiogenesis.’ (score: 0.0)\n",
            "Document #767: ‘The cyclin-dependent cell cycle checkpoint kinase inhibitor- CDKNlA (also known as p) gene harbours promoter G-quadruplex.’ (score: 0.0)\n",
            "Document #766: ‘The intramolecular G-quadruplex structures are formed mostly through the abundant guanine sequence in the promoter region of this specific pathway.’ (score: 0.0)\n",
            "Document #765: ‘Through recent research on the role of G-quadruplex function in vivo, the stabilization of G-quadruplex structures was shown to regulate VEGF gene transcription, with inhibition of transcription factors in this pathway.’ (score: 0.0)\n",
            "Document #764: ‘The formation of an intramolecular G-quadruplex structure has been shown through studies on the polypurine tract of the promoter region of the VEGF gene.’ (score: 0.0)\n",
            "Document #763: ‘Another gene pathway deals with the VEGF gene, Vascular Endothelial Growth Factor, which remains involved in the process of angiogenesis or the formation of new blood vessels.’ (score: 0.0)\n",
            "Document #762: ‘Interaction of c-Myc promoter G-quadruplex with NMH was shown to regulate c-Myc in cancer cells in 00 Regulation of c-myc through Human telomerase reverse transcriptase (hTERT) is also directly regulated through promoter G-quadruplex by interaction with the transcription factor NMH where epigenetic modifications were dependent on NMH-G-quadruplex association.Recently, hTERT epigenetic regulation reported to be mediated through interaction of hTERT promoter G­ quadruplex with the telomeric factor TRF.’ (score: 0.0)\n",
            "Document #785: ‘One type ofligand involving a Quindoline derivative, SYUIQ-0, utilizes the stabilization of G-quadruplexes in promoter regions to inhibit the production of both the c-Myc protein product and the human telomerase reverse transcriptase (hTERT).’ (score: 0.0)\n",
            "Document #787: ‘Further research remains necessary for the discovery of a single gene target to minimize unwanted reactivity with more efficient antitumor activity.’ (score: 0.0)\n",
            "Document #812: ‘These methods can be broadly defined into two classes: biophysical and biochemical methods.  Biochemical techniques were employed to interrogate G-quadruplex formation in a longer sequence context.’ (score: 0.0)\n",
            "Document #788: ‘One way of inducing or stabilizing G-quadruplex formation is to introduce a molecule which can bind to the G-quadruplex structure.’ (score: 0.0)\n",
            "Document #811: ‘A number of experimental methods have been developed to identify G-quadruplexes.’ (score: 0.0)\n",
            "Document #810: ‘This suggests that the sequences may be under positive selection enabled by the evolution of systems capable of suppressing non-B structure formation.’ (score: 0.0)\n",
            "Document #809: ‘the frequency) of these motifs has increased rapidly in the eumetazoa for which complete genomic sequences are available.’ (score: 0.0)\n",
            "Document #808: ‘In one study, it was found that the observed number per base pair (i.e.’ (score: 0.0)\n",
            "Document #807: ‘Although the rule effectively identifies sites of G-quadruplex formation, it also identifies a subset of the imperfect homopurine mirror repeats capable of triplex formationand C-strand i-motif formation.0 Moreover, these sequences also have the capacity to form slipped and foldback structures that are implicit intermediates in the formation of both quadruplex and triplex DNA structures.’ (score: 0.0)\n",
            "Document #806: ‘This rule has been widely used in on-line algorithms.’ (score: 0.0)\n",
            "Document #805: ‘Generally, a simple pattern match is used for searching for possible intrastrand quadruplex forming sequences: d(G+N_G+N_G+N_G+), where N is any nucleotide base (including guanine).’ (score: 0.0)\n",
            "Document #804: ‘Identifying and predicting sequences which have the capacity to form quadruplexes is an important tool in further understanding their role.’ (score: 0.0)\n",
            "Document #803: ‘Ligands with mobile side chains, ones that are able to rotate around its center chromophore, associate more strongly to G-quadruplexes because conformation of the G loops and the ligand side chains can align.’ (score: 0.0)\n",
            "Document #802: ‘Furthermore, the hydrogen bonds of ligands with smaller side chains are shorter and therefore stronger.’ (score: 0.0)\n",
            "Document #801: ‘It has been found that ligands with smaller side chains bind better to the quadruplex because smaller ligands have more concentrated electron density.’ (score: 0.0)\n",
            "Document #800: ‘The way in which TMPyP binds to G's is similar to MM, with the ring stacking onto the external G-quartet and side chains associating to the loops of G's.? When designing ligands to be bound to G-quadruplexes, the ligands have a higher affinity for parallel folded G-quadruplexes.’ (score: 0.0)\n",
            "Document #799: ‘TMPyP, a cationic porphyrin, is a more well known G binding ligand that helps to repress c-Myc.’ (score: 0.0)\n",
            "Document #798: ‘What makes this binding strong is the fluidity in the position of the loops to better associate with the ligand side chains.’ (score: 0.0)\n",
            "Document #797: ‘The quartet and the chromophore are bound with a :rc-:rc bond while the side chains and loops are not bound but are in close proximity.’ (score: 0.0)\n",
            "Document #796: ‘When bound, MM's central chromophore is situated on top of the’ terminal G-quartet and the side chains of the ligand associate to the loops of the quadruplex.’ (score: 0.0)\n",
            "Document #795: ‘The shape of the ligand is vital because it closely matches the quadruplex which has stacked quartets and the loops of nucleic acids holding it together.’ (score: 0.0)\n",
            "Document #794: ‘MM, a ligand that binds selectively for a quadruplex on the BCL- promoter, is shaped with a central core and  side chains branching sterically out.’ (score: 0.0)\n",
            "Document #793: ‘The binding of ligands to G-quadruplexes is vital for anti-cancer pursuits because G-quadruplexes are found typically at translocation hot spots.’ (score: 0.0)\n",
            "Document #792: ‘Cationic porphyrins have been shown to bind intercalatively with G-quadruplexes, as well as the molecule telomestatin.’ (score: 0.0)\n",
            "Document #791: ‘This has become an increasingly large field of research in genetics, biochemistry, and pharmacology.’ (score: 0.0)\n",
            "Document #790: ‘These ligands can be naturally occurring or synthetic.’ (score: 0.0)\n",
            "Document #789: ‘A number of ligands, which can be both small molecules and proteins, can bind to the G-quadruplex.’ (score: 0.0)\n",
            "Document #652: virus group Satellite nucleic acids, Single-stranded satellite DNAs, Family Alphasatellitidae (encoding a replication initiator protein) Family Tolecusatellitidae (encoding a pathogenicity determinant βC1)and Double-stranded satellite RNAs Single-stranded satellite RNAs.' (score: 0.0)\n",
            "Document #650: 'Satellite viruses include the Single-stranded RNA satellite viruses (unnamed family), Aumaivirus – Maize white line mosaic satellite.' (score: 0.0)\n",
            "Document #435: ‘Conversely, the cytoskeleton seems unnecessary in virus-induced atypical SGs.’ (score: 0.0)\n",
            "Document #502: ‘Arrows: SG-positive cells.’ (score: 0.0)\n",
            "Document #513: ‘This complex pre¬vents N protein from self-assembling and, thus, favors nucleocapsid assembly and viral transcription (26).’ (score: 0.0)\n",
            "Document #512: ‘Likewise, Sendai virus PP forms complexes with unassembled N protein (N°).’ (score: 0.0)\n",
            "Document #511: ‘However, this binding is blocked when the N-protein is complexed with PP (NS) (50).’ (score: 0.0)\n",
            "Document #510: ‘The N-protein of VSV seems to bind to any available RNA species, irrespective of sequence.’ (score: 0.0)\n",
            "Document #509: ‘In paramyxoviruses, such as mumps, N-protein en¬capsulates full-length viral negative sense genomic and positive sense antigenomic RNAs to form the helical nucleocapsid template (25).’ (score: 0.0)\n",
            "Document #508: ‘In the course of RSV infection, viral inclusion bodies, which are marked by anti-N and -P anti-bodies, are spatially separated from SGs (22).’ (score: 0.0)\n",
            "Document #507: ‘In our study, NP partially localized in SGs, whereas almost no colocalization of PP and TIA-1 was observed, which is consistent with previous re¬ports.’ (score: 0.0)\n",
            "Document #506: ‘The line of evidence suggests that viral replication, but not viral protein, is essential for v-SGs.’ (score: 0.0)\n",
            "Document #505: ‘not trigger SG formation.’ (score: 0.0)\n",
            "Document #504: ‘Right: Intensity pro.le of linear region of interest (ROI) across the HeLa cell costained with SG markers and mRNA.’ (score: 0.0)\n",
            "Document #503: ‘Arrowheads: SG-negative cells.’ (score: 0.0)\n",
            "Document #501: ‘Anti-TIA-1 (A, B)and anti-TIAR(C, D) were used as an SG marker.’ (score: 0.0)\n",
            "Document #649: 'Satellite-like nucleic acids resemble satellite nucleic acids, in that they replicate with the aid of helper viruses. However they differ in that they can encode functions that can contribute to the success of their helper viruses; while they are sometimes considered to be genomic elements of their helper viruses, they are not always found within their helper viruses.' (score: 0.0)\n",
            "Document #500: ‘Probes speci.cally hybridized to polyadenylated mRNA (green) (A, C ), NP mRNA (red) (B, D)were used to testspeci.c components in SGs.’ (score: 0.0)\n",
            "Document #499: ‘At the indicated time points, cells were .xed and processed for FISH.’ (score: 0.0)\n",
            "Document #498: ‘HeLa cells were mock treated or infected with NDV at an MOI of 1 for 6, 12, and 18 h.’ (score: 0.0)\n",
            "Document #497: ‘Cellular, but not viral mRNA, is predominately recruited in NDV-induced SGs.’ (score: 0.0)\n",
            "Document #496: ‘Both NDV and RSV induce SG formation, and encode no C protein, suggesting that the C protein is a key player in determining the fate of SGs in Paramyxovirus infections.’ (score: 0.0)\n",
            "Document #495: ‘Our data demonstrate that NDV induced stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #494: ‘In respiratory syncytial virus (RSV), another member of the Paramyxoviridae family, SGs are triggered at 12 hpi and increase throughout the 24 h period of infection (22).’ (score: 0.0)\n",
            "Document #493: ‘It is suggested that MV and SeV encode C protein (46, 47), which limits the generation of intracellular double-stranded RNA and activation of PKR to inhibit eIF2a phosphoryla¬tion (48, 49).’ (score: 0.0)\n",
            "Document #492: ‘Measles virus (MV) and Sendai virus (SeV), 2 members of the Paramyxoviridae family, however, do not induce SGs during their infection.’ (score: 0.0)\n",
            "Document #491: To date, several viruses are known to modulate SG formation to promote their replication.’ (score: 0.0)\n",
            "Document #490: ‘Viral (score: 0.0)\n",
            "Document #514: ‘Besides, a recent report showed that NDV positive-but not negative-stranded RNA is located in SGs (51).’ (score: 0.0)\n",
            "Document #515: ‘In line with that, in our study, viral negative-stranded gRNA was not recruited to SGs.’ (score: 0.0)\n",
            "Document #516: ‘Therefore, we specu¬late that a fraction of the NDV NP was recruited to v-SGs through binding to cellular or viral positive-stranded RNAs, and additional unassembled NP° may form complexes with the PP and remains excluded from v-SGs.’ (score: 0.0)\n",
            "Document #517: ‘However, this hypothesis needs to be investi¬gated further.’ (score: 0.0)\n",
            "Document #540: 2) blockage of SGs using siTIA-1 inhibited viral protein synthesis and extracellular virus yields; 3)NDV-induced SGs predominantly recruited cellular mRNA, whereas large amounts of NDV mRNA were excluded; and 4)de¬pletion of endogenous TIA-1/R and TIAR reduced NDV mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #539: ‘Overall, we found that: 1) NDV induced SGs to inhibit global protein translation through eIF2a phosphorylation; (score: 0.0)\n",
            "Document #538: ‘TIA-1/R knockdown led to a decrease in viral protein synthesis and extracellular virus yields, especially at the middle stage of infection.’ (score: 0.0)\n",
            "Document #537: ‘Similarly, in our study, NDV triggers stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #536: ‘In these cases, the depletion of SG key compo¬nents, such as TIA-1, TIAR, or G3BP, results in decreased virus replication, indicating the proviral function of SG proteins.’ (score: 0.0)\n",
            "Document #535: ‘The symbiotic relationship between SG formation and virus infection suggests that SG is a beneficial factor rather than an intrinsic defense mechanism during infections of some viruses.’ (score: 0.0)\n",
            "Document #534: ‘In other words, the formation of SGs is accompanied by virus infection (18, 22).’ (score: 0.0)\n",
            "Document #533: ‘In contrast to this, HCV and RSV trigger stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #532: ‘Most viruses exploit several mechanisms to antagonize SG formation, such as G3BP cleavage and PKR inactivation (15, 16, 47).’ (score: 0.0)\n",
            "Document #531: Correspondingly, viruses use several mechanisms to evade or even take advantage of SGs.’ (score: 0.0)\n",
            "Document #530: ‘An earlier study showed that disruption of microtubules (score: 0.0)\n",
            "Document #529: ‘The function of microtubules in SG formation under physical and chemical stimuli varies in different reports.’ (score: 0.0)\n",
            "Document #528: ‘Furthermore, PKR/eIF2a pathwayplayedakeyroleinNDV-induced SGs.’ (score: 0.0)\n",
            "Document #527: We confirmed the association between the PKR/eIF2a pathway and v-SG formation, indicating that NDV in¬fection activated PKR/eIF2a pathway significantly and inhibited global translation.’ (score: 0.0)\n",
            "Document #526: ‘In this report, (score: 0.0)\n",
            "Document #525: ‘Viral cytoplasmic bodies induced by SV5 infection do not colocalize with cellular markers for SGs.’ (score: 0.0)\n",
            "Document #524: ‘Paramyxovirus simian virus 5 (SV5) is a weak ac¬tivator of the PKR/eIF2a signaling pathway and thereby does not globally inhibit translation (52).’ (score: 0.0)\n",
            "Document #523: ‘Moreover, TIA-1 was colocalized with G3BP1, rPS6, eIF4E, eIF4G, or eIF3A in both ARS-treated cells and NDV-infected cells, indicating that NDV triggers bona fide SGs.’ (score: 0.0)\n",
            "Document #522: ‘In our study, CHX treat¬ment caused the disassembly of NDV-induced SGs.’ (score: 0.0)\n",
            "Document #521: ‘At the late stage of poliovirus infection, virus-induced SGs contain TIA but lack translation initiation factors, mRNA binding proteins, and most polyadenylated mRNA (17).’ (score: 0.0)\n",
            "Document #520: ‘VSV-induced pseudo-SGs lack eIF3 and eIF4A (37).’ (score: 0.0)\n",
            "Document #519: ‘An E3 mutant vaccinia virus induces AVGs, which are atypical SGs that neither contain the 40S ribosomal subunit, nor disassemble when treated with polysome-stabilizing drugs such as CHX and emetine (20).’ (score: 0.0)\n",
            "Document #518: ‘The cytoplasmic granules induced by viral infec¬tions are distinct from typical SGs in both composition and function.’ (score: 0.0)\n",
            "Document #489: ‘FISH was used to detect NDV negative-strand genome RNA.’ (score: 0.0)\n",
            "Document #488: Because v-SGs regulate protein translation in the course of virus infection, we asked whether SGs play a role in virus genome replication.’ (score: 0.0)\n",
            "Document #487: ‘Depletion of endogenous TIA-1 does not affect virus genome replication (score: 0.0)\n",
            "Document #460: ‘Poliovirus-induced SGs are also stable over the course of infection (44).’ (score: 0.0)\n",
            "Document #458: ‘7, sixth column).’ (score: 0.0)\n",
            "Document #457: ‘The over¬lapping intensity profiles of the SG markers and mRNA (as visualized in the line profiles in the fifth column) further demonstrate the colocalization pro¬file between cellular mRNA and SG markers, whereas most viral mRNA was not localized with SG markers (Fig.’ (score: 0.0)\n",
            "Document #456: ‘However, large amounts of NP mRNA were distributed throughout the cytoplasm and excluded from-SGs (Fig.7B, D).’ (score: 0.0)\n",
            "Document #455: ‘At 12 and 18 hpi, a small portion of NP mRNA was localized with TIA-1 or TIAR.’ (score: 0.0)\n",
            "Document #454: ‘Meanwhile, no obvious SGs were observed in NP mRNA–positive cells at 6 hpi.’ (score: 0.0)\n",
            "Document #453: ‘NP mRNA was first detected in some cells infected for 6 h with NDV, compared with no positive fluorescence in mock-treated cells.’ (score: 0.0)\n",
            "Document #452: ‘To verify whether NDV mRNA is a component of SGs, we examined infected cells with FISH probes that hybridize to viral NP mRNA.’ (score: 0.0)\n",
            "Document #451: ‘Because Poly (A) tail has also been found on the messenger RNAs of NDV (43), v-SGs may contain both viral and host mRNA.’ (score: 0.0)\n",
            "Document #450: ‘In contrast, almost all polyadenylated mRNA was recruited to v-SGs at 12 and 18 hpi (arrow).’ (score: 0.0)\n",
            "Document #449: ‘diffusely dispersed in the cytoplasm (arrowhead).’ (score: 0.0)\n",
            "Document #448: ‘Arrows: transfected cells.’ (score: 0.0)\n",
            "Document #447: ‘The cells were collected, .xed, and visualized at 12 hpi.’ (score: 0.0)\n",
            "Document #446: ‘Cells were pulsed with 1 mM puromycin for 1 h at 37°C and then subjected to Western blot analysis.B) HeLa cells were transfected with PJX40-Flag-WT eIF2a, S51A, and S51D mutant eIF2a, followed by treatment at 24 h after transfection with infection of NDV at an MOI of 1.’ (score: 0.0)\n",
            "Document #445: ‘It was noted that in uninfected cells or cells containing no SG dots, polyadenylated mRNA was Phosphorylation of eIF2a at Ser-51 is required for v-SGs.A) HeLa cells were mock treated or infected with NDV at an MOI of 1.’ (score: 0.0)\n",
            "Document #444: ‘Polyadenylated mRNA was specifically recruited to virus-induced SGs marked by both TIA-1 and TIAR.’ (score: 0.0)\n",
            "Document #443: ‘FISH was used to detect total cellular mRNA, as well as viral NP mRNA.’ (score: 0.0)\n",
            "Document #442: Having found that v-SGs benefit the replication of NDV by manipulating the translation system, we checked whether the SG formation and stalled trans¬lation initiation were predominantly directed against viral or host protein translation.’ (score: 0.0)\n",
            "Document #441: ‘NDV-induced SGs contain predominantly cellular mRNA but not viral mRNA (score: 0.0)\n",
            "Document #440: ‘Noc treatment induced the formation of small and circular granules, compared to the large and irregular SGs in NDV-infected cells.’ (score: 0.0)\n",
            "Document #439: ‘In cells treated with Noc, obvious depolymerization of microtubules was observed, but the formation of NDV-induced SGs was unaffected.’ (score: 0.0)\n",
            "Document #438: ‘Accordingly, to confirm whether NDV-induced SGs were canonical SGs, we evaluated the role of microtubules and microfilaments in the formation of v-SGs.’ (score: 0.0)\n",
            "Document #437: ‘One characteristic of SG-like structures is that they are not dependent on a cellular microtubule or microfilament network.’ (score: 0.0)\n",
            "Document #436: ‘Previous reports showed that vesicular stomatitis virus (VSV)-induced SG-like structures are distinct from canonical SGs (37).’ (score: 0.0)\n",
            "Document #459: ‘Similarly, SGs in poliovirus infected cells are largely devoid of positive-sense poliovirus RNA, but contain polyadenylated RNA.’ (score: 0.0)\n",
            "Document #461: ‘Overall, these results indicate that NDV-induced SGs predominantly recruit cellular mRNA, whereas a large portion of viral mRNA seem to escape from v-SGs.’ (score: 0.0)\n",
            "Document #486: ‘The results indicate that G3BP1 is also necessary for NDV replication by trans¬lation regulation.’ (score: 0.0)\n",
            "Document #462: ‘Depletion of endogenous TIA-1 and TIAR reduced NP mRNA levels in polysomes (score: 0.0)\n",
            "Document #485: ‘9A, C).’ (score: 0.0)\n",
            "Document #484: ‘Similar to TIA-1 and TIAR, siG3BP1 resulted in the significant up-regulation of global translation levels (Figs.’ (score: 0.0)\n",
            "Document #483: ‘We next monitored global protein synthesis after NDV infection.’ (score: 0.0)\n",
            "Document #482: ‘In G3BP1-depleted cells infected with NDV, the expression of the viral NP and virus yields were significantly lower than in cells transfected with scrambled siRNAs, especially at 12 hpi.’ (score: 0.0)\n",
            "Document #481: ‘HeLa Cells were transfected with 2 siRNAs specific to G3BP1 and then infected with NDV.’ (score: 0.0)\n",
            "Document #480: ‘As expected, SG formation was ef¬ficiently inhibited in siG3BP1-transfected cells upon treatment .’ (score: 0.0)\n",
            "Document #479: ‘To verify whether the translation regulation is driven by v-SGs or TIA-1/R per se, we knocked down G3BP1, another key SG component, and evaluated virus rep¬lication efficiency.’ (score: 0.0)\n",
            "Document #478: ‘Depletion of endogenous G3BP1, another key SG component, also reduced NDV replication on HeLa cells.’ (score: 0.0)\n",
            "Document #477: ‘These results demonstrate that depletion of endogenous TIA-1 and TIAR reduces NP mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #476: ‘8C).’ (score: 0.0)\n",
            "Document #475: ‘However, lower NP mRNA levels in TIA-1/TIAR-knockdown cells, compared with that in scrambled cells, was detected in fractions 5–12 (Fig.’ (score: 0.0)\n",
            "Document #474: ‘The results showed that, in fractions 1–4, the NP mRNA level in siTIA-1/TIAR-treated cells was similar to that in control cells.’ (score: 0.0)\n",
            "Document #473: ‘To verify whether knockdown of endogenous TIA-1/TIAR influences the viral mRNA profile in individual fractions, NDV NP mRNA was detected by absolute real-time PCR quantification.’ (score: 0.0)\n",
            "Document #472: ‘8B).’ (score: 0.0)\n",
            "Document #471: ‘Similar to the ribosome profile results, the amounts of PABP and rPS6 proteins showed no sig¬nificant differences between siRNA and control groups (Fig.’ (score: 0.0)\n",
            "Document #470: ‘TIA-1, TIAR, and PABP, as typical SG markers, were all found mainly in monosomes in virus infection.’ (score: 0.0)\n",
            "Document #469: ‘Western blot analysis of individual fractions for ri¬bosomal protein S6 showed that compared with mock infection, under virus infection, rPS6 was detected in fractions 1–8, suggesting that small ribosomal protein tends to shift to lower density fractions that contain monosomes.’ (score: 0.0)\n",
            "Document #468: ‘8A).’ (score: 0.0)\n",
            "Document #467: ‘The ab¬sorbance of individual fractions distinguished mono¬somes (fractions 1–4) from polysomes (fractions 5–12) (Fig.’ (score: 0.0)\n",
            "Document #466: ‘To elucidate the mechanisms how TIA-1/TIAR regulate virus in¬fection, sucrose gradient analysis was performed to sepa¬rate monosomes and polysomes before viral mRNA quantification in TIA-1/TIAR-knockdown cells.’ (score: 0.0)\n",
            "Document #465: ‘Because v-SGs contain predominantly cellular mRNA, then we asked whether disruption of SGs would lead to the release of cellular mRNA that competitively binds to polysomes and thereby reduces viral mRNA translation.’ (score: 0.0)\n",
            "Document #464: ‘In addition to mRNA, SGs per se are highly dynamic domains containing mRNA accompanied by dynamic shuttling of TIA-1 and are in equilibrium with polysomes (35).’ (score: 0.0)\n",
            "Document #463: In tomato cells exposed to heat shock, mRNA transfers from polysomes to SGs, whereas mRNA moves from SGs to polysomes when cells recover from heat shock (45).’ (score: 0.0)\n",
            "Document #541: ‘Therefore, we propose that host cells respond to NDV infection by SG formation.’ (score: 0.0)\n",
            "Document #542: ‘The degradation of SGs by siTIA-1 leads to the release of cel¬lular mRNA from SGs to the cytoplasm.’ (score: 0.0)\n",
            "Document #543: ‘The released host mRNA competitively binds to polysomes and thereby re¬duces the translation ratio of viral mRNA, ultimately leading to a decrease of viral translation level and extra¬cellular virus yields.’ (score: 0.0)\n",
            "Document #623: 'Unlike the system of binomial nomenclature adopted in cellular species, there is currently no standardized form for virus species names. At present, the ICTV mandates that a species name must contain as few words as possible while remaining distinct, and must not only contain the word virus and the host name.[9] Species names often take the form of [Disease] virus, particularly for higher plants and animals. In 2019, the ICTV published a proposal to adopt a more formalized system of binomial nomenclature for virus species names, to be voted on in 2020.[10] However, some virologists later objected to the potential naming system change, arguing that the debate came while many in the field were preoccupied due to the COVID-19 pandemic.[11].' (score: 0.0)\n",
            "Document #621: 'Viral classification starts at the level of realm and continues as follows, with the taxonomic suffixes in parentheses:[8]Realm (-viria), Subrealm (-vira), Kingdom (-virae),Subkingdom (-virites), Phylum (-viricota), Subphylum (-viricotina), Class (-viricetes),Subclass (-viricetidae), Order (-virales),.' (score: 0.0)\n",
            "Document #620: 'The International Committee on Taxonomy of Viruses began to devise and implement rules for the naming and classification of viruses early in the 1970s, an effort that continues to the present. The ICTV is the only body charged by the International Union of Microbiological Societies with the task of developing, refining, and maintaining a universal virus taxonomy.[7] The system shares many features with the classification system of cellular organisms, such as taxon structure. However, some differences exist, such as the universal use of italics for all taxonomic names, unlike in the International Code of Nomenclature for algae, fungi, and plants and International Code of Zoological Nomenclature.[8].' (score: 0.0)\n",
            "Document #619: Viruses sensu stricto are defined operationally by the ICTV as a type of MGE (mobile genetic element) that encodes at least one protein that is a major component of the virion encasing the nucleic acid of the respective MGE and therefore the gene encoding the major virion protein itself or MGEs that are clearly demonstrable to be members of a line of evolutionary descent of such major virion protein-encoding entities. Any monophyletic group of MGEs that originates from a virion protein-encoding ancestor should be classified as a group of viruses.' (score: 0.0)\n",
            "Document #618: 'The currently accepted and formal definition of a 'virus' was accepted by the ICTV Executive Committee in November 2020 and ratified in March 2021, and is as follows:[6].' (score: 0.0)\n",
            "Document #617: 'The actual criteria used vary by the taxon, and can be inconsistent (arbitrary similarity thresholds) or unrelated to lineage (geography) at times.[5] The matter is, for many, not yet settled.[2].' (score: 0.0)\n",
            "Document #616: 'In July 2013, the ICTV definition of species changed to state: \"A species is a monophyletic group of viruses whose properties can be distinguished from those of other species by multiple criteria.\"[3] These criteria include the structure of the capsid, the existence of an envelope, the gene expression program for its proteins, host range, pathogenicity, and most importantly genetic sequence similarity and phylogenetic relationship.[4].' (score: 0.0)\n",
            "Document #615: 'Species form the basis for any biological classification system. Before 1982, it was thought that viruses could not be made to fit Ernst Mayr's reproductive concept of species, and so were not amenable to such treatment. In 1982, the ICTV started to define a species as \"a cluster of strains\" with unique identifying qualities. In 1991, the more specific principle that a virus species is a polythetic class of viruses that constitutes a replicating lineage and occupies a particular ecological niche was adopted.[2].' (score: 0.0)\n",
            "Document #614: 'A catalogue of all the world's known viruses has been proposed and, in 2013, some preliminary efforts were underway.[1].' (score: 0.0)\n",
            "Document #613: Viruses are classified by phenotypic characteristics, such as morphology, nucleic acid type, mode of replication, host organisms, and the type of disease they cause. The formal taxonomic classification of viruses is the responsibility of the International Committee on Taxonomy of Viruses (ICTV) system, although the Baltimore classification system can be used to place viruses into one of seven groups based on their manner of mRNA synthesis. Specific naming conventions and further classification guidelines are set out by the ICTV.' (score: 0.0)\n",
            "Document #612: Virus classification is the process of naming viruses and placing them into a taxonomic system similar to the classification systems used for cellular organisms.' (score: 0.0)\n",
            "Document #611:  (score: 0.0)\n",
            "Document #610: ‘At the same time, we report the first evidence that VHS physically interacts with ICP27 both in lysates of infected cells and in transfected cells.’ (score: 0.0)\n",
            "Document #609: ‘Consistent with this finding is the evidence that VHS does not accumulate in ICP27 virus-infected cells and that the VHS brought into cells by ICP27 virions is less effective that that brought into cells by wild-type virus.’ (score: 0.0)\n",
            "Document #608: ‘Indeed, at late times, the IEX-1 is degraded in ICP27 virus-infected cells at a rate similar to that in noninfected or VHS virus-infected cells.’ (score: 0.0)\n",
            "Document #607: ‘In this report, we have reexamined the degradation of GAPDH and IEX-1 mRNAs at early and late times in infection.’ (score: 0.0)\n",
            "Document #606: ‘In the description of their studies, they confused the cleavage events occurring during the early hours after in¬fection when the mRNAs are cleaved and partially degraded in a VHS-but not a ICP27-dependent manner with events taking place at late time intervals.’ (score: 0.0)\n",
            "Document #605: ‘They concluded that ICP27 is required to stabilize full-length, polyadenylated IEX mRNA and that VHS is not involved in this process.’ (score: 0.0)\n",
            "Document #604: ‘The time frame of their studies was selected on the basis of the presence of abundant amounts of ICP27 in the infected cells.’ (score: 0.0)\n",
            "Document #603: ‘Corcoran and colleagues (6) analyzed the degradation of AU-rich IEX-1 mRNA at relatively late times after infection.’ (score: 0.0)\n",
            "Document #602: ‘A dearth or lack of enzymes necessary to degrade the 5 cleavage product containing an intact cap may account for the lingering of the cleavage prod¬uct for many hours.’ (score: 0.0)\n",
            "Document #601: ‘The 3 product of the cleavage would be rapidly degraded 5 to 3 in the same man¬ner as the GAPDH mRNA.’ (score: 0.0)\n",
            "Document #600: ‘In the presence of TTP bound to AREs, VHS is expected to bind TTP and cleave the mRNA at the AREs.’ (score: 0.0)\n",
            "Document #599: ‘This is the normal pathway of degradation of mRNA and would lead to 5-to-3 exonucleolytic degradation of the cleaved RNA.’ (score: 0.0)\n",
            "Document #622: Suborder (-virineae), Family (-viridae),Subfamily (-virinae), Genus (-virus),Subgenus (-virus), and Species.' (score: 0.0)\n",
            "Document #624: 'As of 2023, all levels of taxa except subrealm, subkingdom, and subclass are used. Six realms, one incertae sedis class, 22.' (score: 0.0)\n",
            "Document #597: ‘In brief, in the course of mRNA translation, it is the poly(A)-binding protein at the 3 end that physically abuts the cap-binding complex at the 5 end of the mRNA.’ (score: 0.0)\n",
            "Document #625: 'incertae sedis families, and two incertae sedis genera are recognized:[12].' (score: 0.0)\n",
            "Document #648: Satellites depend on co-infection of a host cell with a helper virus for productive multiplication. Their nucleic acids have substantially distinct nucleotide sequences from either their helper virus or host. When a satellite subviral agent encodes the coat protein in which it is encapsulated, it is then called a satellite virus.' (score: 0.0)\n",
            "Document #647: 1and Genus Elaviroid and subsequent type species of Eggplant latent viroid.' (score: 0.0)\n",
            "Document #646: 'species of Avocado sunblotch viroid, Family Pospiviroidae , Genus Pospiviroid and subsequent type species of Potato spindle tuber viroid, Genus Hostuviroid and subsequent type species of Hop stunt viroid, Genus Cocadviroid and subsequent type species of Coconut cadang-cadang viroid, Genus Apscaviroid and subequent type species of Apple scar skin viroid, Genus Coleviroid and subsequet type species of Coleus blumei viroid.' (score: 0.0)\n",
            "Document #645: 'The system was not accepted by others due to its neglect of morphological similarities.[20]Infectious agents are smaller than viruses and have only some of their properties.[21][22] Since 2015, the ICTV has allowed them to be classified in a similar way as viruses are. Viroids include the Family Avsunviroidae[24], Genus Avsunviroid and subsequent type.' (score: 0.0)\n",
            "Document #644: Holmes (1948) used a Linnaean taxonomy with binomial nomenclature to classify viruses into 3 groups under one order, Virales. Group I includes Phaginae  which helps sustain bacterial evolution for survival under abiotic stress such as drought. Group II includes Phytophaginae Phaginae  which infects plants in order to reduce biomass when there are extreme frost, drought or environmental threat conditions to survival. Group III Zoophaginae which is part of the same infectious species that would target animals in order to activate or modify genomic signaling properties for survival in changed environmental conditions that would otherwise threaten offspring survival. .' (score: 0.0)\n",
            "Document #643: Reoviridaevirus family, such as the Reovirus, rotavirushas a NakedcapsidIcosahedralcapsid symmetry, dsnucleic acid type and belongs to the Group IIIof viruses. Picornaviridaevirus family, such as the Enterovirus, rhinovirus, hepatovirus, cardiovirus, aphthovirus, poliovirus, parechovirus, erbovirus, kobuvirus, teschovirus, coxsackiehas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Caliciviridaevirus family, such as the Norwalk virushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Togaviridaevirus family, such as the Eastern equine encephalitishas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Arenaviridaevirus family, such as the Lymphocytic choriomeningitis virus, Lassa feverhas a EnvelopedcapsidComplexcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Flaviviridaevirus family, such as the Dengue virus, hepatitis C virus, yellow fever virus, Zika virushas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Orthomyxoviridaevirus family, such as the Influenzavirus A, influenzavirus B, influenzavirus C, isavirus, thogotovirushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Paramyxoviridaevirus family, such as the Measles virus, mumps virus, respiratory syncytial virus, Rinderpest virus, canine distemper virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Bunyaviridaevirus family, such as the California encephalitis virus, Sin nombre virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Rhabdoviridaevirus family, such as the Rabies virus, Vesicular stomatitishas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Filoviridaevirus family, such as the Ebola virus, Marburg virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Coronaviridaevirus family, such as the Human coronavirus 229E, Human coronavirus NL63, Human coronavirus OC43, Human coronavirus HKU1, Middle East respiratory syndrome-related coronavirus, Severe acute respiratory syndrome coronavirus, and Severe acute respiratory syndrome coronavirus 2has a EnvelopedcapsidHelicalcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Astroviridaevirus family, such as the Astrovirushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Bornaviridaevirus family, such as the Borna disease virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Arteriviridaevirus family, such as the Arterivirus, equine arteritis virushas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Hepeviridaevirus family, such as the Hepatitis E virushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. All viruses that encode a reverse transcriptase (also known as RT or RNA-dependent DNA polymerase) are members of the class Revtraviricetes, within the phylum Arterviricota, kingdom Pararnavirae, and realm Riboviria. The class Blubervirales contains the single family Hepadnaviridae of DNA RT (reverse transcribing) viruses; all other RT viruses are members of the class Ortervirales.[19] Group VI: viruses possess single-stranded RNA viruses that replicate through a DNA intermediate. The retroviruses are included in this group, of which HIV is a member. Group VII: viruses possess double-stranded DNA genomes and replicate using reverse transcriptase. The hepatitis B virus can be found in this group.' (score: 0.0)\n",
            "Document #642: All viruses that have an RNA genome, and that encode an RNA-dependent RNA polymerase (RdRp), are members of the kingdom Orthornavirae, within the realm Riboviria.[18] Group III viruses possess double-stranded RNA genomes, e.g. rotavirus. Group IV viruses possess positive-sense single-stranded RNA genomes. Many well known viruses are found in this group, including the picornaviruses (which is a family of viruses that includes well-known viruses like Hepatitis A virus, enteroviruses, rhinoviruses, poliovirus, and foot-and-mouth virus), SARS virus, hepatitis C virus, yellow fever virus, and rubella virus. Group V: viruses possess negative-sense single-stranded RNA genomes. Ebola and Marburg viruses are well known members of this group, along with influenza virus, measles, mumps and rabies.' (score: 0.0)\n",
            "Document #641: Herpesviridae are enveloped virion with icosahedral capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #640: Poxviridae are Complex  coats virion with complex coats capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #639: Parvoviridae are naked virion with icosahedral capsid symmetry and ss nucleic acid types and belong to Group II viruses.' (score: 0.0)\n",
            "Document #638: Papovaviridaeare naked virion with icosahedral capsid symmetry and dscircular  nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #637: Adenoviridae are naked virion with icosahedral capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #636: Group II  viruses possess single-stranded DNA such as the pleolipoviridae (HHPV1, HRPV1). Group II viruses have an enveloped virion with ss nucleic acid type, ds nucleic acid type, and may be linear or cicular  The anelioviridiae are also group II viriuses that are not enveloped and called naked, and whose capsid symmetry is icosahedral and whose nucleic acid type is ss circular.' (score: 0.0)\n",
            "Document #635: Group I viruses possess double-stranded DNA. Viruses that cause chickenpox and herpes are found here.' (score: 0.0)\n",
            "Document #634: Viruses with a DNA genome, except for the DNA reverse transcribing viruses, are members of three of the four recognized viral realms: Duplodnaviria, Monodnaviria, and Varidnaviria. But the incertae sedis order Ligamenvirales, and many other incertae sedis families and genera, are also used to classify DNA viruses. The domains Duplodnaviria and Varidnaviria consist of double-stranded DNA viruses; other double-stranded DNA viruses are incertae sedis. The domain Monodnaviria consists of single-stranded DNA viruses that generally encode a HUH endonuclease; other single-stranded DNA viruses are incertae sedis.' (score: 0.0)\n",
            "Document #633: 'placed in one of the seven following groups:[16].' (score: 0.0)\n",
            "Document #632: 'Baltimore classification (first defined in 1971) is a classification system that places viruses into one of seven groups depending on a combination of their nucleic acid (DNA or RNA), strandedness (single-stranded or double-stranded), sense, and method of replication. Named after David Baltimore, a Nobel Prize-winning biologist, these groups are designated by Roman numerals. Other classifications are determined by the disease caused by the virus or its morphology, neither of which are satisfactory due to different viruses either causing the same disease or looking very similar. In addition, viral structures are often difficult to determine under the microscope. Classifying viruses according to their genome means that those in a given category will all behave in a similar fashion, offering some indication of how to proceed with further research. Viruses can be in I  dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses), II: ssDNA viruses (+ strand or \"sense\") DNA (e.g. Parvoviruses), III: dsRNA viruses (e.g. Reoviruses), IV:(+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses) V: (−)ssRNA viruses (− strand or antisense) RNA (e.g. Orthomyxoviruses, Rhabdoviruses), VI: ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses) VII: dsDNA-RT viruses DNA with RNA intermediate in life-cycle (e.g. Hepadnaviruses), .' (score: 0.0)\n",
            "Document #631: 'The ICTV has gradually added many higher-level taxa using relationships in protein folds. All four realms defined in the 2019 release are defined by the presence of a protein of a certain structural family.[15].' (score: 0.0)\n",
            "Document #630: 'It has been suggested that similarity in virion assembly and structure observed for certain viral groups infecting hosts from different domains of life (e.g., bacterial tectiviruses and eukaryotic adenoviruses or prokaryotic Caudovirales and eukaryotic herpesviruses) reflects an evolutionary relationship between these viruses.[13] Therefore, structural relationship between viruses has been suggested to be used as a basis for defining higher-level taxa – structure-based viral lineages – that could complement the ICTV classification scheme of 2010.[14].' (score: 0.0)\n",
            "Document #629: 'Incertae sedis genera: Dinodnavirus, Rhizidiovirus.' (score: 0.0)\n",
            "Document #628: 'Itzamnaviridae Naldaviricetes Ovaliviridae Plasmaviridae Polydnaviriformidae Portogloboviridae Pospiviroidae Rhodogtaviriformidae Spiraviridae Thaspiviridae Tolecusatellitidae.' (score: 0.0)\n",
            "Document #627: 'Alphasatellitidae Ampullaviridae Anelloviridae Avsunviroidae Bartogtaviriformidae Bicaudaviridae Brachygtaviriformidae Clavaviridae Fuselloviridae Globuloviridae Guttaviridae Halspiviridae.' (score: 0.0)\n",
            "Document #626: 'Realms include: Duplodnaviria, Monodnaviria, Adnaviria, Ribozyviria, Riboviria, and Varidnaviria while Incertae sedis families and classes include:.' (score: 0.0)\n",
            "Document #598: ‘In the absence of AREs, VHS is predicted to bind the cap-binding complex and decap the mRNA.’ (score: 0.0)\n",
            "Document #596: ‘We proposed a model of degradation of intrinsi¬cally stable mRNAs (GAPDH) and of ARE mRNAs on the basis of known structures of translated mRNAs, known path¬ways of mRNA decay, and the interaction of VHS protein with the cap-binding complex and TTP.’ (score: 0.0)\n",
            "Document #544: ‘Semliki forest virus causes translation inhibition of host cellular proteins through eIF2a phosphorylation.’ (score: 0.0)\n",
            "Document #569: ‘A recent study reported that, at relatively late times after infection, ARE RNAs are rapidly degraded in cells infected with ICP27 mutant virus and concluded that ICP27 “stabilizes” ARE mRNAs.’ (score: 0.0)\n",
            "Document #567: ‘Stress response RNAs containing AU-rich elements (AREs) in the 3 untranslated region (3 UTR) are dead¬enylated and cleaved, but the cleavage products persist for hours, in contrast to the short half-lives of ARE-containing mRNAs in uninfected cells.’ (score: 0.0)\n",
            "Document #566: ‘Pre-existing stable mRNAs (e.g., GAPDH [glyceraldehyde-3-phosphate dehydrogenase]) are rapidly degraded.’ (score: 0.0)\n",
            "Document #565: ‘The virion host shutoff (VHS) RNase tegument protein released into cells by infecting virus has two effects.’ (score: 0.0)\n",
            "Document #564: ‘Understanding how NDV manipulates stress re¬sponses will lay the groundwork for developing more ef¬fective oncolytic viruses and novel anticancer strategies.’ (score: 0.0)\n",
            "Document #563: ‘However, the hypothesis should be investigated further.’ (score: 0.0)\n",
            "Document #562: ‘That NDV efficiently and selectively replicates in and kills tumor and chicken cells, but not in other normal mammalian cells leads us to spec¬ulate whether the cell stress responses, such as autophagy and SGs, contribute to the tumor-selective replication of NDV.’ (score: 0.0)\n",
            "Document #561: ‘The relationship between autophagy and NDV has been identified in lung cancer cells and human glioma cells (57–59).’ (score: 0.0)\n",
            "Document #560: ‘We also found that NDV infection in¬duced cytoplasmic SG dots in chicken fibroblast cells DF-1 and primary chicken embryo fibroblast cells, and ectopic expression of GFP tagged chicken G3BP1 increased NDV replication (unpublished results).’ (score: 0.0)\n",
            "Document #559: ‘TIA-1 and TIAR regulated NDV replication by manipulating cellular protein translation on HeLa cells, where NDV selectively replicates (55, 56).’ (score: 0.0)\n",
            "Document #558: ‘In our current study, NDV triggered stable formation of bona fide SGs through the PKR/eIF2a pathway.’ (score: 0.0)\n",
            "Document #557: ‘In our study, knockdown of SG key components TIA-1, TIAR, or G3BP1 reduced NDV protein expression and extracellular virus yields, demonstrating that SGs play a proviral role in NDV infection.’ (score: 0.0)\n",
            "Document #556: ‘It is possible that both pro-and antiviral effects of SGs are at play and that the virus has developed mechanisms to overcome the antiviral effects of SGs.’ (score: 0.0)\n",
            "Document #555: ‘However, they may only play a marginal role in NDV replication, which was confirmed by the little or some¬what negative effect of siG3BPs on viral N and L protein expression (51).’ (score: 0.0)\n",
            "Document #554: ‘In tumor cells, SGs play an important role in recognizing uncapped viral RNA and facilitating IFN-b production.’ (score: 0.0)\n",
            "Document #553: ‘NDV infections induce IFN-b production in both normal and tumor cells (24, 55); however, pretreatment with endogenous or exogenous IFN-b has little or no effect on reducing the NDV yield or replication in tumor cells, although IFN-b treatment completely inhibits NDV replication in normal cells (55).’ (score: 0.0)\n",
            "Document #552: ‘A recent publication refuted the claim that NDV-induced SGs enhance IFN induction on HeLa cells; however, no virus replication data were shown (51).’ (score: 0.0)\n",
            "Document #551: ‘Similarly, in our study, siG3BP1 inhibited viral protein expression and extracellular virus yields, and meanwhile led to the increase in global protein trans¬lation, indicating that the translation balance is mediated by v-SGs rather than TIA-1/R per se.’ (score: 0.0)\n",
            "Document #550: ‘RSV replication is impaired in G3BP knockdown cells compared to its replication in control cells (22).’ (score: 0.0)\n",
            "Document #549: ‘For instance, G3BP1 and G3BP2 synergistically regulate the formation of SGs in various human cells (54).’ (score: 0.0)\n",
            "Document #548: ‘Except for TIA-1/R, other cellular proteins are in¬volved in SG assembly.’ (score: 0.0)\n",
            "Document #547: ‘In addition, TIA-1/R interacts with West Nile virus RNA and facilitates virus replication (53).’ (score: 0.0)\n",
            "Document #546: ‘Both TIA-1 and TIAR are RNA binding proteins and function in a similar manner in translation inhibition and SG formation (1, 11, 39).’ (score: 0.0)\n",
            "Document #545: ‘The translational shutoff is delayed in TIA-12/2 murine embryonic fibroblasts (11), indicating TIA function on the inhibition, which is in accord with our results.’ (score: 0.0)\n",
            "Document #568: ‘At late times, the VHS RNase is neutralized by the viral structural proteins VP16 and VP22.’ (score: 0.0)\n",
            "Document #570: ‘The rates of degradation of ARE mRNA at early times (3 h) after infection with the wild type or the ICP27 mutant virus are virtually identical, and hence ICP27 plays no role in this process.’ (score: 0.0)\n",
            "Document #595: ‘In addition, this laboratory has shown that VHS is active and degrades mRNA in polyribo-somes (49).’ (score: 0.0)\n",
            "Document #571: ‘ In non complementing cells, VHS RNase or VP22 is not synthesized.’ (score: 0.0)\n",
            "Document #594: ‘The VHS protein was also reported to bind cap-binding proteins (9, 17, 18, 19, 31).’ (score: 0.0)\n",
            "Document #593: (i) its mRNA contains AREs (3), (ii) TTP physically interacts with VHS protein (15), and (iii) in uninfected cells, TTP binds AREs and sequesters the mRNAs for degradation by exo¬somes (5, 28).’ (score: 0.0)\n",
            "Document #592: ‘TTP is of particular interest for three reasons: (score: 0.0)\n",
            "Document #591: ‘Lastly, a subset of mRNAs exemplified by GADD45 and tristetraprolin (TTP) directed the synthesis of corresponding proteins and were not degraded (14).’ (score: 0.0)\n",
            "Document #590: ‘Whereas the 3 cleavage product rapidly disappeared, the 5 portion of the cleaved RNA lingered for many hours.’ (score: 0.0)\n",
            "Document #589: ‘In contrast, stress response mRNAs contain¬ing AU-rich elements (AREs) in the 3 untranslated region (3 UTR), exemplified by IEX-1, c-Fos, IkB, or cox-2, which in uninfected cells share very short half-lives (1, 4, 51), were deadenylated and cleaved at or near the AREs (14).’ (score: 0.0)\n",
            "Document #588: ‘Thus, mRNAs that are highly stable in uninfected cells, exemplified by GAPDH (glyceraldehyde-3-phosphate dehydrogenase) and actin mRNAs, were rapidly degraded.’ (score: 0.0)\n",
            "Document #587: ‘This laboratory was the first to report that the degradation of mRNA during the early hours after infection by wild-type virus was highly selective (14, 16, 45).’ (score: 0.0)\n",
            "Document #586: 1991 that VHS mediated the degradation of some mRNAs but spared other mRNAs, most notably, histone mRNAs, the issue was still debated as late as 2004 (reviewed in reference 39).’ (score: 0.0)\n",
            "Document #585: ‘Even though Soreson and colleagues (41) reported as early as (score: 0.0)\n",
            "Document #584: ‘Most of the early work was done on model viral mRNAs, and these studies sustained the conclusion that VHS activity was not selective.’ (score: 0.0)\n",
            "Document #583: ‘Studies on the incorporation of radioactive amino acids into proteins after infection in cells infected with wild-type virus but not with a mutant lacking VHS activity led to the speculation that VHS activity is nonselective (23, 29, 30, 36).’ (score: 0.0)\n",
            "Document #582: ‘Relevant to this report are the following.’ (score: 0.0)\n",
            "Document #581: ‘In this report, we show that VHS interacts with still another viral protein, ICP27, and that this protein is required to block degradation of specific mRNAs at times when ICP27 becomes available for this func¬tion.’ (score: 0.0)\n",
            "Document #580: ‘More recent studies from this laboratory showed that an additional late viral protein, VP22, is required for the neutralization of the RNase activity of VHS protein (48).’ (score: 0.0)\n",
            "Document #579: ‘Initial reports indicated that the neutralizing protein is VP16 (27, 38).’ (score: 0.0)\n",
            "Document #578: ‘The VHS RNase is active during the first several hours after infection, and then it is neutralized by proteins made late in infection.’ (score: 0.0)\n",
            "Document #577: ‘Although it was long sus¬pected that VHS is an RNase (11, 12, 22, 32, 52), unambiguous evidence that VHS is an endoribonuclease with a substrate specificity of RNase A emerged from studies of a protein purified to homogeneity (46, 47).’ (score: 0.0)\n",
            "Document #576: ‘The herpes simplex virus type 1 (HSV-1) UL41 gene encod¬ing a protein (virion host shutoff [VHS]) that is carried by the virion into the infected cell and causes the shutoff of host protein synthesis was first reported by Frenkel and co workers in the late 1980s (24, 25, 26, 43).’ (score: 0.0)\n",
            "Document #575: ‘(iii) Concurrently, we present the first evidence that VHS RNase interacts with ICP27 most likely when bound to cap-and poly(A)-binding proteins, respectively.’ (score: 0.0)\n",
            "Document #574: ‘Hence the rapid degradation of ARE mRNAs noted in ICP27 mutant-infected cells at late times is similar to that taking place in mock-infected or in VHS RNase mutant-virus-infected cells and does not by itself support the hypothesis that ICP27 stabilizes ARE mRNAs.’ (score: 0.0)\n",
            "Document #573: ‘The VHS RNase brought into the cells by the ICP27 virus is reduced in potency relative to that of wild-type virus.’ (score: 0.0)\n",
            "Document #572: ‘Therefore, the only VHS that is active is brought into cells by the ICP27 mutant.’ (score: 0.0)\n",
            "Document #0: 'G-quadruplexes (G4s) are non-canonical nucleic acids secondary structures that form within guanine-rich strands of regulatory genomic regions.’ (score: 0.0)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from sklearn.feature_extraction.text import TfidfVectorizer\n",
        "from sklearn.metrics.pairwise import cosine_similarity\n",
        "\n",
        "# This vectorizer will convert our documents into a matrix of TF-IDF features\n",
        "vectorizer = TfidfVectorizer()\n",
        "\n",
        "# Vectorize our documents\n",
        "tfidf_matrix = vectorizer.fit_transform(corpus)  # Change 'documents' to 'corpus'\n",
        "\n",
        "def search(query):\n",
        "    # Vectorize the query to the same feature space as our documents\n",
        "    query_vec = vectorizer.transform([query])\n",
        "\n",
        "    # Compute the cosine similarity between the query and all documents\n",
        "    sims = cosine_similarity(query_vec, tfidf_matrix).flatten()\n",
        "\n",
        "    # Sort the document indices by their similarity score\n",
        "    ranking = sims.argsort()[::-1]\n",
        "\n",
        "    # Print out the ranking\n",
        "    for i in range(len(ranking)):\n",
        "        print(f\"Document #{ranking[i]}: {corpus[ranking[i]]} (score: {sims[ranking[i]]})\")  # Change 'documents' to 'corpus'\n",
        "\n",
        "# Let's try it out\n",
        "search(\"HIV-1\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "3LLgHHvpR2EN",
        "outputId": "d7edfa58-aef6-4045-fd7c-c6358e585111"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Document #643: Reoviridaevirus family, such as the Reovirus, rotavirushas a NakedcapsidIcosahedralcapsid symmetry, dsnucleic acid type and belongs to the Group IIIof viruses. Picornaviridaevirus family, such as the Enterovirus, rhinovirus, hepatovirus, cardiovirus, aphthovirus, poliovirus, parechovirus, erbovirus, kobuvirus, teschovirus, coxsackiehas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Caliciviridaevirus family, such as the Norwalk virushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Togaviridaevirus family, such as the Eastern equine encephalitishas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Arenaviridaevirus family, such as the Lymphocytic choriomeningitis virus, Lassa feverhas a EnvelopedcapsidComplexcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Flaviviridaevirus family, such as the Dengue virus, hepatitis C virus, yellow fever virus, Zika virushas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Orthomyxoviridaevirus family, such as the Influenzavirus A, influenzavirus B, influenzavirus C, isavirus, thogotovirushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Paramyxoviridaevirus family, such as the Measles virus, mumps virus, respiratory syncytial virus, Rinderpest virus, canine distemper virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Bunyaviridaevirus family, such as the California encephalitis virus, Sin nombre virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Rhabdoviridaevirus family, such as the Rabies virus, Vesicular stomatitishas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Filoviridaevirus family, such as the Ebola virus, Marburg virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Coronaviridaevirus family, such as the Human coronavirus 229E, Human coronavirus NL63, Human coronavirus OC43, Human coronavirus HKU1, Middle East respiratory syndrome-related coronavirus, Severe acute respiratory syndrome coronavirus, and Severe acute respiratory syndrome coronavirus 2has a EnvelopedcapsidHelicalcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Astroviridaevirus family, such as the Astrovirushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Bornaviridaevirus family, such as the Borna disease virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Arteriviridaevirus family, such as the Arterivirus, equine arteritis virushas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Hepeviridaevirus family, such as the Hepatitis E virushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. All viruses that encode a reverse transcriptase (also known as RT or RNA-dependent DNA polymerase) are members of the class Revtraviricetes, within the phylum Arterviricota, kingdom Pararnavirae, and realm Riboviria. The class Blubervirales contains the single family Hepadnaviridae of DNA RT (reverse transcribing) viruses; all other RT viruses are members of the class Ortervirales.[19] Group VI: viruses possess single-stranded RNA viruses that replicate through a DNA intermediate. The retroviruses are included in this group, of which HIV is a member. Group VII: viruses possess double-stranded DNA genomes and replicate using reverse transcriptase. The hepatitis B virus can be found in this group.' (score: 0.022489110254878906)\n",
            "Document #867:  (score: 0.0)\n",
            "Document #283: ‘The macrodomain possesses ADP-ribosylhydrolase activity (the ability to remove ADP-ribose from conjugated proteins), and this enzymatic activity is critical for replication and virulence of viruses from several viral families [92,101–104].’ (score: 0.0)\n",
            "Document #294: ‘Consistently, genetic depletion of G3BP1 severely affects viral replication and translation.’ (score: 0.0)\n",
            "Document #293: ‘The proteome of G3BP1 granules contains several proteins that are present in viral replication complexes, suggesting a pro-viral role of G3BP1 during MNV infection.’ (score: 0.0)\n",
            "Document #292: remodels G3BP1 interactome in infected cells and induces cytoplasmic G3BP1 granules during infection that are distinct from SGs.’ (score: 0.0)\n",
            "Document #291: ‘MNV (score: 0.0)\n",
            "Document #290: ‘Caliciviruses Remodel the G3BP1 Interactome during Infection The Caliciviridae family comprises small non-enveloped positive-strand RNA viruses, including human norovirus (HuNoV) and the closely related murine norovirus (MNV).’ (score: 0.0)\n",
            "Document #289: ‘4.2.2.’ (score: 0.0)\n",
            "Document #288: both Old World and New World alphaviruses, G3BP1 plays a pro-viral role only for Old World alphaviruses.’ (score: 0.0)\n",
            "Document #287: ‘Instead, these viruses sequester other SG components, such as FXR1, FXR2, and (score: 0.0)\n",
            "Document #286: ‘Notably, the New World alphaviruses do not have the FGDF motif and thus do not bind G3BP1.’ (score: 0.0)\n",
            "Document #285: ‘Infection with the mutant virus results in delayed SG disassembly and reduced viral structural protein synthesis in neuronal cells, suggesting that G3BP1 ADP-ribosylation regulates virus production [47,105].’ (score: 0.0)\n",
            "Document #284: ‘G3BP1 ADP-ribosylation, a key driver of SG assembly, is reduced in cells infected by wild-type CHIKV, but not a mutant virus that lacks hydrolase activity.’ (score: 0.0)\n",
            "Document #282: ‘Recently, our group discovered that the SG disassembly for CHIKV is partly mediated by the macrodomain at the N-terminus of nsP3.’ (score: 0.0)\n",
            "Document #296: ‘Similar to alphaviruses, G3BP1 in MNV-infected cells recruit ribosomes to viral RNA and initiate translation [31].’ (score: 0.0)\n",
            "Document #281: ‘However, SFV without FGDF motifs still disassembles SGs, but at a slower rate suggesting an additional mechanism for SG disassembly [77].’ (score: 0.0)\n",
            "Document #280: ‘The binding of nsP3 with G3BP1 is a proposed mechanism for SG disassembly where nsP3-G3BP1 binding inhibits the interaction between other SG components and G3BP1.’ (score: 0.0)\n",
            "Document #279: ‘For initiation of infection, G3BP1 facilitates CHIKV, but not to SFV, translation of genomic RNA by enriching translation factors at cytopathic vacuoles [100].’ (score: 0.0)\n",
            "Document #278: ‘In infected cells, G3BP1 is sequestered through interaction of the NTF2L domain with FGDF motifs in the C-terminal hypervariable region of non-structural protein 3 (nsP3).’ (score: 0.0)\n",
            "Document #277: ‘Cells lacking both G3BP1/G3BP2 do not support replication of the Old World alphaviruses but do support replication of the New World encephalitic alphaviruses, indicating a virus-specific importance of G3BP1 for infection.’ (score: 0.0)\n",
            "Document #276: ‘Notably, the alphavirus genus can be categorized into the Old World alphaviruses (CHIKV, SINV, SFV) and the New World alphaviruses (VEEV, EEEV) based on their geographical origin.’ (score: 0.0)\n",
            "Document #275: ‘Togaviruses Sequester G3BP1 to Facilitate Viral Replication and Translation Togaviridae is a family of enveloped, icosahedral viruses with a positive-sense single-stranded RNA genome including the mosquito-borne alphaviruses that cause rash and arthritis such as CHIKV, SFV, and SING.’ (score: 0.0)\n",
            "Document #274: ‘4.2.1.’ (score: 0.0)\n",
            "Document #273: ‘In this section, we will discuss the different strategies employed by viruses that use G3BP1 to promote viral replication.’ (score: 0.0)\n",
            "Document #272: ‘The proposed pro-viral functions of G3BP1 include: stabilizing viral mRNAs to prevent degradation, facilitating viral protein translation by recruiting translation factors and ribosomes, acting as a scaffold to build viral replication complexes, and amplifying the viral genome.’ (score: 0.0)\n",
            "Document #295: ‘In addition, knockdown of MNV-induced G3BP1 interactors reduces MNV replication, suggesting that the G3BP1 interactome consists of host factors that are required for efficient viral replication complex formation [84,107].’ (score: 0.0)\n",
            "Document #297: ‘During MNV infection, both PKR and GCN2 phosphorylated eIF2a; however, SG formation is not observed in the infected cells, even though they can still form SGs upon arsenite treatment, suggesting that MNV decouples the SG formation from eIF2a phosphorylation during virus infection.’ (score: 0.0)\n",
            "Document #270: ‘For example, during CHIKV infection, depletion of both G3BP1 and G3BP2 significantly decreases viral RNA levels, protein expression, and subsequent viral progeny produc¬tion [23].’ (score: 0.0)\n",
            "Document #298: ‘In contrast, feline calicivirus cleaves G3BP1 through the viral 3C-like proteinase NS6Pro [94].’ (score: 0.0)\n",
            "Document #321: ‘Neither deletion nor overexpression of G3BP1, G3BP2, or both affects viral titers.’ (score: 0.0)\n",
            "Document #320: ‘Depletion or Overexpression of G3BP1 Does Not Affect PRRSV Infection PRRSV—a single-stranded positive-sense enveloped virus in the Arteriviridae family— does not require G3BP1, yet the protein is closely associated with viral replication complexes.’ (score: 0.0)\n",
            "Document #319: ‘4.4.1.’ (score: 0.0)\n",
            "Document #318: ‘Though G3BP1 is a predominant target for many viruses, there are some viruses whose replication is unaffected by the expression level of G3BP1.’ (score: 0.0)\n",
            "Document #317: ‘Interestingly, HCV requires only G3BP1, but not G3BP2 whose knockdown does not alter viral replication [74,75].’ (score: 0.0)\n",
            "Document #316: UTR of viral RNA and is involved in genome amplification [74].’ (score: 0.0)\n",
            "Document #315: ‘During HCV infection, G3BP1 interacts with the 50¬ (score: 0.0)\n",
            "Document #314: ‘In addition, in ZIKV-infected cells, G3BP1 interacts with capsid protein and co-localizes with envelope protein, suggesting a possible role for G3BP1 in virion assembly [70–72].’ (score: 0.0)\n",
            "Document #313: ‘G3BP1 depletion reduces viral replication, titer, and viral protein levels, while overexpression of G3BP1 enhances viral titers.’ (score: 0.0)\n",
            "Document #312: ‘ZIKV and HCV Require G3BP1 for Proper Viral Replication In both ZIKV-and HCV-infected cells, G3BP1 associates with the viral replication complex.’ (score: 0.0)\n",
            "Document #311: ‘4.3.2.’ (score: 0.0)\n",
            "Document #310: and whether a similar decoy phenomenon may be used by other viruses to counter the production of ISG proteins and promote viral replication is unknown.’ (score: 0.0)\n",
            "Document #309: ‘Not all flavivirus sfRNAs have this property (score: 0.0)\n",
            "Document #308: ‘As an IFN countermeasure during DENV-2 infection of Huh-7 cells, a long noncoding RNA from the subgenomic region of flavivirus RNA (sfRNA) produced by all flaviviruses binds G3BP1, G3BP2, and Caprin1, and thereby inhibits the translation of ISGs [21].’ (score: 0.0)\n",
            "Document #307: ‘In DENV-2-infected cells, G3BP1 is also associated with ISG mRNAs and facilitates their translation to inhibit viral replication.’ (score: 0.0)\n",
            "Document #306: ‘The expression of viral protein is reduced in cells containing G3BP1 granules, whereas knockdown of G3BP1 enhances protein level and viral titers.’ (score: 0.0)\n",
            "Document #305: ‘Although the phosphorylation of eIF2a by PKR occurs at the late infection stage [73], it is unclear whether infected cells induce SGs.’ (score: 0.0)\n",
            "Document #304: ‘Dengue Virus (DENV) Modulates G3BP1 Function by Interacting with 30UTR vRNA DENV infection induces G3BP1 punctate-like structures at the early stage of infection.’ (score: 0.0)\n",
            "Document #303: ‘4.3.1.’ (score: 0.0)\n",
            "Document #302: ‘In this section, we describe how the G3BP1 function is modulated by DENV, ZIKV, and HCV.’ (score: 0.0)\n",
            "Document #301: utilize G3BP1 at different stages of infection.’ (score: 0.0)\n",
            "Document #300: ‘G3BP1 Is Differentially Utilized within the Flaviviridae Family Although G3BP1 influences viral propagation of Flaviviridae family, different members (score: 0.0)\n",
            "Document #299: ‘4.3.’ (score: 0.0)\n",
            "Document #271: ‘Binding of the viral genome by G3BP1 and sequestering it into viral replication complexes may be important to prevent G3BP1-mediated activation of the innate immune response.’ (score: 0.0)\n",
            "Document #269: ‘Such pro-viral activities have been reported for several viruses including CHIKV, hepatitis C virus, and respiratory syncytial virus [23,74,99].’ (score: 0.0)\n",
            "Document #323: ‘The reason for G3BP1 association with viral replication complexes and the mechanism and role of SG assembly at late infection stages are not known [85,86].’ (score: 0.0)\n",
            "Document #242: ‘Here we summarize the strategies shared by viruses in the Picornaviridae and Coronaviridae families to counteract host defense.’ (score: 0.0)\n",
            "Document #240: ‘G3BP1 also assembles SGs in response to infection-mediated translation arrest and limits the availability of translation factors for viral protein synthesis.’ (score: 0.0)\n",
            "Document #239: ‘In response to IFN, G3BP1, G3BP2, and Caprin 1 promote ISG translation to synthesize anti-viral factors.’ (score: 0.0)\n",
            "Document #238: ‘In virus-infected cells, G3BP1 binds to dsRNA resulting from the viral replication intermediates through the RGG domain and elevates RIG-I induced IFN-ß mRNA expression [30].’ (score: 0.0)\n",
            "Document #237: ‘G3BP1 enhances the immune response against invading viruses by inducing the expression of IFN and translation of ISGs.’ (score: 0.0)\n",
            "Document #236: ‘Here, we explore the mechanisms by which different viruses regulate G3BP1 function during infection.’ (score: 0.0)\n",
            "Document #235: (2) recruiting ribosomes to initiate viral RNA translation; (3) serving as a scaffold to build viral replication complexes; (4) assembling SGs; (5) stabilizing and enhancing the IFN-ß response; and (6) activating cGAS and RIG-I pathways.’ (score: 0.0)\n",
            "Document #234: ‘Depending on the virus type, G3BP1 plays different roles in the viral life cycle, including (1) stabilizing viral RNA; (score: 0.0)\n",
            "Document #233: ‘G3BP1 facilitates replication of some viruses whereas it restricts viral spread in other viruses.’ (score: 0.0)\n",
            "Document #232: ‘G3BP1 controls several key cellular processes, including mRNA metabolism, ribosomal quality control, immune response, and SG assembly.’ (score: 0.0)\n",
            "Document #231: ‘Here, we describe the role of G3BP1 during specific viral infections and the ways by which G3BP1 is targeted.’ (score: 0.0)\n",
            "Document #230: ‘On the contrary, viruses cleave G3BP1 when it exerts an anti-viral role (Table 2).’ (score: 0.0)\n",
            "Document #229: ‘In general, viruses use their genome-encoded proteins to sequester G3BP1 if it is required for viral replication.’ (score: 0.0)\n",
            "Document #228: ‘Further clues are obtained from identifying the strategies that viruses employ to target G3BP1—i.e., sequestration, cleavage, or degradation—and modulate its function during infection (Figure 4).’ (score: 0.0)\n",
            "Document #227: ‘Phosphorylation of G3BP1 by BTK is critical for dimerization and phase separation to limit viral spread [88].’ (score: 0.0)\n",
            "Document #226: ‘showed that G3BP1 is phosphorylated at tyrosine residue 40 by Bruton’s tyrosine kinase (BTK).’ (score: 0.0)\n",
            "Document #225: ‘Using polyinosinic:polycytidylic acid as a mimic for viral replication double-stranded intermediates, Kim et al.’ (score: 0.0)\n",
            "Document #224: ‘Similarly, arginine methylation status of G3BP1 partly recruits Tudor domain-containing protein (TDRD3) to regulate innate immune response against enterovirus species [60].’ (score: 0.0)\n",
            "Document #223: ‘Thus, post-translationally modified G3BP1 may act as an antiviral protein, while the unmodified or differently modified G3BP1 could play a pro-viral role during CHIKV infection.’ (score: 0.0)\n",
            "Document #222: ‘However, G3BP1 is also required for establishing CHIKV replication complexes, and post-translational modifications required for this process have not been defined [23].’ (score: 0.0)\n",
            "Document #221: ‘For example, ADP-ribosylated G3BP1 is crucial for SG assembly, given that chikungunya virus (CHIKV) actively reduces G3BP1 ADP-ribosylation, disrupting SGs with translation factors released from the con¬densed state [47].’ (score: 0.0)\n",
            "Document #220: ‘One way by which G3BP1 may regulate its function dur¬ing infection is through post-translational modifications.’ (score: 0.0)\n",
            "Document #219: ‘G3BP1 can either promote or inhibit the viral life cycle, but the molecular mechanisms are not completely understood.’ (score: 0.0)\n",
            "Document #218: ‘SG status during infection against different viruses.’ (score: 0.0)\n",
            "Document #241: ‘To counter these antiviral functions, viruses target G3BP1—either cleaving or sequestering it during infection.’ (score: 0.0)\n",
            "Document #243: ‘Picornaviruses Inhibit G3BP1 Function through Protease-Mediated Cleavage Picornaviruses are small, non-enveloped, positive-sense RNA viruses that include EMCV, EV, PV, FMDV, and CVB3.’ (score: 0.0)\n",
            "Document #268: ‘Although G3BP1 synergizes host immune responses to fight viral invasion, some viruses require G3BP1 for efficient infection.’ (score: 0.0)\n",
            "Document #244: ‘G3BP1 negatively regulates Picornaviridae infection by binding the 30 untranslated regions (30UTRs) or Internal Ribosome Entry Site (IRES) regions of viral RNAs to inhibit viral replication [53,95].’ (score: 0.0)\n",
            "Document #267: ‘In COVID-19 patients and mice, the N protein interacts with G3BP1, suppresses SG formation, and potentiates viral infection by antagonizing the G3BP1-mediated host innate immune response pathway [97,98].’ (score: 0.0)\n",
            "Document #266: highlighted the potential antiviral role of G3BP1 in the lung tissue of COVID-19 patients and SARS-CoV-2-infected mice.’ (score: 0.0)\n",
            "Document #265: ‘Recent studies (score: 0.0)\n",
            "Document #264: ‘However, the role of G3BP1 remains unclear given the conflicting reports on the effect of G3BP1 knockdown in structural protein and viral RNA levels [81–83,96].’ (score: 0.0)\n",
            "Document #263: ‘Though it is unclear whether SARS-CoV-2 induces SGs, G3BP1–nucleocapsid binding blocks the interaction of SG proteins such as PKR and USP10 with G3BP1 in arsenite-treated cells [82].’ (score: 0.0)\n",
            "Document #262: ‘The intrin¬sically disordered region (IDR) at the N-terminal region of nucleocapsid is required for condensation with G3BP1, and this IDR is also crucial for viral particle production.’ (score: 0.0)\n",
            "Document #261: ‘SARS-CoV-2—the virus responsible for the COVID-19 pandemic—modulates G3BP1 function by sequestration and condensation with the nucleocapsid protein.’ (score: 0.0)\n",
            "Document #260: ‘Expression of non-cleavable G3BP1 results in persistent SGs and reduces viral replication, but the mechanism behind G3BP1-mediated reduction of viral replication remains unexplored [79,80].’ (score: 0.0)\n",
            "Document #259: ‘However, these viruses utilize different strategies to modulate G3BP1 function; for example, PEDV induces cleavage of G3BP1 between D168 and D169 by caspase-8 to regu¬late SG dynamics in infected cells.’ (score: 0.0)\n",
            "Document #258: ‘Overexpression of G3BP1 reduces viral replication, protein synthesis, and virus production, while G3BP1 knockdown enhances viral replica¬tion.’ (score: 0.0)\n",
            "Document #257: ‘Similar to Picornaviridae, G3BP1 nega¬tively regulates coronavirus infection.’ (score: 0.0)\n",
            "Document #256: ‘Coronaviruses Modulate G3BP1 Function Either by Cleavage or Sequestration Coronaviridae is a family of enveloped viruses with large single-stranded positive-sense RNA genomes and includes PEDV and SARS-CoV-2.’ (score: 0.0)\n",
            "Document #255: the fundamental mechanism employed by Picornaviridae to restrict antiviral activities of G3BP1.’ (score: 0.0)\n",
            "Document #254: ‘Thus, protease-mediated cleavage or degradation of G3BP1 is (score: 0.0)\n",
            "Document #253: ‘FMDV infection also causes degradation of G3BP1 through autophagy [62].’ (score: 0.0)\n",
            "Document #252: ‘Catalytic mutant C41A or leaderless FMDV efficiently inhibits arsenite-induced SG formation in infected cells [63].’ (score: 0.0)\n",
            "Document #251: ‘FMDV, which uses the leader protein to cleave G3BP1, also causes SG disassem¬bly.’ (score: 0.0)\n",
            "Document #250: ‘Consistent with the observed effect on G3BP1 cleavage, knockdown of G3BP1 increases viral protein synthesis and virus production, whereas G3BP1 overexpression shows an inhibitory effect [61,64–66].’ (score: 0.0)\n",
            "Document #249: ‘SG disassembly correlates temporally with G3BP1 cleavage, and the expression of the non-cleavable G3BP1 Q325E mutant results in sustained IFN-ß mRNA expression and SG persistence.’ (score: 0.0)\n",
            "Document #248: ‘SGs are observed early in infection but disassembled at late stages of infection.’ (score: 0.0)\n",
            "Document #247: ‘During Picornaviruses infection, PKR is activated by double-stranded RNA intermedi¬ates, phosphorylated eIF2a, and induces SG formation.’ (score: 0.0)\n",
            "Document #246: ‘G3BP1 cleavage is essential for the expression of viral proteins, and expression of the non-cleavable G3BP1 Q325E mutant suppresses viral replication [61–69].’ (score: 0.0)\n",
            "Document #245: ‘To counteract these effects, the 3C protease or leader protein of these viruses cleaves G3BP1 between Q325 and Q326 at late infection stages.’ (score: 0.0)\n",
            "Document #322: ‘In contrast to the above-described viruses, PRRSV induces SGs only at late stages of infection (48–72-h post-infection) through PERK-, rather than PKR-mediated eIF2a phosphorylation.’ (score: 0.0)\n",
            "Document #324: ‘Intriguingly, one study suggested a pro-viral function of SG assembly during PRRSV infection.’ (score: 0.0)\n",
            "Document #432: ‘Microtubule disruption leads to the formation of small SGs after NDV infection (score: 0.0)\n",
            "Document #392: ‘AVG formation during poxvirus infection leads to adecreaseinbothviral translation and virus titers (20, 21).’ (score: 0.0)\n",
            "Document #403: ‘After invasion of host cells by NDV, the NP-wrapped viral genomic RNA, as well as other viral components, is released into the cytoplasm (27).’ (score: 0.0)\n",
            "Document #402: ‘The PP is essential for viral RNA synthesis from bona fide templates (26).’ (score: 0.0)\n",
            "Document #401: ‘The NP encapsu¬lates the RNA genome, which serves as the template for viral transcription and replication (25).’ (score: 0.0)\n",
            "Document #400: ‘The NDV genome contains 6 genes that encode 2 glycoproteins, the fusion protein and hemagglutinin-neuraminidase protein, the matrix protein, and the ribonucleoprotein complex, com¬posed of the nucleocapsid protein (NP), phosphopro¬tein (PP), and large polymerase, responsible for viral transcription and replication (23).’ (score: 0.0)\n",
            "Document #399: ‘Oncolytic NDV can selectively replicate in tumor cells and is an important antineo¬plastic agent (24).’ (score: 0.0)\n",
            "Document #398: ‘NDV is a member of the Paramyxoviridae family with a negative-sense, nonsegmented, single-stranded RNA genome (23).’ (score: 0.0)\n",
            "Document #397: ‘Depletion of endogenous G3BP inhibits SG formation and significantly decreases the viral titers and RNA amounts (22).’ (score: 0.0)\n",
            "Document #396: ‘Respiratory syncytial virus (RSV) also triggers canoni¬cal SGs.’ (score: 0.0)\n",
            "Document #395: ‘For instance, knockdown of SG proteins TIA-1, TIAR, or Ras GTPase¬activating protein-binding protein 1 (G3BP1) impairs HCV RNA replication, virus assembly, and egress (18).’ (score: 0.0)\n",
            "Document #394: ‘However, in situations where the virus induces bona fide SG formation, SG components seem to be required for virus infection.’ (score: 0.0)\n",
            "Document #393: ‘These findings support the hy¬pothesis that SGs (or SG-like structures) are cellular antiviral responses.’ (score: 0.0)\n",
            "Document #391: ‘In other cases, viruses induce SG-like structures (e.g., antiviral granules, AVGs) that re¬strict virus replication.’ (score: 0.0)\n",
            "Document #325: ‘Cells infected (score: 0.0)\n",
            "Document #390: ‘A mutant JEV carrying a core protein incapable of binding to caprin-1 abrogates the SG inhibitory ability and exhibits lower propagation in vitro and in vivo (19).’ (score: 0.0)\n",
            "Document #389: ‘As an example, Japanese encephalitis virus (JEV) inhibits SG formation through interaction between the viral core protein and caprin-1.’ (score: 0.0)\n",
            "Document #388: ‘In cases where viral infection does not induce SGs, accumulat¬ing evidence demonstrates an inverse correlation be¬tween SG formation and viral propagation.’ (score: 0.0)\n",
            "Document #387: ‘SGs generally play an antiviral role (19–21).’ (score: 0.0)\n",
            "Document #386: ‘For example, SGs are served at 48 h post infection (hpi) with HCV and increase to speak at 72hpi (18).’ (score: 0.0)\n",
            "Document #385: ‘Other viru¬ses induce stable SG formation.’ (score: 0.0)\n",
            "Document #384: ‘For example, during the early stages of picor¬navirus infection, v-SGs can be observed but then disappear, probably because of the viral 3C protease, which is responsible for the cleavage of the Ras-GAP SH3 domain-binding protein (G3BP) (15–17).’ (score: 0.0)\n",
            "Document #383: ‘Some viruses induce transient SG for¬mation.’ (score: 0.0)\n",
            "Document #382: ‘For example, wild-type influenza A virus does not induce SGs but an NS1¬mutant virus triggers SG formation in a PKR-dependent manner (13, 14).’ (score: 0.0)\n",
            "Document #381: ‘Some viruses inhibit SG formation.’ (score: 0.0)\n",
            "Document #380: ‘The pattern of virus-induced (v)-SGs varies in the virus species.’ (score: 0.0)\n",
            "Document #404: ‘The NP encapsulates (2) viral gRNA as a template to produce (+) cRNA and mRNA.’ (score: 0.0)\n",
            "Document #405: ‘Viral mRNAs are capped and polyadenylated by L-protein during pro¬tein synthesis (28, 29).’ (score: 0.0)\n",
            "Document #406: ‘The translation process relies on cellular translational machinery that is present within the host cell cytoplasm, eliciting a strong stress re¬sponse of cells.’ (score: 0.0)\n",
            "Document #407: ‘In this study, the effect of NDV-induced SGs on its replication and the possible mechanism were investigated.’ (score: 0.0)\n",
            "Document #430: ‘4C–E).’ (score: 0.0)\n",
            "Document #429: ‘Depletion of endogenous PKR by siRNA reduced the level of eIF2a phosphory¬lation and the percentage of cells with SGs significantly (Fig.’ (score: 0.0)\n",
            "Document #428: ‘4B).’ (score: 0.0)\n",
            "Document #427: ‘Western blot analysis showed that PKR and down¬stream eIF2a was phosphorylated after ARS treatment and virus infection (Fig.’ (score: 0.0)\n",
            "Document #426: ‘4A).’ (score: 0.0)\n",
            "Document #425: ‘PKR were colocalized with either TIA-1 or G3BP1, respectively, after infection (Fig.’ (score: 0.0)\n",
            "Document #424: ‘To examine whether PKR acts as an upstream component of eIF2a in the formation of v-SGs, cells were costained for PKR and TIA-1/G3BP1.’ (score: 0.0)\n",
            "Document #423: ‘3B).’ (score: 0.0)\n",
            "Document #422: ‘(Fig.’ (score: 0.0)\n",
            "Document #421: ‘Conversely, S51D eIF2a transfection led to the accumulation of SG dots that appeared larger than v-SGs.’ (score: 0.0)\n",
            "Document #420: ‘No SG dots were observed in S51A eIF2a¬transfected cells, compared with the SG dots observed in WT eIF2a-transfected cells.’ (score: 0.0)\n",
            "Document #419: ‘To elucidate further the potential role of eIF2a in v-SGs, we measured SG formation by transfecting 2 eIF2a mutant plasmids, S51D and S51A eIF2a, which mimic hyper-and hypo phosphorylated forms of eIF2a,re¬spectively.’ (score: 0.0)\n",
            "Document #418: ‘We noted that the decreased levels of eIF2a and b-actin at 24 hpi were the result of the cell shedding caused by virus infection.’ (score: 0.0)\n",
            "Document #417: ‘Correspondingly, the viral NP accumulated at the late stage of infection.’ (score: 0.0)\n",
            "Document #416: ‘As expected, NDV infection reduced host protein translation over the time).’ (score: 0.0)\n",
            "Document #415: ‘The decrease of eIF2-GTP-tRNAMet ultimately leads to a global decrease in protein syn¬thesis (39).’ (score: 0.0)\n",
            "Document #414: ‘Phosphorylation of eIF2a leads to SG assembly by reducing the levels of the ternary complex eIF2-GTP-tRNAMet.’ (score: 0.0)\n",
            "Document #413: ‘3A).’ (score: 0.0)\n",
            "Document #412: ‘We measured eIF2a phosphoryla¬tion and found that NDV triggered eIF2a phosphorylation at the late stage of infection, consistent with the appear¬ance of v-SGs (Fig.’ (score: 0.0)\n",
            "Document #411: The activation of PKR and eIF2a leads to SG formation in virus-infected cells (38).’ (score: 0.0)\n",
            "Document #410: ‘PKR/eIF2a signaling cascade is essential for NDV-induced SGs (score: 0.0)\n",
            "Document #409: ‘Taken together, these results suggest that NDV induces canonical SGs, which is similar to the SGs formed under oxidative stress, indicating that ongoing protein synthesis is essential for NDV-induced SGs.’ (score: 0.0)\n",
            "Document #408: ‘Our results demonstrate that NDV-induced SGs facilitate viral replication through ma-nipulating host protein translation.’ (score: 0.0)\n",
            "Document #379: ‘Recent years have seen a growing list of viruses with replication that is associated with SG formation (12).’ (score: 0.0)\n",
            "Document #378: ‘Some viruses escape translation inhibition through affect¬ing SG formation (11).’ (score: 0.0)\n",
            "Document #377: ‘Phosphory¬lation of eIF2a also leads to SG formation.’ (score: 0.0)\n",
            "Document #350: ‘Virus-induced SGs function in different ways, depending on the species of virus; however, the mechanism of SG regulation of virus replication is not well understood.’ (score: 0.0)\n",
            "Document #348: ‘Therefore, not all G3BP1-positive structures are SGs, and should be stained with other markers to ascertain their identities.’ (score: 0.0)\n",
            "Document #347: ‘Given that the interacting partners of G3BP1 are modulated during infection to form new complexes in a virus-specific manner, different terms have been used to differentiate for these viral-induced protein complexes, including atypical SGs (EV), antiviral SGs (IBV), or nsP3 foci (alphaviruses).’ (score: 0.0)\n",
            "Document #346: ‘During norovirus infection, the G3BP1 interactome is modulated to inhibit the formation of SGs.’ (score: 0.0)\n",
            "Document #345: ‘In SIV-infected cells, G3BP1 stably interacts with viral proteins throughout the life cycle, whereas some SG components interact in a time-dependent manner.’ (score: 0.0)\n",
            "Document #344: ‘Recent proteomic studies provided some insights into the G3BP1 interactome during infection.’ (score: 0.0)\n",
            "Document #343: G3BP1 plays critical yet distinct roles during infection by different families of the virus.’ (score: 0.0)\n",
            "Document #342: ‘Furthermore, the differences in the role (score: 0.0)\n",
            "Document #341: ‘Given that G3BP1 is also modified by phosphorylation and ubiquitylation, it is worth checking the status of G3BP1 for other post-translational modifications at different infection stages and determine how these modifications affect a viral life cycle [33,42].’ (score: 0.0)\n",
            "Document #340: ‘Alphaviruses and coronaviruses encode an enzyme to remove ADP-ribosylation indicating that the modified form of G3BP1 may possess antiviral functions.’ (score: 0.0)\n",
            "Document #339: ‘G3BP1 methylation recruits TDRD3 into SGs to regulate innate immune response [60].’ (score: 0.0)\n",
            "Document #338: ‘Intriguingly, the post-translational modification status of heavily regulated G3BP1 may determine the functional consequences of interaction.’ (score: 0.0)\n",
            "Document #337: ‘Besides, some viruses possess conserved motifs (e.g., FGDF) within their proteins to bind and modulate G3BP1 func¬tion [76,109].’ (score: 0.0)\n",
            "Document #336: ‘For pro-viral roles, the NTF2L and EGG domains of G3BP1 play a crucial role to bind viral proteins or viral RNA, respectively.’ (score: 0.0)\n",
            "Document #335: ‘If G3BP1 is required for viral replication, viruses do not cleave G3BP1, but instead recruit it to build viral replication complexes (e.g., alphaviruses).’ (score: 0.0)\n",
            "Document #334: ‘In general, if G3BP1 exerts anti-viral activity, then viruses cleave or sequester G3BP1 (e.g., polioviruses, flaviviruses).’ (score: 0.0)\n",
            "Document #333: ‘These functional modulations of G3BP1 by viruses impact the viral life cycle either posi¬tively or negatively.’ (score: 0.0)\n",
            "Document #332: ‘Viruses employ at least three strategies to target and modulate G3BP1 function during infection: (1) enzymatic activity such as proteolytic cleavage or removing post-translational modification (e.g., poliovirus and alphavirus); (2) binding or sequestration (e.g., alphavirus and coronavirus); and (3) redistribution or remodeling of G3BP1 interaction (e.g., norovirus).’ (score: 0.0)\n",
            "Document #331: ‘Although G3BP1 is sequestered within the viral inclusions, the exact role of G3BP1 in EBOV infection is not well understood.’ (score: 0.0)\n",
            "Document #330: ‘VP5 expression affects IFNa/ß expression and PKR kinase activation in infected cells.’ (score: 0.0)\n",
            "Document #329: ‘Although infection-mediated SG assembly is not observed, EBOV inhibits arsenite-induced SG assembly.’ (score: 0.0)\n",
            "Document #328: ‘G3BP1 Is Sequestered within Ebola Virus Inclusions EBOV, a single-stranded negative-sense RNA virus, does not induce SG assembly but sequesters SG components G3BP1, eEF2, eIF2, and eIF3 through the viral protein VP5 [55,87].’ (score: 0.0)\n",
            "Document #327: ‘4.4.2.’ (score: 0.0)\n",
            "Document #326: PRRSV induces SGs, recruits G3BP1 and the viral replicase protein nsP1ß, thereby inhibiting G3BP1-mediated PKR activation [108].’ (score: 0.0)\n",
            "Document #349: ‘Mammalian cells respond to various environmental stressors to form stress granules (SGs) by arresting cytoplasmic mRNA, protein translation element, and RNA binding proteins.’ (score: 0.0)\n",
            "Document #351: ‘In this study, Newcastle disease virus (NDV) triggered stable formation of bona fide SGs on HeLa cells through activating the protein kinase R (PKR)/eIF2a pathway.’ (score: 0.0)\n",
            "Document #376: ‘In this case, viruses such as hepatitis C(HCV) keep on translating through IRES structure in the presence of activated PKR and phosphorylation of eIF2a (10).’ (score: 0.0)\n",
            "Document #352: ‘NDV-induced SGs contained classic SG markers T-cell internal antigen (TIA)-1, Ras GTPase-activating protein-binding protein (G3BP)-1, eukaryotic initiation factors, and small ribosomal subunit, which could be disassembled in the presence of cycloheximide.’ (score: 0.0)\n",
            "Document #375: ‘Some of theseRNA viruses induce phosphorylation of eIF2a and shut-off of global protein synthesis.’ (score: 0.0)\n",
            "Document #374: ‘Many other RNA viruses contain an internal ribosome entry site (IRES) located in the 59-untranslated region of the genomic (g)RNA, which drives cap-independent ini¬tiation of translation of the viral message (9).’ (score: 0.0)\n",
            "Document #373: ‘For instance, the influenza virus nonstructural protein NS1 interacts with eIF4GI and PABP-1, as well as with viral mRNAs, promoting the specific recruitment of 43S complexes to the viral mRNAs (8).’ (score: 0.0)\n",
            "Document #372: ‘They adopt a se¬ries of methods to rob host translation systems.’ (score: 0.0)\n",
            "Document #371: ‘As obligate intracellular parasites, viruses rely on the host cell translation machinery.’ (score: 0.0)\n",
            "Document #370: ‘This process ultimately leads to reduced translation initiation rates and to SG formation (6, 7).’ (score: 0.0)\n",
            "Document #369: ‘The other non canonical mechanism of SG induction is independent of eIF2a and involves inhibiting the activity of eIF4A, eIF4G, and eIF4E, 3 subunits of the eIF4F complex.’ (score: 0.0)\n",
            "Document #368: ‘The phosphorylation of eIF2a at Ser-51 is mediated by several kinases, including the heme-regulated inhibitor kinase, general control nonderepressible kinase, pro¬tein kinase R (PKR), and the PKR-like endoplasmic re¬ticulum kinase (5).’ (score: 0.0)\n",
            "Document #367: ‘The RNA-binding proteins T-cell internal antigen-1 (TIA-1) and TIA-1¬related (TIAR) protein are then shuttled from the nucleus to the cytoplasm to sequester untranslated mRNAs at SGs (1).’ (score: 0.0)\n",
            "Document #366: ‘When cells are exposed to stressors, phosphorylation of eIF2a leads to a decreased abundance of eIF2-GTP-tRNAMet (2).’ (score: 0.0)\n",
            "Document #365: ‘The eIF2-GTP-tRNAMet ter¬nary complex associates with the 40S complex and scans for an initiation codon (2).’ (score: 0.0)\n",
            "Document #364: ‘The central region of eIF4G binds to the 40S ribosome subunit, and the N terminus recruits mRNA through binding to poly(A)¬binding protein (PABP) (4).’ (score: 0.0)\n",
            "Document #363: ‘eIF4F consists of 3 initiation factors: eIF4E, eIF4G, and eIF4A.’ (score: 0.0)\n",
            "Document #362: ‘Conventional cap-dependent translation of most cellular mRNA commences with the recognition of 7¬methyl-GTP cap by the eukaryotic initiation factor (eIF) 4F complex of translation initiation factors (3).’ (score: 0.0)\n",
            "Document #361: ‘In this case, stress granules (SGs) are induced to arrest cytoplas¬mic mRNA, protein translation factors, and RNA binding proteins (1, 2).’ (score: 0.0)\n",
            "Document #360: oxidative stress, heat shock, and UV irradiation.’ (score: 0.0)\n",
            "Document #359: ‘Mammalian cells enter a relatively stable state by tempo¬rarily inhibiting cellular protein translation when exposed to various environmental stressors, such as virus infection, (score: 0.0)\n",
            "Document #358: ‘These results demonstrate that NDV triggers stable formation of bona fide SGs, which benefit viral protein translation and virus replication by arresting cellular mRNA.’ (score: 0.0)\n",
            "Document #357: ‘Deletion of TIA-1 or TIAR reduced NP mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #356: ‘FISH revealed that NDV-induced SGs contained predominantly cellular mRNA rather than viral mRNA.’ (score: 0.0)\n",
            "Document #355: ‘Results showed that depletion of TIA-1 or TIAR inhibited viral protein synthesis, reduced extracellular virus yields, but increased global protein translation.’ (score: 0.0)\n",
            "Document #354: ‘Furthermore, the role of SGs on NDV replication was investigated by knockdown of TIA-1 and TIA-1-related (TIAR) protein, the 2 critical components involved in SG formation from the HeLa cells, followed by NDV infection.’ (score: 0.0)\n",
            "Document #353: ‘Treatment with nocodazole, a microtubule disruption drug, led to the formation of relatively small and circular granules, indicating that NDV infection induces canonical SGs.’ (score: 0.0)\n",
            "Document #217: ‘Table 1.’ (score: 0.0)\n",
            "Document #216: ‘On the other hand, the presence of SGs may facilitate the switch from viral translation to genome amplification by sequestering translation factors into the condensed state (Figure 3) [23].’ (score: 0.0)\n",
            "Document #215: ‘Thus, viruses may target SGs not only to modulate G3BP1 function but also to disassemble SGs to counteract anti-viral mechanisms in the infected cells.’ (score: 0.0)\n",
            "Document #67: ‘We show that nsp1ß is a stress-responsive protein, enters virus-induced stress granules (SGs) during infection, and repurposes SGs into a proviral platform, where it co-opts the SG core component G3BP1 to interact with PKR in a regulated manner.’ (score: 0.0)\n",
            "Document #78: ‘The activated PKR phosphorylates the a-subunit of eukaryotic translation initiation fac¬tor 2 (eIF2a) to induce general translation arrest and to stimulate formation of stress gran¬ules (SGs) (7, 10), a kind of non membrane bound cytoplasmic aggregate that mainly include stalled translation initiation complexes, together with nucleating factors like Ras GTPase-activating protein-binding protein 1 (G3BP1), T cell intracytoplasmic antigen 1 (TIA1), and many others (11, 12).’ (score: 0.0)\n",
            "Document #77: ‘Alternatively, PKR can be activated via heterodimerization by an endogenous dsRNA binding protein, namely PKR-activating protein (PACT) (8).’ (score: 0.0)\n",
            "Document #76: ‘Binding of DRBMs to viral dsRNAs or 50 triphosphate RNAs induces conformational changes of PKR, leading to autophos¬phorylation of its KD (8, 9).’ (score: 0.0)\n",
            "Document #75: ‘Structurally, PKR is composed of two tan¬dem evolutionarily conserved dsRNA-binding motifs (DRBMs) in the N terminus, an effector/catalytic kinase domain (KD) in the C terminus, and a middle .flexible linker region connecting the two domains (2, 6, 7).’ (score: 0.0)\n",
            "Document #74: ‘This protein was initially discovered as a key component of interferon (IFN)-stimulated genes (3, 4) and later identified as a pattern-recognition receptor that specifically recog¬nizes double-stranded RNAs (dsRNA) (5, 6), a common intermediate produced during infections of both DNA and RNA viruses.’ (score: 0.0)\n",
            "Document #73: ‘At the virus–host interface, one such critical host player is the protein kinase R (PKR), a 551-amino acid-long serine threonine kinase (1, 2).’ (score: 0.0)\n",
            "Document #72: Eukaryotic cells employ an amazingly wide variety of intracellular defenses to restrict invading viral pathogens.’ (score: 0.0)\n",
            "Document #71: ‘PKR j stress granules j PRRSV j in.ammation j nsp1ß (score: 0.0)\n",
            "Document #70: ‘Thus, PRRSV has evolved a unique strategy to evade PKR restriction to suppress host in.ammatory responses.’ (score: 0.0)\n",
            "Document #69: ‘The viral mutant carrying the corresponding mutations induces elevated level of PKR phosphoryla¬tion and pronounced production of in.ammatory cytokines (e.g., tumor necrosis factor, interleukin [IL]-6, and IL-8), whereas small-interfering RNA knockdown of PKR or treatment with C16, a PKR inhibitor, blocks this effect.’ (score: 0.0)\n",
            "Document #68: ‘RNA interference silencing of G3BP1 or mutation of speci.c nsp1ß residues (VS19GG) can abolish the antagonization of PKR activation.’ (score: 0.0)\n",
            "Document #66: ‘Here, we report that this event is to suppress cellular in.ammation and is mediated by the viral replicase protein nsp1ß.’ (score: 0.0)\n",
            "Document #214: ‘Given that some of the key anti-viral proteins (such as ADAR1, PARP13/ZAP, OAS, IRF3, IRF7, TBK1, and RNase L) localize to SGs and activate the anti-viral response, SGs may negatively affect viral replication [56–60].’ (score: 0.0)\n",
            "Document #65: ‘However, this molecule is inactivated in the cells infected with porcine reproductive and respiratory syndrome virus (PRRSV), an economically devastating pathogen to the world swine industry.’ (score: 0.0)\n",
            "Document #64: ‘Protein kinase R (PKR) is a critical host restriction factor against invading viral pathogens.’ (score: 0.0)\n",
            "Document #63: ‘So when is a domain really a domain? And if the native state isn’t always the folded state, which state do we use to determine membership in a domain family? Although our definition of a domain may need to be adapted and expanded, we can take comfort that, for now, when focusing on typical globular domains one only needs to ensure appropriate filtering of the data.’ (score: 0.0)\n",
            "Document #62: ‘The proteome is now seen as existing on a continuum of varying structural content, with structure not necessarily present under physiological conditions, and disorder and order interchangeable states dependent upon the environment and protein interactions.’ (score: 0.0)\n",
            "Document #61: ‘The challenge of structural domain categorization is only going to become more complex.’ (score: 0.0)\n",
            "Document #60: ‘At some point we will gather sufficient knowledge to consider categorization of IDPs into domain families, a somewhat daunting notion.’ (score: 0.0)\n",
            "Document #59: ‘Re-examination of our rejected targets here not only demonstrates the importance of filtering structural databases and applying a consistent descriptor for structural domains, it also raises questions for the near future.’ (score: 0.0)\n",
            "Document #58: ‘This reiterates how important it was to survey and re-qualify our consensus data prior to Dynameomics.’ (score: 0.0)\n",
            "Document #57: ‘Fortunately, our consensus approach, along with manual inspection, eliminated many of the domains with confirmed or putative disorder.’ (score: 0.0)\n",
            "Document #56: ‘Inspection of our rejected targets here revealed that many contain disordered regions.’ (score: 0.0)\n",
            "Document #55: ‘When we updated our CDD in 2009, the field of intrinsically disordered proteins was still an emerging field.’ (score: 0.0)\n",
            "Document #79: ‘Noticeably, SGs are increasingly emerging as a critical signaling platform for antiviral immunity and a key regulator of PKR activation (10, 13).’ (score: 0.0)\n",
            "Document #80: ‘Additionally, the activated PKR critically mediates the induction of interferons and modu¬lates cellular in.ammatory responses to build up an antiviral state by inducing sustained activation of NF-.B via activating I.B kinase (IKK) and others (14–16).’ (score: 0.0)\n",
            "Document #81: ‘To overcome PKR restriction, many viruses have evolved various strategies (2).’ (score: 0.0)\n",
            "Document #82: ‘They specify viral products to either interact directly with PKR (17–19), or sequester and degrade dsRNA (20, 21), or target host PKR regulators, such as ADAR1, P58IPK, and PACT (22–24).’ (score: 0.0)\n",
            "Document #105: ‘it does not require other viral factors, but rather depends on the cellular stress itself.’ (score: 0.0)\n",
            "Document #104: ‘Thus, the sorting of nsp1ß into SGs is an intrinsic property of this and colocalizes with PKR in SGs during infection.’ (score: 0.0)\n",
            "Document #103:  (score: 0.0)\n",
            "Document #102: ‘Treatment with other ER stress inducers (arsenite [Ars] and thapsigargin [TG]) obtained the same result (Fig.’ (score: 0.0)\n",
            "Document #101:  (score: 0.0)\n",
            "Document #100: ‘3A, Right).’ (score: 0.0)\n",
            "Document #99: ‘Indeed, treatment of MARC-145 cells with dithiothreitol (DTT), an endoplasmic reticulum (ER) stress inducer, readily led to relocation of ecto¬pically expressed nsp1ß to SGs (Fig.’ (score: 0.0)\n",
            "Document #98: ‘If this is the case, then exposure of nsp1ß in an arti.cial stress condition should allow it to enter SG.’ (score: 0.0)\n",
            "Document #97: ‘Since PRRSV prefers to induce a stress environment (e.g., induction of unfolded protein responses, phosphorylation of eIF2a, and so forth) conducive to replication (34), we hypothesized that nsp1ß might be a stress-responsive protein.’ (score: 0.0)\n",
            "Document #96: ‘PRRSV nsp1ß Is a Stress-Responsive Protein and Colocalizes with PKR in SGs During Infection.’ (score: 0.0)\n",
            "Document #95: ‘Strategically, PRRSV repur¬poses antiviral SGs into a proviral platform to overcome the PKR restriction.’ (score: 0.0)\n",
            "Document #94: ‘Here, we show that the primary pur¬pose of this selective targeting of PKR signaling is to suppress the cellular in.ammatory responses.’ (score: 0.0)\n",
            "Document #93: ‘We have previously reported that the eIF2a phosphorylation is conducive to utilization of ATF4 for viral replication (34).’ (score: 0.0)\n",
            "Document #92: ‘Interestingly, the activity of PKR is disabled in PRRSV-infected cells while the virus still allows the phosphorylation of its substrate eIF2a (32–34).’ (score: 0.0)\n",
            "Document #91: ‘These findings shed light on the mechanisms of PRRSV-mediated suppression of cellular inflammation, which might help in the rationale design of the vaccine, and reveal a unique strategy for viral evasion of PKR and SG restriction.’ (score: 0.0)\n",
            "Document #90: ‘Mechanistically, PRRSV takes advantage of virus-induced stress granules (SGs) and reprograms the antiviral SGs into a proviral platform by utilizing viral replicase protein nsp1ß to co-opt G3BP1 to inhibit PKR activation.’ (score: 0.0)\n",
            "Document #89: ‘Here, we show that this selective targeting of PKR signaling down-regulates cellular inflammatory responses.’ (score: 0.0)\n",
            "Document #88: ‘Enigmatically, PKR activity is disabled by porcine reproductive and respiratory syndrome virus (PRRSV) during infection, while the virus still allows the phosphorylation of its substrate eIF2a.’ (score: 0.0)\n",
            "Document #87: ‘Evolutionarily conserved protein kinase R (PKR) is a powerful mediator of antiviral responses.’ (score: 0.0)\n",
            "Document #86: ‘Conse¬quently, the currently available PRRS vaccines are unable to induce sterilizing immunity (30), and the viral persistence in turn facilitates recombination among different strains and allows selection of immune escape mutants in the .eld (30, 31).’ (score: 0.0)\n",
            "Document #85: ‘One of the most prominent features of PRRSV is subversion of host immunity, as manifested by crippled induction of interferon and in.ammation, leading to inefficient cytotoxic T cell responses (26, 28, 29).’ (score: 0.0)\n",
            "Document #84: ‘PRRSV has remained a major threat to the worldwide swine production ever since its .rst emergence in late 1980s (27).’ (score: 0.0)\n",
            "Document #83: ‘The focus of this report is the interplay of PKR signaling with porcine reproductive and respiratory syndrome virus (PRRSV), an economically devastating swine pathogen and a positive-stranded RNA virus in the family Arteriviridae of the order Nido¬virales (25, 26).’ (score: 0.0)\n",
            "Document #54: ‘Many of the structures assigned to domain families in CATH, SCOP and Dali were not strictly domains in a structural sense.’ (score: 0.0)\n",
            "Document #53: ‘In the course of this work, it became clear that – in some cases - there was much ambiguity as to what qualifies as a domain and what does not.’ (score: 0.0)\n",
            "Document #52: ‘In our Dynameomics project we generated a consensus set of protein folds and selected targets based on the physical attributes of what constituted a globular structural domain.’ (score: 0.0)\n",
            "Document #25: ‘Another 86 folds are composite domains where the structural unit is interrupted by structural elements from a second domain or chain.’ (score: 0.0)\n",
            "Document #23: ‘However, they typically form part of large complexes (PDB ID: 1eq2 in Fig.’ (score: 0.0)\n",
            "Document #22: ‘The 585 folds labeled as “non autonomous”, are not significantly larger than those folds in the other groupings, with only 11 containing over 450 residues.’ (score: 0.0)\n",
            "Document #21:  (score: 0.0)\n",
            "Document #20: ‘Moreover, putative G4-forming sequences have been found in coding and non-coding regions of the human transcriptome, i.e., open reading frames and untranslated regions.’ (score: 0.0)\n",
            "Document #19: ‘In general, G4 sequences are non-randomly distributed but mainly clustered in pivotal genomic regions, namely telomeres, gene promoters and DNA replication origins (15).’ (score: 0.0)\n",
            "Document #18: ‘These data indicate that G4s are mostly suppressed in chromatin and that, in turn, they may influence the occupancy and positioning of nucleosomes.’ (score: 0.0)\n",
            "Document #17: ‘However, mapping of G4s in chromatin by G4 ChIP-sequencing with an anti-G4 antibody (13) or foot printing (14) retrieved only about 10000 G4s in highly transcribed regulatory nucleosome-depleted chromatin regions.’ (score: 0.0)\n",
            "Document #16: ‘These data have been corroborated by 'G4-seq' high-throughput sequencing method, which identified about 700000 G4s (12).’ (score: 0.0)\n",
            "Document #15: ‘Computational analysis using different algorithms (9,10) indicated that 300000 and up to around 3 000000 potential G4-forming sequences may form in the human genome, correlated with specific gene functions (11).’ (score: 0.0)\n",
            "Document #14: ‘While RNA G4s are mostly locked in a parallel conformation due to the 2 -hydroxyl group in the sugar which exclusively allows the anti orientation (2), DNA G4s are in principle characterized by higher topological diversity, even though the majority of DNA G4s examined so far adopt the parallel topology.’ (score: 0.0)\n",
            "Document #13: ‘The anti conformation characterizes a parallel folding, while antiparallel G4s are found to adopt both syn and anti orientations (8) (Figure 1).’ (score: 0.0)\n",
            "Document #12: ‘Strands orientation defines the parallel, antiparallel or mixed topology of G4s, which is directly correlated to the conformational state, anti or syn, of the glycosidic bond between the G base and the sugar (1).’ (score: 0.0)\n",
            "Document #11: ‘G4s can fold intramolecularly from a single G-rich strand, or intermolecularly through dimerization or tetramerization of separate filaments: research of biologically relevant G4s has mainly focused on monomolecular G4s (3,4); however, intermolecular G4s are gaining increasing attention (5-7).’ (score: 0.0)\n",
            "Document #10: ‘G4s are highly polymorphic structures whose topology can be influenced by variations in strand stoichiometry and polarity, as well as by the nature and length of loops and their location in the sequence.’ (score: 0.0)\n",
            "Document #9: ‘Each tetrad is composed of four G residues that are linked by the sugar-phosphate backbone and connected through Hoogsteen-type hydrogen bonds.’ (score: 0.0)\n",
            "Document #8: 'G-quadruplexes (G4s) are nucleic acids secondary structures that can form within DNA (1) or RNA (2) guanine (G)-rich strands, when two or more G-tetrads stack on top of each other and coordinate monovalent cations, such as K+ and Na+.’ (score: 0.0)\n",
            "Document #7: ‘The current state of the art of G4 ligands in antiviral research is also reported, with particular focus on the structural and physicochemical requirements for optimal biological activity.’ (score: 0.0)\n",
            "Document #6: ‘This review aims to provide an updated close-up of the literature on G4s in viruses.’ (score: 0.0)\n",
            "Document #5: ‘In general, G4 ligands showed promising antiviral activity, with G4-mediated mechanisms of action both at the genome and transcript level.’ (score: 0.0)\n",
            "Document #4: ‘In this context, G4 ligands have been developed and tested both as tools to study the complexity of G4mediated mechanisms in the viral life cycle, and as therapeutic agents.’ (score: 0.0)\n",
            "Document #3: ‘Viral G4s are usually located in regulatory regions of the genome and implicated in the control of key viral processes; in some cases, they have been involved also in viral latency.’ (score: 0.0)\n",
            "Document #2: ‘Indeed, G4s have been reported in several viruses, including those involved in recent epidemics, such as the Zika and Ebola viruses.’ (score: 0.0)\n",
            "Document #1: ‘G4s have been extensively described in the human genome, especially in telomeres and oncogene promoters; in recent years the presence of G4s in viruses has at-attracted increasing interest.’ (score: 0.0)\n",
            "Document #24: ‘Most of these non-autonomous folds, being structured in the context of a huge complex or in the presence of binding partners, were rejected because they had large regions buried within a complex making them unlikely to form stable structure in isolation.’ (score: 0.0)\n",
            "Document #26: ‘Examples of these folds are the Rossman, immunoglobulin-like, and citrate synthase domain 1 folds: PDB IDs 1eq2, 1i31 and 1css, respectively, in Figure 4C.’ (score: 0.0)\n",
            "Document #51: ‘If the complexity of attempting to categorize well-structured proteins into domain families has taught us anything, we can be sure that as the gaps in protein fold space are filled, defining a domain will only become more challenging.’ (score: 0.0)\n",
            "Document #27: ‘Although it may seem strange to have structural units with spatially separated sequences, remember that these composite domains are present in the underlying databases as a consequence of the different methods used for categorization.’ (score: 0.0)\n",
            "Document #50: ‘However, at the start of such a revolution in understanding protein structure, or non-structure, it becomes even more important to be able to confidently define domain boundaries and group similar folds together before including ‘dynamic’ or ‘disordered’ protein fold families.’ (score: 0.0)\n",
            "Document #49: ‘Moreover, some disordered regions have already shown attributes consistent with domains, leading to consideration of disordered domain classification [54].’ (score: 0.0)\n",
            "Document #48: ‘Perhaps, this is an indication we should start to consider “dynamic domain” families? After all, conservation of disordered regions has been demonstrated [53].’ (score: 0.0)\n",
            "Document #47: ‘Many disordered proteins have some fold or structure, albeit less compact than the “typical” native states of globular proteins, with many able to form different conformations with different binding partners.’ (score: 0.0)\n",
            "Document #46: ‘It is possible that as we learn more, the folds we previously rejected for not being bona fide domains may be considered as such and we should be open to adapting or expanding the definition of domains.’ (score: 0.0)\n",
            "Document #45: ‘However, there has been increasingly more evidence accruing over the last years that all is not what it first seemed [25].’ (score: 0.0)\n",
            "Document #44: ‘Although there were some exceptions to this rule, they were believed to be just that.’ (score: 0.0)\n",
            "Document #43: ‘The dogma that the native, biologically active state is folded and structured is biased because of the limitations of experimental techniques.’ (score: 0.0)\n",
            "Document #42: ‘Much of what was believed for the last 50 years regarding the relationship between structure and function is now in question as more and more proteins with a local or global lack of structure are identified [25].’ (score: 0.0)\n",
            "Document #41: ‘Accordingly, these rejected non-autonomous folds have a high incidence of known and putative disordered regions.’ (score: 0.0)\n",
            "Document #40: ‘Coupled folding and binding is synonymous with the presence of disorder in unbound states.’ (score: 0.0)\n",
            "Document #39: ‘The heat shock locus HSLU domain (PDB ID: 1ht2) has two predicted disordered regions, one that matches with missing coordinates in the X-ray crystal structures [51, 52] and both are listed in DisProt.’ (score: 0.0)\n",
            "Document #38: ‘The C-terminal domain of topoisomerase II (PDB ID: 1bjt) has a region of missing coordinates representing an acidic region sensitive to protease degradation, leading to the suggestion that it is disordered [49, 50].’ (score: 0.0)\n",
            "Document #37: ‘NMR indicates that the unbound form of the CBP domain is 100% disordered [48].’ (score: 0.0)\n",
            "Document #36: ‘Although shown here in the complexed, structured form with the ACTR domain shaded in gray (Fig.’ (score: 0.0)\n",
            "Document #35: ‘The other domain is the receptor activator (ACTR) domain, which exists as a molten globule until it forms a complex with the CBP domain [48].’ (score: 0.0)\n",
            "Document #34: ‘A first of its kind, this fold representative was one of two domains that are disordered in isolation but become structured upon forming a heterodimer.’ (score: 0.0)\n",
            "Document #33: ‘The CBP domain (PDB ID: 1kbh) is an established IDP [47].’ (score: 0.0)\n",
            "Document #32: ‘Three of these in particular, the C-terminal domain of topoisomerase II (1bjt), cAMP response binding protein (CBP, 1kbh) domain, and heat shock locus HSLU (1ht2), have disorder predictions that correlate with regions already highlighted in DisProt (Fig.’ (score: 0.0)\n",
            "Document #31: ‘Greater disorder was indicated in 43 of these folds, which were either listed in DisProt or had predictions for more than 30 contiguous disordered residues, or both.’ (score: 0.0)\n",
            "Document #30: ‘Some 182 of the 488 targets had predicted disordered regions at least ten residues in length and approximately 70 had missing coordinates, some coincidental with disordered regions as observed above for the crystal structures.’ (score: 0.0)\n",
            "Document #29: ‘Disorder predictions were performed on the 488 continuous folds; the fragmented nature of the 86 composite domains prohibited meaningful sequence analysis.’ (score: 0.0)\n",
            "Document #28: ‘For example, at one of the top-levels in the classification hierarchy of CATH, structures are grouped where they have a similar spatial arrangement of secondary structure elements, irrespective of chain directionality or continuity [46].’ (score: 0.0)\n",
            "Document #106: ‘PKR was also relocalized to PRRSV-induced SGs (Fig.3C).’ (score: 0.0)\n",
            "Document #107: ‘This observation is consistent with previous reports that PKR is a stress-responsive protein and that its activation is dependent on SGs (10, 13, 36).’ (score: 0.0)\n",
            "Document #108: ‘In infected cells, PKR colocalized well with nsp1ß and the SG markers, whereas in mock-infected cells it exhibited a diffusive distribution (Fig.’ (score: 0.0)\n",
            "Document #188: ‘The local concentration of G3BP1, cGAS, and PKR within cytoplasmic foci is proposed to enhance PKR activity.’ (score: 0.0)\n",
            "Document #186: ‘G3BP1 also interacts with cGAS to form large complexes that enhance interaction with viral DNA to induce IFN [30,34,36].’ (score: 0.0)\n",
            "Document #185: ‘For example, the RGG domain of G3BP1 binds the RIG-I helicase domain and viral dsRNA to prevent RIG-I degradation and enhance Ifn-b mRNA expression [30].’ (score: 0.0)\n",
            "Document #184: ‘G3BP1 facilitates the initial IFN-ß response by binding and augmenting the function of viral RNA and DNA sensing proteins in the cytoplasm.’ (score: 0.0)\n",
            "Document #183: ‘G3BP1 regulates early innate responses through its ability to bind nucleic acids and proteins in these signaling pathways.’ (score: 0.0)\n",
            "Document #182: ‘Here, we focus on the role of G3BP1 during viral infection and the various strategies utilized by viruses to exploit or counteract G3BP1 functions.’ (score: 0.0)\n",
            "Document #181: ‘Given that G3BP1 impacts the life cycle of many viruses, it is important to distinguish what specific functional roles are played by G3BP1 at different infection stages.’ (score: 0.0)\n",
            "Document #180: ‘Knockout of both G3BP1 and G3BP2 is necessary to completely inhibit SG formation induced by many stressors, implying their importance to form protein–protein and protein–RNA interactions within SGs [29].’ (score: 0.0)\n",
            "Document #179: ‘SGs are a class of cytoplasmic biomolecular condensates enriched with mRNA–protein complexes assembled in response to stress, including viral infection.’ (score: 0.0)\n",
            "Document #178: ‘Well-studied functions of G3BP1 include participation in the innate response to infection through activation of viral nucleic acid sensors RIG-I and cGAS and as an essential component of stress granules (SGs) [29,30,32–35].’ (score: 0.0)\n",
            "Document #177: ‘For example, some viruses require G3BP1 for efficient viral replication, while the replication of other viruses is negatively affected by G3BP1 expression.’ (score: 0.0)\n",
            "Document #176: ‘Emergent data suggest that G3BP1 is one of the key proteins that impact the viral life cycle both positively and negatively [30,31].’ (score: 0.0)\n",
            "Document #175: ‘For example, knockdown of G3BP1 increases the expression of G3BP2 to compensate for the loss of G3BP1 expression and function, but not vice versa [23,29].’ (score: 0.0)\n",
            "Document #174: ‘The functions of G3BP1 and G3BP2 are redundant in some, but not all cases [29].’ (score: 0.0)\n",
            "Document #173: ‘The RRM domain is essential for RNA binding and RGG domain increases RNA binding as well as mediates protein–protein interactions.’ (score: 0.0)\n",
            "Document #172: ‘For instance, the NTF2L domain interacts with FGDF sequence motifs present in many proteins and the PxxP domain interacts with SH3 motifs.’ (score: 0.0)\n",
            "Document #171: ‘The N-terminal domains contain motifs for protein binding while the C-terminal domains bind nucleic acids.’ (score: 0.0)\n",
            "Document #170: ‘These domains include N-terminal nuclear transport factor 2-like (NTF2L), followed by an acidic domain, proline rich region (PxxP), an RNA recognition motif (RRM), and an arginine and glycine-rich region (RGG) at the C-terminus [27,28].’ (score: 0.0)\n",
            "Document #169: ‘G3BP1 and its paralog G3BP2 possess similar protein domain architecture composed of regions for protein, RNA and DNA interactions (Figure 1).’ (score: 0.0)\n",
            "Document #168: ‘G3BP1 function is further modulated by post-translational modifications and also implicated in many diseases such as cancer, neurodegeneration, and viral infection—the latter is the focus of this review [22–26].’ (score: 0.0)\n",
            "Document #167: ‘G3BP1 is highly conserved from yeast to humans and plays a key role in RNA metabolism, such as regulating the stability and translation of a subset of mRNAs, binding to specific transcripts during stress conditions, and activating interferon (IFN)-stimulated gene expression [18–21].’ (score: 0.0)\n",
            "Document #166: ‘protein-binding protein G3BP1 is an RNA-binding protein targeted by many viruses [17].’ (score: 0.0)\n",
            "Document #165: ‘Concomitantly, induction of cellular immunity is delayed, usually 1 mo following infection, and it is maintained at a low level, contributing to viral persistence in vivo (26, 30).’ (score: 0.0)\n",
            "Document #164: ‘PRRSV is a rather “sneaky” virus, and the infections often lead to low level production of in.ammatory cytokines in the early infec¬tion (26, 28, 29).’ (score: 0.0)\n",
            "Document #187: ‘The binding of the N-terminal region of cGAS with G3BP1 is required for activation of the RNA-dependent protein kinase PKR leading to co-localization as small cytoplasmic foci [35].’ (score: 0.0)\n",
            "Document #189: ‘G3BP1 is required for activation of cGAS as depletion of G3BP1 dramatically reduces DNA-induced IFN production [34].’ (score: 0.0)\n",
            "Document #162: ‘Insight into the Inflammation Regulation by PRRSV.’ (score: 0.0)\n",
            "Document #190: ‘These data suggest that G3BP1-mediated interactions with signaling molecules facilitate the initiation of downstream innate response pathway signaling.’ (score: 0.0)\n",
            "Document #213: ‘As demonstrated by one of the earliest studies using Semliki Forest virus (SFV), infection-induced translation arrest results in the assembly of transient SGs.’ (score: 0.0)\n",
            "Document #212: ‘These patterns highlight the importance of modulating SGs upon virus infection (Table 1).’ (score: 0.0)\n",
            "Document #211: ‘At least four distinct temporal patterns of SGs have been observed during infection with different viruses: (1) stable, (2) transient, (3) oscillating, or (4) no SGs (active blocking of SG induction) [53–55].’ (score: 0.0)\n",
            "Document #210: ‘In light of the viral infection, it may be possible that at the early infection stage, an increase in viral RNA concentration may relieve G3BP1 from its auto-inhibitory state, triggering G3BP1 condensation, with the protein serving as the scaffold to recruit other proteins/RNA required for viral replication.’ (score: 0.0)\n",
            "Document #209: ‘Increasing local RNA concentrations can increase their binding to G3BP1, relieving G3BP1 from the autoinhibitory state and resulting in more RNA recruitment and less aberrant RNA–RNA interaction [50–52].’ (score: 0.0)\n",
            "Document #208: ‘In the normal state, G3BP1 adopts a compact, closed auto-inhibitory state with intramolecular interaction between acidic and arginine-rich regions.’ (score: 0.0)\n",
            "Document #207: ‘G3BP1 is the central node of the protein–protein interaction network within SGs, where RNA serves as a molecular trigger for phase separation.’ (score: 0.0)\n",
            "Document #206: ‘Recent studies provided more mechanistic insights into the role of G3BP1 as the scaffold for SG assembly.’ (score: 0.0)\n",
            "Document #205: ‘These data suggest that G3BP1/G3BP2 act as a scaffold that recruits other proteins into SGs [47,50].’ (score: 0.0)\n",
            "Document #204: ‘RNAi-mediated knockdown or genetic knockout of both G3BP1 and G3BP2 results in cells that do not form SGs in response to most stressors, emphasizing the importance of G3BP1/G3BP2 in SG assembly [29].’ (score: 0.0)\n",
            "Document #203: ‘G3BP1 and G3BP2 form the core protein–protein interaction network which binds translation factors, mRNA binding proteins, long and translationally incompetent mRNAs as well as specific transcripts during stress conditions [19,48,49].’ (score: 0.0)\n",
            "Document #202: ‘Of note, G3BP1 itself undergoes several post-translational modifications, including phosphorylation, methylation, ubiquitination, and ADP-ribosylation during stress conditions [42,44,47].’ (score: 0.0)\n",
            "Document #201: ‘Several post-translational modifications (PTMs) control the dynamics of SGs [33,41–46].’ (score: 0.0)\n",
            "Document #200: ‘These untranslated mRNAs bound with ribonucleoproteins (mRNPs) cluster together with G3BP1, G3BP2, and several other RNA binding proteins to form RNA/protein-rich condensates in the cytoplasm.’ (score: 0.0)\n",
            "Document #199: ‘Once the elongating ribosomes run off from polysomes, the stalled transla¬tion initiation results in a sudden influx of untranslated mRNAs in the cytoplasm.’ (score: 0.0)\n",
            "Document #198: ‘The initiation complex containing the phospho¬rylated eIF2a becomes unavailable for recycling, which leads to a stall in translation initiation [40].’ (score: 0.0)\n",
            "Document #197: ‘The activated kinase phosphorylates the translation initiation factor eIF2a subunit [38,39].’ (score: 0.0)\n",
            "Document #196: ‘Upon stress, one of the following five kinases is activated: double-stranded RNA (dsRNA)-dependent PKR, protein kinase R-like endoplasmic reticulum kinase (PERK), heme-regulated inhibitor (HRI), general control nonderepressible-2 (GCN2), or microtubule affinity-regulating kinase 2 (MARK2).’ (score: 0.0)\n",
            "Document #195: ‘SGs assemble upon stress-induced translation arrest that results in the accumulation of free untranslated mRNAs and mRNA binding proteins in the cyto¬plasm [37].’ (score: 0.0)\n",
            "Document #194: A well-known function of G3BP1 during stress stimuli is its involvement in SG as¬sembly (Figure 2b).’ (score: 0.0)\n",
            "Document #193: ‘G3BP1 Forms the Core of Stress Granules (score: 0.0)\n",
            "Document #192: ‘For example, G3BP1, G3BP2, and their binding partner Caprin1 positively regulate the translation of multiple ISGs in response to IFN (Figure 2a) [21].’ (score: 0.0)\n",
            "Document #191: ‘G3BP1 can also facilitate the expression and translation of IFN-stimulated genes (ISGs).’ (score: 0.0)\n",
            "Document #163: ‘Our results also unexpectedly revealed a regulatory mechanism for PRRSV-mediated suppression of host cellular in.ammatory responses.’ (score: 0.0)\n",
            "Document #161: ‘Overall, these results suggest a strategy of two birds (G3BP1 and PKR) with one stone (nsp1ß).’ (score: 0.0)\n",
            "Document #109: ‘3C).’ (score: 0.0)\n",
            "Document #134: ‘9A and B).’ (score: 0.0)\n",
            "Document #132: ‘Only two mutants (VS19GG and GGK16AAA) were successfully rescued.’ (score: 0.0)\n",
            "Document #131: ‘ (score: 0.0)\n",
            "Document #130: G3BP1 Contributes to nsp1ß-Mediated Inhibition of PKR Activ¬ity.’ (score: 0.0)\n",
            "Document #129:  (score: 0.0)\n",
            "Document #128: ‘Together, these results suggest that the nsp1ß-PKR interaction is regulated and facilitated by cellular factors during infection.’ (score: 0.0)\n",
            "Document #127: ‘Consistent with this notion, nsp1ß showed partial colocalization with Flag-DRBM in the cytoplasm, but not with the full-length Flag-PKR or Flag-KD in transfected cells (Fig.’ (score: 0.0)\n",
            "Document #126: ‘4C), suggesting that a conformational change of PKR is likely necessary for the interaction.’ (score: 0.0)\n",
            "Document #125: ‘4D,lane 11). (score: 0.0)\n",
            "Document #124: ‘4D,lane 10),but the TF-DRBM mutant was abundantly recovered (Fig.’ (score: 0.0)\n",
            "Document #123: ‘To our surprise, GST-nsp1ß was unable to pull down the full-length PKR (TF-PKR) (Fig.’ (score: 0.0)\n",
            "Document #122: ‘Specifically, PRRSV nsp1ß was expressed as a GST fusion protein, whereas PKR or its deriva¬tives (DRBM and KD) were fused to the C terminus of bacterial chaperone trigger factor (TF) to promote protein solubility (Fig.’ (score: 0.0)\n",
            "Document #121: ‘The relevant proteins were expressed in a bacterial system.’ (score: 0.0)\n",
            "Document #120: ‘To test whether the interaction requires any eukaryotic factor, we carried out the in vitro binding assay.’ (score: 0.0)\n",
            "Document #119: ‘The co-IP assay showed that the mutant Flag-DRBM retained the ability to interact with nsp1ß to the level of full-length PKR (Fig.C, lane 6), whereas the mutant Flag-KD failed to interact suggesting that DRBM is the key region for binding to nsp1ß.’ (score: 0.0)\n",
            "Document #118: ‘WT PKR and its derivatives (Flag-DRBM and Flag-KD) were expressed either individually or in combination with HA-nsp1ß in HEK-293FT cells.’ (score: 0.0)\n",
            "Document #117: ‘The nsp1ß-binding region was further mapped by construction of PKR truncation mutants containing only the DRBM or the KD.’ (score: 0.0)\n",
            "Document #116: ‘The anti¬bodies to PKR could readily pull down nsp1ß from PRRSV-infected MARC-145 cell lysates and the same was true under the condition of transfection (Fig.’ (score: 0.0)\n",
            "Document #115: ‘We tested this hypothesis by coimmunoprecipitation assay (co-IP).’ (score: 0.0)\n",
            "Document #114: ‘It is likely that there is an interaction between nsp1ß and PKR.’ (score: 0.0)\n",
            "Document #113: ‘Regulated Interaction between nsp1ß and PKR.’ (score: 0.0)\n",
            "Document #112: ‘Thus, these findings suggest that SGs are likely the battleground for nsp1ß to counteract PKR.’ (score: 0.0)\n",
            "Document #111: ‘3D).’ (score: 0.0)\n",
            "Document #110: ‘In transfected cells, nsp1ß did not colocalize with PKR, but they became colocalized once the cells were stressed with SG inducers (e.g., DTT or Ars) (Fig.’ (score: 0.0)\n",
            "Document #133: ‘The mutant GGK16AAA displayed a growth property similar to the parental virus in both MARC-145 cells and PAMs, whereas the growth of VS19GG was reduced (Fig.’ (score: 0.0)\n",
            "Document #135: ‘Similar to that in the transfection condition, the interaction of nsp1ß with G3BP1 and PKR were almost diminished in VS19GG infected cells (Fig.’ (score: 0.0)\n",
            "Document #160: ‘Certainly, this is a different story from PKR antagonization and warrants further exploration in the future.’ (score: 0.0)\n",
            "Document #136: ‘This was accom¬panied by a concurrent elevation of PKR phosphorylation in both cell types (score: 0.0)\n",
            "Document #159: ‘9A and B).’ (score: 0.0)\n",
            "Document #158: ‘This notion is in line with the observation that the mutant VS19GG had a reduced growth rate in both cell types (Fig.’ (score: 0.0)\n",
            "Document #157: ‘8B and C).’ (score: 0.0)\n",
            "Document #156: ‘This seemingly confusing result may be explained by WT nsp1ß normally inhibiting the antiviral function of G3BP1 via interaction, whereas the mutant nsp1ß VS19GG fails to do so due to loss of interaction (Fig.’ (score: 0.0)\n",
            "Document #155: ‘10).’ (score: 0.0)\n",
            "Document #154: ‘As so, cation of the mutant VS19GG (Fig.’ (score: 0.0)\n",
            "Document #153: ‘Cytoplasmic SGs are increasingly emerging as a critical immune signaling platform to detect and restrict invading viral pathogens (10, 11).’ (score: 0.0)\n",
            "Document #152: ‘Repurposing SG into a Proviral Platform.’ (score: 0.0)\n",
            "Document #151: ‘Asterisks (*) indicate the statistical significance: *P < 0.05; ***P < 0.001; NS, no significance.’ (score: 0.0)\n",
            "Document #150: ‘Statistical analysis was performed by two-tailed Student’s t test, and error bars indicate means ± SDs.’ (score: 0.0)\n",
            "Document #149: ‘The relative abundance of p-PKR in each treat¬ment was expressed as fold-changes compared to the intensity of p-PKR in the .rst lane after being normalized against PKR and then ß-actin in the same lane.’ (score: 0.0)\n",
            "Document #148: ‘The signals of protein bands were determined by ImageJ.’ (score: 0.0)\n",
            "Document #147: ‘(B) The same as A, except cells were transfected to express HA-nsp1ß.(C and D) The same as A and B, except that siRNAs were used to target TIA1.’ (score: 0.0)\n",
            "Document #146: ‘The cells were then subject to Western blot analysis with antibodies to the indicated proteins.’ (score: 0.0)\n",
            "Document #145: ‘At 36 h post transfection, the cells were infected with PRRSV strain JXwn06 at an MOI of 0.1 for 36 h before being treated with poly(I:C) for 12 h.’ (score: 0.0)\n",
            "Document #144: ‘(A) MARC-145 cells were transfected with siRNAs targeting G3BP1 or scram¬bled siRNA (siNC).’ (score: 0.0)\n",
            "Document #143: ‘On the other hand, despite a reduced viral growth, the mutant G3BP1 contributes to nsp1ß-mediated inhibition of PKR activation.’ (score: 0.0)\n",
            "Document #142: ‘In contrast, production of IFN-ß mRNA was highly inducible As compared with WT virus, there was a slight up-regulation of IFN-ß mRNA, but for both mutants, suggesting that the mutational effect is less likely due to PKR activation.’ (score: 0.0)\n",
            "Document #141: ‘In all treat¬ments, the CT values for IFN-a mRNA were near or over 35 cycles , and therefore the fold-changes are not reliable.’ (score: 0.0)\n",
            "Document #140: ‘To investigate the biological significance, we measured the production of interferons and in.ammatory cytokines by RT-qPCR in PRRSV-infected cells.’ (score: 0.0)\n",
            "Document #139: ‘The Mutant nsp1ß VS19GG Induces Inflammatory Responses via G3BP1-PKR Axis.’ (score: 0.0)\n",
            "Document #138: ‘Thus, nsp1ß is responsible for inhibiting PKR activation dur¬ing infection.’ (score: 0.0)\n",
            "Document #137: as opposed to both WT and the mutant nsp1ß GGK16AAA .’ (score: 0.0)\n",
            "Document #431: ‘Together, these results demonstrate that NDV-induced SGs are dependent on the PKR/eIF2a signaling pathway.’ (score: 0.0)\n",
            "Document #433: SGs are considered to be dynamic structures.’ (score: 0.0)\n",
            "Document #866:  (score: 0.0)\n",
            "Document #719: ‘The direct visualization of G-quadruplex structures in human cellsas well as the co-crystal structure of an RNA helicase bound to a G-quadruplexes have provided important confirmations of their relevance to cell biology.’ (score: 0.0)\n",
            "Document #730: ‘Its concentration is used as a measurement of oxidative stress within a cell.’ (score: 0.0)\n",
            "Document #729: ‘Due to Guanine having a lower electron reduction potential than the other nucleotides bases,-oxo-'-deoxyguanosine (-oxo-dG), is a known major product of DNA oxidation.’ (score: 0.0)\n",
            "Document #728: ‘Guanine (G) bases in G-quadruplex have the lowest redox potential causing it to be more susceptible to the formation of -oxoguanine (-oxoG), an endogenous oxidized DNA base damage in the genome.’ (score: 0.0)\n",
            "Document #727: ‘These enzymes participate in BER to repair certain DNA lesions such as ,-dihydro--oxoguanine (-oxoG), which forms under oxidative stress to guanine bases.’ (score: 0.0)\n",
            "Document #726: ‘The BER pathway is signalled when it indicates an oxidative DNA base damage, where structures like, -0xoguanine-DNA glycosylase  (OGG), APE and G-quadruplex play a huge role in its repair.’ (score: 0.0)\n",
            "Document #725: ‘Base excision repair processes in cells have been proved to be reduced with aging as its components in the mitochondria begin to decline, which can lead to the formation of many diseases such as Alzheimer's disease (AD)_These G-quadruplex structures are said to be formed in the promoter regions of DNA through superhelicity, which favors the unwinding of the double helical structure of DNA and in turn loops the strands to form G­ quadruplex structures in guanine rich regions.’ (score: 0.0)\n",
            "Document #724: ‘AP site damage was found to be predominant in PQS regions of the genome, where formation of G-quadruplex structures is regulated and promoted by the DNA repair process, base excision repair (BER).’ (score: 0.0)\n",
            "Document #723: ‘AP site damage was also more prevalent in certain regions of the genome that contain specific active promoter and enhancer markers, some of which were linked to regions responsible for lung adenocarcinoma and colon cancer.’ (score: 0.0)\n",
            "Document #722: ‘Both of these genome-wide mapping sequencing methods, ChIP-sequencing and ARP, have indicated that AP site damage occurrence is nonrandom.’ (score: 0.0)\n",
            "Document #721: ‘T-loops and G-quadruplexes are described as the two tertiary DNA structures that protect telomere ends and regulate telomere length. Many of the genome regulatory processes have been linked to the formation of G-quadruplex structures, attributable to the huge role it plays in DNA repair of apurinic/apyrimidinic sites also known as AP sites. A new technique to map AP sites has been developed known as AP-seq which utilizes a biotin-labeled aldehyde-reactive probe (ARP) to tag certain regions of the genome where AP site damage occurrence has been significant. Another genome-wide mapping sequencing method known as ChIP-sequencing, was utilized to map both; damage in AP sites, and the enzyme responsible for its repair, AP endonuclease  (APE).’ (score: 0.0)\n",
            "Document #720: ‘The potential positive and negative roles of quadruplexes in telomere replication and function remains controversial.’ (score: 0.0)\n",
            "Document #718: ‘The roles of quadruplex structure in translation control are not as well explored.’ (score: 0.0)\n",
            "Document #651: 'virus Papanivirus – Panicum mosaic satellite virus Virtovirus – Tobacco mosaic satellite virus Albetovirus – Tobacco necrosis satellite virus, Family Sarthroviridae, Macronovirus – Macrobrachium satellite virus 1 (extra small virus) (unnamed genus) – Nilaparvata lugens commensal X virus (unnamed genus) – Chronic bee-paralysis satellite virus, Family Lavidaviridae – Virophages Single-stranded DNA satellite viruses, Genus Dependoparvovirus – Adeno-associated.' (score: 0.0)\n",
            "Document #717: ‘There are many studies that implicate quadruplexes in both positive and negative transcriptional regulation, including epigenetic regulation of genes like hTERT.  Function of G-quadruplexes have also been reported in allowing programmed recombination of immunologlobin heavy genes and the pilin antigenic variation system of the pathogenic Neisseria.’ (score: 0.0)\n",
            "Document #716: ‘ The DEAH/RHA helicase, DHX, has also been identified as a key G-quadruplex resolvase_o A metastasis suppressor protein NMH (also known as NME) was found to directly interact with G quadruplex in the promoter of the c-myc gene, and transcriptionally regulate c-myc. More recently, NMH was reported to interact with G-quadruplex in the promoter of the human telomerase (hTERT) gene and regulate hTERT expression.  The telomere-binding-factor- (TRF or TERF) was shown to bind to thousands of non-telomeric G-quadruplexes in the human genome by TRF ChIP-seq.’ (score: 0.0)\n",
            "Document #715: ‘ Quadruplex formation may be potentially damaging for a cell; the helicases WRN and Bloom syndrome protein have a high affinity for resolving DNA G-quadruplexes.’ (score: 0.0)\n",
            "Document #714: ‘It has been suggested that quadruplex formation plays a role in immunoglobulin heavy chain switching. As cells have evolved mechanisms for resolving (i.e., unwinding) quadruplexes that form.’ (score: 0.0)\n",
            "Document #713: ‘In another model, quadruplex formed at the non-coding DNA strand helps to maintain an open conformation of the coding DNA strand and enhance an expression of the respective gene.’ (score: 0.0)\n",
            "Document #712: ‘One model is shown below, with G­ quadruplex formation in or near a promoter blocking transcription of the gene, and hence de­ activating it.’ (score: 0.0)\n",
            "Document #711: ‘There are several possible models for how quadruplexes could influence gene activity, either by upregulation or downregulation.’ (score: 0.0)\n",
            "Document #710: ‘coli.’ (score: 0.0)\n",
            "Document #709: ‘A similar studies have identified putative G-quadruplexes in prokaryotes, namely the bacterium E.’ (score: 0.0)\n",
            "Document #708: ‘ Genome-wide surveys based on a quadruplex folding rule have been performed, which have identified ,000 Putative Quadruplex Sequences (PQS) in the human genome, although not all of these probably form in vivo.’ (score: 0.0)\n",
            "Document #707: ‘ Other genes shown to form G-quadruplexes in their promoter regions include the chicken -globin gene, human ubiquitin­ ligase RFP, and the proto-oncogenes c-kit, bcl-, VEGF, H-ras and N-ras.’ (score: 0.0)\n",
            "Document #731: ‘When DNA undergoes oxidative damage, a possible structural change in guanine, after ionizing radiation, gives rise to an enol form, -0H-Gua.’ (score: 0.0)\n",
            "Document #732: ‘This oxidative product is formed through a tautomeric shift from the original damage guanine, -oxo-Gua, and represents DNA damage that causes changes in the structure.’ (score: 0.0)\n",
            "Document #733: ‘This form allows for the base excision repair (BER) enzyme OGG to bind and remove the oxidative damage with the help of APE, resulting in an AP site.’ (score: 0.0)\n",
            "Document #734: ‘ Moreover, an AP site is a location in DNA that has neither a purine or a pyrimidine base due to DNA damage, they are the most prevalent type of endogenous DNA damage in cells.’ (score: 0.0)\n",
            "Document #757: ‘Some of these oncogenes include c-KIT, PDGF-A, c-Myc and VEGF, showing the importance of this secondary structure in cancer growth and development.’ (score: 0.0)\n",
            "Document #756: ‘The structures most present in the promoter regions of these oncogenes tend to be parallel-stranded G-quadruplex DNA structures.’ (score: 0.0)\n",
            "Document #755: ‘Along with the association of G-quadruplexes in telomeric regions of DNA, G-quadruplex structures have been identified in various human proto-oncogene promoter regions.’ (score: 0.0)\n",
            "Document #754: ‘For therapeutic advances, stabilizing the G­ quadruplexes of cancerous cells can inhibit cell growth and replication leading to the cell's death.’ (score: 0.0)\n",
            "Document #753: ‘This causes replication of damaged and cancerous cells.’ (score: 0.0)\n",
            "Document #752: ‘However, in cancer cells that have mutated helicase these complexes cannot be unwound and leads to potential damage of the cell.’ (score: 0.0)\n",
            "Document #751: ‘G-quadruplexes can inhibit normal cell function, and in healthy cells, are easily and readily unwound by helicase.’ (score: 0.0)\n",
            "Document #750: ‘Therefore, we can conclude that APE has two important roles in genome regulation- Stabilizing the formation of g-quadruplex structures and loading the transcriptional factors onto the AP site G-quadruplex forming sequences are prevalent in eukaryotic cells, especially in telomeres,  untranslated strands, and translocation hot spots.’ (score: 0.0)\n",
            "Document #749: ‘However, not all G-quadruplex structures require APE for formation, in fact some of them formed greater G-quadruplex structures in its absence.’ (score: 0.0)\n",
            "Document #748: ‘When a study downregulated the presence of APE and AcAPE in the cell, the formation of G-quadruplex structures was inhibited, which proves the importance of APE for the formation of these structures.’ (score: 0.0)\n",
            "Document #747: ‘Deacetylation of AcAPE is the driving force behind the loading of these TFs, where APE dissociates from the G-quadruplex structures.’ (score: 0.0)\n",
            "Document #746: ‘After this stage the APE gets acetylated by multiple lysine residues on the chromatin, forming acetylated APE (AcAPE). AcAPE is very crucial to the BER pathway as it acts as a transcriptional coactivator or corepressor, functioning to load transcription factors (TF) into the site of damage allowing it to regulate the gene expression.  AcAPE is also very important since it allows APE to bind for longer periods of time by delay of its dissociation from the sequence, allowing the repair process to be more efficient.’ (score: 0.0)\n",
            "Document #745: ‘This promotes formation of G-quadruplex structures by the folding of the stand.0 This looping process brings four bases in close proximity that will be held together by Hoogsteen base pairing.’ (score: 0.0)\n",
            "Document #744: ‘This AP site then signals cells to engage APE binding, which binds to the open duplex region.SThe binding of APE then plays an important role by stabilizing the formation of G-quadruplex structures in that region.’ (score: 0.0)\n",
            "Document #743: ‘Once the cell is aware of oxidative stress and damage, it recruits OGG to the site, whose main function is to initiate the BER pathway.OGG does so by cleaving the oxidized base and thus creating an AP site, primarily through the process of negative superhelicity.’ (score: 0.0)\n",
            "Document #742: ‘The oxidation of certain purine bases, like guanine, forms oxidized nucleotides that impairs DNA function by mismatching nucleotides in the sequences.This is more common in PQS sequences which form oxidized structures, such as - oxoguanine.’ (score: 0.0)\n",
            "Document #741: ‘APE is considered to be very crucial as AP site damage is known to be the most recurring type of endogenous damage to DNA.’ (score: 0.0)\n",
            "Document #740: ‘APE is mainly in charge of repairing damage caused to AP sites through the BER pathway.’ (score: 0.0)\n",
            "Document #739: ‘AP endonuclease  (APE) is an enzyme responsible for the promotion and the formation of G­ quadruplex structures.’ (score: 0.0)\n",
            "Document #738: ‘Upon insertion of -oxo-dG into thymidine kinase gene of humans, it was determine that if -oxo-dG was left unchecked and not repaired by BER, it can lead to frequent mutations and eventually carcinogenesis.’ (score: 0.0)\n",
            "Document #737: ‘The possible inactivation allows for un-repaired DNA damages to gather in non-replicating cells, like muscle, and can cause aging as well.  Moreover, oxidative DNA damage like -oxo-dG contributes to carcinogenesis through the modulation of gene expression, or the induction of mutations. On the condition that -oxo-dG is repaired by BER, parts of the repair protein is left behind which can lead to epigenetic alterations, or the modulation of gene expression.’ (score: 0.0)\n",
            "Document #736: ‘With the use of genome-wide ChIP-sequencing analyses, cell-based assays, and in vitro biochemical analyses, a connection has been made between oxidized DNA base-derived AP sites, and the formation of the G-quadruplex.  Furthermore, the concentration of -oxo-dG is a known biomarker of oxidative stress within a cell, and excessive amount of oxidative stress has been linked to carcinogenesis and other diseases_SWhen produced, -oxo-dG, has the ability to inactivate OGG, thus preventing the repair of DNA damage caused by the oxidation of guanine.’ (score: 0.0)\n",
            "Document #735: ‘AP sites can be generated spontaneously or after the cleavage of modified bases, like -0H-Gua.iJThe generation of an AP site enables the melting of the duplex DNA to unmask the PQS, adopting aG-quadruplex fold.’ (score: 0.0)\n",
            "Document #706: ‘The proto-oncogene c-myc forms a quadruplex in a nuclease hypersensitive region critical for gene activity.’ (score: 0.0)\n",
            "Document #705: ‘In addition, there was found more than one-billion-year conserved G-quadruplex locus in plants and algae, in gene encoding large subunit of RNA polymerase _i Although these studies predicted G-quadruplex-mediated gene regulation, it is unlikely that all pGs would form in vivo.’ (score: 0.0)\n",
            "Document #704: ‘Here also, like vertebrates G-quadruplexes were enriched within gene promoters.’ (score: 0.0)\n",
            "Document #677: ‘The hydrogen bonds in these layers are represented by blue dashed lines.’ (score: 0.0)\n",
            "Document #675: ‘The backbone is represented by a tube.’ (score: 0.0)\n",
            "Document #674: ‘These structures, formed from two separate G-D Structure of the intramolecular human telomeric G-quadruplex in potassium solution.’ (score: 0.0)\n",
            "Document #673: ‘Longer sequences, which contain two contiguous runs of three or more guanine bases, where the guanine regions are separated by one or more bases, only require two such sequences to provide enough guanine bases to form a quadruplex.’ (score: 0.0)\n",
            "Document #672: ‘The term G DNA was originally reserved for these tetramolecular structures that might play a role in meiosis However, as currently used in molecular biology, the term G can mean G-quadruplexes of any molecularity.’ (score: 0.0)\n",
            "Document #671: ‘Such a quadruplex is described as tetramolecular, reflecting the requirement of four separate strands.’ (score: 0.0)\n",
            "Document #670: ‘Short sequences, consisting of only a single contiguous run of three or more guanine bases, require four individual strands to form a quadruplex.’ (score: 0.0)\n",
            "Document #669: ‘The length of the nucleic acid sequences involved in tetrad formation determines how the quadruplex folds.’ (score: 0.0)\n",
            "Document #668: ‘Early evidence for the formation of G­ quadruplexes in vivo in cells was established by isolating them from cells, and later by the observation that specific DNA helicases could be identified where small molecules specific for these DNA structures accumulated in cells.’ (score: 0.0)\n",
            "Document #667: ‘Current research consists of identifying the biological function of these G­ Quadruplex structures for specific oncogenes and discovering effective therapeutic treatments for cancer based on interactions with G-quadruplexes.’ (score: 0.0)\n",
            "Document #666: ‘With the abundance of G-quadruplexes in vivo, these structures hold a biologically relevant role through interactions with the promoter regions of oncogenes and the telomeric regions of DNA strands.’ (score: 0.0)\n",
            "Document #665: ‘In 00, the prevalence of G-quadruplexes within gene promoters of several bacterial genomes was reported predicting G-quadruplex-mediated gene regulation.’ (score: 0.0)\n",
            "Document #664: ‘Interest in in vivo function of G-quadruplexes surged after large scale genome­ wide analysis showed the prevalence of potential G-quadruplex (pG)-forming sequences within gene promoters of human, chimpanzee, mouse, and rat - presented in the First International G-quadruplex Meeting held in April 00 in Louisville, Kentucky.’ (score: 0.0)\n",
            "Document #663: ‘The importance of discovering G-quadruplex structure was described through the statement, \"If G-quadruplexes form so readily in vitro, Nature will have found a way of using them in vivo\" - Aaron Klug, Nobel Prize Winner in Chemistry ().’ (score: 0.0)\n",
            "Document #662: ‘Their activities in basic genetic processes are an active area of research in telomere, gene regulation, and functional genomics research. The identification of structures with a high guanine association became apparent in the early 0s, through the identification of gel-like substances associated with guanines. More specifically, this research detailed the four-stranded DNA structures with a high association of guanines, which was later identified in eukaryotic telomeric regions of DNA.’ (score: 0.0)\n",
            "Document #661: ‘G-quadruplex structures can be computationally predicted from DNA or RNA sequence motifs, but their actual structures can be quite varied within and between the motifs, which can number over 00,000 per genome.’ (score: 0.0)\n",
            "Document #660: ‘They can be formed of DNA, RNA, LNA, and PNA, and may be intramolecular, bimolecular, or tetramolecular. Depending on the direction of the strands or parts of a strand that form the tetrads, structures may be described as parallel or antiparallel.’ (score: 0.0)\n",
            "Document #659: ‘The quadruplex structure is further stabilized by the presence of a cation, especially potassium, which sits in a central channel between each pair of tetrads.’ (score: 0.0)\n",
            "Document #658: ‘The placement and bonding to form G-quadruplexes is not random and serve very unusual functional purposes.’ (score: 0.0)\n",
            "Document #657: ‘Four guanine bases can associate through Hoogsteen hydrogen bonding to form a square planar structure called a guanine tetrad (G-tetrad or G-quartet), and two or more guanine tetrads (from G-tracts, continuous runs of guanine) can stack on top of each other to form a G-quadruplex.’ (score: 0.0)\n",
            "Document #656: ‘They are helical in shape and contain guanine tetrads that can form from one, two or four strands The unimolecular forms often occur naturally near the ends of the chromosomes, better known as the telomeric regions, and in transcriptional regulatory regions of multiple genes, both  in  microbes  and  across vertebrates SJincluding oncogenes in humans.’ (score: 0.0)\n",
            "Document #655: In molecular biology, G-quadruplex secondary structures (G) are formed in nucleic acids by sequences that are rich in guanine.’ (score: 0.0)\n",
            "Document #654: Viruses can be placed in one of the seven following groups including the dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses), ssDNA viruses (+ strand or \"sense\") DNA (e.g. Parvoviruses), dsRNA viruses (e.g. Reoviruses), (+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses) V, or ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses) VII.' (score: 0.0)\n",
            "Document #653: Defective interfering particles are defective viruses that have lost their ability to replicate except in the presence of a helper virus, which is normally the parental virus. They can also interfere with the helper virus.' (score: 0.0)\n",
            "Document #676: ‘The center of this structure contains three layers of G-tetrads.’ (score: 0.0)\n",
            "Document #678: ‘ Guanine-rich strands, are termed bimolecular quadruplexes.’ (score: 0.0)\n",
            "Document #703: ‘coli and hundreds of other microbial genomes.’ (score: 0.0)\n",
            "Document #679: ‘Finally, sequences which contain four distinct runs of guanine bases can form stable quadruplex structures by themselves, and a quadruplex formed entirely from a single strand is called an intramolecular quadruplex.’ (score: 0.0)\n",
            "Document #702: ‘Similarly, large number of G­ quadruplexes were found in the E.’ (score: 0.0)\n",
            "Document #701: ‘A large number of the non-telomeric G-quadruplexes were found within gene promoters, and were conserved across the species.’ (score: 0.0)\n",
            "Document #700: ‘Analysis of human, chimpanzee, mouse and rat genomes showed enormous number of potential G-quadruplex (pG)-forming sequences in non-telomeric regions.’ (score: 0.0)\n",
            "Document #699: ‘Quadruplexes are present in locations other than at the telomere.’ (score: 0.0)\n",
            "Document #698: ‘This is an active target of drug discovery, including telomestatin.’ (score: 0.0)\n",
            "Document #697: ‘The presence of these structures prevent telomere elongation by the telomerase complex.  Telomeric repeats in a variety of organisms have been shown to form these quadruplex structures in vitro, and subsequently they have also been shown to form in vivo. The human telomeric repeat (which is the same for all vertebrates) consists of many repeats of the sequenced (GGTTAG), and the quadruplexes formed by this structure can be in bead-like structures of  nm to  nm in size and have been well studied by NMR, TEM and X-ray crystal structure determination. The formation of these quadruplexes in telomeres has been shown to decrease the activity of the enzyme telomerase, which is responsible for maintaining length of telomeres and is involved in around % of all cancers.’ (score: 0.0)\n",
            "Document #696: ‘At these’ end protrusions, the G-rich overhang can form secondary structures such as G-quadruplexes if the overhang is longer than four TTAGGG repeats.’ (score: 0.0)\n",
            "Document #695: ‘The heterodimeric complex ribonucleoprotein enzyme telomerase adds TTAGGG repeats at the’ end of DNA strands.’ (score: 0.0)\n",
            "Document #694: ‘The repeats end with a’ protrusion of between 0 and 0 single-stranded TTAGGG repeats.’ (score: 0.0)\n",
            "Document #693: ‘These telomeric regions are characterized by long regions of double-stranded CCCTAA:TTAGGG repeats.’ (score: 0.0)\n",
            "Document #692: ‘Telomeres, rich in guanine and with a propensity to form g-quadruplexes, are located at the terminal ends of chromosomes and help maintain genome integrity by protecting these vulnerable terminal ends from instability.’ (score: 0.0)\n",
            "Document #691: ‘Telomeres function to provide this signaling.’ (score: 0.0)\n",
            "Document #690: ‘Because repair enzymes would naturally recognize ends of linear chromosomes as damaged DNA and would process them as such to harmful effect for the cell, clear signaling and tight regulation is needed at the ends of linear chromosomes.’ (score: 0.0)\n",
            "Document #689: ‘In addition, the propensity of g­ quadruplex formation during transcription in RNA sequences with the potential to form mutually exclusive hairpin or G-quadruplex structures depends heavily on the position of the hairpin-forming sequence.’ (score: 0.0)\n",
            "Document #688: ‘This effect seems to be mediated by alteration of the hydration of the DNA and its effect on Hoogsteen base pair bonding These quadruplexes seemed to readily occur at the ends of chromosome.’ (score: 0.0)\n",
            "Document #687: ‘For instance, quantitative assessments of the thermodynamics of molecular crowding indicate that the antiparallel g-quadruplex is stabilized by molecular crowding.’ (score: 0.0)\n",
            "Document #686: ‘Depending on cell type and cell cycle, mediating factors such as DNA-binding proteins on chromatin, composed of DNA tightly wound around histone proteins, and other environmental conditions and stresses affect the dynamic formation of quadruplexes.’ (score: 0.0)\n",
            "Document #685: ‘Following sequencing of the human genome, many guanine-rich sequences that had the potential to form quadruplexes were discovered.’ (score: 0.0)\n",
            "Document #684: ‘In quadruplexes formed from double-stranded DNA, possible interstrand topologies have also been discussed.  Interstrand quadruplexes contain guanines that originate from both strands of dsDNA.’ (score: 0.0)\n",
            "Document #683: ‘The loops joining runs of guanine bases in intramolecular antiparallel quadruplexes are either diagonal, joining two diagonally opposite runs of guanine bases, or lateral (edgewise) type loops, joining two adjacent runs of guanine base pairs.’ (score: 0.0)\n",
            "Document #682: ‘If one or more of the runs of guanine bases has a’-' direction opposite to the other runs of guanine bases, the quadruplex is said to have adopted an antiparallel topology.’ (score: 0.0)\n",
            "Document #681: ‘For intramolecular quadruplexes, this means that any loop regions present must be of the propeller type, positioned to the sides of the quadruplex.’ (score: 0.0)\n",
            "Document #680: ‘Depending on how the individual runs of guanine bases are arranged in a bimolecular or intramolecular quadruplex, a quadruplex can adopt one of a number of topologies with varying loop configurations. If all strands of DNA proceed in the same direction, the quadruplex is termed parallel.’ (score: 0.0)\n",
            "Document #758: ‘While the formation of G­ quadruplex structure vary to some extent for the different promoter regions of oncogenes, the consistent stabilization of these structures have been found in cancer development.’ (score: 0.0)\n",
            "Document #759: ‘Current therapeutic research actively focuses on targeting this stabilization of G-quadruplex structures to arrest unregulated cell growth and division.’ (score: 0.0)\n",
            "Document #760: ‘One particular gene region, the c-myc pathway, plays an integral role in the regulation of a protein product, c-Myc.’ (score: 0.0)\n",
            "Document #840: ‘Small-molecule ligands, composed primarily of lead, can target GGGGCC repeat regions as well and ultimately decreased both repeat-associated non-ATG translation and RNA foci in neuron cells derived from patients with Amyotrophic lateral sclerosis (ALS).’ (score: 0.0)\n",
            "Document #838: ‘A disadvantage of using small-molecule ligands as a therapeutic technique is that specificity is difficult to manage due to the variability of G-quadruplexes in their primary sequences, orientation, thermodynamic stability, and nucleic acid strand stoichiometry.’ (score: 0.0)\n",
            "Document #837: ‘Approximately ,000 various G­ quadruplex ligands exist in which they are able to interact via their aromatic rings; this allows the small-molecule ligands to stack on the planar terminal tetrads within the G-quadruplex regions.’ (score: 0.0)\n",
            "Document #836: ‘These can be used to target G-quadruplex regions that cause neurological disorders.’ (score: 0.0)\n",
            "Document #835: ‘Decoys have been successfully demonstrated to inhibit oncogenic KRAS in SCIO mice leading to reduced tumour growth and increased median survival time. Another commonly used technique is the utilization of small-molecule ligands.’ (score: 0.0)\n",
            "Document #834: ‘When introduced to cancer cells the decoy can intercept associated transcription factors and bind them leading to the regulation of gene expression.’ (score: 0.0)\n",
            "Document #833: ‘These decoys are typically composed of a G-rich sequence that can form a stable G-quadruplex structure and a short linker region that can be modified to optimize their properties.’ (score: 0.0)\n",
            "Document #832: ‘The strategy involves designing synthetic oligonucleotides that mimic the G-quadruplex structure and compete with the endogenous G-quadruplexes for binding to transcription factors.’ (score: 0.0)\n",
            "Document #831: ‘Antisense oligonucleotides (ASOs) are commonly used to target Corf RNA of the G-quadruplex GGGGCC expansion repeat region, which has lowered the toxicity in cellular models of Corf.0sJ00 ASOs have previously been used to restore normal phenotypes in other neurological diseases that have gain-of-function mechanisms, the only difference is that it was used in the absence of G­ quadruplex expansion repeat regions.0J0JoJThe G-quadruplex decoy strategy is another promising approach for targeting cancer cells by exploiting the unique structural features of the G-quadruplex.’ (score: 0.0)\n",
            "Document #830: ‘Therefore, these techniques are especially advantageous for targeting neurological diseases that have a gain-of-function mechanism, which is when the altered gene product has a new function or new expression of a gene; this has been detected in the Corf (chromosome  open reading frame ).0Antisense therapy is the process by which synthesized strands of nucleic acids are used to bind directly and specifically to the mRNA produced by a certain gene, which will inactivate it.’ (score: 0.0)\n",
            "Document #829: ‘This repeat expansion promotes DNA methylation and other epigenetic heterochromatin modifications of FMR that prevent the transcription of the gene, leading to pathological low levels of FMRP.0J0Antisense-mediated interventions and small-molecule ligands are common strategies used to target neurological diseases linked to G-quadruplex expansion repeats.’ (score: 0.0)\n",
            "Document #828: ‘ FMRP acts as a negative regulator of translation, and its binding stabilizes G-quadruplex structures in mRNA transcripts, inhibiting ribosome elongation of mRNA in the neuron's dendrite and controlling the timing of the transcript's expression.0o Mutations of this gene can cause the development of Fragile X Syndrome, autism, and other neurological disorders.0 Specifically, Fragile X Syndrome is caused by an increase from 0 to over 00 CGG repeats within exon  of the FMR gene.’ (score: 0.0)\n",
            "Document #827: ‘Fragile X mental retardation protein (FMRP) is a widely expressed protein coded by the FMR gene that binds to G-quadruplex secondary structures in neurons and is involved in synaptic plasticity.’ (score: 0.0)\n",
            "Document #826: ‘Nucleolin is involved in the synthesis and maturation of ribosomes within the nucleus, and separation of nucleolin by the mutated RNA transcripts impairs nucleolar function and ribosomal RNA synthesis.’ (score: 0.0)\n",
            "Document #825: ‘Normal individuals typically have around  to  GC repeats, but individuals with FTD or ALS have from 00 to several thousand GC repeats_The transcribed RNA of these repeats have been shown to form stable G-quadruplexes, with evidence showing that the GC repeats in DNA have the ability to form mixed parallel-antiparallel G- quadruplex structures as well_ These RNA transcripts containing GC repeats were shown to bind and separate a wide variety of proteins, including nucleolin.’ (score: 0.0)\n",
            "Document #824: ‘These two diseases have a causal relationship to GGGGCC (GC) repeats within the st intron of Corf gene.’ (score: 0.0)\n",
            "Document #823: ‘The second mechanism is through mutations that affect the expression of G-quadruplex binding proteins, as seen in the fragile X mental retardation gene  (FMR) gene and Fragile X Syndrome. The Corf gene codes for the protein Corf which is found throughout the brain in neuronal cytoplasm and at presynaptic terminals. Mutations of the Corf gene have been linked to the development of FTD and ALS.’ (score: 0.0)\n",
            "Document #822: ‘The first is through expansions of G-repeats within genes that lead to the formation of G-quadruplex structures that directly cause disease, as is the case with the Corf gene and amyotrophic lateral sclerosis (ALS) or frontotemporal dementia (FTD).’ (score: 0.0)\n",
            "Document #821: ‘G-quadruplexes have been implicated in neurological disorders through two main mechanisms.’ (score: 0.0)\n",
            "Document #820: ‘The novel approach combines solid-state nanopores and DNA nanotechnology for label-free detection of G-quadruplexes, for their mapping on dsDNA, and for monitoring G-quadruplex formation.’ (score: 0.0)\n",
            "Document #819: ‘Firstly, it was shown that biological nanopores can detect G-quadruplexes based on size exclusion and specific interaction of G-quadruplex and protein nanocavity.’ (score: 0.0)\n",
            "Document #818: ‘Another approach for detection of G-quadruplexes includes nanopore-based methods.’ (score: 0.0)\n",
            "Document #817: ‘Likewise, the thermostability of the G-quadruplex structure can be identified by observing the UV signal at  nm_ Upon G-quadruplex melting, the UV absorbance at  nm decreases, leading to a hypochromic shift that is a distinctive feature of G-quadruplex structure.’ (score: 0.0)\n",
            "Document #816: ‘To verify G-quadruplex formation, one should also perform the CD experiments under non-G-quadruplex stabilizing (Li+) and G-quadruplex stabilizing conditions (such as K+ or with G-quadruplex ligands), and scan toward the far-UV region (0- 0 nm).’ (score: 0.0)\n",
            "Document #839: ‘As of now, no single small-molecule ligand has been able to be perfectly specific for a single G-quadruplex sequence.  However, a cationic porphyrin known as TMPyP is able to bind to the Corf GGGGCC repeat region, which causes the G-quadruplex repeat region to unfold and lose its interactions with proteins causing it to lose its functionality.’ (score: 0.0)\n",
            "Document #841: ‘This provides evidence that small-molecule ligands are an effective and efficient process to target GGGGCC regions, and that specificity for small-molecule ligand binding is a feasible goal for the scientific community.’ (score: 0.0)\n",
            "Document #814: ‘The topology of the G-quadruplex structure can be determined by monitoring the positive or negative circular dichroism (CD) signals at specific wavelengths.’ (score: 0.0)\n",
            "Document #842: ‘Metal complexes have a number of features that make them particularly suitable as G DNA binders and therefore as potential drugs.’ (score: 0.0)\n",
            "Document #865: ‘These similarities have medical applications in terms of studying the structure and functions of both immune proteins and stress proteins, as well as the role each plays in combating disease.’ (score: 0.0)\n",
            "Document #864: ‘In addition, many stress proteins overlap with immune proteins.’ (score: 0.0)\n",
            "Document #863: ‘Early research has suggested that cells which are better able to synthesize stress proteins and do so at the appropriate time are better able to withstand damage caused by ischemia and reperfusion.’ (score: 0.0)\n",
            "Document #862: ‘Other toxins that either directly or indirectly lead to the release of free radicals can generate misfolded proteins.’ (score: 0.0)\n",
            "Document #861: ‘For example, many heavy metals can react with sulthydryl groups stabilizing proteins,  resulting  in  conformational changes.’ (score: 0.0)\n",
            "Document #860: ‘Many toxins end up activating similar stress proteins to heat or other stress-induced pathways because it is fairly common for some types of toxins to achieve their effects - at least in part - by denaturing vital cellular proteins.’ (score: 0.0)\n",
            "Document #859: ‘Heat shock proteins can also participate in marking malformed proteins for degradation via ubiquitin tags.’ (score: 0.0)\n",
            "Document #858: ‘This role is especially crucial since elevated temperature would, on its own, increase the concentrations of malformed proteins.’ (score: 0.0)\n",
            "Document #857: ‘These can help defend a cell against damage by acting as 'chaperons' in protein folding, ensuring that proteins assume their necessary shape and do not become denatured.’ (score: 0.0)\n",
            "Document #856: The heat shock response involves a class of stress proteins called heat shock proteins.’ (score: 0.0)\n",
            "Document #855: ‘Indeed, if a cell under normal (meaning unstressed) conditions has denatured proteins artificially injected into it, it will trigger a stress response.’ (score: 0.0)\n",
            "Document #854: ‘Because conditions such as high temperatures often cause proteins to denature, this mechanism enables cells to determine when they are subject to high temperature without the need of specialized thermosensitive proteins.’ (score: 0.0)\n",
            "Document #853: ‘Although stress response pathways are mediated in different ways depending on the stressor involved, cell type, etc., a general characteristic of many pathways - especially ones where heat is the principal stressor - is that they are initiated by the presence and detection of denatured proteins.’ (score: 0.0)\n",
            "Document #852: ‘As expected, such cells were also highly vulnerable to stress, and ceased to be viable at elevated temperature ranges.’ (score: 0.0)\n",
            "Document #851: ‘For example, studies in Drosophila have indicated that when DNA encoding certain stress proteins exhibit mutation defects, the resulting cells have impaired or lost abilities such as normal mitotic division and proteasome-mediated protein degradation.’ (score: 0.0)\n",
            "Document #850: ‘Stress proteins can exhibit widely varied functions within a cell- both during normal life processes and in response to stress.’ (score: 0.0)\n",
            "Document #849: ‘The essential character of these stress proteins in promoting the survival of cells has contributed to them being remarkably well conserved across phyla, with nearly identical stress proteins being expressed in the simplest prokaryotic cells as well as the most complex eukaryotic ones.’ (score: 0.0)\n",
            "Document #848: ‘Stress proteins often are further subdivided into two general categories: those that only are activated by stress, or those that are involved both in stress responses and in normal cellular functioning.’ (score: 0.0)\n",
            "Document #847: ‘Cellular stress responses are primarily mediated through what are classified as stress proteins.’ (score: 0.0)\n",
            "Document #846: ‘The various processes involved in cellular stress responses serve the adaptive purpose of protecting a cell against unfavorable environmental conditions, both through short term mechanisms that minimize acute damage to the cell's overall integrity, and through longer term mechanisms which provide the cell a measure of resiliency against similar adverse conditions.’ (score: 0.0)\n",
            "Document #845: ‘Cellular stress responses can also be caused by some viral infections.’ (score: 0.0)\n",
            "Document #844: ‘Cellular stress response is the wide range of molecular changes that cells undergo in response to environmental stressors, including extremes of temperature, exposure to toxins, and mechanical damage.’ (score: 0.0)\n",
            "Document #843: ‘While the metal plays largely a structural role in most G binders, there are also examples where it interacts directly with Gs by electrostatic interactions or direct coordination with nucleobases. (score: 0.0)\n",
            "Document #815: ‘Parallel G-quadruplexes have negative and positive CD signals at 0 and  nm, respectively, whereas antiparallel G-quadruplexes place these signals at  and  nm, respectively.’ (score: 0.0)\n",
            "Document #813: ‘In the DNA polymerase stop assay, the formation of a G-quadruplex in a DNA template can act as a roadblock and cause polymerase stalling, which halts the primer extension.  The dimethyl sulfate (DMS) followed by the piperidine cleavage assay is based on the fact that the formation of a G­ quadruplex will prohibit the N guanine methylation caused by DMS, leading to a protection pattern observed at the DNA G-quadruplex region after piperidine cleavage.’ (score: 0.0)\n",
            "Document #761: ‘With this product, the c-Myc protein functions in the processes of apoptosis and cell growth or development and as a transcriptional control on human telomerase reverse transcriptase.’ (score: 0.0)\n",
            "Document #786: ‘This main pathway of targeting this region results in the lack of telomerase elongation, leading to arrested cell development.’ (score: 0.0)\n",
            "Document #784: ‘To address this issue analog of TMPyP, it was synthesized known as Me which targets only G quadruplex DNA which inhibits cancer growth more effectively than TMPyP_ Ligand design and development remains an important field of research into therapeutic reagents due to the abundance of G-quadruplexes and their multiple conformational differences.’ (score: 0.0)\n",
            "Document #783: ‘However, TMPyP has been limited for used due to its non-selectivity toward cancer cell telomeres and normal double stranded DNA (dsDNA).’ (score: 0.0)\n",
            "Document #782: ‘These complexes have a high affinity for porphyrin rings which makes them effective anticancer agents.’ (score: 0.0)\n",
            "Document #781: ‘Telomeres are generally made up of G-quadruplexes and remain important targets for therapeutic research and discoveries.’ (score: 0.0)\n",
            "Document #780: ‘However, research has also identified the presence of G-quadruplex structures within this region due to the interaction of TMPyP with this promoter sequence.’ (score: 0.0)\n",
            "Document #779: ‘The presence of a guanine-rich sequence in the promoter region of PDGF-A has exhibited the ability to form intramolecular parallel G­ quadruplex structures and remains suggested to play a role in transcriptional regulation of PDGF-A.’ (score: 0.0)\n",
            "Document #778: ‘High levels of expression of PDGF have been associated with increased cell growth and cancer.’ (score: 0.0)\n",
            "Document #777: ‘Another oncogene pathway involving PDGF-A, platelet-derived growth factor, involves the process of wound healing and function as mitogenic growth factors for cells.’ (score: 0.0)\n",
            "Document #776: ‘The research on this pathway suggested the formation of a G­ quadruplex in the promoter region and an applicable target for therapeutic treatments.’ (score: 0.0)\n",
            "Document #775: ‘In certain types of cancers, the RET protein has shown increased expression levels.’ (score: 0.0)\n",
            "Document #774: ‘The guanine rich sequence in the promoter region for this pathway exudes a necessity for baseline transcription of this receptor tyrosine kinase.’ (score: 0.0)\n",
            "Document #773: ‘ The RET oncogene functions in the transcription of kinase which has been abundant in certain types of cancer.’ (score: 0.0)\n",
            "Document #772: ‘Current research on this pathway is focusing on discovering the biological function of this specific quadruplex formation on the c-kit pathway, while this quadruplex sequence has been noticed in various species.’ (score: 0.0)\n",
            "Document #771: ‘The rich guanine sequence of this promoter region has shown the ability to form a variety of quadruplexes.’ (score: 0.0)\n",
            "Document #770: ‘However, more research is necessary to determine whether the formation of G-quadruplex regulates the expression of this gene in a positive or negative manner.i The c-kit oncogene deals with a pathway that encodes an RTK, which was shown to have elevated expression levels in certain types of cancer.’ (score: 0.0)\n",
            "Document #769: ‘Through recent research into this specific gene pathway, the polypurine and polypyrimidine region allows for the transcription of this specific gene and the formation of an intramolecular G-quadruplex structure.’ (score: 0.0)\n",
            "Document #768: ‘Interaction of this G-quadruplex with TRF (also known as TERF) resulted in epigenetic regulation of p, which was tested using the G-quadruplex-binding ligand A.oHypoxia inducible factor la, HIF-a, remains involved in cancer signaling through its binding to Hypoxia Response Element, HRE, in the presence of hypoxia to begin the process of angiogenesis.’ (score: 0.0)\n",
            "Document #767: ‘The cyclin-dependent cell cycle checkpoint kinase inhibitor- CDKNlA (also known as p) gene harbours promoter G-quadruplex.’ (score: 0.0)\n",
            "Document #766: ‘The intramolecular G-quadruplex structures are formed mostly through the abundant guanine sequence in the promoter region of this specific pathway.’ (score: 0.0)\n",
            "Document #765: ‘Through recent research on the role of G-quadruplex function in vivo, the stabilization of G-quadruplex structures was shown to regulate VEGF gene transcription, with inhibition of transcription factors in this pathway.’ (score: 0.0)\n",
            "Document #764: ‘The formation of an intramolecular G-quadruplex structure has been shown through studies on the polypurine tract of the promoter region of the VEGF gene.’ (score: 0.0)\n",
            "Document #763: ‘Another gene pathway deals with the VEGF gene, Vascular Endothelial Growth Factor, which remains involved in the process of angiogenesis or the formation of new blood vessels.’ (score: 0.0)\n",
            "Document #762: ‘Interaction of c-Myc promoter G-quadruplex with NMH was shown to regulate c-Myc in cancer cells in 00 Regulation of c-myc through Human telomerase reverse transcriptase (hTERT) is also directly regulated through promoter G-quadruplex by interaction with the transcription factor NMH where epigenetic modifications were dependent on NMH-G-quadruplex association.Recently, hTERT epigenetic regulation reported to be mediated through interaction of hTERT promoter G­ quadruplex with the telomeric factor TRF.’ (score: 0.0)\n",
            "Document #785: ‘One type ofligand involving a Quindoline derivative, SYUIQ-0, utilizes the stabilization of G-quadruplexes in promoter regions to inhibit the production of both the c-Myc protein product and the human telomerase reverse transcriptase (hTERT).’ (score: 0.0)\n",
            "Document #787: ‘Further research remains necessary for the discovery of a single gene target to minimize unwanted reactivity with more efficient antitumor activity.’ (score: 0.0)\n",
            "Document #812: ‘These methods can be broadly defined into two classes: biophysical and biochemical methods.  Biochemical techniques were employed to interrogate G-quadruplex formation in a longer sequence context.’ (score: 0.0)\n",
            "Document #788: ‘One way of inducing or stabilizing G-quadruplex formation is to introduce a molecule which can bind to the G-quadruplex structure.’ (score: 0.0)\n",
            "Document #811: ‘A number of experimental methods have been developed to identify G-quadruplexes.’ (score: 0.0)\n",
            "Document #810: ‘This suggests that the sequences may be under positive selection enabled by the evolution of systems capable of suppressing non-B structure formation.’ (score: 0.0)\n",
            "Document #809: ‘the frequency) of these motifs has increased rapidly in the eumetazoa for which complete genomic sequences are available.’ (score: 0.0)\n",
            "Document #808: ‘In one study, it was found that the observed number per base pair (i.e.’ (score: 0.0)\n",
            "Document #807: ‘Although the rule effectively identifies sites of G-quadruplex formation, it also identifies a subset of the imperfect homopurine mirror repeats capable of triplex formationand C-strand i-motif formation.0 Moreover, these sequences also have the capacity to form slipped and foldback structures that are implicit intermediates in the formation of both quadruplex and triplex DNA structures.’ (score: 0.0)\n",
            "Document #806: ‘This rule has been widely used in on-line algorithms.’ (score: 0.0)\n",
            "Document #805: ‘Generally, a simple pattern match is used for searching for possible intrastrand quadruplex forming sequences: d(G+N_G+N_G+N_G+), where N is any nucleotide base (including guanine).’ (score: 0.0)\n",
            "Document #804: ‘Identifying and predicting sequences which have the capacity to form quadruplexes is an important tool in further understanding their role.’ (score: 0.0)\n",
            "Document #803: ‘Ligands with mobile side chains, ones that are able to rotate around its center chromophore, associate more strongly to G-quadruplexes because conformation of the G loops and the ligand side chains can align.’ (score: 0.0)\n",
            "Document #802: ‘Furthermore, the hydrogen bonds of ligands with smaller side chains are shorter and therefore stronger.’ (score: 0.0)\n",
            "Document #801: ‘It has been found that ligands with smaller side chains bind better to the quadruplex because smaller ligands have more concentrated electron density.’ (score: 0.0)\n",
            "Document #800: ‘The way in which TMPyP binds to G's is similar to MM, with the ring stacking onto the external G-quartet and side chains associating to the loops of G's.? When designing ligands to be bound to G-quadruplexes, the ligands have a higher affinity for parallel folded G-quadruplexes.’ (score: 0.0)\n",
            "Document #799: ‘TMPyP, a cationic porphyrin, is a more well known G binding ligand that helps to repress c-Myc.’ (score: 0.0)\n",
            "Document #798: ‘What makes this binding strong is the fluidity in the position of the loops to better associate with the ligand side chains.’ (score: 0.0)\n",
            "Document #797: ‘The quartet and the chromophore are bound with a :rc-:rc bond while the side chains and loops are not bound but are in close proximity.’ (score: 0.0)\n",
            "Document #796: ‘When bound, MM's central chromophore is situated on top of the’ terminal G-quartet and the side chains of the ligand associate to the loops of the quadruplex.’ (score: 0.0)\n",
            "Document #795: ‘The shape of the ligand is vital because it closely matches the quadruplex which has stacked quartets and the loops of nucleic acids holding it together.’ (score: 0.0)\n",
            "Document #794: ‘MM, a ligand that binds selectively for a quadruplex on the BCL- promoter, is shaped with a central core and  side chains branching sterically out.’ (score: 0.0)\n",
            "Document #793: ‘The binding of ligands to G-quadruplexes is vital for anti-cancer pursuits because G-quadruplexes are found typically at translocation hot spots.’ (score: 0.0)\n",
            "Document #792: ‘Cationic porphyrins have been shown to bind intercalatively with G-quadruplexes, as well as the molecule telomestatin.’ (score: 0.0)\n",
            "Document #791: ‘This has become an increasingly large field of research in genetics, biochemistry, and pharmacology.’ (score: 0.0)\n",
            "Document #790: ‘These ligands can be naturally occurring or synthetic.’ (score: 0.0)\n",
            "Document #789: ‘A number of ligands, which can be both small molecules and proteins, can bind to the G-quadruplex.’ (score: 0.0)\n",
            "Document #652: virus group Satellite nucleic acids, Single-stranded satellite DNAs, Family Alphasatellitidae (encoding a replication initiator protein) Family Tolecusatellitidae (encoding a pathogenicity determinant βC1)and Double-stranded satellite RNAs Single-stranded satellite RNAs.' (score: 0.0)\n",
            "Document #650: 'Satellite viruses include the Single-stranded RNA satellite viruses (unnamed family), Aumaivirus – Maize white line mosaic satellite.' (score: 0.0)\n",
            "Document #434: ‘The move¬ment of canonical SGs is associated with cytoskeletal ele¬ments, especially microtubules (40–42).’ (score: 0.0)\n",
            "Document #501: ‘Anti-TIA-1 (A, B)and anti-TIAR(C, D) were used as an SG marker.’ (score: 0.0)\n",
            "Document #512: ‘Likewise, Sendai virus PP forms complexes with unassembled N protein (N°).’ (score: 0.0)\n",
            "Document #511: ‘However, this binding is blocked when the N-protein is complexed with PP (NS) (50).’ (score: 0.0)\n",
            "Document #510: ‘The N-protein of VSV seems to bind to any available RNA species, irrespective of sequence.’ (score: 0.0)\n",
            "Document #509: ‘In paramyxoviruses, such as mumps, N-protein en¬capsulates full-length viral negative sense genomic and positive sense antigenomic RNAs to form the helical nucleocapsid template (25).’ (score: 0.0)\n",
            "Document #508: ‘In the course of RSV infection, viral inclusion bodies, which are marked by anti-N and -P anti-bodies, are spatially separated from SGs (22).’ (score: 0.0)\n",
            "Document #507: ‘In our study, NP partially localized in SGs, whereas almost no colocalization of PP and TIA-1 was observed, which is consistent with previous re¬ports.’ (score: 0.0)\n",
            "Document #506: ‘The line of evidence suggests that viral replication, but not viral protein, is essential for v-SGs.’ (score: 0.0)\n",
            "Document #505: ‘not trigger SG formation.’ (score: 0.0)\n",
            "Document #504: ‘Right: Intensity pro.le of linear region of interest (ROI) across the HeLa cell costained with SG markers and mRNA.’ (score: 0.0)\n",
            "Document #503: ‘Arrowheads: SG-negative cells.’ (score: 0.0)\n",
            "Document #502: ‘Arrows: SG-positive cells.’ (score: 0.0)\n",
            "Document #500: ‘Probes speci.cally hybridized to polyadenylated mRNA (green) (A, C ), NP mRNA (red) (B, D)were used to testspeci.c components in SGs.’ (score: 0.0)\n",
            "Document #649: 'Satellite-like nucleic acids resemble satellite nucleic acids, in that they replicate with the aid of helper viruses. However they differ in that they can encode functions that can contribute to the success of their helper viruses; while they are sometimes considered to be genomic elements of their helper viruses, they are not always found within their helper viruses.' (score: 0.0)\n",
            "Document #499: ‘At the indicated time points, cells were .xed and processed for FISH.’ (score: 0.0)\n",
            "Document #498: ‘HeLa cells were mock treated or infected with NDV at an MOI of 1 for 6, 12, and 18 h.’ (score: 0.0)\n",
            "Document #497: ‘Cellular, but not viral mRNA, is predominately recruited in NDV-induced SGs.’ (score: 0.0)\n",
            "Document #496: ‘Both NDV and RSV induce SG formation, and encode no C protein, suggesting that the C protein is a key player in determining the fate of SGs in Paramyxovirus infections.’ (score: 0.0)\n",
            "Document #495: ‘Our data demonstrate that NDV induced stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #494: ‘In respiratory syncytial virus (RSV), another member of the Paramyxoviridae family, SGs are triggered at 12 hpi and increase throughout the 24 h period of infection (22).’ (score: 0.0)\n",
            "Document #493: ‘It is suggested that MV and SeV encode C protein (46, 47), which limits the generation of intracellular double-stranded RNA and activation of PKR to inhibit eIF2a phosphoryla¬tion (48, 49).’ (score: 0.0)\n",
            "Document #492: ‘Measles virus (MV) and Sendai virus (SeV), 2 members of the Paramyxoviridae family, however, do not induce SGs during their infection.’ (score: 0.0)\n",
            "Document #491: To date, several viruses are known to modulate SG formation to promote their replication.’ (score: 0.0)\n",
            "Document #490: ‘Viral (score: 0.0)\n",
            "Document #489: ‘FISH was used to detect NDV negative-strand genome RNA.’ (score: 0.0)\n",
            "Document #513: ‘This complex pre¬vents N protein from self-assembling and, thus, favors nucleocapsid assembly and viral transcription (26).’ (score: 0.0)\n",
            "Document #514: ‘Besides, a recent report showed that NDV positive-but not negative-stranded RNA is located in SGs (51).’ (score: 0.0)\n",
            "Document #515: ‘In line with that, in our study, viral negative-stranded gRNA was not recruited to SGs.’ (score: 0.0)\n",
            "Document #516: ‘Therefore, we specu¬late that a fraction of the NDV NP was recruited to v-SGs through binding to cellular or viral positive-stranded RNAs, and additional unassembled NP° may form complexes with the PP and remains excluded from v-SGs.’ (score: 0.0)\n",
            "Document #539: ‘Overall, we found that: 1) NDV induced SGs to inhibit global protein translation through eIF2a phosphorylation; (score: 0.0)\n",
            "Document #538: ‘TIA-1/R knockdown led to a decrease in viral protein synthesis and extracellular virus yields, especially at the middle stage of infection.’ (score: 0.0)\n",
            "Document #537: ‘Similarly, in our study, NDV triggers stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #536: ‘In these cases, the depletion of SG key compo¬nents, such as TIA-1, TIAR, or G3BP, results in decreased virus replication, indicating the proviral function of SG proteins.’ (score: 0.0)\n",
            "Document #535: ‘The symbiotic relationship between SG formation and virus infection suggests that SG is a beneficial factor rather than an intrinsic defense mechanism during infections of some viruses.’ (score: 0.0)\n",
            "Document #534: ‘In other words, the formation of SGs is accompanied by virus infection (18, 22).’ (score: 0.0)\n",
            "Document #533: ‘In contrast to this, HCV and RSV trigger stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #532: ‘Most viruses exploit several mechanisms to antagonize SG formation, such as G3BP cleavage and PKR inactivation (15, 16, 47).’ (score: 0.0)\n",
            "Document #531: Correspondingly, viruses use several mechanisms to evade or even take advantage of SGs.’ (score: 0.0)\n",
            "Document #530: ‘An earlier study showed that disruption of microtubules (score: 0.0)\n",
            "Document #529: ‘The function of microtubules in SG formation under physical and chemical stimuli varies in different reports.’ (score: 0.0)\n",
            "Document #528: ‘Furthermore, PKR/eIF2a pathwayplayedakeyroleinNDV-induced SGs.’ (score: 0.0)\n",
            "Document #527: We confirmed the association between the PKR/eIF2a pathway and v-SG formation, indicating that NDV in¬fection activated PKR/eIF2a pathway significantly and inhibited global translation.’ (score: 0.0)\n",
            "Document #526: ‘In this report, (score: 0.0)\n",
            "Document #525: ‘Viral cytoplasmic bodies induced by SV5 infection do not colocalize with cellular markers for SGs.’ (score: 0.0)\n",
            "Document #524: ‘Paramyxovirus simian virus 5 (SV5) is a weak ac¬tivator of the PKR/eIF2a signaling pathway and thereby does not globally inhibit translation (52).’ (score: 0.0)\n",
            "Document #523: ‘Moreover, TIA-1 was colocalized with G3BP1, rPS6, eIF4E, eIF4G, or eIF3A in both ARS-treated cells and NDV-infected cells, indicating that NDV triggers bona fide SGs.’ (score: 0.0)\n",
            "Document #522: ‘In our study, CHX treat¬ment caused the disassembly of NDV-induced SGs.’ (score: 0.0)\n",
            "Document #521: ‘At the late stage of poliovirus infection, virus-induced SGs contain TIA but lack translation initiation factors, mRNA binding proteins, and most polyadenylated mRNA (17).’ (score: 0.0)\n",
            "Document #520: ‘VSV-induced pseudo-SGs lack eIF3 and eIF4A (37).’ (score: 0.0)\n",
            "Document #519: ‘An E3 mutant vaccinia virus induces AVGs, which are atypical SGs that neither contain the 40S ribosomal subunit, nor disassemble when treated with polysome-stabilizing drugs such as CHX and emetine (20).’ (score: 0.0)\n",
            "Document #518: ‘The cytoplasmic granules induced by viral infec¬tions are distinct from typical SGs in both composition and function.’ (score: 0.0)\n",
            "Document #517: ‘However, this hypothesis needs to be investi¬gated further.’ (score: 0.0)\n",
            "Document #488: Because v-SGs regulate protein translation in the course of virus infection, we asked whether SGs play a role in virus genome replication.’ (score: 0.0)\n",
            "Document #487: ‘Depletion of endogenous TIA-1 does not affect virus genome replication (score: 0.0)\n",
            "Document #486: ‘The results indicate that G3BP1 is also necessary for NDV replication by trans¬lation regulation.’ (score: 0.0)\n",
            "Document #459: ‘Similarly, SGs in poliovirus infected cells are largely devoid of positive-sense poliovirus RNA, but contain polyadenylated RNA.’ (score: 0.0)\n",
            "Document #457: ‘The over¬lapping intensity profiles of the SG markers and mRNA (as visualized in the line profiles in the fifth column) further demonstrate the colocalization pro¬file between cellular mRNA and SG markers, whereas most viral mRNA was not localized with SG markers (Fig.’ (score: 0.0)\n",
            "Document #456: ‘However, large amounts of NP mRNA were distributed throughout the cytoplasm and excluded from-SGs (Fig.7B, D).’ (score: 0.0)\n",
            "Document #455: ‘At 12 and 18 hpi, a small portion of NP mRNA was localized with TIA-1 or TIAR.’ (score: 0.0)\n",
            "Document #454: ‘Meanwhile, no obvious SGs were observed in NP mRNA–positive cells at 6 hpi.’ (score: 0.0)\n",
            "Document #453: ‘NP mRNA was first detected in some cells infected for 6 h with NDV, compared with no positive fluorescence in mock-treated cells.’ (score: 0.0)\n",
            "Document #452: ‘To verify whether NDV mRNA is a component of SGs, we examined infected cells with FISH probes that hybridize to viral NP mRNA.’ (score: 0.0)\n",
            "Document #451: ‘Because Poly (A) tail has also been found on the messenger RNAs of NDV (43), v-SGs may contain both viral and host mRNA.’ (score: 0.0)\n",
            "Document #450: ‘In contrast, almost all polyadenylated mRNA was recruited to v-SGs at 12 and 18 hpi (arrow).’ (score: 0.0)\n",
            "Document #449: ‘diffusely dispersed in the cytoplasm (arrowhead).’ (score: 0.0)\n",
            "Document #448: ‘Arrows: transfected cells.’ (score: 0.0)\n",
            "Document #447: ‘The cells were collected, .xed, and visualized at 12 hpi.’ (score: 0.0)\n",
            "Document #446: ‘Cells were pulsed with 1 mM puromycin for 1 h at 37°C and then subjected to Western blot analysis.B) HeLa cells were transfected with PJX40-Flag-WT eIF2a, S51A, and S51D mutant eIF2a, followed by treatment at 24 h after transfection with infection of NDV at an MOI of 1.’ (score: 0.0)\n",
            "Document #445: ‘It was noted that in uninfected cells or cells containing no SG dots, polyadenylated mRNA was Phosphorylation of eIF2a at Ser-51 is required for v-SGs.A) HeLa cells were mock treated or infected with NDV at an MOI of 1.’ (score: 0.0)\n",
            "Document #444: ‘Polyadenylated mRNA was specifically recruited to virus-induced SGs marked by both TIA-1 and TIAR.’ (score: 0.0)\n",
            "Document #443: ‘FISH was used to detect total cellular mRNA, as well as viral NP mRNA.’ (score: 0.0)\n",
            "Document #442: Having found that v-SGs benefit the replication of NDV by manipulating the translation system, we checked whether the SG formation and stalled trans¬lation initiation were predominantly directed against viral or host protein translation.’ (score: 0.0)\n",
            "Document #441: ‘NDV-induced SGs contain predominantly cellular mRNA but not viral mRNA (score: 0.0)\n",
            "Document #440: ‘Noc treatment induced the formation of small and circular granules, compared to the large and irregular SGs in NDV-infected cells.’ (score: 0.0)\n",
            "Document #439: ‘In cells treated with Noc, obvious depolymerization of microtubules was observed, but the formation of NDV-induced SGs was unaffected.’ (score: 0.0)\n",
            "Document #438: ‘Accordingly, to confirm whether NDV-induced SGs were canonical SGs, we evaluated the role of microtubules and microfilaments in the formation of v-SGs.’ (score: 0.0)\n",
            "Document #437: ‘One characteristic of SG-like structures is that they are not dependent on a cellular microtubule or microfilament network.’ (score: 0.0)\n",
            "Document #436: ‘Previous reports showed that vesicular stomatitis virus (VSV)-induced SG-like structures are distinct from canonical SGs (37).’ (score: 0.0)\n",
            "Document #435: ‘Conversely, the cytoskeleton seems unnecessary in virus-induced atypical SGs.’ (score: 0.0)\n",
            "Document #458: ‘7, sixth column).’ (score: 0.0)\n",
            "Document #460: ‘Poliovirus-induced SGs are also stable over the course of infection (44).’ (score: 0.0)\n",
            "Document #485: ‘9A, C).’ (score: 0.0)\n",
            "Document #461: ‘Overall, these results indicate that NDV-induced SGs predominantly recruit cellular mRNA, whereas a large portion of viral mRNA seem to escape from v-SGs.’ (score: 0.0)\n",
            "Document #484: ‘Similar to TIA-1 and TIAR, siG3BP1 resulted in the significant up-regulation of global translation levels (Figs.’ (score: 0.0)\n",
            "Document #483: ‘We next monitored global protein synthesis after NDV infection.’ (score: 0.0)\n",
            "Document #482: ‘In G3BP1-depleted cells infected with NDV, the expression of the viral NP and virus yields were significantly lower than in cells transfected with scrambled siRNAs, especially at 12 hpi.’ (score: 0.0)\n",
            "Document #481: ‘HeLa Cells were transfected with 2 siRNAs specific to G3BP1 and then infected with NDV.’ (score: 0.0)\n",
            "Document #480: ‘As expected, SG formation was ef¬ficiently inhibited in siG3BP1-transfected cells upon treatment .’ (score: 0.0)\n",
            "Document #479: ‘To verify whether the translation regulation is driven by v-SGs or TIA-1/R per se, we knocked down G3BP1, another key SG component, and evaluated virus rep¬lication efficiency.’ (score: 0.0)\n",
            "Document #478: ‘Depletion of endogenous G3BP1, another key SG component, also reduced NDV replication on HeLa cells.’ (score: 0.0)\n",
            "Document #477: ‘These results demonstrate that depletion of endogenous TIA-1 and TIAR reduces NP mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #476: ‘8C).’ (score: 0.0)\n",
            "Document #475: ‘However, lower NP mRNA levels in TIA-1/TIAR-knockdown cells, compared with that in scrambled cells, was detected in fractions 5–12 (Fig.’ (score: 0.0)\n",
            "Document #474: ‘The results showed that, in fractions 1–4, the NP mRNA level in siTIA-1/TIAR-treated cells was similar to that in control cells.’ (score: 0.0)\n",
            "Document #473: ‘To verify whether knockdown of endogenous TIA-1/TIAR influences the viral mRNA profile in individual fractions, NDV NP mRNA was detected by absolute real-time PCR quantification.’ (score: 0.0)\n",
            "Document #472: ‘8B).’ (score: 0.0)\n",
            "Document #471: ‘Similar to the ribosome profile results, the amounts of PABP and rPS6 proteins showed no sig¬nificant differences between siRNA and control groups (Fig.’ (score: 0.0)\n",
            "Document #470: ‘TIA-1, TIAR, and PABP, as typical SG markers, were all found mainly in monosomes in virus infection.’ (score: 0.0)\n",
            "Document #469: ‘Western blot analysis of individual fractions for ri¬bosomal protein S6 showed that compared with mock infection, under virus infection, rPS6 was detected in fractions 1–8, suggesting that small ribosomal protein tends to shift to lower density fractions that contain monosomes.’ (score: 0.0)\n",
            "Document #468: ‘8A).’ (score: 0.0)\n",
            "Document #467: ‘The ab¬sorbance of individual fractions distinguished mono¬somes (fractions 1–4) from polysomes (fractions 5–12) (Fig.’ (score: 0.0)\n",
            "Document #466: ‘To elucidate the mechanisms how TIA-1/TIAR regulate virus in¬fection, sucrose gradient analysis was performed to sepa¬rate monosomes and polysomes before viral mRNA quantification in TIA-1/TIAR-knockdown cells.’ (score: 0.0)\n",
            "Document #465: ‘Because v-SGs contain predominantly cellular mRNA, then we asked whether disruption of SGs would lead to the release of cellular mRNA that competitively binds to polysomes and thereby reduces viral mRNA translation.’ (score: 0.0)\n",
            "Document #464: ‘In addition to mRNA, SGs per se are highly dynamic domains containing mRNA accompanied by dynamic shuttling of TIA-1 and are in equilibrium with polysomes (35).’ (score: 0.0)\n",
            "Document #463: In tomato cells exposed to heat shock, mRNA transfers from polysomes to SGs, whereas mRNA moves from SGs to polysomes when cells recover from heat shock (45).’ (score: 0.0)\n",
            "Document #462: ‘Depletion of endogenous TIA-1 and TIAR reduced NP mRNA levels in polysomes (score: 0.0)\n",
            "Document #540: 2) blockage of SGs using siTIA-1 inhibited viral protein synthesis and extracellular virus yields; 3)NDV-induced SGs predominantly recruited cellular mRNA, whereas large amounts of NDV mRNA were excluded; and 4)de¬pletion of endogenous TIA-1/R and TIAR reduced NDV mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #541: ‘Therefore, we propose that host cells respond to NDV infection by SG formation.’ (score: 0.0)\n",
            "Document #542: ‘The degradation of SGs by siTIA-1 leads to the release of cel¬lular mRNA from SGs to the cytoplasm.’ (score: 0.0)\n",
            "Document #622: Suborder (-virineae), Family (-viridae),Subfamily (-virinae), Genus (-virus),Subgenus (-virus), and Species.' (score: 0.0)\n",
            "Document #620: 'The International Committee on Taxonomy of Viruses began to devise and implement rules for the naming and classification of viruses early in the 1970s, an effort that continues to the present. The ICTV is the only body charged by the International Union of Microbiological Societies with the task of developing, refining, and maintaining a universal virus taxonomy.[7] The system shares many features with the classification system of cellular organisms, such as taxon structure. However, some differences exist, such as the universal use of italics for all taxonomic names, unlike in the International Code of Nomenclature for algae, fungi, and plants and International Code of Zoological Nomenclature.[8].' (score: 0.0)\n",
            "Document #619: Viruses sensu stricto are defined operationally by the ICTV as a type of MGE (mobile genetic element) that encodes at least one protein that is a major component of the virion encasing the nucleic acid of the respective MGE and therefore the gene encoding the major virion protein itself or MGEs that are clearly demonstrable to be members of a line of evolutionary descent of such major virion protein-encoding entities. Any monophyletic group of MGEs that originates from a virion protein-encoding ancestor should be classified as a group of viruses.' (score: 0.0)\n",
            "Document #618: 'The currently accepted and formal definition of a 'virus' was accepted by the ICTV Executive Committee in November 2020 and ratified in March 2021, and is as follows:[6].' (score: 0.0)\n",
            "Document #617: 'The actual criteria used vary by the taxon, and can be inconsistent (arbitrary similarity thresholds) or unrelated to lineage (geography) at times.[5] The matter is, for many, not yet settled.[2].' (score: 0.0)\n",
            "Document #616: 'In July 2013, the ICTV definition of species changed to state: \"A species is a monophyletic group of viruses whose properties can be distinguished from those of other species by multiple criteria.\"[3] These criteria include the structure of the capsid, the existence of an envelope, the gene expression program for its proteins, host range, pathogenicity, and most importantly genetic sequence similarity and phylogenetic relationship.[4].' (score: 0.0)\n",
            "Document #615: 'Species form the basis for any biological classification system. Before 1982, it was thought that viruses could not be made to fit Ernst Mayr's reproductive concept of species, and so were not amenable to such treatment. In 1982, the ICTV started to define a species as \"a cluster of strains\" with unique identifying qualities. In 1991, the more specific principle that a virus species is a polythetic class of viruses that constitutes a replicating lineage and occupies a particular ecological niche was adopted.[2].' (score: 0.0)\n",
            "Document #614: 'A catalogue of all the world's known viruses has been proposed and, in 2013, some preliminary efforts were underway.[1].' (score: 0.0)\n",
            "Document #613: Viruses are classified by phenotypic characteristics, such as morphology, nucleic acid type, mode of replication, host organisms, and the type of disease they cause. The formal taxonomic classification of viruses is the responsibility of the International Committee on Taxonomy of Viruses (ICTV) system, although the Baltimore classification system can be used to place viruses into one of seven groups based on their manner of mRNA synthesis. Specific naming conventions and further classification guidelines are set out by the ICTV.' (score: 0.0)\n",
            "Document #612: Virus classification is the process of naming viruses and placing them into a taxonomic system similar to the classification systems used for cellular organisms.' (score: 0.0)\n",
            "Document #611:  (score: 0.0)\n",
            "Document #610: ‘At the same time, we report the first evidence that VHS physically interacts with ICP27 both in lysates of infected cells and in transfected cells.’ (score: 0.0)\n",
            "Document #609: ‘Consistent with this finding is the evidence that VHS does not accumulate in ICP27 virus-infected cells and that the VHS brought into cells by ICP27 virions is less effective that that brought into cells by wild-type virus.’ (score: 0.0)\n",
            "Document #608: ‘Indeed, at late times, the IEX-1 is degraded in ICP27 virus-infected cells at a rate similar to that in noninfected or VHS virus-infected cells.’ (score: 0.0)\n",
            "Document #607: ‘In this report, we have reexamined the degradation of GAPDH and IEX-1 mRNAs at early and late times in infection.’ (score: 0.0)\n",
            "Document #606: ‘In the description of their studies, they confused the cleavage events occurring during the early hours after in¬fection when the mRNAs are cleaved and partially degraded in a VHS-but not a ICP27-dependent manner with events taking place at late time intervals.’ (score: 0.0)\n",
            "Document #605: ‘They concluded that ICP27 is required to stabilize full-length, polyadenylated IEX mRNA and that VHS is not involved in this process.’ (score: 0.0)\n",
            "Document #604: ‘The time frame of their studies was selected on the basis of the presence of abundant amounts of ICP27 in the infected cells.’ (score: 0.0)\n",
            "Document #603: ‘Corcoran and colleagues (6) analyzed the degradation of AU-rich IEX-1 mRNA at relatively late times after infection.’ (score: 0.0)\n",
            "Document #602: ‘A dearth or lack of enzymes necessary to degrade the 5 cleavage product containing an intact cap may account for the lingering of the cleavage prod¬uct for many hours.’ (score: 0.0)\n",
            "Document #601: ‘The 3 product of the cleavage would be rapidly degraded 5 to 3 in the same man¬ner as the GAPDH mRNA.’ (score: 0.0)\n",
            "Document #600: ‘In the presence of TTP bound to AREs, VHS is expected to bind TTP and cleave the mRNA at the AREs.’ (score: 0.0)\n",
            "Document #599: ‘This is the normal pathway of degradation of mRNA and would lead to 5-to-3 exonucleolytic degradation of the cleaved RNA.’ (score: 0.0)\n",
            "Document #598: ‘In the absence of AREs, VHS is predicted to bind the cap-binding complex and decap the mRNA.’ (score: 0.0)\n",
            "Document #621: 'Viral classification starts at the level of realm and continues as follows, with the taxonomic suffixes in parentheses:[8]Realm (-viria), Subrealm (-vira), Kingdom (-virae),Subkingdom (-virites), Phylum (-viricota), Subphylum (-viricotina), Class (-viricetes),Subclass (-viricetidae), Order (-virales),.' (score: 0.0)\n",
            "Document #623: 'Unlike the system of binomial nomenclature adopted in cellular species, there is currently no standardized form for virus species names. At present, the ICTV mandates that a species name must contain as few words as possible while remaining distinct, and must not only contain the word virus and the host name.[9] Species names often take the form of [Disease] virus, particularly for higher plants and animals. In 2019, the ICTV published a proposal to adopt a more formalized system of binomial nomenclature for virus species names, to be voted on in 2020.[10] However, some virologists later objected to the potential naming system change, arguing that the debate came while many in the field were preoccupied due to the COVID-19 pandemic.[11].' (score: 0.0)\n",
            "Document #596: ‘We proposed a model of degradation of intrinsi¬cally stable mRNAs (GAPDH) and of ARE mRNAs on the basis of known structures of translated mRNAs, known path¬ways of mRNA decay, and the interaction of VHS protein with the cap-binding complex and TTP.’ (score: 0.0)\n",
            "Document #624: 'As of 2023, all levels of taxa except subrealm, subkingdom, and subclass are used. Six realms, one incertae sedis class, 22.' (score: 0.0)\n",
            "Document #648: Satellites depend on co-infection of a host cell with a helper virus for productive multiplication. Their nucleic acids have substantially distinct nucleotide sequences from either their helper virus or host. When a satellite subviral agent encodes the coat protein in which it is encapsulated, it is then called a satellite virus.' (score: 0.0)\n",
            "Document #647: 1and Genus Elaviroid and subsequent type species of Eggplant latent viroid.' (score: 0.0)\n",
            "Document #646: 'species of Avocado sunblotch viroid, Family Pospiviroidae , Genus Pospiviroid and subsequent type species of Potato spindle tuber viroid, Genus Hostuviroid and subsequent type species of Hop stunt viroid, Genus Cocadviroid and subsequent type species of Coconut cadang-cadang viroid, Genus Apscaviroid and subequent type species of Apple scar skin viroid, Genus Coleviroid and subsequet type species of Coleus blumei viroid.' (score: 0.0)\n",
            "Document #645: 'The system was not accepted by others due to its neglect of morphological similarities.[20]Infectious agents are smaller than viruses and have only some of their properties.[21][22] Since 2015, the ICTV has allowed them to be classified in a similar way as viruses are. Viroids include the Family Avsunviroidae[24], Genus Avsunviroid and subsequent type.' (score: 0.0)\n",
            "Document #644: Holmes (1948) used a Linnaean taxonomy with binomial nomenclature to classify viruses into 3 groups under one order, Virales. Group I includes Phaginae  which helps sustain bacterial evolution for survival under abiotic stress such as drought. Group II includes Phytophaginae Phaginae  which infects plants in order to reduce biomass when there are extreme frost, drought or environmental threat conditions to survival. Group III Zoophaginae which is part of the same infectious species that would target animals in order to activate or modify genomic signaling properties for survival in changed environmental conditions that would otherwise threaten offspring survival. .' (score: 0.0)\n",
            "Document #642: All viruses that have an RNA genome, and that encode an RNA-dependent RNA polymerase (RdRp), are members of the kingdom Orthornavirae, within the realm Riboviria.[18] Group III viruses possess double-stranded RNA genomes, e.g. rotavirus. Group IV viruses possess positive-sense single-stranded RNA genomes. Many well known viruses are found in this group, including the picornaviruses (which is a family of viruses that includes well-known viruses like Hepatitis A virus, enteroviruses, rhinoviruses, poliovirus, and foot-and-mouth virus), SARS virus, hepatitis C virus, yellow fever virus, and rubella virus. Group V: viruses possess negative-sense single-stranded RNA genomes. Ebola and Marburg viruses are well known members of this group, along with influenza virus, measles, mumps and rabies.' (score: 0.0)\n",
            "Document #641: Herpesviridae are enveloped virion with icosahedral capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #640: Poxviridae are Complex  coats virion with complex coats capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #639: Parvoviridae are naked virion with icosahedral capsid symmetry and ss nucleic acid types and belong to Group II viruses.' (score: 0.0)\n",
            "Document #638: Papovaviridaeare naked virion with icosahedral capsid symmetry and dscircular  nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #637: Adenoviridae are naked virion with icosahedral capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #636: Group II  viruses possess single-stranded DNA such as the pleolipoviridae (HHPV1, HRPV1). Group II viruses have an enveloped virion with ss nucleic acid type, ds nucleic acid type, and may be linear or cicular  The anelioviridiae are also group II viriuses that are not enveloped and called naked, and whose capsid symmetry is icosahedral and whose nucleic acid type is ss circular.' (score: 0.0)\n",
            "Document #635: Group I viruses possess double-stranded DNA. Viruses that cause chickenpox and herpes are found here.' (score: 0.0)\n",
            "Document #634: Viruses with a DNA genome, except for the DNA reverse transcribing viruses, are members of three of the four recognized viral realms: Duplodnaviria, Monodnaviria, and Varidnaviria. But the incertae sedis order Ligamenvirales, and many other incertae sedis families and genera, are also used to classify DNA viruses. The domains Duplodnaviria and Varidnaviria consist of double-stranded DNA viruses; other double-stranded DNA viruses are incertae sedis. The domain Monodnaviria consists of single-stranded DNA viruses that generally encode a HUH endonuclease; other single-stranded DNA viruses are incertae sedis.' (score: 0.0)\n",
            "Document #633: 'placed in one of the seven following groups:[16].' (score: 0.0)\n",
            "Document #632: 'Baltimore classification (first defined in 1971) is a classification system that places viruses into one of seven groups depending on a combination of their nucleic acid (DNA or RNA), strandedness (single-stranded or double-stranded), sense, and method of replication. Named after David Baltimore, a Nobel Prize-winning biologist, these groups are designated by Roman numerals. Other classifications are determined by the disease caused by the virus or its morphology, neither of which are satisfactory due to different viruses either causing the same disease or looking very similar. In addition, viral structures are often difficult to determine under the microscope. Classifying viruses according to their genome means that those in a given category will all behave in a similar fashion, offering some indication of how to proceed with further research. Viruses can be in I  dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses), II: ssDNA viruses (+ strand or \"sense\") DNA (e.g. Parvoviruses), III: dsRNA viruses (e.g. Reoviruses), IV:(+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses) V: (−)ssRNA viruses (− strand or antisense) RNA (e.g. Orthomyxoviruses, Rhabdoviruses), VI: ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses) VII: dsDNA-RT viruses DNA with RNA intermediate in life-cycle (e.g. Hepadnaviruses), .' (score: 0.0)\n",
            "Document #631: 'The ICTV has gradually added many higher-level taxa using relationships in protein folds. All four realms defined in the 2019 release are defined by the presence of a protein of a certain structural family.[15].' (score: 0.0)\n",
            "Document #630: 'It has been suggested that similarity in virion assembly and structure observed for certain viral groups infecting hosts from different domains of life (e.g., bacterial tectiviruses and eukaryotic adenoviruses or prokaryotic Caudovirales and eukaryotic herpesviruses) reflects an evolutionary relationship between these viruses.[13] Therefore, structural relationship between viruses has been suggested to be used as a basis for defining higher-level taxa – structure-based viral lineages – that could complement the ICTV classification scheme of 2010.[14].' (score: 0.0)\n",
            "Document #629: 'Incertae sedis genera: Dinodnavirus, Rhizidiovirus.' (score: 0.0)\n",
            "Document #628: 'Itzamnaviridae Naldaviricetes Ovaliviridae Plasmaviridae Polydnaviriformidae Portogloboviridae Pospiviroidae Rhodogtaviriformidae Spiraviridae Thaspiviridae Tolecusatellitidae.' (score: 0.0)\n",
            "Document #627: 'Alphasatellitidae Ampullaviridae Anelloviridae Avsunviroidae Bartogtaviriformidae Bicaudaviridae Brachygtaviriformidae Clavaviridae Fuselloviridae Globuloviridae Guttaviridae Halspiviridae.' (score: 0.0)\n",
            "Document #626: 'Realms include: Duplodnaviria, Monodnaviria, Adnaviria, Ribozyviria, Riboviria, and Varidnaviria while Incertae sedis families and classes include:.' (score: 0.0)\n",
            "Document #625: 'incertae sedis families, and two incertae sedis genera are recognized:[12].' (score: 0.0)\n",
            "Document #597: ‘In brief, in the course of mRNA translation, it is the poly(A)-binding protein at the 3 end that physically abuts the cap-binding complex at the 5 end of the mRNA.’ (score: 0.0)\n",
            "Document #595: ‘In addition, this laboratory has shown that VHS is active and degrades mRNA in polyribo-somes (49).’ (score: 0.0)\n",
            "Document #543: ‘The released host mRNA competitively binds to polysomes and thereby re¬duces the translation ratio of viral mRNA, ultimately leading to a decrease of viral translation level and extra¬cellular virus yields.’ (score: 0.0)\n",
            "Document #568: ‘At late times, the VHS RNase is neutralized by the viral structural proteins VP16 and VP22.’ (score: 0.0)\n",
            "Document #566: ‘Pre-existing stable mRNAs (e.g., GAPDH [glyceraldehyde-3-phosphate dehydrogenase]) are rapidly degraded.’ (score: 0.0)\n",
            "Document #565: ‘The virion host shutoff (VHS) RNase tegument protein released into cells by infecting virus has two effects.’ (score: 0.0)\n",
            "Document #564: ‘Understanding how NDV manipulates stress re¬sponses will lay the groundwork for developing more ef¬fective oncolytic viruses and novel anticancer strategies.’ (score: 0.0)\n",
            "Document #563: ‘However, the hypothesis should be investigated further.’ (score: 0.0)\n",
            "Document #562: ‘That NDV efficiently and selectively replicates in and kills tumor and chicken cells, but not in other normal mammalian cells leads us to spec¬ulate whether the cell stress responses, such as autophagy and SGs, contribute to the tumor-selective replication of NDV.’ (score: 0.0)\n",
            "Document #561: ‘The relationship between autophagy and NDV has been identified in lung cancer cells and human glioma cells (57–59).’ (score: 0.0)\n",
            "Document #560: ‘We also found that NDV infection in¬duced cytoplasmic SG dots in chicken fibroblast cells DF-1 and primary chicken embryo fibroblast cells, and ectopic expression of GFP tagged chicken G3BP1 increased NDV replication (unpublished results).’ (score: 0.0)\n",
            "Document #559: ‘TIA-1 and TIAR regulated NDV replication by manipulating cellular protein translation on HeLa cells, where NDV selectively replicates (55, 56).’ (score: 0.0)\n",
            "Document #558: ‘In our current study, NDV triggered stable formation of bona fide SGs through the PKR/eIF2a pathway.’ (score: 0.0)\n",
            "Document #557: ‘In our study, knockdown of SG key components TIA-1, TIAR, or G3BP1 reduced NDV protein expression and extracellular virus yields, demonstrating that SGs play a proviral role in NDV infection.’ (score: 0.0)\n",
            "Document #556: ‘It is possible that both pro-and antiviral effects of SGs are at play and that the virus has developed mechanisms to overcome the antiviral effects of SGs.’ (score: 0.0)\n",
            "Document #555: ‘However, they may only play a marginal role in NDV replication, which was confirmed by the little or some¬what negative effect of siG3BPs on viral N and L protein expression (51).’ (score: 0.0)\n",
            "Document #554: ‘In tumor cells, SGs play an important role in recognizing uncapped viral RNA and facilitating IFN-b production.’ (score: 0.0)\n",
            "Document #553: ‘NDV infections induce IFN-b production in both normal and tumor cells (24, 55); however, pretreatment with endogenous or exogenous IFN-b has little or no effect on reducing the NDV yield or replication in tumor cells, although IFN-b treatment completely inhibits NDV replication in normal cells (55).’ (score: 0.0)\n",
            "Document #552: ‘A recent publication refuted the claim that NDV-induced SGs enhance IFN induction on HeLa cells; however, no virus replication data were shown (51).’ (score: 0.0)\n",
            "Document #551: ‘Similarly, in our study, siG3BP1 inhibited viral protein expression and extracellular virus yields, and meanwhile led to the increase in global protein trans¬lation, indicating that the translation balance is mediated by v-SGs rather than TIA-1/R per se.’ (score: 0.0)\n",
            "Document #550: ‘RSV replication is impaired in G3BP knockdown cells compared to its replication in control cells (22).’ (score: 0.0)\n",
            "Document #549: ‘For instance, G3BP1 and G3BP2 synergistically regulate the formation of SGs in various human cells (54).’ (score: 0.0)\n",
            "Document #548: ‘Except for TIA-1/R, other cellular proteins are in¬volved in SG assembly.’ (score: 0.0)\n",
            "Document #547: ‘In addition, TIA-1/R interacts with West Nile virus RNA and facilitates virus replication (53).’ (score: 0.0)\n",
            "Document #546: ‘Both TIA-1 and TIAR are RNA binding proteins and function in a similar manner in translation inhibition and SG formation (1, 11, 39).’ (score: 0.0)\n",
            "Document #545: ‘The translational shutoff is delayed in TIA-12/2 murine embryonic fibroblasts (11), indicating TIA function on the inhibition, which is in accord with our results.’ (score: 0.0)\n",
            "Document #544: ‘Semliki forest virus causes translation inhibition of host cellular proteins through eIF2a phosphorylation.’ (score: 0.0)\n",
            "Document #567: ‘Stress response RNAs containing AU-rich elements (AREs) in the 3 untranslated region (3 UTR) are dead¬enylated and cleaved, but the cleavage products persist for hours, in contrast to the short half-lives of ARE-containing mRNAs in uninfected cells.’ (score: 0.0)\n",
            "Document #569: ‘A recent study reported that, at relatively late times after infection, ARE RNAs are rapidly degraded in cells infected with ICP27 mutant virus and concluded that ICP27 “stabilizes” ARE mRNAs.’ (score: 0.0)\n",
            "Document #594: ‘The VHS protein was also reported to bind cap-binding proteins (9, 17, 18, 19, 31).’ (score: 0.0)\n",
            "Document #570: ‘The rates of degradation of ARE mRNA at early times (3 h) after infection with the wild type or the ICP27 mutant virus are virtually identical, and hence ICP27 plays no role in this process.’ (score: 0.0)\n",
            "Document #593: (i) its mRNA contains AREs (3), (ii) TTP physically interacts with VHS protein (15), and (iii) in uninfected cells, TTP binds AREs and sequesters the mRNAs for degradation by exo¬somes (5, 28).’ (score: 0.0)\n",
            "Document #592: ‘TTP is of particular interest for three reasons: (score: 0.0)\n",
            "Document #591: ‘Lastly, a subset of mRNAs exemplified by GADD45 and tristetraprolin (TTP) directed the synthesis of corresponding proteins and were not degraded (14).’ (score: 0.0)\n",
            "Document #590: ‘Whereas the 3 cleavage product rapidly disappeared, the 5 portion of the cleaved RNA lingered for many hours.’ (score: 0.0)\n",
            "Document #589: ‘In contrast, stress response mRNAs contain¬ing AU-rich elements (AREs) in the 3 untranslated region (3 UTR), exemplified by IEX-1, c-Fos, IkB, or cox-2, which in uninfected cells share very short half-lives (1, 4, 51), were deadenylated and cleaved at or near the AREs (14).’ (score: 0.0)\n",
            "Document #588: ‘Thus, mRNAs that are highly stable in uninfected cells, exemplified by GAPDH (glyceraldehyde-3-phosphate dehydrogenase) and actin mRNAs, were rapidly degraded.’ (score: 0.0)\n",
            "Document #587: ‘This laboratory was the first to report that the degradation of mRNA during the early hours after infection by wild-type virus was highly selective (14, 16, 45).’ (score: 0.0)\n",
            "Document #586: 1991 that VHS mediated the degradation of some mRNAs but spared other mRNAs, most notably, histone mRNAs, the issue was still debated as late as 2004 (reviewed in reference 39).’ (score: 0.0)\n",
            "Document #585: ‘Even though Soreson and colleagues (41) reported as early as (score: 0.0)\n",
            "Document #584: ‘Most of the early work was done on model viral mRNAs, and these studies sustained the conclusion that VHS activity was not selective.’ (score: 0.0)\n",
            "Document #583: ‘Studies on the incorporation of radioactive amino acids into proteins after infection in cells infected with wild-type virus but not with a mutant lacking VHS activity led to the speculation that VHS activity is nonselective (23, 29, 30, 36).’ (score: 0.0)\n",
            "Document #582: ‘Relevant to this report are the following.’ (score: 0.0)\n",
            "Document #581: ‘In this report, we show that VHS interacts with still another viral protein, ICP27, and that this protein is required to block degradation of specific mRNAs at times when ICP27 becomes available for this func¬tion.’ (score: 0.0)\n",
            "Document #580: ‘More recent studies from this laboratory showed that an additional late viral protein, VP22, is required for the neutralization of the RNase activity of VHS protein (48).’ (score: 0.0)\n",
            "Document #579: ‘Initial reports indicated that the neutralizing protein is VP16 (27, 38).’ (score: 0.0)\n",
            "Document #578: ‘The VHS RNase is active during the first several hours after infection, and then it is neutralized by proteins made late in infection.’ (score: 0.0)\n",
            "Document #577: ‘Although it was long sus¬pected that VHS is an RNase (11, 12, 22, 32, 52), unambiguous evidence that VHS is an endoribonuclease with a substrate specificity of RNase A emerged from studies of a protein purified to homogeneity (46, 47).’ (score: 0.0)\n",
            "Document #576: ‘The herpes simplex virus type 1 (HSV-1) UL41 gene encod¬ing a protein (virion host shutoff [VHS]) that is carried by the virion into the infected cell and causes the shutoff of host protein synthesis was first reported by Frenkel and co workers in the late 1980s (24, 25, 26, 43).’ (score: 0.0)\n",
            "Document #575: ‘(iii) Concurrently, we present the first evidence that VHS RNase interacts with ICP27 most likely when bound to cap-and poly(A)-binding proteins, respectively.’ (score: 0.0)\n",
            "Document #574: ‘Hence the rapid degradation of ARE mRNAs noted in ICP27 mutant-infected cells at late times is similar to that taking place in mock-infected or in VHS RNase mutant-virus-infected cells and does not by itself support the hypothesis that ICP27 stabilizes ARE mRNAs.’ (score: 0.0)\n",
            "Document #573: ‘The VHS RNase brought into the cells by the ICP27 virus is reduced in potency relative to that of wild-type virus.’ (score: 0.0)\n",
            "Document #572: ‘Therefore, the only VHS that is active is brought into cells by the ICP27 mutant.’ (score: 0.0)\n",
            "Document #571: ‘ In non complementing cells, VHS RNase or VP22 is not synthesized.’ (score: 0.0)\n",
            "Document #0: 'G-quadruplexes (G4s) are non-canonical nucleic acids secondary structures that form within guanine-rich strands of regulatory genomic regions.’ (score: 0.0)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/ipykernel/ipkernel.py:283: DeprecationWarning: `should_run_async` will not call `transform_cell` automatically in the future. Please pass the result to `transformed_cell` argument and any exception that happen during thetransform in `preprocessing_exc_tuple` in IPython 7.17 and above.\n",
            "  and should_run_async(code)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from sklearn.feature_extraction.text import TfidfVectorizer\n",
        "from sklearn.metrics.pairwise import cosine_similarity\n",
        "\n",
        "# This vectorizer will convert our documents into a matrix of TF-IDF features\n",
        "vectorizer = TfidfVectorizer()\n",
        "\n",
        "# Vectorize our documents\n",
        "tfidf_matrix = vectorizer.fit_transform(corpus)  # Change 'documents' to 'corpus'\n",
        "\n",
        "def search(query):\n",
        "    # Vectorize the query to the same feature space as our documents\n",
        "    query_vec = vectorizer.transform([query])\n",
        "\n",
        "    # Compute the cosine similarity between the query and all documents\n",
        "    sims = cosine_similarity(query_vec, tfidf_matrix).flatten()\n",
        "\n",
        "    # Sort the document indices by their similarity score\n",
        "    ranking = sims.argsort()[::-1]\n",
        "\n",
        "    # Print out the ranking\n",
        "    for i in range(len(ranking)):\n",
        "        print(f\"Document #{ranking[i]}: {corpus[ranking[i]]} (score: {sims[ranking[i]]})\")  # Change 'documents' to 'corpus'\n",
        "\n",
        "# Let's try it out\n",
        "search(\"HSP90\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9zDgdKUfRKT2",
        "outputId": "cefa42f8-eee5-42c2-fbc2-cdc7487e7232"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Document #867:  (score: 0.0)\n",
            "Document #284: ‘G3BP1 ADP-ribosylation, a key driver of SG assembly, is reduced in cells infected by wild-type CHIKV, but not a mutant virus that lacks hydrolase activity.’ (score: 0.0)\n",
            "Document #295: ‘In addition, knockdown of MNV-induced G3BP1 interactors reduces MNV replication, suggesting that the G3BP1 interactome consists of host factors that are required for efficient viral replication complex formation [84,107].’ (score: 0.0)\n",
            "Document #294: ‘Consistently, genetic depletion of G3BP1 severely affects viral replication and translation.’ (score: 0.0)\n",
            "Document #293: ‘The proteome of G3BP1 granules contains several proteins that are present in viral replication complexes, suggesting a pro-viral role of G3BP1 during MNV infection.’ (score: 0.0)\n",
            "Document #292: remodels G3BP1 interactome in infected cells and induces cytoplasmic G3BP1 granules during infection that are distinct from SGs.’ (score: 0.0)\n",
            "Document #291: ‘MNV (score: 0.0)\n",
            "Document #290: ‘Caliciviruses Remodel the G3BP1 Interactome during Infection The Caliciviridae family comprises small non-enveloped positive-strand RNA viruses, including human norovirus (HuNoV) and the closely related murine norovirus (MNV).’ (score: 0.0)\n",
            "Document #289: ‘4.2.2.’ (score: 0.0)\n",
            "Document #288: both Old World and New World alphaviruses, G3BP1 plays a pro-viral role only for Old World alphaviruses.’ (score: 0.0)\n",
            "Document #287: ‘Instead, these viruses sequester other SG components, such as FXR1, FXR2, and (score: 0.0)\n",
            "Document #286: ‘Notably, the New World alphaviruses do not have the FGDF motif and thus do not bind G3BP1.’ (score: 0.0)\n",
            "Document #285: ‘Infection with the mutant virus results in delayed SG disassembly and reduced viral structural protein synthesis in neuronal cells, suggesting that G3BP1 ADP-ribosylation regulates virus production [47,105].’ (score: 0.0)\n",
            "Document #283: ‘The macrodomain possesses ADP-ribosylhydrolase activity (the ability to remove ADP-ribose from conjugated proteins), and this enzymatic activity is critical for replication and virulence of viruses from several viral families [92,101–104].’ (score: 0.0)\n",
            "Document #324: ‘Intriguingly, one study suggested a pro-viral function of SG assembly during PRRSV infection.’ (score: 0.0)\n",
            "Document #282: ‘Recently, our group discovered that the SG disassembly for CHIKV is partly mediated by the macrodomain at the N-terminus of nsP3.’ (score: 0.0)\n",
            "Document #281: ‘However, SFV without FGDF motifs still disassembles SGs, but at a slower rate suggesting an additional mechanism for SG disassembly [77].’ (score: 0.0)\n",
            "Document #280: ‘The binding of nsP3 with G3BP1 is a proposed mechanism for SG disassembly where nsP3-G3BP1 binding inhibits the interaction between other SG components and G3BP1.’ (score: 0.0)\n",
            "Document #279: ‘For initiation of infection, G3BP1 facilitates CHIKV, but not to SFV, translation of genomic RNA by enriching translation factors at cytopathic vacuoles [100].’ (score: 0.0)\n",
            "Document #278: ‘In infected cells, G3BP1 is sequestered through interaction of the NTF2L domain with FGDF motifs in the C-terminal hypervariable region of non-structural protein 3 (nsP3).’ (score: 0.0)\n",
            "Document #277: ‘Cells lacking both G3BP1/G3BP2 do not support replication of the Old World alphaviruses but do support replication of the New World encephalitic alphaviruses, indicating a virus-specific importance of G3BP1 for infection.’ (score: 0.0)\n",
            "Document #276: ‘Notably, the alphavirus genus can be categorized into the Old World alphaviruses (CHIKV, SINV, SFV) and the New World alphaviruses (VEEV, EEEV) based on their geographical origin.’ (score: 0.0)\n",
            "Document #275: ‘Togaviruses Sequester G3BP1 to Facilitate Viral Replication and Translation Togaviridae is a family of enveloped, icosahedral viruses with a positive-sense single-stranded RNA genome including the mosquito-borne alphaviruses that cause rash and arthritis such as CHIKV, SFV, and SING.’ (score: 0.0)\n",
            "Document #274: ‘4.2.1.’ (score: 0.0)\n",
            "Document #273: ‘In this section, we will discuss the different strategies employed by viruses that use G3BP1 to promote viral replication.’ (score: 0.0)\n",
            "Document #272: ‘The proposed pro-viral functions of G3BP1 include: stabilizing viral mRNAs to prevent degradation, facilitating viral protein translation by recruiting translation factors and ribosomes, acting as a scaffold to build viral replication complexes, and amplifying the viral genome.’ (score: 0.0)\n",
            "Document #296: ‘Similar to alphaviruses, G3BP1 in MNV-infected cells recruit ribosomes to viral RNA and initiate translation [31].’ (score: 0.0)\n",
            "Document #297: ‘During MNV infection, both PKR and GCN2 phosphorylated eIF2a; however, SG formation is not observed in the infected cells, even though they can still form SGs upon arsenite treatment, suggesting that MNV decouples the SG formation from eIF2a phosphorylation during virus infection.’ (score: 0.0)\n",
            "Document #298: ‘In contrast, feline calicivirus cleaves G3BP1 through the viral 3C-like proteinase NS6Pro [94].’ (score: 0.0)\n",
            "Document #299: ‘4.3.’ (score: 0.0)\n",
            "Document #322: ‘In contrast to the above-described viruses, PRRSV induces SGs only at late stages of infection (48–72-h post-infection) through PERK-, rather than PKR-mediated eIF2a phosphorylation.’ (score: 0.0)\n",
            "Document #321: ‘Neither deletion nor overexpression of G3BP1, G3BP2, or both affects viral titers.’ (score: 0.0)\n",
            "Document #320: ‘Depletion or Overexpression of G3BP1 Does Not Affect PRRSV Infection PRRSV—a single-stranded positive-sense enveloped virus in the Arteriviridae family— does not require G3BP1, yet the protein is closely associated with viral replication complexes.’ (score: 0.0)\n",
            "Document #319: ‘4.4.1.’ (score: 0.0)\n",
            "Document #318: ‘Though G3BP1 is a predominant target for many viruses, there are some viruses whose replication is unaffected by the expression level of G3BP1.’ (score: 0.0)\n",
            "Document #317: ‘Interestingly, HCV requires only G3BP1, but not G3BP2 whose knockdown does not alter viral replication [74,75].’ (score: 0.0)\n",
            "Document #316: UTR of viral RNA and is involved in genome amplification [74].’ (score: 0.0)\n",
            "Document #315: ‘During HCV infection, G3BP1 interacts with the 50¬ (score: 0.0)\n",
            "Document #314: ‘In addition, in ZIKV-infected cells, G3BP1 interacts with capsid protein and co-localizes with envelope protein, suggesting a possible role for G3BP1 in virion assembly [70–72].’ (score: 0.0)\n",
            "Document #313: ‘G3BP1 depletion reduces viral replication, titer, and viral protein levels, while overexpression of G3BP1 enhances viral titers.’ (score: 0.0)\n",
            "Document #312: ‘ZIKV and HCV Require G3BP1 for Proper Viral Replication In both ZIKV-and HCV-infected cells, G3BP1 associates with the viral replication complex.’ (score: 0.0)\n",
            "Document #311: ‘4.3.2.’ (score: 0.0)\n",
            "Document #310: and whether a similar decoy phenomenon may be used by other viruses to counter the production of ISG proteins and promote viral replication is unknown.’ (score: 0.0)\n",
            "Document #309: ‘Not all flavivirus sfRNAs have this property (score: 0.0)\n",
            "Document #308: ‘As an IFN countermeasure during DENV-2 infection of Huh-7 cells, a long noncoding RNA from the subgenomic region of flavivirus RNA (sfRNA) produced by all flaviviruses binds G3BP1, G3BP2, and Caprin1, and thereby inhibits the translation of ISGs [21].’ (score: 0.0)\n",
            "Document #307: ‘In DENV-2-infected cells, G3BP1 is also associated with ISG mRNAs and facilitates their translation to inhibit viral replication.’ (score: 0.0)\n",
            "Document #306: ‘The expression of viral protein is reduced in cells containing G3BP1 granules, whereas knockdown of G3BP1 enhances protein level and viral titers.’ (score: 0.0)\n",
            "Document #305: ‘Although the phosphorylation of eIF2a by PKR occurs at the late infection stage [73], it is unclear whether infected cells induce SGs.’ (score: 0.0)\n",
            "Document #304: ‘Dengue Virus (DENV) Modulates G3BP1 Function by Interacting with 30UTR vRNA DENV infection induces G3BP1 punctate-like structures at the early stage of infection.’ (score: 0.0)\n",
            "Document #303: ‘4.3.1.’ (score: 0.0)\n",
            "Document #302: ‘In this section, we describe how the G3BP1 function is modulated by DENV, ZIKV, and HCV.’ (score: 0.0)\n",
            "Document #301: utilize G3BP1 at different stages of infection.’ (score: 0.0)\n",
            "Document #300: ‘G3BP1 Is Differentially Utilized within the Flaviviridae Family Although G3BP1 influences viral propagation of Flaviviridae family, different members (score: 0.0)\n",
            "Document #271: ‘Binding of the viral genome by G3BP1 and sequestering it into viral replication complexes may be important to prevent G3BP1-mediated activation of the innate immune response.’ (score: 0.0)\n",
            "Document #270: ‘For example, during CHIKV infection, depletion of both G3BP1 and G3BP2 significantly decreases viral RNA levels, protein expression, and subsequent viral progeny produc¬tion [23].’ (score: 0.0)\n",
            "Document #269: ‘Such pro-viral activities have been reported for several viruses including CHIKV, hepatitis C virus, and respiratory syncytial virus [23,74,99].’ (score: 0.0)\n",
            "Document #242: ‘Here we summarize the strategies shared by viruses in the Picornaviridae and Coronaviridae families to counteract host defense.’ (score: 0.0)\n",
            "Document #240: ‘G3BP1 also assembles SGs in response to infection-mediated translation arrest and limits the availability of translation factors for viral protein synthesis.’ (score: 0.0)\n",
            "Document #239: ‘In response to IFN, G3BP1, G3BP2, and Caprin 1 promote ISG translation to synthesize anti-viral factors.’ (score: 0.0)\n",
            "Document #238: ‘In virus-infected cells, G3BP1 binds to dsRNA resulting from the viral replication intermediates through the RGG domain and elevates RIG-I induced IFN-ß mRNA expression [30].’ (score: 0.0)\n",
            "Document #237: ‘G3BP1 enhances the immune response against invading viruses by inducing the expression of IFN and translation of ISGs.’ (score: 0.0)\n",
            "Document #236: ‘Here, we explore the mechanisms by which different viruses regulate G3BP1 function during infection.’ (score: 0.0)\n",
            "Document #235: (2) recruiting ribosomes to initiate viral RNA translation; (3) serving as a scaffold to build viral replication complexes; (4) assembling SGs; (5) stabilizing and enhancing the IFN-ß response; and (6) activating cGAS and RIG-I pathways.’ (score: 0.0)\n",
            "Document #234: ‘Depending on the virus type, G3BP1 plays different roles in the viral life cycle, including (1) stabilizing viral RNA; (score: 0.0)\n",
            "Document #233: ‘G3BP1 facilitates replication of some viruses whereas it restricts viral spread in other viruses.’ (score: 0.0)\n",
            "Document #232: ‘G3BP1 controls several key cellular processes, including mRNA metabolism, ribosomal quality control, immune response, and SG assembly.’ (score: 0.0)\n",
            "Document #231: ‘Here, we describe the role of G3BP1 during specific viral infections and the ways by which G3BP1 is targeted.’ (score: 0.0)\n",
            "Document #230: ‘On the contrary, viruses cleave G3BP1 when it exerts an anti-viral role (Table 2).’ (score: 0.0)\n",
            "Document #229: ‘In general, viruses use their genome-encoded proteins to sequester G3BP1 if it is required for viral replication.’ (score: 0.0)\n",
            "Document #228: ‘Further clues are obtained from identifying the strategies that viruses employ to target G3BP1—i.e., sequestration, cleavage, or degradation—and modulate its function during infection (Figure 4).’ (score: 0.0)\n",
            "Document #227: ‘Phosphorylation of G3BP1 by BTK is critical for dimerization and phase separation to limit viral spread [88].’ (score: 0.0)\n",
            "Document #226: ‘showed that G3BP1 is phosphorylated at tyrosine residue 40 by Bruton’s tyrosine kinase (BTK).’ (score: 0.0)\n",
            "Document #225: ‘Using polyinosinic:polycytidylic acid as a mimic for viral replication double-stranded intermediates, Kim et al.’ (score: 0.0)\n",
            "Document #224: ‘Similarly, arginine methylation status of G3BP1 partly recruits Tudor domain-containing protein (TDRD3) to regulate innate immune response against enterovirus species [60].’ (score: 0.0)\n",
            "Document #223: ‘Thus, post-translationally modified G3BP1 may act as an antiviral protein, while the unmodified or differently modified G3BP1 could play a pro-viral role during CHIKV infection.’ (score: 0.0)\n",
            "Document #222: ‘However, G3BP1 is also required for establishing CHIKV replication complexes, and post-translational modifications required for this process have not been defined [23].’ (score: 0.0)\n",
            "Document #221: ‘For example, ADP-ribosylated G3BP1 is crucial for SG assembly, given that chikungunya virus (CHIKV) actively reduces G3BP1 ADP-ribosylation, disrupting SGs with translation factors released from the con¬densed state [47].’ (score: 0.0)\n",
            "Document #220: ‘One way by which G3BP1 may regulate its function dur¬ing infection is through post-translational modifications.’ (score: 0.0)\n",
            "Document #219: ‘G3BP1 can either promote or inhibit the viral life cycle, but the molecular mechanisms are not completely understood.’ (score: 0.0)\n",
            "Document #218: ‘SG status during infection against different viruses.’ (score: 0.0)\n",
            "Document #241: ‘To counter these antiviral functions, viruses target G3BP1—either cleaving or sequestering it during infection.’ (score: 0.0)\n",
            "Document #243: ‘Picornaviruses Inhibit G3BP1 Function through Protease-Mediated Cleavage Picornaviruses are small, non-enveloped, positive-sense RNA viruses that include EMCV, EV, PV, FMDV, and CVB3.’ (score: 0.0)\n",
            "Document #268: ‘Although G3BP1 synergizes host immune responses to fight viral invasion, some viruses require G3BP1 for efficient infection.’ (score: 0.0)\n",
            "Document #244: ‘G3BP1 negatively regulates Picornaviridae infection by binding the 30 untranslated regions (30UTRs) or Internal Ribosome Entry Site (IRES) regions of viral RNAs to inhibit viral replication [53,95].’ (score: 0.0)\n",
            "Document #267: ‘In COVID-19 patients and mice, the N protein interacts with G3BP1, suppresses SG formation, and potentiates viral infection by antagonizing the G3BP1-mediated host innate immune response pathway [97,98].’ (score: 0.0)\n",
            "Document #266: highlighted the potential antiviral role of G3BP1 in the lung tissue of COVID-19 patients and SARS-CoV-2-infected mice.’ (score: 0.0)\n",
            "Document #265: ‘Recent studies (score: 0.0)\n",
            "Document #264: ‘However, the role of G3BP1 remains unclear given the conflicting reports on the effect of G3BP1 knockdown in structural protein and viral RNA levels [81–83,96].’ (score: 0.0)\n",
            "Document #263: ‘Though it is unclear whether SARS-CoV-2 induces SGs, G3BP1–nucleocapsid binding blocks the interaction of SG proteins such as PKR and USP10 with G3BP1 in arsenite-treated cells [82].’ (score: 0.0)\n",
            "Document #262: ‘The intrin¬sically disordered region (IDR) at the N-terminal region of nucleocapsid is required for condensation with G3BP1, and this IDR is also crucial for viral particle production.’ (score: 0.0)\n",
            "Document #261: ‘SARS-CoV-2—the virus responsible for the COVID-19 pandemic—modulates G3BP1 function by sequestration and condensation with the nucleocapsid protein.’ (score: 0.0)\n",
            "Document #260: ‘Expression of non-cleavable G3BP1 results in persistent SGs and reduces viral replication, but the mechanism behind G3BP1-mediated reduction of viral replication remains unexplored [79,80].’ (score: 0.0)\n",
            "Document #259: ‘However, these viruses utilize different strategies to modulate G3BP1 function; for example, PEDV induces cleavage of G3BP1 between D168 and D169 by caspase-8 to regu¬late SG dynamics in infected cells.’ (score: 0.0)\n",
            "Document #258: ‘Overexpression of G3BP1 reduces viral replication, protein synthesis, and virus production, while G3BP1 knockdown enhances viral replica¬tion.’ (score: 0.0)\n",
            "Document #257: ‘Similar to Picornaviridae, G3BP1 nega¬tively regulates coronavirus infection.’ (score: 0.0)\n",
            "Document #256: ‘Coronaviruses Modulate G3BP1 Function Either by Cleavage or Sequestration Coronaviridae is a family of enveloped viruses with large single-stranded positive-sense RNA genomes and includes PEDV and SARS-CoV-2.’ (score: 0.0)\n",
            "Document #255: the fundamental mechanism employed by Picornaviridae to restrict antiviral activities of G3BP1.’ (score: 0.0)\n",
            "Document #254: ‘Thus, protease-mediated cleavage or degradation of G3BP1 is (score: 0.0)\n",
            "Document #253: ‘FMDV infection also causes degradation of G3BP1 through autophagy [62].’ (score: 0.0)\n",
            "Document #252: ‘Catalytic mutant C41A or leaderless FMDV efficiently inhibits arsenite-induced SG formation in infected cells [63].’ (score: 0.0)\n",
            "Document #251: ‘FMDV, which uses the leader protein to cleave G3BP1, also causes SG disassem¬bly.’ (score: 0.0)\n",
            "Document #250: ‘Consistent with the observed effect on G3BP1 cleavage, knockdown of G3BP1 increases viral protein synthesis and virus production, whereas G3BP1 overexpression shows an inhibitory effect [61,64–66].’ (score: 0.0)\n",
            "Document #249: ‘SG disassembly correlates temporally with G3BP1 cleavage, and the expression of the non-cleavable G3BP1 Q325E mutant results in sustained IFN-ß mRNA expression and SG persistence.’ (score: 0.0)\n",
            "Document #248: ‘SGs are observed early in infection but disassembled at late stages of infection.’ (score: 0.0)\n",
            "Document #247: ‘During Picornaviruses infection, PKR is activated by double-stranded RNA intermedi¬ates, phosphorylated eIF2a, and induces SG formation.’ (score: 0.0)\n",
            "Document #246: ‘G3BP1 cleavage is essential for the expression of viral proteins, and expression of the non-cleavable G3BP1 Q325E mutant suppresses viral replication [61–69].’ (score: 0.0)\n",
            "Document #245: ‘To counteract these effects, the 3C protease or leader protein of these viruses cleaves G3BP1 between Q325 and Q326 at late infection stages.’ (score: 0.0)\n",
            "Document #323: ‘The reason for G3BP1 association with viral replication complexes and the mechanism and role of SG assembly at late infection stages are not known [85,86].’ (score: 0.0)\n",
            "Document #325: ‘Cells infected (score: 0.0)\n",
            "Document #866:  (score: 0.0)\n",
            "Document #393: ‘These findings support the hy¬pothesis that SGs (or SG-like structures) are cellular antiviral responses.’ (score: 0.0)\n",
            "Document #404: ‘The NP encapsulates (2) viral gRNA as a template to produce (+) cRNA and mRNA.’ (score: 0.0)\n",
            "Document #403: ‘After invasion of host cells by NDV, the NP-wrapped viral genomic RNA, as well as other viral components, is released into the cytoplasm (27).’ (score: 0.0)\n",
            "Document #402: ‘The PP is essential for viral RNA synthesis from bona fide templates (26).’ (score: 0.0)\n",
            "Document #401: ‘The NP encapsu¬lates the RNA genome, which serves as the template for viral transcription and replication (25).’ (score: 0.0)\n",
            "Document #400: ‘The NDV genome contains 6 genes that encode 2 glycoproteins, the fusion protein and hemagglutinin-neuraminidase protein, the matrix protein, and the ribonucleoprotein complex, com¬posed of the nucleocapsid protein (NP), phosphopro¬tein (PP), and large polymerase, responsible for viral transcription and replication (23).’ (score: 0.0)\n",
            "Document #399: ‘Oncolytic NDV can selectively replicate in tumor cells and is an important antineo¬plastic agent (24).’ (score: 0.0)\n",
            "Document #398: ‘NDV is a member of the Paramyxoviridae family with a negative-sense, nonsegmented, single-stranded RNA genome (23).’ (score: 0.0)\n",
            "Document #397: ‘Depletion of endogenous G3BP inhibits SG formation and significantly decreases the viral titers and RNA amounts (22).’ (score: 0.0)\n",
            "Document #396: ‘Respiratory syncytial virus (RSV) also triggers canoni¬cal SGs.’ (score: 0.0)\n",
            "Document #395: ‘For instance, knockdown of SG proteins TIA-1, TIAR, or Ras GTPase¬activating protein-binding protein 1 (G3BP1) impairs HCV RNA replication, virus assembly, and egress (18).’ (score: 0.0)\n",
            "Document #394: ‘However, in situations where the virus induces bona fide SG formation, SG components seem to be required for virus infection.’ (score: 0.0)\n",
            "Document #392: ‘AVG formation during poxvirus infection leads to adecreaseinbothviral translation and virus titers (20, 21).’ (score: 0.0)\n",
            "Document #326: PRRSV induces SGs, recruits G3BP1 and the viral replicase protein nsP1ß, thereby inhibiting G3BP1-mediated PKR activation [108].’ (score: 0.0)\n",
            "Document #391: ‘In other cases, viruses induce SG-like structures (e.g., antiviral granules, AVGs) that re¬strict virus replication.’ (score: 0.0)\n",
            "Document #390: ‘A mutant JEV carrying a core protein incapable of binding to caprin-1 abrogates the SG inhibitory ability and exhibits lower propagation in vitro and in vivo (19).’ (score: 0.0)\n",
            "Document #389: ‘As an example, Japanese encephalitis virus (JEV) inhibits SG formation through interaction between the viral core protein and caprin-1.’ (score: 0.0)\n",
            "Document #388: ‘In cases where viral infection does not induce SGs, accumulat¬ing evidence demonstrates an inverse correlation be¬tween SG formation and viral propagation.’ (score: 0.0)\n",
            "Document #387: ‘SGs generally play an antiviral role (19–21).’ (score: 0.0)\n",
            "Document #386: ‘For example, SGs are served at 48 h post infection (hpi) with HCV and increase to speak at 72hpi (18).’ (score: 0.0)\n",
            "Document #385: ‘Other viru¬ses induce stable SG formation.’ (score: 0.0)\n",
            "Document #384: ‘For example, during the early stages of picor¬navirus infection, v-SGs can be observed but then disappear, probably because of the viral 3C protease, which is responsible for the cleavage of the Ras-GAP SH3 domain-binding protein (G3BP) (15–17).’ (score: 0.0)\n",
            "Document #383: ‘Some viruses induce transient SG for¬mation.’ (score: 0.0)\n",
            "Document #382: ‘For example, wild-type influenza A virus does not induce SGs but an NS1¬mutant virus triggers SG formation in a PKR-dependent manner (13, 14).’ (score: 0.0)\n",
            "Document #381: ‘Some viruses inhibit SG formation.’ (score: 0.0)\n",
            "Document #405: ‘Viral mRNAs are capped and polyadenylated by L-protein during pro¬tein synthesis (28, 29).’ (score: 0.0)\n",
            "Document #406: ‘The translation process relies on cellular translational machinery that is present within the host cell cytoplasm, eliciting a strong stress re¬sponse of cells.’ (score: 0.0)\n",
            "Document #407: ‘In this study, the effect of NDV-induced SGs on its replication and the possible mechanism were investigated.’ (score: 0.0)\n",
            "Document #408: ‘Our results demonstrate that NDV-induced SGs facilitate viral replication through ma-nipulating host protein translation.’ (score: 0.0)\n",
            "Document #431: ‘Together, these results demonstrate that NDV-induced SGs are dependent on the PKR/eIF2a signaling pathway.’ (score: 0.0)\n",
            "Document #430: ‘4C–E).’ (score: 0.0)\n",
            "Document #429: ‘Depletion of endogenous PKR by siRNA reduced the level of eIF2a phosphory¬lation and the percentage of cells with SGs significantly (Fig.’ (score: 0.0)\n",
            "Document #428: ‘4B).’ (score: 0.0)\n",
            "Document #427: ‘Western blot analysis showed that PKR and down¬stream eIF2a was phosphorylated after ARS treatment and virus infection (Fig.’ (score: 0.0)\n",
            "Document #426: ‘4A).’ (score: 0.0)\n",
            "Document #425: ‘PKR were colocalized with either TIA-1 or G3BP1, respectively, after infection (Fig.’ (score: 0.0)\n",
            "Document #424: ‘To examine whether PKR acts as an upstream component of eIF2a in the formation of v-SGs, cells were costained for PKR and TIA-1/G3BP1.’ (score: 0.0)\n",
            "Document #423: ‘3B).’ (score: 0.0)\n",
            "Document #422: ‘(Fig.’ (score: 0.0)\n",
            "Document #421: ‘Conversely, S51D eIF2a transfection led to the accumulation of SG dots that appeared larger than v-SGs.’ (score: 0.0)\n",
            "Document #420: ‘No SG dots were observed in S51A eIF2a¬transfected cells, compared with the SG dots observed in WT eIF2a-transfected cells.’ (score: 0.0)\n",
            "Document #419: ‘To elucidate further the potential role of eIF2a in v-SGs, we measured SG formation by transfecting 2 eIF2a mutant plasmids, S51D and S51A eIF2a, which mimic hyper-and hypo phosphorylated forms of eIF2a,re¬spectively.’ (score: 0.0)\n",
            "Document #418: ‘We noted that the decreased levels of eIF2a and b-actin at 24 hpi were the result of the cell shedding caused by virus infection.’ (score: 0.0)\n",
            "Document #417: ‘Correspondingly, the viral NP accumulated at the late stage of infection.’ (score: 0.0)\n",
            "Document #416: ‘As expected, NDV infection reduced host protein translation over the time).’ (score: 0.0)\n",
            "Document #415: ‘The decrease of eIF2-GTP-tRNAMet ultimately leads to a global decrease in protein syn¬thesis (39).’ (score: 0.0)\n",
            "Document #414: ‘Phosphorylation of eIF2a leads to SG assembly by reducing the levels of the ternary complex eIF2-GTP-tRNAMet.’ (score: 0.0)\n",
            "Document #413: ‘3A).’ (score: 0.0)\n",
            "Document #412: ‘We measured eIF2a phosphoryla¬tion and found that NDV triggered eIF2a phosphorylation at the late stage of infection, consistent with the appear¬ance of v-SGs (Fig.’ (score: 0.0)\n",
            "Document #411: The activation of PKR and eIF2a leads to SG formation in virus-infected cells (38).’ (score: 0.0)\n",
            "Document #410: ‘PKR/eIF2a signaling cascade is essential for NDV-induced SGs (score: 0.0)\n",
            "Document #409: ‘Taken together, these results suggest that NDV induces canonical SGs, which is similar to the SGs formed under oxidative stress, indicating that ongoing protein synthesis is essential for NDV-induced SGs.’ (score: 0.0)\n",
            "Document #380: ‘The pattern of virus-induced (v)-SGs varies in the virus species.’ (score: 0.0)\n",
            "Document #379: ‘Recent years have seen a growing list of viruses with replication that is associated with SG formation (12).’ (score: 0.0)\n",
            "Document #378: ‘Some viruses escape translation inhibition through affect¬ing SG formation (11).’ (score: 0.0)\n",
            "Document #351: ‘In this study, Newcastle disease virus (NDV) triggered stable formation of bona fide SGs on HeLa cells through activating the protein kinase R (PKR)/eIF2a pathway.’ (score: 0.0)\n",
            "Document #349: ‘Mammalian cells respond to various environmental stressors to form stress granules (SGs) by arresting cytoplasmic mRNA, protein translation element, and RNA binding proteins.’ (score: 0.0)\n",
            "Document #348: ‘Therefore, not all G3BP1-positive structures are SGs, and should be stained with other markers to ascertain their identities.’ (score: 0.0)\n",
            "Document #347: ‘Given that the interacting partners of G3BP1 are modulated during infection to form new complexes in a virus-specific manner, different terms have been used to differentiate for these viral-induced protein complexes, including atypical SGs (EV), antiviral SGs (IBV), or nsP3 foci (alphaviruses).’ (score: 0.0)\n",
            "Document #346: ‘During norovirus infection, the G3BP1 interactome is modulated to inhibit the formation of SGs.’ (score: 0.0)\n",
            "Document #345: ‘In SIV-infected cells, G3BP1 stably interacts with viral proteins throughout the life cycle, whereas some SG components interact in a time-dependent manner.’ (score: 0.0)\n",
            "Document #344: ‘Recent proteomic studies provided some insights into the G3BP1 interactome during infection.’ (score: 0.0)\n",
            "Document #343: G3BP1 plays critical yet distinct roles during infection by different families of the virus.’ (score: 0.0)\n",
            "Document #342: ‘Furthermore, the differences in the role (score: 0.0)\n",
            "Document #341: ‘Given that G3BP1 is also modified by phosphorylation and ubiquitylation, it is worth checking the status of G3BP1 for other post-translational modifications at different infection stages and determine how these modifications affect a viral life cycle [33,42].’ (score: 0.0)\n",
            "Document #340: ‘Alphaviruses and coronaviruses encode an enzyme to remove ADP-ribosylation indicating that the modified form of G3BP1 may possess antiviral functions.’ (score: 0.0)\n",
            "Document #339: ‘G3BP1 methylation recruits TDRD3 into SGs to regulate innate immune response [60].’ (score: 0.0)\n",
            "Document #338: ‘Intriguingly, the post-translational modification status of heavily regulated G3BP1 may determine the functional consequences of interaction.’ (score: 0.0)\n",
            "Document #337: ‘Besides, some viruses possess conserved motifs (e.g., FGDF) within their proteins to bind and modulate G3BP1 func¬tion [76,109].’ (score: 0.0)\n",
            "Document #336: ‘For pro-viral roles, the NTF2L and EGG domains of G3BP1 play a crucial role to bind viral proteins or viral RNA, respectively.’ (score: 0.0)\n",
            "Document #335: ‘If G3BP1 is required for viral replication, viruses do not cleave G3BP1, but instead recruit it to build viral replication complexes (e.g., alphaviruses).’ (score: 0.0)\n",
            "Document #334: ‘In general, if G3BP1 exerts anti-viral activity, then viruses cleave or sequester G3BP1 (e.g., polioviruses, flaviviruses).’ (score: 0.0)\n",
            "Document #333: ‘These functional modulations of G3BP1 by viruses impact the viral life cycle either posi¬tively or negatively.’ (score: 0.0)\n",
            "Document #332: ‘Viruses employ at least three strategies to target and modulate G3BP1 function during infection: (1) enzymatic activity such as proteolytic cleavage or removing post-translational modification (e.g., poliovirus and alphavirus); (2) binding or sequestration (e.g., alphavirus and coronavirus); and (3) redistribution or remodeling of G3BP1 interaction (e.g., norovirus).’ (score: 0.0)\n",
            "Document #331: ‘Although G3BP1 is sequestered within the viral inclusions, the exact role of G3BP1 in EBOV infection is not well understood.’ (score: 0.0)\n",
            "Document #330: ‘VP5 expression affects IFNa/ß expression and PKR kinase activation in infected cells.’ (score: 0.0)\n",
            "Document #329: ‘Although infection-mediated SG assembly is not observed, EBOV inhibits arsenite-induced SG assembly.’ (score: 0.0)\n",
            "Document #328: ‘G3BP1 Is Sequestered within Ebola Virus Inclusions EBOV, a single-stranded negative-sense RNA virus, does not induce SG assembly but sequesters SG components G3BP1, eEF2, eIF2, and eIF3 through the viral protein VP5 [55,87].’ (score: 0.0)\n",
            "Document #327: ‘4.4.2.’ (score: 0.0)\n",
            "Document #350: ‘Virus-induced SGs function in different ways, depending on the species of virus; however, the mechanism of SG regulation of virus replication is not well understood.’ (score: 0.0)\n",
            "Document #352: ‘NDV-induced SGs contained classic SG markers T-cell internal antigen (TIA)-1, Ras GTPase-activating protein-binding protein (G3BP)-1, eukaryotic initiation factors, and small ribosomal subunit, which could be disassembled in the presence of cycloheximide.’ (score: 0.0)\n",
            "Document #377: ‘Phosphory¬lation of eIF2a also leads to SG formation.’ (score: 0.0)\n",
            "Document #353: ‘Treatment with nocodazole, a microtubule disruption drug, led to the formation of relatively small and circular granules, indicating that NDV infection induces canonical SGs.’ (score: 0.0)\n",
            "Document #376: ‘In this case, viruses such as hepatitis C(HCV) keep on translating through IRES structure in the presence of activated PKR and phosphorylation of eIF2a (10).’ (score: 0.0)\n",
            "Document #375: ‘Some of theseRNA viruses induce phosphorylation of eIF2a and shut-off of global protein synthesis.’ (score: 0.0)\n",
            "Document #374: ‘Many other RNA viruses contain an internal ribosome entry site (IRES) located in the 59-untranslated region of the genomic (g)RNA, which drives cap-independent ini¬tiation of translation of the viral message (9).’ (score: 0.0)\n",
            "Document #373: ‘For instance, the influenza virus nonstructural protein NS1 interacts with eIF4GI and PABP-1, as well as with viral mRNAs, promoting the specific recruitment of 43S complexes to the viral mRNAs (8).’ (score: 0.0)\n",
            "Document #372: ‘They adopt a se¬ries of methods to rob host translation systems.’ (score: 0.0)\n",
            "Document #371: ‘As obligate intracellular parasites, viruses rely on the host cell translation machinery.’ (score: 0.0)\n",
            "Document #370: ‘This process ultimately leads to reduced translation initiation rates and to SG formation (6, 7).’ (score: 0.0)\n",
            "Document #369: ‘The other non canonical mechanism of SG induction is independent of eIF2a and involves inhibiting the activity of eIF4A, eIF4G, and eIF4E, 3 subunits of the eIF4F complex.’ (score: 0.0)\n",
            "Document #368: ‘The phosphorylation of eIF2a at Ser-51 is mediated by several kinases, including the heme-regulated inhibitor kinase, general control nonderepressible kinase, pro¬tein kinase R (PKR), and the PKR-like endoplasmic re¬ticulum kinase (5).’ (score: 0.0)\n",
            "Document #367: ‘The RNA-binding proteins T-cell internal antigen-1 (TIA-1) and TIA-1¬related (TIAR) protein are then shuttled from the nucleus to the cytoplasm to sequester untranslated mRNAs at SGs (1).’ (score: 0.0)\n",
            "Document #366: ‘When cells are exposed to stressors, phosphorylation of eIF2a leads to a decreased abundance of eIF2-GTP-tRNAMet (2).’ (score: 0.0)\n",
            "Document #365: ‘The eIF2-GTP-tRNAMet ter¬nary complex associates with the 40S complex and scans for an initiation codon (2).’ (score: 0.0)\n",
            "Document #364: ‘The central region of eIF4G binds to the 40S ribosome subunit, and the N terminus recruits mRNA through binding to poly(A)¬binding protein (PABP) (4).’ (score: 0.0)\n",
            "Document #363: ‘eIF4F consists of 3 initiation factors: eIF4E, eIF4G, and eIF4A.’ (score: 0.0)\n",
            "Document #362: ‘Conventional cap-dependent translation of most cellular mRNA commences with the recognition of 7¬methyl-GTP cap by the eukaryotic initiation factor (eIF) 4F complex of translation initiation factors (3).’ (score: 0.0)\n",
            "Document #361: ‘In this case, stress granules (SGs) are induced to arrest cytoplas¬mic mRNA, protein translation factors, and RNA binding proteins (1, 2).’ (score: 0.0)\n",
            "Document #360: oxidative stress, heat shock, and UV irradiation.’ (score: 0.0)\n",
            "Document #359: ‘Mammalian cells enter a relatively stable state by tempo¬rarily inhibiting cellular protein translation when exposed to various environmental stressors, such as virus infection, (score: 0.0)\n",
            "Document #358: ‘These results demonstrate that NDV triggers stable formation of bona fide SGs, which benefit viral protein translation and virus replication by arresting cellular mRNA.’ (score: 0.0)\n",
            "Document #357: ‘Deletion of TIA-1 or TIAR reduced NP mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #356: ‘FISH revealed that NDV-induced SGs contained predominantly cellular mRNA rather than viral mRNA.’ (score: 0.0)\n",
            "Document #355: ‘Results showed that depletion of TIA-1 or TIAR inhibited viral protein synthesis, reduced extracellular virus yields, but increased global protein translation.’ (score: 0.0)\n",
            "Document #354: ‘Furthermore, the role of SGs on NDV replication was investigated by knockdown of TIA-1 and TIA-1-related (TIAR) protein, the 2 critical components involved in SG formation from the HeLa cells, followed by NDV infection.’ (score: 0.0)\n",
            "Document #217: ‘Table 1.’ (score: 0.0)\n",
            "Document #216: ‘On the other hand, the presence of SGs may facilitate the switch from viral translation to genome amplification by sequestering translation factors into the condensed state (Figure 3) [23].’ (score: 0.0)\n",
            "Document #215: ‘Thus, viruses may target SGs not only to modulate G3BP1 function but also to disassemble SGs to counteract anti-viral mechanisms in the infected cells.’ (score: 0.0)\n",
            "Document #67: ‘We show that nsp1ß is a stress-responsive protein, enters virus-induced stress granules (SGs) during infection, and repurposes SGs into a proviral platform, where it co-opts the SG core component G3BP1 to interact with PKR in a regulated manner.’ (score: 0.0)\n",
            "Document #78: ‘The activated PKR phosphorylates the a-subunit of eukaryotic translation initiation fac¬tor 2 (eIF2a) to induce general translation arrest and to stimulate formation of stress gran¬ules (SGs) (7, 10), a kind of non membrane bound cytoplasmic aggregate that mainly include stalled translation initiation complexes, together with nucleating factors like Ras GTPase-activating protein-binding protein 1 (G3BP1), T cell intracytoplasmic antigen 1 (TIA1), and many others (11, 12).’ (score: 0.0)\n",
            "Document #77: ‘Alternatively, PKR can be activated via heterodimerization by an endogenous dsRNA binding protein, namely PKR-activating protein (PACT) (8).’ (score: 0.0)\n",
            "Document #76: ‘Binding of DRBMs to viral dsRNAs or 50 triphosphate RNAs induces conformational changes of PKR, leading to autophos¬phorylation of its KD (8, 9).’ (score: 0.0)\n",
            "Document #75: ‘Structurally, PKR is composed of two tan¬dem evolutionarily conserved dsRNA-binding motifs (DRBMs) in the N terminus, an effector/catalytic kinase domain (KD) in the C terminus, and a middle .flexible linker region connecting the two domains (2, 6, 7).’ (score: 0.0)\n",
            "Document #74: ‘This protein was initially discovered as a key component of interferon (IFN)-stimulated genes (3, 4) and later identified as a pattern-recognition receptor that specifically recog¬nizes double-stranded RNAs (dsRNA) (5, 6), a common intermediate produced during infections of both DNA and RNA viruses.’ (score: 0.0)\n",
            "Document #73: ‘At the virus–host interface, one such critical host player is the protein kinase R (PKR), a 551-amino acid-long serine threonine kinase (1, 2).’ (score: 0.0)\n",
            "Document #72: Eukaryotic cells employ an amazingly wide variety of intracellular defenses to restrict invading viral pathogens.’ (score: 0.0)\n",
            "Document #71: ‘PKR j stress granules j PRRSV j in.ammation j nsp1ß (score: 0.0)\n",
            "Document #70: ‘Thus, PRRSV has evolved a unique strategy to evade PKR restriction to suppress host in.ammatory responses.’ (score: 0.0)\n",
            "Document #69: ‘The viral mutant carrying the corresponding mutations induces elevated level of PKR phosphoryla¬tion and pronounced production of in.ammatory cytokines (e.g., tumor necrosis factor, interleukin [IL]-6, and IL-8), whereas small-interfering RNA knockdown of PKR or treatment with C16, a PKR inhibitor, blocks this effect.’ (score: 0.0)\n",
            "Document #68: ‘RNA interference silencing of G3BP1 or mutation of speci.c nsp1ß residues (VS19GG) can abolish the antagonization of PKR activation.’ (score: 0.0)\n",
            "Document #66: ‘Here, we report that this event is to suppress cellular in.ammation and is mediated by the viral replicase protein nsp1ß.’ (score: 0.0)\n",
            "Document #214: ‘Given that some of the key anti-viral proteins (such as ADAR1, PARP13/ZAP, OAS, IRF3, IRF7, TBK1, and RNase L) localize to SGs and activate the anti-viral response, SGs may negatively affect viral replication [56–60].’ (score: 0.0)\n",
            "Document #65: ‘However, this molecule is inactivated in the cells infected with porcine reproductive and respiratory syndrome virus (PRRSV), an economically devastating pathogen to the world swine industry.’ (score: 0.0)\n",
            "Document #64: ‘Protein kinase R (PKR) is a critical host restriction factor against invading viral pathogens.’ (score: 0.0)\n",
            "Document #63: ‘So when is a domain really a domain? And if the native state isn’t always the folded state, which state do we use to determine membership in a domain family? Although our definition of a domain may need to be adapted and expanded, we can take comfort that, for now, when focusing on typical globular domains one only needs to ensure appropriate filtering of the data.’ (score: 0.0)\n",
            "Document #62: ‘The proteome is now seen as existing on a continuum of varying structural content, with structure not necessarily present under physiological conditions, and disorder and order interchangeable states dependent upon the environment and protein interactions.’ (score: 0.0)\n",
            "Document #61: ‘The challenge of structural domain categorization is only going to become more complex.’ (score: 0.0)\n",
            "Document #60: ‘At some point we will gather sufficient knowledge to consider categorization of IDPs into domain families, a somewhat daunting notion.’ (score: 0.0)\n",
            "Document #59: ‘Re-examination of our rejected targets here not only demonstrates the importance of filtering structural databases and applying a consistent descriptor for structural domains, it also raises questions for the near future.’ (score: 0.0)\n",
            "Document #58: ‘This reiterates how important it was to survey and re-qualify our consensus data prior to Dynameomics.’ (score: 0.0)\n",
            "Document #57: ‘Fortunately, our consensus approach, along with manual inspection, eliminated many of the domains with confirmed or putative disorder.’ (score: 0.0)\n",
            "Document #56: ‘Inspection of our rejected targets here revealed that many contain disordered regions.’ (score: 0.0)\n",
            "Document #55: ‘When we updated our CDD in 2009, the field of intrinsically disordered proteins was still an emerging field.’ (score: 0.0)\n",
            "Document #79: ‘Noticeably, SGs are increasingly emerging as a critical signaling platform for antiviral immunity and a key regulator of PKR activation (10, 13).’ (score: 0.0)\n",
            "Document #80: ‘Additionally, the activated PKR critically mediates the induction of interferons and modu¬lates cellular in.ammatory responses to build up an antiviral state by inducing sustained activation of NF-.B via activating I.B kinase (IKK) and others (14–16).’ (score: 0.0)\n",
            "Document #81: ‘To overcome PKR restriction, many viruses have evolved various strategies (2).’ (score: 0.0)\n",
            "Document #82: ‘They specify viral products to either interact directly with PKR (17–19), or sequester and degrade dsRNA (20, 21), or target host PKR regulators, such as ADAR1, P58IPK, and PACT (22–24).’ (score: 0.0)\n",
            "Document #105: ‘it does not require other viral factors, but rather depends on the cellular stress itself.’ (score: 0.0)\n",
            "Document #104: ‘Thus, the sorting of nsp1ß into SGs is an intrinsic property of this and colocalizes with PKR in SGs during infection.’ (score: 0.0)\n",
            "Document #103:  (score: 0.0)\n",
            "Document #102: ‘Treatment with other ER stress inducers (arsenite [Ars] and thapsigargin [TG]) obtained the same result (Fig.’ (score: 0.0)\n",
            "Document #101:  (score: 0.0)\n",
            "Document #100: ‘3A, Right).’ (score: 0.0)\n",
            "Document #99: ‘Indeed, treatment of MARC-145 cells with dithiothreitol (DTT), an endoplasmic reticulum (ER) stress inducer, readily led to relocation of ecto¬pically expressed nsp1ß to SGs (Fig.’ (score: 0.0)\n",
            "Document #98: ‘If this is the case, then exposure of nsp1ß in an arti.cial stress condition should allow it to enter SG.’ (score: 0.0)\n",
            "Document #97: ‘Since PRRSV prefers to induce a stress environment (e.g., induction of unfolded protein responses, phosphorylation of eIF2a, and so forth) conducive to replication (34), we hypothesized that nsp1ß might be a stress-responsive protein.’ (score: 0.0)\n",
            "Document #96: ‘PRRSV nsp1ß Is a Stress-Responsive Protein and Colocalizes with PKR in SGs During Infection.’ (score: 0.0)\n",
            "Document #95: ‘Strategically, PRRSV repur¬poses antiviral SGs into a proviral platform to overcome the PKR restriction.’ (score: 0.0)\n",
            "Document #94: ‘Here, we show that the primary pur¬pose of this selective targeting of PKR signaling is to suppress the cellular in.ammatory responses.’ (score: 0.0)\n",
            "Document #93: ‘We have previously reported that the eIF2a phosphorylation is conducive to utilization of ATF4 for viral replication (34).’ (score: 0.0)\n",
            "Document #92: ‘Interestingly, the activity of PKR is disabled in PRRSV-infected cells while the virus still allows the phosphorylation of its substrate eIF2a (32–34).’ (score: 0.0)\n",
            "Document #91: ‘These findings shed light on the mechanisms of PRRSV-mediated suppression of cellular inflammation, which might help in the rationale design of the vaccine, and reveal a unique strategy for viral evasion of PKR and SG restriction.’ (score: 0.0)\n",
            "Document #90: ‘Mechanistically, PRRSV takes advantage of virus-induced stress granules (SGs) and reprograms the antiviral SGs into a proviral platform by utilizing viral replicase protein nsp1ß to co-opt G3BP1 to inhibit PKR activation.’ (score: 0.0)\n",
            "Document #89: ‘Here, we show that this selective targeting of PKR signaling down-regulates cellular inflammatory responses.’ (score: 0.0)\n",
            "Document #88: ‘Enigmatically, PKR activity is disabled by porcine reproductive and respiratory syndrome virus (PRRSV) during infection, while the virus still allows the phosphorylation of its substrate eIF2a.’ (score: 0.0)\n",
            "Document #87: ‘Evolutionarily conserved protein kinase R (PKR) is a powerful mediator of antiviral responses.’ (score: 0.0)\n",
            "Document #86: ‘Conse¬quently, the currently available PRRS vaccines are unable to induce sterilizing immunity (30), and the viral persistence in turn facilitates recombination among different strains and allows selection of immune escape mutants in the .eld (30, 31).’ (score: 0.0)\n",
            "Document #85: ‘One of the most prominent features of PRRSV is subversion of host immunity, as manifested by crippled induction of interferon and in.ammation, leading to inefficient cytotoxic T cell responses (26, 28, 29).’ (score: 0.0)\n",
            "Document #84: ‘PRRSV has remained a major threat to the worldwide swine production ever since its .rst emergence in late 1980s (27).’ (score: 0.0)\n",
            "Document #83: ‘The focus of this report is the interplay of PKR signaling with porcine reproductive and respiratory syndrome virus (PRRSV), an economically devastating swine pathogen and a positive-stranded RNA virus in the family Arteriviridae of the order Nido¬virales (25, 26).’ (score: 0.0)\n",
            "Document #54: ‘Many of the structures assigned to domain families in CATH, SCOP and Dali were not strictly domains in a structural sense.’ (score: 0.0)\n",
            "Document #53: ‘In the course of this work, it became clear that – in some cases - there was much ambiguity as to what qualifies as a domain and what does not.’ (score: 0.0)\n",
            "Document #52: ‘In our Dynameomics project we generated a consensus set of protein folds and selected targets based on the physical attributes of what constituted a globular structural domain.’ (score: 0.0)\n",
            "Document #25: ‘Another 86 folds are composite domains where the structural unit is interrupted by structural elements from a second domain or chain.’ (score: 0.0)\n",
            "Document #23: ‘However, they typically form part of large complexes (PDB ID: 1eq2 in Fig.’ (score: 0.0)\n",
            "Document #22: ‘The 585 folds labeled as “non autonomous”, are not significantly larger than those folds in the other groupings, with only 11 containing over 450 residues.’ (score: 0.0)\n",
            "Document #21:  (score: 0.0)\n",
            "Document #20: ‘Moreover, putative G4-forming sequences have been found in coding and non-coding regions of the human transcriptome, i.e., open reading frames and untranslated regions.’ (score: 0.0)\n",
            "Document #19: ‘In general, G4 sequences are non-randomly distributed but mainly clustered in pivotal genomic regions, namely telomeres, gene promoters and DNA replication origins (15).’ (score: 0.0)\n",
            "Document #18: ‘These data indicate that G4s are mostly suppressed in chromatin and that, in turn, they may influence the occupancy and positioning of nucleosomes.’ (score: 0.0)\n",
            "Document #17: ‘However, mapping of G4s in chromatin by G4 ChIP-sequencing with an anti-G4 antibody (13) or foot printing (14) retrieved only about 10000 G4s in highly transcribed regulatory nucleosome-depleted chromatin regions.’ (score: 0.0)\n",
            "Document #16: ‘These data have been corroborated by 'G4-seq' high-throughput sequencing method, which identified about 700000 G4s (12).’ (score: 0.0)\n",
            "Document #15: ‘Computational analysis using different algorithms (9,10) indicated that 300000 and up to around 3 000000 potential G4-forming sequences may form in the human genome, correlated with specific gene functions (11).’ (score: 0.0)\n",
            "Document #14: ‘While RNA G4s are mostly locked in a parallel conformation due to the 2 -hydroxyl group in the sugar which exclusively allows the anti orientation (2), DNA G4s are in principle characterized by higher topological diversity, even though the majority of DNA G4s examined so far adopt the parallel topology.’ (score: 0.0)\n",
            "Document #13: ‘The anti conformation characterizes a parallel folding, while antiparallel G4s are found to adopt both syn and anti orientations (8) (Figure 1).’ (score: 0.0)\n",
            "Document #12: ‘Strands orientation defines the parallel, antiparallel or mixed topology of G4s, which is directly correlated to the conformational state, anti or syn, of the glycosidic bond between the G base and the sugar (1).’ (score: 0.0)\n",
            "Document #11: ‘G4s can fold intramolecularly from a single G-rich strand, or intermolecularly through dimerization or tetramerization of separate filaments: research of biologically relevant G4s has mainly focused on monomolecular G4s (3,4); however, intermolecular G4s are gaining increasing attention (5-7).’ (score: 0.0)\n",
            "Document #10: ‘G4s are highly polymorphic structures whose topology can be influenced by variations in strand stoichiometry and polarity, as well as by the nature and length of loops and their location in the sequence.’ (score: 0.0)\n",
            "Document #9: ‘Each tetrad is composed of four G residues that are linked by the sugar-phosphate backbone and connected through Hoogsteen-type hydrogen bonds.’ (score: 0.0)\n",
            "Document #8: 'G-quadruplexes (G4s) are nucleic acids secondary structures that can form within DNA (1) or RNA (2) guanine (G)-rich strands, when two or more G-tetrads stack on top of each other and coordinate monovalent cations, such as K+ and Na+.’ (score: 0.0)\n",
            "Document #7: ‘The current state of the art of G4 ligands in antiviral research is also reported, with particular focus on the structural and physicochemical requirements for optimal biological activity.’ (score: 0.0)\n",
            "Document #6: ‘This review aims to provide an updated close-up of the literature on G4s in viruses.’ (score: 0.0)\n",
            "Document #5: ‘In general, G4 ligands showed promising antiviral activity, with G4-mediated mechanisms of action both at the genome and transcript level.’ (score: 0.0)\n",
            "Document #4: ‘In this context, G4 ligands have been developed and tested both as tools to study the complexity of G4mediated mechanisms in the viral life cycle, and as therapeutic agents.’ (score: 0.0)\n",
            "Document #3: ‘Viral G4s are usually located in regulatory regions of the genome and implicated in the control of key viral processes; in some cases, they have been involved also in viral latency.’ (score: 0.0)\n",
            "Document #2: ‘Indeed, G4s have been reported in several viruses, including those involved in recent epidemics, such as the Zika and Ebola viruses.’ (score: 0.0)\n",
            "Document #1: ‘G4s have been extensively described in the human genome, especially in telomeres and oncogene promoters; in recent years the presence of G4s in viruses has at-attracted increasing interest.’ (score: 0.0)\n",
            "Document #24: ‘Most of these non-autonomous folds, being structured in the context of a huge complex or in the presence of binding partners, were rejected because they had large regions buried within a complex making them unlikely to form stable structure in isolation.’ (score: 0.0)\n",
            "Document #26: ‘Examples of these folds are the Rossman, immunoglobulin-like, and citrate synthase domain 1 folds: PDB IDs 1eq2, 1i31 and 1css, respectively, in Figure 4C.’ (score: 0.0)\n",
            "Document #51: ‘If the complexity of attempting to categorize well-structured proteins into domain families has taught us anything, we can be sure that as the gaps in protein fold space are filled, defining a domain will only become more challenging.’ (score: 0.0)\n",
            "Document #27: ‘Although it may seem strange to have structural units with spatially separated sequences, remember that these composite domains are present in the underlying databases as a consequence of the different methods used for categorization.’ (score: 0.0)\n",
            "Document #50: ‘However, at the start of such a revolution in understanding protein structure, or non-structure, it becomes even more important to be able to confidently define domain boundaries and group similar folds together before including ‘dynamic’ or ‘disordered’ protein fold families.’ (score: 0.0)\n",
            "Document #49: ‘Moreover, some disordered regions have already shown attributes consistent with domains, leading to consideration of disordered domain classification [54].’ (score: 0.0)\n",
            "Document #48: ‘Perhaps, this is an indication we should start to consider “dynamic domain” families? After all, conservation of disordered regions has been demonstrated [53].’ (score: 0.0)\n",
            "Document #47: ‘Many disordered proteins have some fold or structure, albeit less compact than the “typical” native states of globular proteins, with many able to form different conformations with different binding partners.’ (score: 0.0)\n",
            "Document #46: ‘It is possible that as we learn more, the folds we previously rejected for not being bona fide domains may be considered as such and we should be open to adapting or expanding the definition of domains.’ (score: 0.0)\n",
            "Document #45: ‘However, there has been increasingly more evidence accruing over the last years that all is not what it first seemed [25].’ (score: 0.0)\n",
            "Document #44: ‘Although there were some exceptions to this rule, they were believed to be just that.’ (score: 0.0)\n",
            "Document #43: ‘The dogma that the native, biologically active state is folded and structured is biased because of the limitations of experimental techniques.’ (score: 0.0)\n",
            "Document #42: ‘Much of what was believed for the last 50 years regarding the relationship between structure and function is now in question as more and more proteins with a local or global lack of structure are identified [25].’ (score: 0.0)\n",
            "Document #41: ‘Accordingly, these rejected non-autonomous folds have a high incidence of known and putative disordered regions.’ (score: 0.0)\n",
            "Document #40: ‘Coupled folding and binding is synonymous with the presence of disorder in unbound states.’ (score: 0.0)\n",
            "Document #39: ‘The heat shock locus HSLU domain (PDB ID: 1ht2) has two predicted disordered regions, one that matches with missing coordinates in the X-ray crystal structures [51, 52] and both are listed in DisProt.’ (score: 0.0)\n",
            "Document #38: ‘The C-terminal domain of topoisomerase II (PDB ID: 1bjt) has a region of missing coordinates representing an acidic region sensitive to protease degradation, leading to the suggestion that it is disordered [49, 50].’ (score: 0.0)\n",
            "Document #37: ‘NMR indicates that the unbound form of the CBP domain is 100% disordered [48].’ (score: 0.0)\n",
            "Document #36: ‘Although shown here in the complexed, structured form with the ACTR domain shaded in gray (Fig.’ (score: 0.0)\n",
            "Document #35: ‘The other domain is the receptor activator (ACTR) domain, which exists as a molten globule until it forms a complex with the CBP domain [48].’ (score: 0.0)\n",
            "Document #34: ‘A first of its kind, this fold representative was one of two domains that are disordered in isolation but become structured upon forming a heterodimer.’ (score: 0.0)\n",
            "Document #33: ‘The CBP domain (PDB ID: 1kbh) is an established IDP [47].’ (score: 0.0)\n",
            "Document #32: ‘Three of these in particular, the C-terminal domain of topoisomerase II (1bjt), cAMP response binding protein (CBP, 1kbh) domain, and heat shock locus HSLU (1ht2), have disorder predictions that correlate with regions already highlighted in DisProt (Fig.’ (score: 0.0)\n",
            "Document #31: ‘Greater disorder was indicated in 43 of these folds, which were either listed in DisProt or had predictions for more than 30 contiguous disordered residues, or both.’ (score: 0.0)\n",
            "Document #30: ‘Some 182 of the 488 targets had predicted disordered regions at least ten residues in length and approximately 70 had missing coordinates, some coincidental with disordered regions as observed above for the crystal structures.’ (score: 0.0)\n",
            "Document #29: ‘Disorder predictions were performed on the 488 continuous folds; the fragmented nature of the 86 composite domains prohibited meaningful sequence analysis.’ (score: 0.0)\n",
            "Document #28: ‘For example, at one of the top-levels in the classification hierarchy of CATH, structures are grouped where they have a similar spatial arrangement of secondary structure elements, irrespective of chain directionality or continuity [46].’ (score: 0.0)\n",
            "Document #106: ‘PKR was also relocalized to PRRSV-induced SGs (Fig.3C).’ (score: 0.0)\n",
            "Document #107: ‘This observation is consistent with previous reports that PKR is a stress-responsive protein and that its activation is dependent on SGs (10, 13, 36).’ (score: 0.0)\n",
            "Document #108: ‘In infected cells, PKR colocalized well with nsp1ß and the SG markers, whereas in mock-infected cells it exhibited a diffusive distribution (Fig.’ (score: 0.0)\n",
            "Document #188: ‘The local concentration of G3BP1, cGAS, and PKR within cytoplasmic foci is proposed to enhance PKR activity.’ (score: 0.0)\n",
            "Document #186: ‘G3BP1 also interacts with cGAS to form large complexes that enhance interaction with viral DNA to induce IFN [30,34,36].’ (score: 0.0)\n",
            "Document #185: ‘For example, the RGG domain of G3BP1 binds the RIG-I helicase domain and viral dsRNA to prevent RIG-I degradation and enhance Ifn-b mRNA expression [30].’ (score: 0.0)\n",
            "Document #184: ‘G3BP1 facilitates the initial IFN-ß response by binding and augmenting the function of viral RNA and DNA sensing proteins in the cytoplasm.’ (score: 0.0)\n",
            "Document #183: ‘G3BP1 regulates early innate responses through its ability to bind nucleic acids and proteins in these signaling pathways.’ (score: 0.0)\n",
            "Document #182: ‘Here, we focus on the role of G3BP1 during viral infection and the various strategies utilized by viruses to exploit or counteract G3BP1 functions.’ (score: 0.0)\n",
            "Document #181: ‘Given that G3BP1 impacts the life cycle of many viruses, it is important to distinguish what specific functional roles are played by G3BP1 at different infection stages.’ (score: 0.0)\n",
            "Document #180: ‘Knockout of both G3BP1 and G3BP2 is necessary to completely inhibit SG formation induced by many stressors, implying their importance to form protein–protein and protein–RNA interactions within SGs [29].’ (score: 0.0)\n",
            "Document #179: ‘SGs are a class of cytoplasmic biomolecular condensates enriched with mRNA–protein complexes assembled in response to stress, including viral infection.’ (score: 0.0)\n",
            "Document #178: ‘Well-studied functions of G3BP1 include participation in the innate response to infection through activation of viral nucleic acid sensors RIG-I and cGAS and as an essential component of stress granules (SGs) [29,30,32–35].’ (score: 0.0)\n",
            "Document #177: ‘For example, some viruses require G3BP1 for efficient viral replication, while the replication of other viruses is negatively affected by G3BP1 expression.’ (score: 0.0)\n",
            "Document #176: ‘Emergent data suggest that G3BP1 is one of the key proteins that impact the viral life cycle both positively and negatively [30,31].’ (score: 0.0)\n",
            "Document #175: ‘For example, knockdown of G3BP1 increases the expression of G3BP2 to compensate for the loss of G3BP1 expression and function, but not vice versa [23,29].’ (score: 0.0)\n",
            "Document #174: ‘The functions of G3BP1 and G3BP2 are redundant in some, but not all cases [29].’ (score: 0.0)\n",
            "Document #173: ‘The RRM domain is essential for RNA binding and RGG domain increases RNA binding as well as mediates protein–protein interactions.’ (score: 0.0)\n",
            "Document #172: ‘For instance, the NTF2L domain interacts with FGDF sequence motifs present in many proteins and the PxxP domain interacts with SH3 motifs.’ (score: 0.0)\n",
            "Document #171: ‘The N-terminal domains contain motifs for protein binding while the C-terminal domains bind nucleic acids.’ (score: 0.0)\n",
            "Document #170: ‘These domains include N-terminal nuclear transport factor 2-like (NTF2L), followed by an acidic domain, proline rich region (PxxP), an RNA recognition motif (RRM), and an arginine and glycine-rich region (RGG) at the C-terminus [27,28].’ (score: 0.0)\n",
            "Document #169: ‘G3BP1 and its paralog G3BP2 possess similar protein domain architecture composed of regions for protein, RNA and DNA interactions (Figure 1).’ (score: 0.0)\n",
            "Document #168: ‘G3BP1 function is further modulated by post-translational modifications and also implicated in many diseases such as cancer, neurodegeneration, and viral infection—the latter is the focus of this review [22–26].’ (score: 0.0)\n",
            "Document #167: ‘G3BP1 is highly conserved from yeast to humans and plays a key role in RNA metabolism, such as regulating the stability and translation of a subset of mRNAs, binding to specific transcripts during stress conditions, and activating interferon (IFN)-stimulated gene expression [18–21].’ (score: 0.0)\n",
            "Document #166: ‘protein-binding protein G3BP1 is an RNA-binding protein targeted by many viruses [17].’ (score: 0.0)\n",
            "Document #165: ‘Concomitantly, induction of cellular immunity is delayed, usually 1 mo following infection, and it is maintained at a low level, contributing to viral persistence in vivo (26, 30).’ (score: 0.0)\n",
            "Document #164: ‘PRRSV is a rather “sneaky” virus, and the infections often lead to low level production of in.ammatory cytokines in the early infec¬tion (26, 28, 29).’ (score: 0.0)\n",
            "Document #187: ‘The binding of the N-terminal region of cGAS with G3BP1 is required for activation of the RNA-dependent protein kinase PKR leading to co-localization as small cytoplasmic foci [35].’ (score: 0.0)\n",
            "Document #189: ‘G3BP1 is required for activation of cGAS as depletion of G3BP1 dramatically reduces DNA-induced IFN production [34].’ (score: 0.0)\n",
            "Document #162: ‘Insight into the Inflammation Regulation by PRRSV.’ (score: 0.0)\n",
            "Document #190: ‘These data suggest that G3BP1-mediated interactions with signaling molecules facilitate the initiation of downstream innate response pathway signaling.’ (score: 0.0)\n",
            "Document #213: ‘As demonstrated by one of the earliest studies using Semliki Forest virus (SFV), infection-induced translation arrest results in the assembly of transient SGs.’ (score: 0.0)\n",
            "Document #212: ‘These patterns highlight the importance of modulating SGs upon virus infection (Table 1).’ (score: 0.0)\n",
            "Document #211: ‘At least four distinct temporal patterns of SGs have been observed during infection with different viruses: (1) stable, (2) transient, (3) oscillating, or (4) no SGs (active blocking of SG induction) [53–55].’ (score: 0.0)\n",
            "Document #210: ‘In light of the viral infection, it may be possible that at the early infection stage, an increase in viral RNA concentration may relieve G3BP1 from its auto-inhibitory state, triggering G3BP1 condensation, with the protein serving as the scaffold to recruit other proteins/RNA required for viral replication.’ (score: 0.0)\n",
            "Document #209: ‘Increasing local RNA concentrations can increase their binding to G3BP1, relieving G3BP1 from the autoinhibitory state and resulting in more RNA recruitment and less aberrant RNA–RNA interaction [50–52].’ (score: 0.0)\n",
            "Document #208: ‘In the normal state, G3BP1 adopts a compact, closed auto-inhibitory state with intramolecular interaction between acidic and arginine-rich regions.’ (score: 0.0)\n",
            "Document #207: ‘G3BP1 is the central node of the protein–protein interaction network within SGs, where RNA serves as a molecular trigger for phase separation.’ (score: 0.0)\n",
            "Document #206: ‘Recent studies provided more mechanistic insights into the role of G3BP1 as the scaffold for SG assembly.’ (score: 0.0)\n",
            "Document #205: ‘These data suggest that G3BP1/G3BP2 act as a scaffold that recruits other proteins into SGs [47,50].’ (score: 0.0)\n",
            "Document #204: ‘RNAi-mediated knockdown or genetic knockout of both G3BP1 and G3BP2 results in cells that do not form SGs in response to most stressors, emphasizing the importance of G3BP1/G3BP2 in SG assembly [29].’ (score: 0.0)\n",
            "Document #203: ‘G3BP1 and G3BP2 form the core protein–protein interaction network which binds translation factors, mRNA binding proteins, long and translationally incompetent mRNAs as well as specific transcripts during stress conditions [19,48,49].’ (score: 0.0)\n",
            "Document #202: ‘Of note, G3BP1 itself undergoes several post-translational modifications, including phosphorylation, methylation, ubiquitination, and ADP-ribosylation during stress conditions [42,44,47].’ (score: 0.0)\n",
            "Document #201: ‘Several post-translational modifications (PTMs) control the dynamics of SGs [33,41–46].’ (score: 0.0)\n",
            "Document #200: ‘These untranslated mRNAs bound with ribonucleoproteins (mRNPs) cluster together with G3BP1, G3BP2, and several other RNA binding proteins to form RNA/protein-rich condensates in the cytoplasm.’ (score: 0.0)\n",
            "Document #199: ‘Once the elongating ribosomes run off from polysomes, the stalled transla¬tion initiation results in a sudden influx of untranslated mRNAs in the cytoplasm.’ (score: 0.0)\n",
            "Document #198: ‘The initiation complex containing the phospho¬rylated eIF2a becomes unavailable for recycling, which leads to a stall in translation initiation [40].’ (score: 0.0)\n",
            "Document #197: ‘The activated kinase phosphorylates the translation initiation factor eIF2a subunit [38,39].’ (score: 0.0)\n",
            "Document #196: ‘Upon stress, one of the following five kinases is activated: double-stranded RNA (dsRNA)-dependent PKR, protein kinase R-like endoplasmic reticulum kinase (PERK), heme-regulated inhibitor (HRI), general control nonderepressible-2 (GCN2), or microtubule affinity-regulating kinase 2 (MARK2).’ (score: 0.0)\n",
            "Document #195: ‘SGs assemble upon stress-induced translation arrest that results in the accumulation of free untranslated mRNAs and mRNA binding proteins in the cyto¬plasm [37].’ (score: 0.0)\n",
            "Document #194: A well-known function of G3BP1 during stress stimuli is its involvement in SG as¬sembly (Figure 2b).’ (score: 0.0)\n",
            "Document #193: ‘G3BP1 Forms the Core of Stress Granules (score: 0.0)\n",
            "Document #192: ‘For example, G3BP1, G3BP2, and their binding partner Caprin1 positively regulate the translation of multiple ISGs in response to IFN (Figure 2a) [21].’ (score: 0.0)\n",
            "Document #191: ‘G3BP1 can also facilitate the expression and translation of IFN-stimulated genes (ISGs).’ (score: 0.0)\n",
            "Document #163: ‘Our results also unexpectedly revealed a regulatory mechanism for PRRSV-mediated suppression of host cellular in.ammatory responses.’ (score: 0.0)\n",
            "Document #161: ‘Overall, these results suggest a strategy of two birds (G3BP1 and PKR) with one stone (nsp1ß).’ (score: 0.0)\n",
            "Document #109: ‘3C).’ (score: 0.0)\n",
            "Document #134: ‘9A and B).’ (score: 0.0)\n",
            "Document #132: ‘Only two mutants (VS19GG and GGK16AAA) were successfully rescued.’ (score: 0.0)\n",
            "Document #131: ‘ (score: 0.0)\n",
            "Document #130: G3BP1 Contributes to nsp1ß-Mediated Inhibition of PKR Activ¬ity.’ (score: 0.0)\n",
            "Document #129:  (score: 0.0)\n",
            "Document #128: ‘Together, these results suggest that the nsp1ß-PKR interaction is regulated and facilitated by cellular factors during infection.’ (score: 0.0)\n",
            "Document #127: ‘Consistent with this notion, nsp1ß showed partial colocalization with Flag-DRBM in the cytoplasm, but not with the full-length Flag-PKR or Flag-KD in transfected cells (Fig.’ (score: 0.0)\n",
            "Document #126: ‘4C), suggesting that a conformational change of PKR is likely necessary for the interaction.’ (score: 0.0)\n",
            "Document #125: ‘4D,lane 11). (score: 0.0)\n",
            "Document #124: ‘4D,lane 10),but the TF-DRBM mutant was abundantly recovered (Fig.’ (score: 0.0)\n",
            "Document #123: ‘To our surprise, GST-nsp1ß was unable to pull down the full-length PKR (TF-PKR) (Fig.’ (score: 0.0)\n",
            "Document #122: ‘Specifically, PRRSV nsp1ß was expressed as a GST fusion protein, whereas PKR or its deriva¬tives (DRBM and KD) were fused to the C terminus of bacterial chaperone trigger factor (TF) to promote protein solubility (Fig.’ (score: 0.0)\n",
            "Document #121: ‘The relevant proteins were expressed in a bacterial system.’ (score: 0.0)\n",
            "Document #120: ‘To test whether the interaction requires any eukaryotic factor, we carried out the in vitro binding assay.’ (score: 0.0)\n",
            "Document #119: ‘The co-IP assay showed that the mutant Flag-DRBM retained the ability to interact with nsp1ß to the level of full-length PKR (Fig.C, lane 6), whereas the mutant Flag-KD failed to interact suggesting that DRBM is the key region for binding to nsp1ß.’ (score: 0.0)\n",
            "Document #118: ‘WT PKR and its derivatives (Flag-DRBM and Flag-KD) were expressed either individually or in combination with HA-nsp1ß in HEK-293FT cells.’ (score: 0.0)\n",
            "Document #117: ‘The nsp1ß-binding region was further mapped by construction of PKR truncation mutants containing only the DRBM or the KD.’ (score: 0.0)\n",
            "Document #116: ‘The anti¬bodies to PKR could readily pull down nsp1ß from PRRSV-infected MARC-145 cell lysates and the same was true under the condition of transfection (Fig.’ (score: 0.0)\n",
            "Document #115: ‘We tested this hypothesis by coimmunoprecipitation assay (co-IP).’ (score: 0.0)\n",
            "Document #114: ‘It is likely that there is an interaction between nsp1ß and PKR.’ (score: 0.0)\n",
            "Document #113: ‘Regulated Interaction between nsp1ß and PKR.’ (score: 0.0)\n",
            "Document #112: ‘Thus, these findings suggest that SGs are likely the battleground for nsp1ß to counteract PKR.’ (score: 0.0)\n",
            "Document #111: ‘3D).’ (score: 0.0)\n",
            "Document #110: ‘In transfected cells, nsp1ß did not colocalize with PKR, but they became colocalized once the cells were stressed with SG inducers (e.g., DTT or Ars) (Fig.’ (score: 0.0)\n",
            "Document #133: ‘The mutant GGK16AAA displayed a growth property similar to the parental virus in both MARC-145 cells and PAMs, whereas the growth of VS19GG was reduced (Fig.’ (score: 0.0)\n",
            "Document #135: ‘Similar to that in the transfection condition, the interaction of nsp1ß with G3BP1 and PKR were almost diminished in VS19GG infected cells (Fig.’ (score: 0.0)\n",
            "Document #160: ‘Certainly, this is a different story from PKR antagonization and warrants further exploration in the future.’ (score: 0.0)\n",
            "Document #136: ‘This was accom¬panied by a concurrent elevation of PKR phosphorylation in both cell types (score: 0.0)\n",
            "Document #159: ‘9A and B).’ (score: 0.0)\n",
            "Document #158: ‘This notion is in line with the observation that the mutant VS19GG had a reduced growth rate in both cell types (Fig.’ (score: 0.0)\n",
            "Document #157: ‘8B and C).’ (score: 0.0)\n",
            "Document #156: ‘This seemingly confusing result may be explained by WT nsp1ß normally inhibiting the antiviral function of G3BP1 via interaction, whereas the mutant nsp1ß VS19GG fails to do so due to loss of interaction (Fig.’ (score: 0.0)\n",
            "Document #155: ‘10).’ (score: 0.0)\n",
            "Document #154: ‘As so, cation of the mutant VS19GG (Fig.’ (score: 0.0)\n",
            "Document #153: ‘Cytoplasmic SGs are increasingly emerging as a critical immune signaling platform to detect and restrict invading viral pathogens (10, 11).’ (score: 0.0)\n",
            "Document #152: ‘Repurposing SG into a Proviral Platform.’ (score: 0.0)\n",
            "Document #151: ‘Asterisks (*) indicate the statistical significance: *P < 0.05; ***P < 0.001; NS, no significance.’ (score: 0.0)\n",
            "Document #150: ‘Statistical analysis was performed by two-tailed Student’s t test, and error bars indicate means ± SDs.’ (score: 0.0)\n",
            "Document #149: ‘The relative abundance of p-PKR in each treat¬ment was expressed as fold-changes compared to the intensity of p-PKR in the .rst lane after being normalized against PKR and then ß-actin in the same lane.’ (score: 0.0)\n",
            "Document #148: ‘The signals of protein bands were determined by ImageJ.’ (score: 0.0)\n",
            "Document #147: ‘(B) The same as A, except cells were transfected to express HA-nsp1ß.(C and D) The same as A and B, except that siRNAs were used to target TIA1.’ (score: 0.0)\n",
            "Document #146: ‘The cells were then subject to Western blot analysis with antibodies to the indicated proteins.’ (score: 0.0)\n",
            "Document #145: ‘At 36 h post transfection, the cells were infected with PRRSV strain JXwn06 at an MOI of 0.1 for 36 h before being treated with poly(I:C) for 12 h.’ (score: 0.0)\n",
            "Document #144: ‘(A) MARC-145 cells were transfected with siRNAs targeting G3BP1 or scram¬bled siRNA (siNC).’ (score: 0.0)\n",
            "Document #143: ‘On the other hand, despite a reduced viral growth, the mutant G3BP1 contributes to nsp1ß-mediated inhibition of PKR activation.’ (score: 0.0)\n",
            "Document #142: ‘In contrast, production of IFN-ß mRNA was highly inducible As compared with WT virus, there was a slight up-regulation of IFN-ß mRNA, but for both mutants, suggesting that the mutational effect is less likely due to PKR activation.’ (score: 0.0)\n",
            "Document #141: ‘In all treat¬ments, the CT values for IFN-a mRNA were near or over 35 cycles , and therefore the fold-changes are not reliable.’ (score: 0.0)\n",
            "Document #140: ‘To investigate the biological significance, we measured the production of interferons and in.ammatory cytokines by RT-qPCR in PRRSV-infected cells.’ (score: 0.0)\n",
            "Document #139: ‘The Mutant nsp1ß VS19GG Induces Inflammatory Responses via G3BP1-PKR Axis.’ (score: 0.0)\n",
            "Document #138: ‘Thus, nsp1ß is responsible for inhibiting PKR activation dur¬ing infection.’ (score: 0.0)\n",
            "Document #137: as opposed to both WT and the mutant nsp1ß GGK16AAA .’ (score: 0.0)\n",
            "Document #432: ‘Microtubule disruption leads to the formation of small SGs after NDV infection (score: 0.0)\n",
            "Document #433: SGs are considered to be dynamic structures.’ (score: 0.0)\n",
            "Document #434: ‘The move¬ment of canonical SGs is associated with cytoskeletal ele¬ments, especially microtubules (40–42).’ (score: 0.0)\n",
            "Document #719: ‘The direct visualization of G-quadruplex structures in human cellsas well as the co-crystal structure of an RNA helicase bound to a G-quadruplexes have provided important confirmations of their relevance to cell biology.’ (score: 0.0)\n",
            "Document #730: ‘Its concentration is used as a measurement of oxidative stress within a cell.’ (score: 0.0)\n",
            "Document #729: ‘Due to Guanine having a lower electron reduction potential than the other nucleotides bases,-oxo-'-deoxyguanosine (-oxo-dG), is a known major product of DNA oxidation.’ (score: 0.0)\n",
            "Document #728: ‘Guanine (G) bases in G-quadruplex have the lowest redox potential causing it to be more susceptible to the formation of -oxoguanine (-oxoG), an endogenous oxidized DNA base damage in the genome.’ (score: 0.0)\n",
            "Document #727: ‘These enzymes participate in BER to repair certain DNA lesions such as ,-dihydro--oxoguanine (-oxoG), which forms under oxidative stress to guanine bases.’ (score: 0.0)\n",
            "Document #726: ‘The BER pathway is signalled when it indicates an oxidative DNA base damage, where structures like, -0xoguanine-DNA glycosylase  (OGG), APE and G-quadruplex play a huge role in its repair.’ (score: 0.0)\n",
            "Document #725: ‘Base excision repair processes in cells have been proved to be reduced with aging as its components in the mitochondria begin to decline, which can lead to the formation of many diseases such as Alzheimer's disease (AD)_These G-quadruplex structures are said to be formed in the promoter regions of DNA through superhelicity, which favors the unwinding of the double helical structure of DNA and in turn loops the strands to form G­ quadruplex structures in guanine rich regions.’ (score: 0.0)\n",
            "Document #724: ‘AP site damage was found to be predominant in PQS regions of the genome, where formation of G-quadruplex structures is regulated and promoted by the DNA repair process, base excision repair (BER).’ (score: 0.0)\n",
            "Document #723: ‘AP site damage was also more prevalent in certain regions of the genome that contain specific active promoter and enhancer markers, some of which were linked to regions responsible for lung adenocarcinoma and colon cancer.’ (score: 0.0)\n",
            "Document #722: ‘Both of these genome-wide mapping sequencing methods, ChIP-sequencing and ARP, have indicated that AP site damage occurrence is nonrandom.’ (score: 0.0)\n",
            "Document #721: ‘T-loops and G-quadruplexes are described as the two tertiary DNA structures that protect telomere ends and regulate telomere length. Many of the genome regulatory processes have been linked to the formation of G-quadruplex structures, attributable to the huge role it plays in DNA repair of apurinic/apyrimidinic sites also known as AP sites. A new technique to map AP sites has been developed known as AP-seq which utilizes a biotin-labeled aldehyde-reactive probe (ARP) to tag certain regions of the genome where AP site damage occurrence has been significant. Another genome-wide mapping sequencing method known as ChIP-sequencing, was utilized to map both; damage in AP sites, and the enzyme responsible for its repair, AP endonuclease  (APE).’ (score: 0.0)\n",
            "Document #720: ‘The potential positive and negative roles of quadruplexes in telomere replication and function remains controversial.’ (score: 0.0)\n",
            "Document #718: ‘The roles of quadruplex structure in translation control are not as well explored.’ (score: 0.0)\n",
            "Document #651: 'virus Papanivirus – Panicum mosaic satellite virus Virtovirus – Tobacco mosaic satellite virus Albetovirus – Tobacco necrosis satellite virus, Family Sarthroviridae, Macronovirus – Macrobrachium satellite virus 1 (extra small virus) (unnamed genus) – Nilaparvata lugens commensal X virus (unnamed genus) – Chronic bee-paralysis satellite virus, Family Lavidaviridae – Virophages Single-stranded DNA satellite viruses, Genus Dependoparvovirus – Adeno-associated.' (score: 0.0)\n",
            "Document #717: ‘There are many studies that implicate quadruplexes in both positive and negative transcriptional regulation, including epigenetic regulation of genes like hTERT.  Function of G-quadruplexes have also been reported in allowing programmed recombination of immunologlobin heavy genes and the pilin antigenic variation system of the pathogenic Neisseria.’ (score: 0.0)\n",
            "Document #716: ‘ The DEAH/RHA helicase, DHX, has also been identified as a key G-quadruplex resolvase_o A metastasis suppressor protein NMH (also known as NME) was found to directly interact with G quadruplex in the promoter of the c-myc gene, and transcriptionally regulate c-myc. More recently, NMH was reported to interact with G-quadruplex in the promoter of the human telomerase (hTERT) gene and regulate hTERT expression.  The telomere-binding-factor- (TRF or TERF) was shown to bind to thousands of non-telomeric G-quadruplexes in the human genome by TRF ChIP-seq.’ (score: 0.0)\n",
            "Document #715: ‘ Quadruplex formation may be potentially damaging for a cell; the helicases WRN and Bloom syndrome protein have a high affinity for resolving DNA G-quadruplexes.’ (score: 0.0)\n",
            "Document #714: ‘It has been suggested that quadruplex formation plays a role in immunoglobulin heavy chain switching. As cells have evolved mechanisms for resolving (i.e., unwinding) quadruplexes that form.’ (score: 0.0)\n",
            "Document #713: ‘In another model, quadruplex formed at the non-coding DNA strand helps to maintain an open conformation of the coding DNA strand and enhance an expression of the respective gene.’ (score: 0.0)\n",
            "Document #712: ‘One model is shown below, with G­ quadruplex formation in or near a promoter blocking transcription of the gene, and hence de­ activating it.’ (score: 0.0)\n",
            "Document #711: ‘There are several possible models for how quadruplexes could influence gene activity, either by upregulation or downregulation.’ (score: 0.0)\n",
            "Document #710: ‘coli.’ (score: 0.0)\n",
            "Document #709: ‘A similar studies have identified putative G-quadruplexes in prokaryotes, namely the bacterium E.’ (score: 0.0)\n",
            "Document #708: ‘ Genome-wide surveys based on a quadruplex folding rule have been performed, which have identified ,000 Putative Quadruplex Sequences (PQS) in the human genome, although not all of these probably form in vivo.’ (score: 0.0)\n",
            "Document #707: ‘ Other genes shown to form G-quadruplexes in their promoter regions include the chicken -globin gene, human ubiquitin­ ligase RFP, and the proto-oncogenes c-kit, bcl-, VEGF, H-ras and N-ras.’ (score: 0.0)\n",
            "Document #731: ‘When DNA undergoes oxidative damage, a possible structural change in guanine, after ionizing radiation, gives rise to an enol form, -0H-Gua.’ (score: 0.0)\n",
            "Document #732: ‘This oxidative product is formed through a tautomeric shift from the original damage guanine, -oxo-Gua, and represents DNA damage that causes changes in the structure.’ (score: 0.0)\n",
            "Document #733: ‘This form allows for the base excision repair (BER) enzyme OGG to bind and remove the oxidative damage with the help of APE, resulting in an AP site.’ (score: 0.0)\n",
            "Document #734: ‘ Moreover, an AP site is a location in DNA that has neither a purine or a pyrimidine base due to DNA damage, they are the most prevalent type of endogenous DNA damage in cells.’ (score: 0.0)\n",
            "Document #757: ‘Some of these oncogenes include c-KIT, PDGF-A, c-Myc and VEGF, showing the importance of this secondary structure in cancer growth and development.’ (score: 0.0)\n",
            "Document #756: ‘The structures most present in the promoter regions of these oncogenes tend to be parallel-stranded G-quadruplex DNA structures.’ (score: 0.0)\n",
            "Document #755: ‘Along with the association of G-quadruplexes in telomeric regions of DNA, G-quadruplex structures have been identified in various human proto-oncogene promoter regions.’ (score: 0.0)\n",
            "Document #754: ‘For therapeutic advances, stabilizing the G­ quadruplexes of cancerous cells can inhibit cell growth and replication leading to the cell's death.’ (score: 0.0)\n",
            "Document #753: ‘This causes replication of damaged and cancerous cells.’ (score: 0.0)\n",
            "Document #752: ‘However, in cancer cells that have mutated helicase these complexes cannot be unwound and leads to potential damage of the cell.’ (score: 0.0)\n",
            "Document #751: ‘G-quadruplexes can inhibit normal cell function, and in healthy cells, are easily and readily unwound by helicase.’ (score: 0.0)\n",
            "Document #750: ‘Therefore, we can conclude that APE has two important roles in genome regulation- Stabilizing the formation of g-quadruplex structures and loading the transcriptional factors onto the AP site G-quadruplex forming sequences are prevalent in eukaryotic cells, especially in telomeres,  untranslated strands, and translocation hot spots.’ (score: 0.0)\n",
            "Document #749: ‘However, not all G-quadruplex structures require APE for formation, in fact some of them formed greater G-quadruplex structures in its absence.’ (score: 0.0)\n",
            "Document #748: ‘When a study downregulated the presence of APE and AcAPE in the cell, the formation of G-quadruplex structures was inhibited, which proves the importance of APE for the formation of these structures.’ (score: 0.0)\n",
            "Document #747: ‘Deacetylation of AcAPE is the driving force behind the loading of these TFs, where APE dissociates from the G-quadruplex structures.’ (score: 0.0)\n",
            "Document #746: ‘After this stage the APE gets acetylated by multiple lysine residues on the chromatin, forming acetylated APE (AcAPE). AcAPE is very crucial to the BER pathway as it acts as a transcriptional coactivator or corepressor, functioning to load transcription factors (TF) into the site of damage allowing it to regulate the gene expression.  AcAPE is also very important since it allows APE to bind for longer periods of time by delay of its dissociation from the sequence, allowing the repair process to be more efficient.’ (score: 0.0)\n",
            "Document #745: ‘This promotes formation of G-quadruplex structures by the folding of the stand.0 This looping process brings four bases in close proximity that will be held together by Hoogsteen base pairing.’ (score: 0.0)\n",
            "Document #744: ‘This AP site then signals cells to engage APE binding, which binds to the open duplex region.SThe binding of APE then plays an important role by stabilizing the formation of G-quadruplex structures in that region.’ (score: 0.0)\n",
            "Document #743: ‘Once the cell is aware of oxidative stress and damage, it recruits OGG to the site, whose main function is to initiate the BER pathway.OGG does so by cleaving the oxidized base and thus creating an AP site, primarily through the process of negative superhelicity.’ (score: 0.0)\n",
            "Document #742: ‘The oxidation of certain purine bases, like guanine, forms oxidized nucleotides that impairs DNA function by mismatching nucleotides in the sequences.This is more common in PQS sequences which form oxidized structures, such as - oxoguanine.’ (score: 0.0)\n",
            "Document #741: ‘APE is considered to be very crucial as AP site damage is known to be the most recurring type of endogenous damage to DNA.’ (score: 0.0)\n",
            "Document #740: ‘APE is mainly in charge of repairing damage caused to AP sites through the BER pathway.’ (score: 0.0)\n",
            "Document #739: ‘AP endonuclease  (APE) is an enzyme responsible for the promotion and the formation of G­ quadruplex structures.’ (score: 0.0)\n",
            "Document #738: ‘Upon insertion of -oxo-dG into thymidine kinase gene of humans, it was determine that if -oxo-dG was left unchecked and not repaired by BER, it can lead to frequent mutations and eventually carcinogenesis.’ (score: 0.0)\n",
            "Document #737: ‘The possible inactivation allows for un-repaired DNA damages to gather in non-replicating cells, like muscle, and can cause aging as well.  Moreover, oxidative DNA damage like -oxo-dG contributes to carcinogenesis through the modulation of gene expression, or the induction of mutations. On the condition that -oxo-dG is repaired by BER, parts of the repair protein is left behind which can lead to epigenetic alterations, or the modulation of gene expression.’ (score: 0.0)\n",
            "Document #736: ‘With the use of genome-wide ChIP-sequencing analyses, cell-based assays, and in vitro biochemical analyses, a connection has been made between oxidized DNA base-derived AP sites, and the formation of the G-quadruplex.  Furthermore, the concentration of -oxo-dG is a known biomarker of oxidative stress within a cell, and excessive amount of oxidative stress has been linked to carcinogenesis and other diseases_SWhen produced, -oxo-dG, has the ability to inactivate OGG, thus preventing the repair of DNA damage caused by the oxidation of guanine.’ (score: 0.0)\n",
            "Document #735: ‘AP sites can be generated spontaneously or after the cleavage of modified bases, like -0H-Gua.iJThe generation of an AP site enables the melting of the duplex DNA to unmask the PQS, adopting aG-quadruplex fold.’ (score: 0.0)\n",
            "Document #706: ‘The proto-oncogene c-myc forms a quadruplex in a nuclease hypersensitive region critical for gene activity.’ (score: 0.0)\n",
            "Document #705: ‘In addition, there was found more than one-billion-year conserved G-quadruplex locus in plants and algae, in gene encoding large subunit of RNA polymerase _i Although these studies predicted G-quadruplex-mediated gene regulation, it is unlikely that all pGs would form in vivo.’ (score: 0.0)\n",
            "Document #704: ‘Here also, like vertebrates G-quadruplexes were enriched within gene promoters.’ (score: 0.0)\n",
            "Document #677: ‘The hydrogen bonds in these layers are represented by blue dashed lines.’ (score: 0.0)\n",
            "Document #675: ‘The backbone is represented by a tube.’ (score: 0.0)\n",
            "Document #674: ‘These structures, formed from two separate G-D Structure of the intramolecular human telomeric G-quadruplex in potassium solution.’ (score: 0.0)\n",
            "Document #673: ‘Longer sequences, which contain two contiguous runs of three or more guanine bases, where the guanine regions are separated by one or more bases, only require two such sequences to provide enough guanine bases to form a quadruplex.’ (score: 0.0)\n",
            "Document #672: ‘The term G DNA was originally reserved for these tetramolecular structures that might play a role in meiosis However, as currently used in molecular biology, the term G can mean G-quadruplexes of any molecularity.’ (score: 0.0)\n",
            "Document #671: ‘Such a quadruplex is described as tetramolecular, reflecting the requirement of four separate strands.’ (score: 0.0)\n",
            "Document #670: ‘Short sequences, consisting of only a single contiguous run of three or more guanine bases, require four individual strands to form a quadruplex.’ (score: 0.0)\n",
            "Document #669: ‘The length of the nucleic acid sequences involved in tetrad formation determines how the quadruplex folds.’ (score: 0.0)\n",
            "Document #668: ‘Early evidence for the formation of G­ quadruplexes in vivo in cells was established by isolating them from cells, and later by the observation that specific DNA helicases could be identified where small molecules specific for these DNA structures accumulated in cells.’ (score: 0.0)\n",
            "Document #667: ‘Current research consists of identifying the biological function of these G­ Quadruplex structures for specific oncogenes and discovering effective therapeutic treatments for cancer based on interactions with G-quadruplexes.’ (score: 0.0)\n",
            "Document #666: ‘With the abundance of G-quadruplexes in vivo, these structures hold a biologically relevant role through interactions with the promoter regions of oncogenes and the telomeric regions of DNA strands.’ (score: 0.0)\n",
            "Document #665: ‘In 00, the prevalence of G-quadruplexes within gene promoters of several bacterial genomes was reported predicting G-quadruplex-mediated gene regulation.’ (score: 0.0)\n",
            "Document #664: ‘Interest in in vivo function of G-quadruplexes surged after large scale genome­ wide analysis showed the prevalence of potential G-quadruplex (pG)-forming sequences within gene promoters of human, chimpanzee, mouse, and rat - presented in the First International G-quadruplex Meeting held in April 00 in Louisville, Kentucky.’ (score: 0.0)\n",
            "Document #663: ‘The importance of discovering G-quadruplex structure was described through the statement, \"If G-quadruplexes form so readily in vitro, Nature will have found a way of using them in vivo\" - Aaron Klug, Nobel Prize Winner in Chemistry ().’ (score: 0.0)\n",
            "Document #662: ‘Their activities in basic genetic processes are an active area of research in telomere, gene regulation, and functional genomics research. The identification of structures with a high guanine association became apparent in the early 0s, through the identification of gel-like substances associated with guanines. More specifically, this research detailed the four-stranded DNA structures with a high association of guanines, which was later identified in eukaryotic telomeric regions of DNA.’ (score: 0.0)\n",
            "Document #661: ‘G-quadruplex structures can be computationally predicted from DNA or RNA sequence motifs, but their actual structures can be quite varied within and between the motifs, which can number over 00,000 per genome.’ (score: 0.0)\n",
            "Document #660: ‘They can be formed of DNA, RNA, LNA, and PNA, and may be intramolecular, bimolecular, or tetramolecular. Depending on the direction of the strands or parts of a strand that form the tetrads, structures may be described as parallel or antiparallel.’ (score: 0.0)\n",
            "Document #659: ‘The quadruplex structure is further stabilized by the presence of a cation, especially potassium, which sits in a central channel between each pair of tetrads.’ (score: 0.0)\n",
            "Document #658: ‘The placement and bonding to form G-quadruplexes is not random and serve very unusual functional purposes.’ (score: 0.0)\n",
            "Document #657: ‘Four guanine bases can associate through Hoogsteen hydrogen bonding to form a square planar structure called a guanine tetrad (G-tetrad or G-quartet), and two or more guanine tetrads (from G-tracts, continuous runs of guanine) can stack on top of each other to form a G-quadruplex.’ (score: 0.0)\n",
            "Document #656: ‘They are helical in shape and contain guanine tetrads that can form from one, two or four strands The unimolecular forms often occur naturally near the ends of the chromosomes, better known as the telomeric regions, and in transcriptional regulatory regions of multiple genes, both  in  microbes  and  across vertebrates SJincluding oncogenes in humans.’ (score: 0.0)\n",
            "Document #655: In molecular biology, G-quadruplex secondary structures (G) are formed in nucleic acids by sequences that are rich in guanine.’ (score: 0.0)\n",
            "Document #654: Viruses can be placed in one of the seven following groups including the dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses), ssDNA viruses (+ strand or \"sense\") DNA (e.g. Parvoviruses), dsRNA viruses (e.g. Reoviruses), (+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses) V, or ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses) VII.' (score: 0.0)\n",
            "Document #653: Defective interfering particles are defective viruses that have lost their ability to replicate except in the presence of a helper virus, which is normally the parental virus. They can also interfere with the helper virus.' (score: 0.0)\n",
            "Document #676: ‘The center of this structure contains three layers of G-tetrads.’ (score: 0.0)\n",
            "Document #678: ‘ Guanine-rich strands, are termed bimolecular quadruplexes.’ (score: 0.0)\n",
            "Document #703: ‘coli and hundreds of other microbial genomes.’ (score: 0.0)\n",
            "Document #679: ‘Finally, sequences which contain four distinct runs of guanine bases can form stable quadruplex structures by themselves, and a quadruplex formed entirely from a single strand is called an intramolecular quadruplex.’ (score: 0.0)\n",
            "Document #702: ‘Similarly, large number of G­ quadruplexes were found in the E.’ (score: 0.0)\n",
            "Document #701: ‘A large number of the non-telomeric G-quadruplexes were found within gene promoters, and were conserved across the species.’ (score: 0.0)\n",
            "Document #700: ‘Analysis of human, chimpanzee, mouse and rat genomes showed enormous number of potential G-quadruplex (pG)-forming sequences in non-telomeric regions.’ (score: 0.0)\n",
            "Document #699: ‘Quadruplexes are present in locations other than at the telomere.’ (score: 0.0)\n",
            "Document #698: ‘This is an active target of drug discovery, including telomestatin.’ (score: 0.0)\n",
            "Document #697: ‘The presence of these structures prevent telomere elongation by the telomerase complex.  Telomeric repeats in a variety of organisms have been shown to form these quadruplex structures in vitro, and subsequently they have also been shown to form in vivo. The human telomeric repeat (which is the same for all vertebrates) consists of many repeats of the sequenced (GGTTAG), and the quadruplexes formed by this structure can be in bead-like structures of  nm to  nm in size and have been well studied by NMR, TEM and X-ray crystal structure determination. The formation of these quadruplexes in telomeres has been shown to decrease the activity of the enzyme telomerase, which is responsible for maintaining length of telomeres and is involved in around % of all cancers.’ (score: 0.0)\n",
            "Document #696: ‘At these’ end protrusions, the G-rich overhang can form secondary structures such as G-quadruplexes if the overhang is longer than four TTAGGG repeats.’ (score: 0.0)\n",
            "Document #695: ‘The heterodimeric complex ribonucleoprotein enzyme telomerase adds TTAGGG repeats at the’ end of DNA strands.’ (score: 0.0)\n",
            "Document #694: ‘The repeats end with a’ protrusion of between 0 and 0 single-stranded TTAGGG repeats.’ (score: 0.0)\n",
            "Document #693: ‘These telomeric regions are characterized by long regions of double-stranded CCCTAA:TTAGGG repeats.’ (score: 0.0)\n",
            "Document #692: ‘Telomeres, rich in guanine and with a propensity to form g-quadruplexes, are located at the terminal ends of chromosomes and help maintain genome integrity by protecting these vulnerable terminal ends from instability.’ (score: 0.0)\n",
            "Document #691: ‘Telomeres function to provide this signaling.’ (score: 0.0)\n",
            "Document #690: ‘Because repair enzymes would naturally recognize ends of linear chromosomes as damaged DNA and would process them as such to harmful effect for the cell, clear signaling and tight regulation is needed at the ends of linear chromosomes.’ (score: 0.0)\n",
            "Document #689: ‘In addition, the propensity of g­ quadruplex formation during transcription in RNA sequences with the potential to form mutually exclusive hairpin or G-quadruplex structures depends heavily on the position of the hairpin-forming sequence.’ (score: 0.0)\n",
            "Document #688: ‘This effect seems to be mediated by alteration of the hydration of the DNA and its effect on Hoogsteen base pair bonding These quadruplexes seemed to readily occur at the ends of chromosome.’ (score: 0.0)\n",
            "Document #687: ‘For instance, quantitative assessments of the thermodynamics of molecular crowding indicate that the antiparallel g-quadruplex is stabilized by molecular crowding.’ (score: 0.0)\n",
            "Document #686: ‘Depending on cell type and cell cycle, mediating factors such as DNA-binding proteins on chromatin, composed of DNA tightly wound around histone proteins, and other environmental conditions and stresses affect the dynamic formation of quadruplexes.’ (score: 0.0)\n",
            "Document #685: ‘Following sequencing of the human genome, many guanine-rich sequences that had the potential to form quadruplexes were discovered.’ (score: 0.0)\n",
            "Document #684: ‘In quadruplexes formed from double-stranded DNA, possible interstrand topologies have also been discussed.  Interstrand quadruplexes contain guanines that originate from both strands of dsDNA.’ (score: 0.0)\n",
            "Document #683: ‘The loops joining runs of guanine bases in intramolecular antiparallel quadruplexes are either diagonal, joining two diagonally opposite runs of guanine bases, or lateral (edgewise) type loops, joining two adjacent runs of guanine base pairs.’ (score: 0.0)\n",
            "Document #682: ‘If one or more of the runs of guanine bases has a’-' direction opposite to the other runs of guanine bases, the quadruplex is said to have adopted an antiparallel topology.’ (score: 0.0)\n",
            "Document #681: ‘For intramolecular quadruplexes, this means that any loop regions present must be of the propeller type, positioned to the sides of the quadruplex.’ (score: 0.0)\n",
            "Document #680: ‘Depending on how the individual runs of guanine bases are arranged in a bimolecular or intramolecular quadruplex, a quadruplex can adopt one of a number of topologies with varying loop configurations. If all strands of DNA proceed in the same direction, the quadruplex is termed parallel.’ (score: 0.0)\n",
            "Document #758: ‘While the formation of G­ quadruplex structure vary to some extent for the different promoter regions of oncogenes, the consistent stabilization of these structures have been found in cancer development.’ (score: 0.0)\n",
            "Document #759: ‘Current therapeutic research actively focuses on targeting this stabilization of G-quadruplex structures to arrest unregulated cell growth and division.’ (score: 0.0)\n",
            "Document #760: ‘One particular gene region, the c-myc pathway, plays an integral role in the regulation of a protein product, c-Myc.’ (score: 0.0)\n",
            "Document #840: ‘Small-molecule ligands, composed primarily of lead, can target GGGGCC repeat regions as well and ultimately decreased both repeat-associated non-ATG translation and RNA foci in neuron cells derived from patients with Amyotrophic lateral sclerosis (ALS).’ (score: 0.0)\n",
            "Document #838: ‘A disadvantage of using small-molecule ligands as a therapeutic technique is that specificity is difficult to manage due to the variability of G-quadruplexes in their primary sequences, orientation, thermodynamic stability, and nucleic acid strand stoichiometry.’ (score: 0.0)\n",
            "Document #837: ‘Approximately ,000 various G­ quadruplex ligands exist in which they are able to interact via their aromatic rings; this allows the small-molecule ligands to stack on the planar terminal tetrads within the G-quadruplex regions.’ (score: 0.0)\n",
            "Document #836: ‘These can be used to target G-quadruplex regions that cause neurological disorders.’ (score: 0.0)\n",
            "Document #835: ‘Decoys have been successfully demonstrated to inhibit oncogenic KRAS in SCIO mice leading to reduced tumour growth and increased median survival time. Another commonly used technique is the utilization of small-molecule ligands.’ (score: 0.0)\n",
            "Document #834: ‘When introduced to cancer cells the decoy can intercept associated transcription factors and bind them leading to the regulation of gene expression.’ (score: 0.0)\n",
            "Document #833: ‘These decoys are typically composed of a G-rich sequence that can form a stable G-quadruplex structure and a short linker region that can be modified to optimize their properties.’ (score: 0.0)\n",
            "Document #832: ‘The strategy involves designing synthetic oligonucleotides that mimic the G-quadruplex structure and compete with the endogenous G-quadruplexes for binding to transcription factors.’ (score: 0.0)\n",
            "Document #831: ‘Antisense oligonucleotides (ASOs) are commonly used to target Corf RNA of the G-quadruplex GGGGCC expansion repeat region, which has lowered the toxicity in cellular models of Corf.0sJ00 ASOs have previously been used to restore normal phenotypes in other neurological diseases that have gain-of-function mechanisms, the only difference is that it was used in the absence of G­ quadruplex expansion repeat regions.0J0JoJThe G-quadruplex decoy strategy is another promising approach for targeting cancer cells by exploiting the unique structural features of the G-quadruplex.’ (score: 0.0)\n",
            "Document #830: ‘Therefore, these techniques are especially advantageous for targeting neurological diseases that have a gain-of-function mechanism, which is when the altered gene product has a new function or new expression of a gene; this has been detected in the Corf (chromosome  open reading frame ).0Antisense therapy is the process by which synthesized strands of nucleic acids are used to bind directly and specifically to the mRNA produced by a certain gene, which will inactivate it.’ (score: 0.0)\n",
            "Document #829: ‘This repeat expansion promotes DNA methylation and other epigenetic heterochromatin modifications of FMR that prevent the transcription of the gene, leading to pathological low levels of FMRP.0J0Antisense-mediated interventions and small-molecule ligands are common strategies used to target neurological diseases linked to G-quadruplex expansion repeats.’ (score: 0.0)\n",
            "Document #828: ‘ FMRP acts as a negative regulator of translation, and its binding stabilizes G-quadruplex structures in mRNA transcripts, inhibiting ribosome elongation of mRNA in the neuron's dendrite and controlling the timing of the transcript's expression.0o Mutations of this gene can cause the development of Fragile X Syndrome, autism, and other neurological disorders.0 Specifically, Fragile X Syndrome is caused by an increase from 0 to over 00 CGG repeats within exon  of the FMR gene.’ (score: 0.0)\n",
            "Document #827: ‘Fragile X mental retardation protein (FMRP) is a widely expressed protein coded by the FMR gene that binds to G-quadruplex secondary structures in neurons and is involved in synaptic plasticity.’ (score: 0.0)\n",
            "Document #826: ‘Nucleolin is involved in the synthesis and maturation of ribosomes within the nucleus, and separation of nucleolin by the mutated RNA transcripts impairs nucleolar function and ribosomal RNA synthesis.’ (score: 0.0)\n",
            "Document #825: ‘Normal individuals typically have around  to  GC repeats, but individuals with FTD or ALS have from 00 to several thousand GC repeats_The transcribed RNA of these repeats have been shown to form stable G-quadruplexes, with evidence showing that the GC repeats in DNA have the ability to form mixed parallel-antiparallel G- quadruplex structures as well_ These RNA transcripts containing GC repeats were shown to bind and separate a wide variety of proteins, including nucleolin.’ (score: 0.0)\n",
            "Document #824: ‘These two diseases have a causal relationship to GGGGCC (GC) repeats within the st intron of Corf gene.’ (score: 0.0)\n",
            "Document #823: ‘The second mechanism is through mutations that affect the expression of G-quadruplex binding proteins, as seen in the fragile X mental retardation gene  (FMR) gene and Fragile X Syndrome. The Corf gene codes for the protein Corf which is found throughout the brain in neuronal cytoplasm and at presynaptic terminals. Mutations of the Corf gene have been linked to the development of FTD and ALS.’ (score: 0.0)\n",
            "Document #822: ‘The first is through expansions of G-repeats within genes that lead to the formation of G-quadruplex structures that directly cause disease, as is the case with the Corf gene and amyotrophic lateral sclerosis (ALS) or frontotemporal dementia (FTD).’ (score: 0.0)\n",
            "Document #821: ‘G-quadruplexes have been implicated in neurological disorders through two main mechanisms.’ (score: 0.0)\n",
            "Document #820: ‘The novel approach combines solid-state nanopores and DNA nanotechnology for label-free detection of G-quadruplexes, for their mapping on dsDNA, and for monitoring G-quadruplex formation.’ (score: 0.0)\n",
            "Document #819: ‘Firstly, it was shown that biological nanopores can detect G-quadruplexes based on size exclusion and specific interaction of G-quadruplex and protein nanocavity.’ (score: 0.0)\n",
            "Document #818: ‘Another approach for detection of G-quadruplexes includes nanopore-based methods.’ (score: 0.0)\n",
            "Document #817: ‘Likewise, the thermostability of the G-quadruplex structure can be identified by observing the UV signal at  nm_ Upon G-quadruplex melting, the UV absorbance at  nm decreases, leading to a hypochromic shift that is a distinctive feature of G-quadruplex structure.’ (score: 0.0)\n",
            "Document #816: ‘To verify G-quadruplex formation, one should also perform the CD experiments under non-G-quadruplex stabilizing (Li+) and G-quadruplex stabilizing conditions (such as K+ or with G-quadruplex ligands), and scan toward the far-UV region (0- 0 nm).’ (score: 0.0)\n",
            "Document #839: ‘As of now, no single small-molecule ligand has been able to be perfectly specific for a single G-quadruplex sequence.  However, a cationic porphyrin known as TMPyP is able to bind to the Corf GGGGCC repeat region, which causes the G-quadruplex repeat region to unfold and lose its interactions with proteins causing it to lose its functionality.’ (score: 0.0)\n",
            "Document #841: ‘This provides evidence that small-molecule ligands are an effective and efficient process to target GGGGCC regions, and that specificity for small-molecule ligand binding is a feasible goal for the scientific community.’ (score: 0.0)\n",
            "Document #814: ‘The topology of the G-quadruplex structure can be determined by monitoring the positive or negative circular dichroism (CD) signals at specific wavelengths.’ (score: 0.0)\n",
            "Document #842: ‘Metal complexes have a number of features that make them particularly suitable as G DNA binders and therefore as potential drugs.’ (score: 0.0)\n",
            "Document #865: ‘These similarities have medical applications in terms of studying the structure and functions of both immune proteins and stress proteins, as well as the role each plays in combating disease.’ (score: 0.0)\n",
            "Document #864: ‘In addition, many stress proteins overlap with immune proteins.’ (score: 0.0)\n",
            "Document #863: ‘Early research has suggested that cells which are better able to synthesize stress proteins and do so at the appropriate time are better able to withstand damage caused by ischemia and reperfusion.’ (score: 0.0)\n",
            "Document #862: ‘Other toxins that either directly or indirectly lead to the release of free radicals can generate misfolded proteins.’ (score: 0.0)\n",
            "Document #861: ‘For example, many heavy metals can react with sulthydryl groups stabilizing proteins,  resulting  in  conformational changes.’ (score: 0.0)\n",
            "Document #860: ‘Many toxins end up activating similar stress proteins to heat or other stress-induced pathways because it is fairly common for some types of toxins to achieve their effects - at least in part - by denaturing vital cellular proteins.’ (score: 0.0)\n",
            "Document #859: ‘Heat shock proteins can also participate in marking malformed proteins for degradation via ubiquitin tags.’ (score: 0.0)\n",
            "Document #858: ‘This role is especially crucial since elevated temperature would, on its own, increase the concentrations of malformed proteins.’ (score: 0.0)\n",
            "Document #857: ‘These can help defend a cell against damage by acting as 'chaperons' in protein folding, ensuring that proteins assume their necessary shape and do not become denatured.’ (score: 0.0)\n",
            "Document #856: The heat shock response involves a class of stress proteins called heat shock proteins.’ (score: 0.0)\n",
            "Document #855: ‘Indeed, if a cell under normal (meaning unstressed) conditions has denatured proteins artificially injected into it, it will trigger a stress response.’ (score: 0.0)\n",
            "Document #854: ‘Because conditions such as high temperatures often cause proteins to denature, this mechanism enables cells to determine when they are subject to high temperature without the need of specialized thermosensitive proteins.’ (score: 0.0)\n",
            "Document #853: ‘Although stress response pathways are mediated in different ways depending on the stressor involved, cell type, etc., a general characteristic of many pathways - especially ones where heat is the principal stressor - is that they are initiated by the presence and detection of denatured proteins.’ (score: 0.0)\n",
            "Document #852: ‘As expected, such cells were also highly vulnerable to stress, and ceased to be viable at elevated temperature ranges.’ (score: 0.0)\n",
            "Document #851: ‘For example, studies in Drosophila have indicated that when DNA encoding certain stress proteins exhibit mutation defects, the resulting cells have impaired or lost abilities such as normal mitotic division and proteasome-mediated protein degradation.’ (score: 0.0)\n",
            "Document #850: ‘Stress proteins can exhibit widely varied functions within a cell- both during normal life processes and in response to stress.’ (score: 0.0)\n",
            "Document #849: ‘The essential character of these stress proteins in promoting the survival of cells has contributed to them being remarkably well conserved across phyla, with nearly identical stress proteins being expressed in the simplest prokaryotic cells as well as the most complex eukaryotic ones.’ (score: 0.0)\n",
            "Document #848: ‘Stress proteins often are further subdivided into two general categories: those that only are activated by stress, or those that are involved both in stress responses and in normal cellular functioning.’ (score: 0.0)\n",
            "Document #847: ‘Cellular stress responses are primarily mediated through what are classified as stress proteins.’ (score: 0.0)\n",
            "Document #846: ‘The various processes involved in cellular stress responses serve the adaptive purpose of protecting a cell against unfavorable environmental conditions, both through short term mechanisms that minimize acute damage to the cell's overall integrity, and through longer term mechanisms which provide the cell a measure of resiliency against similar adverse conditions.’ (score: 0.0)\n",
            "Document #845: ‘Cellular stress responses can also be caused by some viral infections.’ (score: 0.0)\n",
            "Document #844: ‘Cellular stress response is the wide range of molecular changes that cells undergo in response to environmental stressors, including extremes of temperature, exposure to toxins, and mechanical damage.’ (score: 0.0)\n",
            "Document #843: ‘While the metal plays largely a structural role in most G binders, there are also examples where it interacts directly with Gs by electrostatic interactions or direct coordination with nucleobases. (score: 0.0)\n",
            "Document #815: ‘Parallel G-quadruplexes have negative and positive CD signals at 0 and  nm, respectively, whereas antiparallel G-quadruplexes place these signals at  and  nm, respectively.’ (score: 0.0)\n",
            "Document #813: ‘In the DNA polymerase stop assay, the formation of a G-quadruplex in a DNA template can act as a roadblock and cause polymerase stalling, which halts the primer extension.  The dimethyl sulfate (DMS) followed by the piperidine cleavage assay is based on the fact that the formation of a G­ quadruplex will prohibit the N guanine methylation caused by DMS, leading to a protection pattern observed at the DNA G-quadruplex region after piperidine cleavage.’ (score: 0.0)\n",
            "Document #761: ‘With this product, the c-Myc protein functions in the processes of apoptosis and cell growth or development and as a transcriptional control on human telomerase reverse transcriptase.’ (score: 0.0)\n",
            "Document #786: ‘This main pathway of targeting this region results in the lack of telomerase elongation, leading to arrested cell development.’ (score: 0.0)\n",
            "Document #784: ‘To address this issue analog of TMPyP, it was synthesized known as Me which targets only G quadruplex DNA which inhibits cancer growth more effectively than TMPyP_ Ligand design and development remains an important field of research into therapeutic reagents due to the abundance of G-quadruplexes and their multiple conformational differences.’ (score: 0.0)\n",
            "Document #783: ‘However, TMPyP has been limited for used due to its non-selectivity toward cancer cell telomeres and normal double stranded DNA (dsDNA).’ (score: 0.0)\n",
            "Document #782: ‘These complexes have a high affinity for porphyrin rings which makes them effective anticancer agents.’ (score: 0.0)\n",
            "Document #781: ‘Telomeres are generally made up of G-quadruplexes and remain important targets for therapeutic research and discoveries.’ (score: 0.0)\n",
            "Document #780: ‘However, research has also identified the presence of G-quadruplex structures within this region due to the interaction of TMPyP with this promoter sequence.’ (score: 0.0)\n",
            "Document #779: ‘The presence of a guanine-rich sequence in the promoter region of PDGF-A has exhibited the ability to form intramolecular parallel G­ quadruplex structures and remains suggested to play a role in transcriptional regulation of PDGF-A.’ (score: 0.0)\n",
            "Document #778: ‘High levels of expression of PDGF have been associated with increased cell growth and cancer.’ (score: 0.0)\n",
            "Document #777: ‘Another oncogene pathway involving PDGF-A, platelet-derived growth factor, involves the process of wound healing and function as mitogenic growth factors for cells.’ (score: 0.0)\n",
            "Document #776: ‘The research on this pathway suggested the formation of a G­ quadruplex in the promoter region and an applicable target for therapeutic treatments.’ (score: 0.0)\n",
            "Document #775: ‘In certain types of cancers, the RET protein has shown increased expression levels.’ (score: 0.0)\n",
            "Document #774: ‘The guanine rich sequence in the promoter region for this pathway exudes a necessity for baseline transcription of this receptor tyrosine kinase.’ (score: 0.0)\n",
            "Document #773: ‘ The RET oncogene functions in the transcription of kinase which has been abundant in certain types of cancer.’ (score: 0.0)\n",
            "Document #772: ‘Current research on this pathway is focusing on discovering the biological function of this specific quadruplex formation on the c-kit pathway, while this quadruplex sequence has been noticed in various species.’ (score: 0.0)\n",
            "Document #771: ‘The rich guanine sequence of this promoter region has shown the ability to form a variety of quadruplexes.’ (score: 0.0)\n",
            "Document #770: ‘However, more research is necessary to determine whether the formation of G-quadruplex regulates the expression of this gene in a positive or negative manner.i The c-kit oncogene deals with a pathway that encodes an RTK, which was shown to have elevated expression levels in certain types of cancer.’ (score: 0.0)\n",
            "Document #769: ‘Through recent research into this specific gene pathway, the polypurine and polypyrimidine region allows for the transcription of this specific gene and the formation of an intramolecular G-quadruplex structure.’ (score: 0.0)\n",
            "Document #768: ‘Interaction of this G-quadruplex with TRF (also known as TERF) resulted in epigenetic regulation of p, which was tested using the G-quadruplex-binding ligand A.oHypoxia inducible factor la, HIF-a, remains involved in cancer signaling through its binding to Hypoxia Response Element, HRE, in the presence of hypoxia to begin the process of angiogenesis.’ (score: 0.0)\n",
            "Document #767: ‘The cyclin-dependent cell cycle checkpoint kinase inhibitor- CDKNlA (also known as p) gene harbours promoter G-quadruplex.’ (score: 0.0)\n",
            "Document #766: ‘The intramolecular G-quadruplex structures are formed mostly through the abundant guanine sequence in the promoter region of this specific pathway.’ (score: 0.0)\n",
            "Document #765: ‘Through recent research on the role of G-quadruplex function in vivo, the stabilization of G-quadruplex structures was shown to regulate VEGF gene transcription, with inhibition of transcription factors in this pathway.’ (score: 0.0)\n",
            "Document #764: ‘The formation of an intramolecular G-quadruplex structure has been shown through studies on the polypurine tract of the promoter region of the VEGF gene.’ (score: 0.0)\n",
            "Document #763: ‘Another gene pathway deals with the VEGF gene, Vascular Endothelial Growth Factor, which remains involved in the process of angiogenesis or the formation of new blood vessels.’ (score: 0.0)\n",
            "Document #762: ‘Interaction of c-Myc promoter G-quadruplex with NMH was shown to regulate c-Myc in cancer cells in 00 Regulation of c-myc through Human telomerase reverse transcriptase (hTERT) is also directly regulated through promoter G-quadruplex by interaction with the transcription factor NMH where epigenetic modifications were dependent on NMH-G-quadruplex association.Recently, hTERT epigenetic regulation reported to be mediated through interaction of hTERT promoter G­ quadruplex with the telomeric factor TRF.’ (score: 0.0)\n",
            "Document #785: ‘One type ofligand involving a Quindoline derivative, SYUIQ-0, utilizes the stabilization of G-quadruplexes in promoter regions to inhibit the production of both the c-Myc protein product and the human telomerase reverse transcriptase (hTERT).’ (score: 0.0)\n",
            "Document #787: ‘Further research remains necessary for the discovery of a single gene target to minimize unwanted reactivity with more efficient antitumor activity.’ (score: 0.0)\n",
            "Document #812: ‘These methods can be broadly defined into two classes: biophysical and biochemical methods.  Biochemical techniques were employed to interrogate G-quadruplex formation in a longer sequence context.’ (score: 0.0)\n",
            "Document #788: ‘One way of inducing or stabilizing G-quadruplex formation is to introduce a molecule which can bind to the G-quadruplex structure.’ (score: 0.0)\n",
            "Document #811: ‘A number of experimental methods have been developed to identify G-quadruplexes.’ (score: 0.0)\n",
            "Document #810: ‘This suggests that the sequences may be under positive selection enabled by the evolution of systems capable of suppressing non-B structure formation.’ (score: 0.0)\n",
            "Document #809: ‘the frequency) of these motifs has increased rapidly in the eumetazoa for which complete genomic sequences are available.’ (score: 0.0)\n",
            "Document #808: ‘In one study, it was found that the observed number per base pair (i.e.’ (score: 0.0)\n",
            "Document #807: ‘Although the rule effectively identifies sites of G-quadruplex formation, it also identifies a subset of the imperfect homopurine mirror repeats capable of triplex formationand C-strand i-motif formation.0 Moreover, these sequences also have the capacity to form slipped and foldback structures that are implicit intermediates in the formation of both quadruplex and triplex DNA structures.’ (score: 0.0)\n",
            "Document #806: ‘This rule has been widely used in on-line algorithms.’ (score: 0.0)\n",
            "Document #805: ‘Generally, a simple pattern match is used for searching for possible intrastrand quadruplex forming sequences: d(G+N_G+N_G+N_G+), where N is any nucleotide base (including guanine).’ (score: 0.0)\n",
            "Document #804: ‘Identifying and predicting sequences which have the capacity to form quadruplexes is an important tool in further understanding their role.’ (score: 0.0)\n",
            "Document #803: ‘Ligands with mobile side chains, ones that are able to rotate around its center chromophore, associate more strongly to G-quadruplexes because conformation of the G loops and the ligand side chains can align.’ (score: 0.0)\n",
            "Document #802: ‘Furthermore, the hydrogen bonds of ligands with smaller side chains are shorter and therefore stronger.’ (score: 0.0)\n",
            "Document #801: ‘It has been found that ligands with smaller side chains bind better to the quadruplex because smaller ligands have more concentrated electron density.’ (score: 0.0)\n",
            "Document #800: ‘The way in which TMPyP binds to G's is similar to MM, with the ring stacking onto the external G-quartet and side chains associating to the loops of G's.? When designing ligands to be bound to G-quadruplexes, the ligands have a higher affinity for parallel folded G-quadruplexes.’ (score: 0.0)\n",
            "Document #799: ‘TMPyP, a cationic porphyrin, is a more well known G binding ligand that helps to repress c-Myc.’ (score: 0.0)\n",
            "Document #798: ‘What makes this binding strong is the fluidity in the position of the loops to better associate with the ligand side chains.’ (score: 0.0)\n",
            "Document #797: ‘The quartet and the chromophore are bound with a :rc-:rc bond while the side chains and loops are not bound but are in close proximity.’ (score: 0.0)\n",
            "Document #796: ‘When bound, MM's central chromophore is situated on top of the’ terminal G-quartet and the side chains of the ligand associate to the loops of the quadruplex.’ (score: 0.0)\n",
            "Document #795: ‘The shape of the ligand is vital because it closely matches the quadruplex which has stacked quartets and the loops of nucleic acids holding it together.’ (score: 0.0)\n",
            "Document #794: ‘MM, a ligand that binds selectively for a quadruplex on the BCL- promoter, is shaped with a central core and  side chains branching sterically out.’ (score: 0.0)\n",
            "Document #793: ‘The binding of ligands to G-quadruplexes is vital for anti-cancer pursuits because G-quadruplexes are found typically at translocation hot spots.’ (score: 0.0)\n",
            "Document #792: ‘Cationic porphyrins have been shown to bind intercalatively with G-quadruplexes, as well as the molecule telomestatin.’ (score: 0.0)\n",
            "Document #791: ‘This has become an increasingly large field of research in genetics, biochemistry, and pharmacology.’ (score: 0.0)\n",
            "Document #790: ‘These ligands can be naturally occurring or synthetic.’ (score: 0.0)\n",
            "Document #789: ‘A number of ligands, which can be both small molecules and proteins, can bind to the G-quadruplex.’ (score: 0.0)\n",
            "Document #652: virus group Satellite nucleic acids, Single-stranded satellite DNAs, Family Alphasatellitidae (encoding a replication initiator protein) Family Tolecusatellitidae (encoding a pathogenicity determinant βC1)and Double-stranded satellite RNAs Single-stranded satellite RNAs.' (score: 0.0)\n",
            "Document #650: 'Satellite viruses include the Single-stranded RNA satellite viruses (unnamed family), Aumaivirus – Maize white line mosaic satellite.' (score: 0.0)\n",
            "Document #435: ‘Conversely, the cytoskeleton seems unnecessary in virus-induced atypical SGs.’ (score: 0.0)\n",
            "Document #502: ‘Arrows: SG-positive cells.’ (score: 0.0)\n",
            "Document #513: ‘This complex pre¬vents N protein from self-assembling and, thus, favors nucleocapsid assembly and viral transcription (26).’ (score: 0.0)\n",
            "Document #512: ‘Likewise, Sendai virus PP forms complexes with unassembled N protein (N°).’ (score: 0.0)\n",
            "Document #511: ‘However, this binding is blocked when the N-protein is complexed with PP (NS) (50).’ (score: 0.0)\n",
            "Document #510: ‘The N-protein of VSV seems to bind to any available RNA species, irrespective of sequence.’ (score: 0.0)\n",
            "Document #509: ‘In paramyxoviruses, such as mumps, N-protein en¬capsulates full-length viral negative sense genomic and positive sense antigenomic RNAs to form the helical nucleocapsid template (25).’ (score: 0.0)\n",
            "Document #508: ‘In the course of RSV infection, viral inclusion bodies, which are marked by anti-N and -P anti-bodies, are spatially separated from SGs (22).’ (score: 0.0)\n",
            "Document #507: ‘In our study, NP partially localized in SGs, whereas almost no colocalization of PP and TIA-1 was observed, which is consistent with previous re¬ports.’ (score: 0.0)\n",
            "Document #506: ‘The line of evidence suggests that viral replication, but not viral protein, is essential for v-SGs.’ (score: 0.0)\n",
            "Document #505: ‘not trigger SG formation.’ (score: 0.0)\n",
            "Document #504: ‘Right: Intensity pro.le of linear region of interest (ROI) across the HeLa cell costained with SG markers and mRNA.’ (score: 0.0)\n",
            "Document #503: ‘Arrowheads: SG-negative cells.’ (score: 0.0)\n",
            "Document #501: ‘Anti-TIA-1 (A, B)and anti-TIAR(C, D) were used as an SG marker.’ (score: 0.0)\n",
            "Document #649: 'Satellite-like nucleic acids resemble satellite nucleic acids, in that they replicate with the aid of helper viruses. However they differ in that they can encode functions that can contribute to the success of their helper viruses; while they are sometimes considered to be genomic elements of their helper viruses, they are not always found within their helper viruses.' (score: 0.0)\n",
            "Document #500: ‘Probes speci.cally hybridized to polyadenylated mRNA (green) (A, C ), NP mRNA (red) (B, D)were used to testspeci.c components in SGs.’ (score: 0.0)\n",
            "Document #499: ‘At the indicated time points, cells were .xed and processed for FISH.’ (score: 0.0)\n",
            "Document #498: ‘HeLa cells were mock treated or infected with NDV at an MOI of 1 for 6, 12, and 18 h.’ (score: 0.0)\n",
            "Document #497: ‘Cellular, but not viral mRNA, is predominately recruited in NDV-induced SGs.’ (score: 0.0)\n",
            "Document #496: ‘Both NDV and RSV induce SG formation, and encode no C protein, suggesting that the C protein is a key player in determining the fate of SGs in Paramyxovirus infections.’ (score: 0.0)\n",
            "Document #495: ‘Our data demonstrate that NDV induced stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #494: ‘In respiratory syncytial virus (RSV), another member of the Paramyxoviridae family, SGs are triggered at 12 hpi and increase throughout the 24 h period of infection (22).’ (score: 0.0)\n",
            "Document #493: ‘It is suggested that MV and SeV encode C protein (46, 47), which limits the generation of intracellular double-stranded RNA and activation of PKR to inhibit eIF2a phosphoryla¬tion (48, 49).’ (score: 0.0)\n",
            "Document #492: ‘Measles virus (MV) and Sendai virus (SeV), 2 members of the Paramyxoviridae family, however, do not induce SGs during their infection.’ (score: 0.0)\n",
            "Document #491: To date, several viruses are known to modulate SG formation to promote their replication.’ (score: 0.0)\n",
            "Document #490: ‘Viral (score: 0.0)\n",
            "Document #514: ‘Besides, a recent report showed that NDV positive-but not negative-stranded RNA is located in SGs (51).’ (score: 0.0)\n",
            "Document #515: ‘In line with that, in our study, viral negative-stranded gRNA was not recruited to SGs.’ (score: 0.0)\n",
            "Document #516: ‘Therefore, we specu¬late that a fraction of the NDV NP was recruited to v-SGs through binding to cellular or viral positive-stranded RNAs, and additional unassembled NP° may form complexes with the PP and remains excluded from v-SGs.’ (score: 0.0)\n",
            "Document #517: ‘However, this hypothesis needs to be investi¬gated further.’ (score: 0.0)\n",
            "Document #540: 2) blockage of SGs using siTIA-1 inhibited viral protein synthesis and extracellular virus yields; 3)NDV-induced SGs predominantly recruited cellular mRNA, whereas large amounts of NDV mRNA were excluded; and 4)de¬pletion of endogenous TIA-1/R and TIAR reduced NDV mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #539: ‘Overall, we found that: 1) NDV induced SGs to inhibit global protein translation through eIF2a phosphorylation; (score: 0.0)\n",
            "Document #538: ‘TIA-1/R knockdown led to a decrease in viral protein synthesis and extracellular virus yields, especially at the middle stage of infection.’ (score: 0.0)\n",
            "Document #537: ‘Similarly, in our study, NDV triggers stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #536: ‘In these cases, the depletion of SG key compo¬nents, such as TIA-1, TIAR, or G3BP, results in decreased virus replication, indicating the proviral function of SG proteins.’ (score: 0.0)\n",
            "Document #535: ‘The symbiotic relationship between SG formation and virus infection suggests that SG is a beneficial factor rather than an intrinsic defense mechanism during infections of some viruses.’ (score: 0.0)\n",
            "Document #534: ‘In other words, the formation of SGs is accompanied by virus infection (18, 22).’ (score: 0.0)\n",
            "Document #533: ‘In contrast to this, HCV and RSV trigger stable formation of bona fide SGs.’ (score: 0.0)\n",
            "Document #532: ‘Most viruses exploit several mechanisms to antagonize SG formation, such as G3BP cleavage and PKR inactivation (15, 16, 47).’ (score: 0.0)\n",
            "Document #531: Correspondingly, viruses use several mechanisms to evade or even take advantage of SGs.’ (score: 0.0)\n",
            "Document #530: ‘An earlier study showed that disruption of microtubules (score: 0.0)\n",
            "Document #529: ‘The function of microtubules in SG formation under physical and chemical stimuli varies in different reports.’ (score: 0.0)\n",
            "Document #528: ‘Furthermore, PKR/eIF2a pathwayplayedakeyroleinNDV-induced SGs.’ (score: 0.0)\n",
            "Document #527: We confirmed the association between the PKR/eIF2a pathway and v-SG formation, indicating that NDV in¬fection activated PKR/eIF2a pathway significantly and inhibited global translation.’ (score: 0.0)\n",
            "Document #526: ‘In this report, (score: 0.0)\n",
            "Document #525: ‘Viral cytoplasmic bodies induced by SV5 infection do not colocalize with cellular markers for SGs.’ (score: 0.0)\n",
            "Document #524: ‘Paramyxovirus simian virus 5 (SV5) is a weak ac¬tivator of the PKR/eIF2a signaling pathway and thereby does not globally inhibit translation (52).’ (score: 0.0)\n",
            "Document #523: ‘Moreover, TIA-1 was colocalized with G3BP1, rPS6, eIF4E, eIF4G, or eIF3A in both ARS-treated cells and NDV-infected cells, indicating that NDV triggers bona fide SGs.’ (score: 0.0)\n",
            "Document #522: ‘In our study, CHX treat¬ment caused the disassembly of NDV-induced SGs.’ (score: 0.0)\n",
            "Document #521: ‘At the late stage of poliovirus infection, virus-induced SGs contain TIA but lack translation initiation factors, mRNA binding proteins, and most polyadenylated mRNA (17).’ (score: 0.0)\n",
            "Document #520: ‘VSV-induced pseudo-SGs lack eIF3 and eIF4A (37).’ (score: 0.0)\n",
            "Document #519: ‘An E3 mutant vaccinia virus induces AVGs, which are atypical SGs that neither contain the 40S ribosomal subunit, nor disassemble when treated with polysome-stabilizing drugs such as CHX and emetine (20).’ (score: 0.0)\n",
            "Document #518: ‘The cytoplasmic granules induced by viral infec¬tions are distinct from typical SGs in both composition and function.’ (score: 0.0)\n",
            "Document #489: ‘FISH was used to detect NDV negative-strand genome RNA.’ (score: 0.0)\n",
            "Document #488: Because v-SGs regulate protein translation in the course of virus infection, we asked whether SGs play a role in virus genome replication.’ (score: 0.0)\n",
            "Document #487: ‘Depletion of endogenous TIA-1 does not affect virus genome replication (score: 0.0)\n",
            "Document #460: ‘Poliovirus-induced SGs are also stable over the course of infection (44).’ (score: 0.0)\n",
            "Document #458: ‘7, sixth column).’ (score: 0.0)\n",
            "Document #457: ‘The over¬lapping intensity profiles of the SG markers and mRNA (as visualized in the line profiles in the fifth column) further demonstrate the colocalization pro¬file between cellular mRNA and SG markers, whereas most viral mRNA was not localized with SG markers (Fig.’ (score: 0.0)\n",
            "Document #456: ‘However, large amounts of NP mRNA were distributed throughout the cytoplasm and excluded from-SGs (Fig.7B, D).’ (score: 0.0)\n",
            "Document #455: ‘At 12 and 18 hpi, a small portion of NP mRNA was localized with TIA-1 or TIAR.’ (score: 0.0)\n",
            "Document #454: ‘Meanwhile, no obvious SGs were observed in NP mRNA–positive cells at 6 hpi.’ (score: 0.0)\n",
            "Document #453: ‘NP mRNA was first detected in some cells infected for 6 h with NDV, compared with no positive fluorescence in mock-treated cells.’ (score: 0.0)\n",
            "Document #452: ‘To verify whether NDV mRNA is a component of SGs, we examined infected cells with FISH probes that hybridize to viral NP mRNA.’ (score: 0.0)\n",
            "Document #451: ‘Because Poly (A) tail has also been found on the messenger RNAs of NDV (43), v-SGs may contain both viral and host mRNA.’ (score: 0.0)\n",
            "Document #450: ‘In contrast, almost all polyadenylated mRNA was recruited to v-SGs at 12 and 18 hpi (arrow).’ (score: 0.0)\n",
            "Document #449: ‘diffusely dispersed in the cytoplasm (arrowhead).’ (score: 0.0)\n",
            "Document #448: ‘Arrows: transfected cells.’ (score: 0.0)\n",
            "Document #447: ‘The cells were collected, .xed, and visualized at 12 hpi.’ (score: 0.0)\n",
            "Document #446: ‘Cells were pulsed with 1 mM puromycin for 1 h at 37°C and then subjected to Western blot analysis.B) HeLa cells were transfected with PJX40-Flag-WT eIF2a, S51A, and S51D mutant eIF2a, followed by treatment at 24 h after transfection with infection of NDV at an MOI of 1.’ (score: 0.0)\n",
            "Document #445: ‘It was noted that in uninfected cells or cells containing no SG dots, polyadenylated mRNA was Phosphorylation of eIF2a at Ser-51 is required for v-SGs.A) HeLa cells were mock treated or infected with NDV at an MOI of 1.’ (score: 0.0)\n",
            "Document #444: ‘Polyadenylated mRNA was specifically recruited to virus-induced SGs marked by both TIA-1 and TIAR.’ (score: 0.0)\n",
            "Document #443: ‘FISH was used to detect total cellular mRNA, as well as viral NP mRNA.’ (score: 0.0)\n",
            "Document #442: Having found that v-SGs benefit the replication of NDV by manipulating the translation system, we checked whether the SG formation and stalled trans¬lation initiation were predominantly directed against viral or host protein translation.’ (score: 0.0)\n",
            "Document #441: ‘NDV-induced SGs contain predominantly cellular mRNA but not viral mRNA (score: 0.0)\n",
            "Document #440: ‘Noc treatment induced the formation of small and circular granules, compared to the large and irregular SGs in NDV-infected cells.’ (score: 0.0)\n",
            "Document #439: ‘In cells treated with Noc, obvious depolymerization of microtubules was observed, but the formation of NDV-induced SGs was unaffected.’ (score: 0.0)\n",
            "Document #438: ‘Accordingly, to confirm whether NDV-induced SGs were canonical SGs, we evaluated the role of microtubules and microfilaments in the formation of v-SGs.’ (score: 0.0)\n",
            "Document #437: ‘One characteristic of SG-like structures is that they are not dependent on a cellular microtubule or microfilament network.’ (score: 0.0)\n",
            "Document #436: ‘Previous reports showed that vesicular stomatitis virus (VSV)-induced SG-like structures are distinct from canonical SGs (37).’ (score: 0.0)\n",
            "Document #459: ‘Similarly, SGs in poliovirus infected cells are largely devoid of positive-sense poliovirus RNA, but contain polyadenylated RNA.’ (score: 0.0)\n",
            "Document #461: ‘Overall, these results indicate that NDV-induced SGs predominantly recruit cellular mRNA, whereas a large portion of viral mRNA seem to escape from v-SGs.’ (score: 0.0)\n",
            "Document #486: ‘The results indicate that G3BP1 is also necessary for NDV replication by trans¬lation regulation.’ (score: 0.0)\n",
            "Document #462: ‘Depletion of endogenous TIA-1 and TIAR reduced NP mRNA levels in polysomes (score: 0.0)\n",
            "Document #485: ‘9A, C).’ (score: 0.0)\n",
            "Document #484: ‘Similar to TIA-1 and TIAR, siG3BP1 resulted in the significant up-regulation of global translation levels (Figs.’ (score: 0.0)\n",
            "Document #483: ‘We next monitored global protein synthesis after NDV infection.’ (score: 0.0)\n",
            "Document #482: ‘In G3BP1-depleted cells infected with NDV, the expression of the viral NP and virus yields were significantly lower than in cells transfected with scrambled siRNAs, especially at 12 hpi.’ (score: 0.0)\n",
            "Document #481: ‘HeLa Cells were transfected with 2 siRNAs specific to G3BP1 and then infected with NDV.’ (score: 0.0)\n",
            "Document #480: ‘As expected, SG formation was ef¬ficiently inhibited in siG3BP1-transfected cells upon treatment .’ (score: 0.0)\n",
            "Document #479: ‘To verify whether the translation regulation is driven by v-SGs or TIA-1/R per se, we knocked down G3BP1, another key SG component, and evaluated virus rep¬lication efficiency.’ (score: 0.0)\n",
            "Document #478: ‘Depletion of endogenous G3BP1, another key SG component, also reduced NDV replication on HeLa cells.’ (score: 0.0)\n",
            "Document #477: ‘These results demonstrate that depletion of endogenous TIA-1 and TIAR reduces NP mRNA levels in polysomes.’ (score: 0.0)\n",
            "Document #476: ‘8C).’ (score: 0.0)\n",
            "Document #475: ‘However, lower NP mRNA levels in TIA-1/TIAR-knockdown cells, compared with that in scrambled cells, was detected in fractions 5–12 (Fig.’ (score: 0.0)\n",
            "Document #474: ‘The results showed that, in fractions 1–4, the NP mRNA level in siTIA-1/TIAR-treated cells was similar to that in control cells.’ (score: 0.0)\n",
            "Document #473: ‘To verify whether knockdown of endogenous TIA-1/TIAR influences the viral mRNA profile in individual fractions, NDV NP mRNA was detected by absolute real-time PCR quantification.’ (score: 0.0)\n",
            "Document #472: ‘8B).’ (score: 0.0)\n",
            "Document #471: ‘Similar to the ribosome profile results, the amounts of PABP and rPS6 proteins showed no sig¬nificant differences between siRNA and control groups (Fig.’ (score: 0.0)\n",
            "Document #470: ‘TIA-1, TIAR, and PABP, as typical SG markers, were all found mainly in monosomes in virus infection.’ (score: 0.0)\n",
            "Document #469: ‘Western blot analysis of individual fractions for ri¬bosomal protein S6 showed that compared with mock infection, under virus infection, rPS6 was detected in fractions 1–8, suggesting that small ribosomal protein tends to shift to lower density fractions that contain monosomes.’ (score: 0.0)\n",
            "Document #468: ‘8A).’ (score: 0.0)\n",
            "Document #467: ‘The ab¬sorbance of individual fractions distinguished mono¬somes (fractions 1–4) from polysomes (fractions 5–12) (Fig.’ (score: 0.0)\n",
            "Document #466: ‘To elucidate the mechanisms how TIA-1/TIAR regulate virus in¬fection, sucrose gradient analysis was performed to sepa¬rate monosomes and polysomes before viral mRNA quantification in TIA-1/TIAR-knockdown cells.’ (score: 0.0)\n",
            "Document #465: ‘Because v-SGs contain predominantly cellular mRNA, then we asked whether disruption of SGs would lead to the release of cellular mRNA that competitively binds to polysomes and thereby reduces viral mRNA translation.’ (score: 0.0)\n",
            "Document #464: ‘In addition to mRNA, SGs per se are highly dynamic domains containing mRNA accompanied by dynamic shuttling of TIA-1 and are in equilibrium with polysomes (35).’ (score: 0.0)\n",
            "Document #463: In tomato cells exposed to heat shock, mRNA transfers from polysomes to SGs, whereas mRNA moves from SGs to polysomes when cells recover from heat shock (45).’ (score: 0.0)\n",
            "Document #541: ‘Therefore, we propose that host cells respond to NDV infection by SG formation.’ (score: 0.0)\n",
            "Document #542: ‘The degradation of SGs by siTIA-1 leads to the release of cel¬lular mRNA from SGs to the cytoplasm.’ (score: 0.0)\n",
            "Document #543: ‘The released host mRNA competitively binds to polysomes and thereby re¬duces the translation ratio of viral mRNA, ultimately leading to a decrease of viral translation level and extra¬cellular virus yields.’ (score: 0.0)\n",
            "Document #623: 'Unlike the system of binomial nomenclature adopted in cellular species, there is currently no standardized form for virus species names. At present, the ICTV mandates that a species name must contain as few words as possible while remaining distinct, and must not only contain the word virus and the host name.[9] Species names often take the form of [Disease] virus, particularly for higher plants and animals. In 2019, the ICTV published a proposal to adopt a more formalized system of binomial nomenclature for virus species names, to be voted on in 2020.[10] However, some virologists later objected to the potential naming system change, arguing that the debate came while many in the field were preoccupied due to the COVID-19 pandemic.[11].' (score: 0.0)\n",
            "Document #621: 'Viral classification starts at the level of realm and continues as follows, with the taxonomic suffixes in parentheses:[8]Realm (-viria), Subrealm (-vira), Kingdom (-virae),Subkingdom (-virites), Phylum (-viricota), Subphylum (-viricotina), Class (-viricetes),Subclass (-viricetidae), Order (-virales),.' (score: 0.0)\n",
            "Document #620: 'The International Committee on Taxonomy of Viruses began to devise and implement rules for the naming and classification of viruses early in the 1970s, an effort that continues to the present. The ICTV is the only body charged by the International Union of Microbiological Societies with the task of developing, refining, and maintaining a universal virus taxonomy.[7] The system shares many features with the classification system of cellular organisms, such as taxon structure. However, some differences exist, such as the universal use of italics for all taxonomic names, unlike in the International Code of Nomenclature for algae, fungi, and plants and International Code of Zoological Nomenclature.[8].' (score: 0.0)\n",
            "Document #619: Viruses sensu stricto are defined operationally by the ICTV as a type of MGE (mobile genetic element) that encodes at least one protein that is a major component of the virion encasing the nucleic acid of the respective MGE and therefore the gene encoding the major virion protein itself or MGEs that are clearly demonstrable to be members of a line of evolutionary descent of such major virion protein-encoding entities. Any monophyletic group of MGEs that originates from a virion protein-encoding ancestor should be classified as a group of viruses.' (score: 0.0)\n",
            "Document #618: 'The currently accepted and formal definition of a 'virus' was accepted by the ICTV Executive Committee in November 2020 and ratified in March 2021, and is as follows:[6].' (score: 0.0)\n",
            "Document #617: 'The actual criteria used vary by the taxon, and can be inconsistent (arbitrary similarity thresholds) or unrelated to lineage (geography) at times.[5] The matter is, for many, not yet settled.[2].' (score: 0.0)\n",
            "Document #616: 'In July 2013, the ICTV definition of species changed to state: \"A species is a monophyletic group of viruses whose properties can be distinguished from those of other species by multiple criteria.\"[3] These criteria include the structure of the capsid, the existence of an envelope, the gene expression program for its proteins, host range, pathogenicity, and most importantly genetic sequence similarity and phylogenetic relationship.[4].' (score: 0.0)\n",
            "Document #615: 'Species form the basis for any biological classification system. Before 1982, it was thought that viruses could not be made to fit Ernst Mayr's reproductive concept of species, and so were not amenable to such treatment. In 1982, the ICTV started to define a species as \"a cluster of strains\" with unique identifying qualities. In 1991, the more specific principle that a virus species is a polythetic class of viruses that constitutes a replicating lineage and occupies a particular ecological niche was adopted.[2].' (score: 0.0)\n",
            "Document #614: 'A catalogue of all the world's known viruses has been proposed and, in 2013, some preliminary efforts were underway.[1].' (score: 0.0)\n",
            "Document #613: Viruses are classified by phenotypic characteristics, such as morphology, nucleic acid type, mode of replication, host organisms, and the type of disease they cause. The formal taxonomic classification of viruses is the responsibility of the International Committee on Taxonomy of Viruses (ICTV) system, although the Baltimore classification system can be used to place viruses into one of seven groups based on their manner of mRNA synthesis. Specific naming conventions and further classification guidelines are set out by the ICTV.' (score: 0.0)\n",
            "Document #612: Virus classification is the process of naming viruses and placing them into a taxonomic system similar to the classification systems used for cellular organisms.' (score: 0.0)\n",
            "Document #611:  (score: 0.0)\n",
            "Document #610: ‘At the same time, we report the first evidence that VHS physically interacts with ICP27 both in lysates of infected cells and in transfected cells.’ (score: 0.0)\n",
            "Document #609: ‘Consistent with this finding is the evidence that VHS does not accumulate in ICP27 virus-infected cells and that the VHS brought into cells by ICP27 virions is less effective that that brought into cells by wild-type virus.’ (score: 0.0)\n",
            "Document #608: ‘Indeed, at late times, the IEX-1 is degraded in ICP27 virus-infected cells at a rate similar to that in noninfected or VHS virus-infected cells.’ (score: 0.0)\n",
            "Document #607: ‘In this report, we have reexamined the degradation of GAPDH and IEX-1 mRNAs at early and late times in infection.’ (score: 0.0)\n",
            "Document #606: ‘In the description of their studies, they confused the cleavage events occurring during the early hours after in¬fection when the mRNAs are cleaved and partially degraded in a VHS-but not a ICP27-dependent manner with events taking place at late time intervals.’ (score: 0.0)\n",
            "Document #605: ‘They concluded that ICP27 is required to stabilize full-length, polyadenylated IEX mRNA and that VHS is not involved in this process.’ (score: 0.0)\n",
            "Document #604: ‘The time frame of their studies was selected on the basis of the presence of abundant amounts of ICP27 in the infected cells.’ (score: 0.0)\n",
            "Document #603: ‘Corcoran and colleagues (6) analyzed the degradation of AU-rich IEX-1 mRNA at relatively late times after infection.’ (score: 0.0)\n",
            "Document #602: ‘A dearth or lack of enzymes necessary to degrade the 5 cleavage product containing an intact cap may account for the lingering of the cleavage prod¬uct for many hours.’ (score: 0.0)\n",
            "Document #601: ‘The 3 product of the cleavage would be rapidly degraded 5 to 3 in the same man¬ner as the GAPDH mRNA.’ (score: 0.0)\n",
            "Document #600: ‘In the presence of TTP bound to AREs, VHS is expected to bind TTP and cleave the mRNA at the AREs.’ (score: 0.0)\n",
            "Document #599: ‘This is the normal pathway of degradation of mRNA and would lead to 5-to-3 exonucleolytic degradation of the cleaved RNA.’ (score: 0.0)\n",
            "Document #622: Suborder (-virineae), Family (-viridae),Subfamily (-virinae), Genus (-virus),Subgenus (-virus), and Species.' (score: 0.0)\n",
            "Document #624: 'As of 2023, all levels of taxa except subrealm, subkingdom, and subclass are used. Six realms, one incertae sedis class, 22.' (score: 0.0)\n",
            "Document #597: ‘In brief, in the course of mRNA translation, it is the poly(A)-binding protein at the 3 end that physically abuts the cap-binding complex at the 5 end of the mRNA.’ (score: 0.0)\n",
            "Document #625: 'incertae sedis families, and two incertae sedis genera are recognized:[12].' (score: 0.0)\n",
            "Document #648: Satellites depend on co-infection of a host cell with a helper virus for productive multiplication. Their nucleic acids have substantially distinct nucleotide sequences from either their helper virus or host. When a satellite subviral agent encodes the coat protein in which it is encapsulated, it is then called a satellite virus.' (score: 0.0)\n",
            "Document #647: 1and Genus Elaviroid and subsequent type species of Eggplant latent viroid.' (score: 0.0)\n",
            "Document #646: 'species of Avocado sunblotch viroid, Family Pospiviroidae , Genus Pospiviroid and subsequent type species of Potato spindle tuber viroid, Genus Hostuviroid and subsequent type species of Hop stunt viroid, Genus Cocadviroid and subsequent type species of Coconut cadang-cadang viroid, Genus Apscaviroid and subequent type species of Apple scar skin viroid, Genus Coleviroid and subsequet type species of Coleus blumei viroid.' (score: 0.0)\n",
            "Document #645: 'The system was not accepted by others due to its neglect of morphological similarities.[20]Infectious agents are smaller than viruses and have only some of their properties.[21][22] Since 2015, the ICTV has allowed them to be classified in a similar way as viruses are. Viroids include the Family Avsunviroidae[24], Genus Avsunviroid and subsequent type.' (score: 0.0)\n",
            "Document #644: Holmes (1948) used a Linnaean taxonomy with binomial nomenclature to classify viruses into 3 groups under one order, Virales. Group I includes Phaginae  which helps sustain bacterial evolution for survival under abiotic stress such as drought. Group II includes Phytophaginae Phaginae  which infects plants in order to reduce biomass when there are extreme frost, drought or environmental threat conditions to survival. Group III Zoophaginae which is part of the same infectious species that would target animals in order to activate or modify genomic signaling properties for survival in changed environmental conditions that would otherwise threaten offspring survival. .' (score: 0.0)\n",
            "Document #643: Reoviridaevirus family, such as the Reovirus, rotavirushas a NakedcapsidIcosahedralcapsid symmetry, dsnucleic acid type and belongs to the Group IIIof viruses. Picornaviridaevirus family, such as the Enterovirus, rhinovirus, hepatovirus, cardiovirus, aphthovirus, poliovirus, parechovirus, erbovirus, kobuvirus, teschovirus, coxsackiehas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Caliciviridaevirus family, such as the Norwalk virushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Togaviridaevirus family, such as the Eastern equine encephalitishas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Arenaviridaevirus family, such as the Lymphocytic choriomeningitis virus, Lassa feverhas a EnvelopedcapsidComplexcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Flaviviridaevirus family, such as the Dengue virus, hepatitis C virus, yellow fever virus, Zika virushas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Orthomyxoviridaevirus family, such as the Influenzavirus A, influenzavirus B, influenzavirus C, isavirus, thogotovirushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Paramyxoviridaevirus family, such as the Measles virus, mumps virus, respiratory syncytial virus, Rinderpest virus, canine distemper virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Bunyaviridaevirus family, such as the California encephalitis virus, Sin nombre virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Rhabdoviridaevirus family, such as the Rabies virus, Vesicular stomatitishas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Filoviridaevirus family, such as the Ebola virus, Marburg virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Coronaviridaevirus family, such as the Human coronavirus 229E, Human coronavirus NL63, Human coronavirus OC43, Human coronavirus HKU1, Middle East respiratory syndrome-related coronavirus, Severe acute respiratory syndrome coronavirus, and Severe acute respiratory syndrome coronavirus 2has a EnvelopedcapsidHelicalcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Astroviridaevirus family, such as the Astrovirushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Bornaviridaevirus family, such as the Borna disease virushas a EnvelopedcapsidHelicalcapsid symmetry, ss(−)nucleic acid type and belongs to the Group Vof viruses. Arteriviridaevirus family, such as the Arterivirus, equine arteritis virushas a EnvelopedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. Hepeviridaevirus family, such as the Hepatitis E virushas a NakedcapsidIcosahedralcapsid symmetry, ssnucleic acid type and belongs to the Group IVof viruses. All viruses that encode a reverse transcriptase (also known as RT or RNA-dependent DNA polymerase) are members of the class Revtraviricetes, within the phylum Arterviricota, kingdom Pararnavirae, and realm Riboviria. The class Blubervirales contains the single family Hepadnaviridae of DNA RT (reverse transcribing) viruses; all other RT viruses are members of the class Ortervirales.[19] Group VI: viruses possess single-stranded RNA viruses that replicate through a DNA intermediate. The retroviruses are included in this group, of which HIV is a member. Group VII: viruses possess double-stranded DNA genomes and replicate using reverse transcriptase. The hepatitis B virus can be found in this group.' (score: 0.0)\n",
            "Document #642: All viruses that have an RNA genome, and that encode an RNA-dependent RNA polymerase (RdRp), are members of the kingdom Orthornavirae, within the realm Riboviria.[18] Group III viruses possess double-stranded RNA genomes, e.g. rotavirus. Group IV viruses possess positive-sense single-stranded RNA genomes. Many well known viruses are found in this group, including the picornaviruses (which is a family of viruses that includes well-known viruses like Hepatitis A virus, enteroviruses, rhinoviruses, poliovirus, and foot-and-mouth virus), SARS virus, hepatitis C virus, yellow fever virus, and rubella virus. Group V: viruses possess negative-sense single-stranded RNA genomes. Ebola and Marburg viruses are well known members of this group, along with influenza virus, measles, mumps and rabies.' (score: 0.0)\n",
            "Document #641: Herpesviridae are enveloped virion with icosahedral capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #640: Poxviridae are Complex  coats virion with complex coats capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #639: Parvoviridae are naked virion with icosahedral capsid symmetry and ss nucleic acid types and belong to Group II viruses.' (score: 0.0)\n",
            "Document #638: Papovaviridaeare naked virion with icosahedral capsid symmetry and dscircular  nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #637: Adenoviridae are naked virion with icosahedral capsid symmetry and ds nucleic acid types and belong to Group I viruses.' (score: 0.0)\n",
            "Document #636: Group II  viruses possess single-stranded DNA such as the pleolipoviridae (HHPV1, HRPV1). Group II viruses have an enveloped virion with ss nucleic acid type, ds nucleic acid type, and may be linear or cicular  The anelioviridiae are also group II viriuses that are not enveloped and called naked, and whose capsid symmetry is icosahedral and whose nucleic acid type is ss circular.' (score: 0.0)\n",
            "Document #635: Group I viruses possess double-stranded DNA. Viruses that cause chickenpox and herpes are found here.' (score: 0.0)\n",
            "Document #634: Viruses with a DNA genome, except for the DNA reverse transcribing viruses, are members of three of the four recognized viral realms: Duplodnaviria, Monodnaviria, and Varidnaviria. But the incertae sedis order Ligamenvirales, and many other incertae sedis families and genera, are also used to classify DNA viruses. The domains Duplodnaviria and Varidnaviria consist of double-stranded DNA viruses; other double-stranded DNA viruses are incertae sedis. The domain Monodnaviria consists of single-stranded DNA viruses that generally encode a HUH endonuclease; other single-stranded DNA viruses are incertae sedis.' (score: 0.0)\n",
            "Document #633: 'placed in one of the seven following groups:[16].' (score: 0.0)\n",
            "Document #632: 'Baltimore classification (first defined in 1971) is a classification system that places viruses into one of seven groups depending on a combination of their nucleic acid (DNA or RNA), strandedness (single-stranded or double-stranded), sense, and method of replication. Named after David Baltimore, a Nobel Prize-winning biologist, these groups are designated by Roman numerals. Other classifications are determined by the disease caused by the virus or its morphology, neither of which are satisfactory due to different viruses either causing the same disease or looking very similar. In addition, viral structures are often difficult to determine under the microscope. Classifying viruses according to their genome means that those in a given category will all behave in a similar fashion, offering some indication of how to proceed with further research. Viruses can be in I  dsDNA viruses (e.g. Adenoviruses, Herpesviruses, Poxviruses), II: ssDNA viruses (+ strand or \"sense\") DNA (e.g. Parvoviruses), III: dsRNA viruses (e.g. Reoviruses), IV:(+)ssRNA viruses (+ strand or sense) RNA (e.g. Coronaviruses, Picornaviruses, Togaviruses) V: (−)ssRNA viruses (− strand or antisense) RNA (e.g. Orthomyxoviruses, Rhabdoviruses), VI: ssRNA-RT viruses (+ strand or sense) RNA with DNA intermediate in life-cycle (e.g. Retroviruses) VII: dsDNA-RT viruses DNA with RNA intermediate in life-cycle (e.g. Hepadnaviruses), .' (score: 0.0)\n",
            "Document #631: 'The ICTV has gradually added many higher-level taxa using relationships in protein folds. All four realms defined in the 2019 release are defined by the presence of a protein of a certain structural family.[15].' (score: 0.0)\n",
            "Document #630: 'It has been suggested that similarity in virion assembly and structure observed for certain viral groups infecting hosts from different domains of life (e.g., bacterial tectiviruses and eukaryotic adenoviruses or prokaryotic Caudovirales and eukaryotic herpesviruses) reflects an evolutionary relationship between these viruses.[13] Therefore, structural relationship between viruses has been suggested to be used as a basis for defining higher-level taxa – structure-based viral lineages – that could complement the ICTV classification scheme of 2010.[14].' (score: 0.0)\n",
            "Document #629: 'Incertae sedis genera: Dinodnavirus, Rhizidiovirus.' (score: 0.0)\n",
            "Document #628: 'Itzamnaviridae Naldaviricetes Ovaliviridae Plasmaviridae Polydnaviriformidae Portogloboviridae Pospiviroidae Rhodogtaviriformidae Spiraviridae Thaspiviridae Tolecusatellitidae.' (score: 0.0)\n",
            "Document #627: 'Alphasatellitidae Ampullaviridae Anelloviridae Avsunviroidae Bartogtaviriformidae Bicaudaviridae Brachygtaviriformidae Clavaviridae Fuselloviridae Globuloviridae Guttaviridae Halspiviridae.' (score: 0.0)\n",
            "Document #626: 'Realms include: Duplodnaviria, Monodnaviria, Adnaviria, Ribozyviria, Riboviria, and Varidnaviria while Incertae sedis families and classes include:.' (score: 0.0)\n",
            "Document #598: ‘In the absence of AREs, VHS is predicted to bind the cap-binding complex and decap the mRNA.’ (score: 0.0)\n",
            "Document #596: ‘We proposed a model of degradation of intrinsi¬cally stable mRNAs (GAPDH) and of ARE mRNAs on the basis of known structures of translated mRNAs, known path¬ways of mRNA decay, and the interaction of VHS protein with the cap-binding complex and TTP.’ (score: 0.0)\n",
            "Document #544: ‘Semliki forest virus causes translation inhibition of host cellular proteins through eIF2a phosphorylation.’ (score: 0.0)\n",
            "Document #569: ‘A recent study reported that, at relatively late times after infection, ARE RNAs are rapidly degraded in cells infected with ICP27 mutant virus and concluded that ICP27 “stabilizes” ARE mRNAs.’ (score: 0.0)\n",
            "Document #567: ‘Stress response RNAs containing AU-rich elements (AREs) in the 3 untranslated region (3 UTR) are dead¬enylated and cleaved, but the cleavage products persist for hours, in contrast to the short half-lives of ARE-containing mRNAs in uninfected cells.’ (score: 0.0)\n",
            "Document #566: ‘Pre-existing stable mRNAs (e.g., GAPDH [glyceraldehyde-3-phosphate dehydrogenase]) are rapidly degraded.’ (score: 0.0)\n",
            "Document #565: ‘The virion host shutoff (VHS) RNase tegument protein released into cells by infecting virus has two effects.’ (score: 0.0)\n",
            "Document #564: ‘Understanding how NDV manipulates stress re¬sponses will lay the groundwork for developing more ef¬fective oncolytic viruses and novel anticancer strategies.’ (score: 0.0)\n",
            "Document #563: ‘However, the hypothesis should be investigated further.’ (score: 0.0)\n",
            "Document #562: ‘That NDV efficiently and selectively replicates in and kills tumor and chicken cells, but not in other normal mammalian cells leads us to spec¬ulate whether the cell stress responses, such as autophagy and SGs, contribute to the tumor-selective replication of NDV.’ (score: 0.0)\n",
            "Document #561: ‘The relationship between autophagy and NDV has been identified in lung cancer cells and human glioma cells (57–59).’ (score: 0.0)\n",
            "Document #560: ‘We also found that NDV infection in¬duced cytoplasmic SG dots in chicken fibroblast cells DF-1 and primary chicken embryo fibroblast cells, and ectopic expression of GFP tagged chicken G3BP1 increased NDV replication (unpublished results).’ (score: 0.0)\n",
            "Document #559: ‘TIA-1 and TIAR regulated NDV replication by manipulating cellular protein translation on HeLa cells, where NDV selectively replicates (55, 56).’ (score: 0.0)\n",
            "Document #558: ‘In our current study, NDV triggered stable formation of bona fide SGs through the PKR/eIF2a pathway.’ (score: 0.0)\n",
            "Document #557: ‘In our study, knockdown of SG key components TIA-1, TIAR, or G3BP1 reduced NDV protein expression and extracellular virus yields, demonstrating that SGs play a proviral role in NDV infection.’ (score: 0.0)\n",
            "Document #556: ‘It is possible that both pro-and antiviral effects of SGs are at play and that the virus has developed mechanisms to overcome the antiviral effects of SGs.’ (score: 0.0)\n",
            "Document #555: ‘However, they may only play a marginal role in NDV replication, which was confirmed by the little or some¬what negative effect of siG3BPs on viral N and L protein expression (51).’ (score: 0.0)\n",
            "Document #554: ‘In tumor cells, SGs play an important role in recognizing uncapped viral RNA and facilitating IFN-b production.’ (score: 0.0)\n",
            "Document #553: ‘NDV infections induce IFN-b production in both normal and tumor cells (24, 55); however, pretreatment with endogenous or exogenous IFN-b has little or no effect on reducing the NDV yield or replication in tumor cells, although IFN-b treatment completely inhibits NDV replication in normal cells (55).’ (score: 0.0)\n",
            "Document #552: ‘A recent publication refuted the claim that NDV-induced SGs enhance IFN induction on HeLa cells; however, no virus replication data were shown (51).’ (score: 0.0)\n",
            "Document #551: ‘Similarly, in our study, siG3BP1 inhibited viral protein expression and extracellular virus yields, and meanwhile led to the increase in global protein trans¬lation, indicating that the translation balance is mediated by v-SGs rather than TIA-1/R per se.’ (score: 0.0)\n",
            "Document #550: ‘RSV replication is impaired in G3BP knockdown cells compared to its replication in control cells (22).’ (score: 0.0)\n",
            "Document #549: ‘For instance, G3BP1 and G3BP2 synergistically regulate the formation of SGs in various human cells (54).’ (score: 0.0)\n",
            "Document #548: ‘Except for TIA-1/R, other cellular proteins are in¬volved in SG assembly.’ (score: 0.0)\n",
            "Document #547: ‘In addition, TIA-1/R interacts with West Nile virus RNA and facilitates virus replication (53).’ (score: 0.0)\n",
            "Document #546: ‘Both TIA-1 and TIAR are RNA binding proteins and function in a similar manner in translation inhibition and SG formation (1, 11, 39).’ (score: 0.0)\n",
            "Document #545: ‘The translational shutoff is delayed in TIA-12/2 murine embryonic fibroblasts (11), indicating TIA function on the inhibition, which is in accord with our results.’ (score: 0.0)\n",
            "Document #568: ‘At late times, the VHS RNase is neutralized by the viral structural proteins VP16 and VP22.’ (score: 0.0)\n",
            "Document #570: ‘The rates of degradation of ARE mRNA at early times (3 h) after infection with the wild type or the ICP27 mutant virus are virtually identical, and hence ICP27 plays no role in this process.’ (score: 0.0)\n",
            "Document #595: ‘In addition, this laboratory has shown that VHS is active and degrades mRNA in polyribo-somes (49).’ (score: 0.0)\n",
            "Document #571: ‘ In non complementing cells, VHS RNase or VP22 is not synthesized.’ (score: 0.0)\n",
            "Document #594: ‘The VHS protein was also reported to bind cap-binding proteins (9, 17, 18, 19, 31).’ (score: 0.0)\n",
            "Document #593: (i) its mRNA contains AREs (3), (ii) TTP physically interacts with VHS protein (15), and (iii) in uninfected cells, TTP binds AREs and sequesters the mRNAs for degradation by exo¬somes (5, 28).’ (score: 0.0)\n",
            "Document #592: ‘TTP is of particular interest for three reasons: (score: 0.0)\n",
            "Document #591: ‘Lastly, a subset of mRNAs exemplified by GADD45 and tristetraprolin (TTP) directed the synthesis of corresponding proteins and were not degraded (14).’ (score: 0.0)\n",
            "Document #590: ‘Whereas the 3 cleavage product rapidly disappeared, the 5 portion of the cleaved RNA lingered for many hours.’ (score: 0.0)\n",
            "Document #589: ‘In contrast, stress response mRNAs contain¬ing AU-rich elements (AREs) in the 3 untranslated region (3 UTR), exemplified by IEX-1, c-Fos, IkB, or cox-2, which in uninfected cells share very short half-lives (1, 4, 51), were deadenylated and cleaved at or near the AREs (14).’ (score: 0.0)\n",
            "Document #588: ‘Thus, mRNAs that are highly stable in uninfected cells, exemplified by GAPDH (glyceraldehyde-3-phosphate dehydrogenase) and actin mRNAs, were rapidly degraded.’ (score: 0.0)\n",
            "Document #587: ‘This laboratory was the first to report that the degradation of mRNA during the early hours after infection by wild-type virus was highly selective (14, 16, 45).’ (score: 0.0)\n",
            "Document #586: 1991 that VHS mediated the degradation of some mRNAs but spared other mRNAs, most notably, histone mRNAs, the issue was still debated as late as 2004 (reviewed in reference 39).’ (score: 0.0)\n",
            "Document #585: ‘Even though Soreson and colleagues (41) reported as early as (score: 0.0)\n",
            "Document #584: ‘Most of the early work was done on model viral mRNAs, and these studies sustained the conclusion that VHS activity was not selective.’ (score: 0.0)\n",
            "Document #583: ‘Studies on the incorporation of radioactive amino acids into proteins after infection in cells infected with wild-type virus but not with a mutant lacking VHS activity led to the speculation that VHS activity is nonselective (23, 29, 30, 36).’ (score: 0.0)\n",
            "Document #582: ‘Relevant to this report are the following.’ (score: 0.0)\n",
            "Document #581: ‘In this report, we show that VHS interacts with still another viral protein, ICP27, and that this protein is required to block degradation of specific mRNAs at times when ICP27 becomes available for this func¬tion.’ (score: 0.0)\n",
            "Document #580: ‘More recent studies from this laboratory showed that an additional late viral protein, VP22, is required for the neutralization of the RNase activity of VHS protein (48).’ (score: 0.0)\n",
            "Document #579: ‘Initial reports indicated that the neutralizing protein is VP16 (27, 38).’ (score: 0.0)\n",
            "Document #578: ‘The VHS RNase is active during the first several hours after infection, and then it is neutralized by proteins made late in infection.’ (score: 0.0)\n",
            "Document #577: ‘Although it was long sus¬pected that VHS is an RNase (11, 12, 22, 32, 52), unambiguous evidence that VHS is an endoribonuclease with a substrate specificity of RNase A emerged from studies of a protein purified to homogeneity (46, 47).’ (score: 0.0)\n",
            "Document #576: ‘The herpes simplex virus type 1 (HSV-1) UL41 gene encod¬ing a protein (virion host shutoff [VHS]) that is carried by the virion into the infected cell and causes the shutoff of host protein synthesis was first reported by Frenkel and co workers in the late 1980s (24, 25, 26, 43).’ (score: 0.0)\n",
            "Document #575: ‘(iii) Concurrently, we present the first evidence that VHS RNase interacts with ICP27 most likely when bound to cap-and poly(A)-binding proteins, respectively.’ (score: 0.0)\n",
            "Document #574: ‘Hence the rapid degradation of ARE mRNAs noted in ICP27 mutant-infected cells at late times is similar to that taking place in mock-infected or in VHS RNase mutant-virus-infected cells and does not by itself support the hypothesis that ICP27 stabilizes ARE mRNAs.’ (score: 0.0)\n",
            "Document #573: ‘The VHS RNase brought into the cells by the ICP27 virus is reduced in potency relative to that of wild-type virus.’ (score: 0.0)\n",
            "Document #572: ‘Therefore, the only VHS that is active is brought into cells by the ICP27 mutant.’ (score: 0.0)\n",
            "Document #0: 'G-quadruplexes (G4s) are non-canonical nucleic acids secondary structures that form within guanine-rich strands of regulatory genomic regions.’ (score: 0.0)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/ipykernel/ipkernel.py:283: DeprecationWarning: `should_run_async` will not call `transform_cell` automatically in the future. Please pass the result to `transformed_cell` argument and any exception that happen during thetransform in `preprocessing_exc_tuple` in IPython 7.17 and above.\n",
            "  and should_run_async(code)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from sklearn.feature_extraction.text import TfidfVectorizer\n",
        "from sklearn.metrics.pairwise import cosine_similarity\n",
        "\n",
        "# This vectorizer will convert our documents into a matrix of TF-IDF features\n",
        "vectorizer = TfidfVectorizer()\n",
        "\n",
        "# Vectorize our documents\n",
        "tfidf_matrix = vectorizer.fit_transform(corpus)  # Change 'documents' to 'corpus'\n",
        "\n",
        "def search(query):\n",
        "    # Vectorize the query to the same feature space as our documents\n",
        "    query_vec = vectorizer.transform([query])\n",
        "\n",
        "    # Compute the cosine similarity between the query and all documents\n",
        "    sims = cosine_similarity(query_vec, tfidf_matrix).flatten()\n",
        "\n",
        "    # Sort the document indices by their similarity score\n",
        "    ranking = sims.argsort()[::-1]\n",
        "\n",
        "    # Print out the ranking\n",
        "    for i in range(len(ranking)):\n",
        "        print(f\"Document #{ranking[i]}: {corpus[ranking[i]]} (score: {sims[ranking[i]]})\")  # Change 'documents' to 'corpus'\n",
        "\n",
        "# Let's try it out\n",
        "search(\"HSP90\")\n"
      ],
      "metadata": {
        "id": "VnK0U9XlTLIS"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}